{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pybel.examples import sialic_acid_graph\n",
    "from pybel_tools.visualization import to_jupyter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"yemravkuxobqfpjt\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"name\": \"Sialic Acid Graph\", \"version\": \"1.0.0\", \"description\": \"The downstream effects of sialic acid in immune signaling\", \"authors\": \"Charles Tapley Hoyt\", \"contact\": \"charles.hoyt@scai.fraunhofer.de\"}, \"pybel_version\": \"0.9.8-dev\", \"namespace_url\": {\"HGNC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/hgnc-human-genes/hgnc-human-genes-20170725.belns\", \"CHEBI\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/chebi/chebi-20170725.belns\", \"GOBP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-biological-process/go-biological-process-20170725.belns\"}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {\"Confidence\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/confidence/confidence-1.0.0.belanno\", \"Species\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/species-taxonomy-id/species-taxonomy-id-20170511.belanno\"}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}, \"namespaces_uncached\": []}, \"nodes\": [{\"function\": \"Complex\", \"members\": [{\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"sialic acid\", \"identifier\": \"26667\", \"cname\": \"sialic acid\"}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CD33\", \"identifier\": \"1659\", \"cname\": \"CD33\"}], \"cname\": \"complex(a(CHEBI:\\\"sialic acid\\\"), p(HGNC:CD33))\", \"id\": [\"Complex\", [\"Abundance\", \"CHEBI\", \"sialic acid\"], [\"Protein\", \"HGNC\", \"CD33\"]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"sialic acid\", \"identifier\": \"26667\", \"cname\": \"sialic acid\", \"id\": [\"Abundance\", \"CHEBI\", \"sialic acid\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CD33\", \"identifier\": \"1659\", \"cname\": \"CD33\", \"id\": [\"Protein\", \"HGNC\", \"CD33\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CD33\", \"identifier\": \"1659\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"cname\": \"p(HGNC:CD33, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"CD33\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PTPN6\", \"identifier\": \"9658\", \"cname\": \"PTPN6\", \"id\": [\"Protein\", \"HGNC\", \"PTPN6\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PTPN11\", \"identifier\": \"9644\", \"cname\": \"PTPN11\", \"id\": [\"Protein\", \"HGNC\", \"PTPN11\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SYK\", \"identifier\": \"11491\", \"cname\": \"SYK\", \"id\": [\"Protein\", \"HGNC\", \"SYK\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"identifier\": \"17761\", \"cname\": \"TREM2\", \"id\": [\"Protein\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TYROBP\", \"identifier\": \"12449\", \"cname\": \"TYROBP\", \"id\": [\"Protein\", \"HGNC\", \"TYROBP\"]}], \"links\": [{\"relation\": \"hasComponent\", \"source\": 0, \"target\": 1, \"key\": -3}, {\"relation\": \"hasComponent\", \"source\": 0, \"target\": 2, \"key\": -3}, {\"relation\": \"increases\", \"evidence\": \"Sialic acid binding activates CD33, resulting in phosphorylation of the CD33 immunoreceptor tyrosine-based inhibitory motif (ITIM) domains and activation of the SHP-1 and SHP-2 tyrosine phosphatases [66, 67].\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"object\": {\"modifier\": \"Activity\"}, \"source\": 0, \"target\": 2, \"key\": 1}, {\"relation\": \"hasVariant\", \"source\": 2, \"target\": 3, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Sialic acid binding activates CD33, resulting in phosphorylation of the CD33 immunoreceptor tyrosine-based inhibitory motif (ITIM) domains and activation of the SHP-1 and SHP-2 tyrosine phosphatases [66, 67].\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"subject\": {\"modifier\": \"Activity\"}, \"source\": 2, \"target\": 3, \"key\": 1}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Sialic acid binding activates CD33, resulting in phosphorylation of the CD33 immunoreceptor tyrosine-based inhibitory motif (ITIM) domains and activation of the SHP-1 and SHP-2 tyrosine phosphatases [66, 67].\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"phos\"}}, \"source\": 3, \"target\": 4, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"evidence\": \"Sialic acid binding activates CD33, resulting in phosphorylation of the CD33 immunoreceptor tyrosine-based inhibitory motif (ITIM) domains and activation of the SHP-1 and SHP-2 tyrosine phosphatases [66, 67].\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"phos\"}}, \"source\": 3, \"target\": 5, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"These phosphatases act on multiple substrates, including Syk, to inhibit immune activation [68, 69].  Hence, CD33 activation leads to increased SHP-1 and SHP-2 activity that antagonizes Syk, inhibiting ITAM-signaling  proteins, possibly including TREM2/DAP12 (Fig. 1, [70, 71]).\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"directlyDecreases\", \"evidence\": \"These phosphatases act on multiple substrates, including Syk, to inhibit immune activation [68, 69].  Hence, CD33 activation leads to increased SHP-1 and SHP-2 activity that antagonizes Syk, inhibiting ITAM-signaling  proteins, possibly including TREM2/DAP12 (Fig. 1, [70, 71]).\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"High\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"source\": 5, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These phosphatases act on multiple substrates, including Syk, to inhibit immune activation [68, 69].  Hence, CD33 activation leads to increased SHP-1 and SHP-2 activity that antagonizes Syk, inhibiting ITAM-signaling  proteins, possibly including TREM2/DAP12 (Fig. 1, [70, 71]).\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"Low\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"source\": 6, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These phosphatases act on multiple substrates, including Syk, to inhibit immune activation [68, 69].  Hence, CD33 activation leads to increased SHP-1 and SHP-2 activity that antagonizes Syk, inhibiting ITAM-signaling  proteins, possibly including TREM2/DAP12 (Fig. 1, [70, 71]).\", \"citation\": {\"type\": \"PubMed\", \"reference\": \"26438529\"}, \"annotations\": {\"Species\": \"9606\", \"Confidence\": \"Low\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"source\": 6, \"target\": 8, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(sialic_acid_graph)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
