{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Abstract\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt)\n",
    "\n",
    "**Estimated Run Time:** 4 minutes\n",
    "\n",
    "This notebooks demonstrates the usage of a variant  the [Network Perturbation Amplitude](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-54) algorithm to analyze an BEL knowledge assembly in the context of a differential gene expression experiment. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import logging\n",
    "import os\n",
    "import sys\n",
    "import time\n",
    "\n",
    "import numpy as np\n",
    "\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "from pandas.tools.plotting import scatter_matrix\n",
    "from matplotlib_venn import venn2\n",
    "from tqdm import tqdm\n",
    "\n",
    "import pybel\n",
    "import pybel_tools\n",
    "from pybel_tools.analysis.heat import Runner, multirun, workflow_aggregate\n",
    "from pybel.constants import *\n",
    "from pybel.canonicalize import calculate_canonical_name\n",
    "from pybel_tools.visualization import to_jupyter\n",
    "from pybel_tools.integration import overlay_type_data\n",
    "from pybel_tools.summary import print_summary\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "%config InlineBackend.figure_format = 'svg'\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Provenance\n",
    "\n",
    "The time of execution, random number generator seed, and the versions of the software packages used are displayed explicitly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "3.6.5 (default, Apr 25 2018, 14:23:58) \n",
      "[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.1)]\n"
     ]
    }
   ],
   "source": [
    "print(sys.version)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Wed Jun 20 14:46:32 2018\n"
     ]
    }
   ],
   "source": [
    "print(time.asctime())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.11.4-dev'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.utils.get_version()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.5.3-dev'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel_tools.utils.get_version()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# seed the random number generator\n",
    "import random\n",
    "\n",
    "random.seed(127)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Local Path Definitions\n",
    "\n",
    "To make this notebook interoperable across many machines, locations to the repositories that contain the data used in this notebook are referenced from the environment, set in `~/.bashrc` to point to the place where the repositories have been cloned. Assuming the repositories have been `git clone`'d into the `~/dev` folder, the entries in `~/.bashrc` should look like:\n",
    "\n",
    "```bash\n",
    "...\n",
    "export BMS_BASE=~/dev/bms\n",
    "...\n",
    "```\n",
    "\n",
    "#### BMS \n",
    "\n",
    "The biological model store (BMS) is the internal Fraunhofer SCAI repository for keeping BEL models under version control. It can be downloaded from https://tor-2.scai.fraunhofer.de/gf/project/bms/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "bms_base = os.environ[\"BMS_BASE\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### OwnCloud\n",
    "\n",
    "The differential gene expression data used in this notebook is currently not published, and is obfuscated with reference through our team's internal data storage system with [OwnCloud](https://owncloud.org/)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "owncloud_base = os.environ[\"OWNCLOUD_BASE\"]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data\n",
    "\n",
    "## Alzheimer's Disease Knowledge Assembly\n",
    "\n",
    "The Alzheimer's Disease Knowledge Assembly has been precompiled with the following command line script, and will be loaded from this format for improved performance. In general, derived data, such as the gpickle representation of a BEL script, are not saved under version control to ensure that the most up-to-date data is always used.\n",
    "\n",
    "```sh\n",
    "pybel convert --path \"$BMS_BASE/aetionomy/alzheimers.bel\" --pickle \"$BMS_BASE/aetionomy/alzheimers.gpickle\"\n",
    "```\n",
    "\n",
    "The BEL script can also be compiled from inside this notebook with the following python code:\n",
    "\n",
    "```python\n",
    ">>> import os\n",
    ">>> import pybel\n",
    ">>> # Input from BEL script\n",
    ">>> bel_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.bel')\n",
    ">>> graph = pybel.from_path(bel_path)\n",
    ">>> # Output to gpickle for fast loading later\n",
    ">>> pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')\n",
    ">>> pybel.to_pickle(graph, pickle_path)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "pickle_path = os.path.join(bms_base, \"aetionomy\", \"alzheimers\", \"alzheimers.gpickle\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "graph = pybel.from_pickle(pickle_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Alzheimer's Disease Knowledge Assembly v5.0.5\n"
     ]
    }
   ],
   "source": [
    "print(graph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "All orthologies are discared before analysis."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.filters.remove_nodes_by_namespace(graph, \"MGI\")\n",
    "pybel_tools.filters.remove_nodes_by_namespace(graph, \"RGD\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To merge the differential gene expression annotations, the entire graph is collapsed to genes using [pybel_tools.mutation.collapse_by_central_dogma_to_genes](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.collapse_by_central_dogma_to_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.mutation.collapse_by_central_dogma_to_genes(graph)\n",
    "pybel_tools.mutation.rewire_variants_to_genes(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 4007\n",
      "Edges: 15140\n",
      "Citations: 1816\n",
      "Authors: 0\n",
      "Network density: 0.0009431821820550931\n",
      "Components: 73\n",
      "Average degree: 3.7783878213127027\n",
      "Compilation warnings: 1435\n"
     ]
    }
   ],
   "source": [
    "print_summary(graph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Differential Gene Expression\n",
    "\n",
    "Differential gene expression data can be obtained from many sources, including ADNI and other large clinical studies. This analysis is concerned with the log-fold-changes on each gene, and not necessarily the p-value. This is better as a data-driven process becuase it does not require a model or multiple hypothesis testing on raw data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "data_path = os.path.join(owncloud_base, \"projects\", \"alzheimers\", \"SevAD.csv\")\n",
    "\n",
    "assert os.path.exists(data_path), \"Data does not exist!\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv(data_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "target_columns = [\"Gene.symbol\", \"logFC\"]\n",
    "\n",
    "# Check that the columns are named properly\n",
    "for target_column in target_columns:\n",
    "    assert target_column in df.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "# throw away columns that don't have gene symbols - these represent control sequences\n",
    "df = df.loc[df[\"Gene.symbol\"].notnull(), target_columns]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene.symbol</th>\n",
       "      <th>logFC</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ZNF616</td>\n",
       "      <td>-4.244691</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DEFB125</td>\n",
       "      <td>3.974393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SNAP23</td>\n",
       "      <td>3.337636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PHLDB2</td>\n",
       "      <td>3.192559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LOC389895</td>\n",
       "      <td>-4.296850</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Gene.symbol     logFC\n",
       "0      ZNF616 -4.244691\n",
       "1     DEFB125  3.974393\n",
       "3      SNAP23  3.337636\n",
       "4      PHLDB2  3.192559\n",
       "5   LOC389895 -4.296850"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# show a summary\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Summary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "On the left, the number of shared elements in the knowledge assembly and differential gene data set are counted.\n",
    "\n",
    "On the right, a histogram of the log-fold-changes shows that the data are normally distributed, as expected for differential gene expression data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Get all HGNC symbols from the BEL graph\n",
    "hgnc_names = pybel.struct.summary.get_names_by_namespace(graph, \"HGNC\")\n",
    "\n",
    "# Get all HGNC symbols from the differential gene expression data set\n",
    "df_names = set(df[\"Gene.symbol\"])\n",
    "\n",
    "# Calculate the intersection of the two gene symbol lists\n",
    "overlapping_hgnc_names = hgnc_names & df_names"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "# The function pybel_tools.integration.overlay_type_data requires a dictionary from gene symbol to log fold changes\n",
    "data = {gene_symbol: log_fc for _, gene_symbol, log_fc in df.itertuples()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"207pt\" version=\"1.1\" viewBox=\"0 0 718 207\" width=\"718pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 207.274375 \n",
       "L 718.916163 207.274375 \n",
       "L 718.916163 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path clip-path=\"url(#p02e85ee312)\" d=\"M 135.461352 107.330274 \n",
       "C 134.775547 103.598605 134.430513 99.812289 134.430513 96.018125 \n",
       "C 134.430513 92.223961 134.775547 88.437645 135.461352 84.705976 \n",
       "C 132.634759 87.793904 131.065705 91.831846 131.065705 96.018125 \n",
       "C 131.065705 100.204404 132.634759 104.242346 135.461352 107.330274 \n",
       "\" style=\"fill:#ff0000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p02e85ee312)\" d=\"M 135.461352 84.705976 \n",
       "C 136.594851 83.467679 137.908057 82.406754 139.356908 81.558799 \n",
       "C 140.805759 80.710845 142.373866 80.085452 144.008611 79.703606 \n",
       "C 145.643355 79.321761 147.326247 79.187781 149.000816 79.306164 \n",
       "C 150.675385 79.424546 152.32269 79.793952 153.887456 80.401985 \n",
       "C 155.452222 81.010018 156.916749 81.8498 158.231895 82.893154 \n",
       "C 159.547041 83.936508 160.697931 85.17163 161.645945 86.557078 \n",
       "C 162.593959 87.942526 163.328375 89.462627 163.82455 91.066375 \n",
       "C 164.320724 92.670123 164.573045 94.339377 164.573045 96.018125 \n",
       "C 164.573045 97.696873 164.320724 99.366127 163.82455 100.969875 \n",
       "C 163.328375 102.573623 162.593959 104.093724 161.645945 105.479172 \n",
       "C 160.697931 106.86462 159.547041 108.099742 158.231895 109.143096 \n",
       "C 156.916749 110.18645 155.452222 111.026232 153.887456 111.634265 \n",
       "C 152.32269 112.242298 150.675385 112.611704 149.000816 112.730086 \n",
       "C 147.326247 112.848469 145.643355 112.714489 144.008611 112.332644 \n",
       "C 142.373866 111.950798 140.805759 111.325405 139.356908 110.477451 \n",
       "C 137.908057 109.629496 136.594851 108.568571 135.461352 107.330274 \n",
       "C 136.860409 114.942958 139.659677 122.230454 143.716789 128.822128 \n",
       "C 147.7739 135.413801 153.01889 141.195978 159.185005 145.874584 \n",
       "C 165.351119 150.55319 172.332021 154.047541 179.772672 156.179919 \n",
       "C 187.213323 158.312297 194.985406 159.045928 202.693643 158.343502 \n",
       "C 210.401879 157.641075 217.91334 155.514704 224.846 152.072532 \n",
       "C 231.77866 148.630361 238.012964 143.931751 243.231844 138.215667 \n",
       "C 248.450724 132.499583 252.564178 125.864602 255.363001 118.648169 \n",
       "C 258.161825 111.431736 259.597752 103.758301 259.597752 96.018125 \n",
       "C 259.597752 88.277949 258.161825 80.604514 255.363001 73.388081 \n",
       "C 252.564178 66.171648 248.450724 59.536667 243.231844 53.820583 \n",
       "C 238.012964 48.104499 231.77866 43.405889 224.846 39.963718 \n",
       "C 217.91334 36.521546 210.401879 34.395175 202.693643 33.692748 \n",
       "C 194.985406 32.990322 187.213323 33.723953 179.772672 35.856331 \n",
       "C 172.332021 37.988709 165.351119 41.48306 159.185005 46.161666 \n",
       "C 153.01889 50.840272 147.7739 56.622449 143.716789 63.214122 \n",
       "C 139.659677 69.805796 136.860409 77.093292 135.461352 84.705976 \n",
       "\" style=\"fill:#008000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p02e85ee312)\" d=\"M 135.461352 107.330274 \n",
       "C 136.594851 108.568571 137.908057 109.629496 139.356908 110.477451 \n",
       "C 140.805759 111.325405 142.373866 111.950798 144.008611 112.332644 \n",
       "C 145.643355 112.714489 147.326247 112.848469 149.000816 112.730086 \n",
       "C 150.675385 112.611704 152.32269 112.242298 153.887456 111.634265 \n",
       "C 155.452222 111.026232 156.916749 110.18645 158.231895 109.143096 \n",
       "C 159.547041 108.099742 160.697931 106.86462 161.645945 105.479172 \n",
       "C 162.593959 104.093724 163.328375 102.573623 163.82455 100.969875 \n",
       "C 164.320724 99.366127 164.573045 97.696873 164.573045 96.018125 \n",
       "C 164.573045 94.339377 164.320724 92.670123 163.82455 91.066375 \n",
       "C 163.328375 89.462627 162.593959 87.942526 161.645945 86.557078 \n",
       "C 160.697931 85.17163 159.547041 83.936508 158.231895 82.893154 \n",
       "C 156.916749 81.8498 155.452222 81.010018 153.887456 80.401985 \n",
       "C 152.32269 79.793952 150.675385 79.424546 149.000816 79.306164 \n",
       "C 147.326247 79.187781 145.643355 79.321761 144.008611 79.703606 \n",
       "C 142.373866 80.085452 140.805759 80.710845 139.356908 81.558799 \n",
       "C 137.908057 82.406754 136.594851 83.467679 135.461352 84.705976 \n",
       "C 134.775547 88.437645 134.430513 92.223961 134.430513 96.018125 \n",
       "C 134.430513 99.812289 134.775547 103.598605 135.461352 107.330274 \n",
       "\" style=\"fill:#b25900;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_1\">\n",
       "    <!-- 95 -->\n",
       "    <defs>\n",
       "     <path d=\"M 10.984375 1.515625 \n",
       "L 10.984375 10.5 \n",
       "Q 14.703125 8.734375 18.5 7.8125 \n",
       "Q 22.3125 6.890625 25.984375 6.890625 \n",
       "Q 35.75 6.890625 40.890625 13.453125 \n",
       "Q 46.046875 20.015625 46.78125 33.40625 \n",
       "Q 43.953125 29.203125 39.59375 26.953125 \n",
       "Q 35.25 24.703125 29.984375 24.703125 \n",
       "Q 19.046875 24.703125 12.671875 31.3125 \n",
       "Q 6.296875 37.9375 6.296875 49.421875 \n",
       "Q 6.296875 60.640625 12.9375 67.421875 \n",
       "Q 19.578125 74.21875 30.609375 74.21875 \n",
       "Q 43.265625 74.21875 49.921875 64.515625 \n",
       "Q 56.59375 54.828125 56.59375 36.375 \n",
       "Q 56.59375 19.140625 48.40625 8.859375 \n",
       "Q 40.234375 -1.421875 26.421875 -1.421875 \n",
       "Q 22.703125 -1.421875 18.890625 -0.6875 \n",
       "Q 15.09375 0.046875 10.984375 1.515625 \n",
       "z\n",
       "M 30.609375 32.421875 \n",
       "Q 37.25 32.421875 41.125 36.953125 \n",
       "Q 45.015625 41.5 45.015625 49.421875 \n",
       "Q 45.015625 57.28125 41.125 61.84375 \n",
       "Q 37.25 66.40625 30.609375 66.40625 \n",
       "Q 23.96875 66.40625 20.09375 61.84375 \n",
       "Q 16.21875 57.28125 16.21875 49.421875 \n",
       "Q 16.21875 41.5 20.09375 36.953125 \n",
       "Q 23.96875 32.421875 30.609375 32.421875 \n",
       "z\n",
       "\" id=\"DejaVuSans-39\"/>\n",
       "     <path d=\"M 10.796875 72.90625 \n",
       "L 49.515625 72.90625 \n",
       "L 49.515625 64.59375 \n",
       "L 19.828125 64.59375 \n",
       "L 19.828125 46.734375 \n",
       "Q 21.96875 47.46875 24.109375 47.828125 \n",
       "Q 26.265625 48.1875 28.421875 48.1875 \n",
       "Q 40.625 48.1875 47.75 41.5 \n",
       "Q 54.890625 34.8125 54.890625 23.390625 \n",
       "Q 54.890625 11.625 47.5625 5.09375 \n",
       "Q 40.234375 -1.421875 26.90625 -1.421875 \n",
       "Q 22.3125 -1.421875 17.546875 -0.640625 \n",
       "Q 12.796875 0.140625 7.71875 1.703125 \n",
       "L 7.71875 11.625 \n",
       "Q 12.109375 9.234375 16.796875 8.0625 \n",
       "Q 21.484375 6.890625 26.703125 6.890625 \n",
       "Q 35.15625 6.890625 40.078125 11.328125 \n",
       "Q 45.015625 15.765625 45.015625 23.390625 \n",
       "Q 45.015625 31 40.078125 35.4375 \n",
       "Q 35.15625 39.890625 26.703125 39.890625 \n",
       "Q 22.75 39.890625 18.8125 39.015625 \n",
       "Q 14.890625 38.140625 10.796875 36.28125 \n",
       "z\n",
       "\" id=\"DejaVuSans-35\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(126.385609 98.7775)scale(0.1 -0.1)\">\n",
       "     <use xlink:href=\"#DejaVuSans-39\"/>\n",
       "     <use x=\"63.623047\" xlink:href=\"#DejaVuSans-35\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_2\">\n",
       "    <!-- 20692 -->\n",
       "    <defs>\n",
       "     <path d=\"M 19.1875 8.296875 \n",
       "L 53.609375 8.296875 \n",
       "L 53.609375 0 \n",
       "L 7.328125 0 \n",
       "L 7.328125 8.296875 \n",
       "Q 12.9375 14.109375 22.625 23.890625 \n",
       "Q 32.328125 33.6875 34.8125 36.53125 \n",
       "Q 39.546875 41.84375 41.421875 45.53125 \n",
       "Q 43.3125 49.21875 43.3125 52.78125 \n",
       "Q 43.3125 58.59375 39.234375 62.25 \n",
       "Q 35.15625 65.921875 28.609375 65.921875 \n",
       "Q 23.96875 65.921875 18.8125 64.3125 \n",
       "Q 13.671875 62.703125 7.8125 59.421875 \n",
       "L 7.8125 69.390625 \n",
       "Q 13.765625 71.78125 18.9375 73 \n",
       "Q 24.125 74.21875 28.421875 74.21875 \n",
       "Q 39.75 74.21875 46.484375 68.546875 \n",
       "Q 53.21875 62.890625 53.21875 53.421875 \n",
       "Q 53.21875 48.921875 51.53125 44.890625 \n",
       "Q 49.859375 40.875 45.40625 35.40625 \n",
       "Q 44.1875 33.984375 37.640625 27.21875 \n",
       "Q 31.109375 20.453125 19.1875 8.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-32\"/>\n",
       "     <path d=\"M 31.78125 66.40625 \n",
       "Q 24.171875 66.40625 20.328125 58.90625 \n",
       "Q 16.5 51.421875 16.5 36.375 \n",
       "Q 16.5 21.390625 20.328125 13.890625 \n",
       "Q 24.171875 6.390625 31.78125 6.390625 \n",
       "Q 39.453125 6.390625 43.28125 13.890625 \n",
       "Q 47.125 21.390625 47.125 36.375 \n",
       "Q 47.125 51.421875 43.28125 58.90625 \n",
       "Q 39.453125 66.40625 31.78125 66.40625 \n",
       "z\n",
       "M 31.78125 74.21875 \n",
       "Q 44.046875 74.21875 50.515625 64.515625 \n",
       "Q 56.984375 54.828125 56.984375 36.375 \n",
       "Q 56.984375 17.96875 50.515625 8.265625 \n",
       "Q 44.046875 -1.421875 31.78125 -1.421875 \n",
       "Q 19.53125 -1.421875 13.0625 8.265625 \n",
       "Q 6.59375 17.96875 6.59375 36.375 \n",
       "Q 6.59375 54.828125 13.0625 64.515625 \n",
       "Q 19.53125 74.21875 31.78125 74.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-30\"/>\n",
       "     <path d=\"M 33.015625 40.375 \n",
       "Q 26.375 40.375 22.484375 35.828125 \n",
       "Q 18.609375 31.296875 18.609375 23.390625 \n",
       "Q 18.609375 15.53125 22.484375 10.953125 \n",
       "Q 26.375 6.390625 33.015625 6.390625 \n",
       "Q 39.65625 6.390625 43.53125 10.953125 \n",
       "Q 47.40625 15.53125 47.40625 23.390625 \n",
       "Q 47.40625 31.296875 43.53125 35.828125 \n",
       "Q 39.65625 40.375 33.015625 40.375 \n",
       "z\n",
       "M 52.59375 71.296875 \n",
       "L 52.59375 62.3125 \n",
       "Q 48.875 64.0625 45.09375 64.984375 \n",
       "Q 41.3125 65.921875 37.59375 65.921875 \n",
       "Q 27.828125 65.921875 22.671875 59.328125 \n",
       "Q 17.53125 52.734375 16.796875 39.40625 \n",
       "Q 19.671875 43.65625 24.015625 45.921875 \n",
       "Q 28.375 48.1875 33.59375 48.1875 \n",
       "Q 44.578125 48.1875 50.953125 41.515625 \n",
       "Q 57.328125 34.859375 57.328125 23.390625 \n",
       "Q 57.328125 12.15625 50.6875 5.359375 \n",
       "Q 44.046875 -1.421875 33.015625 -1.421875 \n",
       "Q 20.359375 -1.421875 13.671875 8.265625 \n",
       "Q 6.984375 17.96875 6.984375 36.375 \n",
       "Q 6.984375 53.65625 15.1875 63.9375 \n",
       "Q 23.390625 74.21875 37.203125 74.21875 \n",
       "Q 40.921875 74.21875 44.703125 73.484375 \n",
       "Q 48.484375 72.75 52.59375 71.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-36\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(196.179148 98.7775)scale(0.1 -0.1)\">\n",
       "     <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "     <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "     <use x=\"127.246094\" xlink:href=\"#DejaVuSans-36\"/>\n",
       "     <use x=\"190.869141\" xlink:href=\"#DejaVuSans-39\"/>\n",
       "     <use x=\"254.492188\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_3\">\n",
       "    <!-- 1495 -->\n",
       "    <defs>\n",
       "     <path d=\"M 12.40625 8.296875 \n",
       "L 28.515625 8.296875 \n",
       "L 28.515625 63.921875 \n",
       "L 10.984375 60.40625 \n",
       "L 10.984375 69.390625 \n",
       "L 28.421875 72.90625 \n",
       "L 38.28125 72.90625 \n",
       "L 38.28125 8.296875 \n",
       "L 54.390625 8.296875 \n",
       "L 54.390625 0 \n",
       "L 12.40625 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-31\"/>\n",
       "     <path d=\"M 37.796875 64.3125 \n",
       "L 12.890625 25.390625 \n",
       "L 37.796875 25.390625 \n",
       "z\n",
       "M 35.203125 72.90625 \n",
       "L 47.609375 72.90625 \n",
       "L 47.609375 25.390625 \n",
       "L 58.015625 25.390625 \n",
       "L 58.015625 17.1875 \n",
       "L 47.609375 17.1875 \n",
       "L 47.609375 0 \n",
       "L 37.796875 0 \n",
       "L 37.796875 17.1875 \n",
       "L 4.890625 17.1875 \n",
       "L 4.890625 26.703125 \n",
       "z\n",
       "\" id=\"DejaVuSans-34\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(136.776779 98.7775)scale(0.1 -0.1)\">\n",
       "     <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "     <use x=\"63.623047\" xlink:href=\"#DejaVuSans-34\"/>\n",
       "     <use x=\"127.246094\" xlink:href=\"#DejaVuSans-39\"/>\n",
       "     <use x=\"190.869141\" xlink:href=\"#DejaVuSans-35\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_4\">\n",
       "    <!-- Alzheimer's Disease KA -->\n",
       "    <defs>\n",
       "     <path d=\"M 34.1875 63.1875 \n",
       "L 20.796875 26.90625 \n",
       "L 47.609375 26.90625 \n",
       "z\n",
       "M 28.609375 72.90625 \n",
       "L 39.796875 72.90625 \n",
       "L 67.578125 0 \n",
       "L 57.328125 0 \n",
       "L 50.6875 18.703125 \n",
       "L 17.828125 18.703125 \n",
       "L 11.1875 0 \n",
       "L 0.78125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-41\"/>\n",
       "     <path d=\"M 9.421875 75.984375 \n",
       "L 18.40625 75.984375 \n",
       "L 18.40625 0 \n",
       "L 9.421875 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-6c\"/>\n",
       "     <path d=\"M 5.515625 54.6875 \n",
       "L 48.1875 54.6875 \n",
       "L 48.1875 46.484375 \n",
       "L 14.40625 7.171875 \n",
       "L 48.1875 7.171875 \n",
       "L 48.1875 0 \n",
       "L 4.296875 0 \n",
       "L 4.296875 8.203125 \n",
       "L 38.09375 47.515625 \n",
       "L 5.515625 47.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-7a\"/>\n",
       "     <path d=\"M 54.890625 33.015625 \n",
       "L 54.890625 0 \n",
       "L 45.90625 0 \n",
       "L 45.90625 32.71875 \n",
       "Q 45.90625 40.484375 42.875 44.328125 \n",
       "Q 39.84375 48.1875 33.796875 48.1875 \n",
       "Q 26.515625 48.1875 22.3125 43.546875 \n",
       "Q 18.109375 38.921875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 75.984375 \n",
       "L 18.109375 75.984375 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.34375 51.125 25.703125 53.5625 \n",
       "Q 30.078125 56 35.796875 56 \n",
       "Q 45.21875 56 50.046875 50.171875 \n",
       "Q 54.890625 44.34375 54.890625 33.015625 \n",
       "z\n",
       "\" id=\"DejaVuSans-68\"/>\n",
       "     <path d=\"M 56.203125 29.59375 \n",
       "L 56.203125 25.203125 \n",
       "L 14.890625 25.203125 \n",
       "Q 15.484375 15.921875 20.484375 11.0625 \n",
       "Q 25.484375 6.203125 34.421875 6.203125 \n",
       "Q 39.59375 6.203125 44.453125 7.46875 \n",
       "Q 49.3125 8.734375 54.109375 11.28125 \n",
       "L 54.109375 2.78125 \n",
       "Q 49.265625 0.734375 44.1875 -0.34375 \n",
       "Q 39.109375 -1.421875 33.890625 -1.421875 \n",
       "Q 20.796875 -1.421875 13.15625 6.1875 \n",
       "Q 5.515625 13.8125 5.515625 26.8125 \n",
       "Q 5.515625 40.234375 12.765625 48.109375 \n",
       "Q 20.015625 56 32.328125 56 \n",
       "Q 43.359375 56 49.78125 48.890625 \n",
       "Q 56.203125 41.796875 56.203125 29.59375 \n",
       "z\n",
       "M 47.21875 32.234375 \n",
       "Q 47.125 39.59375 43.09375 43.984375 \n",
       "Q 39.0625 48.390625 32.421875 48.390625 \n",
       "Q 24.90625 48.390625 20.390625 44.140625 \n",
       "Q 15.875 39.890625 15.1875 32.171875 \n",
       "z\n",
       "\" id=\"DejaVuSans-65\"/>\n",
       "     <path d=\"M 9.421875 54.6875 \n",
       "L 18.40625 54.6875 \n",
       "L 18.40625 0 \n",
       "L 9.421875 0 \n",
       "z\n",
       "M 9.421875 75.984375 \n",
       "L 18.40625 75.984375 \n",
       "L 18.40625 64.59375 \n",
       "L 9.421875 64.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-69\"/>\n",
       "     <path d=\"M 52 44.1875 \n",
       "Q 55.375 50.25 60.0625 53.125 \n",
       "Q 64.75 56 71.09375 56 \n",
       "Q 79.640625 56 84.28125 50.015625 \n",
       "Q 88.921875 44.046875 88.921875 33.015625 \n",
       "L 88.921875 0 \n",
       "L 79.890625 0 \n",
       "L 79.890625 32.71875 \n",
       "Q 79.890625 40.578125 77.09375 44.375 \n",
       "Q 74.3125 48.1875 68.609375 48.1875 \n",
       "Q 61.625 48.1875 57.5625 43.546875 \n",
       "Q 53.515625 38.921875 53.515625 30.90625 \n",
       "L 53.515625 0 \n",
       "L 44.484375 0 \n",
       "L 44.484375 32.71875 \n",
       "Q 44.484375 40.625 41.703125 44.40625 \n",
       "Q 38.921875 48.1875 33.109375 48.1875 \n",
       "Q 26.21875 48.1875 22.15625 43.53125 \n",
       "Q 18.109375 38.875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.1875 51.21875 25.484375 53.609375 \n",
       "Q 29.78125 56 35.6875 56 \n",
       "Q 41.65625 56 45.828125 52.96875 \n",
       "Q 50 49.953125 52 44.1875 \n",
       "z\n",
       "\" id=\"DejaVuSans-6d\"/>\n",
       "     <path d=\"M 41.109375 46.296875 \n",
       "Q 39.59375 47.171875 37.8125 47.578125 \n",
       "Q 36.03125 48 33.890625 48 \n",
       "Q 26.265625 48 22.1875 43.046875 \n",
       "Q 18.109375 38.09375 18.109375 28.8125 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 20.953125 51.171875 25.484375 53.578125 \n",
       "Q 30.03125 56 36.53125 56 \n",
       "Q 37.453125 56 38.578125 55.875 \n",
       "Q 39.703125 55.765625 41.0625 55.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-72\"/>\n",
       "     <path d=\"M 17.921875 72.90625 \n",
       "L 17.921875 45.796875 \n",
       "L 9.625 45.796875 \n",
       "L 9.625 72.90625 \n",
       "z\n",
       "\" id=\"DejaVuSans-27\"/>\n",
       "     <path d=\"M 44.28125 53.078125 \n",
       "L 44.28125 44.578125 \n",
       "Q 40.484375 46.53125 36.375 47.5 \n",
       "Q 32.28125 48.484375 27.875 48.484375 \n",
       "Q 21.1875 48.484375 17.84375 46.4375 \n",
       "Q 14.5 44.390625 14.5 40.28125 \n",
       "Q 14.5 37.15625 16.890625 35.375 \n",
       "Q 19.28125 33.59375 26.515625 31.984375 \n",
       "L 29.59375 31.296875 \n",
       "Q 39.15625 29.25 43.1875 25.515625 \n",
       "Q 47.21875 21.78125 47.21875 15.09375 \n",
       "Q 47.21875 7.46875 41.1875 3.015625 \n",
       "Q 35.15625 -1.421875 24.609375 -1.421875 \n",
       "Q 20.21875 -1.421875 15.453125 -0.5625 \n",
       "Q 10.6875 0.296875 5.421875 2 \n",
       "L 5.421875 11.28125 \n",
       "Q 10.40625 8.6875 15.234375 7.390625 \n",
       "Q 20.0625 6.109375 24.8125 6.109375 \n",
       "Q 31.15625 6.109375 34.5625 8.28125 \n",
       "Q 37.984375 10.453125 37.984375 14.40625 \n",
       "Q 37.984375 18.0625 35.515625 20.015625 \n",
       "Q 33.0625 21.96875 24.703125 23.78125 \n",
       "L 21.578125 24.515625 \n",
       "Q 13.234375 26.265625 9.515625 29.90625 \n",
       "Q 5.8125 33.546875 5.8125 39.890625 \n",
       "Q 5.8125 47.609375 11.28125 51.796875 \n",
       "Q 16.75 56 26.8125 56 \n",
       "Q 31.78125 56 36.171875 55.265625 \n",
       "Q 40.578125 54.546875 44.28125 53.078125 \n",
       "z\n",
       "\" id=\"DejaVuSans-73\"/>\n",
       "     <path id=\"DejaVuSans-20\"/>\n",
       "     <path d=\"M 19.671875 64.796875 \n",
       "L 19.671875 8.109375 \n",
       "L 31.59375 8.109375 \n",
       "Q 46.6875 8.109375 53.6875 14.9375 \n",
       "Q 60.6875 21.78125 60.6875 36.53125 \n",
       "Q 60.6875 51.171875 53.6875 57.984375 \n",
       "Q 46.6875 64.796875 31.59375 64.796875 \n",
       "z\n",
       "M 9.8125 72.90625 \n",
       "L 30.078125 72.90625 \n",
       "Q 51.265625 72.90625 61.171875 64.09375 \n",
       "Q 71.09375 55.28125 71.09375 36.53125 \n",
       "Q 71.09375 17.671875 61.125 8.828125 \n",
       "Q 51.171875 0 30.078125 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-44\"/>\n",
       "     <path d=\"M 34.28125 27.484375 \n",
       "Q 23.390625 27.484375 19.1875 25 \n",
       "Q 14.984375 22.515625 14.984375 16.5 \n",
       "Q 14.984375 11.71875 18.140625 8.90625 \n",
       "Q 21.296875 6.109375 26.703125 6.109375 \n",
       "Q 34.1875 6.109375 38.703125 11.40625 \n",
       "Q 43.21875 16.703125 43.21875 25.484375 \n",
       "L 43.21875 27.484375 \n",
       "z\n",
       "M 52.203125 31.203125 \n",
       "L 52.203125 0 \n",
       "L 43.21875 0 \n",
       "L 43.21875 8.296875 \n",
       "Q 40.140625 3.328125 35.546875 0.953125 \n",
       "Q 30.953125 -1.421875 24.3125 -1.421875 \n",
       "Q 15.921875 -1.421875 10.953125 3.296875 \n",
       "Q 6 8.015625 6 15.921875 \n",
       "Q 6 25.140625 12.171875 29.828125 \n",
       "Q 18.359375 34.515625 30.609375 34.515625 \n",
       "L 43.21875 34.515625 \n",
       "L 43.21875 35.40625 \n",
       "Q 43.21875 41.609375 39.140625 45 \n",
       "Q 35.0625 48.390625 27.6875 48.390625 \n",
       "Q 23 48.390625 18.546875 47.265625 \n",
       "Q 14.109375 46.140625 10.015625 43.890625 \n",
       "L 10.015625 52.203125 \n",
       "Q 14.9375 54.109375 19.578125 55.046875 \n",
       "Q 24.21875 56 28.609375 56 \n",
       "Q 40.484375 56 46.34375 49.84375 \n",
       "Q 52.203125 43.703125 52.203125 31.203125 \n",
       "z\n",
       "\" id=\"DejaVuSans-61\"/>\n",
       "     <path d=\"M 9.8125 72.90625 \n",
       "L 19.671875 72.90625 \n",
       "L 19.671875 42.09375 \n",
       "L 52.390625 72.90625 \n",
       "L 65.09375 72.90625 \n",
       "L 28.90625 38.921875 \n",
       "L 67.671875 0 \n",
       "L 54.6875 0 \n",
       "L 19.671875 35.109375 \n",
       "L 19.671875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-4b\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(7.2 125.856784)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#DejaVuSans-41\"/>\n",
       "     <use x=\"68.408203\" xlink:href=\"#DejaVuSans-6c\"/>\n",
       "     <use x=\"96.191406\" xlink:href=\"#DejaVuSans-7a\"/>\n",
       "     <use x=\"148.681641\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "     <use x=\"212.060547\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"273.583984\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"301.367188\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "     <use x=\"398.779297\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"460.302734\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"501.416016\" xlink:href=\"#DejaVuSans-27\"/>\n",
       "     <use x=\"528.90625\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"581.005859\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"612.792969\" xlink:href=\"#DejaVuSans-44\"/>\n",
       "     <use x=\"689.794922\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"717.578125\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"769.677734\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"831.201172\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"892.480469\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"944.580078\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"1006.103516\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1037.890625\" xlink:href=\"#DejaVuSans-4b\"/>\n",
       "     <use x=\"1103.451172\" xlink:href=\"#DejaVuSans-41\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_5\">\n",
       "    <!-- E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 9.8125 72.90625 \n",
       "L 55.90625 72.90625 \n",
       "L 55.90625 64.59375 \n",
       "L 19.671875 64.59375 \n",
       "L 19.671875 43.015625 \n",
       "L 54.390625 43.015625 \n",
       "L 54.390625 34.71875 \n",
       "L 19.671875 34.71875 \n",
       "L 19.671875 8.296875 \n",
       "L 56.78125 8.296875 \n",
       "L 56.78125 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-45\"/>\n",
       "     <path d=\"M 4.890625 31.390625 \n",
       "L 31.203125 31.390625 \n",
       "L 31.203125 23.390625 \n",
       "L 4.890625 23.390625 \n",
       "z\n",
       "\" id=\"DejaVuSans-2d\"/>\n",
       "     <path d=\"M 59.515625 10.40625 \n",
       "L 59.515625 29.984375 \n",
       "L 43.40625 29.984375 \n",
       "L 43.40625 38.09375 \n",
       "L 69.28125 38.09375 \n",
       "L 69.28125 6.78125 \n",
       "Q 63.578125 2.734375 56.6875 0.65625 \n",
       "Q 49.8125 -1.421875 42 -1.421875 \n",
       "Q 24.90625 -1.421875 15.25 8.5625 \n",
       "Q 5.609375 18.5625 5.609375 36.375 \n",
       "Q 5.609375 54.25 15.25 64.234375 \n",
       "Q 24.90625 74.21875 42 74.21875 \n",
       "Q 49.125 74.21875 55.546875 72.453125 \n",
       "Q 61.96875 70.703125 67.390625 67.28125 \n",
       "L 67.390625 56.78125 \n",
       "Q 61.921875 61.421875 55.765625 63.765625 \n",
       "Q 49.609375 66.109375 42.828125 66.109375 \n",
       "Q 29.4375 66.109375 22.71875 58.640625 \n",
       "Q 16.015625 51.171875 16.015625 36.375 \n",
       "Q 16.015625 21.625 22.71875 14.15625 \n",
       "Q 29.4375 6.6875 42.828125 6.6875 \n",
       "Q 48.046875 6.6875 52.140625 7.59375 \n",
       "Q 56.25 8.5 59.515625 10.40625 \n",
       "z\n",
       "\" id=\"DejaVuSans-47\"/>\n",
       "     <path d=\"M 39.40625 66.21875 \n",
       "Q 28.65625 66.21875 22.328125 58.203125 \n",
       "Q 16.015625 50.203125 16.015625 36.375 \n",
       "Q 16.015625 22.609375 22.328125 14.59375 \n",
       "Q 28.65625 6.59375 39.40625 6.59375 \n",
       "Q 50.140625 6.59375 56.421875 14.59375 \n",
       "Q 62.703125 22.609375 62.703125 36.375 \n",
       "Q 62.703125 50.203125 56.421875 58.203125 \n",
       "Q 50.140625 66.21875 39.40625 66.21875 \n",
       "z\n",
       "M 39.40625 74.21875 \n",
       "Q 54.734375 74.21875 63.90625 63.9375 \n",
       "Q 73.09375 53.65625 73.09375 36.375 \n",
       "Q 73.09375 19.140625 63.90625 8.859375 \n",
       "Q 54.734375 -1.421875 39.40625 -1.421875 \n",
       "Q 24.03125 -1.421875 14.8125 8.828125 \n",
       "Q 5.609375 19.09375 5.609375 36.375 \n",
       "Q 5.609375 53.65625 14.8125 63.9375 \n",
       "Q 24.03125 74.21875 39.40625 74.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-4f\"/>\n",
       "     <path d=\"M 31.78125 34.625 \n",
       "Q 24.75 34.625 20.71875 30.859375 \n",
       "Q 16.703125 27.09375 16.703125 20.515625 \n",
       "Q 16.703125 13.921875 20.71875 10.15625 \n",
       "Q 24.75 6.390625 31.78125 6.390625 \n",
       "Q 38.8125 6.390625 42.859375 10.171875 \n",
       "Q 46.921875 13.96875 46.921875 20.515625 \n",
       "Q 46.921875 27.09375 42.890625 30.859375 \n",
       "Q 38.875 34.625 31.78125 34.625 \n",
       "z\n",
       "M 21.921875 38.8125 \n",
       "Q 15.578125 40.375 12.03125 44.71875 \n",
       "Q 8.5 49.078125 8.5 55.328125 \n",
       "Q 8.5 64.0625 14.71875 69.140625 \n",
       "Q 20.953125 74.21875 31.78125 74.21875 \n",
       "Q 42.671875 74.21875 48.875 69.140625 \n",
       "Q 55.078125 64.0625 55.078125 55.328125 \n",
       "Q 55.078125 49.078125 51.53125 44.71875 \n",
       "Q 48 40.375 41.703125 38.8125 \n",
       "Q 48.828125 37.15625 52.796875 32.3125 \n",
       "Q 56.78125 27.484375 56.78125 20.515625 \n",
       "Q 56.78125 9.90625 50.3125 4.234375 \n",
       "Q 43.84375 -1.421875 31.78125 -1.421875 \n",
       "Q 19.734375 -1.421875 13.25 4.234375 \n",
       "Q 6.78125 9.90625 6.78125 20.515625 \n",
       "Q 6.78125 27.484375 10.78125 32.3125 \n",
       "Q 14.796875 37.15625 21.921875 38.8125 \n",
       "z\n",
       "M 18.3125 54.390625 \n",
       "Q 18.3125 48.734375 21.84375 45.5625 \n",
       "Q 25.390625 42.390625 31.78125 42.390625 \n",
       "Q 38.140625 42.390625 41.71875 45.5625 \n",
       "Q 45.3125 48.734375 45.3125 54.390625 \n",
       "Q 45.3125 60.0625 41.71875 63.234375 \n",
       "Q 38.140625 66.40625 31.78125 66.40625 \n",
       "Q 25.390625 66.40625 21.84375 63.234375 \n",
       "Q 18.3125 60.0625 18.3125 54.390625 \n",
       "z\n",
       "\" id=\"DejaVuSans-38\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(197.014132 171.686734)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#DejaVuSans-45\"/>\n",
       "     <use x=\"63.183594\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"99.314453\" xlink:href=\"#DejaVuSans-47\"/>\n",
       "     <use x=\"176.804688\" xlink:href=\"#DejaVuSans-45\"/>\n",
       "     <use x=\"239.988281\" xlink:href=\"#DejaVuSans-4f\"/>\n",
       "     <use x=\"318.699219\" xlink:href=\"#DejaVuSans-44\"/>\n",
       "     <use x=\"395.701172\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"431.785156\" xlink:href=\"#DejaVuSans-35\"/>\n",
       "     <use x=\"495.408203\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "     <use x=\"559.03125\" xlink:href=\"#DejaVuSans-38\"/>\n",
       "     <use x=\"622.654297\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_6\">\n",
       "    <!-- Gene Overlap -->\n",
       "    <defs>\n",
       "     <path d=\"M 54.890625 33.015625 \n",
       "L 54.890625 0 \n",
       "L 45.90625 0 \n",
       "L 45.90625 32.71875 \n",
       "Q 45.90625 40.484375 42.875 44.328125 \n",
       "Q 39.84375 48.1875 33.796875 48.1875 \n",
       "Q 26.515625 48.1875 22.3125 43.546875 \n",
       "Q 18.109375 38.921875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.34375 51.125 25.703125 53.5625 \n",
       "Q 30.078125 56 35.796875 56 \n",
       "Q 45.21875 56 50.046875 50.171875 \n",
       "Q 54.890625 44.34375 54.890625 33.015625 \n",
       "z\n",
       "\" id=\"DejaVuSans-6e\"/>\n",
       "     <path d=\"M 2.984375 54.6875 \n",
       "L 12.5 54.6875 \n",
       "L 29.59375 8.796875 \n",
       "L 46.6875 54.6875 \n",
       "L 56.203125 54.6875 \n",
       "L 35.6875 0 \n",
       "L 23.484375 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-76\"/>\n",
       "     <path d=\"M 18.109375 8.203125 \n",
       "L 18.109375 -20.796875 \n",
       "L 9.078125 -20.796875 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.390625 \n",
       "Q 20.953125 51.265625 25.265625 53.625 \n",
       "Q 29.59375 56 35.59375 56 \n",
       "Q 45.5625 56 51.78125 48.09375 \n",
       "Q 58.015625 40.1875 58.015625 27.296875 \n",
       "Q 58.015625 14.40625 51.78125 6.484375 \n",
       "Q 45.5625 -1.421875 35.59375 -1.421875 \n",
       "Q 29.59375 -1.421875 25.265625 0.953125 \n",
       "Q 20.953125 3.328125 18.109375 8.203125 \n",
       "z\n",
       "M 48.6875 27.296875 \n",
       "Q 48.6875 37.203125 44.609375 42.84375 \n",
       "Q 40.53125 48.484375 33.40625 48.484375 \n",
       "Q 26.265625 48.484375 22.1875 42.84375 \n",
       "Q 18.109375 37.203125 18.109375 27.296875 \n",
       "Q 18.109375 17.390625 22.1875 11.75 \n",
       "Q 26.265625 6.109375 33.40625 6.109375 \n",
       "Q 40.53125 6.109375 44.609375 11.75 \n",
       "Q 48.6875 17.390625 48.6875 27.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-70\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(154.003916 16.318125)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#DejaVuSans-47\"/>\n",
       "     <use x=\"77.490234\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"139.013672\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"202.392578\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"263.916016\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"295.703125\" xlink:href=\"#DejaVuSans-4f\"/>\n",
       "     <use x=\"374.414062\" xlink:href=\"#DejaVuSans-76\"/>\n",
       "     <use x=\"433.59375\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"495.117188\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"536.230469\" xlink:href=\"#DejaVuSans-6c\"/>\n",
       "     <use x=\"564.013672\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"625.292969\" xlink:href=\"#DejaVuSans-70\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_5\">\n",
       "    <path d=\"M 381.798667 169.718125 \n",
       "L 708.216163 169.718125 \n",
       "L 708.216163 22.318125 \n",
       "L 381.798667 22.318125 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 396.635826 169.718125 \n",
       "L 426.310144 169.718125 \n",
       "L 426.310144 169.6924 \n",
       "L 396.635826 169.6924 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 426.310144 169.718125 \n",
       "L 455.984462 169.718125 \n",
       "L 455.984462 169.525188 \n",
       "L 426.310144 169.525188 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 455.984462 169.718125 \n",
       "L 485.65878 169.718125 \n",
       "L 485.65878 166.451059 \n",
       "L 455.984462 166.451059 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 485.65878 169.718125 \n",
       "L 515.333097 169.718125 \n",
       "L 515.333097 151.723535 \n",
       "L 485.65878 151.723535 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 515.333097 169.718125 \n",
       "L 545.007415 169.718125 \n",
       "L 545.007415 76.220861 \n",
       "L 515.333097 76.220861 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 545.007415 169.718125 \n",
       "L 574.681733 169.718125 \n",
       "L 574.681733 29.337173 \n",
       "L 545.007415 29.337173 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 574.681733 169.718125 \n",
       "L 604.356051 169.718125 \n",
       "L 604.356051 144.250443 \n",
       "L 574.681733 144.250443 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 604.356051 169.718125 \n",
       "L 634.030368 169.718125 \n",
       "L 634.030368 165.730761 \n",
       "L 604.356051 165.730761 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 634.030368 169.718125 \n",
       "L 663.704686 169.718125 \n",
       "L 663.704686 169.229351 \n",
       "L 634.030368 169.229351 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path clip-path=\"url(#p4f58d71c9f)\" d=\"M 663.704686 169.718125 \n",
       "L 693.379004 169.718125 \n",
       "L 693.379004 169.64095 \n",
       "L 663.704686 169.64095 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"md6b89837fe\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"407.163376\" xlink:href=\"#md6b89837fe\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- 4 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 35.5 \n",
       "L 73.1875 35.5 \n",
       "L 73.1875 27.203125 \n",
       "L 10.59375 27.203125 \n",
       "z\n",
       "\" id=\"DejaVuSans-2212\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(399.792282 184.316562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
       "       <use x=\"83.789062\" xlink:href=\"#DejaVuSans-34\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"478.09186\" xlink:href=\"#md6b89837fe\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- 2 -->\n",
       "      <g transform=\"translate(470.720766 184.316562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
       "       <use x=\"83.789062\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"549.020343\" xlink:href=\"#md6b89837fe\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(545.839093 184.316562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"619.948827\" xlink:href=\"#md6b89837fe\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- 2 -->\n",
       "      <g transform=\"translate(616.767577 184.316562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"690.877311\" xlink:href=\"#md6b89837fe\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- 4 -->\n",
       "      <g transform=\"translate(687.696061 184.316562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_12\">\n",
       "     <!-- Log-Fold-Change -->\n",
       "     <defs>\n",
       "      <path d=\"M 9.8125 72.90625 \n",
       "L 19.671875 72.90625 \n",
       "L 19.671875 8.296875 \n",
       "L 55.171875 8.296875 \n",
       "L 55.171875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-4c\"/>\n",
       "      <path d=\"M 30.609375 48.390625 \n",
       "Q 23.390625 48.390625 19.1875 42.75 \n",
       "Q 14.984375 37.109375 14.984375 27.296875 \n",
       "Q 14.984375 17.484375 19.15625 11.84375 \n",
       "Q 23.34375 6.203125 30.609375 6.203125 \n",
       "Q 37.796875 6.203125 41.984375 11.859375 \n",
       "Q 46.1875 17.53125 46.1875 27.296875 \n",
       "Q 46.1875 37.015625 41.984375 42.703125 \n",
       "Q 37.796875 48.390625 30.609375 48.390625 \n",
       "z\n",
       "M 30.609375 56 \n",
       "Q 42.328125 56 49.015625 48.375 \n",
       "Q 55.71875 40.765625 55.71875 27.296875 \n",
       "Q 55.71875 13.875 49.015625 6.21875 \n",
       "Q 42.328125 -1.421875 30.609375 -1.421875 \n",
       "Q 18.84375 -1.421875 12.171875 6.21875 \n",
       "Q 5.515625 13.875 5.515625 27.296875 \n",
       "Q 5.515625 40.765625 12.171875 48.375 \n",
       "Q 18.84375 56 30.609375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-6f\"/>\n",
       "      <path d=\"M 45.40625 27.984375 \n",
       "Q 45.40625 37.75 41.375 43.109375 \n",
       "Q 37.359375 48.484375 30.078125 48.484375 \n",
       "Q 22.859375 48.484375 18.828125 43.109375 \n",
       "Q 14.796875 37.75 14.796875 27.984375 \n",
       "Q 14.796875 18.265625 18.828125 12.890625 \n",
       "Q 22.859375 7.515625 30.078125 7.515625 \n",
       "Q 37.359375 7.515625 41.375 12.890625 \n",
       "Q 45.40625 18.265625 45.40625 27.984375 \n",
       "z\n",
       "M 54.390625 6.78125 \n",
       "Q 54.390625 -7.171875 48.1875 -13.984375 \n",
       "Q 42 -20.796875 29.203125 -20.796875 \n",
       "Q 24.46875 -20.796875 20.265625 -20.09375 \n",
       "Q 16.0625 -19.390625 12.109375 -17.921875 \n",
       "L 12.109375 -9.1875 \n",
       "Q 16.0625 -11.328125 19.921875 -12.34375 \n",
       "Q 23.78125 -13.375 27.78125 -13.375 \n",
       "Q 36.625 -13.375 41.015625 -8.765625 \n",
       "Q 45.40625 -4.15625 45.40625 5.171875 \n",
       "L 45.40625 9.625 \n",
       "Q 42.625 4.78125 38.28125 2.390625 \n",
       "Q 33.9375 0 27.875 0 \n",
       "Q 17.828125 0 11.671875 7.65625 \n",
       "Q 5.515625 15.328125 5.515625 27.984375 \n",
       "Q 5.515625 40.671875 11.671875 48.328125 \n",
       "Q 17.828125 56 27.875 56 \n",
       "Q 33.9375 56 38.28125 53.609375 \n",
       "Q 42.625 51.21875 45.40625 46.390625 \n",
       "L 45.40625 54.6875 \n",
       "L 54.390625 54.6875 \n",
       "z\n",
       "\" id=\"DejaVuSans-67\"/>\n",
       "      <path d=\"M 9.8125 72.90625 \n",
       "L 51.703125 72.90625 \n",
       "L 51.703125 64.59375 \n",
       "L 19.671875 64.59375 \n",
       "L 19.671875 43.109375 \n",
       "L 48.578125 43.109375 \n",
       "L 48.578125 34.8125 \n",
       "L 19.671875 34.8125 \n",
       "L 19.671875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-46\"/>\n",
       "      <path d=\"M 45.40625 46.390625 \n",
       "L 45.40625 75.984375 \n",
       "L 54.390625 75.984375 \n",
       "L 54.390625 0 \n",
       "L 45.40625 0 \n",
       "L 45.40625 8.203125 \n",
       "Q 42.578125 3.328125 38.25 0.953125 \n",
       "Q 33.9375 -1.421875 27.875 -1.421875 \n",
       "Q 17.96875 -1.421875 11.734375 6.484375 \n",
       "Q 5.515625 14.40625 5.515625 27.296875 \n",
       "Q 5.515625 40.1875 11.734375 48.09375 \n",
       "Q 17.96875 56 27.875 56 \n",
       "Q 33.9375 56 38.25 53.625 \n",
       "Q 42.578125 51.265625 45.40625 46.390625 \n",
       "z\n",
       "M 14.796875 27.296875 \n",
       "Q 14.796875 17.390625 18.875 11.75 \n",
       "Q 22.953125 6.109375 30.078125 6.109375 \n",
       "Q 37.203125 6.109375 41.296875 11.75 \n",
       "Q 45.40625 17.390625 45.40625 27.296875 \n",
       "Q 45.40625 37.203125 41.296875 42.84375 \n",
       "Q 37.203125 48.484375 30.078125 48.484375 \n",
       "Q 22.953125 48.484375 18.875 42.84375 \n",
       "Q 14.796875 37.203125 14.796875 27.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-64\"/>\n",
       "      <path d=\"M 64.40625 67.28125 \n",
       "L 64.40625 56.890625 \n",
       "Q 59.421875 61.53125 53.78125 63.8125 \n",
       "Q 48.140625 66.109375 41.796875 66.109375 \n",
       "Q 29.296875 66.109375 22.65625 58.46875 \n",
       "Q 16.015625 50.828125 16.015625 36.375 \n",
       "Q 16.015625 21.96875 22.65625 14.328125 \n",
       "Q 29.296875 6.6875 41.796875 6.6875 \n",
       "Q 48.140625 6.6875 53.78125 8.984375 \n",
       "Q 59.421875 11.28125 64.40625 15.921875 \n",
       "L 64.40625 5.609375 \n",
       "Q 59.234375 2.09375 53.4375 0.328125 \n",
       "Q 47.65625 -1.421875 41.21875 -1.421875 \n",
       "Q 24.65625 -1.421875 15.125 8.703125 \n",
       "Q 5.609375 18.84375 5.609375 36.375 \n",
       "Q 5.609375 53.953125 15.125 64.078125 \n",
       "Q 24.65625 74.21875 41.21875 74.21875 \n",
       "Q 47.75 74.21875 53.53125 72.484375 \n",
       "Q 59.328125 70.75 64.40625 67.28125 \n",
       "z\n",
       "\" id=\"DejaVuSans-43\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(502.741009 197.994687)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-4c\"/>\n",
       "      <use x=\"55.697266\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"116.878906\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"180.355469\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "      <use x=\"216.439453\" xlink:href=\"#DejaVuSans-46\"/>\n",
       "      <use x=\"273.912109\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"335.09375\" xlink:href=\"#DejaVuSans-6c\"/>\n",
       "      <use x=\"362.876953\" xlink:href=\"#DejaVuSans-64\"/>\n",
       "      <use x=\"426.353516\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "      <use x=\"462.4375\" xlink:href=\"#DejaVuSans-43\"/>\n",
       "      <use x=\"532.261719\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "      <use x=\"595.640625\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "      <use x=\"656.919922\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "      <use x=\"720.298828\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"783.775391\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m1b213fddce\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"169.718125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(368.436167 173.517344)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"143.993193\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- 2000 -->\n",
       "      <g transform=\"translate(349.348667 147.792412)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"190.869141\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"118.268262\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- 4000 -->\n",
       "      <g transform=\"translate(349.348667 122.06748)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"190.869141\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"92.54333\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- 6000 -->\n",
       "      <g transform=\"translate(349.348667 96.342549)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"190.869141\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"66.818398\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- 8000 -->\n",
       "      <g transform=\"translate(349.348667 70.617617)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-38\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"190.869141\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"381.798667\" xlink:href=\"#m1b213fddce\" y=\"41.093466\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- 10000 -->\n",
       "      <g transform=\"translate(342.986167 44.892685)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"190.869141\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"254.492188\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_19\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 14.796875 27.296875 \n",
       "Q 14.796875 17.390625 18.875 11.75 \n",
       "Q 22.953125 6.109375 30.078125 6.109375 \n",
       "Q 37.203125 6.109375 41.296875 11.75 \n",
       "Q 45.40625 17.390625 45.40625 27.296875 \n",
       "Q 45.40625 37.203125 41.296875 42.84375 \n",
       "Q 37.203125 48.484375 30.078125 48.484375 \n",
       "Q 22.953125 48.484375 18.875 42.84375 \n",
       "Q 14.796875 37.203125 14.796875 27.296875 \n",
       "z\n",
       "M 45.40625 8.203125 \n",
       "Q 42.578125 3.328125 38.25 0.953125 \n",
       "Q 33.9375 -1.421875 27.875 -1.421875 \n",
       "Q 17.96875 -1.421875 11.734375 6.484375 \n",
       "Q 5.515625 14.40625 5.515625 27.296875 \n",
       "Q 5.515625 40.1875 11.734375 48.09375 \n",
       "Q 17.96875 56 27.875 56 \n",
       "Q 33.9375 56 38.25 53.625 \n",
       "Q 42.578125 51.265625 45.40625 46.390625 \n",
       "L 45.40625 54.6875 \n",
       "L 54.390625 54.6875 \n",
       "L 54.390625 -20.796875 \n",
       "L 45.40625 -20.796875 \n",
       "z\n",
       "\" id=\"DejaVuSans-71\"/>\n",
       "      <path d=\"M 8.5 21.578125 \n",
       "L 8.5 54.6875 \n",
       "L 17.484375 54.6875 \n",
       "L 17.484375 21.921875 \n",
       "Q 17.484375 14.15625 20.5 10.265625 \n",
       "Q 23.53125 6.390625 29.59375 6.390625 \n",
       "Q 36.859375 6.390625 41.078125 11.03125 \n",
       "Q 45.3125 15.671875 45.3125 23.6875 \n",
       "L 45.3125 54.6875 \n",
       "L 54.296875 54.6875 \n",
       "L 54.296875 0 \n",
       "L 45.3125 0 \n",
       "L 45.3125 8.40625 \n",
       "Q 42.046875 3.421875 37.71875 1 \n",
       "Q 33.40625 -1.421875 27.6875 -1.421875 \n",
       "Q 18.265625 -1.421875 13.375 4.4375 \n",
       "Q 8.5 10.296875 8.5 21.578125 \n",
       "z\n",
       "M 31.109375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-75\"/>\n",
       "      <path d=\"M 48.78125 52.59375 \n",
       "L 48.78125 44.1875 \n",
       "Q 44.96875 46.296875 41.140625 47.34375 \n",
       "Q 37.3125 48.390625 33.40625 48.390625 \n",
       "Q 24.65625 48.390625 19.8125 42.84375 \n",
       "Q 14.984375 37.3125 14.984375 27.296875 \n",
       "Q 14.984375 17.28125 19.8125 11.734375 \n",
       "Q 24.65625 6.203125 33.40625 6.203125 \n",
       "Q 37.3125 6.203125 41.140625 7.25 \n",
       "Q 44.96875 8.296875 48.78125 10.40625 \n",
       "L 48.78125 2.09375 \n",
       "Q 45.015625 0.34375 40.984375 -0.53125 \n",
       "Q 36.96875 -1.421875 32.421875 -1.421875 \n",
       "Q 20.0625 -1.421875 12.78125 6.34375 \n",
       "Q 5.515625 14.109375 5.515625 27.296875 \n",
       "Q 5.515625 40.671875 12.859375 48.328125 \n",
       "Q 20.21875 56 33.015625 56 \n",
       "Q 37.15625 56 41.109375 55.140625 \n",
       "Q 45.0625 54.296875 48.78125 52.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-63\"/>\n",
       "      <path d=\"M 32.171875 -5.078125 \n",
       "Q 28.375 -14.84375 24.75 -17.8125 \n",
       "Q 21.140625 -20.796875 15.09375 -20.796875 \n",
       "L 7.90625 -20.796875 \n",
       "L 7.90625 -13.28125 \n",
       "L 13.1875 -13.28125 \n",
       "Q 16.890625 -13.28125 18.9375 -11.515625 \n",
       "Q 21 -9.765625 23.484375 -3.21875 \n",
       "L 25.09375 0.875 \n",
       "L 2.984375 54.6875 \n",
       "L 12.5 54.6875 \n",
       "L 29.59375 11.921875 \n",
       "L 46.6875 54.6875 \n",
       "L 56.203125 54.6875 \n",
       "z\n",
       "\" id=\"DejaVuSans-79\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(336.90648 122.315781)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-46\"/>\n",
       "      <use x=\"57.410156\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"98.492188\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"160.015625\" xlink:href=\"#DejaVuSans-71\"/>\n",
       "      <use x=\"223.492188\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"286.871094\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"348.394531\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "      <use x=\"411.773438\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "      <use x=\"466.753906\" xlink:href=\"#DejaVuSans-79\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 381.798667 169.718125 \n",
       "L 381.798667 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 708.216163 169.718125 \n",
       "L 708.216163 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path d=\"M 381.798667 169.718125 \n",
       "L 708.216163 169.718125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 381.798667 22.318125 \n",
       "L 708.216163 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_20\">\n",
       "    <!-- Distribution of Log-Fold-Changes in E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 18.3125 70.21875 \n",
       "L 18.3125 54.6875 \n",
       "L 36.8125 54.6875 \n",
       "L 36.8125 47.703125 \n",
       "L 18.3125 47.703125 \n",
       "L 18.3125 18.015625 \n",
       "Q 18.3125 11.328125 20.140625 9.421875 \n",
       "Q 21.96875 7.515625 27.59375 7.515625 \n",
       "L 36.8125 7.515625 \n",
       "L 36.8125 0 \n",
       "L 27.59375 0 \n",
       "Q 17.1875 0 13.234375 3.875 \n",
       "Q 9.28125 7.765625 9.28125 18.015625 \n",
       "L 9.28125 47.703125 \n",
       "L 2.6875 47.703125 \n",
       "L 2.6875 54.6875 \n",
       "L 9.28125 54.6875 \n",
       "L 9.28125 70.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-74\"/>\n",
       "     <path d=\"M 48.6875 27.296875 \n",
       "Q 48.6875 37.203125 44.609375 42.84375 \n",
       "Q 40.53125 48.484375 33.40625 48.484375 \n",
       "Q 26.265625 48.484375 22.1875 42.84375 \n",
       "Q 18.109375 37.203125 18.109375 27.296875 \n",
       "Q 18.109375 17.390625 22.1875 11.75 \n",
       "Q 26.265625 6.109375 33.40625 6.109375 \n",
       "Q 40.53125 6.109375 44.609375 11.75 \n",
       "Q 48.6875 17.390625 48.6875 27.296875 \n",
       "z\n",
       "M 18.109375 46.390625 \n",
       "Q 20.953125 51.265625 25.265625 53.625 \n",
       "Q 29.59375 56 35.59375 56 \n",
       "Q 45.5625 56 51.78125 48.09375 \n",
       "Q 58.015625 40.1875 58.015625 27.296875 \n",
       "Q 58.015625 14.40625 51.78125 6.484375 \n",
       "Q 45.5625 -1.421875 35.59375 -1.421875 \n",
       "Q 29.59375 -1.421875 25.265625 0.953125 \n",
       "Q 20.953125 3.328125 18.109375 8.203125 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 75.984375 \n",
       "L 18.109375 75.984375 \n",
       "z\n",
       "\" id=\"DejaVuSans-62\"/>\n",
       "     <path d=\"M 37.109375 75.984375 \n",
       "L 37.109375 68.5 \n",
       "L 28.515625 68.5 \n",
       "Q 23.6875 68.5 21.796875 66.546875 \n",
       "Q 19.921875 64.59375 19.921875 59.515625 \n",
       "L 19.921875 54.6875 \n",
       "L 34.71875 54.6875 \n",
       "L 34.71875 47.703125 \n",
       "L 19.921875 47.703125 \n",
       "L 19.921875 0 \n",
       "L 10.890625 0 \n",
       "L 10.890625 47.703125 \n",
       "L 2.296875 47.703125 \n",
       "L 2.296875 54.6875 \n",
       "L 10.890625 54.6875 \n",
       "L 10.890625 58.5 \n",
       "Q 10.890625 67.625 15.140625 71.796875 \n",
       "Q 19.390625 75.984375 28.609375 75.984375 \n",
       "z\n",
       "\" id=\"DejaVuSans-66\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(396.102415 16.318125)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#DejaVuSans-44\"/>\n",
       "     <use x=\"77.001953\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"104.785156\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"156.884766\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "     <use x=\"196.09375\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"237.207031\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"264.990234\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "     <use x=\"328.466797\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "     <use x=\"391.845703\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "     <use x=\"431.054688\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"458.837891\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"520.019531\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"583.398438\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"615.185547\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"676.367188\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"711.572266\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"743.359375\" xlink:href=\"#DejaVuSans-4c\"/>\n",
       "     <use x=\"799.056641\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"860.238281\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "     <use x=\"923.714844\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"959.798828\" xlink:href=\"#DejaVuSans-46\"/>\n",
       "     <use x=\"1017.271484\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"1078.453125\" xlink:href=\"#DejaVuSans-6c\"/>\n",
       "     <use x=\"1106.236328\" xlink:href=\"#DejaVuSans-64\"/>\n",
       "     <use x=\"1169.712891\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"1205.796875\" xlink:href=\"#DejaVuSans-43\"/>\n",
       "     <use x=\"1275.621094\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "     <use x=\"1339\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"1400.279297\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"1463.658203\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "     <use x=\"1527.134766\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"1588.658203\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"1640.757812\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1672.544922\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"1700.328125\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"1763.707031\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1795.494141\" xlink:href=\"#DejaVuSans-45\"/>\n",
       "     <use x=\"1858.677734\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"1894.808594\" xlink:href=\"#DejaVuSans-47\"/>\n",
       "     <use x=\"1972.298828\" xlink:href=\"#DejaVuSans-45\"/>\n",
       "     <use x=\"2035.482422\" xlink:href=\"#DejaVuSans-4f\"/>\n",
       "     <use x=\"2114.193359\" xlink:href=\"#DejaVuSans-44\"/>\n",
       "     <use x=\"2191.195312\" xlink:href=\"#DejaVuSans-2d\"/>\n",
       "     <use x=\"2227.279297\" xlink:href=\"#DejaVuSans-35\"/>\n",
       "     <use x=\"2290.902344\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "     <use x=\"2354.525391\" xlink:href=\"#DejaVuSans-38\"/>\n",
       "     <use x=\"2418.148438\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p02e85ee312\">\n",
       "   <rect height=\"147.4\" width=\"150.764807\" x=\"119.949325\" y=\"22.318125\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p4f58d71c9f\">\n",
       "   <rect height=\"147.4\" width=\"326.417495\" x=\"381.798667\" y=\"22.318125\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x10cccff28>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "fix, (rax, lax) = plt.subplots(1, 2, figsize=(10, 3))\n",
    "\n",
    "lax.set_title(\"Distribution of Log-Fold-Changes in E-GEOD-5281\")\n",
    "lax.set_xlabel(\"Log-Fold-Change\")\n",
    "lax.set_ylabel(\"Frequency\")\n",
    "lax.hist(list(data.values()))\n",
    "\n",
    "rax.set_title(\"Gene Overlap\")\n",
    "venn2(\n",
    "    [hgnc_names, df_names], set_labels=[\"Alzheimer's Disease KA\", \"E-GEOD-5281\"], ax=rax\n",
    ")\n",
    "plt.tight_layout()\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Integration\n",
    "\n",
    "Finally, the differential gene expression data are ovelayed on the BEL graph with [pybel_tools.integration.overlay_type_data](http://pybel-tools.readthedocs.io/en/latest/integration.html#pybel_tools.integration.overlay_type_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "overlay_type_data(graph, data, GENE, \"HGNC\", overwrite=False, impute=0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Subgraph Generation\n",
    "\n",
    "A subgraph is induced around an example biological process, Inflammatory Response, by selecting all of the source nodes of upstream causal edges, where this process is the target node. These networks are then enriched by expandind around all of those nodes' upstream edges as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "inflamation_node = BIOPROCESS, \"GOBP\", \"inflammatory response\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 1/2\n",
    "\n",
    "All of the nodes that have upstream causal relations to Inflammatory Response are added."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 41\n",
      "Edges: 49\n",
      "Citations: 25\n",
      "Authors: 0\n",
      "Network density: 0.029878048780487804\n",
      "Components: 1\n",
      "Average degree: 1.1951219512195121\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='bqvxdaujygremoqt'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Caspase subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\", \"type\": \"PubMed\"}, \"evidence\": \"Results show that p53 has two binding sites located at the cathepsin D promoter gene and that cathepsin D participates in p53-dependent apoptotic process. Cathepsin D showed augmented activity soon after it was released and that was accompanied by increased levels of p53 protein, a cathepsin D transcription factor [16]. Therefore, the mechanism responsible for increase in cathepsin D activity might be an effect of increased synthesis regulated by p53. Cathepsin B has also been implicated in the activation of the pro-inflammatory caspases-1 and -11, and the cleavage of Bcl-2 family member Bid which may lead to cytochrome c release from the mitochondria and subsequent caspase activation\", \"hash\": \"4d0d7140a5909395eaa64fd2e1d1a19c43489426d8068708dcc3f53d1868ee0c258cc77d443e0b23f085eefc90e66c532b09fd1de19205f4c35c227a022debbf\", \"key\": 0, \"line\": 6816, \"relation\": \"increases\", \"source\": 0, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Caspase subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\", \"type\": \"PubMed\"}, \"evidence\": \"Results show that p53 has two binding sites located at the cathepsin D promoter gene and that cathepsin D participates in p53-dependent apoptotic process. Cathepsin D showed augmented activity soon after it was released and that was accompanied by increased levels of p53 protein, a cathepsin D transcription factor [16]. Therefore, the mechanism responsible for increase in cathepsin D activity might be an effect of increased synthesis regulated by p53. Cathepsin B has also been implicated in the activation of the pro-inflammatory caspases-1 and -11, and the cleavage of Bcl-2 family member Bid which may lead to cytochrome c release from the mitochondria and subsequent caspase activation\", \"hash\": \"2f73f4611836b09217020233933297dace251942d1121292ecdb5820074d6346c3333b26f7b2b5be8c48ccc71454f07cd58f789a28843132b0cd853da12994c5\", \"key\": 1, \"line\": 34733, \"relation\": \"increases\", \"source\": 0, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 2, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 2, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 2, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Eicosanoids signaling subgraph\": true}}, \"citation\": {\"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\", \"type\": \"PubMed\"}, \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"hash\": \"62ad85a9706f042fc0514e548495bac731dabd77cc0e6f46ace64f21dc1dce159f5d027f54c95ba8b3913ae23509d47066f9b8e3b2bf05673e67c98e612e5c7b\", \"key\": 0, \"line\": 21487, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Neuroprotection subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2011 Jan 5;6(1):e15816. doi: 10.1371/journal.pone.0015816.\", \"reference\": \"21246057\", \"type\": \"PubMed\"}, \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"hash\": \"97fae9c366f62e66ff0b238c5c1705a82a5d94b3caac73694f20fc3a51cb26ce53f3fcee06beb8e060f83d351bb9ae39d8f346e4b39999afcd7cdbad8a482268\", \"key\": 0, \"line\": 23979, \"relation\": \"decreases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Neuroprotection subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"PLoS One2011\", \"reference\": \"21246057\", \"type\": \"PubMed\"}, \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"hash\": \"d052a498e9cbd694dd2663d48f7eb1b6580e5dae32cbbab6753f86c4bb889f14f92d4b68505253a0fcb1d1f8e571b71faadc38922cb14d2f031bf13a58c0c284\", \"key\": 1, \"line\": 24486, \"relation\": \"decreases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 5, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Acta Pharmacol Sin. 2006 Jan;27(1):41-9.\", \"reference\": \"16364209\", \"type\": \"PubMed\"}, \"evidence\": \"Recent studies showed that melatonin, an indoleamine secreted by the pineal gland, may play an important/ role in aging and AD as an antioxidant and neuroprotector. Melatonin decreases during aging and patients with AD have a/ more profound reduction in this hormone.Melatonin efficiently protects neuronal cells from Abeta-mediated toxicity via/ antioxidant and anti-amyloid properties: it not only inhibits Abeta generation, but also arrests the formation of amyloid/ fibrils by a structure-dependent interaction with Abeta. Our recent studies have demonstrated that melatonin efficiently/ attenuates Alzheimer-like tau hyperphosphorylation. Although the exact mechanism is still not fully understood, a direct/ regulatory influence of melatonin on the activities of protein kinases and protein phosphatases is proposed. Additionally,/ melatonin also plays a role in protecting cholinergic neurons and in anti-inflammation.\", \"hash\": \"435f4be8d47ed098b0f842af523a6ee7b5615ad23da92f890b99cba93c5d9a79982aa7d40c0a1b63e874ecd24c8c50110d03ff052d13dc17c3e810d1d34685ef\", \"key\": 0, \"line\": 38197, \"relation\": \"decreases\", \"source\": 6, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"MeSHDisease\": {\"Alzheimer Disease\": true}, \"Subgraph\": {\"Calcium-dependent signal transduction\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\", \"type\": \"PubMed\"}, \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"hash\": \"d984cd93f761a599d1f2d6a990f3a17eee7cd6523353e073b604925c0676c1cace5d5285c254b08ac6dc83fc71cfc7d7f48d14c4fddf8bfb2dd60e337c937468\", \"key\": 0, \"line\": 38361, \"relation\": \"increases\", \"source\": 7, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 8, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Macrophages\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Journal of immunology (Baltimore, Md. : 1950); Vol. 188; Iss. 3\", \"reference\": \"22198949\", \"type\": \"PubMed\"}, \"evidence\": \"By coexpressing TLR1 or TLR6 in TLR2-transgenic HEK293 cells or silencing tlrs genes in RAW264.7 macrophages, we observed that TLR2-mediated Abeta42-triggered inflammatory activation was enhanced by TLR1 and suppressed by TLR6.\", \"hash\": \"bc2de36e8558b33575359865a546207fd7bb4a5551b44594549f2f01a9089bedc8293bb07b48c1bc36189328a8c0540c7b81988a7e58b737ddb6b9bd824c0cae\", \"key\": 0, \"line\": 40173, \"relation\": \"increases\", \"source\": 9, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Journal of immunology (Baltimore, Md. : 1950); Vol. 188; Iss. 3\", \"reference\": \"22198949\", \"type\": \"PubMed\"}, \"evidence\": \"Our study demonstrated that TLR2 is a primary receptor for Abeta to trigger neuroinflammatory activation and suggested that inhibition of TLR2 in microglia could be beneficial in Alzheimer's disease pathogenesis.\", \"hash\": \"fb05191de6c158ba88acff261c256a747e584525a46926ee21b3b6e7aef7fed524408ce9d1f9776a01086003d9a33ea6e6cad728b452adb7467d7d865c973234\", \"key\": 1, \"line\": 40186, \"relation\": \"increases\", \"source\": 9, \"target\": 1}, {\"annotations\": {\"Species\": {\"10090\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PloS one; Vol. 9; Iss. 3\", \"reference\": \"24621589\", \"type\": \"PubMed\"}, \"evidence\": \"Notably, nicardipine also showed significant anti-neuroinflammatory effects on microglial activation in mice in vivo.The present study is the first to report a novel inhibitory role of nicardipine on neuroinflammation and provides a new candidate agent for the development of therapies for inflammation-related neurodegenerative diseases.\", \"hash\": \"7aee2ef467ef229df9673007e85299b3638a4ff14f3f1b1c58258d11176b1d243b379e22ad187d4b8c37894d3ab90e0efbaf2ce7c7b35cd5a69a4681ca75eb70\", \"key\": 0, \"line\": 41679, \"relation\": \"decreases\", \"source\": 10, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 11, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2003 Aug;86(4):907-16.\", \"reference\": \"12887689\", \"type\": \"PubMed\"}, \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"hash\": \"6c97d7801fb2bca1f9bf2b84edd6d1bbd2c8c91c205727563e691fddee08cdfb489df944b9035f1ed1579fe5a773c6732af39f6a38db837f3277d5fe4b633c64\", \"key\": 0, \"line\": 42168, \"relation\": \"decreases\", \"source\": 12, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 13, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 13, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"93cd2ce794c083e46cf2d6cc4e5dae4ccc9dd90d32971c8d11db85bf88788423ac092519856d67e1fd0f6044694023a158f5b4cad327f0008de50fa1f7577ea4\", \"key\": 0, \"line\": 39270, \"relation\": \"increases\", \"source\": 14, \"target\": 1}, {\"annotations\": {\"Encode_Feature_Types\": {\"Promoter\": true}, \"MeSHAnatomy\": {\"Hippocampus\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\", \"type\": \"PubMed\"}, \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"hash\": \"f4ae4e87227ac55e1c5a823b80f3c19a0fd563c9c6be1bbdb720545f08ca61c2388f6f6b4e63715432ec61990c92069bfce8195393cbd5a24ef29514034fe19e\", \"key\": 0, \"line\": 45205, \"relation\": \"increases\", \"source\": 15, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"N Engl J Med. 2013 Jan 10;368(2):117-27. doi: 10.1056/NEJMoa1211851. Epub 2012 Nov 14.\", \"reference\": \"23150934\", \"type\": \"PubMed\"}, \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"hash\": \"6fd472ec009aaa5034c7f787784aea064f912ae73bcb620047643a92fe9c0bc9dec67af8315ff6b72ed0555f733035fb80ab5cf96a37dd999e7b54861fd5305d\", \"key\": 1, \"line\": 45645, \"relation\": \"decreases\", \"source\": 16, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Complement system subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2008 Nov 14;283(46):31315-22. doi: 10.1074/jbc.M805371200. Epub 2008 Sep 18.\", \"reference\": \"18801740\", \"type\": \"PubMed\"}, \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"hash\": \"37eb74446a87252302a649479b6c270acb8263a38de56dba278260957f5c2ae8fb85c6720e05aad3829e63cb6d168a0e42aafb0a75852263620d146591edd9c3\", \"key\": 0, \"line\": 9123, \"relation\": \"decreases\", \"source\": 17, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"db3ea32cc348e56d341202a5c397cdd070fc58c904ae34888e216a78e7108c4cbf0435e117c57369c25348feaf310bb658d466e76af2972766b5f28048f400a0\", \"key\": 0, \"line\": 9509, \"relation\": \"decreases\", \"source\": 18, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"cb1103fac72f40f6b5700a2a048666fb9879f73c11f21d72f4a47dcd8e1477b0b744d77907cc2c5d88d67065a798a9bda3469e3777c997b1d4ae4e87fd68eb33\", \"key\": 0, \"line\": 9511, \"relation\": \"increases\", \"source\": 19, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 20, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 21, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 22, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 23, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 24, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 25, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 25, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 26, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 27, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 28, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 28, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 29, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 29, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"4950a0ac1d27fbea839a8904a352a327e8632a2e6555df7a0d963e61ac2c953ab9d04a33a643893f8c4fb9b61846dd327b5dfa472ee6aee121c62bf2c1e5574b\", \"key\": 0, \"line\": 39272, \"relation\": \"increases\", \"source\": 30, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 31, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 32, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 33, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\", \"type\": \"PubMed\"}, \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"hash\": \"287970aa3d22a0c143e02a1156dc171f03af2892024ca68e686f2511c4248c5ceaab813389f54295270a03585869d4940398ffe41e09c70c776375e2c2090794\", \"key\": 0, \"line\": 38419, \"relation\": \"increases\", \"source\": 34, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 35, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 36, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\", \"type\": \"PubMed\"}, \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"hash\": \"81e32aa7776d082960910d09b5e53eafe56e17b6397bfffb96111f688c33301e0d1b8a3d78533d79f522e23360ea291f9da3efe9e0c0075a9d3f7cb9df996b59\", \"key\": 0, \"line\": 38417, \"relation\": \"increases\", \"source\": 37, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2015 Jun;36(6):2018-23. doi: 10.1016/j.neurobiolaging.2015.03.001\", \"reference\": \"25865441\", \"type\": \"PubMed\"}, \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Abeta42+ subjects but not in Abeta42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"hash\": \"20375a264d8c8c5eb4893054811491c1df4fda89560721a37772406f90ddbbddd014ad6ca23c932cf20f5ccddfd6cef24285cd0dcfe75afbc34159347723b428\", \"key\": 0, \"line\": 45241, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Cell\": {\"monocyte\": true, \"neutrophil\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\", \"type\": \"PubMed\"}, \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"hash\": \"ce87734784feaf2736ff7a47899c05f8790617453784d8f233a51880672aac09661be5262dea4dca491fd9f5902d9d43331cb6d38d46a5477205c0340bf855a4\", \"key\": 0, \"line\": 46433, \"relation\": \"increases\", \"source\": 40, \"target\": 1}], \"multigraph\": true, \"nodes\": [{\"cname\": \"composite(p(HGNC:CASP1), p(HGNC:SCAF11))\", \"function\": \"Composite\", \"id\": [\"Composite\", [\"Protein\", \"HGNC\", \"CASP1\"], [\"Protein\", \"HGNC\", \"SCAF11\"]], \"members\": [{\"function\": \"Protein\", \"name\": \"CASP1\", \"namespace\": \"HGNC\"}, {\"function\": \"Protein\", \"name\": \"SCAF11\", \"namespace\": \"HGNC\"}]}, {\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"leukotriene\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"], \"name\": \"leukotriene\", \"namespace\": \"CHEBI\"}, {\"cname\": \"neuroprotectin\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"ADO\", \"neuroprotectin\"], \"name\": \"neuroprotectin\", \"namespace\": \"ADO\"}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"melatonin\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"CHEBI\", \"melatonin\"], \"name\": \"melatonin\", \"namespace\": \"CHEBI\"}, {\"cname\": \"Neurofibrillary Tangles\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"], \"name\": \"Neurofibrillary Tangles\", \"namespace\": \"MESHCS\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"complex(p(HGNC:APP, frag(672_713)), p(MGI:Tlr2))\", \"function\": \"Complex\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], [\"Protein\", \"MGI\", \"Tlr2\"]], \"members\": [{\"function\": \"Protein\", \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"function\": \"Protein\", \"name\": \"Tlr2\", \"namespace\": \"MGI\"}]}, {\"cname\": \"Nicardipine\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"CHEBI\", \"Nicardipine\"], \"name\": \"Nicardipine\", \"namespace\": \"CHEBI\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"noradrenaline\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"CHEBI\", \"noradrenaline\"], \"name\": \"noradrenaline\", \"namespace\": \"CHEBI\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"PTGS2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTGS2\"], \"name\": \"PTGS2\", \"namespace\": \"HGNC\", \"weight\": -1.07023648}, {\"cname\": \"CX3CR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"], \"name\": \"CX3CR1\", \"namespace\": \"HGNC\", \"weight\": -0.81994598}, {\"cname\": \"TREM2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"], \"name\": \"TREM2\", \"namespace\": \"HGNC\", \"weight\": -0.40134828}, {\"cname\": \"CFH\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CFH\"], \"name\": \"CFH\", \"namespace\": \"HGNC\", \"weight\": 1.86557178}, {\"cname\": \"IRAK1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"], \"name\": \"IRAK1\", \"namespace\": \"HGNC\", \"weight\": 0.72985984}, {\"cname\": \"IRAK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"], \"name\": \"IRAK2\", \"namespace\": \"HGNC\", \"weight\": -0.26455035}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"PLA2G4A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G4A\"], \"name\": \"PLA2G4A\", \"namespace\": \"HGNC\", \"weight\": 0.52335304}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"MT-CO2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MT-CO2\"], \"name\": \"MT-CO2\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"PLA2G7\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"], \"name\": \"PLA2G7\", \"namespace\": \"HGNC\", \"weight\": -0.52445663}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}, {\"cname\": \"CHI3L1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"], \"name\": \"CHI3L1\", \"namespace\": \"HGNC\", \"weight\": 0.14672037}, {\"cname\": \"TREM1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"], \"name\": \"TREM1\", \"namespace\": \"HGNC\", \"weight\": 1.13272121}]}, \"#bqvxdaujygremoqt\", 1000, 650, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "subgraph = pybel_tools.mutation.get_upstream_causal_subgraph(graph, inflamation_node)\n",
    "\n",
    "print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 2/2\n",
    "\n",
    "All of the nodes upstream of the nodes within the subgraph are added as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 343\n",
      "Edges: 763\n",
      "Citations: 423\n",
      "Authors: 0\n",
      "Network density: 0.006504356128416279\n",
      "Components: 1\n",
      "Average degree: 2.2244897959183674\n"
     ]
    }
   ],
   "source": [
    "pybel_tools.mutation.expand_upstream_causal_subgraph(graph, subgraph)\n",
    "print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Post-Processing\n",
    "\n",
    "Inconsistient edges are thrown away, literature bias is removed, and nodes on the periphery of the subgraph that are not annotated with experimental data are also removed, since they cannot contribute to the propogation of scores."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.mutation.remove_inconsistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.mutation.collapse_consistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.generation.remove_unweighted_leaves(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "pybel_tools.generation.remove_unweighted_sources(subgraph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Nodes: 181\n",
      "Edges: 520\n",
      "Citations: 323\n",
      "Authors: 0\n",
      "Network density: 0.01596071209330878\n",
      "Components: 1\n",
      "Average degree: 2.8729281767955803\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='mkudpqcofahtyrge'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 22}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"b097a471f1fb37c8206da83f8cf86668534b3469343940202384cda9e85acb294bb50b9c69df75eb070f8392a22732f08cbce6fb1c38da3372e7e0fb62533325\", \"key\": 0, \"line\": 42188, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1993 Dec;124(2):289-98.\", \"reference\": \"8287928\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"hash\": \"ad1f29aa4d840d0ddc1e3d8a3da37591bcca66d1a611768e9d74312f777831f3a22c72a324e8b244eb9b2b7cc5dd9d31937cd75ef2691979fbb2e4738b8a2517\", \"key\": 1, \"line\": 42344, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 21}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 22}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 22}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 23}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 23}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 25}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 26}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 27}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 29}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 30}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 32}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Eicosanoids signaling subgraph\": true}}, \"citation\": {\"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\", \"type\": \"PubMed\"}, \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"hash\": \"62ad85a9706f042fc0514e548495bac731dabd77cc0e6f46ace64f21dc1dce159f5d027f54c95ba8b3913ae23509d47066f9b8e3b2bf05673e67c98e612e5c7b\", \"key\": 0, \"line\": 21487, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"MeSHDisease\": {\"Alzheimer Disease\": true}, \"Subgraph\": {\"Calcium-dependent signal transduction\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\", \"type\": \"PubMed\"}, \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"hash\": \"d984cd93f761a599d1f2d6a990f3a17eee7cd6523353e073b604925c0676c1cace5d5285c254b08ac6dc83fc71cfc7d7f48d14c4fddf8bfb2dd60e337c937468\", \"key\": 0, \"line\": 38361, \"relation\": \"increases\", \"source\": 4, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 5, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 5, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 5, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 6, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"4f0cba06b674c0e9847c0fb9884cf859ca04ba644c3c6b28c077cfa4cf1a974bb209c0cdda26733a5f6ada7743d7cff85624c046373e88f3e78122b2cf2fefdd\", \"key\": 0, \"line\": 3554, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"hash\": \"1e78a3aea456a94331af060b6acca92b9fdf5ce40ee68f9a2179f513be1a8ceda93198b68dd0e65d3bfe46b251ef6be30b4738befcf651d1ad9812d63050d6ff\", \"key\": 1, \"line\": 6181, \"relation\": \"increases\", \"source\": 6, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"fecdb06cc63b0946512b0d57bf1de37ccecd3633343e9cdb5671d07531cbc04802c0ae1a71e6534814b9ee6f0ce399b25ebf475f8dc97d2a63c0c8b9caf93358\", \"key\": 2, \"line\": 35325, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 6, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of neuroinflammation; Vol. 11; Iss. \", \"reference\": \"24694234\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid- (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"hash\": \"44f25dfab4dbfc410250e4ab86280e73c9b1bfcfbc6522fde6dc3387fbd91dd4570fd4d1f31c89c05cfe99ff3bb39aa0088f31cb1af2aabb632aab2a044978db\", \"key\": 3, \"line\": 39478, \"relation\": \"increases\", \"source\": 6, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"5223c465d4674837b2596c4aba54d38ff3a70148ac3773bdea1fef6a41b73da935452e9285019f43c1a992ce58915bcc5d5dc6f7ce82b9bb0cfa7a70782bde87\", \"key\": 4, \"line\": 42189, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\", \"type\": \"PubMed\"}, \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"hash\": \"8c09ee5bf342ee7449995131e6d0c4db263272a6d2dd2bb358cc00c503190f0fa93ad93e22d2bb631fffcc0aa52ea8037aa9b3ba2ed283b1b32e1d60e75bac8c\", \"key\": 0, \"line\": 4875, \"relation\": \"increases\", \"source\": 6, \"target\": 21}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 7, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"93cd2ce794c083e46cf2d6cc4e5dae4ccc9dd90d32971c8d11db85bf88788423ac092519856d67e1fd0f6044694023a158f5b4cad327f0008de50fa1f7577ea4\", \"key\": 0, \"line\": 39270, \"relation\": \"increases\", \"source\": 8, \"target\": 0}, {\"annotations\": {\"Encode_Feature_Types\": {\"Promoter\": true}, \"MeSHAnatomy\": {\"Hippocampus\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\", \"type\": \"PubMed\"}, \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"hash\": \"f4ae4e87227ac55e1c5a823b80f3c19a0fd563c9c6be1bbdb720545f08ca61c2388f6f6b4e63715432ec61990c92069bfce8195393cbd5a24ef29514034fe19e\", \"key\": 0, \"line\": 45205, \"relation\": \"increases\", \"source\": 9, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"N Engl J Med. 2013 Jan 10;368(2):117-27. doi: 10.1056/NEJMoa1211851. Epub 2012 Nov 14.\", \"reference\": \"23150934\", \"type\": \"PubMed\"}, \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"hash\": \"6fd472ec009aaa5034c7f787784aea064f912ae73bcb620047643a92fe9c0bc9dec67af8315ff6b72ed0555f733035fb80ab5cf96a37dd999e7b54861fd5305d\", \"key\": 1, \"line\": 45645, \"relation\": \"decreases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Complement system subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2008 Nov 14;283(46):31315-22. doi: 10.1074/jbc.M805371200. Epub 2008 Sep 18.\", \"reference\": \"18801740\", \"type\": \"PubMed\"}, \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"hash\": \"37eb74446a87252302a649479b6c270acb8263a38de56dba278260957f5c2ae8fb85c6720e05aad3829e63cb6d168a0e42aafb0a75852263620d146591edd9c3\", \"key\": 0, \"line\": 9123, \"relation\": \"decreases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"db3ea32cc348e56d341202a5c397cdd070fc58c904ae34888e216a78e7108c4cbf0435e117c57369c25348feaf310bb658d466e76af2972766b5f28048f400a0\", \"key\": 0, \"line\": 9509, \"relation\": \"decreases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"cb1103fac72f40f6b5700a2a048666fb9879f73c11f21d72f4a47dcd8e1477b0b744d77907cc2c5d88d67065a798a9bda3469e3777c997b1d4ae4e87fd68eb33\", \"key\": 0, \"line\": 9511, \"relation\": \"increases\", \"source\": 13, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 17, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 18, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 19, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 20, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 21, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 22, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 22, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 23, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 23, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"4950a0ac1d27fbea839a8904a352a327e8632a2e6555df7a0d963e61ac2c953ab9d04a33a643893f8c4fb9b61846dd327b5dfa472ee6aee121c62bf2c1e5574b\", \"key\": 0, \"line\": 39272, \"relation\": \"increases\", \"source\": 24, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 25, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 26, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 27, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\", \"type\": \"PubMed\"}, \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"hash\": \"287970aa3d22a0c143e02a1156dc171f03af2892024ca68e686f2511c4248c5ceaab813389f54295270a03585869d4940398ffe41e09c70c776375e2c2090794\", \"key\": 0, \"line\": 38419, \"relation\": \"increases\", \"source\": 28, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 29, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 30, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\", \"type\": \"PubMed\"}, \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"hash\": \"81e32aa7776d082960910d09b5e53eafe56e17b6397bfffb96111f688c33301e0d1b8a3d78533d79f522e23360ea291f9da3efe9e0c0075a9d3f7cb9df996b59\", \"key\": 0, \"line\": 38417, \"relation\": \"increases\", \"source\": 31, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 32, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2015 Jun;36(6):2018-23. doi: 10.1016/j.neurobiolaging.2015.03.001\", \"reference\": \"25865441\", \"type\": \"PubMed\"}, \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Abeta42+ subjects but not in Abeta42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"hash\": \"20375a264d8c8c5eb4893054811491c1df4fda89560721a37772406f90ddbbddd014ad6ca23c932cf20f5ccddfd6cef24285cd0dcfe75afbc34159347723b428\", \"key\": 0, \"line\": 45241, \"relation\": \"increases\", \"source\": 33, \"target\": 0}, {\"annotations\": {\"Cell\": {\"monocyte\": true, \"neutrophil\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\", \"type\": \"PubMed\"}, \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"hash\": \"ce87734784feaf2736ff7a47899c05f8790617453784d8f233a51880672aac09661be5262dea4dca491fd9f5902d9d43331cb6d38d46a5477205c0340bf855a4\", \"key\": 0, \"line\": 46433, \"relation\": \"increases\", \"source\": 34, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\", \"type\": \"PubMed\"}, \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"hash\": \"62a714940bbf2a987459f0f1ea2ba6cb18225d84a005ef4acd724471be68904eac690c753b4fe8c67fde307a6614b96850223bf4abc9b6697f24e3a1d8eabe78\", \"key\": 0, \"line\": 333, \"relation\": \"increases\", \"source\": 35, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\", \"type\": \"PubMed\"}, \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"hash\": \"9507be44f48975feeaa091f63e9d8065fbf49cce63403b085cca409b26add27293dc703f872812fb21242402a4c7050be8ebcd30615e2ed408200d2d3b8d2bbf\", \"key\": 0, \"line\": 334, \"relation\": \"increases\", \"source\": 36, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"Acta Neurol Scand. 2011 Jun;123(6):419-23. doi: 10.1111/j.1600-0404.2010.01400.x.\", \"reference\": \"20597865\", \"type\": \"PubMed\"}, \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (Abeta). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"hash\": \"57f2a1c2cf5b5b4426cc8f0a1fd58882f31305c97e98a7445ec47945e0f494953682109ad1530bdd15b40a60b98d9e3f1f09b01bc18ef2ec13c2ade8b399f380\", \"key\": 0, \"line\": 1394, \"relation\": \"increases\", \"source\": 37, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"APOE subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"hash\": \"4ff37aed1dff031464f7317722e71837fd1c645d64b19aba2f50864adbe345c71e0fdd9889bd44f7c5d3cff94791b2d03960951ea3c3c514e6af8af12053043e\", \"key\": 0, \"line\": 5448, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"hash\": \"9611c060f68621342e28f093e7e53a7c40cbf068840fca398e68181c4d976d97d158c1a28f5ed46275bfccf16e7dd8e564fffb236cedf71dd3dd1383fd1a0e5e\", \"key\": 1, \"line\": 42750, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"APOE subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Brain Nerve. 2010 Jul;62(7):757-67.\", \"reference\": \"20675880\", \"type\": \"PubMed\"}, \"evidence\": \" Recently, many amyloid PET-positive and cognitively normal subjects were found in PiB-PET studies. PiB-PET studies on healthy subjects have also shown that apolipoprotein (APO) E4 boosts the accumulation of amyloid-beta and may consequently accelerate the pathogenesis of AD\", \"hash\": \"7f3ce52e7f54543dfea6c18dc72c228244e884d1572a5d79b9921bf2b0426de8258e3a7c4a7ebe04c3e7c1be38e099ba44ac9f17e182b6d51f7a6e0e7f551f92\", \"key\": 2, \"line\": 4138, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"APOE subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Opin Lipidol. 2007 Jun;18(3):325-32.\", \"reference\": \"17495608\", \"type\": \"PubMed\"}, \"evidence\": \"Apolipoprotein E is the main lipid carrier in the brain and the best-established risk factor for late-onset Alzheimer's disease. Intracellular cholesterol levels influence the generation of amyloid-beta peptides, the toxic species thought to be a primary cause of Alzheimer's disease. Finally, compounds that modulate cholesterol metabolism affect amyloid-beta generation.\", \"hash\": \"af674edfaee5ff5ab94d0df254f26eeb5dd4885a69ec3804d056ba2341b91a50cbd75ef7bad92f88c1f89894fecc18321317b7252598c5bc011539c9ab67828f\", \"key\": 3, \"line\": 4797, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Amyloidogenic subgraph\": true, \"Synuclein subgraph\": true}}, \"citation\": {\"name\": \"Nat Neurosci2008\", \"reference\": \"18297066\", \"type\": \"PubMed\"}, \"evidence\": \" ApoE was not protective, but was injurious, as deletion of ApoE delayed the neurodegeneration caused by alpha-synuclein and suppressed the accumulation of Abeta. \", \"hash\": \"4fb645415d28170ac75c96174694e80a1ba2abf6b964d41ff65a80aa2990519f48a5678be85835f0f5eb542a648cf44eed46a486e5f16e9a7b881444e9711098\", \"key\": 4, \"line\": 25171, \"relation\": \"increases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron2008\", \"reference\": \"18549781\", \"type\": \"PubMed\"}, \"evidence\": \"We report that ApoE plays a role in facilitating the proteolytic clearance of soluble Abeta from the brain. \", \"hash\": \"5f0d902cad6a19acbde3e4e7d729c5e85c51dac22e97d9b35de9f5261787f86f526f3d2fe1c419f05368baf1b60d13b83ae2ec91768c067becf8c13395f4129e\", \"key\": 5, \"line\": 25182, \"relation\": \"decreases\", \"source\": 38, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\", \"type\": \"Online Resource\"}, \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"hash\": \"a369b8b544826472c182060c07667da24390b7a2451c464e6bfb804b21cd65fca0edb0ae687c13ce535039bd1ab2be4c8de41d27c54fbe85c7fd31f0a7839c3f\", \"key\": 0, \"line\": 7416, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"CellStructure\": {\"Centrosome\": true}, \"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Reelin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686\", \"type\": \"PubMed\"}, \"evidence\": \"Upon binding, LRP8 transduces reelin signaling during neuronal development, and recent evidence has indicated that it interacts with the NR2A and NR2B subunits of NMDA receptor, being involved in neuronal functions such as maturation of NMDA receptor composition in the hippocampus, and the regulation of long-term potentiation. Subsequently, a novel interaction between reelin and AbetaPP was discovered, leading to increase in the cell surface levels of AbetaPP and affecting AbetaPP processing and Abeta production. It was shown that reelin signaling in excitatory synapses can restore normal synaptic plasticity, which is impaired by oligomeric Abeta peptides at concentrations within the range detectable in the brains of AD patients. At high concentrations of Abeta peptides, reelin can no longer overcome the Abeta-induced functional suppression, and this condition coincides with a complete blockade of the reelin-dependent phosphorylation of NR2 subunits in NMDA receptors. This reversal requires the LRP receptor-dependent activation of tyrosine kinases of the Src family.\", \"hash\": \"ce6551d5f1ac212df016c7fea7dd6863abb79c54feaebeed7497b0eb80396879cab283b7fcd56a8763ec2f77c37b3406a6c35b274ae5340be974bcd8c2399b74\", \"key\": 1, \"line\": 2208, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"intracellular\", \"namespace\": \"GOCC\"}, \"toLoc\": {\"name\": \"extracellular space\", \"namespace\": \"GOCC\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Biofactors. 2012 Mar;38(2):98-106. doi: 10.1002/biof.199. Epub 2012 Mar 23.\", \"reference\": \"22447723\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's disease (AD), one of the major causes of disability and mortality in Western societies, is a progressive age-related neurodegenerative disorder. Increasing evidence suggests that the etiology of AD may involve disruptions of zinc (Zn) homeostasis. This review discusses current evidence supporting a potential role of Zn and zinc transporters (ZnTs) in processing of the amyloid beta protein precursor (APP) and amyloid beta (Abeta) peptide generation and aggregation.\", \"hash\": \"0adb16b8d7b9df4cc86126b467c92a06377f9db241939e9d0e4b1f80936b40fa2aaecd44673dde752be340b798558016ab3f3efbe23fb9126f33618e6737cdb6\", \"key\": 2, \"line\": 2334, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"APOE subgraph\": true, \"Cholesterol metabolism subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2010 Jan 1;5(1):e8556\", \"reference\": \"20049331\", \"type\": \"PubMed\"}, \"evidence\": \"The low-density lipoprotein receptor (LDLR) has the highest affinity for apoE and plays an important role in brain cholesterol metabolism.These data suggest that increased APP expression and Abeta exposure alters microtubule function, leading to reduced transport of LDLR to the plasma membrane. Consequent deleterious effects on apoE uptake and function will have implications for AD pathogenesis and/or progression\", \"hash\": \"ca42ac90c016deb88ce9ee2d5347526c45a10d046a3a73e79c5075916ee70406977c79feef83471691d11c2c88d7b83f76a05668dfc9f22713bc640c9a9f11dc\", \"key\": 3, \"line\": 3499, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"DYRK1A subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\", \"type\": \"PubMed\"}, \"evidence\": \"DYRK1A phosphorylates APP at Thr688 and enhances APP amyloidogenic cleavage, resulting in increased level of Abeta40/42, formation of toxic Abeta oligomers and deposition of fibrillar amyloid in plaques. Brain amyloidosis contributes to loss of neuronal function and possibly also to neuronal loss. Moreover, elevated levels of Abeta may up-regulate DYRK1A expression and enhance the contribution of overexpressed DYRK1A to neurofibrillary degeneration and -amyloidosis.\", \"hash\": \"0a2db9da1c57c10c467a7b03d50bc5581d9f89b24673ace5a4aecde059af59cfd7dc49ab4d13bb94488066806aa273f49469b833b53daa8da677006929678019\", \"key\": 4, \"line\": 4069, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Endoplasmic reticulum-Golgi protein export\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\", \"type\": \"PubMed\"}, \"evidence\": \"Depletion of Hrs and Tsg101, acting early in the multivesicular body pathway, retained APP in early endosomes and reduced Abeta(40) production. Conversely, depletion of CHMP6 and VPS4, acting late in the pathway, rerouted endosomal APP to the TGN for enhanced APP processing. We found that VPS35 (retromer)-mediated APP recycling to the TGN was required for efficient Abeta(40) production.\", \"hash\": \"1b0ee9796535f9a41e815a4c5ca57aae59c319b72eb3049acfdb253aabcaffb7d1ed1548fcc6610e942739a1fbf5af7550bec510e632b3c41f9e006fd0378423\", \"key\": 5, \"line\": 4248, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"trans-Golgi Network\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Endoplasmic reticulum-Golgi protein export\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A. 2012 Jul 24;109(30):E2077-82. Epub 2012 Jun 18\", \"reference\": \"22711829\", \"type\": \"PubMed\"}, \"evidence\": \" An interruption of the bidirectional trafficking of APP between the TGN and endosomes, particularly retromer-mediated retrieval of APP from early endosomes to the TGN, resulted in the accumulation of endocytosed APP in early endosomes with reduced APP processing. These data suggest that Abeta(40) is generated predominantly in the TGN, relying on an endocytosed pool of APP recycled from early endosomes to the TGN.\", \"hash\": \"844a7502f34af7b79cfe398578353a62d67a83ea45ecf9f463f058baef0964aee4aa9f83cd2f1b66c50ece01330c411b540ccdbde942538d2faab73a36de8034\", \"key\": 6, \"line\": 4256, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"trans-Golgi Network\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\", \"type\": \"PubMed\"}, \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"hash\": \"9027fa4e243a4b3d9d2ed99c83a3e7a4b126ffbf1e4a9d69c305f899d88240fd659ea470f941ef6a31ea2701b99278931a7e154bccdfdaf009d4a9ba47ca0b59\", \"key\": 7, \"line\": 4819, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Cell Surface Extensions\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Synaptic vesicle endocytosis subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Cirrito et al. [26] also show that synaptic activity-induced increase in endocytosis drives more APP into the endocytic compartment, ultimately resulting in increased A production and release. A produced in the endocytic pathway is then brought to the cell surface where it is released into the extracellular fluid [70]. Inhibition of endocytosis reduces APP internalization and reduces A production and release in cell lines\", \"hash\": \"3d501898432add2aefa8adef699ce67abbfd4ac86974391efe968c52e7ba8b7e0d628a85eaf4ac043e0838471e253df38de46bf41d67e061cb4895066fc92983\", \"key\": 8, \"line\": 6464, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Cell Membrane\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Brain Res Mol Brain Res2001\", \"reference\": \"11744168\", \"type\": \"PubMed\"}, \"evidence\": \"Expression of familial Alzheimer's disease (FAD) mutants of APP in primary neurons causes both intracellular accumulation of the C-terminal beta-secretase cleavage product of APP and increased secretion of Abeta, and eventually results in apoptotic death of the cells.\", \"hash\": \"b7957b16c1fc01a385eca570c8fb85ad0c20454d8bd72bb98aba4680cb5917a44e1355d29dc69a460ab8c7aca625a753471cb0e03ba914f83d23a5f565bfc1a3\", \"key\": 9, \"line\": 25633, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\", \"type\": \"PubMed\"}, \"evidence\": \"The formation of beta A4 amyloid in the brains of individuals with Alzheimer's disease requires the proteolytic cleavage of amyloid precursor protein.\", \"hash\": \"04ae59fb12a21f5bb6a4ba12fde83857e6d2a568c6e9db7076adf9979d698e378cfc5f5e0c6aa01ce7db33c251c8ff00acdf6adec827e734753bc24409f9769f\", \"key\": 10, \"line\": 33241, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"International journal of biochemistry and cell biology1999\", \"reference\": \"10605825\", \"type\": \"PubMed\"}, \"evidence\": \"Several lines of evidence suggest that cathepsin D, the major lysosomal/endosomal aspartic protease, may be involved in this process. \", \"hash\": \"14c8b50c94e80e48877b503576ea5cfbb766212e20871e1f86b12f968831969adef79aa7324e75530e848e32f83f81f72ab8a035777fdef08f980bfdbf1e85fd\", \"key\": 11, \"line\": 33245, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochem J. 2012 Mar 1;442(2):413-22.\", \"reference\": \"22115042\", \"type\": \"PubMed\"}, \"evidence\": \"The death of cholinergic neurons in the cerebral cortex and certain subcortical regions is linked to irreversible dementia relevant to AD (Alzheimer's disease). Although multiple studies have shown that expression of a FAD (familial AD)-linked APP (amyloid Abeta precursor protein) or a PS (presenilin) mutant, but not that of wild-type APP or PS, induced neuronal death by activating intracellular death signals, it remains to be addressed how these signals are interrelated and what the key molecule involved in this process is. In the present study, we show that the PS1-mediated (or possibly the PS2-mediated) signal is essential for the APP-mediated death in a gamma-secretase-independent manner and vice versa. MOCA (modifier of cell adhesion), which was originally identified as being a PS- and Rac1-binding protein, is a common downstream constituent of these neuronal death signals. Detailed molecular analysis indicates that MOCA is a key molecule of the AD-relevant neuronal death signals that links the PS-mediated death signal with the APP-mediated death signal at a point between Rac1 [or Cdc42 (cell division cycle 42)] and ASK1 (apoptotic process signal-regulating kinase 1).\", \"hash\": \"757f997e96efd6c3d42c5e8a0cc0d12101a594d65f7a38b34249d538fd4efb80fb6b9dbbcf76b78468d0038689ed051e4a23b9c449c899368c5ff805a0acb468\", \"key\": 12, \"line\": 34699, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Axons\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Axonal transport subgraph\": true}}, \"citation\": {\"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\", \"type\": \"PubMed\"}, \"evidence\": \"Defects in axonal transport are an early pathological feature in Alzheimer's disease and defective APP transport is known to increase ABeta production. We show that calsyntenin-1 and APP are co-transported through axons and that siRNA-induced loss of calsyntenin-1 markedly disrupts axonal transport of APP. Thus, perturbation to axonal transport of APP on calsyntenin-1 containing carriers induces alterations to APP processing that increase production of ABeta. Together, our findings suggest that disruption of calsyntenin-1-associated axonal transport of APP is a pathogenic mechanism in Alzheimer's disease.\", \"hash\": \"ef628714488254f08f4f81ba4295ec7fc54fc9b71a554e4875eb5602d843737a063f453e6272f81850b2b99e89862a72c6f4dabec231e8b218e0c48794c4a84f\", \"key\": 13, \"line\": 35118, \"relation\": \"decreases\", \"source\": 39, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Axons\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Cell Membrane\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"BMC Neurosci. 2008 Dec 3;9 Suppl 2:S2.\", \"reference\": \"19090990\", \"type\": \"PubMed\"}, \"evidence\": \"Iron was further demonstrated to modulate expression of the Alzheimer's amyloid precursor holo-protein (APP) by a mechanism similar to that of regulation of ferritin-L and -H mRNA translation through an iron-responsive element (IRE) in their 5' untranslated regions (UTRs). Here, we discuss two aspects of the link between iron and AD, in relation to the recently discovered IRE in the 5'UTR of APP mRNA. The first is the physiological aspect: a compensatory neuroprotective response of amyloid- protein (ABeta) in reducing iron-induced neurotoxicity. Thus, given that ABeta possesses iron chelation sites, it is hypothesized that OS-induced intracellular iron may stimulate APP holo-protein translation (via the APP 5'UTR) and subsequently the generation of its cleavage product, ABeta, as a compensatory response that eventually reduces OS.\", \"hash\": \"dbc9950ba819372952ee7c5617ad91718f0e81017a81c3fda8423ddc90fcda60905995e5fc95bbdbbb6fe920746e950aaa2ae7a137a218a4385cc587e2e2971a\", \"key\": 14, \"line\": 36890, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2015;12(6):563-71.\", \"reference\": \"26027813\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, based on timing of changes in cortical thinning and Abeta deposition such as Abeta deposition after cortical thinning; cortical thinning after Abeta deposition, or concurrent Abeta deposition and cortical thinning, we identified three types of relationships between cortical thinning and Abeta deposition: (1) Abeta-associated cortical thinning; (2) Abeta-independent cortical thinning; and (3) Abeta deposition only without cortical thinning. Taken together, these findings suggest that Abeta-weighted cortical thickness values can be used as an objective biomarker of structural changes caused by amyloid pathology in the brain.\", \"hash\": \"3bea54be6c07667c3543ea95fe7865bf3dfda0a0044e554e5cab5282d2413de8a72228610ea32301c552d050e72318e5bef108d5e8dd7053aa3c45c2681dd518\", \"key\": 15, \"line\": 45315, \"relation\": \"increases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"CellStructure\": {\"Golgi Apparatus\": true}, \"Subgraph\": {\"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\", \"type\": \"PubMed\"}, \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"hash\": \"54b330744baef4593d181b257a65dbf66796d885165a8fd34b2e4e2d87da047d5314cd7d3ace7f94f5136618ecc552128fd5172362bbd3a721027c5d8b8508db\", \"key\": 16, \"line\": 54637, \"relation\": \"decreases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\", \"type\": \"PubMed\"}, \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"hash\": \"affb6ede9e36bc0177a6bcec83f1fd21d00a86073b078c71861a85ef86dbd760b35c2e00bbd9e0f2064c7aa7f7e0a39512f59de3d290705cb61be4738f942562\", \"key\": 17, \"line\": 54643, \"relation\": \"decreases\", \"source\": 39, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"BMB Rep. 2010 Oct;43(10):656-63.\", \"reference\": \"21034527\", \"type\": \"PubMed\"}, \"evidence\": \"Although the role of AICD as a gene transcription inducer is still controversial, several targets of AICD- or APP-mediated transcriptional activation are reportedly regulated by APP nuclear signaling, including APP itself, BACE, Tip60, GSK-3, p53, Mn-SOD, KAI1, Neprilysin, transgelin, a2actin, S100a9, and other genes\", \"hash\": \"691407333deb4313ad6f04885d0598e284e77a93c309fb574ffee01452b632edb91665905149b8b6370c11f8887fc8b86c1ab8c17342ea9b0e460000325bceac\", \"key\": 0, \"line\": 36969, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 39, \"subject\": {\"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\", \"type\": \"Online Resource\"}, \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"hash\": \"cd83fe5946b9dec94f2b8acf2a6ad7c7d18b4bf5dd0efd5915de5a1ce46b67ed7b92cb37cd505c193b69e31cd5e1b14278669a73fb63928c9ed397ecc5138950\", \"key\": 0, \"line\": 7418, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2008\", \"reference\": \"17995932\", \"type\": \"PubMed\"}, \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"hash\": \"cc795b46c19e1e374cc33509ef38b36402402c8536afa60757ae83a4fa1353598dcea3b5cd97ac20b619d7bb7e9295e70edf902faa7fe31ca73852dff6aa1b78\", \"key\": 1, \"line\": 25969, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"EMBO journal2002\", \"reference\": \"12374741\", \"type\": \"PubMed\"}, \"evidence\": \"This cleavage is similar to the PS-dependent gamma-secretase cleavage of the beta-amyloid precursor protein (betaAPP). \", \"hash\": \"1de8037509b87ebd76474b19a5c1ae7ebe6abb484fe53d994e2ab75b81cdcca0d274222463174ec38ef771e6a09758337491e85394c970efb1f9e0b5458c73f1\", \"key\": 2, \"line\": 30759, \"relation\": \"increases\", \"source\": 40, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett2002\", \"reference\": \"11814648\", \"type\": \"PubMed\"}, \"evidence\": \"Presenilin-1 (PS1) is intimately involved in cleavage of amyloid precursor protein to form beta-amyloid peptides, certain forms of which aggregate in the brains of patients with Alzheimer's disease (AD).\", \"hash\": \"28bd8592e5b18de5ec3033617141baaa1492003900985e3da1200b1dc532d1182154bfb507393d3c30a776127547d65ee01b9b2da69e863f3c907bb7711b53ff\", \"key\": 3, \"line\": 30899, \"relation\": \"increases\", \"source\": 40, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol2000\", \"reference\": \"10670705\", \"type\": \"PubMed\"}, \"evidence\": \"It was hypothesized that PS1 might directly cleave APP.\", \"hash\": \"9f170114be09b1beecc11f11bfa30b7bebece6bd21358ee671d06d22aaefb809a2ba510aeb7e4ffc00966c111121880495de98448dfdb01910b6374339994545\", \"key\": 4, \"line\": 30939, \"relation\": \"increases\", \"source\": 40, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun1997\", \"reference\": \"9344855\", \"type\": \"PubMed\"}, \"evidence\": \"Therefore, our results show that PS1 binds to APP directly and suggest that the PS1 protein itself is involved in the metabolism of beta-amyloid peptide.\", \"hash\": \"28d9533232df8f55528e24eb2528646b401bf149a6c30edf7949fa6a809486243896aca0a351b293220ef0049e5f40a5a1a2210b4b66f1777e77fe2532c534c6\", \"key\": 5, \"line\": 30949, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry1998\", \"reference\": \"9843412\", \"type\": \"PubMed\"}, \"evidence\": \"Our results indicate PS1 and APP can interact in the ER and Golgi, where PS1 is required for proper gamma-secretase processing of APP CTFs, and that PS1 mutations augment Abeta42 levels principally in Golgi-like vesicles.\", \"hash\": \"9c0e50faedeee26cfbfeffdbe077dbd31dde77a7ca71c399c56011489af1888b03a47b37ca4ce56302bbdcd35224d90b36019d075445a8b7cb5452ced6f2cf66\", \"key\": 6, \"line\": 30960, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem1999\", \"reference\": \"10037471\", \"type\": \"PubMed\"}, \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"hash\": \"61c5eb56d6305f5314e283430101f4242b7b0d9c4d2148ca13ae64bb1c3a21d11e7670bbc501f381627383ed5fe27501dbbdb2658b849a37dbe90218db1116ff\", \"key\": 7, \"line\": 31153, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2002\", \"reference\": \"12147673\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"hash\": \"cf051825f25363ca30a55ff4ef4f399ff56f0c377dbb9bc8348dad6b8a3a04895e1f75aeb6229e268165d4ad4249ae6a6d7e7d252bbd47f214b31fdf6ec87fe2\", \"key\": 8, \"line\": 31273, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\", \"type\": \"PubMed\"}, \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"hash\": \"8eac060b387cb98878cf3b645e5256deabe6f0e00bbb1bc66980f3a21b809e024d80f04ab69ca9e3117e470216925ba6260040655773a8e48551bc5711da538c\", \"key\": 9, \"line\": 31298, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Encode_Feature_Types\": {\"Promoter\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2005 Sep 30;386(2):69-71.\", \"reference\": \"16040194\", \"type\": \"PubMed\"}, \"evidence\": \"hypomethylation of the promoter of the presenilin 1 (PS1) gene, which will lead to overexpression of presenilin 1 and, consequently, to increased Abeta(1-42) (Abeta42) formation \", \"hash\": \"dfe73debd5f8780a649e30cf2eb06b8f919aff16c3848fc4811b20e31fa369d4fde781e0f1cd88eeef5935b06dbb658483bb4b1ba1824aed77fb283eedd8ba7e\", \"key\": 10, \"line\": 44147, \"relation\": \"increases\", \"source\": 40, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\", \"type\": \"Online Resource\"}, \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"hash\": \"e9205eee245a4f7b2365bbef506b2be373e2bdce91ba8760213bd2f2e07457bbf7b8113dea026ca04bf80c551f5bcb0182132d8513124961a204150ce3c4a4b9\", \"key\": 0, \"line\": 7419, \"relation\": \"increases\", \"source\": 41, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem1999\", \"reference\": \"10037471\", \"type\": \"PubMed\"}, \"evidence\": \"Missense substitutions in the presenilin 1 (PS1) and presenilin 2 (PS2) proteins are associated with early-onset familial Alzheimer's disease.\", \"hash\": \"235ecdd24aa63937c941563d3b21bb739b307ddfec10a919327d5a35b97a87c0022e01a06629755c3ec8444444e3c115c92c9a0192863afa60120bfdb7409174\", \"key\": 1, \"line\": 31154, \"relation\": \"increases\", \"source\": 41, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2002\", \"reference\": \"12147673\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid beta-peptide (Abeta) is generated by the consecutive cleavages of beta- and gamma-secretase.\", \"hash\": \"4dd471c53978cf13defa9d0a48c426f3e4f19473d73e5d03d5ffdfe01d27be0db368159887968d9b6e0cb0024f4edad17a55bb5f96b68e8634b7711f1c5b3575\", \"key\": 2, \"line\": 31274, \"relation\": \"increases\", \"source\": 41, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Neurochemistry2003\", \"reference\": \"12603837\", \"type\": \"PubMed\"}, \"evidence\": \"Abundant biochemical and genetic evidence suggests that presenilins are catalytic components of gamma-secretase, the protease responsible for generating the Alzheimer amyloid beta-protein.\", \"hash\": \"c7335d98dce1f3436a68d7548b357821b6052961b42a2c45c40a0d368926c7c884fca77a050b50ab8238233cec6919a15fa64ee51309cb2b7f30a921c580103a\", \"key\": 3, \"line\": 31299, \"relation\": \"increases\", \"source\": 41, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true, \"Ubiquitin degradation subgraph\": true}}, \"citation\": {\"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\", \"type\": \"PubMed\"}, \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"hash\": \"35c86c7b4187254a44b41ace9c40b6f02cebb77f5ec65f24c1aea4684e0fd319c5559207e2ad6d25273444d457dc9f11a5200ada0cc1d6df619d2c6ac97ac3fe\", \"key\": 0, \"line\": 7633, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}, \"UserdefinedCellLine\": {\"App transgenic\": true}}, \"citation\": {\"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\", \"type\": \"PubMed\"}, \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"hash\": \"a1285d90e9a0e37e8292450927ca369a11a036d30cd5e1330d21c7c80fc00839032436fb5003dbc44ec973e5b1331a79ec9319216a8ff9100c9b408990a88514\", \"key\": 1, \"line\": 42833, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\", \"type\": \"PubMed\"}, \"evidence\": \"The peroxisome proliferator-activated receptor- (PPAR-) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR- agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) 4 allele\", \"hash\": \"4d6417496a4e65e6f05a440f216b62a864908a2f7438c9070a67c16b1661263addeaa841728d4bdf13c5496356f729e9e43fac4f7eae2fb26eb32e478a8f9b7a\", \"key\": 2, \"line\": 5055, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\", \"type\": \"PubMed\"}, \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"hash\": \"2041cc4eadebce2d0785206ebe1f3bc4bbd33bee28158af3051343e9d6efd4e3d6c39da4c8b99d7b9b002da9e631f9e431455fe344291bef302171015ac3c3f6\", \"key\": 3, \"line\": 32549, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2008 Jun 18;5:25\", \"reference\": \"18564425\", \"type\": \"PubMed\"}, \"evidence\": \"these findings suggest the existence of a down-regulation of PPAR under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"hash\": \"52c7af7eeb14005a2656123f368f946df9b41d787844b225b11fde7d525598f5eca34364b49e023dd86ed2db163a622d1857f93f0e20733a63d525b3895ac802\", \"key\": 4, \"line\": 37387, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2011 Mar;32(3):547.e1-6. Epub 2009 Aug 6.\", \"reference\": \"19660836\", \"type\": \"PubMed\"}, \"evidence\": \"The peroxisome proliferator-activated receptor- (PPAR-) is a ligand-inducible transcription factor that suppresses microglial inflammatory responses and inhibits amyloid beta (Abeta) production through promoting cholesterol efflux from glial cells. PPAR- agonists have been advanced as a new disease altering approach to Alzheimer's disease (AD), with rosiglitazone therapy having improved cognition in those AD patients that did not possess an Apolipoprotein E (APOE) 4 allele\", \"hash\": \"c05ab175b50ea7c28dbd4819fb4dd3b475bcdc3e1c101d0a6373c12a9378cf77eb03e2d530cf2ca2c5650d43b5e9f872ad6792dd292ff5b4f797ff51778f137f\", \"key\": 5, \"line\": 42701, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"Medical Hypotheses2006\", \"reference\": \"16828233\", \"type\": \"PubMed\"}, \"evidence\": \"PPARgamma activity decreases Abeta production by promoting harmless catabolism of amyloid precursor protein while blocking the up-regulatory impact of cytokines on beta-secretase expression.\", \"hash\": \"2041cc4eadebce2d0785206ebe1f3bc4bbd33bee28158af3051343e9d6efd4e3d6c39da4c8b99d7b9b002da9e631f9e431455fe344291bef302171015ac3c3f6\", \"key\": 6, \"line\": 42920, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2008 Jun 18;5:25\", \"reference\": \"18564425\", \"type\": \"PubMed\"}, \"evidence\": \"these findings suggest the existence of a down-regulation of PPAR under inflammatory conditions, which would result in an increase in BACE1 transcription and Abeta generation. \", \"hash\": \"429c2b0547d0f970312071ffc2c023b6c4032df6ccb96dba221c1898fc2aba1b107c1c7635cb27199376f98cb16632c40a33b866191d4bb9f272757a271e3540\", \"key\": 7, \"line\": 43011, \"relation\": \"decreases\", \"source\": 42, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"1f152646e5eeb466c01b4a76f0b889567c1eff9944d55d6062b38636e48d94cd97bd042c3d2da094729a5e7247fdb326d45b074c353da3beb646493969aae698\", \"key\": 0, \"line\": 38439, \"relation\": \"decreases\", \"source\": 42, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"02e19a7ab21de9829336cd1cb4b03667dc4f323bc3b9d794d98bd78f9fc17938989417f18096e0bc8a5ee19203bff29a1a8379a7a07b66a6a46e50259b17f723\", \"key\": 0, \"line\": 38438, \"relation\": \"decreases\", \"source\": 42, \"target\": 21}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b0af811fee9172729953f98d376a105e8f48988f42f2527511c165ff2c106b4e745856adcae2deeedfd40904174e510c974fa211f56aa9eb157fe8d0e01a29b9\", \"key\": 0, \"line\": 38450, \"relation\": \"decreases\", \"source\": 42, \"target\": 23}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5fea2af2bec805f6c3e7f7de7f8b1d71453ff2c868492da56f16c7727cbfe0659d056ebb51827c0f3c835f21c588ca9226f2bb64585ec25a0da9d1972e7ba338\", \"key\": 0, \"line\": 38448, \"relation\": \"decreases\", \"source\": 42, \"target\": 26}, {\"annotations\": {\"Subgraph\": {\"Peroxisome proliferator activated receptor subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"9eb16750fbe14b51db2ca432a84ff5d10fe167990077f49498bf2255cda6c3ebf5c680cd8da51a3bcc32107f8f3a972242b9c4ec33c12203b0e8ea7447a6536e\", \"key\": 0, \"line\": 38449, \"relation\": \"decreases\", \"source\": 42, \"target\": 32}, {\"annotations\": {\"Subgraph\": {\"Autophagy signaling subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2006 Dec 20;26(51):13279-86.\", \"reference\": \"17182778\", \"type\": \"PubMed\"}, \"evidence\": \"we found that the Nogo-66 receptor (NgR) interacts physically with both Abeta and the amyloid precursor protein (APP). The inverse correlation of Abeta levels with NgR levels within the brain may reflect regulation of Abeta production and/or Abeta clearance.\", \"hash\": \"cda5368c3b06e097c04365eb28e6394e2a5a283a3ac91a2dd8b7156619eb25ea2487d8065bdc732df633612cd4587dd3529cde9b0283f8b86e4b201e307e1244\", \"key\": 0, \"line\": 37186, \"relation\": \"decreases\", \"source\": 43, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Apr;6(2):118-31\", \"reference\": \"19355846\", \"type\": \"PubMed\"}, \"evidence\": \"Lower-expression of PS1 and over-expression of IDE or NEP may be helpful in potentially lowering brain Abeta levels in subjects with AD\", \"hash\": \"d5c5e6585484d31f082c8997f9f7bfc27853514c1508a17d74e0e29164e362328bbb7f9dbb63c94c2314d3671587c35e48007ea7da703745c239e0f520687c32\", \"key\": 0, \"line\": 37196, \"relation\": \"directlyDecreases\", \"source\": 44, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Endothelin subgraph\": true, \"Insulin signal transduction\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\", \"type\": \"PubMed\"}, \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) Enzy matic degradation by activated microglia or by insulin de grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptor related protein (LRP) either directly or after binding to apol ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"hash\": \"0dfa447c9cf4e2a3ebd16ed404e13f85150567d9d4b362efa593f08a1de5afd7e9273c6bfc26ed1eab511dbda89fc1f62d6ec5435eba58e2af36a73f5f0fbd2c\", \"key\": 1, \"line\": 8734, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 44, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\", \"type\": \"PubMed\"}, \"evidence\": \"Since Abeta is secreted extracellularly, and deposits outside the neuronal cells in the AD brain, we took an approach to screen any secreted protease(s) in neuronal and non-neuronal cell culture media for the ability to degrade Abeta. Among all secreted proteases from the cells, only IDE degraded Abeta. We found that under physiological conditions IDE is secreted at high levels from the microglial cells, and degrades Abeta extracellularly [73]. Purified IDE from rat liver and brain was shown to degrade Abeta effectively. IDE is present in the soluble fractions from human brains, and binds and degrades Abeta specifically [57] and [71]. Primary cultured neurons were also shown to clear Abeta via extracellular IDE as well as IDE on the cell surface [90]. IDE from brain homogenates degrades different forms of Abeta: Abeta40, Abeta42 and an Abeta mutant in one type of AD (Dutch Variant 1-40 Q) [61] and [71]. Abeta42 is the longer form of Abeta and more abundant in the AD brain.\", \"hash\": \"8df12621d34ddfbe947dc14f6f229af9a976ed796eade4c75b937b9d0161d81f9150be01d7afcb4363b47111a8553e8c9196e7d87bad40facebdeefc27fef716\", \"key\": 2, \"line\": 8777, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 44, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Nitric oxide subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmune Pharmacol. 2012 Mar;7(1):165-72\", \"reference\": \"22227962\", \"type\": \"PubMed\"}, \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid  degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid  clearance.\", \"hash\": \"d136bd681ccda779ea6b37fd44f96a98632015710e7ca9fccc35890af10bb4b971782e1c5a32ba9a1b913991de66ff02fd72be1a5c9d33834e7ccc96b7460275\", \"key\": 3, \"line\": 22017, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 44, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Nitric oxide subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmune Pharmacol. 2012 Mar;7(1):165-72\", \"reference\": \"22227962\", \"type\": \"PubMed\"}, \"evidence\": \"These data suggest that NOS2 upregulation impairs amyloid beta degradation through negative regulation of IDE activity and thus loss of NOS2 activity will positively influence amyloid beta clearance.\", \"hash\": \"0465fc4d42d44445ef2edffbf183c2a07b573695b1e7d8bb687b36b5e01dd271d1c6c016b494efd4ff9324c911ded7b691cd4858596dec77e40ed4141fa41442\", \"key\": 4, \"line\": 37797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 44, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Apr;6(2):118-31\", \"reference\": \"19355846\", \"type\": \"PubMed\"}, \"evidence\": \"Lower-expression of PS1 and over-expression of IDE or NEP may be helpful in potentially lowering brain Abeta levels in subjects with AD\", \"hash\": \"d3b63893857f347ee1cf92fcab8b7ee390f611aeef2be2464d0219cd59d7a8065fa14d74a6a66eb0964ec3630c030e331c18711df962ed3af3a2de37aa39ce1a\", \"key\": 0, \"line\": 37197, \"relation\": \"directlyDecreases\", \"source\": 45, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"MAPK-ERK subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2009 Dec 9;29(49):15472-8\", \"reference\": \"20007471\", \"type\": \"PubMed\"}, \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"hash\": \"d2d0a1438a70b640470df446911cd08e723db24bd777236b339719efaccd3b06e1f0fe8c51d6a14cf72f11748690bac3b8c170578cbee7e6b730cf6049ec4539\", \"key\": 0, \"line\": 37243, \"relation\": \"directlyDecreases\", \"source\": 46, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"MAPK-ERK subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2009 Dec 9;29(49):15472-8\", \"reference\": \"20007471\", \"type\": \"PubMed\"}, \"evidence\": \"Using screening approaches in primary neurons, we identified brain-derived neurotrophic factor (BDNF) as a major inducer of Sorla that activates receptor gene transcription through the ERK (extracellular regulated kinase) pathway.These findings demonstrate that the beneficial effects ascribed to BDNF in APP metabolism act through induction of Sorla that encodes a negative regulator of neuronal APP processing\", \"hash\": \"e2638964833345731f0e809d79d45004b13aeb1ecf94cca9f94c3dadaa35f7b93df2b14b07e1463ff5a01283e8ceb73a589241022a4575dd7fb72b7f66f7978d\", \"key\": 0, \"line\": 37244, \"relation\": \"directlyDecreases\", \"source\": 47, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"BMC Syst Biol. 2012 Jun 22;6(1):74. [Epub ahead of print]\", \"reference\": \"22727043\", \"type\": \"PubMed\"}, \"evidence\": \"SORLA is proposed to act as a retention factor for APP in the TGN, preventing the release of precursor molecules into the processing pathways. Consequently, over-expression of SORLA in neurons prevents the targeting of APP from TGN to the cell surface and to endosomes and reduces the production of Abeta peptides [9-11]. The importance of SORLA for AD is further supported by low levels of receptor expression seen in patients suffering from the disease\", \"hash\": \"dd31508a3f53156f614ef4520392f1ac0c5b19349cae069f0e3fecbba4015f7a76dd1e18f8991f5adca2acc371d72fac7f4ee856e10c70b300a41907c809acd8\", \"key\": 1, \"line\": 35153, \"relation\": \"directlyDecreases\", \"source\": 47, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2012 Apr 3. doi: 10.1038/mp.2012.15. [Epub ahead of print]\", \"reference\": \"22472873\", \"type\": \"PubMed\"}, \"evidence\": \"SORL1 encodes the Sortilin-related receptor LR11/SorLA, a protein involved in the control of amyloid beta peptide production\", \"hash\": \"28de48b9df8af9cb3eee5c942b986552dbe6f8b8aee1ee1cd72a859b798b149310d4e035ca6098a4192508c1d643af35e51116b5d1c8bd8e8b30037f5a91bdb8\", \"key\": 2, \"line\": 35183, \"relation\": \"decreases\", \"source\": 47, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"Sci Rep. 2016 Mar 21;6:23362. doi: 10.1038/srep23362.\", \"reference\": \"26996954\", \"type\": \"PubMed\"}, \"evidence\": \"SORL1 regulates and traps APP in the Golgi apparatus and reduces the production of the amyloid-beta peptide (Abeta), the main component of senile plaques in AD4. Cell biology experiments have indicated that SORL1 underexperssion leads to overproduction of Abeta3 , whereas SORL1 overexpression considerably reduces cellular APP and extracellular Abeta5. Although the underlying mechanisms of SORL1 in AD and neurodegenerative processes remain unclear, a recent comprehensive review on multiple functional domains of SORL1 suggested that SORL1 can assume various characters in AD pathogenesis\", \"hash\": \"cd15802dae432d6cf4d84d237d7b70fb410dfb76ae45544026cc54cd42de647f4a91f71fc31d76d847fd902946a449d7a0cd3a5eda5757654c821ea3bb44bab1\", \"key\": 3, \"line\": 54641, \"relation\": \"increases\", \"source\": 47, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2011;6(10):e24588. Epub 2011 Oct 26\", \"reference\": \"22046233\", \"type\": \"PubMed\"}, \"evidence\": \"We previously reported that genetic variants in SORCS1 increase the risk of AD, that over-expression of SorCS1 reduces gamma-secretase activity and ABeta levels, and that SorCS1 suppression increases gamma-secretase processing of APP and ABeta levels. We now explored the effect of variation in SORCS1 on memory.Variation in intron 1 in SORCS1 is associated with memory changes in AD\", \"hash\": \"3bb7f79e535f93f0118c86a21f6cc49c1d4d6ad272f36b77b52c38b330678cf2da3395395e763a8bc535847442f533b633668530c47e7bea122c1782a2ef1372\", \"key\": 0, \"line\": 37273, \"relation\": \"directlyDecreases\", \"source\": 48, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Ann Neurol. 2011 Jan;69(1):47-64. doi: 10.1002/ana.22308\", \"reference\": \"21280075\", \"type\": \"PubMed\"}, \"evidence\": \"Sorting mechanisms that cause the amyloid precursor protein (APP) and the -secretases and gamma-secretases to colocalize in the same compartment play an important role in the regulation of ABeta production in Alzheimer's disease (AD). We and others have reported that genetic variants in the Sortilin-related receptor (SORL1) increased the risk of AD, that SORL1 is involved in trafficking of APP, and that underexpression of SORL1 leads to overproduction of ABeta. Here we explored the role of one of its homologs, the sortilin-related VPS10 domain containing receptor 1 (SORCS1), in AD.SorCS1 influenced APP processing. While overexpression of SorCS1 reduced gamma-secretase activity and ABeta levels, the suppression of SorCS1 increased gamma-secretase processing of APP and the levels of ABeta.\", \"hash\": \"4365d38db5c9b7c8efd4215eda0f103550e159c49eb35177152e65c8954c1ba991690ad71bb26b718ca0395df89669cfebee3efb39ca661879afeb9f73b7afbf\", \"key\": 1, \"line\": 35173, \"relation\": \"directlyDecreases\", \"source\": 48, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"Psychogeriatrics. 2012 Dec;12(4):215-25. doi: 10.1111/j.1479-8301.2012.00405.x.\", \"reference\": \"23279143\", \"type\": \"PubMed\"}, \"evidence\": \"Both in vitro and in vivo, overexpression of the sortilin-related VPS10 domain containing receptor 1 (SORCS1) protein lowers amyloid- generation.\", \"hash\": \"c39170019d95843261f4306cbb2ee068e5d3f5ba4cca21de1337b149fd8532534db9f874b8be9859aeda4930053746b6715515ee942138ed2a77549e94e9c1f1\", \"key\": 2, \"line\": 11911, \"relation\": \"decreases\", \"source\": 48, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Molecular Psychiatry\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by A mediates neurotoxicity, we measured the effects of A and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krppel-like factor-10) that, when individually silenced, protected against A neurotoxicity and/or tau phosphorylation.\", \"hash\": \"cefcb067c0a2307a0e5564c5cc969447e29046c32c9fb95643154fd549e28ad9d3b19c9754b9efd9273e94b20b14c9422ec8f38d28f915b8148fe69511336c90\", \"key\": 0, \"line\": 57204, \"relation\": \"decreases\", \"source\": 49, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Molecular Psychiatry\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by A mediates neurotoxicity, we measured the effects of A and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krppel-like factor-10) that, when individually silenced, protected against A neurotoxicity and/or tau phosphorylation.\", \"hash\": \"2418706b383b83be4933062f361ca835ad8d124daae6d19343d1ddf9355e361b3c2b8d869d984220f38c8fcca92720319cfd158bacc2c711b340b3075daf1aa7\", \"key\": 0, \"line\": 57207, \"relation\": \"decreases\", \"source\": 50, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Molecular Psychiatry\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"To further elucidate how the clusterin-dependent induction of Dkk1 by A mediates neurotoxicity, we measured the effects of A and Dkk1 protein on whole-genome expression in primary neurons, finding a common pathway suggestive of activation of wnt-planar cell polarity (PCP)-c-Jun N-terminal kinase (JNK) signalling leading to the induction of genes including EGR1 (early growth response-1), NAB2 (Ngfi-A-binding protein-2) and KLF10 (Krppel-like factor-10) that, when individually silenced, protected against A neurotoxicity and/or tau phosphorylation.\", \"hash\": \"17ccc13e8dc69692bbd13bcd2316046aa46b4f525114a0ac459a7449affd337b62b87464ffddacda1065ee43b37596155b52af4cdb395e57aa36007438dedcd5\", \"key\": 0, \"line\": 57208, \"relation\": \"decreases\", \"source\": 51, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\", \"type\": \"PubMed\"}, \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomallysosomal pathway because they exhibit - and -secretase like-activity [32]. For this reason, the endosomallysosomal pathway is a site for cleavage of the APP into smaller -amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"hash\": \"5c5130e4331a5ef8ae4fa714d8cb95fcc9487d5fef0d80ec771da53dd20d01df9da4358fa73d396172bf5d3b1bb20d996fcc459bd443196e952e45691ec6825f\", \"key\": 0, \"line\": 6879, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 52, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Complement system subgraph\": true}}, \"citation\": {\"name\": \"Neurology. 2012 Jan 31;78(5):334-41. Epub 2012 Jan 18\", \"reference\": \"22262751\", \"type\": \"PubMed\"}, \"evidence\": \"The CR1 variant rs6656401 influences risk and recurrence of CAA-ICH, as well as the severity of vascular amyloid deposition\", \"hash\": \"807af34d659938fd883bcdf3ca014b8714a8d405dcb2d1080c63e733b5bf31343c59772b34750df6240d68c7b2bf1ebc848b5add6f5436f55b9cebb2aadebd2f\", \"key\": 0, \"line\": 3449, \"relation\": \"increases\", \"source\": 53, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Caspase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron. 2008 Oct 23;60(2):247-57\", \"reference\": \"18957217\", \"type\": \"PubMed\"}, \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"hash\": \"8669dff6a61d28741c851b0e9a43b3a766537cfdaa4f202c46bf39e9cc872ec9695d2546113c1132f71fdba7a5b09cda0f7a5f879f2b05801a0b8d9afcab9bbd\", \"key\": 0, \"line\": 37260, \"relation\": \"decreases\", \"source\": 54, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Endosomal lysosomal subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 1996 Sep-Oct;17(5):717-22.\", \"reference\": \"8892344\", \"type\": \"PubMed\"}, \"evidence\": \"Clusterin inhibits the aggregation of A beta, which might be neuroprotective according to the aggregation-toxicity hypothesis of A beta. However, clusterin enhanced the oxidative stress of A beta.\", \"hash\": \"f82d0d3d5014d91107eb98ff3a1182c65053de01f60459e0e9121385ef17af9e2c3901f88cf891a8202a23cd8b3a9184e7aac0d6b190153a1d272262d5e07851\", \"key\": 0, \"line\": 35274, \"relation\": \"decreases\", \"source\": 55, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Endosomal lysosomal subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Feb 5;285(6):3532-9. Epub 2009 Dec 7\", \"reference\": \"19996109\", \"type\": \"PubMed\"}, \"evidence\": \"We have previously proposed that CLU-client complexes serve as vehicles to dispose of damaged misfolded extracellular proteins in vivo via receptor-mediated endocytosis\", \"hash\": \"e69ddbc7878b7a1f78fe989e75f760a2e6e973980f8902b864db985a5506611778b6ce88533a21151ea31f3bc14c050b74013d0a76a9338373d444f7500f0cc4\", \"key\": 1, \"line\": 35304, \"relation\": \"directlyDecreases\", \"source\": 55, \"subject\": {\"effect\": {\"name\": \"chap\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"DKK1 subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2014 Jan;19(1):88-98. doi: 10.1038/mp.2012.163. Epub 2012 Nov 20.\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"Knockdown of clusterin in primary neurons reduced A toxicity and DKK1 upregulation and, conversely, A increased intracellular clusterin and decreased clusterin protein secretion, resulting in the p53-dependent induction of DKK1. \", \"hash\": \"de47cfe8ff7f4e5fefb2e37d94da07d7f69034d1dda2f6d1e21a1a259249c4f6e45b0a51c12679b98fb8d0843c71edee6666d0376c2257438e6d863d5e123351\", \"key\": 2, \"line\": 56089, \"relation\": \"decreases\", \"source\": 55, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Apr 26. [Epub ahead of print]\", \"reference\": \"22539346\", \"type\": \"PubMed\"}, \"evidence\": \"These data indicate that PICALM, an adaptor protein involved in clathrin-dependent endocytosis, regulates APP internalization and subsequent Abeta generation. PICALM contributes to amyloid plaque load in brain likely via its effect on Abeta metabolism.\", \"hash\": \"dedcc6f86fcd068bc98ae850889572687d280959574664861ecb05c541dd51420252db5c0f1f4ec556bcf70dbec5bc108859cfff3d371c25372e4ac38e154652\", \"key\": 0, \"line\": 1564, \"relation\": \"increases\", \"source\": 56, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ATP binding cassette transport subgraph\": true, \"Cholesterol metabolism subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2008 Jul;106(2):793-804. Epub 2008 Apr 19\", \"reference\": \"18429932\", \"type\": \"PubMed\"}, \"evidence\": \"These studies indicate that ABCA7 has the capacity to stimulate cellular cholesterol efflux to apoE discs and regulate APP processing resulting in an inhibition of Abeta production\", \"hash\": \"1675e1bcf850df211ca35618d78abaa0b30abd64b70fa43b148ea13eff023eb6bbc1cd091c2e0230187e68e4e5e88d11b50932acae15d7ca1b6106435ba74e06\", \"key\": 0, \"line\": 37209, \"relation\": \"decreases\", \"source\": 57, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Akt substrates such as mammalian target of rapamycin (mTOR; Ser2448) and decreased levels of cell-cycle inhibitors (p27kip1) are found in AD temporal cortex when compared to controls . GSK-3a has been implicated in the production of A peptide while increased GSK-3 activity has been implicated in tau hyperphosphorylation and neuronal cell death\", \"hash\": \"fcb8cdeec6dd98d84943c514a308fb4591989886b21fdf6fc1f018a9106a16491e3e8ba8fc59fa2ac5da1052460a025d596074112a2c7f8d78b9acf67de19a5d\", \"key\": 0, \"line\": 6478, \"relation\": \"increases\", \"source\": 58, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\", \"type\": \"PubMed\"}, \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"hash\": \"413b2cf28abf95a0abb5852424bce2c0169c7c87be941a298cbe3a01aedeb5deefecd243b3e0243f031cd4a1546f1cb766d17f215e5ff04ed1c79f0f7632152f\", \"key\": 1, \"line\": 31969, \"relation\": \"increases\", \"source\": 58, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell Neurosci. 2010 Nov;45(3):306-15. Epub 2010 Jul 14.\", \"reference\": \"20637285\", \"type\": \"PubMed\"}, \"evidence\": \"In the brain, megalin is expressed in brain capillaries, ependymal cells and choroid plexus, where it participates in the clearance of brain amyloid beta-peptide (Abeta) complex.Additionally, given that FE65 mediates the interaction between the low density lipoprotein receptor-related protein-1 and the amyloid precursor protein (APP) to modulate the rate of APP internalization from the cell surface, we hypothesize that megalin could also interact with APP in neurons.\", \"hash\": \"ee55acdb3fa85e12e90d05258861efcfa0a54f8ce98c329005ec1fbb33a83ddf8a7112f7dff5b6f7a80acbb7732a944ec77fea155563a746e1fcd5a95b04052e\", \"key\": 0, \"line\": 37401, \"relation\": \"directlyDecreases\", \"source\": 59, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Mol Biol. 1997 Sep 26;272(3):348-61.\", \"reference\": \"9325095\", \"type\": \"PubMed\"}, \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"hash\": \"fa3964580a109110dbd461d560430f46c18a0fd34f8febe5c3ac4ea436006d80c1ace3107c8fafd17a0cec2a055f1c59503335db7f776b794d54337bacec1fff\", \"key\": 0, \"line\": 23954, \"relation\": \"increases\", \"source\": 60, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"FEBS J2008\", \"reference\": \"18205831\", \"type\": \"PubMed\"}, \"evidence\": \" The fact that acetylcholinesterase accelerates amyloid formation and the effect is sensitive to peripherical anionic site blockers of the enzyme, suggests that specific and new EC 3.1.1.7 (acetylcholinesterase) inhibitors may well provide an attractive possibility for treating Alzheimer's disease. \", \"hash\": \"b1840536ee367ef1a9eb1e961847e488b0494aab3be5448a382ac1339276eec25a903a145712db49246fae8c43f7e9855e5c0508313c69d968c4be87eacbad02\", \"key\": 1, \"line\": 24385, \"relation\": \"increases\", \"source\": 60, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Mol Biol1997\", \"reference\": \"9325095\", \"type\": \"PubMed\"}, \"evidence\": \"Acetylcholinesterase (AChE), an enzyme involved in the hydrolysis of the neurotransmitter acetylcholine, consistently colocalizes with the amyloid deposits characteristic of Alzheimer's disease and may contribute to the generation of amyloid proteins and/or physically affect fibril assembly.\", \"hash\": \"f4cf117697fd1348ce323d4b5ff03814c6a320dc9d832ba05c56f6a1116024382486350ea97e6ce89a7bdb1dae5c4f093b390a1618d7e91f8990e0872c4cf51d\", \"key\": 2, \"line\": 24411, \"relation\": \"increases\", \"source\": 60, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"Eur J Pharmacol. 2004 Apr 19;490(1-3):115-25\", \"reference\": \"15094078\", \"type\": \"PubMed\"}, \"evidence\": \"As far as the metabolism of amyloid precursor protein (APP) in late-onset sporadic Alzheimer disease is concerned, neuronal insulin receptor dysfunction may result in the intracellular accumulation of Abeta and in subsequent cellular damage.\", \"hash\": \"b8c7c446e644a9149fb7aa66f8b11710a90ac3a1c6841658c84be8d902f68c353dca27ad5d064b6ba6dda961baf60172af96d613a6919a1d2c1862ce60bc5c93\", \"key\": 0, \"line\": 20306, \"relation\": \"increases\", \"source\": 61, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Subgraph\": {\"Nitric oxide subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\", \"type\": \"PubMed\"}, \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"hash\": \"844f018120412e8f1861150ac576bd75a85392e46c06cf3e3293814b2bffac5d33612eb3b9f799e0f39ed317b894d1e9d6f7de37327b5f56e211937d8beb33e0\", \"key\": 0, \"line\": 17310, \"relation\": \"decreases\", \"source\": 62, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Subgraph\": {\"Nitric oxide subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\", \"type\": \"PubMed\"}, \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"hash\": \"36de6541a4bf189a25d8104d3306d73e4895bb55638f580a64146574062e3ef3ca48616f686235ac94b2de94e694044ad8f34aec68f0305d7204faef7ef10379\", \"key\": 1, \"line\": 17343, \"relation\": \"decreases\", \"source\": 62, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\", \"type\": \"PubMed\"}, \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"hash\": \"3985c0befb697515f6a09ac582af9dec4f0571fd5101125d69175e48bb06c9198376a43f7129e3174a1335b29405388f7381f3f56ca35d45a8a7f804b895ff66\", \"key\": 0, \"line\": 1123, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Beta secretase subgraph\": true, \"Response to oxidative stress\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\", \"type\": \"PubMed\"}, \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in Abeta-secretase and, thus, an increase of Abeta generation in neuronal cells expressing mutant PS2\", \"hash\": \"d7a0996a6871a2c46cfd3f485f3119e66a161933fa0f4ae1f4714ad2f6bad47ad7941d618729ea6f4c0f7238fa6d4d533c3b74042222c907100cb6ea30fbb3bf\", \"key\": 1, \"line\": 1130, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Cell\": {\"microglial cell\": true}, \"Condition\": {\"Normal Healthy State\": true}, \"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\", \"type\": \"PubMed\"}, \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"hash\": \"b683f11c3f31151eca5373cb12b0ece7f848b509ee484855d3dba40456e4c26fe574ab8c3bc9ca44259baa1c7d5748ba303319a14207f48f4a2dac2255e7df60\", \"key\": 2, \"line\": 3063, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2008 Oct 31;283(44):29615-9. Epub 2008 Jul 23.\", \"reference\": \"18650430\", \"type\": \"PubMed\"}, \"evidence\": \"Interestingly, Fe65 acts as a functional linker between APP and LRP (another type I membrane protein containing two NPXY endocytosis motifs) in modulating endocytic APP trafficking and Abeta production. A conformational change introduced by phosphorylation at Thr668 (14 amino acids proximal to the YENPTY motif) interferes with Fe65 binding to APP and facilitates BACE1 and gamma-secretase cleavage of APP.\", \"hash\": \"64f560834168c61da137b1b2583a7e180907aec665474404d5be1b9ef49f656555498291385a6365702a58e6b13c3482d54aad4b8bcfb594440f1ace791d693a\", \"key\": 3, \"line\": 3255, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\", \"type\": \"PubMed\"}, \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"hash\": \"4c16ca4bd22886b664bbef6b0736e59578d354f9e4f4b2ca9c2be129879b34ee2a494010e438a8217f71b30c75b71671678c8f604a3ef587704bbaf5e3f5efcf\", \"key\": 4, \"line\": 4820, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Cell Surface Extensions\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\", \"type\": \"PubMed\"}, \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"hash\": \"4aef1216128b267e12345ee8137d3404e4a48b72e620ab29989b0445d9a2782b24d9cae8adecfee40bc26a0b2db295fe350d32c4056019448bbaaee931446be2\", \"key\": 5, \"line\": 5089, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Nerve growth factor subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\", \"type\": \"PubMed\"}, \"evidence\": \"CerebraiiR/IGF-1 signaling in APP metabolism. Binding of insulin and IGF-1 to their receptors leads to autophosphorylation of the -subunit of the IR/IGF-IR and recruitment of IRS-1 /-2. IRS proteins activate mainly two pathways, the PI3 kinase pathway, and the RAS-RAF-MAP kinase cascade. Tau phosphorylation via IR/IGF-1 signaling is influenced by GSK-3and the tau phosphatase PP2A. Inter estingly, mTOR signaling couples the activity ofPP2A and GSK-3in a way that the activities of both enzymes change always in the same direction. suggesting that only a dysregulation of either PP2A or GSK-3induces tau hyperphosphorylation. The PI3K pathway as well as the RAS-RAF-MAPK cascade regulate different transcription factors involved in transcriptional regulation of metabolism and clearance of  amyloid. APP is cleaved by a.-, -secretase (BACE-1) and y-secretase (presenilin). - and subsequent y-cleavage of APP leads to generation of -amyloid 1 -4o/ t-42 IGF-1 signaling promotes a switch lrom TrkA to p75NTR expression leading to increased -secretase activity due to an upregulation of BACE-1 expression. Recent data suggest that not only BACE-1 is int1uenced by IRIIGF- IR mediated signals but also a. secretase activity is stimulated by the PI3K pathway.\", \"hash\": \"af007329e4b6392ec4d51df4d8884164a1eeb870826d88d8799a87af1a8cdfcda00ddd8a3a33529f02d4519ab120d575937a6400137f2450be1d87e6514343c7\", \"key\": 6, \"line\": 8396, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Apoptosis signaling subgraph\": true, \"Beta secretase subgraph\": true, \"Response to oxidative stress\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\", \"type\": \"PubMed\"}, \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in -secretase and, thus, an increase of A generation in neuronal cells expressing mutant PS2\", \"hash\": \"cdfeb45feb409b5f42a0c024c5cc6d92fe52d682a3f85c465bee955cf8b3a65499b1d81453724505d32bb9467a89e5b4cd03043dcf71436699a97a40546a9cd0\", \"key\": 7, \"line\": 10345, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"MAPK-ERK subgraph\": true, \"Response to oxidative stress\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol. 2012 Feb;71(2):130-9\", \"reference\": \"22249458\", \"type\": \"PubMed\"}, \"evidence\": \"oxidative stress-mediated ERK activation contributes to increases in -secretase and, thus, an increase of A generation in neuronal cells expressing mutant PS2\", \"hash\": \"9f4b7bf4c08e6de8ea8465de31a1653effbb421f1fbc4e0c54516b9ae24f62f1f3ac600f1c669710fb178d5e6d2d20136c5ee110d3c921cfd6145539bd675a63\", \"key\": 8, \"line\": 10349, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Beta secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Pharm Pharmacol. 2011 Aug;63(8):1111-8\", \"reference\": \"21718295\", \"type\": \"PubMed\"}, \"evidence\": \"In this study, we aimed to investigate the possibility of P-gp as a potential therapeutic target for Alzheimer's disease by examining the impact of P-gp up-regulation on the clearance of Abeta, a neuropathological hallmark of Alzheimer's disease.Uptake studies for-radiolabelled Abeta Approximately 10-35% decrease in Abeta intracellular accumulation was observed in cells treated with rifampicin, dexamethasone, caffeine, verapamil, hyperforin, -estradiol and pentylenetetrazole compared with control.\", \"hash\": \"dc044cb024de57d51051bf1078f352fe84cdb3f3ef47d13be6766f1a092371149b8bac461cbddb25a3dad4ae79a7e527ff73d65c0f7cf2922fd63c7b76e02adf\", \"key\": 9, \"line\": 17910, \"relation\": \"decreases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Immunoglobulin subgraph\": true, \"T cells signaling\": true}}, \"citation\": {\"name\": \"FASEB J. 2009 Aug;23(8):2639-49. doi: 10.1096/fj.08-126383. Epub 2009 Mar 30.\", \"reference\": \"19332646\", \"type\": \"PubMed\"}, \"evidence\": \"RAGE stimulates functional BACE1 expression through NFAT1 activation, resulting in more Abeta production and deposition in the brain.\", \"hash\": \"2015b5bd541db44bfe0d8a623f112595cccc79262c29327e5c362272dcf38f01972317eb8df1de211e59f19662015d4ac5f9497807087bcdd956cd07874f7f62\", \"key\": 10, \"line\": 24191, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Beta secretase subgraph\": true, \"Neuroprotection subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"PLoS One2011\", \"reference\": \"21246057\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, NPD1 suppresses Abeta42 peptide shedding by down-regulating beta-secretase-1 (BACE1) while activating the -secretase ADAM10 and up-regulating sAPPalpha, thus shifting the cleavage of betaAPP holoenzyme from an amyloidogenic into the non-amyloidogenic pathway.\", \"hash\": \"fcd31e4443bb0a31fdf28840dc27f2dbfe618468ac0ae258d5632df290b75a952a89e77ecc520ab7dbc70a5f1237b7704e89cdabb30920f9abc2ff3a5258c226\", \"key\": 11, \"line\": 24473, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Beta secretase subgraph\": true, \"Neuroprotection subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\", \"type\": \"PubMed\"}, \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the alphaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway.\", \"hash\": \"78d522b4ba0d91191a2d9104f01fc4270174c46a90104edd7f886ad9d0ce398b0db5e195578753a85f3c834217d74a262e2dc23e7e5c1a95e86ee54b3d712f97\", \"key\": 12, \"line\": 24515, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A1999\", \"reference\": \"10500121\", \"type\": \"PubMed\"}, \"evidence\": \"A pathogenic mutation at codons 670/671 in APP (APP Swedish) leads to enhanced cleavage at the beta-secretase scissile bond and increased Abeta formation.\", \"hash\": \"3685d71a0a4085f589a513417ef14e664415fc1075934a1146efe00d18e06947051ca81df8ad8b20ecd3b9e57c183b6dfc2ae2e1ec99e2b3de0ec0083d6bd8f1\", \"key\": 13, \"line\": 25577, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2000\", \"reference\": \"10956649\", \"type\": \"PubMed\"}, \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation.\", \"hash\": \"03456dc2c577ebff103e604a4f8b2a9ea4e11cd80887c81fc46ccf01af09bc6b59190ebdf72be1536c5f3e94a0ce91acf2241b82fb35bdac30f0515f083859ed\", \"key\": 14, \"line\": 25608, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Histol Histopathol2002\", \"reference\": \"11813874\", \"type\": \"PubMed\"}, \"evidence\": \"A beta is generated by the sequential intracellular cleavage of APP by beta-secretase to generate the N-terminal end of A beta, and intramembranous cleavage by gamma-secretase to generate the C-terminal end.\", \"hash\": \"7135229ad541b6921f0d7080f2c1961bf7a15fc570f0eec6143729c090bd6094ff5a534e99ad831730aff93db46e89b3b62ef0064732f6f0ab49fd67b76c8bdc\", \"key\": 15, \"line\": 25651, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Acta Biochim Pol2004\", \"reference\": \"15218540\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Secretase, a beta-site amyloid precursor protein (APP) cleaving enzyme (BACE), participates in the secretion of beta-amyloid peptides (Abeta), the major components of the toxic amyloid plaques found in the brains of patients with Alzheimer's disease (AD).\", \"hash\": \"424f7dda80dbb778a5c19d1480218c27f2e40a23a4166e67165ee1c979e5a49a143c4e757bed52eb0984bdc014c99e167c43243a80e2245f1f5288df774815e2\", \"key\": 16, \"line\": 25682, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2004\", \"reference\": \"15452128\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid-beta (Abeta) the primary component of the senile plaques found in Alzheimer's disease (AD) is generated by the rate-limiting cleavage of amyloid precursor protein (APP) by beta-secretase followed by gamma-secretase cleavage.\", \"hash\": \"ecbf36cedf993fdcc336f619e665887ee0d0f1189ec7afd192d883c268cce943fa67d040fce1c42f6ee905649d8778386294238ba14a35e384814cc6160fcb0a\", \"key\": 17, \"line\": 25709, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci2004\", \"reference\": \"15465276\", \"type\": \"PubMed\"}, \"evidence\": \"The generation of beta-amyloid peptides requires the enzymatic activity of the beta-site APP-cleaving enzyme 1 (BACE1).\", \"hash\": \"f31364dbde1ed5029c1596f19b325d927cab63a4a5a0635afd78425b0f8e4d5b4d8602007877fd1282904bbf5a3e4854e303d359ada9eb12306e11641b4d53b9\", \"key\": 18, \"line\": 25722, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2005\", \"reference\": \"15663471\", \"type\": \"PubMed\"}, \"evidence\": \"The generation of beta-amyloid peptides by proteolytical processing of the amyloid precursor protein (APP) requires the enzymatic activity of the beta-site APP cleaving enzyme 1 (BACE1).\", \"hash\": \"b06ff8675ae1abde7f997eade9daa3c0a5dbb12dc35934982b4a45738ee31c45967d9c1c09d3986d311e71ac46c15b45cd9e3b52c2142630e840b818d6f1966e\", \"key\": 19, \"line\": 25737, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"J Biol Chem2005\", \"reference\": \"15737955\", \"type\": \"PubMed\"}, \"evidence\": \"Using an assay that incorporates full-length recombinant APP as a substrate for beta-secretase (BACE), we have identified a series of compounds that inhibit APP processing, but do not affect the cleavage of peptide substrates by BACE1.\", \"hash\": \"5d206cd809c4b694ea6b71648643e538f3873ff6c2292466e5275ea63e25ca404716dbfdb117a4776c83a57071740eebe7374bea0c03ee707635402e259f1403\", \"key\": 20, \"line\": 25769, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2005\", \"reference\": \"15824102\", \"type\": \"PubMed\"}, \"evidence\": \"Sequential processing of amyloid precursor protein (APP) by membrane-bound proteases, BACE1 and gamma-secretase, plays a crucial role in the pathogenesis of Alzheimer disease.\", \"hash\": \"ca934db0094479e753885b66e6b038cdf37247c240e35e435d9aae3f802887191e1655f77c172b48e12e5bb4a2589e085378b2b25c169fd52da0195316044242\", \"key\": 21, \"line\": 25779, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"Proc Natl Acad Sci U S A2005\", \"reference\": \"15890777\", \"type\": \"PubMed\"}, \"evidence\": \"gamma-Secretase is a membrane protein complex that cleaves the beta-amyloid precursor protein (APP) within the transmembrane region, after prior processing by beta-secretase, producing amyloid beta peptides Abeta(40) and Abeta(42).\", \"hash\": \"75931cd508e01de15e76a93f1118c15447dbb6a08394f09249ccb816d22d353a75c18fdba5acff5e33eae3c8be65daaaa03c50d232e336371d2b83b4a2a80786\", \"key\": 22, \"line\": 25787, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2005\", \"reference\": \"16027115\", \"type\": \"PubMed\"}, \"evidence\": \"Release of Abeta from the amyloid precursor protein (APP) requires proteolysis by the beta-site APP-cleaving enzyme (BACE1).\", \"hash\": \"9e44a5a1d9ae45bd4d5a278e000ccbdc49f1a39fd52af318218dd2217fa5be7437b3491715a85129094e3f39bd013bfd447840dc25eed47aba9e9ca53f2ef259\", \"key\": 23, \"line\": 25796, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Res2006\", \"reference\": \"16290302\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta is produced from beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase. beta-Secretase has been identified as beta-site APP cleaving enzyme1 (BACE1).\", \"hash\": \"7936486b07a9f15dda829f3e538aa32f9722c46370e49ba25a442d3ece6ef36ab94fb9af2cbb568cc2e44e59ef2738eb6a82922d46d29be6118fa5d52f8871ee\", \"key\": 24, \"line\": 25831, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"RNA2006\", \"reference\": \"16888322\", \"type\": \"PubMed\"}, \"evidence\": \"The beta-amyloid peptide (Abeta) is a major component of the Alzheimer's disease (AD)-associated senile plaques and is generated by sequential cleavage of the beta-amyloid precursor protein (APP) by beta-secretase and gamma-secretase.\", \"hash\": \"9dc29b5d2455102ba6f2b69442abe2c94ab7022ccedebf8db2b5e3ebeec6f57570b41b9df09b97da951bf79d59586dbb2345e0cb05d9f7f41d20d29130aabcc6\", \"key\": 25, \"line\": 25847, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Microbiol2006\", \"reference\": \"17205046\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid beta-peptide (Abeta), which is a product of the proteolytic effect of beta-secretase (BACE) on an amyloid precursor protein, is closely associated with Alzheimer's disease (AD) pathogenesis.\", \"hash\": \"a5250268820b3c0de1358be04141c8e5f92671dce891c59b4a0915778b6e80a8503af0f94272cd9fe3c3efa925386ee6d367869ab1d12ebae4b9c1fc21d28542\", \"key\": 26, \"line\": 25864, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol2007\", \"reference\": \"17255335\", \"type\": \"PubMed\"}, \"evidence\": \"TNF-alpha directly stimulated beta-site APP-cleaving enzyme (BACE1) expression and enhanced beta-processing of APP in astrocytes.\", \"hash\": \"bfb2d71d4cdb12e8966f5238dcf7035b7deb29e2f2efe62798b67839f4ea72af5a70a26f9bc571c1130108cfd50fb19df2273f4b0889e963e1e1ab2fba536cc5\", \"key\": 27, \"line\": 25874, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Peptides2007\", \"reference\": \"17293005\", \"type\": \"PubMed\"}, \"evidence\": \"The peptides not only inhibited the formation of APPbeta (a soluble N-terminal fragment of APP cleaved by beta-secretase), but also significantly reduced Abeta40 production.\", \"hash\": \"9820107d49841d3d509b907205bc414774ac4293d6100a63075827983ac54bb86377d36912f7642d5b131c8bbefcc75f3058c1969098cac913a9bf003a0adaeb\", \"key\": 28, \"line\": 25890, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17360493\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's disease (AD) is associated with accumulation of the neurotoxic peptide amyloid-beta (Abeta), which is produced by sequential cleavage of amyloid precursor protein (APP) by the aspartyl protease beta-secretase and the presenilin-dependent protease gamma-secretase.\", \"hash\": \"9a407b23260255ae4682498d214faac560c9bc1b17ef59506f7025e85ff85c77fb651729fef38804b02f204b86b107498fa164b61fa1a8d9828bac2ad75828da\", \"key\": 29, \"line\": 25899, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurodegener Dis2007\", \"reference\": \"17536186\", \"type\": \"PubMed\"}, \"evidence\": \"Such an approach limits APP processing by beta-secretase, mainly through the endocytic pathway, and overcomes some of the limitations of BACE inhibition.\", \"hash\": \"cd807c30c1adb1df433a6069a0db7d7731d306a2b68be945efa3010b7b9c30953c2b45bd2c7a22bfbb966e0f651b444d1acc0e9b7f568d3b0df9ffe2d955b678\", \"key\": 30, \"line\": 25913, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\", \"type\": \"PubMed\"}, \"evidence\": \"Proteolytic processing of the amyloid precursor protein (APP) by beta-secretase, beta-site APP cleaving enzyme (BACE1), is the initial step in the production of the amyloid beta (Abeta) peptide, which is involved in the pathogenesis of Alzheimer's disease.\", \"hash\": \"59292f16668a57bbcc6af17d7f50615df2c2d3b021d11a64fd771de8119f75eb98697f1c45fe4776c79d297783726d97e064f9a9942c8d1e49d7b79f8024e3cc\", \"key\": 31, \"line\": 25922, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Brain Res2007\", \"reference\": \"17586478\", \"type\": \"PubMed\"}, \"evidence\": \"To address this issue, temporal neocortex of 27 AD and 21 non-demented control brains was examined to assess mRNA levels of APP isoforms (total APP, APP containing the Kunitz protease inhibitor domain [APP-KPI] and APP770) and APP metabolic enzymatic partners (the APP cleaving enzymes beta-secretase [BACE] and presenilin-1 [PS-1], and putative clearance molecules, low-density lipoprotein receptor protein [LRP] and apolipoprotein E [apoE]).\", \"hash\": \"006682b114961428886293af178215e43ea17af9d19ebf3ae16e4b81e75c04a5e408d95571030ac51bb81d424c8fb123919ce57429a27c73baf5d8169fbb2fc0\", \"key\": 32, \"line\": 25941, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Nat Clin Pract Neurol2007\", \"reference\": \"17611486\", \"type\": \"PubMed\"}, \"evidence\": \"Pathologically, AD is characterized by the deposition in the brain of amyloid-beta peptides derived from proteolysis of amyloid precursor protein (APP) by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"hash\": \"2ecb2574f759acaf21e600114b1d22cd2bf647504fc8cedcfc5ac0e4c58fb794d81f43ff503c4083b6ec3f5ada387382e60109281c798bf35bf6e04427778f07\", \"key\": 33, \"line\": 25950, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res2007\", \"reference\": \"17908048\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta is generated by proteolytic processing of amyloid precursor protein (APP) via beta and gamma-secretases.\", \"hash\": \"2975cab20e167990770a9156298f0fa29e2c9af15a55b366d6564ab4d0489031a576aaaa8b07e2755e7ffbb9d13bb67a9da7b1ff1fc1f372448c30b7259619b8\", \"key\": 34, \"line\": 25959, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2008\", \"reference\": \"17995932\", \"type\": \"PubMed\"}, \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"hash\": \"b49fc1a279189502e96eec28cf27fe12bceaa78f6946d8243146aec13ea5e52113df096ae5baf29a5cd8206ca723aafa81d5938029eaf2cf398dc3ef29be5de9\", \"key\": 35, \"line\": 25968, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18162398\", \"type\": \"PubMed\"}, \"evidence\": \"Proteolytic cleavage of amyloid precursor protein by beta-secretase (BACE-1) and gamma-secretase leads to formation of beta-amyloid (A beta) a key component of amyloid plaques, which are considered the hallmark of Alzheimer's disease.\", \"hash\": \"2255e03d14156c38781204b88c9e99858bbc73dc768b9bac4af3690e644df41db874bed67c94b3eb43e3e0943d1c9ba1359420c85508e9131369679a5af64e1b\", \"key\": 36, \"line\": 25995, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2008\", \"reference\": \"18308724\", \"type\": \"PubMed\"}, \"evidence\": \"Here we describe that the two enzymatic activities responsible for Abeta production, beta-secretase and gamma-secretase, are inhibited in parallel by cholesterol reduction.\", \"hash\": \"278e8c2811a3a08f45a1bac32df7c0d2b432615562698b347b4e1a8d6a605dda7ba42977bb0e6b4fc8b64d8ea4a9b51d91ae7f08c914a234a440866b5e7bcc4e\", \"key\": 37, \"line\": 26004, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2008\", \"reference\": \"18410513\", \"type\": \"PubMed\"}, \"evidence\": \"There was also an induced expression of the key APP processing enzyme i.e. beta-site APP cleaving enzyme 1 in both high fat/cholesterol-fed C57BL/6 and LDLR-/- mice accompanied by an increased generation of C-terminal fragments of APP.\", \"hash\": \"655f94f857c60ff69dd932765b80ce90625d4d9e2d1432095c083a4bc91e36c410fef5b8fca2eda29a4321a9b3a5829589c3cdc70ff41c802b32957fb1ab173e\", \"key\": 38, \"line\": 26019, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry2008\", \"reference\": \"18570439\", \"type\": \"PubMed\"}, \"evidence\": \"The study provides new cell-based assays for the profiling of small molecule inhibitors of QC and points to conspicuous differences in processing of APP depending on sequence at the beta-secretase cleavage site.\", \"hash\": \"6307ee10eea16ac916d2059f3487a8581b4256f965fb212cd54e65c4828f5e3e1c8717c8debd5f9057b1102f4dcf73281c0ae9246657cac1532f2286044ba323\", \"key\": 39, \"line\": 26031, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"Scand J Clin Lab Invest2009\", \"reference\": \"18609117\", \"type\": \"PubMed\"}, \"evidence\": \"The Abeta peptide results from cleavage of APP initially by BACE1 to produce the C99 fragment and releases soluble APPbeta (sAPPbeta); C99 is then further cleaved by gamma-secretase leading to the Abeta peptide.\", \"hash\": \"efbf96dfc01ab172ab51fa77ac81c3bba96e49cf7a548ed6729ece5de8130fdbcda99ae8b52088a20fbfa15f28fe32e4aa4d35a2838aaea4b438c2b5b9aa161f\", \"key\": 40, \"line\": 26042, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun2008\", \"reference\": \"18675254\", \"type\": \"PubMed\"}, \"evidence\": \"Here, we show that lysines 587 and 595 of APP, which are immediately adjacent to the site of beta-secretase cleavage, are covalently pmodified by SUMO proteins in vivo.\", \"hash\": \"cd4850f2f4b54ff206291593be3528e37723138ff985690f461b8a23836d31586122b2f713d22f021ce4057435d21e5eaec82cfeb17243c7a48c2ba0358af36a\", \"key\": 41, \"line\": 26051, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Hypoxia response subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2009\", \"reference\": \"19196431\", \"type\": \"PubMed\"}, \"evidence\": \"Specifically, hypoxia significantly increases beta-site APP cleaving enzyme (BACE1) gene transcription through the over-expression of hypoxia inducible factor 1alpha, resulting in increased BACE1 secretase activity and amyloid-beta production.\", \"hash\": \"94fcfad6a3ecdf488a11da47793e94bebd9c536a7ef88d596ebb2f8cf82e8af3d085123c6ec12b47e4217b3d707512c1cc51d9f83fb4bbe2121b4bdb956cc4aa\", \"key\": 42, \"line\": 26065, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Scand J Clin Lab Invest2009\", \"reference\": \"19199126\", \"type\": \"PubMed\"}, \"evidence\": \"It is produced from amyloid precursor protein (APP) by proteolytic processing dependent on the beta-site APP-cleaving enzyme 1 (BACE1) and gamma-secretase, and is degraded by a broad range of proteases.\", \"hash\": \"e5999a5cd62c6f517ea77e675430ae5a3ef0fe9b16e21d2d1164d223280a63549cd6ee8aa751fe23731f1b4ae74ffb4480b935012adf89239d0f174b9a842867\", \"key\": 43, \"line\": 26075, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Divers2010\", \"reference\": \"19330459\", \"type\": \"PubMed\"}, \"evidence\": \"Three-dimensional quantitative structure-activity relationship (3D-QSAR) pmodels were developed based on comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA), on a series of 43 hydroxyethylamine derivatives, acting as potent inhibitors of beta-site amyloid precursor protein (APP) cleavage enzyme (BACE-1).\", \"hash\": \"b0ff6cc26ed7f986c2121a97bb711208700c60b390456f4ef1af93b7f58ac29497a61152cee42a5857225d2f04b19bce08b73fd4dd51cdb1e7eda00d149f90a2\", \"key\": 44, \"line\": 26108, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci Res2009\", \"reference\": \"19405102\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is a membrane-bound protease that is essential for the production of beta-amyloid protein (Abeta).\", \"hash\": \"145d6c80c21a014752cd269f7ea35eb342defefb558a3bdda788092307cd715e26c6f3b3d06ed52502b1ff317979c0f6539ba64c5907e6dba586030410613f2f\", \"key\": 45, \"line\": 26129, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Hum Mutat2009\", \"reference\": \"19462468\", \"type\": \"PubMed\"}, \"evidence\": \"Expression levels of the amyloid precursor protein (APP) and beta-site amyloid (Abeta) cleaving enzyme 1 (BACE1) have been implicated in Alzheimer disease (AD) progression.\", \"hash\": \"3e1285df244088b3d9ba4f16ae904b99ac3865e57d224379e864c7c1884d0127372c293fd7752cbb580e78ed8ccee9a388e194751a6874e3329719d624ed8bd3\", \"key\": 46, \"line\": 26145, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging2011\", \"reference\": \"19464074\", \"type\": \"PubMed\"}, \"evidence\": \" The effect of nicotine (2 mg/(kg day)) on Abeta-induced spatial learning and memory impairments was assessed by evaluation of performance in the radial arm water maze (RAWM), in vivo electrophysiological recordings of early-phase long-term potentiation (E-LTP) in urethane-anesthetized rats, and immunoblot analysis to determine changes in the levels of beta-site amyloid precursor protein (APP)-cleaving enzyme (BACE), Abeta and memory-related protei\", \"hash\": \"0001cf7daec598685163bc424f6d58f57bab80359b67cfe1cc20195a863ce50340209134f3685bde4efe5913e6c1f50fe8c2663101eb44eb7e2afc8cfab9d381\", \"key\": 47, \"line\": 26157, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neuromolecular Med2009\", \"reference\": \"19669607\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme (BACE1) is the rate-limiting enzyme for production of beta-amyloid peptides (Abeta), which are proposed to drive the pathological changes found in Alzheimer's disease (AD).\", \"hash\": \"4574a2d40274f3b0b725b68c2a30276e1f845b148068e11f65073b5637a34619476d4aa733e24bda79f9ee3be803afbb37bd49772a797769b9e836099c56c496\", \"key\": 48, \"line\": 26166, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Exp Gerontol2010\", \"reference\": \"19698775\", \"type\": \"PubMed\"}, \"evidence\": \"Core CSF biomarkers include secreted Abeta and amyloid precursor protein (APP) isoforms, Abeta oligomers and beta-site APP-cleaving enzyme 1 (BACE1).\", \"hash\": \"d1f4946af27c1a1646c55c2acece596776a2d8ae4d09f6927b07ae5002d2cf58afd21fddec87d3641d097207c19c89bc00ff288107a3999dd468939c1adc1eb4\", \"key\": 49, \"line\": 26175, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"FASEB J2010\", \"reference\": \"19729516\", \"type\": \"PubMed\"}, \"evidence\": \" In the amyloidogenic pathway, a small proportion of APP is cleaved by beta- and gamma-secretases, known as beta-site APP-cleaving enzyme 1 (BACE1) and presenilin, respectively, leading to the secretion of Abeta peptides.\", \"hash\": \"a24bea60869f5a22a5ab5b9f88bb11b36cf69715772a2bac90e70381f0443e6ccaed9463fcc5852eca6ab45d98118614d6aa2de8a2ab541c526cff926a29fd7a\", \"key\": 50, \"line\": 26188, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci Res2010\", \"reference\": \"19885829\", \"type\": \"PubMed\"}, \"evidence\": \"A beta is generated upon the sequential proteolytic cleavage of transmembrane amyloid precursor protein (APP) by two membrane-bound proteases, beta-secretase (BACE1) and the gamma-secretase complex comprising presenilin 1 (PS1), nicastrin, APH-1 and PEN-2.\", \"hash\": \"87d1ead943f86a074d51e264f19468670dfef0086308366f89e4a727b2c844bc9741a2db161f49bfd1224a52693247a1ecf27e040106bdeb41ea80a046212042\", \"key\": 51, \"line\": 26211, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Prion2009\", \"reference\": \"19887909\", \"type\": \"PubMed\"}, \"evidence\": \"In Alzheimer disease, this feedback loop is disrupted, and the increased level of Abeta oligomers bind to PrP(C) and prevent it from regulating BACE1 activity. PrP(C) interacts with and inhibits the beta-secretase BACE1, the rate-limiting enzyme in the production of Abeta.\", \"hash\": \"007d4d537ed1434f5df0dc2f5151adbc6c4140680754d832c7490aa4cb0347c65e2fabc58719c4b41b08ec73f68f0b12ed4f1c2a425de42782d27b528e0fb3e1\", \"key\": 52, \"line\": 26220, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2010\", \"reference\": \"20067575\", \"type\": \"PubMed\"}, \"evidence\": \"The beta-site APP cleaving enzyme (BACE1) is responsible for the first step in the production of the beta-amyloid protein of Alzheimer's disease.\", \"hash\": \"1c155d4c9640dade4ea1547b07ace24d0e6b975fb1fe93f0d980f58ba5a170ddf1fbb5df50dbcf32f9078e2ccc1707a5656ae2c6f270423611613aebc1e80e24\", \"key\": 53, \"line\": 26233, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2010\", \"reference\": \"20089133\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation that is central to the pathophysiology of Alzheimer's disease (AD). Therefore, lowering Abeta levels by BACE1 manipulations represents a key therapeutic strategy, but it remains unclear whether partial inhibition of BACE1, as expected for AD treatments, can improve memory deficits.\", \"hash\": \"7cdcc1b8956fb966b83758d5a31a89551d147d0235cd7dbe62a4529a2755824cf1b6d4a775304358c27fabfbeed19aec3a86c2fcf07198edc67dc2b9abfa9ac3\", \"key\": 54, \"line\": 26245, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun2010\", \"reference\": \"20171164\", \"type\": \"PubMed\"}, \"evidence\": \"We previously indicated that amyloid beta (Abeta) augments protein levels of beta-site amyloid precursor protein cleaving enzyme-1 (BACE-1) through oxidative stress.\", \"hash\": \"36975553d211e31dcc68b8c7758474fd3209f4994362d8c6def4427e84ae47da1b23f32590c698033b1dc65470e3e93f059a2eaa8d1cd8e12deb29565a6dff47\", \"key\": 55, \"line\": 26323, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"Curr Alzheimer Res2010\", \"reference\": \"20205669\", \"type\": \"PubMed\"}, \"evidence\": \"Familial Alzheimer's disease mutations in presenilin 1 do not alter levels of the secreted amyloid-beta protein precursor generated by beta-secretase cleavage.\", \"hash\": \"46788979c02fe622106bed5e8b6d9d75d98898823128748e8b9e68a543416888beb27f3d37a612d5eb26dbd6b5dee99e33ffa2904c74271f5f8fa366e99f592d\", \"key\": 56, \"line\": 26330, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochim Biophys Acta2010\", \"reference\": \"20303415\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta is derived from a type I transmembrane protein, amyloid precursor protein (APP), by the sequential proteolytic events mediated by beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase.\", \"hash\": \"82e9a01fd87c566bd17a82cea6d92e897b108c9407026418e6e51e866fa2191feb4eb0adebbfd3aa14b7ee50180d590e8068afc97c164778871c6f094a16ac18\", \"key\": 57, \"line\": 26340, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Brain2010\", \"reference\": \"20409323\", \"type\": \"PubMed\"}, \"evidence\": \"BACKGROUND: A central feature of Alzheimer's disease is the cleavage of the amyloid precursor protein (APP) to form beta-amyloid peptide (Abeta) by the beta-secretase and gamma-secretase enzymes.\", \"hash\": \"a79debcf0fbc66f96734c3f3f3a3b4d294ee54c00d53c31e0eb279f51fdab0b1b8781a51848a8066276621f9e7f895bd205eca5ddac42488ef4daa6dccba072b\", \"key\": 58, \"line\": 26350, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol2010\", \"reference\": \"20451519\", \"type\": \"PubMed\"}, \"evidence\": \"Basic understanding of the activities of the amyloid beta peptide (Abeta) and associated proteins such as beta-site APP-cleaving enzyme 1 (BACE1) is necessary to develop effective medical responses to AD.\", \"hash\": \"9f4134d8f5e421056e69b5bd70c91b26edaae07c8eab8856305bdecb00821a0d924b17fa23f8c83921bb6667eaa59685cc3511c887c6fe9beaebee51b84be792\", \"key\": 59, \"line\": 26366, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol2010\", \"reference\": \"20685197\", \"type\": \"PubMed\"}, \"evidence\": \"Cleavage of APP by BACE1 is the first proteolytic step in the production of amyloid-beta (Abeta), which accumulates in senile plaques in Alzheimer's disease.\", \"hash\": \"a0966c77a3fd204b125b3551a81624653531be1711cc3e3af9ca94a8c9e90e3358b6dbfab8de9131a34c8b1203bb6df603c2baea639fbb8a3db636d47d1d14af\", \"key\": 60, \"line\": 26438, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett2010\", \"reference\": \"20727383\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta is generated from the beta-amyloid precursor protein (APP) through the proteolysis of beta-site APP cleaving enzyme 1 (BACE1) and gamma-secretase. \", \"hash\": \"8bad408f0dcc955a8d3e01b2b33162fc7ad155ced8ff2e526b430532640eb958dbf1859c11ea1b4c40749ea209e2b012d2a9ea16ed0fbbdbcdc3f486ea455fd5\", \"key\": 61, \"line\": 26448, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"BMB Rep2010\", \"reference\": \"21034535\", \"type\": \"PubMed\"}, \"evidence\": \"The Swedish mutation (K595N/M596L) of amyloid precursor protein (APP-swe) has been known to increase abnormal cleavage of cellular APP by Beta-secretase (BACE), which causes tau protein hyperphosphorylation and early-onset Alzheimer's disease (AD).\", \"hash\": \"7e27f2be5bc2285fe69f53fcbba118a7ea9b12b3a5f576c7beb68b82330c8830fe0e6e4b1745f4bd218c60014038c72a069eed00154da4f89b706aabe15eb9c6\", \"key\": 62, \"line\": 26460, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Neurobiol2011\", \"reference\": \"21431475\", \"type\": \"PubMed\"}, \"evidence\": \" In human neural cells overexpressing beta-amyloid precursor protein (betaAPP), the lipid mediator suppressed Abeta42 shedding by downregulating beta-secretase (BACE1) while activating the alpha-secretase (ADAM10), thus shifting the betaAPP cleavage from the noxious amyloidogenic pathway into a non-amyloidogenic, neurotrophic pathway. \", \"hash\": \"74e6bce61076dbcdf9afe284f5e69f7cfeb9156947f1542fd9f836c19ff896d514344c0363f0f287e60e9b121675c40e71b6b92e313ca2f2d19d419255258174\", \"key\": 63, \"line\": 26472, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci Res2011\", \"reference\": \"21433051\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid-beta peptide (Abeta) is generated by sequential cleavage of the amyloid precursor protein (APP) by beta-site amyloid precursor protein cleaving enzyme 1 (beta-secretase, or BACE1) and gamma-secretase. \", \"hash\": \"915608c220697151d2f34ae97d030809e6d735259554768ad53899d524c94ff3ae1207bc0801485e105e8d241c039f341c5443bd9b432f7b95b3c63710801fd8\", \"key\": 64, \"line\": 26482, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Nat Med1998\", \"reference\": \"9623986\", \"type\": \"PubMed\"}, \"evidence\": \"To release the amyloidogenic peptide A beta from the Alzheimer amyloid precursor protein (APP), two secretases act sequentially: first, beta-secretase cleaves close to the membrane within the ectodomain and then gamma-secretase cuts within the transmembrane domain.\", \"hash\": \"fdda05c5f260606855a38f636a00ae6d1c190436d22ddd7e89fa7f13cf2e53a795d08e6fdd60aad02d286ec1705e5da22327de111819099f6e7ba94e64a89a67\", \"key\": 65, \"line\": 26493, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol1998\", \"reference\": \"9775403\", \"type\": \"PubMed\"}, \"evidence\": \"Proteolytic processing of APP by beta-secretase, on the other hand, exposes the N-terminus of beta-amyloid, which is liberated after gamma-secretase cleavage at the variable amyloid C-terminus.\", \"hash\": \"579aa3ae20a8db5480d45ac9aec37085a1129dea67c697e0dfa4a1068fcf4591aa575f434db6f2c6b1ef9a492e606c80deb8628b9932304c5f7674dbb7c3a97e\", \"key\": 66, \"line\": 26503, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Mol Neurosci1998\", \"reference\": \"10096041\", \"type\": \"PubMed\"}, \"evidence\": \"Pulse-chase experiments revealed beta-secretase cleavage from immature full-length amyloid precursor protein harboring the Swedish mutation.\", \"hash\": \"259cd8fc8e9683674e96c2fa7f05217f918c831497768b7669dfdc60fb620f596fb405e0e1fe60fea8d955e1fe2b5f7e71d780452239f5b5564b20f02d5e80a4\", \"key\": 67, \"line\": 26520, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Nature1999\", \"reference\": \"10591213\", \"type\": \"PubMed\"}, \"evidence\": \"Solubilized Asp2 protein cleaves a synthetic APP peptide substrate at the beta-secretase site, and the rate of cleavage is increased tenfold by a mutation associated with early-onset Alzheimer's disease in Sweden.\", \"hash\": \"f7a89a05e680987c3746c8f3cd409adc6b8e1791efdb7734cf4029d13a6265936e925895a2ccab5f4f6e7614f853942c91818da72ade907341c8f1d7f6e97912\", \"key\": 68, \"line\": 26530, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell Neurosci1999\", \"reference\": \"10656250\", \"type\": \"PubMed\"}, \"evidence\": \"Transient expression of Asp 2 in cells expressing APP causes an increase in the secretion of the N-terminal fragment of APP and an increase in the cell-associated C-terminal beta-secretase APP fragment.\", \"hash\": \"f695265062baa06eb81a2613db47fcbf7663af0dfb30008f366ef277521ef7f8c40cec1f825f4928202cef8794f9ef41d57757b92a6f5e34fe619991a46715ba\", \"key\": 69, \"line\": 26540, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10677483\", \"type\": \"PubMed\"}, \"evidence\": \"Recombinant memapsin 2 specifically hydrolyzed peptides derived from the beta-secretase site of both the wild-type and Swedish mutant beta-amyloid precursor protein (APP) with over 60-fold increase of catalytic efficiency for the latter. Human aspartic protease memapsin 2 cleaves the beta-secretase site of beta-amyloid precursor protein.\", \"hash\": \"a8cbe89889ef7ec5efb27a90a55176504b3e5d57e4ec56fdbcb5a7c233434fa95b3f3b4f7b5ee80ce8d9a365316564a045e099b2748c1bc7be66b079cc44d4f0\", \"key\": 70, \"line\": 26551, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis2000\", \"reference\": \"11114266\", \"type\": \"PubMed\"}, \"evidence\": \"IFNgamma in combination with TNFalpha or IL-1beta seems to trigger Abeta production by supporting beta-secretase cleavage of the immature APP molecule.\", \"hash\": \"cf0ffb412729f31922f5a714d1e69edc0e05524cc5fcefe122a6e6dd19ffa3aebb14cde54836db7e5cc836557a6550a78725a3e686a9ae264b175ab1b3f2a2cc\", \"key\": 71, \"line\": 26561, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Traffic2001\", \"reference\": \"11737828\", \"type\": \"PubMed\"}, \"evidence\": \"Amyloid precursor protein/P-selectin, which is sorted from early to late endosomes, undergoes significantly less alpha-secretase cleavage, and more beta-secretase cleavage, than amyloid precursor protein/P-selectin768A, a mutant that recycles more efficiently to the cell surface.\", \"hash\": \"c136c4b50429cfb5a45664af67273835e218767e25a24fbb2e42844b6ddc205a65b449cc1f4f3f92f494179ca9e578d474755dc9aad4ebaf46b62476968efc5d\", \"key\": 72, \"line\": 26576, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol2002\", \"reference\": \"11839594\", \"type\": \"PubMed\"}, \"evidence\": \"Homeostatic APPsbetaswe levels with aging suggest that progressive amyloid deposition in brain results not from increased beta-secretase cleavage of APP but from impaired Abeta/amyloid clearance mechanisms.\", \"hash\": \"2ec823a80fac8befcb0ca1973d97017893a191e3eb2f6eb44dbcefb5d6dcaf175b1a2bc0b9d538d7caa5244f019fe6e4e4fa22199d82431406477d3dfd11a88b\", \"key\": 73, \"line\": 26586, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2002\", \"reference\": \"11953452\", \"type\": \"PubMed\"}, \"evidence\": \"ELISA analysis of beta-secretase cleavage of the Swedish amyloid precursor protein in the secretory and endocytic pathways.\", \"hash\": \"838d6b8dcfc590c4c884718fc0f2f6049a804460e0d2109112a1af8859eea317d2050d71312e8fdbc3d1c7708ce771fd2e62784d812fdcdc164faa977d746698\", \"key\": 74, \"line\": 26595, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2002\", \"reference\": \"11953458\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Site APP-cleaving enzyme (BACE) initiates the processing of the amyloid precursor protein (APP) leading to the generation of beta-amyloid, the main component of Alzheimer's disease senile plaques.\", \"hash\": \"3beb86218fc5fdc264c7d4bcad60faa428c1e2919a19e277dea53066f5b62290a3be9116e980098e10350ff37e2e9f629db572dffe8e4f578dcfc915c24104a2\", \"key\": 75, \"line\": 26605, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Ann Neurol2002\", \"reference\": \"12112088\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-secretase cleavage represents the first step in the generation of Abeta polypeptides and initiates the amyloid cascade that leads to neurodegeneration in Alzheimer's disease.\", \"hash\": \"8a059f8e37b066b0b380fe73b204a71a039ea9e9c4c6fe7b268634f3490e07cc4a5053680c4bbc8270692ffbecbc5ea5865fc0e5997e52a59e949c27adcfb6af\", \"key\": 76, \"line\": 26615, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging2002\", \"reference\": \"12470797\", \"type\": \"PubMed\"}, \"evidence\": \"Our recent knockout studies show that BACE1 is critical for Abeta generation, but the knockout mice show an otherwise normal phenotype, raising the possibility that therapeutic BACE1 inhibition could be accomplished without major mechanism based toxicity.\", \"hash\": \"75cfbeb40c2b1dca938379d2b2cd89bdd002e22c68f3eabc78c686dedfddff5650692bb36de46cd7b630185becd8a948b32f2e58d6237770b5ec942acfb89f6c\", \"key\": 77, \"line\": 26638, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2003\", \"reference\": \"12473667\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis.\", \"hash\": \"3683dd1b59b9cbbf4d6beb943ad2bbe552b0d1933630ac8d16e266fb7acc28ecead1400ebccb3836c91e3f02aadaa61d91c0c46780922334dec520350b04c2de\", \"key\": 78, \"line\": 26648, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2003\", \"reference\": \"12515826\", \"type\": \"PubMed\"}, \"evidence\": \"APP is cleaved either by beta-secretase or by alpha-secretase to initiate amyloidogenic (release of A beta) or nonamyloidogenic processing of APP, respectively.\", \"hash\": \"003bb7146dd1549f9636fc044125783c49ed4db9bd6ff94ddbd2e4cf8c7c93fcdae9c29454de309b9436ee7a2f36d757cde24eb66eada15849ea6d4680a7c357\", \"key\": 79, \"line\": 26659, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2003\", \"reference\": \"12603825\", \"type\": \"PubMed\"}, \"evidence\": \"Employing a superior BACE1 cleavage sequence to probe cellular APP processing.\", \"hash\": \"bdcfba69e59c21e641110dd01f194974b748ca18e7fce516eccd54ce417fd1b0c5c50ac7deec1f21c9739d6f2c11fa7d5bae9ea82dbc632ce37098f8bd5a4f08\", \"key\": 80, \"line\": 26668, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Mol Neurosci2003\", \"reference\": \"14501002\", \"type\": \"PubMed\"}, \"evidence\": \"An integral membrane aspartyl protease, BACE, is responsible for beta-secretase processing of the beta-amyloid precursor protein (APP) to the large secreted sAPPbeta and membrane-bound CTFbeta of 99 residues.\", \"hash\": \"b439c78f20269c02d8d6533bdc8ae826b6f31e5e9ce3bf50d3a96b5eb885d2c279ab7c6511c68295b3a9b39b1b524739c6e8425127bca626fbf64c5d486b6956\", \"key\": 81, \"line\": 26678, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2003\", \"reference\": \"14530380\", \"type\": \"PubMed\"}, \"evidence\": \"Here we report evidence that heparan sulfate (HS) interacts with beta-site APP-cleaving enzyme (BACE) 1 and regulates its cleavage of APP.\", \"hash\": \"92ed6f28a8af727c8dfee5d6794a79b43a81f0651228d9332e935ae0ead7140d973d1f98ca6c1426fa38015476519c407ca32f7a8afbc61d1768a0da6743fef5\", \"key\": 82, \"line\": 26687, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2003\", \"reference\": \"14557249\", \"type\": \"PubMed\"}, \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"hash\": \"692b479b50d5c137aedd9a722bba1b599ca63fd22d9da14090a2ee4399edd6431127f2662e54d78a7e36417c7ef22adba4356decde0d2aaf6867b802fd293e33\", \"key\": 83, \"line\": 26698, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2004\", \"reference\": \"14600149\", \"type\": \"PubMed\"}, \"evidence\": \"Our results indicate that BACE1 siRNA specifically impacts on beta-cleavage of APP and may be a potential therapeutic approach for treating AD.\", \"hash\": \"9e85128ecfd705cfc3a27ba0c26a066a2d65edaa46053c210f9e2c35be9d8f070827b22acf9ef1adbe97533b61d931c5f5004def8fbb7f02cb55fa7b97ad1d00\", \"key\": 84, \"line\": 26709, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Anal Biochem2003\", \"reference\": \"14622952\", \"type\": \"PubMed\"}, \"evidence\": \"The cerebral deposition of amyloid beta-peptide (Abeta) is a major factor in the etiology of Alzheimer's disease. beta-Secretase (BACE) initiates the generation of Abeta by cleaving the amyloid precursor protein at the beta-site and is therefore a prime target for therapeutic intervention.\", \"hash\": \"2ba80406e5477e463545ec15347bd5ce58128fb90e23371e82e929208122f7801d998000c72e09c3a1a0326738101fcc5120485b19ae0cffaba32fdc7aef6ae1\", \"key\": 85, \"line\": 26720, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"Mol Cell Biol2004\", \"reference\": \"14701757\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1, the major beta-secretase involved in cleaving APP, has been identified as a type 1 membrane-associated aspartyl protease.\", \"hash\": \"aa7a5b09c38db2af4454cc32ae208cceca4e3c1d82d507e2d96ed04437c2d1021136a6ce42573c5c98129a79427aafb74c6693e878beaf728188b6044f741416\", \"key\": 86, \"line\": 26728, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neuron2004\", \"reference\": \"14715132\", \"type\": \"PubMed\"}, \"evidence\": \"beta-site APP cleaving enzyme 1 (BACE1) is the beta-secretase enzyme required for generating pathogenic beta-amyloid (Abeta) peptides in Alzheimer's disease (AD).\", \"hash\": \"68f6e568664694be76bce630393d41f316d4832a4718913e095c2bca19780dbd9e122af2f43b8f7ca6d11b227446523c8c8db39e49163f4c6f6cae52c6ba2212\", \"key\": 87, \"line\": 26738, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Glycoconj J2004\", \"reference\": \"14973371\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's beta-secretase (BACE1) is a membrane-bound protease that cleaves the amyloid precursor protein (APP) in the trans-Golgi network, an initial step in the pathogenesis of Alzheimer's disease.\", \"hash\": \"ffdaf4bbe527c2948e3aa1ec60c201803d74723478eafdf3aad7244fe11d621fe1c90c7ce9dfeeb426a6954c726647426d3255c54982ae319b4fea0653e1cfbf\", \"key\": 88, \"line\": 26748, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neural Transm2004\", \"reference\": \"15057522\", \"type\": \"PubMed\"}, \"evidence\": \"The activity of BACE-1 as measured by the formation of the cleavage product of amyloid beta precursor protein, transiently increased up to 48 h after injury, but returned to basal level 7 days post injury.\", \"hash\": \"7324da911796c4b2de491e18f94247e96a3fa1219624928b182de64be7466483406d4da248aa96f7e2bddfd2c0fe100ecf2b9625632900f813db03d3c4c2f4da\", \"key\": 89, \"line\": 26764, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell Neurosci2004\", \"reference\": \"15080893\", \"type\": \"PubMed\"}, \"evidence\": \"BACE is an aspartyl protease that cleaves the amyloid precursor protein (APP) at the beta-secretase cleavage site and is involved in Alzheimer's disease.\", \"hash\": \"4b30b6f7a784f855f585743c690c63b85f2e1db607ec953cae715a724f600d0472487cf7579d8d6b71031ca8b0c1342bac17046a9a24110e08b1f7e75ef330a9\", \"key\": 90, \"line\": 26774, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci Res2004\", \"reference\": \"15211591\", \"type\": \"PubMed\"}, \"evidence\": \"The beta-site APP-cleaving enzyme (BACE1) has been identified as the key enzyme leading to beta-amyloid formation, and cholinergic mechanisms have been shown to control APP processing.\", \"hash\": \"a681a809d2774f7f252fe0d0fca5621aea7145ebfdef5ff23d9eb1f1f739120586d268fc326112ea6fc87a4ead6eb265a3a362c348ec2a1536681a0f06c055c0\", \"key\": 91, \"line\": 26785, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Mol Neurosci2004\", \"reference\": \"15314262\", \"type\": \"PubMed\"}, \"evidence\": \"Because alpha-secretase and BACE-1 cleave APP within the secretory pathway, it is likely that the two enzymes compete for the APP substrate.\", \"hash\": \"eea648711259ef886fec64d95c16a7e46739daf6d7f071847da765910d6da1aacfc0896a595a5df2deb599004811d9dbcc9cd55157899c258188af8f7cc2d587\", \"key\": 92, \"line\": 26804, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Glycoconj J2004\", \"reference\": \"15467394\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer's beta-secretase (BACE1) cleaves amyloid precursor protein to produce amyloid beta-peptide, which is a crucial initiation process of the pathogenesis of Alzheimer's disease.\", \"hash\": \"0627c8198372d888a06e56f81c243fcf42d8bab15851ba8c7c48c8f22d6b88f18085895bc6af53e124ba8e9cc22d6a9bcd21671d8e5e602c4d77f8c8981ea23c\", \"key\": 93, \"line\": 26814, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Proteome Res2004\", \"reference\": \"15473697\", \"type\": \"PubMed\"}, \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"hash\": \"9efe7b9cc754842d3027290e27cd83b3d55f716da7d172a8c80d4d17dda4a5fd0766f9d4b8274113cf24fbd440d6068838f61117ac5d00a1698dc0b62d7a35d3\", \"key\": 94, \"line\": 26824, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2004\", \"reference\": \"15584903\", \"type\": \"PubMed\"}, \"evidence\": \"Preventing Abeta42 production with an M596I mutation (beta-), which blocks beta-secretase cleavage of APP, or by treatment with a gamma-secretase inhibitor increased the resistance of APP(FAD)-expressing cells to apoptotic process.\", \"hash\": \"9d6af3f1161c4e1438a1a60c7fa4f85f5646a5474157a8aa22f412b7b8f619fc3441d0f6b13947c0824b373fd415fcb3af55d10ea5ff803b69ba2e2ceaa9644a\", \"key\": 95, \"line\": 26836, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2005\", \"reference\": \"15642747\", \"type\": \"PubMed\"}, \"evidence\": \"Although pmodest overexpression enhanced amyloid deposition, high BACE overexpression inhibited amyloid formation despite increased beta-cleavage of APP. BACE overexpression alters the subcellular processing of APP and inhibits Abeta deposition in vivo.\", \"hash\": \"2b71717398b1e9ccab34d06964b87f91475bb975019841b9c8e5b726bfb9dce10acfba136f6f72521242b842646cb1d7ce82c6c15ef30d7d9dc9c82a8f300cdd\", \"key\": 96, \"line\": 26846, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Biol2005\", \"reference\": \"15767460\", \"type\": \"PubMed\"}, \"evidence\": \"Here, we established a novel approach to regulate production of Abeta based on intracellular expression of single chain antibodies (intrabodies) raised to an epitope adjacent to the beta-secretase cleavage site of human APP.\", \"hash\": \"db7b87bfb149e3a8103763b1586f6194c713a38d698574d002768f616c22723c356da9f04ecc3ae63e1a0e128d624a52befa8e51de56b2218f4efe2ab6cc00c3\", \"key\": 97, \"line\": 26856, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"FASEB J2005\", \"reference\": \"15857888\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 increased APP cleavage at the beta-site and Abeta production whereas BACE2 did not.\", \"hash\": \"1b8108e69ffd86ebf81e60c0638f87888c77da7976aa0f604614cd57941f7707303f2239c4c192b9a97cb8f68eb1f7f0e5a978eb355a6569515b8e39047e01b4\", \"key\": 98, \"line\": 26865, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Structure2005\", \"reference\": \"16216580\", \"type\": \"PubMed\"}, \"evidence\": \"The aspartic protease beta-secretase (BACE) cleaves the amyloid precursor protein into a 42 residue beta-peptide, which is the principal biochemical marker of Alzheimer's disease.\", \"hash\": \"16a5724c07b19a3538f14f5137b3a7ecfeb88cf38670da2618c76232eb3a2f9ef589822d8645a0044e65c9acbb2b407e4166a6801bc056b655c97ecb76ec3661\", \"key\": 99, \"line\": 26875, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2005\", \"reference\": \"16306400\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-secretase [beta-site amyloid precursor protein-cleaving enzyme 1 (BACE1)] is the key rate-limiting enzyme for the production of the beta-amyloid (Abeta) peptide involved in the pathogenesis of Alzheimer's disease (AD).\", \"hash\": \"af59628d714c0d3c44e5565b6b8daf8fa999e34990bd66a0e5f1c2d0ad24b29d8cae183f775a2db54750fdece6cc4cbfa094db37df462762baac0be1ff170ad6\", \"key\": 100, \"line\": 26886, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"EMBO J2006\", \"reference\": \"16407971\", \"type\": \"PubMed\"}, \"evidence\": \"This was associated with inefficient plasminogen binding and plasmin activation, the displacement of beta-secretase (BACE) from DRMs to APP-containing membrane fractions, increased beta-cleavage of APP and high levels of Abeta peptides.\", \"hash\": \"090e2fa7124c64cf8a55cc11d6999934cb9504d26c81598a069fcb14bdd04fc4e6fa1d67f99d1a114d28ba6ca634362f44d61fb6734e2eb7691778c37cae8ad7\", \"key\": 101, \"line\": 26895, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry2006\", \"reference\": \"16716081\", \"type\": \"PubMed\"}, \"evidence\": \"Heparin can promote beta-secretase cleavage of APP in neuroblastoma cells.\", \"hash\": \"f2676c061c9c3ef83975532cf84a13d46eef1126c876f56ee41342122d5f7037ffc1fd43977d48f134004ffb478af4fd5f48d831528a14ee77db3387473daa79\", \"key\": 102, \"line\": 26904, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Bull Mem Acad R Med Belg2005\", \"reference\": \"16768248\", \"type\": \"PubMed\"}, \"evidence\": \"In AD, the specific inhibition of beta- or beta-secretase activities would decrease the production of Abeta from its precursor, in such a way that its relative concentration could be low enough to avoid the formation of aggregates.\", \"hash\": \"981d00852e9361e62b0d1bd14f2ef648d5a979ec2d15de88c743d330376f043db50196249b4ae955df933e3705a13b890c371b13b8b50647205d349ca8fce59d\", \"key\": 103, \"line\": 26915, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Mol Biol2006\", \"reference\": \"16979658\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"hash\": \"87d4d98783aad9ea03c98434d54ebbac0e3a3b3684e5d5916c9e893c94f9b9ffa404e9e08c9cd48c6cf5ef5b94b1d45d2239435e1491feb0fd7bd21e0df52c4e\", \"key\": 104, \"line\": 26959, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\", \"type\": \"PubMed\"}, \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"hash\": \"789cf95ff6c739b3ae09305ec05cd304a99a2bb3d296bbc25aaf70af9da290cea8cf668652952d7103ce7078358ca7ef5ce0be0a8e76dc2b39495ce8b453f1ff\", \"key\": 105, \"line\": 26971, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Hypoxia response subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17121991\", \"type\": \"PubMed\"}, \"evidence\": \"Hypoxia up-regulated beta-secretase cleavage of APP and amyloid-beta protein (Abeta) production by increasing BACE1 gene transcription and expression both in vitro and in vivo.\", \"hash\": \"2a2fe23c16b0600bf10d92fba9d5b93b04c7584844a1294ef57150fa344fca4e86902729087011475183dbf521c69ec30e84124ae73b7a8ecc87d94be5be6b7e\", \"key\": 106, \"line\": 26988, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2007\", \"reference\": \"17156133\", \"type\": \"PubMed\"}, \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of Alzheimer's disease.\", \"hash\": \"1255da51f42fb409c902bea4ade55463861e56b58ef2bd98c7f15fb52fac4c9035eba030ad9964ab1bfa54018e0e8476e38a46a68348b2c989ef91e58fdd48e8\", \"key\": 107, \"line\": 26998, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"hash\": \"ba034390874b269ed6058484a396e8ad2d397cc17696f6ea7ac8344c12202050be96b88ee68415de522a0a51e38b0117486af0107aa127c2b11d993807141fa7\", \"key\": 108, \"line\": 27007, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2007\", \"reference\": \"17409228\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) (beta-secretase) initiates generation of beta-amyloid (Abeta), which plays an early role in Alzheimer's disease (AD).\", \"hash\": \"1eba508c03cbee66232c05ccec8f10b819d583c1cb20a4a7cd56fd1790fcaf53675018b06d635108b74a77f725762ab293f9f8c43ce479ad6cb1d2ad8fba2008\", \"key\": 109, \"line\": 27019, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Comput Biol Chem2007\", \"reference\": \"17500040\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-secretase is a potential target for inhibitory drugs against Alzheimer's disease as it cleaves amyloid precursor protein (APP) to form insoluble amyloid plaques and vascular deposits in the brain.\", \"hash\": \"8fbe12d55c478e49fcf4cb8c22491c2c3a66892f26228ac1ab7c6adf5737d9a0ca02f887a1fe75eb795dc6edb774f6310d2ac30e070ceb08e3cd5a5d312e72dc\", \"key\": 110, \"line\": 27029, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Anal Bioanal Chem2007\", \"reference\": \"17541560\", \"type\": \"PubMed\"}, \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach to the treatment of Alzheimer's disease.\", \"hash\": \"9669e8274819dda281205791b57ffc43bec2be8b5bd5b37d3e66aa2277958ea03832d41d43767dd5d49d8adad381d9d8d0b909c391a8806e408d683b27d64e91\", \"key\": 111, \"line\": 27044, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neuron2007\", \"reference\": \"17553422\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-site APP-cleaving enzyme (BACE) is required for production of the Alzheimer's disease (AD)-associated Abeta protein.\", \"hash\": \"e565d543d8e120b8ddf5f0312d46bced392233b7a6e3a5fb48e6d7888f4b090f847593b67f84122d2aff7e8b65ea3b5bf25e149b96e472df7ad1434b5edfda05\", \"key\": 112, \"line\": 27059, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2007\", \"reference\": \"17616527\", \"type\": \"PubMed\"}, \"evidence\": \"The aspartyl protease beta-site amyloid precursor protein cleaving enzyme 1 (BACE1) initiates processing of amyloid precursor protein (APP) into amyloid beta (Abeta) peptide, the major component of Alzheimer disease (AD) plaques.\", \"hash\": \"05d4afbddd787d6eebaeafd00c9fcbc89b76f6ae9137dd90857a0c073092170aa4d5ecefc8967fbae6118555a7a5dd9d71572c8e70d00ffe928b287fff6af890\", \"key\": 113, \"line\": 27069, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Bioorg Med Chem Lett2008\", \"reference\": \"18068983\", \"type\": \"PubMed\"}, \"evidence\": \"The proteolytic enzyme beta-secretase (BACE-1) produces amyloid beta (Abeta) peptide, the primary constituent of neurofibrillary plaques, implicated in Alzheimer's disease, by cleavage of the amyloid precursor protein.\", \"hash\": \"28b73a08c551705aa9b21981da856dbd120f77491074b4c48ef52a9cbbeeb078c461027c8fedbc3144d69273881b55a70b4f256a30405d3389f89f4bc7142003\", \"key\": 114, \"line\": 27079, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"J Biol Chem2008\", \"reference\": \"18184658\", \"type\": \"PubMed\"}, \"evidence\": \"After inhibitor treatment, the improved memory function was accompanied by reduced amyloid plaque load, decreased Abeta40 and Abeta42, and reduced C-terminal beta-secretase fragment derived from APP by beta-secretase.\", \"hash\": \"9caf1fb0adf05db463ddb5d4d9c52665f43482b691071fff16da5b39dd5fcd7e081af9bae9d45aef3d0cfb73571984b7b2a3fb04abddb488e08106862a91225a\", \"key\": 115, \"line\": 27087, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2008\", \"reference\": \"18263584\", \"type\": \"PubMed\"}, \"evidence\": \"Beta-amyloid (Abeta) peptides that accumulate in Alzheimer disease are generated from the beta-amyloid precursor protein (betaAPP) by cleavages by beta-secretase BACE1 and by presenilin-dependent gamma-secretase activities.\", \"hash\": \"ffb8d82993a8b9b83c0e7a9424c6bf5f94dff81cfdfbed0e46075a0b82c7a15f200cd20b95af5e70f3e0113b80e1ace565a0d7cc8e04ae16540bd86ed4cb624a\", \"key\": 116, \"line\": 27097, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochim Biophys Acta2008\", \"reference\": \"18295609\", \"type\": \"PubMed\"}, \"evidence\": \"Consequently, inhibition of BACE-1, a rate-limiting enzyme in the production of Abeta, is an attractive therapeutic approach for the treatment of AD.\", \"hash\": \"9fcd2631d5dd272c4e6c9729bb2d8cf754db16834221be45e7aa62f631bc37560f51c91b7387a62808a9cdc579208bf646c5c8ac0335fc2039718fa11d691ad3\", \"key\": 117, \"line\": 27107, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\", \"type\": \"PubMed\"}, \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"hash\": \"6fe5f7f84b739403d062a0b9e7b728809706e5ef9e923c24723006efbd24e447a321f8730936f7c2f530cb99b59ff57e247360806b14c4d40351514bf0135007\", \"key\": 118, \"line\": 27122, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"CNS Neurol Disord Drug Targets2008\", \"reference\": \"18673212\", \"type\": \"PubMed\"}, \"evidence\": \"BACE-1 cleavage is limiting for the production of Abeta, making it a particularly good drug target for the generation of inhibitors that lower Abeta.\", \"hash\": \"39173ebb3c6bf4e88d210ff6ed69cca39b6765be4f0462d6c8513e06d1dc726366f221f72a2724f4761d2eb8981424b8d4f2c56ba68bcd6b4f542dc06b50cce6\", \"key\": 119, \"line\": 27132, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biol Chem2008\", \"reference\": \"18979625\", \"type\": \"PubMed\"}, \"evidence\": \"One group described the poor kinetics of BACE 1 for cleaving the wild-type (WT) beta-secretase site of APP found in most AD patients.\", \"hash\": \"918792193f93faaa338fccb286ed92593a669988f903e09ea29695886851165700289f7a865d8a7ae786bd5739ef5aedc8d0251f9614fcfbd769ebadaa8188ce\", \"key\": 120, \"line\": 27141, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"J Biol Chem2009\", \"reference\": \"19074428\", \"type\": \"PubMed\"}, \"evidence\": \"These results indicate that post-translational S-palmitoylation of BACE1 is not required for APP processing, and that BACE1 can efficiently cleave APP in both raft and non-raft microdomains.\", \"hash\": \"8dff389a02150a2bf9dfc8505dccb313bc1eb7176d9ea271eb98997bbea064fb829b8d316c5d7adcc413085ef0844df9f416554119f9636325f59de5a7d9054e\", \"key\": 121, \"line\": 27149, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"FASEB J2009\", \"reference\": \"19332646\", \"type\": \"PubMed\"}, \"evidence\": \"Because beta-site APP-cleaving enzyme 1 (BACE1), an essential protease for Abeta production, is up-regulated in cells overexpressing RAGE and in RAGE-injected brains of Tg2576 mice, the molecular mechanisms underlying RAGE, BACE1 expression, and Abeta production were examined.\", \"hash\": \"bc7b3c6a138ca39bb3f9d4e588720b833ca21ea7b4196081bd0ddf6049ac9c2f85fe58156e1902e6e46d845f6f9a30eaf02f8c32ce7561e97ce08eb8cc17cce6\", \"key\": 122, \"line\": 27160, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2009\", \"reference\": \"19366692\", \"type\": \"PubMed\"}, \"evidence\": \"However, unlike BRI2, the binding of BRI3 to the beta-secretase cleaved APP C-terminal fragment is negligible and BRI3 does not cause the massive accumulation of this APP fragment, suggesting that, unlike BRI2, BRI3 is a poor gamma-cleavage inhibitor.\", \"hash\": \"e8be7f86652418225bae4ab99a214ee433ff0a89db6189874ed1211d6e43196dce5ca371df893d3a8ab7501000003029ffcc1df5312e5ac33156beb5d0cde187\", \"key\": 123, \"line\": 27172, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2009\", \"reference\": \"19442147\", \"type\": \"PubMed\"}, \"evidence\": \"BACE 1 (beta-site APP-cleaving enzyme 1 or beta-secretase), the key enzyme required for generating Abeta from the beta-amyloid precursor protein (APP), is regarded as an ideal target for AD therapeutic drug design.\", \"hash\": \"92043cc6008b692fe1e435127acf053037c31b635e7754891b2d75a044ff1264cd073ba6c7f08f3db092edadbd2fb9b38ea9b4572ef3d534047c2bb572d3a2c4\", \"key\": 124, \"line\": 27182, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2009\", \"reference\": \"19828790\", \"type\": \"PubMed\"}, \"evidence\": \"The beta-secretase, beta-site amyloid precursor protein cleaving enzyme (BACE1; also called Asp2, memapsin 2), is the enzyme responsible for initiating Abeta generation.\", \"hash\": \"5ba34bc8093adbeca03ae27171ea00154591f7fcd580e0014ce2c1b7801c11d5747d311b8da5e3d791105b74d872d6a6ef7871c191e37b12fc07888b544779b3\", \"key\": 125, \"line\": 27192, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Eur J Neurosci2009\", \"reference\": \"20092570\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 initiates the amyloidogenic processing of APP; therefore, early or active amyloidogenic loci might exhibit site-specific BACE1 and Abeta elevation.\", \"hash\": \"2b29db021bee11b6f14f71fd7e415bba2b4e2f4d15cc5e55044ab511ddb29980957debbc5ebfbd00f6d139083e2bd0602e65669c709e60b87affb199cf9191d3\", \"key\": 126, \"line\": 27202, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2010\", \"reference\": \"20097758\", \"type\": \"PubMed\"}, \"evidence\": \"Taken together, ATXN1 functions as a genetic risk pmodifier that contributes to AD pathogenesis through a loss-of-function mechanism by regulating beta-secretase cleavage of APP and Abeta levels.\", \"hash\": \"331758577d5bf5822a0a21f3ef4dd4fa7e1507555f7f5fbbe9eddaac36881109478fde195234504e9722c78a5894e58cf5d1f3af1ae99457da3eb7448c5e0b69\", \"key\": 127, \"line\": 27212, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\", \"type\": \"PubMed\"}, \"evidence\": \"Three proteases that are involved in the processing of amyloid precursor protein-alpha-secretase, beta-secretase and gamma-secretase-are of particular interest as they are central to the generation and pmodulation of amyloid-beta peptide and can be targeted by small compounds in vitro and in vivo.\", \"hash\": \"a3740353d1e253194dd668a2853ecf42fadfa3f82a8bb920f32024ad4449f9b935a14a733bbb5ac3404d080a34fdb1504073d082853257a466d53383b157956b\", \"key\": 128, \"line\": 27232, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Sci Transl Med2010\", \"reference\": \"20371462\", \"type\": \"PubMed\"}, \"evidence\": \"Although pmoderate decreases of either gamma-secretase or BACE1 are not associated with mechanism-based toxicities, they provide only pmodest benefits in reducing Abeta in the brains of APPswe/PS1DeltaE9 mice.\", \"hash\": \"2270b7e5bb15a0e422d526e0c9087c397a5ad4c8e5aa90cfb691cb0ae456fef4d04d1f35d2535e69a1e5b497fc9dd9271639da461ae5b458a91f240f24c3ef36\", \"key\": 129, \"line\": 27260, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neuropathology2010\", \"reference\": \"20374499\", \"type\": \"PubMed\"}, \"evidence\": \"Reticulons are a group of membrane-bound proteins involved in diverse cellular functions, and are suggested to act as inhibitors of beta-secretase enzyme 1 (BACE1) activity that cleaves amyloid precursor protein. \", \"hash\": \"3d2c86263d7008046223d19153de92c4c01553e66395d9914844160380931991eeb7069ac0827af814b7cfbafc029702dc21dd725d4550020c7ef2d9a921ea90\", \"key\": 130, \"line\": 27269, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2010\", \"reference\": \"20592218\", \"type\": \"PubMed\"}, \"evidence\": \"Here we tested the possibility of targeting the cellular environment of beta-secretase cleavage instead of the beta-secretase enzyme itself. beta-Secretase has an acidic pH optimum and cleaves the amyloid precursor protein in the acidic endosomes.\", \"hash\": \"3f341d7c8fc8dad9759df45a26f697668ef900a29da7b774631338c2d01a99550b1e0086e8e36d7b22e8bd8edabaa50815b175e7bcd1af2009aa549a63c730d7\", \"key\": 131, \"line\": 27280, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res2010\", \"reference\": \"20704561\", \"type\": \"PubMed\"}, \"evidence\": \"ABCA2 expression promoted b-secretase (BACE1) cleavage of APP not at the common Asp1 amino acid site (beta-site) of Abeta in APP but at the Glu11 site (beta'-site) to increase C89 carboxyl-terminal fragment levels (beta'-CTF/C89). \", \"hash\": \"ae8618cc9532920ec80f7028b9f9fa2bc06cb71dfc34a5341817d0b670be8f878e76ee1e6df8f7456acfc40d14d526003e4d8c5d2b16539ee2006cf0d877c1a3\", \"key\": 132, \"line\": 27289, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2010\", \"reference\": \"20720123\", \"type\": \"PubMed\"}, \"evidence\": \"We discovered a nonpeptidic compound, TAK-070, that inhibited BACE1, a rate-limiting protease for the generation of Abeta peptides that are considered causative for Alzheimer's disease (AD), in a noncompetitive manner.\", \"hash\": \"89f79fd172904f2a7bd20adfcf639a45d561d0c93047296a3371387758105f8f1bec9292a36201c519e898d5319ba7b07b34599c6883e301a2e327b93b9f2d9a\", \"key\": 133, \"line\": 27302, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Eur J Neurosci2010\", \"reference\": \"20726888\", \"type\": \"PubMed\"}, \"evidence\": \"Western blot analyses detected increased levels of BACE1 protein and beta-site-cleavage amyloid precursor protein C-terminal fragments in plaque-bearing human and monkey cortex relative to controls. \", \"hash\": \"6f94846d37bf0d9a5294d2d0aa933f7461e42f40126326bcc538a895a0a67b38ab64c667edff76ab6f7612ae54f1ebe24b0ede2c768150e95056b45afc060982\", \"key\": 134, \"line\": 27312, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"PLoS One2010\", \"reference\": \"20886088\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Site APP-cleaving enzyme 1 (BACE1) initiates amyloid-beta (Abeta) generation and thus represents a prime therapeutic target in treating Alzheimer's disease (AD).\", \"hash\": \"792b22f6309b71e6ce767422053d83aa2ed0d27a211af9c31e3ab51ecb9e49b020163bc32bdd74c48cab5cb0f1e7a1f61846ec9cae0096d63935271c9c19df98\", \"key\": 135, \"line\": 27322, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Mol Brain2010\", \"reference\": \"21059265\", \"type\": \"PubMed\"}, \"evidence\": \"Importantly, recent evidence reveals that expression and activity levels of the beta-site APP cleaving enzyme 1 (BACE1), which initiates amyloid-beta (Abeta) production, are elevated in AD brains.\", \"hash\": \"f126539db478132f0715cf52ba42ba4a9899aa185808aa47d5fb86ef2bcc10e6810ab82feab8870602b06bed9726bd48d8db3d9c2e49aa8db637c23382f2e717\", \"key\": 136, \"line\": 27332, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2011\", \"reference\": \"21245145\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 (beta-site beta-amyloid precursor protein (APP)-cleaving enzyme 1) mediates the first proteolytic cleavage of APP, leading to amyloid beta-peptide (Abeta) production.\", \"hash\": \"e2751000f803fadf703233daaa7be8b058e56c03fff358cc476c488471c8e871b371fa09684a1f95639cf3502fdedb13e41b6d2b7c4354ff3707f3a2096d9e55\", \"key\": 137, \"line\": 27342, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem1995\", \"reference\": \"7876155\", \"type\": \"PubMed\"}, \"evidence\": \"Increased beta-secretase cleavage of APP after introduction of the Swedish double mutation causes apical missorting of about 20% of beta-secretase-cleaved APP.\", \"hash\": \"332dacd791729d89890e9da7c4855c78de37fa0101be7250d78a4268271d39748594c531cd25c7f7ac39b785dc772aae5ddcd66b78a9c960326062d230da95be\", \"key\": 138, \"line\": 27351, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem1996\", \"reference\": \"8621560\", \"type\": \"PubMed\"}, \"evidence\": \"Taken together, the data suggest that the processing pathway for betaPP is similar for both betaPP-wt and betaPP-sw cells and that increased Abeta production by betaPP-sw cells arises from enhanced cleavage of mutant betaPP by beta-secretase, the as-yet unidentified enzyme(s) that cleaves at the NH2 terminus of Abeta.\", \"hash\": \"756a6c5ea1ef74d1a76e1a3a19cb290e78facc92f383c1ed4295f923e7b4ecbc7e6ef18abb10017ffa8042bcd3cee82d8bc364be1156a7f09ce4e90e64e16d8c\", \"key\": 139, \"line\": 27362, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurochem Int1997\", \"reference\": \"9152995\", \"type\": \"PubMed\"}, \"evidence\": \"Heparin promotes beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"hash\": \"7bf191ce9c08ba064693b69bba7a104190c0e9e95fbcad7a5be5a0f4b86c1c248674844e6c44042df5b2e802790311199f4bdcb169d2feee2b03129f4683aff5\", \"key\": 140, \"line\": 27371, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biol Chem2010\", \"reference\": \"20731541\", \"type\": \"PubMed\"}, \"evidence\": \"Sequential proteolytic cleavage of APP by beta-secretase and gamma-secretase liberates Abeta from APP.\", \"hash\": \"7d0d0d21fa025412f61666681e084ccd511a0326f4bf0df41157d26f607a6534e958246d7062069b094e4ccb027e1d100ab594906058e06581e5ed17f4f7041c\", \"key\": 141, \"line\": 27381, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Nat Med1997\", \"reference\": \"9055862\", \"type\": \"PubMed\"}, \"evidence\": \"Formation of stable complexes between two Alzheimer's disease gene products: presenilin-2 and beta-amyloid precursor protein.\", \"hash\": \"9f84e518baec87edb882403c7bdb98c13d30dcd1a55e0a30c800911e46b24841b7a71406e6dd130aae1095e6d5dc71fce2edf5fd564a2cbb921cf30b55722249\", \"key\": 142, \"line\": 27390, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"citation\": {\"name\": \"J Biol Chem2000\", \"reference\": \"10956649\", \"type\": \"PubMed\"}, \"evidence\": \"The novel transmembrane aspartic protease BACE (for Beta-site APP Cleaving Enzyme) is the beta-secretase that cleaves amyloid precursor protein to initiate beta-amyloid formation. As such, BACE is a prime therapeutic target for the treatment of Alzheimer's disease. \", \"hash\": \"ade7d85f40ef1ccc846b001c7b4f9f1fba64f7866c08b8a2ec40add5db25d73ea7d37f3e76597ca59ab3ee101175d4c7c090a7c14634c71ec310abb5dda61c2c\", \"key\": 143, \"line\": 27759, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Cell Sci2004\", \"reference\": \"15466887\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Secretase (BACE) carries out the first of two proteolysis steps to generate the amyloid-beta peptides that accumulate in the senile plaques in Alzheimer's disease (AD). \", \"hash\": \"3a66d4d591447cb2f8a65c062e0d95cb872276f92715d9596c92fbeed96e1ed3f76a027f09d0471439ec131a9d690ea8d8123cbcd17e8b97bb8478e16e008251\", \"key\": 144, \"line\": 27784, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Mol Biol2006\", \"reference\": \"16979658\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 is a membrane-bound aspartyl protease that specifically cleaves amyloid precursor protein (APP) at the beta-secretase site.\", \"hash\": \"87d4d98783aad9ea03c98434d54ebbac0e3a3b3684e5d5916c9e893c94f9b9ffa404e9e08c9cd48c6cf5ef5b94b1d45d2239435e1491feb0fd7bd21e0df52c4e\", \"key\": 145, \"line\": 27847, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2003\", \"reference\": \"12473667\", \"type\": \"PubMed\"}, \"evidence\": \"BACE1 is a membrane-bound aspartic protease that cleaves the amyloid precursor protein (APP) at the beta-secretase site, a critical step in the Alzheimer disease pathogenesis. \", \"hash\": \"22dff091908b9ed9cee964ffa491b4cb6e1ac8c0e89b860675be1d884adffe8b6f1432eab9c75291a4d436cbbfa2084ba84082fd218aca5f38bb42ba0737fbdb\", \"key\": 146, \"line\": 27867, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\", \"type\": \"PubMed\"}, \"evidence\": \"LRRTM3 promotes processing of amyloid-precursor protein by BACE1 \", \"hash\": \"daed403ac2aa881e3dc1b2061e5c11b65e736f0a1191e97bad22dee8643338ab58da0546a12c4752ae864456eb140d7c3cee76af72d4a9cb7d55a6d591c7a119\", \"key\": 147, \"line\": 30285, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2004\", \"reference\": \"15189355\", \"type\": \"PubMed\"}, \"evidence\": \"The complex mediates the intramembraneous proteolysis of beta-secretase cleaved beta-amyloid precursor protein (APP) leading to the secretion of the Alzheimer's disease-associated amyloid beta-peptide (Abeta).\", \"hash\": \"ae78f06d2af7644e28ff6f653b2eca6263b7f8e0f526822a44ef8de2b6ecb2fb5bdbfc359943e82a125e19187da27035181c26316ab2a4b7435f0938c8adffa0\", \"key\": 148, \"line\": 30534, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2002\", \"reference\": \"11724784\", \"type\": \"PubMed\"}, \"evidence\": \"The familial Alzheimer's disease gene product amyloid beta precursor protein (APP) is sequentially processed by beta- and gamma-secretases to generate the Abeta peptide.\", \"hash\": \"312e8fdba738ae0c8143797f4da64786ca1c00d37841e3fffa0aaa5ef0524dca96fc6543846fb1122326fd5cac1e1b679190ef3f9e0d87b570302c81ed52eabd\", \"key\": 149, \"line\": 31512, \"relation\": \"increases\", \"source\": 63, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Physiological levels of ABeta also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta in homeostatic plasticity [29]. ABeta may have an important physiological role in synapse elimination during brain development\", \"hash\": \"a54a2244c096cff42d43f854774a6fd606d8d0c33196010a9ac63e6536194ca9dfcb517dce0ed7e595d2405eea46b1a35f838034de019f73d973952a1882347f\", \"key\": 150, \"line\": 35588, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Eur J Pharmacol. 2012 Jan 30;675(1-3):15-21. Epub 2011 Dec 7.\", \"reference\": \"22166376\", \"type\": \"PubMed\"}, \"evidence\": \"The level of Abeta-site APP-cleaving enzyme 1 (BACE1) has been documented to increase in the brains of patients with Alzheimer's disease, which has resulted in elevation of Abeta-amyloid (Abeta) peptides.\", \"hash\": \"7f69ce1686de043ec2a473ff6bd0185ff041b47d5078d77ef91f916f80738d85545e3162f794182a99800f8bb91a9128f3eef9f63bca8d56b3625548b7e35a20\", \"key\": 151, \"line\": 42713, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"Species\": {\"10090\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Epigenetic modification subgraph\": true}}, \"citation\": {\"name\": \"Free Radic Biol Med. 2009 May 1;46(9):1241-9. doi: 10.1016/j.freeradbiomed.2009.02.006. Epub 2009 Feb 23.\", \"reference\": \"19245828\", \"type\": \"PubMed\"}, \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"hash\": \"d4c4206360e51619adaecfbb591cee8f1ae8332c5f02064c19313930c0c976e4e7a3bea8fd9bf5ba22008e81c890d465fb988b28c91d08e5e2668bb2d5a76c51\", \"key\": 152, \"line\": 43364, \"relation\": \"increases\", \"source\": 63, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Insulin signal transduction\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A. 2009 Mar 10;106(10):3907-12. doi: 10.1073/pnas.0807991106. Epub 2009 Feb 23.\", \"reference\": \"19237574\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin modulates metabolism of -amyloid precursor protein (APP) in neurons, decreasing the intracellular accumulation of -amyloid (Abeta) peptides, which are pivotal in AD pathogenesis. The present study investigates whether the widely prescribed insulin-sensitizing drug, metformin (GlucophageR), affects APP metabolism and Abeta generation in various cell models. We demonstrate that metformin, at doses that lead to activation of the AMP-activated protein kinase (AMPK), significantly increases the generation of both intracellular and extracellular Abeta species\", \"hash\": \"5cc3fc9603ac2b72115a3b83a2a8e2a8e547e0b60d18b4d1375ec845b68a6b3c1edf441ab73eefd3c4fc4e5de5ef6ac6f20510c91107bd8527d744483dda44d3\", \"key\": 0, \"line\": 7754, \"relation\": \"decreases\", \"source\": 64, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Insulin signal transduction\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2006 Feb;27(2):190-8. Epub 2005 Feb 17.\", \"reference\": \"16399206\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin and insulin receptors were shown to decrease in a normal brain with aging, but increase in AD brains [29]. Several basic science studies have explored and shown the relationship between the increased insulin and AD pathology in the aspects other than Abeta degradation alone. For example, insulin increases the secretion of Abeta into extracellular space [31], stimulates tau phosphorylation to form neurofibrillary tangles, and impairs insulin signal transduction [32] and [40] (reviewed by Gasparini and Hoyer). Insulin also affected APP processing in vivo, a critical molecular step in generating Abeta, to secrete sAPP [13], [16] and [81]. In addition, Abeta reduces insulin binding to insulin receptors [95].\", \"hash\": \"89e216c64068278456f1b8d891f39a2ad25e7e2a8d5dc1e421d5b004511f705c9ff0a981821ca3e7644395e299c9895d739e232fb66056332b4a120f91282fc8\", \"key\": 1, \"line\": 8817, \"object\": {\"effect\": {\"fromLoc\": {\"name\": \"intracellular\", \"namespace\": \"GOCC\"}, \"toLoc\": {\"name\": \"extracellular space\", \"namespace\": \"GOCC\"}}, \"modifier\": \"Translocation\"}, \"relation\": \"increases\", \"source\": 64, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Akt subgraph\": true, \"Cholesterol metabolism subgraph\": true, \"Sphingolipid metabolic subgraph\": true}}, \"citation\": {\"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\", \"type\": \"PubMed\"}, \"evidence\": \"AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"hash\": \"3f6cb15c25754ff1c1a9d6cdcd81318590d5681382e2828674ab2193fbaf8a0c3b97b4d8188f3bb7c11b62d425adc3fd30ae3e4f2ed69c47c1a10cf904377e78\", \"key\": 0, \"line\": 3154, \"relation\": \"increases\", \"source\": 65, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Sphingolipid metabolic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\", \"type\": \"PubMed\"}, \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"hash\": \"b071a6457dbbea26cd1100f39284ebf22a4337b2a92c5b7adbf10d4ecf246a4137e2a1a68c9b91ea67c689f4f1c1d12da26932bb9b53cf9132ba4bd52d7fdb29\", \"key\": 0, \"line\": 2731, \"relation\": \"increases\", \"source\": 66, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"a91fcf7cdb86639c1e4e6c86d01be8e8d353a2f8008125b0aa57e5d3b76585a8f89e400ca12c2dd82ea3c9050fa20da771491e05289be3229cef6e8dbdebeedc\", \"key\": 0, \"line\": 35545, \"relation\": \"increases\", \"source\": 67, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"a1c0a506cdbeaf6cc5d62e1d2779aa480fa35673070f869ee92c25d3f009791539e255bc3f167567a4c15eade01cb267e9182c6a06a26d32101a2e475a981431\", \"key\": 1, \"line\": 35572, \"relation\": \"increases\", \"source\": 67, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of Neurochemistry2004\", \"reference\": \"14756819\", \"type\": \"PubMed\"}, \"evidence\": \"The phosphotyrosine binding domain of the neuronal protein X11alpha/mint-1 binds to the C-terminus of amyloid precursor protein (APP) and inhibits catabolism to beta-amyloid (Abeta), but the mechanism of this effect is unclear.\", \"hash\": \"a580ed5e57d8c4c3c7506b13129e6add058cd4a3f78caa71866ad1b6e176736e9bcbf5d262743c89c8f2140d728fa48b848c8af747515d34fbf0a6e19bf35ccf\", \"key\": 0, \"line\": 32886, \"relation\": \"decreases\", \"source\": 68, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuroscience2003\", \"reference\": \"12849748\", \"type\": \"PubMed\"}, \"evidence\": \"The neuronal adaptor X11alpha interacts with the conserved -GYENPTY- sequence in the C-terminus of amyloid precursor protein (APP) or its Swedish mutation (APPswe) to inhibit Abeta40 and Abeta42 secretion.\", \"hash\": \"be6879174a158880d337ec7690d25628bd342e951628c9898378138612c5c4aeeff8f3a58bde7e95b918cf9623da13f1ae49716ca2377278a3c611edf3733cc9\", \"key\": 1, \"line\": 32902, \"relation\": \"decreases\", \"source\": 68, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686 \", \"type\": \"PubMed\"}, \"evidence\": \"On the contrary, X11 stabilizes ABetaPP conformation in membrane, inhibiting ABeta secretion in cultured cells [45], likely impairing ABetaPP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptotic process. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABetaPP. The expression of JIP1b stabilizes immature ABetaPP and decreases the ABetaPP ectodomain, ABeta40/42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"hash\": \"6c587a987a9ecdac77744ff3c62d10172b29af0cefdc7667dc53b46aa9d5c335e70c67f0f20bbe534b34fe9bffb7a51c1e2333a89385db4fb1632cd9cfd8a9bc\", \"key\": 2, \"line\": 37042, \"relation\": \"decreases\", \"source\": 68, \"target\": 1}, {\"annotations\": {\"Species\": {\"6239\": true}, \"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true, \"Notch signaling subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2003 May 30;278(22):20117-23. Epub 2003 Mar 14.\", \"reference\": \"12639958\", \"type\": \"PubMed\"}, \"evidence\": \"PEN-2 is an integral membrane protein that is a necessary component of the gamma-secretase complex, which is central in the pathogenesis of Alzheimer's disease and is also required for Notch signaling. In the absence of PEN-2, Notch signaling fails to guide normal development in Caenorhabditis elegans, and amyloid beta peptide is not generated from the amyloid precursor protein\", \"hash\": \"f8650dd9bb1f4f953c49f3f769c9a09d6fd08d9b2e0c82cf46f780e26854867ca0c4ad85cd3aa80f4ebf29c3c851c389b84c6218024e95dad3829b6c184425bf\", \"key\": 0, \"line\": 12262, \"relation\": \"decreases\", \"source\": 69, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"820e3a9d5bf223ccd540b134ec1759bbed21069b2f241f5a429575f98bea45e828927b0409401afa46be5fbdb393b6d1de5a055f6a26b458f8aca3579ba0fe81\", \"key\": 0, \"line\": 3550, \"relation\": \"increases\", \"source\": 70, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"0baadf98e54af0b25c87d91c9ca545c3292d1af7452de0893409d6b150bf9e7bed05348db07665d53fcfe6a755095457630f8cfb1f00d2a782e997f31fa60b2e\", \"key\": 0, \"line\": 3553, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 70, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2008 Feb;5(1):15-25\", \"reference\": \"18288927\", \"type\": \"PubMed\"}, \"evidence\": \"it has been demonstrated that LRP1 might bind to APP independent of the KPI domain in APP. This APP - LRP1 interaction is facilitated through a trimeric complex of APP-FE65-LRP1, which has a functional role in APP processing. Along with LRP1, APP is transported from the early secretory compartments to the cell surface and subsequently internalised into the endosomal / lysosomal compartments. Recent investigations indicate that ApoER2 and SorLA fulfil a similar role in shifting APP localisation in the cell, which affects APP processing and the production of the APP derived amyloid beta-peptide (Abeta)\", \"hash\": \"b1a113e17c62ceb801bf59916e1ba4f8f069be43d5c1d0ae9b11205e750f558e73a7a9b5bcb7ebc41d80c61caa81539ad36621d92075d173f4660fa7e2148229\", \"key\": 0, \"line\": 35229, \"relation\": \"decreases\", \"source\": 71, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Liver\": true}, \"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2008 Feb;5(1):15-25\", \"reference\": \"18288927\", \"type\": \"PubMed\"}, \"evidence\": \"Following this export Abeta is degraded in the liver, where LRP1 potentially conducts the removal of Abeta from the blood stream\", \"hash\": \"00f3582d3839405b21cd0396b9a29bf2a98a3d63a8429434056d1590337dbb2cd3df1ee1a90edefa8aabf516cb412af6e5edc945cb5be1c9100b63f3141f6af7\", \"key\": 1, \"line\": 35236, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 71, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"Endosomal lysosomal subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2007 Apr 11;27(15):4052-60.\", \"reference\": \"17428983\", \"type\": \"PubMed\"}, \"evidence\": \" Abeta is a fragment of amyloid-beta precursor protein (APP) generated in neurons by two proteases, beta- and gamma-secretases. APP and beta-secretase, both present on cell surface, are endocytosed into endosomes to produce Abeta. The molecular mechanism by which neurons trigger the production of Abeta is poorly understood. We describe here evidence that the binding of lipid-carrying apolipoprotein E (ApoE) to receptor apolipoprotein E receptor 2 (ApoER2) triggers the endocytosis of APP, beta-secretase, and ApoER2 in neuroblastoma cells, leading to the production of Abeta. This mechanism, mediated by adaptor protein X11alpha or X11beta (X11alpha/beta), whose PTB (phosphotyrosine-binding) domain binds to APP and a newly recognized motif in the cytosolic domain of ApoER2. Isomorphic form ApoE4 triggers the production of more Abeta than by ApoE2 or ApoE3; thus, it may play a role in the genetic risk of ApoE4 for the sporadic AD.\", \"hash\": \"99d6ac8ac3ebce2b4fbf6477098bd665e6b9f15fabefe3e95d699ebe28b4af12f4fbbd244cf296b7d2c13966e03c6b2ef37980efcb0d5c2a6b11fb53e4b5263b\", \"key\": 0, \"line\": 4821, \"relation\": \"increases\", \"source\": 72, \"subject\": {\"effect\": {\"fromLoc\": {\"name\": \"Cell Surface Extensions\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Endosomes\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2000\", \"reference\": \"10931940\", \"type\": \"PubMed\"}, \"evidence\": \"These data suggest that BACE2 contributes to Abeta production in individuals bearing the Flemish mutation, and that selective inhibition of these highly similar proteases may be feasible and therapeutically advantageous.\", \"hash\": \"34b930b81a64341525bccfa0ed13c5d0f151a36146d2114748530f77147cfa6869d658fe7515684913975598fd94bf369c245cc44ecc2e5a3e17ee5c59f51fbc\", \"key\": 1, \"line\": 25598, \"relation\": \"increases\", \"source\": 72, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett2002\", \"reference\": \"12052539\", \"type\": \"PubMed\"}, \"evidence\": \"BACE2 cleaves the amyloid precursor protein at the beta-secretase site and is thought to contribute to amyloid beta protein production.\", \"hash\": \"ac96b306fac33a7c75b1e7d9d6bfdd1dd619cf04d7c46afc7c13eb877d4c6463a9a38648f07b132705dd8779bfbe3f9b21e1403350f053185a91f9561c24a914\", \"key\": 2, \"line\": 25660, \"relation\": \"increases\", \"source\": 72, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Proteome Res2004\", \"reference\": \"15473697\", \"type\": \"PubMed\"}, \"evidence\": \"Mapping to the Down syndrome critical region (chromosome 21) and identified as a homologue of BACE1, BACE2 also cleaves amyloid precursor protein at the beta-site.\", \"hash\": \"7d698aff1261d42055c492b6133d3f21f7fc45982254d863265615ae4d7628a9b4ef75c4a9fde19ae4824de459963c1f6b8a997f44668985f8ceb7a048220254\", \"key\": 3, \"line\": 26826, \"relation\": \"increases\", \"source\": 72, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Physiological levels of ABeta also have trophic and neuroprotective actions in trophic deprived conditions [95]. Many ABeta has a physiological role in normal synapse function. In organotypic hippocampal slices, BACE activity is increased by synaptic activity and the resulting ABeta peptides depress excitatory transmission through a-amino-3-hydroxyl-5-methyl-4-isoxazole-propionate (AMPA) and N-methyl-D-aspartic acid (NMDA) receptors, suggesting a role for ABeta in homeostatic plasticity [29]. ABeta may have an important physiological role in synapse elimination during brain development\", \"hash\": \"99f050b76fdf9f7dfea6374a987e7b48f6d2922c8078cef757102b37ce7966fc373042ed06b2e5eb5b808c4eccc8fc3f76de1ce244d3fe75294324d5ecc58937\", \"key\": 4, \"line\": 35590, \"relation\": \"increases\", \"source\": 72, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Insulin signal transduction\": true}}, \"citation\": {\"name\": \"Neural Plast2005\", \"reference\": \"16444902\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin and IGF-I may pmodulate brain levels of insulin degrading enzyme, which would also lead to an accumulation of Abeta amyloid.\", \"hash\": \"3a046e2901d286baa6de6e8de7877b88b1d87dc9a1f5bccd5a2216b0078bc9290a4a3ae54f7829e2e3da19f460fd46ec880dc32265fd58564539fe80cff6aa2f\", \"key\": 0, \"line\": 32118, \"relation\": \"increases\", \"source\": 73, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Nuclear factor Kappa beta subgraph\": true}}, \"citation\": {\"name\": \"Int J Neuropsychopharmacol. 2012 Feb;15(1):77-90\", \"reference\": \"21329555\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, NF-B p65 expression leads to up-regulated -secretase cleavage and Abeta production, while non-steroidal anti-inflammatory drugs (NSAIDs) inhibited BACE1 transcriptional activation induced by strong NF-B activator tumour necrosis factor-alpha (TNF-).\", \"hash\": \"8b4633a2d9e2cf0b4d5a74fb6767a5f84349bbb36ae6d7f1bcdc1c634090bcaf6e22bcfb8f78dc6d37a3765ddd0b8e5d424bc6f67bd3116b9196dcd678f240b6\", \"key\": 0, \"line\": 15271, \"relation\": \"increases\", \"source\": 74, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"hash\": \"89d55a34b4871d7b42a86d5afa5e04afc539a3b5de3569df1a0e3a5e5b548e1039420c23dd3f9cc4593ae74fd48d68baff61b572ba029150622fb34d5888d4b3\", \"key\": 0, \"line\": 2612, \"relation\": \"increases\", \"source\": 75, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Subgraph\": {\"Nitric oxide subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\", \"type\": \"PubMed\"}, \"evidence\": \"First, Pin1 inhibits the production of Abeta, and enhances the activity of eNOS. Second, Abeta and eNOS form a mutual inhibition system. Third, the well-balanced feedback signaling loop avoids the development of AD, HTN, and CAA by inhibiting the frequent pathological characteristics of these diseases, including Abeta deposition in cerebral microvessels and cerebral microbleeds.\", \"hash\": \"4c1bc5ca09a3ce8e55b81e9b356eb0d6f5079e5a5305431ad39606feceac6e9aa673aa2b4fa74cc0d424a79116748dfb49bdc3405e69ab97b8163addeaecb2a1\", \"key\": 1, \"line\": 17308, \"relation\": \"decreases\", \"source\": 75, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Subgraph\": {\"Nitric oxide subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Med Hypotheses. 2014 Feb;82(2):145-50\", \"reference\": \"24332564\", \"type\": \"PubMed\"}, \"evidence\": \"To explore the molecular mechanism underlying AD, HTN, and CAA, we hypothesize a feedback signaling loop consisted of Pin1, eNOS, and Abeta. Pin1 and eNOS mainly inhibit Abeta deposition in cerebral vessels, cerebral microbleeds, and elevation of blood pressure, preventing the development of AD, HTN, and CAA, however, Abeta plays an opposite role and aggravates these diseases.\", \"hash\": \"c6d5b26a1621b365753b8a99ba3700d19a3f38dfd0f9cd725f5d088b4c2190f62eaba24e37f0bc9938f6528738f0d726de278e124d395bf27ad657d9108bf0e1\", \"key\": 2, \"line\": 17342, \"relation\": \"decreases\", \"source\": 75, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 2;287(10):6969-73. Epub 2011 Dec 19.\", \"reference\": \"22184106\", \"type\": \"PubMed\"}, \"evidence\": \"Alzheimer disease (AD) is characterized by the presence of senile plaques of amyloid-Abeta (Abeta) peptides derived from amyloid precursor protein (APP) and neurofibrillary tangles made of hyperphosphorylated Tau. Increasing APP gene dosage or expression has been shown to cause familial early-onset AD. However, whether and how protein stability of APP is regulated is unclear. The prolyl isomerase Pin1 and glycogen synthase kinase-3Abeta (GSK3Abeta) have been shown to have the opposite effects on APP processing and Tau hyperphosphorylation, relevant to the pathogenesis of AD. However, nothing is known about their relationship. In this study, we found that Pin1 binds to the pT330-P motif in GSK3Abeta to inhibit its kinase activity. Furthermore, Pin1 promotes protein turnover of APP by inhibiting GSK3Abeta activity. A point mutation either at Thr-330, the Pin1-binding site in GSK3Abeta, or at Thr-668, the GSK3Abeta phosphorylation site in APP, abolished the regulation of GSK3Abeta activity, Thr-668 phosphorylation, and APP stability by Pin1, resulting in reduced non-amyloidogenic APP processing and increased APP levels. These results uncover a novel role of Pin1 in inhibiting GSK3Abeta kinase activity to reduce APP protein levels, providing a previously unrecognized mechanism by which Pin1 protects against Alzheimer disease.\", \"hash\": \"91941bb6ed550b6111ec107f178a450ac4ae6fd006dfece6fc506a08e8cb0834196c1d46f8a494e5dc55ea6251d061f5c8c2f688a0acb4c46a1378dc7e4edabf\", \"key\": 0, \"line\": 2610, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 75, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Calpastatin-calpain subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2010\", \"reference\": \"20595388\", \"type\": \"PubMed\"}, \"evidence\": \"We found that overexpression of endogenous calpain inhibitor calpastatin (CAST) under the control of the calcium/calpmodulin-dependent protein kinase II promoter in APP/PS1 mice caused a remarkable decrease of amyloid plaque burdens and prevented Tau phosphorylation and the loss of synapses.\", \"hash\": \"5f96521c42985b07e73ae896d55c4e5e542007a7be0f1c85de0cd695762b74a376975723997ec8f90dd936525bd8ea4fc81ca6aeb925f0efe0b9a589c532ba9d\", \"key\": 0, \"line\": 26387, \"relation\": \"decreases\", \"source\": 76, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Calpastatin-calpain subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2010\", \"reference\": \"20595388\", \"type\": \"PubMed\"}, \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"hash\": \"d58992eb3bc4777c977c3f61d206d8baf9db0ca29dfcf08784ff099d73f4c75efb890f2388b924679e685e1f1dc832b0f7e5c9888d5a864d7c39bba5cc07e82a\", \"key\": 1, \"line\": 26414, \"relation\": \"decreases\", \"source\": 76, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Calpastatin-calpain subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2010\", \"reference\": \"20595388\", \"type\": \"PubMed\"}, \"evidence\": \" Interestingly, treatment of cultured primary neurons with amyloid-beta (Abeta) peptides caused an increase in the level of beta-site APP-cleaving enzyme 1 (BACE1), the key enzyme responsible for APP processing and Abeta production. This effect was inhibited by CAST overexpression. \", \"hash\": \"d58992eb3bc4777c977c3f61d206d8baf9db0ca29dfcf08784ff099d73f4c75efb890f2388b924679e685e1f1dc832b0f7e5c9888d5a864d7c39bba5cc07e82a\", \"key\": 2, \"line\": 26416, \"relation\": \"decreases\", \"source\": 76, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Nuclear factor Kappa beta subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett2007\", \"reference\": \"17223266\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, we show that some NF-kappaB inhibitors decrease sAPPbeta and APP-CTFbeta suggesting that they reduce the beta-secretase cleavage of APP.\", \"hash\": \"2d94cdec98a416e9fd2b920691628d9233c6ce85f840bb7494f6bdc4b6ca2a497605d6af07ebdcfde63b37af3316b8db6bc3f109b60b60525d798327c70c9a6d\", \"key\": 0, \"line\": 27009, \"relation\": \"increases\", \"source\": 77, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Nuclear factor Kappa beta subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Jul 13;287(29):24573-84. Epub 2012 May 31.\", \"reference\": \"22654105\", \"type\": \"PubMed\"}, \"evidence\": \"Under physiological conditions, NFkappaB lowers the transcriptional activity of the promoters of betaAPP, beta-secretase (beta-site APP-cleaving enzyme 1, BACE1), and of the four protein components (Aph-1, Pen-2, nicastrin, presenilin-1, or presenilin-2) of the -secretase in HEK293 cells. This was accompanied by a reduction of both protein levels and enzymatic activities, thereby ultimately yielding lower amounts of Abeta and AICD (APP intracellular domain).\", \"hash\": \"230365f13d2404627c4e63bb10e0d9b990f75403bf522f1d2438bdbc7c3ed699d5b17ef0dac87701122a93a9d777ec04eabfce29e39d24e9c782a556f7f49e8e\", \"key\": 1, \"line\": 37164, \"relation\": \"decreases\", \"source\": 77, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"DKK1 subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2014 Jan;19(1):88-98. doi: 10.1038/mp.2012.163. Epub 2012 Nov 20.\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"Although the mechanism of Ab action in the pathogenesis of Alzheimers disease (AD) has remained elusive, it is known to increase the expression of the antagonist of canonical wnt signalling, Dickkopf-1 (Dkk1), whereas the silencing of Dkk1 blocks Ab neurotoxicity.\", \"hash\": \"7456feda5db4808e773327d229efb019998996f68e734001d457dcc8e2cfb167d5aab3cafb549afae81bce27c5adf4fcb5da9396064c0d6ea3daa0e38f63a9f7\", \"key\": 0, \"line\": 56064, \"relation\": \"decreases\", \"source\": 78, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"DKK1 subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2014 Jan;19(1):88-98. doi: 10.1038/mp.2012.163. Epub 2012 Nov 20.\", \"reference\": \"23164821\", \"type\": \"PubMed\"}, \"evidence\": \"Caricasole et al demonstrated that the A peptide fragment, A25-35, induces neuronal expression of the wnt antagonist Dkk1 and that silencing of DKK1 blocks A neurotoxicity.\", \"hash\": \"2f3b2bfb377450a13638b6a4c8a3e102187f4c542ae1eecca5876a81ba5025bad661f5cfb868b4512669f3206ee3f9399371dd1c401c5dd1e6326ffc9f0c98bd\", \"key\": 1, \"line\": 56077, \"relation\": \"decreases\", \"source\": 78, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"DKK1 subgraph\": true}}, \"citation\": {\"name\": \"FASEB J.\", \"reference\": \"25384422\", \"type\": \"PubMed\"}, \"evidence\": \"In this study, activation of the Wnt pathway by overexpression of the agonist Wnt3a or -catenin or by inhibition of glycogen kinase synthase-3 in N2a cells resulted in a reduction in A levels and in the activity and expression of BACE1 (-APP cleaving enzyme). Conversely, inhibition of the pathway by transfection of the antagonists secreted frizzled receptor protein-1 or dickkopf-1 produced the opposite effects.\", \"hash\": \"1cf50bcffc3b9c8056ac29fdcc000ee86aa6be1d6707da3e06c8cc815490b20d72ade13684ce69b953938fef819acf1a015af908409f35e3fe392cd57e5161a4\", \"key\": 2, \"line\": 56374, \"relation\": \"increases\", \"source\": 78, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"DKK1 subgraph\": true}}, \"citation\": {\"name\": \"Biomed Res Int.\", \"reference\": \"24883305\", \"type\": \"PubMed\"}, \"evidence\": \"More importantly, persistent activation of Wnt signaling through Wnt ligands, or inhibition of negative regulators of Wnt signaling, such as Dickkopf-1 (DKK-1) and glycogen synthase kinase-3  (GSK-3  ) that are hyperactive in the disease state, is able to protect against A  toxicity and ameliorate cognitive performance in AD.\", \"hash\": \"c60c88a64dd1e7ed47411f985089e8242bf1de965492518ac13064c6a91ffa090fb09c778322960d37f6aabe6ce790fbc8efc732dc888787c3f246a18263b699\", \"key\": 3, \"line\": 56422, \"relation\": \"decreases\", \"source\": 78, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Cholesterol metabolism subgraph\": true, \"Non-amyloidogenic subgraph\": true}, \"UserdefinedCellLine\": {\"Neuroblastoma cell\": true}}, \"citation\": {\"name\": \"Neuroscience. 2011 Sep 15;191:107-17. doi: 10.1016/j.neuroscience.2011.03.011. Epub 2011 Mar 22.\", \"reference\": \"21396986\", \"type\": \"PubMed\"}, \"evidence\": \"Seladin-1 was considered a novel neuroprotective factor, because of its anti-apoptotic activity. Subsequently, it was demonstrated that seladin-1 has also enzymatic activity [3--hydroxysterol delta-24-reductase, (DHCR24)], which catalyzes the synthesis of cholesterol from desmosterol. The amount of membrane cholesterol may play an important role both in protecting neuronal cells against toxic insults and in inhibiting the production of -amyloid. We demonstrated that seladin-1 overexpression increases the amount of membrane cholesterol and induces resistance against -amyloid aggregates in neuroblastoma cells, whereas a specific inhibitor of DHCR24 increased cell vulnerability. We also hypothesized that seladin-1 might be a mediator of the neuroprotective effects of estrogens. We first demonstrated that, in human fetal neuroepithelial cells (FNC), 17-estradiol, raloxifene, and tamoxifen exert protective effects against -amyloid toxicity and oxidative stress. In addition, these molecules significantly increased the expression of seladin-1 and the amount of cell cholesterol. Then, we showed that, upon seladin-1 silencing, the protective effects of estrogens were abolished, thus indicating this factor as a fundamental mediator of estrogen-mediated neuroprotection, at least in FNC cells. Furthermore, we detected the presence of functionally active half-palindromic estrogen responsive elements upstream the coding region of the seladin-1 gene. Overall, our results indicate that seladin-1 may be viewed as a multi-faceted protein, which conjugates both the neuroprotective properties of estrogens and the important functions of cholesterol in maintaining brain homeostasis\", \"hash\": \"6c98e05061c152cf9bcd5406af4e49893edf6cab2de8f966aba9d441b612d70e957b9361cef4dc259dbf4e603d2bc4f098f3c7587dc4ee75e6122ae816f4ce1c\", \"key\": 0, \"line\": 12383, \"relation\": \"decreases\", \"source\": 79, \"target\": 1}, {\"annotations\": {\"UserdefinedCellLine\": {\"Neuroblastoma cell\": true}}, \"citation\": {\"reference\": \"29183403\", \"type\": \"PubMed\"}, \"evidence\": \"Rs142787485 in RAB10 confers significant protection against AD (p value=0.0184, odds ratio=0.5853). Moreover, we replicated this association in an independent series of unrelated individuals (p value=0.028, odds ratio=0.69) and used a gene-based test to confirm a role for RAB10 variants in modifying AD risk (p value=0.002). Experimentally, we demonstrated that knockdown of RAB10 resulted in a significant decrease in A42 (p value=0.0003) and in the A42/A40 ratio (p value=0.0001) in neuroblastoma cells. We also found that RAB10 expression is significantly elevated in human AD brains (p value=0.04).\", \"hash\": \"6816f20ca87f9c02b1d93849b2987b205ec74c86b314a29f0f0285fa1e5c344135490dd13dd8f8d57f43674efd66a1e4c75c3e751016af16cd2eb878ccfefde5\", \"key\": 0, \"line\": 57432, \"relation\": \"increases\", \"source\": 80, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\", \"type\": \"PubMed\"}, \"evidence\": \"The role of miR-124 on the expression of Abeta-site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the Abeta-amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"hash\": \"948c4f0e50d29f70031189eb4d1d1f3540fa2a48fd9cb37a8ef81f525baf09acbabca83b2394890bda2b3124d7f8891b75862a5cfef5f8fa4c2d79a45cfb9be6\", \"key\": 0, \"line\": 3210, \"relation\": \"increases\", \"source\": 81, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Toxicol Lett. 2012 Feb 25;209(1):94-105. Epub 2011 Dec 8.\", \"reference\": \"22178568\", \"type\": \"PubMed\"}, \"evidence\": \"The role of miR-124 on the expression of -site APP cleaving enzyme 1 (BACE1), an important cleavager of amyloid precursor protein that plays a pivotal role in the -amyloid production, was studied in this paper using cellular models for Alzheimer' disease (AD) of cultured PC12 cell lines and primary cultured hippocampal neurons. The aim of the present study was to uncover novel potential miR-124 targets and shed light on its function in the cellular AD model. MiR-124 expression was steadily altered when its mimic and inhibitor were transfected in vitro. The results showed the expression of BACE1, one of the potential functional downstream targets of miR-124, was well correlated with cell death induced by Abeta neurotoxicity, and its expression level could be up- and down-regulated by suppression or over expression of miR-124 level respectively. These findings suggest that miR-124 may work as a basilic regulating factor to alleviate cell death in the process of AD by targeting BACE1, play an essential role in the control of BACE1 gene expression, and might be considered as a novel therapeutic target in treating AD.\", \"hash\": \"b7a7bd2b73d102d12f12555fd118df94d8b1abb1dd5bb58447e444f24a0eff14f32d63df0ac8687eae028b6b07f4b8dedc3895142158d41ce58f12d67a18e608\", \"key\": 1, \"line\": 9775, \"relation\": \"increases\", \"source\": 81, \"target\": 1}, {\"annotations\": {\"Encode_Feature_Types\": {\"Promoter\": true}}, \"citation\": {\"name\": \"PLoS One. 2014 Jul 22;9(7):e103067. doi: 10.1371/journal.pone.0103067. eCollection 2014.\", \"reference\": \"25051175\", \"type\": \"PubMed\"}, \"evidence\": \"Our results revealed that histone H3 acetylation in PS1 and BACE1 promoters is markedly increased in N2a/APPswe cells\", \"hash\": \"dfd925144677b127422a93d292e8672dc99e19092c68365598fdcf3397985a14ee774f78181bbfaadcbda33c5a5a47ba99c6c7e9dc9bba623073f0d85066711e\", \"key\": 2, \"line\": 44739, \"relation\": \"increases\", \"source\": 81, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol2000\", \"reference\": \"10850859\", \"type\": \"PubMed\"}, \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"hash\": \"b5ef61de3a6c8b6e72de0045cc262aae8c51ae8171065e46ad4eae988822aea5a452a86b70ee810c6acae247c18a1c44a5249c07e2f32a3792ee41dcef1f506e\", \"key\": 0, \"line\": 23580, \"relation\": \"increases\", \"source\": 82, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Neuropathol Exp Neurol. 2000 Jun;59(6):471-6.\", \"reference\": \"10850859\", \"type\": \"PubMed\"}, \"evidence\": \" IL-1 levels are elevated in Alzheimer brain, and overexpression of IL-1 is associated with beta-amyloid plaque progression.\", \"hash\": \"0a5d6ff121f76b87286619d4440cc5ebc55a5d5fa7537a959f082d905510876e8ec97038df76d479e7d91e24fe631d69a3051b774ee4d0d09fbd7e5764cdebb0\", \"key\": 1, \"line\": 23863, \"relation\": \"increases\", \"source\": 82, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Amyloidogenic subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2008 Mar;104(5):1387-93. Epub 2007 Nov 16.\", \"reference\": \"18021299\", \"type\": \"PubMed\"}, \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"hash\": \"0b1168aa59b2267b5991a90689cba92b0448f1cd0219fa2a131dc5307210f500d0b20cd46681dc3ea20f04848134350466749accc0a5f41a4e299839e773b685\", \"key\": 2, \"line\": 24071, \"relation\": \"decreases\", \"source\": 82, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Interleukin signaling subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem2008\", \"reference\": \"18021299\", \"type\": \"PubMed\"}, \"evidence\": \"we unexpectedly found that IL-1beta significantly enhanced alpha-cleavage, indicated by increases in sAPPalpha and C83, but reduced beta-cleavage, indicated by decreases in sAPPbeta and Abeta40/42, in human neuroblastoma SK-N-SH cells. IL-1beta did not significantly alter the mRNA levels of BACE1, ADAM-9, and ADAM-10, but up-regulated that of TACE by threefold.\", \"hash\": \"98f2c6587f067cb4befdc05882f756ce246055e2ae6061d0d4606b69d9e9fcf26fb1abfb0dae932709b117a1de3285af2fe1b9d7788aa0b72cd8ec7832a92096\", \"key\": 3, \"line\": 24584, \"relation\": \"decreases\", \"source\": 82, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Akt subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Nat Rev Neurol2010\", \"reference\": \"20139999\", \"type\": \"PubMed\"}, \"evidence\": \"We show that knock-down of ATXN1 significantly increases the levels of both Abeta40 and Abeta42. This effect could be rescued with concurrent overexpression of ATXN1. Moreover, overexpression of ATXN1 decreased Abeta levels. \", \"hash\": \"b36f5284fa7660d58034cea632714e624b7c50ec51fbf2e4f96bd5a4d089ed1990d645662cd23f57b1930e2c550025b075825fa3a68192d3175b897955f9642b\", \"key\": 0, \"line\": 27244, \"relation\": \"decreases\", \"source\": 83, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Sphingolipid metabolic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\", \"type\": \"PubMed\"}, \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"hash\": \"591773c6b6240de7e81becb57a9057d0eb5ee8d3390f9776f69a13ed5aa7a40f040760ebd3fc7bbb44b6696ad3f52757fc3b1e53227ca181eafaf5d719eb7706\", \"key\": 0, \"line\": 2733, \"relation\": \"increases\", \"source\": 84, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing A levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"4b324d20930d9954fd5cc0b807641ddf73ea4c86c9fc2a8d3fb1e31e64951782a6b51494385fe8ae04147a1faa0b769054459172e554a413d934dec334d1065e\", \"key\": 0, \"line\": 6453, \"relation\": \"decreases\", \"source\": 85, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"FEBS Lett2010\", \"reference\": \"20849853\", \"type\": \"PubMed\"}, \"evidence\": \"Our laboratory has previously shown that EGCG can increase non-amyloidogenic processing of APP through promotion of the beta-secretase ADAM10, which consequently reduced Abeta deposition and improved cognition in AD mice\", \"hash\": \"b3f3d9c76d83c962ff8844780dcd96003067fb2781ec024f7da58c8dee156cfcda73ac2447f454cd9c4ab75b84c840793cbebe1f77c34e2c4ef39749529bd3b3\", \"key\": 1, \"line\": 24459, \"relation\": \"decreases\", \"source\": 85, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2013 Aug 26;548:90-4. doi: 10.1016/j.neulet.2013.05.018. Epub 2013 May 15.\", \"reference\": \"23685131\", \"type\": \"PubMed\"}, \"evidence\": \"The -secretase complex comprises presenilins (PS1 or PS2), nicastrin, APH-1, and PEN-2. Herein, we find that PEN-2 can interact with ferritin light chain (FTL), an important component of the iron storage protein ferritin. In addition, we show that overexpression of FTL increases the protein levels of PEN-2 and PS1 amino-terminal fragment (NTF) and promotes -secretase activity for more production of Abeta and notch intracellular domain (NICD). Furthermore, iron treatments increase the levels of FTL, PEN-2 and PS1 NTF and promote -secretase-mediated NICD production. Moreover, downregulation of FTL decreases the levels of PEN-2 and PS1 NTF. Together, our results suggest that iron can increase -secretase activity through promoting the level of FTL that interacts with and stabilizes PEN-2, providing a new molecular link between iron, PEN-2/-secretase and Abeta generation in AD.\", \"hash\": \"90aebbb7cf8088e1ea7297a8c90e5a19a610009c0b9076db2ff6483efbbabedbbe08f96e0b491334cf8e38789fb0700cbcaae55b94489d5f19e05054c6d8c88e\", \"key\": 0, \"line\": 12284, \"relation\": \"increases\", \"source\": 86, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Brain Res Bull. 2012 Sep 1;88(6):596-601. Epub 2012 Jun 19.\", \"reference\": \"22721728\", \"type\": \"PubMed\"}, \"evidence\": \"Furthermore, overexpression of miR-195 in N2a/APP decreased the level of Abeta, while inhibition of miR-195 resulted in an increase of Abeta. Thus, we demonstrated that miR-195 could downregulate the level of Abeta by inhibiting the translation of BACE1. We conclude that miR-195 might provide a therapeutic strategy for AD\", \"hash\": \"bb8f9143fefd16b7ae739753142cc1653eda9d1f6963800112ff05b8e4d2e84a440e70531ac03a30eaa200d31f348f43b5daaf3422fc5d44e5931aae186a7233\", \"key\": 0, \"line\": 37325, \"relation\": \"directlyDecreases\", \"source\": 87, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry. 1999 Aug 31;38(35):11570-6\", \"reference\": \"10471309\", \"type\": \"PubMed\"}, \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"hash\": \"23e3ae62d72d43c2c828e1821ec7128fdcb2b1a98a35b5518b7f15780ba04bd1ae7f6728dc98e0a176b70174940f0376d20bb316a1d355451844d15e0b48a67a\", \"key\": 0, \"line\": 18973, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 88, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Cell\": {\"regular cardiac myocyte\": true}, \"Species\": {\"9606\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2003 May 23;278(21):19054-61\", \"reference\": \"12754271\", \"type\": \"PubMed\"}, \"evidence\": \"In the presence of plasminogen there was robust degradation of A beta that was added to the HCSM cells resulting in restoration of cell viability.\", \"hash\": \"448800fbd23af90233ee31f434efb3e36d2f5f7edc95cfed08f3c59dfe26977a19f5f9077991c9648db51c82835a52d7750523d008cea6cd9b31b137543034cc\", \"key\": 1, \"line\": 20662, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 88, \"target\": 1}, {\"annotations\": {\"MeSHDisease\": {\"Neurodegenerative Diseases\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Response DNA damage\": true}}, \"citation\": {\"name\": \"PLoS One. 2011;6(10):e26420\", \"reference\": \"22046282\", \"type\": \"PubMed\"}, \"evidence\": \"Activated gadd153 can generate oxidative damage and reactive oxygen species (ROS), increase -amyloid (Abeta) levels, disturb iron homeostasis and induce inflammation as well as cell death, which are all pathological hallmarks of AD.\", \"hash\": \"a5a9eca361a488c7f347c4dd9d662b997bb57f71b0956cabdbd89f9486bf831dd6987a2586d041eff166ef296ddfb1ebb6656c4d873527407dfc9e341748b913\", \"key\": 0, \"line\": 20164, \"relation\": \"increases\", \"source\": 89, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Axonal transport subgraph\": true, \"Calsyntenin subgraph\": true}}, \"citation\": {\"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\", \"type\": \"PubMed\"}, \"evidence\": \"Here, we show that siRNA-mediated loss of calsyntenin-1 in cultured neurons alters APP processing to increase production of Abeta. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased Abeta levels.\", \"hash\": \"d0e9d87da50e99a3d5546479f61794629712f8e9e6d9e48fa4294ad4d93663f5d45caed36f0002fe660e63fde1f937ffb6f42d27eec88f8139f3ac1476957516\", \"key\": 0, \"line\": 149, \"relation\": \"increases\", \"source\": 90, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Axons\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Calsyntenin subgraph\": true}}, \"citation\": {\"name\": \"Hum Mol Genet. 2012 Mar 27. [Epub ahead of print]\", \"reference\": \"22434822\", \"type\": \"PubMed\"}, \"evidence\": \"Understanding the mechanisms that control processing of the amyloid precursor protein (APP) to produce amyloid- (ABeta) peptide represents a key area of Alzheimer's disease research. We also show that calsyntenin-1 is reduced in Alzheimer's disease brains and that the extent of this reduction correlates with increased ABeta levels. Calsyntenin-1 is a ligand for kinesin-1 light chains and APP is transported through axons on kinesin-1 molecular motors.\", \"hash\": \"c77f12fc49ed0014196c87e0d65954db726c4f671dadb939b5d3327fc775b3c240c1038f644af74dad712b895466f82c4226df8d9048ae3c37d814359ad008d9\", \"key\": 1, \"line\": 35105, \"relation\": \"decreases\", \"source\": 90, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Confidence\": {\"High\": true}, \"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"PLoS One. 2012;7(1):e29755. Epub 2012 Jan 17\", \"reference\": \"22272245\", \"type\": \"PubMed\"}, \"evidence\": \"Differentiated neural cultures derived from KLC1-suppressed hESC contained neural rosettes but further differentiation revealed obvious morphological changes along with reduced levels of microtubule-associated neural proteins, including Tau and less secreted Abeta, supporting the previously established connection between KLC1, Tau and Abeta\", \"hash\": \"7e0fb43274bcf1896cdfebff1cf3aa797cd5944ac2b1f46f3c2e0e9d827125fbe3cab0dc473e5ba7e5fa900bffda9be793e2b3dc3699b22714c7ec40e39f1878\", \"key\": 0, \"line\": 1068, \"object\": {\"effect\": {\"fromLoc\": {\"name\": \"intracellular\", \"namespace\": \"GOCC\"}, \"toLoc\": {\"name\": \"extracellular space\", \"namespace\": \"GOCC\"}}, \"modifier\": \"Translocation\"}, \"relation\": \"increases\", \"source\": 91, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Axonal transport subgraph\": true, \"Innate immune system subgraph\": true}}, \"citation\": {\"name\": \"Bioessays. 2015 Feb;37(2):131-41. doi: 10.1002/bies.201400131. Epub 2014 Nov 13.\", \"reference\": \"25394182\", \"type\": \"PubMed\"}, \"evidence\": \"Using a novel, unbiased genetic screen, Morihara et al. identified kinesin light chain-1 splice variant E (KLC1vE) as a modifier of A accumulation...First, KLC1vE reduces APP transport, leading to A accumulation.\", \"hash\": \"a44cf22fb95cab804a09e9ad26bac8d9fd4ba07d0179aa8db71e27c541afcb215cb1ed52790482bb99adacc705a217c4e5fe8d9a65f6a94220ef28d1c4ed9899\", \"key\": 1, \"line\": 56681, \"relation\": \"increases\", \"source\": 91, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Gamma secretase subgraph\": true, \"Notch signaling subgraph\": true}}, \"citation\": {\"name\": \"Journal of Biological Chemistry2002\", \"reference\": \"12297508\", \"type\": \"PubMed\"}, \"evidence\": \"Similar to the loss of presenilin or nicastrin, the inactivation of endogenous mAPH-1 using small interfering RNAs results in the decrease of presenilin levels, accumulation of gamma-secretase substrates (APP carboxyl-terminal fragments), and reduction of gamma-secretase products (amyloid-beta peptides and the intracellular domains of APP and Notch). \", \"hash\": \"ad2a8e36ec45e23e3f6ff5280a8c8d4b58efddbb3227d7e08204808008e39b000af33843977c2ea4d92c6b0d19d29d2bd282f4616e1a708c56ffc5bfd26b2d6d\", \"key\": 0, \"line\": 32995, \"relation\": \"increases\", \"source\": 92, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"G-protein-mediated signaling\": true, \"Gamma secretase subgraph\": true}}, \"citation\": {\"name\": \"Nat Med. 2013 Jan;19(1):22-4. doi: 10.1038/nm.3053.\", \"reference\": \"23296004\", \"type\": \"PubMed\"}, \"evidence\": \" In this issue of Nature Medicine, Thathiah et al.4 now provide provocative evidence that the adaptor protein  mediates the Abeta-altering effects of these GPCRs by promoting Abeta generation. This newly uncovered function of -arrestin 2 suggests it could be targeted to decrease amyloid pathology in patients with Alzheimer's disease. Production of the amyloid- peptide in Alzheimer's disease by the -secretase complex can be regulated by certain G proteincoupled receptors. This regulation seems to be mediated by -arrestin-2, whose expression was found to be elevated in Alzheimer's disease brains.Recruitment of -arrestin 2 to a GPCR leads to interaction with the -secretase complex via the Aph-1 subunit. Other members of the complex include presenilin-1 (PS-1), nicastrin (Nct) and Pen-2. The complex then moves laterally into lipid rafts, where -secretase activation is enhanced. Internalization may also occur to localize -secretase to late endosomes, where its activation is also increased. Cleavage of APP by -secretase (BACE1) to release soluble APP (sAPPb) followed by -secretase produces Abeta and APP intracellular domain (AICD). Increased production and secretion of Abeta from cells can lead to extracellular Abeta aggregation in the form of plaques. Mutagenesis of GPR3 in regions of the protein that specifically interact with either G protein or -arrestin 2 further showed that -arrestin 2, not G protein, mediates the ability of GPR3 to increase Abeta levels.\", \"hash\": \"c538aa2233805f6a443537b0b6eb95c06c4896d0e6fb297c00ac8401a6454b735fb05809c716dc30eb83c5e6429a39a135e2cdc29fcf182b9692377239dc1ee0\", \"key\": 0, \"line\": 251, \"relation\": \"increases\", \"source\": 93, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Alpha 2 macroglobulin subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis2003\", \"reference\": \"14678766\", \"type\": \"PubMed\"}, \"evidence\": \"alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"hash\": \"6f96fcbe4bd0f4ad9a3c4dc822671b11e4c2569a24cfb2ca23e9db643626a5e99f72a6d66d9e08e9d869028bd343f8d8749c73e35063c3868f670c69bde903b5\", \"key\": 0, \"line\": 23530, \"relation\": \"decreases\", \"source\": 94, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Alpha 2 macroglobulin subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem1998\", \"reference\": \"9489740\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, we investigated whether alpha2-macroglobulin (alpha2M), a protein present in neuritic plaques and elevated in Alzheimer's disease brain, is a potential regulatory factor for A beta fibril formation\", \"hash\": \"4a0a37b5f7dc052c80d9d2eea7cc285c3dcd4fe1d4fd22f47ef780d664b36e24f8ad9f2493eaa3ac17738b2336952774c958c22a459a2941cb061f5007aa5feb\", \"key\": 1, \"line\": 23568, \"relation\": \"increases\", \"source\": 94, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Alpha 2 macroglobulin subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2003 Dec;14(3):504-12.\", \"reference\": \"14678766\", \"type\": \"PubMed\"}, \"evidence\": \"alpha(2)-Macroglobulin (alpha(2)M) is an abundant plasma/extracellular space protein implicated in clearance of amyloid beta (Abeta)\", \"hash\": \"d14ae730cffdcf390cedcbc6281534e5ccc8d55e65241f9e27b1fff0fc20aaefd4793eb816c40378131890b34465edb246303610bb310542921c047851854d51\", \"key\": 2, \"line\": 23830, \"relation\": \"decreases\", \"source\": 94, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Caspase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron. 2008 Oct 23;60(2):247-57\", \"reference\": \"18957217\", \"type\": \"PubMed\"}, \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"hash\": \"bf35cb8a2234d76e44d4e77f6e152459ecf941e0dacb4e85a9fbb204e5357f6b79647b09dc2c9482594adcc857fff8944ec5ed63956b61223b905c9ad9437079\", \"key\": 0, \"line\": 37257, \"relation\": \"decreases\", \"source\": 95, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Neuroprotection subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2009 Feb;14(2):206-22. Epub 2008 Sep 23\", \"reference\": \"18813209\", \"type\": \"PubMed\"}, \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"hash\": \"b1da971d962ef44c1e4591cd080bbf407dc588bc0f40ba77c86505746c02c994f3c5186836edafd87e41b19379534e8bbe205de68ad9732b28c95d3c2b1ca814\", \"key\": 0, \"line\": 743, \"relation\": \"decreases\", \"source\": 96, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Neuroprotection subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Mol Psychiatry. 2009 Feb;14(2):206-22. Epub 2008 Sep 23\", \"reference\": \"18813209\", \"type\": \"PubMed\"}, \"evidence\": \"Humanin (HN) is a short neuroprotective peptide abolishing Abeta neurotoxicity.\", \"hash\": \"9551568466f54b9a23fef549c3e8433995178b089e0247e9327ae53eb0ffb07e23ce69f34adab4942d574c02b5967b11c3ef75590d1ef31849317420ddb831e1\", \"key\": 1, \"line\": 37173, \"relation\": \"decreases\", \"source\": 96, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"CRH subgraph\": true}}, \"citation\": {\"name\": \"EMBO J. 2015 Jun 12;34(12):1674-86. doi: 10.15252/embj.201488795.\", \"reference\": \"25964433\", \"type\": \"PubMed\"}, \"evidence\": \"In cells, CRF treatment increases Abeta production and triggers CRF receptor 1 (CRFR1) and -secretase internalization. Co-immunoprecipitation studies establish that -secretase associates with CRFR1; this is mediated by -arrestin binding motifs. \", \"hash\": \"b25a4e73022f0e38a93e5d5effc37580a925ea1a63c55b8b91e1fdf745c9dbfc968df009ce8c889daf125292dc80efb2b215a40ac070997533afe576d8cd4d78\", \"key\": 0, \"line\": 55915, \"relation\": \"increases\", \"source\": 97, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Autophagy signaling subgraph\": true, \"Chaperone subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2011 Aug 17;31(33):11992-2000. doi: 10.1523/JNEUROSCI.0945-11.2011.\", \"reference\": \"21849559\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta deposits in AD, parenchymal as well as (cap)CAA and dyshoric angiopathy, are associated with a local inflammatory reaction, including activation of microglial cells and astrocytes that, among others, produce cytokines and reactive oxygen species. This neuroinflammatory reaction may account for at least part of the cognitive decline. In previous studies we observed that small heat shock proteins (sHsps) are associated with Abeta deposits in AD. In this study the molecular chaperones Hsp20, HspB8 and HspB2B3 were found to colocalize with CAA and capCAA in AD brains. In addition, Hsp20, HspB8 and HspB2B3 colocalized with intercellular adhesion molecule 1 (ICAM-1) in capCAA-associated dyshoric angiopathy. Furthermore, we demonstrated that Hsp20, HspB8 and HspB2B3 induced production of interleukin 8, soluble ICAM-1 and monocyte chemoattractant protein 1 by human leptomeningeal smooth muscle cells and human brain astrocytes in vitro and that Hsp27 inhibited production of transforming growth factor beta 1 and CD40 ligand. Our results suggest a central role for sHsps in the neuroinflammatory reaction in AD and CAA and thus in contributing to cognitive decline.\", \"hash\": \"4bf6507996522722f62518ab84c82658d5f29a54a324c40756d114f7df57afe816760fec43d5f1c5cbd857bcc6588836d31444d96776ea82b996a3d4f46613a7\", \"key\": 0, \"line\": 38828, \"relation\": \"increases\", \"source\": 98, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of Biological Chemistry2001\", \"reference\": \"11517218\", \"type\": \"PubMed\"}, \"evidence\": \"We report here that (i) a single amino acid mutation at the Thr-668 residue of APP695, located 14 amino acids toward the amino-terminal end from the (682)YENPTY(687) motif, reduced the interaction between members of the Fe65 family of proteins and APP, whereas interaction of APP with the phosphotyrosine interaction domain of other APP binders such as X11-like and mammalian disabled-1 was not influenced by this mutation; (ii) the phosphorylation of APP at Thr-668 diminished the interaction of APP with Fe65 by causing a conformational change in the cytoplasmic domain that contains the Fe65-binding motif, YENPTY; and (iii) the expression of Fe65 slightly suppressed maturation of APP and decreased production of beta-amyloid (Abeta).\", \"hash\": \"f8c0e1616fb3c3adc0564734b9f537511ee48a0b3d33954eb103396ff5b54b5d92025dbb65937ea61eef562f64a674a4593c956edef3456e823002b1ace6b2ee\", \"key\": 0, \"line\": 32195, \"relation\": \"decreases\", \"source\": 99, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686 \", \"type\": \"PubMed\"}, \"evidence\": \"On the contrary, X11 stabilizes ABetaPP conformation in membrane, inhibiting ABeta secretion in cultured cells [45], likely impairing ABetaPP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptotic process. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABetaPP. The expression of JIP1b stabilizes immature ABetaPP and decreases the ABetaPP ectodomain, ABeta40/42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"hash\": \"7625bc7d99a6c5a4943eb9b4020cac08182d283c61789d83426ee4d296127fbcf54c3ee62b2742b56608f797c385e635d7ad75a8cc2fe956a30114b612b9f45d\", \"key\": 0, \"line\": 37044, \"relation\": \"decreases\", \"source\": 100, \"target\": 1}, {\"citation\": {\"name\": \"Neuroreport. 2012 Feb 15;23(3):146-51. doi: 10.1097/WNR.0b013e32834f1934.\", \"reference\": \"22222501\", \"type\": \"PubMed\"}, \"evidence\": \" Overexpression of neuronal adaptor protein X11 has been shown to decrease the production of amyloid-\", \"hash\": \"c8347c691892c345373cb20a6645cc8f0b93f2ebaeed397d8ceb58ce35daa48f49111d2379c50e36b2879a750fe8140ed568ffca8cebd37af0dfd9edc050998c\", \"key\": 1, \"line\": 44199, \"relation\": \"decreases\", \"source\": 100, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Gamma secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686 \", \"type\": \"PubMed\"}, \"evidence\": \"On the contrary, X11 stabilizes ABetaPP conformation in membrane, inhibiting ABeta secretion in cultured cells [45], likely impairing ABetaPP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptotic process. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABetaPP. The expression of JIP1b stabilizes immature ABetaPP and decreases the ABetaPP ectodomain, ABeta40/42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"hash\": \"143707704837bab2602c506cc5eed4fb9348459d7ee0c7f504afbbf26cd178132cc0c2ced37cb3e7bcc83d33630b7205b23d73cda3a721d1721ec10076157e65\", \"key\": 0, \"line\": 37046, \"relation\": \"decreases\", \"source\": 101, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"Ann N Y Acad Sci2006\", \"reference\": \"17185504\", \"type\": \"PubMed\"}, \"evidence\": \"Our in vivo studies using transgenic mice have shown that overexpression of LRP in central nervous system (CNS) neurons increases soluble brain Abeta and this increase correlates with deficits in memory.\", \"hash\": \"df729c7317cc3b8c80345613868a619430e995d59f8dac9c6b361285f95cfef9e6b9a19a2ba830a4e4dabb89deb10b4a576fcf31beec6502b4ab034a12becafb\", \"key\": 0, \"line\": 25454, \"relation\": \"increases\", \"source\": 102, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron2009\", \"reference\": \"20005821\", \"type\": \"PubMed\"}, \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"hash\": \"113829b0ba6ba341a3e6fe44b2d500477fe0fc370eb03629d83a33b90a33eea2c7b9d76987a631328fa8122a5f55818283fe4611099091344e8802f5f0c1176d\", \"key\": 1, \"line\": 32737, \"relation\": \"decreases\", \"source\": 102, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Low density lipoprotein subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron2009\", \"reference\": \"20005821\", \"type\": \"PubMed\"}, \"evidence\": \"Furthermore, LDLR overexpression dramatically reduced A beta aggregation and enhanced A beta clearance from the brain extracellular space. \", \"hash\": \"5de3e2b0538cd5bede9e360dfc584c3fb87f44f09867ba4c10abb2ceedd1dd3ecb6d63403cee1740669509220ecbc7b677c2539f4c93d5ca4822323efec642c3\", \"key\": 2, \"line\": 32738, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 102, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Endosomal lysosomal subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"3bbac5790b82165862f4809f8035306c15daca66ed1e49fe9100333cac4cf5d5e5bee6d648318d6f622d5584fce6ba26a518ca82ebabdf6aea36b3e1a9c43ab2\", \"key\": 3, \"line\": 35327, \"object\": {\"effect\": {\"fromLoc\": {\"name\": \"COP-Coated Vesicles\", \"namespace\": \"MESHCS\"}, \"toLoc\": {\"name\": \"Lysosomes\", \"namespace\": \"MESHCS\"}}, \"modifier\": \"Translocation\"}, \"relation\": \"increases\", \"source\": 102, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"705d19da05c70e00b896a0e8f94e45a97c71d94dc845decc291d23cd15f23c486ac554812d3beaf08b826cfc39c50f7a8028d143379fcde3c96533a1ec81a1d0\", \"key\": 0, \"line\": 35542, \"relation\": \"increases\", \"source\": 103, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"e74bcda1296c0bca03bc75bde3f58b3c016a40ece1392b0185bfa5f5464c6c7e44a5ad9de4efece52abf8a621e983436d4deea4b0747b450f2df8f213af9ae64\", \"key\": 1, \"line\": 35569, \"relation\": \"increases\", \"source\": 103, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"45ad85a6bac9e29f5af87f31a6ce5df6c1359e8787148c64d066708052d2e9458533157a4cf8a164aeca8d2921be09f741b2f50fc6fcba244013cbd8a7afa7e0\", \"key\": 0, \"line\": 35541, \"relation\": \"increases\", \"source\": 104, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"2db6dd8e70d8a17f5e82ad310ba3b696586c0ce498412b40f1390b9b0f7a1561c3274a1162bb66c48e91441e14dd3e99dccea6e10db1890986510619c747840a\", \"key\": 1, \"line\": 35568, \"relation\": \"increases\", \"source\": 104, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Sphingolipid metabolic subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2011 Oct 12;31(41):14820-30.\", \"reference\": \"21994399\", \"type\": \"PubMed\"}, \"evidence\": \"SPT is the first rate limiting enzyme in the de novo ceramide synthesis pathway (Hannun and Obeid, 2008). Activation of SPT elevates ceramide levels (Perry et al., 2000) and inhibition of SPT decreases ceramide levels (Hojjati et al., 2005; Patil et al., 2007) and neuronal cell death by Abeta (Cutler et al., 2004), supporting SPT as an important regulator of ceramide. SPT is a heterodimer composed of serine palmitoyltransferase long chain 1 (SPTLC1) and either serine palmitoyltransferase long chain 2 (SPTLC2) or serine palmitoyltransferase long chain 3 (SPTLC3) (Rotthier et al., 2010). In the brain, SPTLC3 is lowly expressed while SPTLC1 and SPTLC2 are the major subunits (Hornemann et al., 2006). However, the regulation of these subunits and in turn SPT is not well understood. Cell culture studies demonstrate that SPT activity increases in response to various stimuli (i.e. etoposide or resveratrol), but without concomitant changes in SPTLC1 and SPTLC2 mRNA levels, which have led researchers to hypothesize that SPT may be post-transcriptionally regulated.\", \"hash\": \"d8532a8f78729cc8aa89288c8533c08e269e1269c1b5f4b5ae633a2dfb3a7346020d1118fd9b9df7aa0eceedfcc5ead1730bb69942e29d944aabc32994571c8b\", \"key\": 0, \"line\": 2732, \"relation\": \"increases\", \"source\": 105, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Cell\": {\"microglial cell\": true}, \"Condition\": {\"Normal Healthy State\": true}, \"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 9;287(11):8424-33. Epub 2012 Jan 20\", \"reference\": \"22267734\", \"type\": \"PubMed\"}, \"evidence\": \"Ex vivo studies show that the levels of BACE1 are tightly regulated by the ATases. Specifically, up-regulation of ATase1 and ATase2 increases the levels of BACE1 and the generation of Abeta while siRNA-mediated down-regulation of either transferase achieves the opposite effects.\", \"hash\": \"5b65d1f760e7cce23e017776b9b1e024609f759dc96e5ce351428402f71de4bf0d98e96c0367bcd3c11210afa209518f81ec2a111ac3869c0457d0c197dc5ab8\", \"key\": 0, \"line\": 3060, \"relation\": \"increases\", \"source\": 106, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron2007\", \"reference\": \"17553422\", \"type\": \"PubMed\"}, \"evidence\": \"RNAi silencing of GGA3 also elevated levels of BACE and Abeta.\", \"hash\": \"7deb58bd004a2e7f646ba97513c3d1b01a381551c0b906873997268d8a27711cbe3a36374b8b9223cac246148d1224ed5c09952d2f412cfbc2b20d47d50681d6\", \"key\": 0, \"line\": 27053, \"relation\": \"decreases\", \"source\": 107, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular overexpression of PrP(C) inhibited the beta-secretase cleavage of APP and reduced Abeta formation. \", \"hash\": \"50f7938045c6d4dde51e40c7c9824711edc3b32c770685ba5036121fad02acf9a2fcc0fc0dd153963814675817209d98f7c1018986a1cc4369ddd4492c488f56\", \"key\": 0, \"line\": 25930, \"relation\": \"decreases\", \"source\": 108, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Expert Opin Ther Targets2008\", \"reference\": \"18479216\", \"type\": \"PubMed\"}, \"evidence\": \"In our opinion, albeit based on limited available data, a future potential therapeutic strategy is to mimic the mechanism by which the normal cellular form of the prion protein inhibits the beta-secretase beta-site amyloid precursor protein cleaving enzyme-1 (BACE1), and hence the production of Abeta.\", \"hash\": \"04d505615a68c5bbd0ceef79cca00d21b377bc9e5216917272a0457543f0c1b22c10096cdddae44ac4c8d18d9f6808469d5fc73fcc534db2cef9e56404a47aed\", \"key\": 1, \"line\": 27124, \"relation\": \"decreases\", \"source\": 108, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17573534\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular prion protein regulates beta-secretase cleavage of the Alzheimer's amyloid precursor protein.\", \"hash\": \"71d10c50e1181319a9311fa787182a94f8303eccf32e345287f323c50d1c60fed9abd1ca04a48a756ce9de8cc0f128a6f44f0c5c12e3170bf02b132b3c71c459\", \"key\": 2, \"line\": 30856, \"relation\": \"decreases\", \"source\": 108, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"PLoS One2013\", \"reference\": \"23577068\", \"type\": \"PubMed\"}, \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"hash\": \"e4fd557082c2dcf8a4f322c44e24a313b6d6cffc27545dd564bab12ace784f18bbe4692d68f845f66392a3bedd783b665d0b37a356b203896e39064d29998d91\", \"key\": 3, \"line\": 30887, \"relation\": \"decreases\", \"source\": 108, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Beta secretase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"PLoS One2013\", \"reference\": \"23577068\", \"type\": \"PubMed\"}, \"evidence\": \"PrP(C) decreases amyloid-beta (Abeta) production, which is involved in AD pathogenesis, by inhibiting beta-secretase (BACE1) activity.\", \"hash\": \"e4fd557082c2dcf8a4f322c44e24a313b6d6cffc27545dd564bab12ace784f18bbe4692d68f845f66392a3bedd783b665d0b37a356b203896e39064d29998d91\", \"key\": 4, \"line\": 30888, \"relation\": \"decreases\", \"source\": 108, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"f69f65e81bc8b21fc0296ffedcf979417ae8bfdb710980858bcd83efa91fff5380915656698ec0e4f6da9b3dc2002c5538cc8c4df81fa4b369dcf512bbdcfe24\", \"key\": 0, \"line\": 35946, \"relation\": \"increases\", \"source\": 109, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"ATP binding cassette transport subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2010 Nov;7(7):566-77.\", \"reference\": \"20704561\", \"type\": \"PubMed\"}, \"evidence\": \" This report identifies ABCA2 as a key regulator of endogenous APP expression and processing and suggests a possible biochemical mechanism linking ABCA2 expression, APP processing and Alzheimer's disease.\", \"hash\": \"3651e2a8099e730b48b31e923bb5a463c73b4f0fd72680dcb0e78cf6b9236346e3b0149eb0a0280fa77683324486832d315186b1faf5b559058de024d21050e8\", \"key\": 0, \"line\": 4284, \"relation\": \"increases\", \"source\": 110, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"TGF-Beta subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"38a46c9d84749117fd9bd2a26f22ad9d1ea28c62a90256f48e3b578ee161e373d672df772def64da7c9bc7094b421d89282617c50b583ca7c64ea63d2402962e\", \"key\": 0, \"line\": 38135, \"relation\": \"increases\", \"source\": 111, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Phosphatidylinositol 3 subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Bull. 2010 Aug;26(4):338-44.\", \"reference\": \"20651816\", \"type\": \"PubMed\"}, \"evidence\": \"It is believed that amyloid-beta peptide (Abeta) plays a central role in the pathogenesis of Alzheimer's disease (AD). Thus, the process of amyloid precursor protein (APP) cleavage is a key event and has raised much attention in the field of AD research. It is proposed that APP, beta- and gamma-secretases are all located on the lipid raft, and the meeting of them is an indispensable step for Abeta generation. Endocytosis can lead to clustering of APP, beta- and gamma-secretases from separate smaller lipid rafts into a larger one. On the other hand, for myristoylated alanine-rich C kinase substrate (MARCKS), phosphorylation by protein kinase C (PKC) or interaction with Ca(2+) can lead to its release from membrane into cytoplasm. This process induces the release of actins and phosphatidylinositol 4, 5-bisphosphate (PIP2), which are important factors for endocytosis. Thus, the present review proposes that MARCKS may be implicated in Abeta generation, by modulating free PIP2 level and actin movement, causing endocytosis\", \"hash\": \"5cf26b6ad8764220b1a79c46d302abe0fa2f89977ef74f03d5963189d8b42283a881292814918a9f01c8004c296e13bafbc6caf26c8ca0ebccb981b26455024a\", \"key\": 0, \"line\": 4354, \"relation\": \"increases\", \"source\": 112, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"6cc861c5f4042db0c10240611e10fe89e48ecd7bb11cfa187c6540be7040ea207b3522232d586933c3f6a4c9b6d2321bd3b46bac972410f5e4fd87159c1b5956\", \"key\": 0, \"line\": 35543, \"relation\": \"increases\", \"source\": 113, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"20520e18c30ea26f2ccdb81db796619ca32f305e3c94f8819606463f5e86ab2ff18e646f2785a24ce71465a31058afb7173b52105e473406636c9cc9aa40bbf3\", \"key\": 1, \"line\": 35570, \"relation\": \"increases\", \"source\": 113, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"275d0ec01a456ec7b84b2dfc4cbd5c63242555d8ca59594ef96587072774dc08f467892bb5c5c76dc230bd60282a74a774a12cbba522abeb3bb7443ab970d3ce\", \"key\": 0, \"line\": 35544, \"relation\": \"increases\", \"source\": 114, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"4d70f31d9c1dfd4db6f1650a83e0b13c4c6e1add76c8857c11b8c394c23f936b26e2ffa4349aa70e279ee608b375adbff5d9da0b8e4cca26d48c9d181aac019a\", \"key\": 1, \"line\": 35571, \"relation\": \"increases\", \"source\": 114, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"In animal models as well as in humans, the activation of muscarinic M1 acetylcholine receptors increases a-secretase cleavage of APP and consequently reduces ABeta levels [63, 64] whereas activation of NMDARs decreases a-secretase cleavage, consequently increasing ABeta levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"8978ccf10f01ead6c128cf4230f6a8334111ca750abdcd1213476e9ac4baffc756f10db0d27019826bebe2999980dce155f1bb4bcebd10159a18fa64b1916c87\", \"key\": 0, \"line\": 35546, \"relation\": \"increases\", \"source\": 115, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glutamatergic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The lower concentrations of aged ABeta42used by Puzzo et al.[83] are close to those found in the normal brain and shown to enhance LTP and memory. Increase in synaptic activity will increase ABeta production while lowering synaptic activity minimizes ABeta production. Similarly specific stimulation of NMDA receptors promotes ABeta production and ABeta in turn depresses synaptic activity. Thus indirectly ABeta also plays a role in suppressing excessive glutamate release. Interestingly, ABeta may play an important role during synapse elimination and stimulatation of neurogenesis in the hippocampus during brain development [93]. These studies provide convincing evidence to show that physiological levels of ABeta have a role in controlling synaptic activity.\", \"hash\": \"3d5bd1f6785a0053a76f17a40b2c42de37f5d8df728c00a6c58f331795311fef97eb95bfa9199e440560467e3344f02736a6a0721834ee3fd948b6ec017fd084\", \"key\": 1, \"line\": 35573, \"relation\": \"increases\", \"source\": 115, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Chemokine signaling subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2011 Oct;179(4):2016-27. Epub 2011 Aug 26.\", \"reference\": \"21872563\", \"type\": \"PubMed\"}, \"evidence\": \" We observed that the neuronal cells primarily produce IL-34 and that microglia express its receptor, colony-stimulating factor 1 receptor. IL-34 promoted microglial proliferation and clearance of soluble oligomeric amyloid- (oAbeta), which mediates synaptic dysfunction and neuronal damage in AD.\", \"hash\": \"0d755ca69ac15a344de5beafcae06bdb83c4604ba64207f471d823dca2e775548d4bdd026d63c1f29ee6cdfabd64d3ebeeff943838ac9cd8fdcb90e972e743ab\", \"key\": 0, \"line\": 4984, \"relation\": \"decreases\", \"source\": 116, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Chaperone subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Hum Mol Genet. 2010 Sep 1;19(17):3440-56. Epub 2010 Jun 22\", \"reference\": \"20570967\", \"type\": \"PubMed\"}, \"evidence\": \"The pharmacological blockage of autophagy resulted in a dramatic increase of mutant SOD1 aggregates. Immunoprecipitation studies, performed during autophagic flux blockage, demonstrated that mutant SOD1 interacts with the HspB8/Bag3/Hsc70/CHIP multiheteromeric complex, known to selectively activate autophagic removal of misfolded proteins. \", \"hash\": \"e558f4dca724148f73e6f93b956b79b5ef102ec1be57d4ee925dd755a5edec67d0c73ac1b5c93e80d26405f843633ca0b53d00947f74cd7c0e4fa80348b16191\", \"key\": 0, \"line\": 37304, \"relation\": \"increases\", \"source\": 117, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing A levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"1d588c891a2c1f29f542d659af46063bc420a7d1526a629933a050446c1d4657ed6937e1104edd74169820451b084e20da6f7694e14e6ec32d365973430841c1\", \"key\": 0, \"line\": 6452, \"relation\": \"decreases\", \"source\": 118, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"ADAM Metallopeptidase subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Activation of NMDARs decreases a-secretase cleavage, consequently increasing A levels [65]. Stimulation with muscarinic agonists or activators of protein kinase C (PKC), such as phorbol esters causes the up-regulation of the a-secretase cleavage of APP\", \"hash\": \"8d7e2b9ba7a41f1f4d6c864a0e1fddd7fc53b0cbf48f18cb4dac8787607821fd9d444c73c9921a50a81e04b63120a7dab61cc77e44c180857732377c55003de9\", \"key\": 0, \"line\": 6454, \"relation\": \"decreases\", \"source\": 119, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis2011\", \"reference\": \"21613740\", \"type\": \"PubMed\"}, \"evidence\": \"Cathepsin B cleaves the wild-type beta-secretase site sequence in AbetaPP to produce Abeta, and cathepsin B inhibitors administered to animal pmodels expressing AbetaPP containing the wild-type beta-secretase site sequence reduce brain Abeta in a manner consistent with beta-secretase inhibition. \", \"hash\": \"5d2b654c8fc2abb634250cd69fdce61ce7f281313ef48823658a553eec9e89dc132ef0d9142b5261270804429f8daee76af446d814f6b81dc99a5086ba183ca7\", \"key\": 0, \"line\": 29123, \"relation\": \"increases\", \"source\": 120, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Caspase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neuron. 2008 Oct 23;60(2):247-57\", \"reference\": \"18957217\", \"type\": \"PubMed\"}, \"evidence\": \"Impaired degradation of amyloid beta (Abeta) peptides could lead to Abeta accumulation, an early trigger of Alzheimer's disease (AD). How Abeta-degrading enzymes are regulated remains largely unknown. Cystatin C (CysC, CST3) is an endogenous inhibitor of cysteine proteases, including cathepsin B (CatB), a recently discovered Abeta-degrading enzyme\", \"hash\": \"c33fbece7420591e6ec10d5dfb612d33d715f4873c12e61166e7b3d6de9ba166e790608283a5364751e05c0b7c297f1749d34dce906e32f3beb0bca282777c4d\", \"key\": 1, \"line\": 37259, \"relation\": \"decreases\", \"source\": 120, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Acta Biochim Biophys Sin (Shanghai). 2009 Jun;41(6):437-45.\", \"reference\": \"19499146\", \"type\": \"PubMed\"}, \"evidence\": \"Lysosomal dysfunction may be the earliest histological change in AD. Amyloid plaques are full of active lysosomal hydrolases, implying that plaques may originate from lysosomal rupture. Cathepsins D and E (intracellular aspartyl proteases) are considered to influence Abeta peptides generation within the endosomallysosomal pathway because they exhibit - and -secretase like-activity [32]. For this reason, the endosomallysosomal pathway is a site for cleavage of the APP into smaller -amyloid-containing peptides, which are then degraded by cathepsins. Inhibition of cathepsins activity causes a rapid and pronounced build-up of potentially amyloidogenic protein fragments [33]. On the other hand, a failure to degrade aggregated Abeta_42 in late endosomes or secondary lysosomes was a mechanism that contributed to intracellular accumulation of Abeta in AD. The cysteine protease cathepsin B in lysosomes degrades A peptides, especially the aggregation-prone species Abeta_42.\", \"hash\": \"562d9fb26dd502241e9f7379a592452c860fe2aaca837b02248abe4da35accb0d9a006fa84ec7b8ef43797640176a5cfab5532c0bff0afbada038955df559495\", \"key\": 0, \"line\": 6880, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 121, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Glucagon subgraph\": true, \"Insulin signal transduction\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"Neuropharmacology. 2013 Sep;72:291-300. doi: 10.1016/j.neuropharm.2013.04.008. Epub 2013 Apr 17.\", \"reference\": \"23603201\", \"type\": \"PubMed\"}, \"evidence\": \"Pharmacological agents, such as dipeptidyl peptidase-4 (DPP-4) inhibitors, which increase the level of glucagon-like peptide-1 (GLP-1) and ameliorate T2D, have become valuable candidates as disease modifying agents in the treatment of AD. In addition, endogenous GLP-1 levels decrease amyloid beta (Abeta) peptide and tau phosphorylation in AD\", \"hash\": \"510267ccbbf5b14a47ae267afb89fc2ecb781ae698cb0c9da53d2bbd61bfadd2b5ee1f6c8cb02fbc0d47eb93e37d5eab8e19eb8a18c9acb76286cc71fcdb2564\", \"key\": 0, \"line\": 12197, \"relation\": \"decreases\", \"source\": 122, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Endothelin subgraph\": true, \"Insulin signal transduction\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\", \"type\": \"PubMed\"}, \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) Enzy matic degradation by activated microglia or by insulin de grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptor related protein (LRP) either directly or after binding to apol ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"hash\": \"c5b8ead416d0d6aa5ad962772b34281c7bac1646b98e4693285cfd9f8816f4040e836f3601b55d3f82991dd10f0b1c711bb104037fd0961e92822cd52b849af0\", \"key\": 0, \"line\": 8738, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 123, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\", \"type\": \"PubMed\"}, \"evidence\": \"The amyloid hypothesis of AD suggests that -amyloid accumulation is the critical event in development of disease (I 49, I 50]. Lots of research has been done on the formation and accumulation of Al3, however, in the last years the mechanism' of amyloid clearance came into focus. For Al3 clearance several mechanisms are known (Fig. 2): i) Enzy matic degradation by activated microglia or by insulin de grading enzyme (IDE), neprilysin, endothelin converting enzyme (ECE), and angiotensin converting enzyme (ACE); ii) Receptor-mediated transport across the blood brain barrier (BBB) by binding to the low-density lipoprotein receptor related protein (LRP) either directly or after binding to apol ipoprotein E (ApoE) and/or a2-macroglobulin (a2M) to be delivered to peripheral sites of degradation, e.g., liver and kidney. (Review in [151 ]). Concerning insulin resistance it has been shown that IDE expression is stimulated by the IR/lGF-1 R cascade (152]. \", \"hash\": \"5a5d3c0667c86b495ee0015d7f9b72d6d1ddd2ef1a287552944b96051c56c0cee74aef0bb810b4768785b667b59bcf07f83eddea628df47c80fe69472c75f3a8\", \"key\": 0, \"line\": 8740, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 124, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\", \"type\": \"PubMed\"}, \"evidence\": \"Lower angiotensin-converting enzyme (ACE) activity could increase the risk of Alzheimer's disease (AD) as ACE functions to degrade amyloid- (Abeta).\", \"hash\": \"9e148343dd5eedfbe47b11706c45bb0bb48f2f04f228665fc41fcb7fc5c258656854484705311f85a38fec15f3d48cdff780d4e1ef0ffb092a91e383da27d989\", \"key\": 1, \"line\": 20790, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"Alzheimers Res Ther. 2014 May 15;6(3):27\", \"reference\": \"24987467\", \"type\": \"PubMed\"}, \"evidence\": \"Lower CSF ACE protein level, and to a lesser extent serum ACE protein level and CSF ACE activity, were associated with lower CSF Abeta, indicating more brain Abeta pathology; adjusted regression coefficients (B) (95% CI) per SD increase were 0.09 (0.04; 0.15), 0.06 (0.00; 0.12) and 0.05 (0.00; 0.11), respectively. Further, lower CSF ACE protein level was associated with lower CSF tau and ptau levels; adjusted B's (95% CI) per SD increase were 0.15 (0.06; 0.25) and 0.17 (0.10; 0.25), respectively.These results strengthen the hypothesis that ACE degrades Abeta.\", \"hash\": \"c166a8201e8ae98480ed30f4928a218c8898783ee7b1bfb037562704b3d889d5e270e7641894f935b9b45d4542b2980cf621713ff24d9ed201aa05fb839477ae\", \"key\": 2, \"line\": 20814, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2001\", \"reference\": \"11604391\", \"type\": \"PubMed\"}, \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"hash\": \"58cd5742d440b206f80c10ea3792b4eb1e84949744a3ef7ed87e75e856787a81c54f56df87acdb6ad028e5b13ef083782e1563836e5f6b895c3d18418c3bdb33\", \"key\": 3, \"line\": 23726, \"relation\": \"decreases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2001\", \"reference\": \"11604391\", \"type\": \"PubMed\"}, \"evidence\": \"These findings led to the hypothesis that ACE may affect susceptibility to AD by degrading A beta and preventing the accumulation of amyloid plaques in vivo.\", \"hash\": \"c546970c8c84bdb5ee294dc7622518502c1006aca87ffe4ab5772c2141a8dcbfcaec1af3072621d5e852aba946f0d225302800d25a8ec21200e92cf30609089e\", \"key\": 4, \"line\": 23732, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"decreases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2001\", \"reference\": \"11604391\", \"type\": \"PubMed\"}, \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"hash\": \"28904097c7ccc24c4e0f9d86a55e2cce03edae51d949f156b25370e63a7223feec9757049a7679265eac729b70dcaa8498c7841d8b765c98d166ec2621c5e843\", \"key\": 5, \"line\": 23736, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"decreases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\", \"type\": \"PubMed\"}, \"evidence\": \"ACE was found to significantly inhibit A beta aggregation in a dose response manner.\", \"hash\": \"30d8821233a2b6a1c6498c850fe116e19687d009c42fe20b02b0339069158993a39ad40fcb21f5428d9010c41dd333aab1ac8f38f97b75aae36455656ef2a10d\", \"key\": 6, \"line\": 23922, \"relation\": \"decreases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Renin-angiotensin subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2001 Dec 21;276(51):47863-8. Epub 2001 Oct 16.\", \"reference\": \"11604391\", \"type\": \"PubMed\"}, \"evidence\": \"The most striking fact was that ACE degraded A beta by cleaving A beta-(1-40) at the site Asp(7)-Ser(8).\", \"hash\": \"76a05db5fb8ca11c8333ff9dfd3a9e0a89c1be2ab30e6461849677ec9e0311a964bee1e9a34ec7647adc6fe798889bc0e401afda4d4bee36f3b70a034266075c\", \"key\": 7, \"line\": 23931, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"decreases\", \"source\": 124, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurochem Int. 2013 Jun;62(8):1048-54. doi: 10.1016/j.neuint.2013.03.017. Epub 2013 Apr 8.\", \"reference\": \"23579020\", \"type\": \"PubMed\"}, \"evidence\": \"for the first time it was shown that the Abeta40 and Abeta42 are substrates of DPPIV. DPPIV prohibits the fibrillation of peptides and promotes disaggregation\", \"hash\": \"262ada3ff3483fca2bc9ac7e248489707ea80a6e0b1c5c1a28b7fe3da5f35aac205e3db6184d928fe01372a148f80fc30dcd128474aaf369b6f09cac37f44228\", \"key\": 0, \"line\": 12224, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 125, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}, \"UserdefinedCellLine\": {\"293APPwt\": true, \"CHOAPPsw\": true}}, \"citation\": {\"name\": \"FASEB J. 2000 Sep;14(12):1837-47.\", \"reference\": \"10973933\", \"type\": \"PubMed\"}, \"evidence\": \"Human bleomycin hydrolase (hBH) is a neutral cysteine protease genetically associated with increased risk for Alzheimer disease. We show here that ectopic expression of hBH in 293APPwt and CHOAPPsw cells altered the processing of amyloid precursor protein (APP) and increased significantly the release of its proteolytic fragment, beta amyloid (Abeta). We also found that hBH interacted and colocalized with APP as determined by subcellular fractionation, in vitro binding assay, and confocal immunolocalization. Metabolic labeling and pulse-chase experiments showed that ectopic hBH expression increased secretion of soluble APPalpha/beta products without changing the half-life of cellular APP. \", \"hash\": \"ae15386b07fbb5d694e6007d36a0445690970b2acba67dc45b9bf28f1a4e9b24fd969b2173bc4178e6c24dca94c100d822133f55749dad07ade4365594d6a188\", \"key\": 0, \"line\": 12404, \"relation\": \"increases\", \"source\": 126, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Cerebrospinal Fluid\": true}, \"Subgraph\": {\"Albumin subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Aug 10;287(33):28163-8. doi: 10.1074/jbc.C112.360800. Epub 2012 Jun 20.\", \"reference\": \"22718756\", \"type\": \"PubMed\"}, \"evidence\": \"Human serum albumin (HSA) binds 95% of Abeta in blood plasma and is thought to inhibit plaque formation in peripheral tissue. However, the role of albumin in binding Abeta in the cerebrospinal fluid has been largely overlooked. Here we investigate the effect of HSA on both Abeta(1-40) and Abeta(1-42) fibril growth. We show that at micromolar cerebrospinal fluid levels, HSA inhibits the kinetics of Abeta fibrillization, significantly increasing the lag time and decreasing the total amount of fibrils produced\", \"hash\": \"fc7dc41c292d4e16ea4a08a0c89edbc408340d7585f22b70986e662b9752434731977723648cac9a12a7b07e50663d05e7b901a87a2508ec5399f564cf5622ee\", \"key\": 0, \"line\": 12465, \"relation\": \"decreases\", \"source\": 127, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Serum\": true}, \"Species\": {\"9606\": true}, \"Subgraph\": {\"Albumin subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Biophys J. 2013 Oct 1;105(7):1700-9\", \"reference\": \"24094411\", \"type\": \"PubMed\"}, \"evidence\": \"Human serum albumin (HSA) is a potent inhibitor of Abeta self-association and this novel, to our knowledge, function of HSA is of potential therapeutic interest for the treatment of Alzheimer's disease.\", \"hash\": \"6942da364d195a74ed07bc19e9ee7d182085c03c57dea07e723defeea736bb9c34e84ff491c474d2956d46157444ccb0b91367b34cab73dea59beea684665fd0\", \"key\": 1, \"line\": 21179, \"relation\": \"decreases\", \"source\": 127, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Leptin subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\", \"type\": \"PubMed\"}, \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"hash\": \"ebf3b2adb22e3b688055616cf50f82d1037f675b490cad1539fb8337afc2b65324ef1e7440c2dedeb94622f2abca201812d7b2030031e3df8f85730d5600fe8d\", \"key\": 0, \"line\": 12665, \"relation\": \"decreases\", \"source\": 128, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer's disease2010\", \"reference\": \"20157255\", \"type\": \"PubMed\"}, \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"hash\": \"ef3dc1fd61374f303602ff4bc10d8a856dc0816715e54d75f8d741929dc88ef952e2a00c9a4a17482a58834c0d3d18af2c79f939b08f351d6909279c6d96f1a6\", \"key\": 1, \"line\": 29980, \"relation\": \"decreases\", \"source\": 128, \"target\": 1}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Leptin subgraph\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2009 May 22;455(3):191-4. doi: 10.1016/j.neulet.2009.03.066. Epub 2009 Mar 25.\", \"reference\": \"19429119\", \"type\": \"PubMed\"}, \"evidence\": \"We have previously demonstrated that Leptin reduces extracellular amyloid beta (Abeta) protein both in vitro and in vivo, and intracellular tau phosphorylation in vitro. Further, we have shown that these effects are dependent on activation of AMP-activated protein kinase (AMPK) in vitro. Herein, we investigated downstream effectors of AMPK signaling directly linked to tau phosphorylation. One such target, of relevance to Alzheimer's disease (AD), may be GSK-3beta, which has been shown to be inactivated by Leptin. We therefore dissected the role of GSK-3beta in mediating Leptin's ability to reduce tau phosphorylation in neuronal cells. Our data suggest that Leptin regulates tau phosphorylation through a pathway involving both AMPK and GSK-3beta. This was based on the following: Leptin and the cell-permeable AMPK activator, 5-aminoimidazole-4-carboxyamide ribonucleoside (AICAR), reduced tau phosphorylation at AD-relevant sites similarly to the GSK-3beta inhibitor, lithium chloride (LiCl). Further, this reduction of tau phosphorylation was mimicked by the downregulation of GSK-3beta, achieved using siRNA technology and antagonized by the ectopic overexpression of GSK-3beta. \", \"hash\": \"cb5917e14e120e53813f84d4c411a6a21727651d80788ea854f66bcb06ccf767dffd3c8201fa1f33097ec932c942e2577a4bb1ebb3de8079914147bfd1d0189b\", \"key\": 0, \"line\": 12676, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 128, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer's disease2010\", \"reference\": \"20157255\", \"type\": \"PubMed\"}, \"evidence\": \"Treatment with leptin reversed the 27-OHC-induced increase in Abeta and phosphorylated tau by decreasing the levels of BACE-1 and GSK-3beta respectively.\", \"hash\": \"f7a004a832bc35850093f0119c0d8630cdc98e89f50dbd9d79a8faa4e4b21690053937291d8e91239b86993a8bcd0925d28f79ab752a2d678ba517b2ecdce4c3\", \"key\": 1, \"line\": 29986, \"relation\": \"decreases\", \"source\": 128, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"9654eb45fa3c1567cc3a59fccfcca775b0d2401a0b004d23972167af4080450dba84a49ff34bb72c549fe27b087a4e9b3f672a79e741562b864384fc7f0833c0\", \"key\": 0, \"line\": 35944, \"relation\": \"increases\", \"source\": 129, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"d94421554f321e49b498cd74d68a51875b6df9a5489fccd379f2c914755b659e4902adc2ccc6efd3091c4d91f5e84c011b426c5a266d84fcee7ae4459b8b58ab\", \"key\": 0, \"line\": 35945, \"relation\": \"increases\", \"source\": 130, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"44f90de358a6fed078898365a6eb54eea48521300e97e11fc70908559e1e87b9ad64dfa7bb806f1869a16e8e8d08705486b55daba155619240d55c31f75bdfc6\", \"key\": 0, \"line\": 35947, \"relation\": \"increases\", \"source\": 131, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"f8f07c8c363a2911ec1d6bc86cc28f5e4fc14715ad6e3954cda8c281d5e871f51a4d992eacbc799ed6f41180b9eb5283cf4e55c6fbc4af4d718a4f0c4c7673cb\", \"key\": 0, \"line\": 35948, \"relation\": \"increases\", \"source\": 132, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"ce8dbd7cd523435fd6628bf9d9939572048d9c95c3278be204e5d41d0d91510adbc11149e0ab7acc1c7bb23a6485d1a7222a1c49dc5008257719b6a8b49d9385\", \"key\": 0, \"line\": 35949, \"relation\": \"increases\", \"source\": 133, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"4166638775df3a6f10568eecd51f634d0f28ae7c65cf96b2e678883f42ce1fd53990e5bf320a2f41e925bc5e2dae5079c245ebef866e54ca78c98edb4ad452d6\", \"key\": 0, \"line\": 35951, \"relation\": \"increases\", \"source\": 134, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"f9441c01d48405f22324a633b5e3356ee5a212e18145fe3f71e012f2fc993b93d1ce5ebab97031a8ef14ecf764f24ee7a513502c1d6d996a107ec1ff91dc902b\", \"key\": 0, \"line\": 35952, \"relation\": \"increases\", \"source\": 135, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Nature2010\", \"reference\": \"20811458\", \"type\": \"PubMed\"}, \"evidence\": \"Here we report the discovery of a novel gamma-secretase activating protein (GSAP) that drastically and selectively increases amyloid-beta production through a mechanism involving its interactions with both gamma-secretase and its substrate, the amyloid precursor protein carboxy-terminal fragment (APP-CTF).\", \"hash\": \"af7bc5199921f947559804e2bdee4ee1c1bbc361abea8f04c23af864d4b98a31ddbf41b97309f8b9f17e41d86fa55dfe4a0aef52cbf1bb5008f8b66b6da69ec8\", \"key\": 0, \"line\": 30256, \"relation\": \"increases\", \"source\": 136, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\", \"type\": \"PubMed\"}, \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"hash\": \"f8d68d49a3bac4731ba378c841b254a1490243028d6dc63328cd470cf421c3f977631334524ece14dd4d9cdfaccd28a895736a211faf599f8787231752e31ee2\", \"key\": 0, \"line\": 16083, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 137, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Clin Neuropathol2001\", \"reference\": \"11327298\", \"type\": \"PubMed\"}, \"evidence\": \"OBJECTIVE: Urinary-type plasminogen activator (uPA) binding to uPA receptor (uPAR) promotes the activation of matrix metalloproteinase-9 (MMP-9), which degrades amyloid beta protein (Abeta) in vitro.\", \"hash\": \"c09e647dcec9e320ccb6dbdc598aea1bae2374c636ff729f78355de8a22acc84596f453a0c0473828ffcfa499dc0940269f0fcb7e6a8dfc52249425a690f2788\", \"key\": 1, \"line\": 30295, \"relation\": \"increases\", \"source\": 137, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2006\", \"reference\": \"16787929\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that MMP-9 can degrade fAbeta and may contribute to ongoing clearance of plaques from amyloid-laden brains.\", \"hash\": \"36afbd72497c12541b99bf93ce0f19e8d807f204e492753c961ac3a888b2ea2d6a0da762041882f0a1b8bf9c889579ad6dd1e36c7d0558e00383b58e4873ab6c\", \"key\": 2, \"line\": 30426, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 137, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun. 1994 Dec 30;205(3):1755-61\", \"reference\": \"7811262\", \"type\": \"PubMed\"}, \"evidence\": \"Our aim was to test if this enzyme could also degrade the insoluble 40-42 residues long A beta peptides purified from Alzheimer Disease brain. Our results indicate that MMP2 hydrolyzes A beta 1-40 and A beta 1-42 peptides at Lys 16-Leu 17, at Leu 34-Met 35, and Met 35-Val 36 peptide bonds.These results suggest that MMP2 has the ability of degrading A beta of AD in vitro.\", \"hash\": \"b08df5225fd0b4be6fd1be78239a64cc0953d6444b6001c182786d7fef0d2416cb7ed3ca12462ffe8815d3208caa18ccd2476937f556a1da1d3a6c40f90f38b7\", \"key\": 0, \"line\": 16022, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 138, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Mol Cell Neurosci. 2012 Apr;49(4):423-9\", \"reference\": \"22402435\", \"type\": \"PubMed\"}, \"evidence\": \"Estrogen activates matrix metalloproteinases-2 and -9 to increase beta amyloid degradation.\", \"hash\": \"f09b086e9addbff86d874cf4d9f6c932b98572e9890fc709d6c436d04f02ee2ad9f2b269357f2ef5d2ab789c1cfb8f9d71f12803946d8681d5e4e88178a05b1c\", \"key\": 1, \"line\": 16081, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 138, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Bcl-2 subgraph\": true}}, \"citation\": {\"name\": \"Neuroscience. 2001;102(1):139-50\", \"reference\": \"11226677\", \"type\": \"PubMed\"}, \"evidence\": \"Cells overexpressing Bcl-xL were significantly protected from beta-amyloid neurotoxicity and staurosporine-induced apoptosis compared to vector-transfected controls.\", \"hash\": \"2ad2efda82ec8bd86e560086fc925541dd8df670b669d24bbb0f3b5d5020c76611952b5850bfe2f622aade388a616b95cb8e4ac94ee367a9ca948cbccf0d0540\", \"key\": 0, \"line\": 16647, \"relation\": \"decreases\", \"source\": 139, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"ATP binding cassette transport subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2005 Dec 30;280(52):43236-42. Epub 2005 Oct 5.\", \"reference\": \"16207708\", \"type\": \"PubMed\"}, \"evidence\": \"Deletion of Abca1 increases Abeta deposition\", \"hash\": \"4029d29f47e8cd438e9c89b749b37b053708b7ff1421c202053c2060cbbc7cabfb575077c60f9c7a1017779c8d98fe376caa4ffb4a7bc64831cf4ac77060f361\", \"key\": 0, \"line\": 24297, \"relation\": \"decreases\", \"source\": 140, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Neurons\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2007 Feb 2;282(5):2851-61. Epub 2006 Nov 22\", \"reference\": \"17121837\", \"type\": \"PubMed\"}, \"evidence\": \"These data indicate that ABCA1 and ABCG1 play a significant role in the regulation of neuronal cholesterol efflux to apoE discs and in suppression of APP processing to generate Abeta peptides.\", \"hash\": \"b968bdb7ef08a8119848876153eec970970624ff974105598cb7f53d51d933aa6d38fb662305e4dc59280eddb520dc9884ea649791f6c91c8288b133446f73df\", \"key\": 1, \"line\": 37222, \"relation\": \"decreases\", \"source\": 140, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Matrix metalloproteinase subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Curr Biol. 2002 Sep 17;12(18):1595-600\", \"reference\": \"12372252\", \"type\": \"PubMed\"}, \"evidence\": \"In contrast, enhanced FasL shedding, by recombinant MMP-7, completely protected neurons from Abeta neurotoxicity.\", \"hash\": \"9f3d22cf2d97043910d2cf78fc6e72fd6ec9341c7a033d4dcf90681cd880d0e0a054d15a26caa7e7a4f3084358f5a8a2c378d92a45d4a2d07467772763bd63a6\", \"key\": 0, \"line\": 18451, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 141, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2014 Mar;35(3):511-9\", \"reference\": \"24126163\", \"type\": \"PubMed\"}, \"evidence\": \"Thus, endogenous tPA, preferentially its aggregate form, could degrade Abeta molecules and maintain low levels of brain Abeta, resulting in the delay of AD pathogenesis.\", \"hash\": \"d87a9358cc3ba73ede29606063d438f5ec3b242272e413802d5cd0d8075ee995d12483b6e0c631e66628754b0e393bd7438fadb83d09a3a4183fdd77ee54c9a8\", \"key\": 0, \"line\": 18895, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 142, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry. 1999 Aug 31;38(35):11570-6\", \"reference\": \"10471309\", \"type\": \"PubMed\"}, \"evidence\": \"Subsequent studies showed that it was plasmin, the product of tPA activation of plasminogen, that specifically cleaved A beta 1-40 in the amino terminal region between Arg5 and His6.\", \"hash\": \"e7b83fb71a542449120fb6b20829846c9768f390a33159ab292aa3f2756d9067ea5737271873fd7d5be9381ac12b49042475259922a6625af6ed36957ea83fee\", \"key\": 1, \"line\": 18974, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 142, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"J Mol Neurosci2003\", \"reference\": \"14501010\", \"type\": \"PubMed\"}, \"evidence\": \"Additionally, plasmin (activated by tPA) attenuates Abeta neurotoxicity by degrading the peptide and rendering it inactive.\", \"hash\": \"249b0664a89f2c5766ec35331508f1e74d4586ad282959968abf62a41ca4c718df179664b7aff2e9ea24c621035b551bc8065b46d69f53b4f385fad26708f937\", \"key\": 2, \"line\": 30266, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 142, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Species\": {\"9606\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2009 Apr;109(2):303-15\", \"reference\": \"19222708\", \"type\": \"PubMed\"}, \"evidence\": \"Thus, neuroserpin inhibition of tissue plasminogen activator activity leads to reduced plasmin and may be responsible for reduced clearance of amyloid-beta in the Alzheimer disease brain.\", \"hash\": \"5d19996d8e2e64abad80076b57fd8ed5a92cfcf5c665c47240c6044b9bc781136fb5ae6707927804fed03f23e65b9b75a0db98579394dd2ba50c8857fedefa3a\", \"key\": 0, \"line\": 19039, \"relation\": \"increases\", \"source\": 143, \"target\": 1}, {\"annotations\": {\"Anatomy\": {\"cerebral blood vessel\": true}, \"Subgraph\": {\"Non-amyloidogenic subgraph\": true, \"Plasminogen activator subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2003 May 23;278(21):19054-61\", \"reference\": \"12754271\", \"type\": \"PubMed\"}, \"evidence\": \"This suggests that increased expression of uPA may initially serve as a protective mechanism leading to localized degradation and clearance of the pathogenic stimulus A beta.\", \"hash\": \"64aca705c6b3dd2465f037ee465728eab001cfd0ce1f963f8c7ea45ed0ab1d4268229cbfe28dadacd4eb3a1e599f55ce66523ebcb01393a29f79923b02de8aee\", \"key\": 0, \"line\": 20669, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 144, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun2009\", \"reference\": \"19523444\", \"type\": \"PubMed\"}, \"evidence\": \"An increase of the Abeta binding protein transthyretin suggests that increased clearance of Abeta underlies the reduction in plaques.\", \"hash\": \"77a5119f87597b769eb77d7dd509a03e81e4ab8484636cd4efb170935689674d2ed8e00d6357fca22052afd15899829751323602a3a48382c6e67f1ccb40924e\", \"key\": 0, \"line\": 27715, \"relation\": \"decreases\", \"source\": 145, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Interleukin signaling subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Immunol2008\", \"reference\": \"18941241\", \"type\": \"PubMed\"}, \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"hash\": \"4b96caea69698940d6bdb584e8e99ee75eaf21d77c7597421e45fcaf7bf3f56a78c96f970e3e8920227f362102300609c1569b66fa6ee99a36a352c9e02a97f7\", \"key\": 0, \"line\": 29804, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 146, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2006\", \"reference\": \"17050537\", \"type\": \"PubMed\"}, \"evidence\": \"Finally, we show that PAT1a promotes APP/APLPs processing, resulting in increased secretion of beta-amyloid peptide\", \"hash\": \"43f2d12e82c6a0bfb2dcd97912a4d6f576ea9a7cd2f952653090066dc1ff95e0bda84f8634a41889b073e2b9a4018c70006e1afc0ea6417c5883536fd06405a1\", \"key\": 0, \"line\": 25544, \"relation\": \"increases\", \"source\": 147, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Apoptosis signaling subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2005\", \"reference\": \"15671026\", \"type\": \"PubMed\"}, \"evidence\": \"PAR-4 (prostate apoptosis response-4) is a leucine zipper protein that was initially identified to be associated with neuronal degeneration and aberrant Abeta production in pmodels of AD.\", \"hash\": \"81255b94b17f274d34e16bd4821a111de248961645f4ac89002bba5bf05beeb8c709308ea1c764f9ac77aeec7bb22329a8b9e09067d70b9f951f1cecdf95b2f0\", \"key\": 0, \"line\": 25760, \"relation\": \"increases\", \"source\": 148, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Wnt signaling subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2007\", \"reference\": \"17360493\", \"type\": \"PubMed\"}, \"evidence\": \" Overexpression of constitutively active CK1epsilon, one of the CK1 isoforms expressed in brain, leads to an increase in Abeta peptide production. \", \"hash\": \"a28c7ccd5a0cfc5ed2e3c21dcdd02b63e2ed56cd51c4d438127a2a46e5ed0c123d9432990d3196ded056104d9c152f47c5b1a73d45b7d6566597fc6f3de71563\", \"key\": 0, \"line\": 25904, \"relation\": \"increases\", \"source\": 149, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry2008\", \"reference\": \"18570439\", \"type\": \"PubMed\"}, \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE.\", \"hash\": \"3b1131d7d89f4f5169b651668cf101797d66d8d8d7f352d9cdbdefb6987531b6323bc6594193933cd4a2f6dcb5f4fd16fac2c7bdd228af02a85abaca489d5b01\", \"key\": 0, \"line\": 26034, \"relation\": \"increases\", \"source\": 150, \"subject\": {\"effect\": {\"name\": \"pep\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Biochemistry2008\", \"reference\": \"18570439\", \"type\": \"PubMed\"}, \"evidence\": \"Here we show that glutaminyl cyclase (QC) catalyzes the formation of Abeta 3(pE)-40/42 after amyloidogenic processing of APP in two different cell lines, applying specific ELISAs and Western blotting based on urea-PAGE. \", \"hash\": \"beff57901df35614049b5deccbd850bf6dfaf748b865ca553945fb057d1632bd9aadec9e622418ba5d2fdd63fed04b2c3321640790aa32313d121acc93a67f59\", \"key\": 1, \"line\": 27404, \"relation\": \"increases\", \"source\": 150, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Biochem Biophys Res Commun2008\", \"reference\": \"18675254\", \"type\": \"PubMed\"}, \"evidence\": \"urther, overexpression of the SUMO E2 enzyme ubc9 along with SUMO-1 results in decreased levels of Abeta aggregates in cells transfected with the familial Alzheimer's disease-associated V642F mutant APP, indicating the potential of up-regulating activity of the cellular sumoylation machinery as an approach against Alzheimer's disease. \", \"hash\": \"ddce00608138523dab8698910513d845eb74f7dfc342bfc76554ea602c306528f457b885482467dbd7b7ad936d7d3c89323504744eec3839c24be3b1d5822a17\", \"key\": 0, \"line\": 26056, \"relation\": \"increases\", \"source\": 151, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2009\", \"reference\": \"19251705\", \"type\": \"PubMed\"}, \"evidence\": \" Importantly, knockdown of endogenous RanBP9 significantly reduced Abeta generation in Chinese hamster ovary cells and in primary neurons, demonstrating its physiological role in BACE1 cleavage of APP. \", \"hash\": \"2ddff6bf45326c2b99a8ab1be0e62231888404135f691cf353a65ef960e7fc51c6271db751005a34f74126ca3136e3d3d06991151ec60d2532ce7a98f8e8ca51\", \"key\": 0, \"line\": 26099, \"relation\": \"decreases\", \"source\": 152, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem2010\", \"reference\": \"20595388\", \"type\": \"PubMed\"}, \"evidence\": \"Consistently, overexpression of calpain in heterologous APP expressing cells up-regulated the level of BACE1 and increased Abeta production. \", \"hash\": \"f07913495b9c9a6769e638bfe4e943af2aaa734259cbf600d72bfa650ec483b1d2697fa4ea1ba2290afe11ce4e3f2276a424010af6a5b31c61867bd1e5598e1b\", \"key\": 0, \"line\": 26423, \"relation\": \"increases\", \"source\": 153, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"Proc Natl Acad Sci U S A2006\", \"reference\": \"17098871\", \"type\": \"PubMed\"}, \"evidence\": \"Using high-throughput siRNA screening technology, we assessed 15,200 genes for their role in Abeta42 secretion and identified leucine-rich repeat transmembrane 3 (LRRTM3) as a neuronal gene that promotes APP processing by BACE1. siRNAs targeting LRRTM3 inhibit the secretion of Abeta40, Abeta42, and sAPPbeta, the N-terminal APP fragment produced by BACE1 cleavage, from cultured cells and primary neurons by up to 60%, whereas overexpression increases Abeta secretion.\", \"hash\": \"9ac4cb571249b9da22488fbaf0ebced9ba7b33fd06a3f6c28a7a86ec6fab66ca582bc73c4dbf6c684c6e885fe0e0560682a250a08f2700541c88bb85facc162d\", \"key\": 0, \"line\": 26972, \"relation\": \"decreases\", \"source\": 154, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci2006\", \"reference\": \"16452662\", \"type\": \"PubMed\"}, \"evidence\": \"Overexpression of NgR decreases Abeta production in neuroblastoma culture, and targeted disruption of NgR expression increases transgenic mouse brain Abeta levels, Abeta plaque deposition, and dystrophic neurites. \", \"hash\": \"96cb83c0d752253f1621e858fdd8ecd352155dbc1c8f0347d7409e852da91c29fe5c125b23561dde93f1a54be593f503757530d280e6e193b58226aa783bda3a\", \"key\": 0, \"line\": 27429, \"relation\": \"decreases\", \"source\": 155, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res2007\", \"reference\": \"18220524\", \"type\": \"PubMed\"}, \"evidence\": \" Deletion of NgR expression increases Abeta plaque deposition in transgenic mice, while excess soluble NgR treatment reduces Abeta plaque deposition in mice.\", \"hash\": \"90366acf9609840f22118db4f12d0be1c9198a2f70d49121f074dbcbbe3d860b5c27f2a50cd213e45786ed71924ce9339f1c466943f30e19689b9805a08f01d7\", \"key\": 1, \"line\": 27451, \"relation\": \"decreases\", \"source\": 155, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Interleukin signaling subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Immunol2008\", \"reference\": \"18941241\", \"type\": \"PubMed\"}, \"evidence\": \"NEP and IDE also contributed to IL-4-induced degradation of Abeta(1-42), b ecause their inhibitors, thiorphan and insulin, respectively, significantly suppressed this activity.\", \"hash\": \"0b543ed2080f6de6f6800e4631da3978a5e7d7bf337a90935c682600379a0734af79bac1d541608cb6456735682a9a53ec7faf2bcfc0eaa05343fb44dca5625f\", \"key\": 0, \"line\": 29803, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 156, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Acetylcholine signaling subgraph\": true, \"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Dysregulation of intracellular calcium signaling has been implicated in the pathogenesis of Alzheimers disease [150]. ABeta is known to act through multiple targets [151] including Ca2+ channels and various receptors in membranes. Synthetic ABeta binds to the calcium permeable nAChRs with high affinity [152]. ABeta42 administered in the low picomolar range activates nAChRs at presynaptic nerve endings of synaptosomes [83, 153]. Under normal conditions, activation of nAChRs is necessary for the ABeta-induced increase in synaptic plasticity and memory [23]. However, it remains to be determined whether these effects are mediated by a direct physical interaction of the peptide with the nAChR. In addition, ABeta enhances transmitter release by transient increase of glutamate release from the presynaptic terminal that results from brief periods of high frequency stimulation with Ca2+ buildup within the terminal that triggers mechanisms of short-term synaptic plasticity [154].\", \"hash\": \"4e3176efe95aa22143617131253fb3410d8bd0b2408bc2a72aa75671c275eb41c4ce9e638222c5e39a53bb21696ab909e651d4e8abd43ff0c4d84d3cd39332f6\", \"key\": 0, \"line\": 35943, \"relation\": \"increases\", \"source\": 157, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Species\": {\"10090\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Epigenetic modification subgraph\": true}}, \"citation\": {\"name\": \"Free Radic Biol Med. 2009 May 1;46(9):1241-9. doi: 10.1016/j.freeradbiomed.2009.02.006. Epub 2009 Feb 23.\", \"reference\": \"19245828\", \"type\": \"PubMed\"}, \"evidence\": \"Down regulation of DNMT results in hypomethylation of BACE1 and APP which are involved in Abeta production and causes upregulation of their protein expression; in turn SP1 transcription factor increases which finally results in Abeta production.\", \"hash\": \"9c544ee70913bf68db1a18ecd67dfcaed4ecef3e2fbd36d8ec273b3f4948a093b2cfa76d6ca10f14c7d575c821cddf86cd7b65d350e2e25c28b5300851984845\", \"key\": 0, \"line\": 43366, \"relation\": \"increases\", \"source\": 158, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Metabolism of steroid hormones subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2014 Apr;35(4):e11-2\", \"reference\": \"24262203\", \"type\": \"PubMed\"}, \"evidence\": \"In CAA, and probably in AD, corticosteroid treatment suppresses the secondarily induced vascular inflammation (Chang et al., 2012, Kloppenborg et al., 2010 and Machida et al., 2012) and the activation of S100A9/Mrp14 (Gebhardt et al., 2002 and Kametani and Ikeda, 2013). Recently, it has been reported that anti-Abeta autoantibodies were progressively reduced in the cerebrospinal fluid of a patient with CAA-related inflammation, following intravenous steroid administration (Piazza et al., 2013). Reduction of S100A9/Mrp14 might induce suppression of amyloid fibril formation (Zhang et al., 2012) and increase phagocytosis of fibrillar Abeta in microglia cells (Kummer et al., 2012), resulting in reduced Abeta amyloid fibril deposition.\", \"hash\": \"47d01145ed63ee9098b87b85afcf73b8a87aef61d1e29f32bd59fb001c1cabca1114d58b5e4f717c648dfb076ea1753ff967fbbecc869c8df63f8b734b3745bd\", \"key\": 0, \"line\": 42476, \"relation\": \"increases\", \"source\": 159, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Acta Neuropathol. 2014 Apr;127(4):507-22\", \"reference\": \"24240735\", \"type\": \"PubMed\"}, \"evidence\": \"Pro-inflammatory S100A9 protein is increasingly recognized as an important contributor to inflammation-related neurodegeneration. Here, we provide insights into S100A9 specific mechanisms of action in Alzheimer's disease (AD). Due to its inherent amyloidogenicity S100A9 contributes to amyloid plaque formation together with Abeta.\", \"hash\": \"a0e1d2f1f6a7d67b1583bf6fb3bcec156565dc5dc088dcb00a076dc0ca15f914c31d047d3f30c560eb52beca72eead74925767fecfadfe26a6039ff9535490e6\", \"key\": 1, \"line\": 42500, \"relation\": \"increases\", \"source\": 159, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"J Neurosci. 2012 Dec 5;32(49):17824-9.\", \"reference\": \"23223301\", \"type\": \"PubMed\"}, \"evidence\": \"We therefore conclude that Mrp14 promotes APP processing and Abeta accumulation under neuroinflammatory conditions.\", \"hash\": \"12c938cd5c922242518cf4e7e30ae94338e837782bd40426ea3fecb48bea7beaa5339494f5ec617ec1eee036f5c80fa1c1786fa685c41d0c1406c290f3d2d325\", \"key\": 2, \"line\": 42540, \"relation\": \"increases\", \"source\": 159, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Autophagy signaling subgraph\": true, \"MAPK-ERK subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"J Toxicol. 2012;2012:187297. Epub 2012 Feb 8.\", \"reference\": \"22496686 \", \"type\": \"PubMed\"}, \"evidence\": \"On the contrary, X11 stabilizes ABetaPP conformation in membrane, inhibiting ABeta secretion in cultured cells [45], likely impairing ABetaPP trafficking to sites containing active gamma-secretase complexes [46]. JIP's are member of JNK-scaffolding family proteins kinases, implicated in different signal pathway, including neuronal apoptotic process. JNK-interacting proteins JIP1b and JIP2 bind to the cytoplasmic tail of ABetaPP. The expression of JIP1b stabilizes immature ABetaPP and decreases the ABetaPP ectodomain, ABeta40/42 and CTFs abundance [47].All these observations suggest that some of these protein-protein interactions may play a role in the modulation of the amyloidogenic pathway and thus might have a role in neurodegeneration.\", \"hash\": \"e7ace59c7c27c31c12b328566106eecc00ca0396055864dbd2c655242248935fc9e6847e70843dfd20da62397caa90b2a7efb9b2247cb58956046641d1d95a46\", \"key\": 0, \"line\": 37039, \"relation\": \"decreases\", \"source\": 160, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Neurons\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2007 Feb 2;282(5):2851-61. Epub 2006 Nov 22\", \"reference\": \"17121837\", \"type\": \"PubMed\"}, \"evidence\": \"These data indicate that ABCA1 and ABCG1 play a significant role in the regulation of neuronal cholesterol efflux to apoE discs and in suppression of APP processing to generate Abeta peptides.\", \"hash\": \"4a65312cfc2b8b3fcaadba1ee32fbce4c03041b45cd05a57bcbf784562f0ff6e0478b36a34d526326d1116df29144192cc09158b75cdd03027e08d82f1ee0bf9\", \"key\": 0, \"line\": 37224, \"relation\": \"decreases\", \"source\": 161, \"subject\": {\"effect\": {\"name\": \"tport\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"FASEB J. 2010 Nov;24(11):4491-502. Epub 2010 Jul 12.\", \"reference\": \"20624932\", \"type\": \"PubMed\"}, \"evidence\": \"Here, we report a novel function of glutamate carboxypeptidase II (GCPII) in Abeta degradation in brain, which is a peptidase involved in N-acetylaspartylglutamate cleavage, folate metabolism, and prostate tumorigenesis.\", \"hash\": \"6566105738dcf8e6a578d72b890c9cd71fccff3276224558c9d4321192ff39f1364e4e4a45daf5dd20d54663511e078a94c2381bfbd32a324f5974fd89f91158\", \"key\": 0, \"line\": 37428, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 162, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"FASEB J. 2010 Nov;24(11):4491-502. Epub 2010 Jul 12.\", \"reference\": \"20624932\", \"type\": \"PubMed\"}, \"evidence\": \"Here, we report a novel function of glutamate carboxypeptidase II (GCPII) in Abeta degradation in brain, which is a peptidase involved in N-acetylaspartylglutamate cleavage, folate metabolism, and prostate tumorigenesis.\", \"hash\": \"1eac7e0b1a35ea3ba25355f900476ac924a8d0ba4978bba13012143d51d5b528d49cfd3ecd97b0c02c10f94254dcd30768b4c3b234519813900f09b265605663\", \"key\": 1, \"line\": 37429, \"relation\": \"decreases\", \"source\": 162, \"target\": 1}, {\"citation\": {\"name\": \"J Alzheimers Dis. 2011;26(3):457-66. doi: 10.3233/JAD-2011-110056.\", \"reference\": \"21673407\", \"type\": \"PubMed\"}, \"evidence\": \"The 5-HT4 receptor may play a role in memory and learning and 5-HT4 receptor activation has been suggested to modulate acetylcholine release and to reduce amyloid- (Abeta) accumulation.\", \"hash\": \"c12504db6dc04368249bb59dac138130545ae101c9d1436139a1d38f10adbca2d93cea99eea6bda54f2e9c17aacaf43c252ff592b222b094153b9a18bcff5fee\", \"key\": 0, \"line\": 46120, \"relation\": \"decreases\", \"source\": 163, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Neurons\": true}, \"Subgraph\": {\"Eicosanoids signaling subgraph\": true}}, \"citation\": {\"name\": \"Med Hypotheses. 2000 Jan;54(1):75-6\", \"reference\": \"10790729\", \"type\": \"PubMed\"}, \"evidence\": \"The formation of endogenous inflammatory lipid mediators, leukotrienes, is initiated by 5-lipoxygenase (5-LOX), which is also expressed in neurons.\", \"hash\": \"94b067f07a525712e11c32875bc263a68ae4b64c512d7da5b93839028a806af90279dfc19a56fc84d55f3635862f840d53e9ea7ad74a530aa41e3902a2a4fd9a\", \"key\": 0, \"line\": 21451, \"relation\": \"increases\", \"source\": 164, \"target\": 2}, {\"annotations\": {\"Subgraph\": {\"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2010 May 26;476(1):18-22. doi: 10.1016/j.neulet.2010.03.054. Epub 2010 Mar 27.\", \"reference\": \"20347935\", \"type\": \"PubMed\"}, \"evidence\": \"Here we report that the levels of a human brain-enriched miRNA-125b are up-regulated in interleukin-6 (IL-6)-stressed normal human astrocytes (NHA), a treatment known to induce astrogliosis. In vitro, anti-miRNA-125b added exogenously to IL-6-stressed NHA cultures attenuated both glial cell proliferation and increased the expression of the cyclin-dependent kinase inhibitor 2A (CDKN2A), a miRNA-125b target and negative regulator of cell growth. A strong positive correlation between miRNA-125b abundance and the glial cell markers glial fibrillary acidic protein (GFAP) and vimentin, and CDKN2A down-regulation was noted in advanced Alzheimer's disease (AD) and in Down's syndrome (DS) brain, chronic neurological disorders associated with astrogliosis. The results suggest that miRNA-125b up-regulation contributes to astrogliosis and to defects in the cell cycle that are characteristic of degenerating brain tissues.\", \"hash\": \"e5d5a8f1e8f72a0527491d9e79243af838822d013f9f65f10d886b993292717af7745c412b6152883b39bfe7cadf184f6ee57bb99f1d710b76760d8a43d7a46a\", \"key\": 0, \"line\": 9343, \"relation\": \"increases\", \"source\": 165, \"target\": 3}, {\"annotations\": {\"Subgraph\": {\"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2010 May 26;476(1):18-22. doi: 10.1016/j.neulet.2010.03.054. Epub 2010 Mar 27.\", \"reference\": \"20347935\", \"type\": \"PubMed\"}, \"evidence\": \"Here we report that the levels of a human brain-enriched miRNA-125b are up-regulated in interleukin-6 (IL-6)-stressed normal human astrocytes (NHA), a treatment known to induce astrogliosis. In vitro, anti-miRNA-125b added exogenously to IL-6-stressed NHA cultures attenuated both glial cell proliferation and increased the expression of the cyclin-dependent kinase inhibitor 2A (CDKN2A), a miRNA-125b target and negative regulator of cell growth. A strong positive correlation between miRNA-125b abundance and the glial cell markers glial fibrillary acidic protein (GFAP) and vimentin, and CDKN2A down-regulation was noted in advanced Alzheimer's disease (AD) and in Down's syndrome (DS) brain, chronic neurological disorders associated with astrogliosis. The results suggest that miRNA-125b up-regulation contributes to astrogliosis and to defects in the cell cycle that are characteristic of degenerating brain tissues.\", \"hash\": \"f0cf38689e0ed756a9512a1d04adf7b4270340ce066eb3937b9fdb00081a198603624c38fa70c2a561fb8e45c1e60eba1e145d8f7115c08a39994074662360d8\", \"key\": 0, \"line\": 9344, \"relation\": \"increases\", \"source\": 166, \"target\": 3}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"MeSHDisease\": {\"Alzheimer Disease\": true}, \"Subgraph\": {\"Calcium-dependent signal transduction\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\", \"type\": \"PubMed\"}, \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"hash\": \"ff26d7d10d8523b2577c4fd052397ead9f44a66806553404ae46b86f82fa7808a06bd6945da18b2f74ba5fbb11b6d85f3f04ed1ad6c990929fb8f486f671df98\", \"key\": 0, \"line\": 38358, \"relation\": \"increases\", \"source\": 167, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"Calcium-dependent signal transduction\": true}}, \"citation\": {\"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\", \"type\": \"PubMed\"}, \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"hash\": \"95a0c09693632fcf6e11792f258a4664622e40ccbf33a600ef8e6247fe29eeffc38d061e60a687df64c48e3227a79df5d27c067f4f5e53d66bb3ae15e9767a96\", \"key\": 0, \"line\": 38328, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 167, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Acta Neuropathol. 2000 Feb;99(2):91-5\", \"reference\": \"10672313\", \"type\": \"PubMed\"}, \"evidence\": \"Stromelysin-1 (MMP-3) plays a central role in activating latent-type MMPs, which are originally secreted as proenzymes.\", \"hash\": \"3bad13277c176674358827ebd5690a918b2f78b92f3c0aa0dd32772d5d682f37992b0db611aba99e6c0c761233127513fb127ef181c325cc034909b933ad422c\", \"key\": 0, \"line\": 18776, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 168, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Regulation of actin cytoskeleton subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane -subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the -subunit, and causes autophosphorylation of the -subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3 and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3 signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta monomers may bind to IGF-1/insulin receptors, as already shown for ABeta oligomers \", \"hash\": \"eacad9f713792216ac71f5d46db2a8c3522b3a6ce4b8c5dfdd61a9f1be920dc0efeecc617b4744c26986527ab1263ca80dddb141b95f13eee66a91e72b0a9ec2\", \"key\": 0, \"line\": 35706, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 169, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Regulation of actin cytoskeleton subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane -subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the -subunit, and causes autophosphorylation of the -subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3 and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3 signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta monomers may bind to IGF-1/insulin receptors, as already shown for ABeta oligomers \", \"hash\": \"eacad9f713792216ac71f5d46db2a8c3522b3a6ce4b8c5dfdd61a9f1be920dc0efeecc617b4744c26986527ab1263ca80dddb141b95f13eee66a91e72b0a9ec2\", \"key\": 1, \"line\": 35712, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 169, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Beta-Catenin subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"The PI3K/Akt signaling: PI3K, a membrane-associated second messenger protein, and its downstream kinase, Akt, are associated with neuronal survival [111] and plasticity [112] via activation of transcription pathways and protein synthesis. PI3K pathway, which required the activation of IGF-1/insulin receptors, is the most convincing prosurvival effect of ABeta42 monomers. The PI3K signaling pathway is important in the transmission of survival signals in many cell types including neurons [113, 114]. The PI3K-Akt signaling cascade, initiated by IRS, is phosphorylated by stimulated insulin- andIGF-receptor tyrosine kinases [115]. One of the kinases known to lie downstream of PI3K is Akt, which can be directly activated by products of PI3K [116] by promoting its phosphorylation at Ser473 and Thr308 [117]. Activated Akt, in turn, phosphorylates a wide range of substrates activating anti-apoptotic (survival) factors and inactivating pro-apoptotic factors [114, 117]. Certain proapoptotic mediators, such as the transcription factor forkhead (FOXO), the tau kinase GSK-3, and the Bcl2 antagonist BAD proteins, are inactivated by Akt [118, 119]. Akt downregulates the activities of GSK-3a and GSK-3 by phosphorylating the former at Ser21 and the latter at Ser9 [118]. Phosphorylation/inactivation of GSK-3, suppresses GSK-3-dependent phosphorylation of tau at residues overphosphorylated in AD and prevents apoptosis of confluent cells. Treatment of cortical neurons with ABeta42 monomers increased Ser9 phosphorylation (inhibition) of the Akt substrate, GSK- 3 [97]. Inhibition of GSK-3 promotes cell survival through a variety of mechanisms including a reduced degradation of -catenin, which then translocates into the nucleus and activates the transcription of protective genes [126].\", \"hash\": \"b2ed1752cd4cb21f0b2619f5262aa738c752639fc12ffa3eaf19d3002cfd3bd9c7fbddedd34333fe509ad21a01081f6f0fa2f67acde04715be121dce923c10fe\", \"key\": 0, \"line\": 35831, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"decreases\", \"source\": 170, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Insulin signal transduction\": true, \"Regulation of actin cytoskeleton subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\", \"type\": \"PubMed\"}, \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane -subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the -subunit, and causes autophosphorylation of the -subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3 and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3 signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta monomers may bind to IGF-1/insulin receptors, as already shown for ABeta oligomers \", \"hash\": \"48ef5c4ef9d7d6873927741a07211e8bc8ba00c606066057f2ddbe2dca7adc00a337a91eb2d8f99c2631b891d96b0421f1ea74e60e343e4f0e01b95af89297a4\", \"key\": 0, \"line\": 35696, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 171, \"target\": 7}, {\"citation\": {\"reference\": \"24653673\", \"type\": \"PubMed\"}, \"evidence\": \" Specific PP2A inhibition has been proven to lead to in vivo deregulation of many major brain Ser/Thr kinases implicated in AD, including GSK3beta (Wang et al., 2010; Louis et al., 2011), cdk5 (Louis et al., 2011; Kimura et al., 2013), extracellular signal-regulated kinase (ERK) and JNK (Kins et al., 2003).\", \"hash\": \"a791a4f8ba452c40512f10834ad6ca00afac330e6551ec231ccb2a1024053aa2501c3b219fb4cd7612eca6daa1d6c384d1dddffc1ae631526c4a16dba375b4ea\", \"key\": 0, \"line\": 57624, \"relation\": \"increases\", \"source\": 172, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 7}, {\"annotations\": {\"Confidence\": {\"High\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Tau protein subgraph\": true}}, \"citation\": {\"name\": \"Journal of Biological Chemistry2005\", \"reference\": \"15522877\", \"type\": \"PubMed\"}, \"evidence\": \"In situ, FRAT-2 significantly increased GSK3 beta-mediated phosphorylation of tau at a primed epitope while not significantly affecting the phosphorylation of unprimed sites.\", \"hash\": \"8b4a08d763fa6cbd3d447834b70c63726ac6eeac0782fa6ed75e7a719f88be04e76228b8630bf2daf9611f490c0edf106af13755c09139fe8d13ca0dba0c1ff7\", \"key\": 0, \"line\": 29663, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 173, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Prostaglandin subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Neuroreport. 2009 Oct 28;20(16):1500-5. doi: 10.1097/WNR.0b013e3283329c05.\", \"reference\": \"19801956\", \"type\": \"PubMed\"}, \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"hash\": \"466098d9c24c5d1fb9def4339ad931f05e137730d1f43f0a47ff7badae1e2b2519d81b57031505ac8941f46a72416f4267d38b55c239d7d6c51bd6801249140f\", \"key\": 0, \"line\": 9183, \"relation\": \"increases\", \"source\": 174, \"target\": 8}, {\"annotations\": {\"Subgraph\": {\"Complement system subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2008 Nov 14;283(46):31315-22. doi: 10.1074/jbc.M805371200. Epub 2008 Sep 18.\", \"reference\": \"18801740\", \"type\": \"PubMed\"}, \"evidence\": \" Here we provide evidence in AD brains of a specific up-regulation of an NF-kappaB-sensitive miRNA-146a highly complementary to the 3'-untranslated region of complement factor H (CFH), an important repressor of the inflammatory response of the brain.Transfection of HN cells using an NF-kappaB-containing pre-miRNA-146a promoter-luciferase reporter construct in stressed HN cells showed significant up-regulation of luciferase activity that paralleled decreases in CFH gene expression. Up-regulation of miRNA-146a coupled to down-regulation of CFH was observed in AD brain and in interleukin-1beta, Abeta42, and/or oxidatively stressed human neural (HN) cells in primary culture.\", \"hash\": \"a6d1b952c33d47f7da66a870af21ed932e3306dab876f253857fa76ea6d16492bd63c5ad6ff952f1b7a2e83f9193b3986b5e2d96ad86e68384a4697df1d2c897\", \"key\": 0, \"line\": 9121, \"relation\": \"decreases\", \"source\": 174, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2011 Jul 20;499(2):109-13. doi: 10.1016/j.neulet.2011.05.044. Epub 2011 May 26.\", \"reference\": \"21640790\", \"type\": \"PubMed\"}, \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"hash\": \"02d2000999bd18356900ee030aaebfa5f7e9dcd7c39743f1852a2702d7e4d19fca49ddce3d665c9c91960eccba0b3078da1fcf3a300b2fe5aab9414ba898a2bf\", \"key\": 1, \"line\": 9635, \"relation\": \"decreases\", \"source\": 174, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Complement system subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Inorg Biochem. 2009 Nov;103(11):1591-5. doi: 10.1016/j.jinorgbio.2009.05.012. Epub 2009 May 24.\", \"reference\": \"19540598\", \"type\": \"PubMed\"}, \"evidence\": \"Here we provide evidence in human neural (HN) cells of an aluminum-sulfate- and reactive oxygen species (ROS)-mediated up-regulation of an NF-kappaB-sensitive miRNA-146a that down-regulates the expression of complement factor H (CFH), an important repressor of inflammation. This NF-kappaB-miRNA-146a-CFH signaling circuit is known to be similarly affected by Abeta42 peptides and in AD brain. These aluminum-sulfate-inducible events were not observed in parallel experiments using iron-, magnesium-, or zinc-sulfate-stressed HN cells. An NF-kappaB-containing miRNA-146a-promoter-luciferase reporter construct transfected into HN cells showed significant up-regulation of miRNA-146a after aluminum-sulfate treatment that corresponded to decreased CFH gene expression. These data suggest that (1) as in AD brain, NF-kappaB-sensitive, miRNA-146a-mediated, modulation of CFH gene expression may contribute to inflammatory responses in aluminum-stressed HN cells, and (2) underscores the potential of nanomolar aluminum to drive genotoxic mechanisms characteristic of neurodegenerative disease processes.\", \"hash\": \"91f9c2d0347e76830fd9c6edf698473b6af11945e829d5e68aab6c306ab89888e848d1e48811d9fa28a3ea11c3ade7b0671c4bf0f917be45cfd29de5367745b6\", \"key\": 2, \"line\": 9161, \"relation\": \"decreases\", \"source\": 174, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Complement system subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neuroreport. 2009 Oct 28;20(16):1500-5. doi: 10.1097/WNR.0b013e3283329c05.\", \"reference\": \"19801956\", \"type\": \"PubMed\"}, \"evidence\": \" We report that infection of human primary neural cells with a high phenotypic reactivator HSV-1 (17syn+) induces upregulation of a brain-enriched microRNA (miRNA)-146a that is associated with proinflammatory signaling in stressed brain cells and Alzheimer's disease. Expression of cytoplasmic phospholipase A2, the inducible prostaglandin synthase cyclooxygenase-2, and the neuroinflammatory cytokine interleukin-1beta were each upregulated. A known miRNA-146a target in the brain, complement factor H, was downregulated. These data suggest a role for HSV-1-induced miRNA-146a in the evasion of HSV-1 from the complement system, and the activation of key elements of the arachidonic acid cascade known to contribute to Alzheimer-type neuropathological change.\", \"hash\": \"11bb4d43ffce2b8863fc3dafef6dacbef67ab044ccc6c22be780812a79360ef8f98f12b6b27e522f63367080040576df3a979e1c16f4fd3196a703096dbfe2dd\", \"key\": 3, \"line\": 9187, \"relation\": \"decreases\", \"source\": 174, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"87a493a1b0db6474f1f4855489b72af5a395cf936ca54fb0732a2445e1546eb63a2f8d3e3f992637c8de20c16893e324e7ca6469e1743f1dc0e7685316174244\", \"key\": 0, \"line\": 9508, \"relation\": \"decreases\", \"source\": 174, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"Neurosci Lett. 2011 Jul 20;499(2):109-13. doi: 10.1016/j.neulet.2011.05.044. Epub 2011 May 26.\", \"reference\": \"21640790\", \"type\": \"PubMed\"}, \"evidence\": \"MicroRNA-146a (miRNA-146a) is an inducible, 22 nucleotide, small RNA over-expressed in Alzheimer's disease (AD) brain. Up-regulated miRNA-146a targets several inflammation-related and membrane-associated messenger RNAs (mRNAs), including those encoding complement factor-H (CFH) and the interleukin-1 receptor associated kinase-1 (IRAK-1), resulting in significant decreases in their expression (p<0.05, ANOVA). In this study we assayed miRNA-146a, CFH, IRAK-1 and tetraspanin-12 (TSPAN12), abundances in primary human neuronal-glial (HNG) co-cultures, in human astroglial (HAG) and microglial (HMG) cells stressed with Abeta42 peptide and tumor necrosis factor alpha (TNFalpha). The results indicate a consistent inverse relationship between miRNA-146a and CFH, IRAK-1 and TSPAN12 expression levels, and indicate that HNG, HAG and HMG cell types each respond differently to Abeta42-peptide+TNFalpha-triggered stress. While the strongest miRNA-146a-IRAK-1 response was found in HAG cells, the largest miRNA-146a-TSPAN12 response was found in HNG cells, and the most significant miRNA-146a-CFH changes were found in HMG cells, the 'resident scavenging macrophages' of the brain.\", \"hash\": \"0ad3f055294235b816fe33d0feb1cc3cf52168b969d1bc3d1c7d8888e98955dd485ce71ce8793410598d69bd7215e57434a8dda6e94d5b42f889468668651107\", \"key\": 1, \"line\": 9636, \"relation\": \"decreases\", \"source\": 174, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true, \"miRNA subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\", \"type\": \"PubMed\"}, \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"hash\": \"6f285c895c1c2a533ccf650d877996181934666c1f861081c56c129d8beeb0c89820a5ff7f09f3b15d0d7273d94088940738a44973e7a05e9423c32d813af2b0\", \"key\": 0, \"line\": 9510, \"relation\": \"increases\", \"source\": 174, \"target\": 13}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Nuclear factor Kappa beta subgraph\": true, \"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2010 May;113(3):772-83. Epub 2010 Feb 17\", \"reference\": \"20180883\", \"type\": \"PubMed\"}, \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"hash\": \"00209c4e181c532f9073d01d4ba5f7df2f4ad05ebfd71e46dce0447f69e4e8a91920c2300bfd0cb457b41f471ff211bf6cbc32649879043a523fcc9dde34e5ae\", \"key\": 0, \"line\": 4367, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 175, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 8}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Nuclear factor Kappa beta subgraph\": true, \"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neurochem. 2010 May;113(3):772-83. Epub 2010 Feb 17\", \"reference\": \"20180883\", \"type\": \"PubMed\"}, \"evidence\": \"Nuclear factor-kappaB activation regulates cyclooxygenase-2 induction in human astrocytes in response to CXCL12: role in neuronal toxicity\", \"hash\": \"d397462f1ee2b75c9bbcbba6db925f3e2f36c645a370f0a127039ffa0b71c52e409903a1fa5e5a6244d72fb06066528510b0382ebcdf166fb3237ed4ccf7e46f\", \"key\": 1, \"line\": 34676, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 175, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"d401e3b4b7216c9fa75862d63b9ef6572b585dff01006c163e48f838631f38b1874e58a6e4858cb70b73c251f2cd86040b835f5c89b99f40f3c40cf23e9c58c7\", \"key\": 0, \"line\": 46329, \"relation\": \"decreases\", \"source\": 176, \"target\": 10}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"38de5edd2c288acb23f9718ca330fd4bf90b6abc66a00ae0ba0e4d237f225de5249117e0aa62d764ed36c4d3246695bbafb21da20cc53b4ed742f1de6a486545\", \"key\": 0, \"line\": 24174, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 177, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"14c4843a13056aa43d6ea7c07bb50205c81ed90f5b8c12430a8eb4d880a28341de5dbe6d47afebfa81c0eb4cc58f160dd8f4377d7fa239c9aa7e0fd6b698959c\", \"key\": 1, \"line\": 24766, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 177, \"target\": 14}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Innate immune system subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2012 Feb;33(2):422.e11-25. doi: 10.1016/j.neurobiolaging.2010.09.017. Epub 2010 Nov 26.\", \"reference\": \"21112127\", \"type\": \"PubMed\"}, \"evidence\": \"CCAAT/enhancer binding protein delta (CEBPD) elevating PTX3 expression inhibits macrophage-mediated phagocytosis of dying neuron cells.\", \"hash\": \"48d1226b5064b2e17888469f12ade18b8b616edf2f65f063b2624b6b6d5c7198a3126e6bdd05c9041e9bd16f4d9bb54409a2cfd3c78138129e5bd19df2ded367\", \"key\": 0, \"line\": 37938, \"relation\": \"increases\", \"source\": 178, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"JAK-STAT signaling subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes\", \"hash\": \"08990abbbc73b730bff8703f1fc857fa53c469b0b9b736a21410bbc7f678c93ea17f0e4005caf34755ef4cde22f92d390d3e21839f1f756cc5366334a0268473\", \"key\": 0, \"line\": 46220, \"relation\": \"increases\", \"source\": 179, \"target\": 33}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"The secreted protein, YKL-40, has been proposed as a biomarker of a variety of human diseases characterized by ongoing inflammation, including chronic neurologic pathologies such as multiple sclerosis and Alzheimer's disease. However, inflammatory mediators and the molecular mechanism responsible for enhanced expression of YKL-40 remained elusive. Using several mouse models of inflammation, we now show that YKL-40 expression correlated with increased expression of both IL-1 and IL-6. Furthermore, IL-1 together with IL-6 or the IL-6 family cytokine, oncostatin M, synergistically upregulated YKL-40 expression in both primary human and mouse astrocytes in vitro. The robust cytokine-driven expression of YKL-40 in astrocytes required both STAT3 and NF-kB binding elements of the YKL-40 promoter. In addition, YKL-40 expression was enhanced by constitutively active STAT3 and inhibited by dominant-negative IkBalpha. Surprisingly, cytokine-driven expression of YKL-40 in astrocytes was independent of the p65 subunit of NF-kB and instead required subunits RelB and p50. Mechanistically, we show that IL-1-induced RelB/p50 complex formation was further promoted by oncostatin M and that these complexes directly bound to the YKL-40 promoter. Moreover, we found that expression of RelB was strongly upregulated during inflammation in vivo and by IL-1 in astrocytes in vitro. We propose that IL-1 and the IL-6 family of cytokines regulate YKL-40 expression during sterile inflammation via both STAT3 and RelB/p50 complexes. These results suggest that IL-1 may regulate the expression of specific anti-inflammatory genes in nonlymphoid tissues via the canonical activation of the RelB/p50 complexes.\", \"hash\": \"cbe082666c6e87ec7638defe84f04e610e595bec8665901537ae6686d625cf3ad415fbb7626e8917936b0fdb808368b1da52a892cb81035448cb9c6f97ee01ae\", \"key\": 0, \"line\": 43111, \"relation\": \"increases\", \"source\": 180, \"target\": 33}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, IL-6 and OSM moderately upregulate YKL-40 expression in human astrocytes [...] demonstrate that YKL-40 expression correlates with the expression of both IL-1 and IL-6\", \"hash\": \"8ba58abb8dd30d6dee12303cdc244770413772a547551ef38c36bc20254307ec79b9ce9ed02c1b18dc48f42a841f1347dd3c04f8565725ee6241ac479ddb0511\", \"key\": 1, \"line\": 46233, \"relation\": \"increases\", \"source\": 180, \"target\": 33}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"leukotriene\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"], \"name\": \"leukotriene\", \"namespace\": \"CHEBI\"}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"Neurofibrillary Tangles\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"], \"name\": \"Neurofibrillary Tangles\", \"namespace\": \"MESHCS\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"PTGS2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTGS2\"], \"name\": \"PTGS2\", \"namespace\": \"HGNC\", \"weight\": -1.07023648}, {\"cname\": \"CX3CR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CX3CR1\"], \"name\": \"CX3CR1\", \"namespace\": \"HGNC\", \"weight\": -0.81994598}, {\"cname\": \"TREM2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TREM2\"], \"name\": \"TREM2\", \"namespace\": \"HGNC\", \"weight\": -0.40134828}, {\"cname\": \"CFH\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CFH\"], \"name\": \"CFH\", \"namespace\": \"HGNC\", \"weight\": 1.86557178}, {\"cname\": \"IRAK1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IRAK1\"], \"name\": \"IRAK1\", \"namespace\": \"HGNC\", \"weight\": 0.72985984}, {\"cname\": \"IRAK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IRAK2\"], \"name\": \"IRAK2\", \"namespace\": \"HGNC\", \"weight\": -0.26455035}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"PLA2G4A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G4A\"], \"name\": \"PLA2G4A\", \"namespace\": \"HGNC\", \"weight\": 0.52335304}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"MT-CO2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MT-CO2\"], \"name\": \"MT-CO2\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"PLA2G7\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLA2G7\"], \"name\": \"PLA2G7\", \"namespace\": \"HGNC\", \"weight\": -0.52445663}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}, {\"cname\": \"CHI3L1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHI3L1\"], \"name\": \"CHI3L1\", \"namespace\": \"HGNC\", \"weight\": 0.14672037}, {\"cname\": \"TREM1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TREM1\"], \"name\": \"TREM1\", \"namespace\": \"HGNC\", \"weight\": 1.13272121}, {\"cname\": \"g(HGNC:APP, var(c.275341G>C))\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.275341G>C\"]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": \"c.275341G>C\", \"kind\": \"hgvs\"}], \"weight\": -0.13701466}, {\"cname\": \"g(HGNC:APP, var(c.717G>C))\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.717G>C\"]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": \"c.717G>C\", \"kind\": \"hgvs\"}], \"weight\": -0.13701466}, {\"cname\": \"g(HGNC:CTSD, var(c.224C>T))\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\", [\"hgvs\", \"c.224C>T\"]], \"name\": \"CTSD\", \"namespace\": \"HGNC\", \"variants\": [{\"identifier\": \"c.224C>T\", \"kind\": \"hgvs\"}], \"weight\": 0.88300839}, {\"cname\": \"APOE\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APOE\"], \"name\": \"APOE\", \"namespace\": \"HGNC\", \"weight\": 0.04406715}, {\"cname\": \"APP\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APP\"], \"name\": \"APP\", \"namespace\": \"HGNC\", \"weight\": -0.13701466}, {\"cname\": \"PSEN1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PSEN1\"], \"name\": \"PSEN1\", \"namespace\": \"HGNC\", \"weight\": 0.00266298}, {\"cname\": \"PSEN2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PSEN2\"], \"name\": \"PSEN2\", \"namespace\": \"HGNC\", \"weight\": -0.02429789}, {\"cname\": \"PPARG\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PPARG\"], \"name\": \"PPARG\", \"namespace\": \"HGNC\", \"weight\": -0.06808657}, {\"cname\": \"RTN4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"RTN4\"], \"name\": \"RTN4\", \"namespace\": \"HGNC\", \"weight\": 0.11441}, {\"cname\": \"IDE\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IDE\"], \"name\": \"IDE\", \"namespace\": \"HGNC\", \"weight\": 0.28059224}, {\"cname\": \"MME\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MME\"], \"name\": \"MME\", \"namespace\": \"HGNC\", \"weight\": 0.036129400000000006}, {\"cname\": \"BDNF\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"BDNF\"], \"name\": \"BDNF\", \"namespace\": \"HGNC\", \"weight\": -0.39074377}, {\"cname\": \"SORL1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SORL1\"], \"name\": \"SORL1\", \"namespace\": \"HGNC\", \"weight\": -0.01427188}, {\"cname\": \"SORCS1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SORCS1\"], \"name\": \"SORCS1\", \"namespace\": \"HGNC\", \"weight\": -1.5152121}, {\"cname\": \"EGR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EGR1\"], \"name\": \"EGR1\", \"namespace\": \"HGNC\", \"weight\": -0.31091638}, {\"cname\": \"NAB2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NAB2\"], \"name\": \"NAB2\", \"namespace\": \"HGNC\", \"weight\": 0.33700701}, {\"cname\": \"KLF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"KLF1\"], \"name\": \"KLF1\", \"namespace\": \"HGNC\", \"weight\": -0.77844709}, {\"cname\": \"CTSD\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\"], \"name\": \"CTSD\", \"namespace\": \"HGNC\", \"weight\": 0.88300839}, {\"cname\": \"CR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CR1\"], \"name\": \"CR1\", \"namespace\": \"HGNC\", \"weight\": 0.02399208}, {\"cname\": \"CST3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CST3\"], \"name\": \"CST3\", \"namespace\": \"HGNC\", \"weight\": -0.03303922}, {\"cname\": \"CLU\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CLU\"], \"name\": \"CLU\", \"namespace\": \"HGNC\", \"weight\": 0.4376103}, {\"cname\": \"PICALM\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PICALM\"], \"name\": \"PICALM\", \"namespace\": \"HGNC\", \"weight\": -0.14693108}, {\"cname\": \"ABCA7\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ABCA7\"], \"name\": \"ABCA7\", \"namespace\": \"HGNC\", \"weight\": -0.98234365}, {\"cname\": \"GSK3A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3A\"], \"name\": \"GSK3A\", \"namespace\": \"HGNC\", \"weight\": -0.45894849}, {\"cname\": \"LRP2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"LRP2\"], \"name\": \"LRP2\", \"namespace\": \"HGNC\", \"weight\": -0.13257198}, {\"cname\": \"ACHE\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ACHE\"], \"name\": \"ACHE\", \"namespace\": \"HGNC\", \"weight\": 0.49236940000000007}, {\"cname\": \"INSR\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"INSR\"], \"name\": \"INSR\", \"namespace\": \"HGNC\", \"weight\": 0.049953899999999996}, {\"cname\": \"NOS3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOS3\"], \"name\": \"NOS3\", \"namespace\": \"HGNC\", \"weight\": -0.1714975}, {\"cname\": \"BACE1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"BACE1\"], \"name\": \"BACE1\", \"namespace\": \"HGNC\", \"weight\": -0.00189721}, {\"cname\": \"INS\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"INS\"], \"name\": \"INS\", \"namespace\": \"HGNC\", \"weight\": 0.99910812}, {\"cname\": \"STK11\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"STK11\"], \"name\": \"STK11\", \"namespace\": \"HGNC\", \"weight\": 0.17049337}, {\"cname\": \"SPTLC1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC1\"], \"name\": \"SPTLC1\", \"namespace\": \"HGNC\", \"weight\": -0.0808145}, {\"cname\": \"GRIN3A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN3A\"], \"name\": \"GRIN3A\", \"namespace\": \"HGNC\", \"weight\": -0.01551801}, {\"cname\": \"APBA1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APBA1\"], \"name\": \"APBA1\", \"namespace\": \"HGNC\", \"weight\": -0.48809369999999996}, {\"cname\": \"PSENEN\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PSENEN\"], \"name\": \"PSENEN\", \"namespace\": \"HGNC\", \"weight\": 1.24788008}, {\"cname\": \"CCR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CCR2\"], \"name\": \"CCR2\", \"namespace\": \"HGNC\", \"weight\": 0.72408115}, {\"cname\": \"LRP1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"LRP1\"], \"name\": \"LRP1\", \"namespace\": \"HGNC\", \"weight\": 0.39313774}, {\"cname\": \"BACE2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"BACE2\"], \"name\": \"BACE2\", \"namespace\": \"HGNC\", \"weight\": -0.34884808}, {\"cname\": \"IGF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IGF1\"], \"name\": \"IGF1\", \"namespace\": \"HGNC\", \"weight\": 0.98026104}, {\"cname\": \"RELA\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"RELA\"], \"name\": \"RELA\", \"namespace\": \"HGNC\", \"weight\": 0.39927783}, {\"cname\": \"PIN1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PIN1\"], \"name\": \"PIN1\", \"namespace\": \"HGNC\", \"weight\": -0.25496963}, {\"cname\": \"CAST\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CAST\"], \"name\": \"CAST\", \"namespace\": \"HGNC\", \"weight\": 0.03364871}, {\"cname\": \"NFKB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NFKB1\"], \"name\": \"NFKB1\", \"namespace\": \"HGNC\", \"weight\": -1.33219344}, {\"cname\": \"DKK1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"DKK1\"], \"name\": \"DKK1\", \"namespace\": \"HGNC\", \"weight\": -1.58173755}, {\"cname\": \"DHCR24\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"DHCR24\"], \"name\": \"DHCR24\", \"namespace\": \"HGNC\", \"weight\": -0.17838582}, {\"cname\": \"RAB10\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"RAB10\"], \"name\": \"RAB10\", \"namespace\": \"HGNC\", \"weight\": 0.07462024}, {\"cname\": \"MIR124-1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR124-1\"], \"name\": \"MIR124-1\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"IL1B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL1B\"], \"name\": \"IL1B\", \"namespace\": \"HGNC\", \"weight\": -1.63771704}, {\"cname\": \"ATXN1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ATXN1\"], \"name\": \"ATXN1\", \"namespace\": \"HGNC\", \"weight\": 0.00954063}, {\"cname\": \"SPTLC3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC3\"], \"name\": \"SPTLC3\", \"namespace\": \"HGNC\", \"weight\": 0.19053622}, {\"cname\": \"ADAM10\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ADAM10\"], \"name\": \"ADAM10\", \"namespace\": \"HGNC\", \"weight\": -0.051464199999999995}, {\"cname\": \"FTL\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FTL\"], \"name\": \"FTL\", \"namespace\": \"HGNC\", \"weight\": 0.70775495}, {\"cname\": \"MIR195\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR195\"], \"name\": \"MIR195\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"PLG\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLG\"], \"name\": \"PLG\", \"namespace\": \"HGNC\", \"weight\": 0.68395652}, {\"cname\": \"DDIT3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"DDIT3\"], \"name\": \"DDIT3\", \"namespace\": \"HGNC\", \"weight\": 1.40355466}, {\"cname\": \"CLSTN1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CLSTN1\"], \"name\": \"CLSTN1\", \"namespace\": \"HGNC\", \"weight\": 0.38076231}, {\"cname\": \"KLC1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"KLC1\"], \"name\": \"KLC1\", \"namespace\": \"HGNC\", \"weight\": -0.09664774}, {\"cname\": \"APH1A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APH1A\"], \"name\": \"APH1A\", \"namespace\": \"HGNC\", \"weight\": -0.42715932}, {\"cname\": \"GPR3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GPR3\"], \"name\": \"GPR3\", \"namespace\": \"HGNC\", \"weight\": 0.14999666}, {\"cname\": \"A2M\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"A2M\"], \"name\": \"A2M\", \"namespace\": \"HGNC\", \"weight\": 1.08160069}, {\"cname\": \"CASP8\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CASP8\"], \"name\": \"CASP8\", \"namespace\": \"HGNC\", \"weight\": -0.15029163}, {\"cname\": \"MTRNR2L2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MTRNR2L2\"], \"name\": \"MTRNR2L2\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"CRH\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CRH\"], \"name\": \"CRH\", \"namespace\": \"HGNC\", \"weight\": -0.2141471}, {\"cname\": \"HSPB8\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"HSPB8\"], \"name\": \"HSPB8\", \"namespace\": \"HGNC\", \"weight\": 0.39864903}, {\"cname\": \"APBB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APBB1\"], \"name\": \"APBB1\", \"namespace\": \"HGNC\", \"weight\": 0.59661584}, {\"cname\": \"APBA2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APBA2\"], \"name\": \"APBA2\", \"namespace\": \"HGNC\", \"weight\": -0.19241516}, {\"cname\": \"APBA3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APBA3\"], \"name\": \"APBA3\", \"namespace\": \"HGNC\", \"weight\": -1.34229521}, {\"cname\": \"LDLR\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"LDLR\"], \"name\": \"LDLR\", \"namespace\": \"HGNC\", \"weight\": -0.01435724}, {\"cname\": \"GRIN2B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2B\"], \"name\": \"GRIN2B\", \"namespace\": \"HGNC\", \"weight\": -0.10430164}, {\"cname\": \"GRIN2A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2A\"], \"name\": \"GRIN2A\", \"namespace\": \"HGNC\", \"weight\": -0.29373059999999995}, {\"cname\": \"SPTLC2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SPTLC2\"], \"name\": \"SPTLC2\", \"namespace\": \"HGNC\", \"weight\": -0.06391096}, {\"cname\": \"NAT8B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NAT8B\"], \"name\": \"NAT8B\", \"namespace\": \"HGNC\", \"weight\": 0.20466742}, {\"cname\": \"GGA3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GGA3\"], \"name\": \"GGA3\", \"namespace\": \"HGNC\", \"weight\": -0.81325519}, {\"cname\": \"PRNP\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PRNP\"], \"name\": \"PRNP\", \"namespace\": \"HGNC\", \"weight\": 0.08884805}, {\"cname\": \"CHRNA4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA4\"], \"name\": \"CHRNA4\", \"namespace\": \"HGNC\", \"weight\": 0.57478705}, {\"cname\": \"ABCA2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ABCA2\"], \"name\": \"ABCA2\", \"namespace\": \"HGNC\", \"weight\": -0.03926027}, {\"cname\": \"TGFB2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TGFB2\"], \"name\": \"TGFB2\", \"namespace\": \"HGNC\", \"weight\": 0.26765361}, {\"cname\": \"MARCKS\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MARCKS\"], \"name\": \"MARCKS\", \"namespace\": \"HGNC\", \"weight\": 0.07542072}, {\"cname\": \"GRIN2C\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2C\"], \"name\": \"GRIN2C\", \"namespace\": \"HGNC\", \"weight\": 0.15747839}, {\"cname\": \"GRIN2D\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN2D\"], \"name\": \"GRIN2D\", \"namespace\": \"HGNC\", \"weight\": -0.44105144}, {\"cname\": \"GRIN3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRIN3B\"], \"name\": \"GRIN3B\", \"namespace\": \"HGNC\", \"weight\": 0.55994342}, {\"cname\": \"IL34\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL34\"], \"name\": \"IL34\", \"namespace\": \"HGNC\", \"weight\": -0.18066962}, {\"cname\": \"SOD1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SOD1\"], \"name\": \"SOD1\", \"namespace\": \"HGNC\", \"weight\": -0.05539543}, {\"cname\": \"ADAM9\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ADAM9\"], \"name\": \"ADAM9\", \"namespace\": \"HGNC\", \"weight\": -0.16119139}, {\"cname\": \"ADAM17\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ADAM17\"], \"name\": \"ADAM17\", \"namespace\": \"HGNC\", \"weight\": 0.21966586}, {\"cname\": \"CTSB\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CTSB\"], \"name\": \"CTSB\", \"namespace\": \"HGNC\", \"weight\": -0.04291486}, {\"cname\": \"CTSE\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CTSE\"], \"name\": \"CTSE\", \"namespace\": \"HGNC\", \"weight\": -1.2097365}, {\"cname\": \"GCG\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GCG\"], \"name\": \"GCG\", \"namespace\": \"HGNC\", \"weight\": -0.03157631}, {\"cname\": \"ECE1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ECE1\"], \"name\": \"ECE1\", \"namespace\": \"HGNC\", \"weight\": -0.09027805}, {\"cname\": \"ACE\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ACE\"], \"name\": \"ACE\", \"namespace\": \"HGNC\", \"weight\": 0.38262923}, {\"cname\": \"DPP4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"DPP4\"], \"name\": \"DPP4\", \"namespace\": \"HGNC\", \"weight\": 0.17409829999999998}, {\"cname\": \"BLMH\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"BLMH\"], \"name\": \"BLMH\", \"namespace\": \"HGNC\", \"weight\": -0.85202484}, {\"cname\": \"ALB\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ALB\"], \"name\": \"ALB\", \"namespace\": \"HGNC\", \"weight\": 0.02354135}, {\"cname\": \"LEP\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"LEP\"], \"name\": \"LEP\", \"namespace\": \"HGNC\", \"weight\": 1.01085844}, {\"cname\": \"CHRNA2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA2\"], \"name\": \"CHRNA2\", \"namespace\": \"HGNC\", \"weight\": -0.15115052}, {\"cname\": \"CHRNA3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA3\"], \"name\": \"CHRNA3\", \"namespace\": \"HGNC\", \"weight\": -0.6279239999999999}, {\"cname\": \"CHRNA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA5\"], \"name\": \"CHRNA5\", \"namespace\": \"HGNC\", \"weight\": 0.4580624}, {\"cname\": \"CHRNA6\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA6\"], \"name\": \"CHRNA6\", \"namespace\": \"HGNC\", \"weight\": -1.13569155}, {\"cname\": \"CHRNA7\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA7\"], \"name\": \"CHRNA7\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"CHRNA9\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA9\"], \"name\": \"CHRNA9\", \"namespace\": \"HGNC\", \"weight\": 0.53800658}, {\"cname\": \"CHRNA10\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA10\"], \"name\": \"CHRNA10\", \"namespace\": \"HGNC\", \"weight\": -0.44528092}, {\"cname\": \"GSAP\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSAP\"], \"name\": \"GSAP\", \"namespace\": \"HGNC\", \"weight\": -0.44248455}, {\"cname\": \"MMP9\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MMP9\"], \"name\": \"MMP9\", \"namespace\": \"HGNC\", \"weight\": -0.32509715}, {\"cname\": \"MMP2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MMP2\"], \"name\": \"MMP2\", \"namespace\": \"HGNC\", \"weight\": -0.47336532}, {\"cname\": \"BCL2L1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"BCL2L1\"], \"name\": \"BCL2L1\", \"namespace\": \"HGNC\", \"weight\": -0.29881302}, {\"cname\": \"ABCA1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ABCA1\"], \"name\": \"ABCA1\", \"namespace\": \"HGNC\", \"weight\": 0.17888996}, {\"cname\": \"MMP7\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MMP7\"], \"name\": \"MMP7\", \"namespace\": \"HGNC\", \"weight\": -1.19454136}, {\"cname\": \"PLAT\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLAT\"], \"name\": \"PLAT\", \"namespace\": \"HGNC\", \"weight\": 1.04346523}, {\"cname\": \"SERPINI1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SERPINI1\"], \"name\": \"SERPINI1\", \"namespace\": \"HGNC\", \"weight\": -1.09712643}, {\"cname\": \"PLAU\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PLAU\"], \"name\": \"PLAU\", \"namespace\": \"HGNC\", \"weight\": -0.44174472}, {\"cname\": \"TTR\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TTR\"], \"name\": \"TTR\", \"namespace\": \"HGNC\", \"weight\": 4.07054127}, {\"cname\": \"IL4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL4\"], \"name\": \"IL4\", \"namespace\": \"HGNC\", \"weight\": 0.91448898}, {\"cname\": \"APPBP2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"APPBP2\"], \"name\": \"APPBP2\", \"namespace\": \"HGNC\", \"weight\": -0.01975785}, {\"cname\": \"PAWR\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PAWR\"], \"name\": \"PAWR\", \"namespace\": \"HGNC\", \"weight\": -0.17425334}, {\"cname\": \"CSNK1A1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CSNK1A1\"], \"name\": \"CSNK1A1\", \"namespace\": \"HGNC\", \"weight\": -0.09186504}, {\"cname\": \"QPCT\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"QPCT\"], \"name\": \"QPCT\", \"namespace\": \"HGNC\", \"weight\": 0.51242099}, {\"cname\": \"UBE2I\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"UBE2I\"], \"name\": \"UBE2I\", \"namespace\": \"HGNC\", \"weight\": -0.15740177}, {\"cname\": \"RANBP9\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"RANBP9\"], \"name\": \"RANBP9\", \"namespace\": \"HGNC\", \"weight\": -0.20181786}, {\"cname\": \"CAPN2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CAPN2\"], \"name\": \"CAPN2\", \"namespace\": \"HGNC\", \"weight\": -0.02208264}, {\"cname\": \"LRRTM3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"LRRTM3\"], \"name\": \"LRRTM3\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"RTN4R\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"RTN4R\"], \"name\": \"RTN4R\", \"namespace\": \"HGNC\", \"weight\": -1.01416883}, {\"cname\": \"HSPA1B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"HSPA1B\"], \"name\": \"HSPA1B\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"CHRNA1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CHRNA1\"], \"name\": \"CHRNA1\", \"namespace\": \"HGNC\", \"weight\": 0.56461989}, {\"cname\": \"SP1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SP1\"], \"name\": \"SP1\", \"namespace\": \"HGNC\", \"weight\": -0.08678866}, {\"cname\": \"S100A9\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"S100A9\"], \"name\": \"S100A9\", \"namespace\": \"HGNC\", \"weight\": -0.60784743}, {\"cname\": \"MAPK8IP2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MAPK8IP2\"], \"name\": \"MAPK8IP2\", \"namespace\": \"HGNC\", \"weight\": -0.07187538}, {\"cname\": \"ABCG1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ABCG1\"], \"name\": \"ABCG1\", \"namespace\": \"HGNC\", \"weight\": -0.17414933}, {\"cname\": \"CPQ\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CPQ\"], \"name\": \"CPQ\", \"namespace\": \"HGNC\", \"weight\": 0.45456024}, {\"cname\": \"HTR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"HTR4\"], \"name\": \"HTR4\", \"namespace\": \"HGNC\", \"weight\": -0.15785455}, {\"cname\": \"ALOX5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"ALOX5\"], \"name\": \"ALOX5\", \"namespace\": \"HGNC\", \"weight\": -0.188697}, {\"cname\": \"MIR125B1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR125B1\"], \"name\": \"MIR125B1\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"MIR125B2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR125B2\"], \"name\": \"MIR125B2\", \"namespace\": \"HGNC\", \"weight\": 0}, {\"cname\": \"S100B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"S100B\"], \"name\": \"S100B\", \"namespace\": \"HGNC\", \"weight\": 0.27537864}, {\"cname\": \"MMP3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MMP3\"], \"name\": \"MMP3\", \"namespace\": \"HGNC\", \"weight\": 0.78577944}, {\"cname\": \"GRB2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GRB2\"], \"name\": \"GRB2\", \"namespace\": \"HGNC\", \"weight\": -0.02552084}, {\"cname\": \"CTNNB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CTNNB1\"], \"name\": \"CTNNB1\", \"namespace\": \"HGNC\", \"weight\": 0.03837192}, {\"cname\": \"PIK3R5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PIK3R5\"], \"name\": \"PIK3R5\", \"namespace\": \"HGNC\", \"weight\": -0.23683106}, {\"cname\": \"PPP2CA\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PPP2CA\"], \"name\": \"PPP2CA\", \"namespace\": \"HGNC\", \"weight\": -0.20521612}, {\"cname\": \"FRAT2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FRAT2\"], \"name\": \"FRAT2\", \"namespace\": \"HGNC\", \"weight\": -0.50393273}, {\"cname\": \"MIR146A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR146A\"], \"name\": \"MIR146A\", \"namespace\": \"HGNC\", \"weight\": -0.53979604}, {\"cname\": \"NFKBIA\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NFKBIA\"], \"name\": \"NFKBIA\", \"namespace\": \"HGNC\", \"weight\": 0.36591919}, {\"cname\": \"MIR34A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MIR34A\"], \"name\": \"MIR34A\", \"namespace\": \"HGNC\", \"weight\": 0.58758949}, {\"cname\": \"HMGB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"HMGB1\"], \"name\": \"HMGB1\", \"namespace\": \"HGNC\", \"weight\": -0.09432351}, {\"cname\": \"CEBPD\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CEBPD\"], \"name\": \"CEBPD\", \"namespace\": \"HGNC\", \"weight\": 0.47557886}, {\"cname\": \"STAT3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"STAT3\"], \"name\": \"STAT3\", \"namespace\": \"HGNC\", \"weight\": 0.04651478}, {\"cname\": \"OSM\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"OSM\"], \"name\": \"OSM\", \"namespace\": \"HGNC\", \"weight\": 0.64942734}]}, \"#mkudpqcofahtyrge\", 1000, 650, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This entire procedure can be run automatically with [pybel_tools.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Running"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "runner = Runner(subgraph, inflamation_node)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "While the algorithm can be immediately run with [pybel_tools.analysis.heat.Runner.run](http://pybel-tools.readthedocs.io/en/latest/heat.html#pybel_tools.analysis.heat.Runner.run), the process is outlined to show the evolution of the graph throughouts its steps. For animation purposes, the functin [pybel_tools.analysis.heat.Runner.run_with_graph_transformation](http://pybel-tools.readthedocs.io/en/latest/heat.html#pybel_tools.analysis.heat.Runner.run_with_graph_transformation) yields a new `BELGraph` at each step through the process as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Leaf: g(HGNC:TREM2)\n",
      "Leaf: g(HGNC:IRAK1)\n",
      "Leaf: g(HGNC:PTGS2)\n",
      "Leaf: a(MESHCS:\"Neurofibrillary Tangles\")\n",
      "Leaf: g(HGNC:CHI3L1)\n",
      "Leaf: g(HGNC:CFH)\n",
      "Leaf: a(CHEBI:leukotriene)\n",
      "Leaf: g(HGNC:IRAK2)\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='dtzjxqobeyqvgrmp'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 14}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 2}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"b097a471f1fb37c8206da83f8cf86668534b3469343940202384cda9e85acb294bb50b9c69df75eb070f8392a22732f08cbce6fb1c38da3372e7e0fb62533325\", \"key\": 0, \"line\": 42188, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1993 Dec;124(2):289-98.\", \"reference\": \"8287928\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"hash\": \"ad1f29aa4d840d0ddc1e3d8a3da37591bcca66d1a611768e9d74312f777831f3a22c72a324e8b244eb9b2b7cc5dd9d31937cd75ef2691979fbb2e4738b8a2517\", \"key\": 1, \"line\": 42344, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 21}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 4, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"4f0cba06b674c0e9847c0fb9884cf859ca04ba644c3c6b28c077cfa4cf1a974bb209c0cdda26733a5f6ada7743d7cff85624c046373e88f3e78122b2cf2fefdd\", \"key\": 0, \"line\": 3554, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"hash\": \"1e78a3aea456a94331af060b6acca92b9fdf5ce40ee68f9a2179f513be1a8ceda93198b68dd0e65d3bfe46b251ef6be30b4738befcf651d1ad9812d63050d6ff\", \"key\": 1, \"line\": 6181, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"fecdb06cc63b0946512b0d57bf1de37ccecd3633343e9cdb5671d07531cbc04802c0ae1a71e6534814b9ee6f0ce399b25ebf475f8dc97d2a63c0c8b9caf93358\", \"key\": 2, \"line\": 35325, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of neuroinflammation; Vol. 11; Iss. \", \"reference\": \"24694234\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid- (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"hash\": \"44f25dfab4dbfc410250e4ab86280e73c9b1bfcfbc6522fde6dc3387fbd91dd4570fd4d1f31c89c05cfe99ff3bb39aa0088f31cb1af2aabb632aab2a044978db\", \"key\": 3, \"line\": 39478, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"5223c465d4674837b2596c4aba54d38ff3a70148ac3773bdea1fef6a41b73da935452e9285019f43c1a992ce58915bcc5d5dc6f7ce82b9bb0cfa7a70782bde87\", \"key\": 4, \"line\": 42189, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\", \"type\": \"PubMed\"}, \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"hash\": \"8c09ee5bf342ee7449995131e6d0c4db263272a6d2dd2bb358cc00c503190f0fa93ad93e22d2bb631fffcc0aa52ea8037aa9b3ba2ed283b1b32e1d60e75bac8c\", \"key\": 0, \"line\": 4875, \"relation\": \"increases\", \"source\": 4, \"target\": 13}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 5, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 8, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 9, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 11, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 13, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 15, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 17, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 18, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 20, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 21, \"target\": 0}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}]}, \"#dtzjxqobeyqvgrmp\", 1000, 400, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "leaves = runner.score_leaves()\n",
    "\n",
    "for leaf in leaves:\n",
    "    print(f\"Leaf: {graph.node_to_bel(leaf)}\")\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "no leaves.\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='ybjfvhurdgpoczts'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 14}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 2}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"b097a471f1fb37c8206da83f8cf86668534b3469343940202384cda9e85acb294bb50b9c69df75eb070f8392a22732f08cbce6fb1c38da3372e7e0fb62533325\", \"key\": 0, \"line\": 42188, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1993 Dec;124(2):289-98.\", \"reference\": \"8287928\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"hash\": \"ad1f29aa4d840d0ddc1e3d8a3da37591bcca66d1a611768e9d74312f777831f3a22c72a324e8b244eb9b2b7cc5dd9d31937cd75ef2691979fbb2e4738b8a2517\", \"key\": 1, \"line\": 42344, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 21}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 4, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"4f0cba06b674c0e9847c0fb9884cf859ca04ba644c3c6b28c077cfa4cf1a974bb209c0cdda26733a5f6ada7743d7cff85624c046373e88f3e78122b2cf2fefdd\", \"key\": 0, \"line\": 3554, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"hash\": \"1e78a3aea456a94331af060b6acca92b9fdf5ce40ee68f9a2179f513be1a8ceda93198b68dd0e65d3bfe46b251ef6be30b4738befcf651d1ad9812d63050d6ff\", \"key\": 1, \"line\": 6181, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"fecdb06cc63b0946512b0d57bf1de37ccecd3633343e9cdb5671d07531cbc04802c0ae1a71e6534814b9ee6f0ce399b25ebf475f8dc97d2a63c0c8b9caf93358\", \"key\": 2, \"line\": 35325, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of neuroinflammation; Vol. 11; Iss. \", \"reference\": \"24694234\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid- (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"hash\": \"44f25dfab4dbfc410250e4ab86280e73c9b1bfcfbc6522fde6dc3387fbd91dd4570fd4d1f31c89c05cfe99ff3bb39aa0088f31cb1af2aabb632aab2a044978db\", \"key\": 3, \"line\": 39478, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"5223c465d4674837b2596c4aba54d38ff3a70148ac3773bdea1fef6a41b73da935452e9285019f43c1a992ce58915bcc5d5dc6f7ce82b9bb0cfa7a70782bde87\", \"key\": 4, \"line\": 42189, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\", \"type\": \"PubMed\"}, \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"hash\": \"8c09ee5bf342ee7449995131e6d0c4db263272a6d2dd2bb358cc00c503190f0fa93ad93e22d2bb631fffcc0aa52ea8037aa9b3ba2ed283b1b32e1d60e75bac8c\", \"key\": 0, \"line\": 4875, \"relation\": \"increases\", \"source\": 4, \"target\": 13}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 5, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 8, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 9, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 11, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 13, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 15, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 17, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 18, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 20, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 21, \"target\": 0}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}]}, \"#ybjfvhurdgpoczts\", 1000, 400, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "leaves = runner.score_leaves()\n",
    "\n",
    "for leaf in leaves:\n",
    "    print(f\"Leaf: {graph.node_to_bel(leaf)}\")\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "no leaves.\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='dkvgqxzpahbmtsjq'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 14}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 2}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"b097a471f1fb37c8206da83f8cf86668534b3469343940202384cda9e85acb294bb50b9c69df75eb070f8392a22732f08cbce6fb1c38da3372e7e0fb62533325\", \"key\": 0, \"line\": 42188, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Autophagy signaling subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1993 Dec;124(2):289-98.\", \"reference\": \"8287928\", \"type\": \"PubMed\"}, \"evidence\": \"beta-Amyloid of Alzheimer's disease induces reactive gliosis that inhibits axonal outgrowth.\", \"hash\": \"ad1f29aa4d840d0ddc1e3d8a3da37591bcca66d1a611768e9d74312f777831f3a22c72a324e8b244eb9b2b7cc5dd9d31937cd75ef2691979fbb2e4738b8a2517\", \"key\": 1, \"line\": 42344, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 21}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 4, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"4f0cba06b674c0e9847c0fb9884cf859ca04ba644c3c6b28c077cfa4cf1a974bb209c0cdda26733a5f6ada7743d7cff85624c046373e88f3e78122b2cf2fefdd\", \"key\": 0, \"line\": 3554, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"hash\": \"1e78a3aea456a94331af060b6acca92b9fdf5ce40ee68f9a2179f513be1a8ceda93198b68dd0e65d3bfe46b251ef6be30b4738befcf651d1ad9812d63050d6ff\", \"key\": 1, \"line\": 6181, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"fecdb06cc63b0946512b0d57bf1de37ccecd3633343e9cdb5671d07531cbc04802c0ae1a71e6534814b9ee6f0ce399b25ebf475f8dc97d2a63c0c8b9caf93358\", \"key\": 2, \"line\": 35325, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of neuroinflammation; Vol. 11; Iss. \", \"reference\": \"24694234\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid- (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"hash\": \"44f25dfab4dbfc410250e4ab86280e73c9b1bfcfbc6522fde6dc3387fbd91dd4570fd4d1f31c89c05cfe99ff3bb39aa0088f31cb1af2aabb632aab2a044978db\", \"key\": 3, \"line\": 39478, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"5223c465d4674837b2596c4aba54d38ff3a70148ac3773bdea1fef6a41b73da935452e9285019f43c1a992ce58915bcc5d5dc6f7ce82b9bb0cfa7a70782bde87\", \"key\": 4, \"line\": 42189, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\", \"type\": \"PubMed\"}, \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"hash\": \"8c09ee5bf342ee7449995131e6d0c4db263272a6d2dd2bb358cc00c503190f0fa93ad93e22d2bb631fffcc0aa52ea8037aa9b3ba2ed283b1b32e1d60e75bac8c\", \"key\": 0, \"line\": 4875, \"relation\": \"increases\", \"source\": 4, \"target\": 13}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 5, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 8, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 9, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 11, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 13, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 15, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 17, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 18, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 20, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 21, \"target\": 0}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}]}, \"#dkvgqxzpahbmtsjq\", 1000, 400, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "leaves = runner.score_leaves()\n",
    "\n",
    "for leaf in leaves:\n",
    "    print(f\"Leaf: {graph.node_to_bel(leaf)}\")\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='gzcpevkqtuqsfyjh'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 14}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 2}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"b097a471f1fb37c8206da83f8cf86668534b3469343940202384cda9e85acb294bb50b9c69df75eb070f8392a22732f08cbce6fb1c38da3372e7e0fb62533325\", \"key\": 0, \"line\": 42188, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 21}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\", \"type\": \"PubMed\"}, \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"hash\": \"51711324bca269e59366fba3ee61b5764f7c8e1ce605a3410a5178ad13f6ed02edd84805382cffa0d68de36f7dc9fe4c7281c4faa42ba5805c5018b3f4cef6ec\", \"key\": 0, \"line\": 42118, \"relation\": \"increases\", \"source\": 4, \"target\": 0}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Chemokine signaling subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\", \"type\": \"PubMed\"}, \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"hash\": \"4f0cba06b674c0e9847c0fb9884cf859ca04ba644c3c6b28c077cfa4cf1a974bb209c0cdda26733a5f6ada7743d7cff85624c046373e88f3e78122b2cf2fefdd\", \"key\": 0, \"line\": 3554, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Beta secretase subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"hash\": \"1e78a3aea456a94331af060b6acca92b9fdf5ce40ee68f9a2179f513be1a8ceda93198b68dd0e65d3bfe46b251ef6be30b4738befcf651d1ad9812d63050d6ff\", \"key\": 1, \"line\": 6181, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"APOE subgraph\": true, \"Low density lipoprotein subgraph\": true}}, \"citation\": {\"name\": \"J Biol Chem. 2012 Mar 1. [Epub ahead of print]\", \"reference\": \"22383525\", \"type\": \"PubMed\"}, \"evidence\": \"Cellular uptake and degradation by glial cells is one means by which ABeta may be cleared from the brain. In the current study, we demonstrate that modulating levels of the low-density lipoprotein receptor (LDLR), a cell surface receptor that regulates the amount of apolipoprotein E (apoE) in the brain, altered both the uptake and degradation of ABeta by astrocytes. Deletion of LDLR caused a decrease in ABeta uptake, while increasing LDLR levels significantly enhanced both the uptake and clearance of ABeta. Increasing LDLR levels also enhanced the cellular degradation of ABeta and facilitated the vesicular transport of ABeta to lysosomes. Despite the fact that LDLR regulated the uptake of apoE by astrocytes, we found that the effect of LDLR on ABeta uptake and clearance occurred in the absence of apoE. Finally, we provide evidence that ABeta can directly bind to LDLR, suggesting an interaction between LDLR and ABeta could be responsible for LDLR-mediated ABeta uptake. Therefore, these results identify LDLR as a receptor that mediates ABeta uptake and clearance by astrocytes, and provide evidence that increasing glial LDLR levels may promote ABeta degradation within the brain\", \"hash\": \"fecdb06cc63b0946512b0d57bf1de37ccecd3633343e9cdb5671d07531cbc04802c0ae1a71e6534814b9ee6f0ce399b25ebf475f8dc97d2a63c0c8b9caf93358\", \"key\": 2, \"line\": 35325, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Journal of neuroinflammation; Vol. 11; Iss. \", \"reference\": \"24694234\", \"type\": \"PubMed\"}, \"evidence\": \"In addition, primary human astrocytes stimulated with the RIG-1 ligand 5'ppp RNA showed increased expression of amyloid precursor protein (APP) and amyloid- (Abeta), supporting the idea that RIG-1 is involved in the pathology of MCI associated with early progression to AD.These findings suggest that RIG-1 may play a critical role in incipient AD.\", \"hash\": \"44f25dfab4dbfc410250e4ab86280e73c9b1bfcfbc6522fde6dc3387fbd91dd4570fd4d1f31c89c05cfe99ff3bb39aa0088f31cb1af2aabb632aab2a044978db\", \"key\": 3, \"line\": 39478, \"relation\": \"increases\", \"source\": 4, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Tumor necrosis factor subgraph\": true}}, \"citation\": {\"name\": \"J Neuroinflammation. 2011 Nov 2;8:150. doi: 10.1186/1742-2094-8-150.\", \"reference\": \"22047170\", \"type\": \"PubMed\"}, \"evidence\": \"Cytokines including TNF-+IFN- increase levels of endogenous BACE1, APP, and Abeta and stimulate amyloidogenic/ APP processing in astrocytes. Oligomeric and fibrillar Abeta42 also increase levels of astrocytic BACE1, APP, and -secretase/ processing. Together, our results suggest a cytokine- and Abeta42-driven feed-forward mechanism that promotes astrocytic Abeta/ production. Given that astrocytes greatly outnumber neurons, activated astrocytes may represent significant sources of Abeta/ during neuroinflammation in AD\", \"hash\": \"5223c465d4674837b2596c4aba54d38ff3a70148ac3773bdea1fef6a41b73da935452e9285019f43c1a992ce58915bcc5d5dc6f7ce82b9bb0cfa7a70782bde87\", \"key\": 4, \"line\": 42189, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Interleukin signaling subgraph\": true}}, \"citation\": {\"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\", \"type\": \"PubMed\"}, \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"hash\": \"8c09ee5bf342ee7449995131e6d0c4db263272a6d2dd2bb358cc00c503190f0fa93ad93e22d2bb631fffcc0aa52ea8037aa9b3ba2ed283b1b32e1d60e75bac8c\", \"key\": 0, \"line\": 4875, \"relation\": \"increases\", \"source\": 4, \"target\": 13}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 5, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 7, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 8, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 9, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 11, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 13, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 15, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 15, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 17, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 18, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 20, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 21, \"target\": 0}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"Astrocyte\", \"function\": \"Abundance\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"], \"name\": \"Astrocyte\", \"namespace\": \"BRCO\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}]}, \"#gzcpevkqtuqsfyjh\", 1000, 400, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "leaves = runner.score_leaves()\n",
    "len(leaves)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "\n",
       "require.config({\n",
       "    paths: {\n",
       "        d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "    }\n",
       "});\n",
       "\n",
       "var elementInnerHTML = \"<div id='ebthqvyacfqdjspz'></div>\";\n",
       "\n",
       "element.append(elementInnerHTML);\n",
       "\n",
       "require(['d3'], function (d3) {\n",
       "    return init_d3_force(d3, {\"directed\": true, \"graph\": {\"annotation_list\": {}, \"annotation_pattern\": {}, \"annotation_url\": {}, \"document_metadata\": {}, \"namespace_pattern\": {}, \"namespace_url\": {}, \"namespaces_uncached\": [], \"pybel_version\": \"0.11.4-dev\"}, \"links\": [{\"annotations\": {\"MeSHAnatomy\": {\"Serum\": true}, \"Subgraph\": {\"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\", \"type\": \"PubMed\"}, \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"hash\": \"e69ef494c7f5b78970b293b1c96c87458c8c55533fb633f00e8b61700982b6e6b69209dd3ddec87e27dcfba2fb4bb7ecedb9c093d2499a46731dac83dfddb8e8\", \"key\": 0, \"line\": 15309, \"relation\": \"increases\", \"source\": 0, \"target\": 13}, {\"annotations\": {\"MeSHAnatomy\": {\"Microglia\": true}, \"Subgraph\": {\"Blood vessel dilation subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\", \"type\": \"PubMed\"}, \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"hash\": \"c00b0f6e3b4e8118ea8e549d11b0f107a9bc29ef1c35bc6ce624d48899be4885b7d157d1e25ba006f76f0c60bd0a701cc737e3d0ef5915c3c2e52d27d3e1aa32\", \"key\": 0, \"line\": 12814, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"During early AD pathogenesis, amyloid beta (Abeta), S100B and IL-1 could bring about a vicious cycle of Abeta/ generation between astrocytes and neurons leading to chronic, sustained and progressive neuroinflammation. In advanced/ stages of AD, TRAIL secreted from astrocytes have been shown to bind to death receptor 5 (DR5) on neurons to trigger / apoptotic process in a caspase-8-dependent manner. Furthermore, astrocytes could be reactivated by TGF1 to generate more Abeta and / to undergo the aggravating astrogliosis. TGF2 was also observed to cooperate with Abeta to cause neuronal demise by / destroying the stability of lysosomes in neurons.\", \"hash\": \"7bc141e5eac4a56069d2287d79185fa00d3bde3db17baac176c07c1b425be4e8fd6077b59f754e9cdc0a46d5d758871d8bcbdc4e42d79186faf0e1ea1ce00680\", \"key\": 1, \"line\": 38125, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\", \"type\": \"PubMed\"}, \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"hash\": \"7bafb73a0cbd250fbbfc2a519af4e67070138aa5c2fe2614b0bb2d8c4e92de46929296fea4aff3bb406637782313c68ed84a3a30faa07781ad5878af7921d303\", \"key\": 2, \"line\": 46344, \"relation\": \"increases\", \"source\": 1, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Prostaglandin subgraph\": true}}, \"citation\": {\"name\": \"J Neuroimmunol. 2008 Mar;195(1-2):81-7. doi: 10.1016/j.jneuroim.2008.01.014. Epub 2008 Mar 18.\", \"reference\": \"18353443\", \"type\": \"PubMed\"}, \"evidence\": \"For example, when the brain is injured, microglia become activated by Abeta deposits and recruit astrocytes / by secreting acute-phase proteins such as complement factors and cytokines. Reactive microglia and astrocytes additionally / generate proinflammatory mediators, including cytokines, chemokines, prostaglandins, neurotoxic secretory products, / reactive oxygen species, and nitric oxide (Griffin et al., 1998; Tuppo and Arias, 2005). Cytokines and chemokines, / in turn, stimulate the synthesis of other enzymes, such as COXs and prostaglandin synthases. In AD, the expression / of COX-2, the inducible isoform, increases in response to inflammatory agents in neurons and glial cells (Pasinetti and / Aisen, 1998; Sairanen et al., 1998). Because COX is the rate-limiting enzyme in the production of prostaglandins (O'Banion,/ 1999; Smith et al., 1991), the increase in COX activity leads to an increase in prostaglandin production (Consilvio et al.,/ 2004). \", \"hash\": \"5c0479426fd22d8b3ffe906a847fe5b7c8d70bf5cfc20a2c3cc529d1ebe57bec61f0f0cdeb563f739262c0cc2f9d20925326ddd07bce69b05fc373eaaa4b096e\", \"key\": 0, \"line\": 38007, \"relation\": \"increases\", \"source\": 1, \"target\": 2}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"e7d961580bdc0d9ad25133ac540e97b343c2a734c71939efee651214b82ed300d2aa3356a8bb5f60a9df947252e26445891266fd31a16bdcc3a95023bee326c8\", \"key\": 0, \"line\": 57261, \"relation\": \"increases\", \"source\": 1, \"target\": 3}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 0, \"line\": 2758, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Disease\": {\"Alzheimer's disease\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"c9593b6ebfe71e25d451ca18952877bde449a50738cb9a7a8dcd17016e343a3617d3d7bcb750d6a6d84a8e0e2ad6b2e74c7ce25a394ecce31999bcffe064b281\", \"key\": 1, \"line\": 2761, \"object\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 2, \"line\": 34551, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true}}, \"citation\": {\"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\", \"type\": \"PubMed\"}, \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"hash\": \"205395c49caa77bd819eb7d3fe3755c68d6b0c0bac8908ca9bdac8787877c512623acf971055b60ff0913d88bee7e15d522df66619e4dde28132065b5b5f82c3\", \"key\": 3, \"line\": 34557, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 4}, {\"annotations\": {\"Subgraph\": {\"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"5b4bfdc099dc91a4652ac5673c9d786beb19715f72a85fcf97c0850de98bb182718388b1026555c1948d945703721de079be3c9aa950605d4a715a52b0eb2703\", \"key\": 0, \"line\": 24176, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Cytokine signaling subgraph\": true, \"Immunoglobulin subgraph\": true}}, \"citation\": {\"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\", \"type\": \"PubMed\"}, \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"hash\": \"339a53cf0a3a8c93db8d3617972f2747cfe65783cfc095701c6a3d4e51ce034dc88d37a720509eff7a7fab82fda44cc8b3671f0753fa072ce4045c676709bf57\", \"key\": 1, \"line\": 24768, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"e0c55f9d5bfbb3bacb9637891fcba284b211b45237e434ed7ff4dccc9ffd4e7cfdfc6948f39c9ed4190b5f3f29f26b0691027a143d2289edb9150330a57e13cf\", \"key\": 2, \"line\": 38301, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 5}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"b30d4e764a1e1d6e07cc0885c620b28fa29fc1ab04edfbdc197764979871388f0749ab98c6ffe63600180688ec93858cad7b89d8d2677efc2fd05ba0a3b054f6\", \"key\": 0, \"line\": 38309, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 6}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"a9031360008a6c5dcbd1e3f2d1a4186f80b237ca4097e8863b28f10b3d24c98c0d9b60d908d1ec82436cfcea52e553e61ed6595c8bdf009eb8a622a6f166b56b\", \"key\": 0, \"line\": 38434, \"relation\": \"increases\", \"source\": 1, \"target\": 7}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"85249bfd3f4823af3dd733887b1872d074e64e6b732458511a7f489678c6d68c54dff0545f5e00b237798045aa07727e494548364652bf540d51118075d79643\", \"key\": 0, \"line\": 1175, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d1b2efdf5e971fb09483644707ec3a1041441da908cf52bd72c0e0ae693def481638edcb6278c1d93e3a5e1a7c3796ebc1563a8e8c6875314c0369669b7c6e59\", \"key\": 1, \"line\": 38305, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 8}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"ea69a90868f9503b77c41a07417fa91247f5157be766beeb93fe9cf44e6e64f6802fa5f071e83022cd85658b0a47ce92fe88a69b71ccb45ce2a7a82b1d2901b8\", \"key\": 0, \"line\": 1199, \"relation\": \"increases\", \"source\": 1, \"target\": 9}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"53a030e345c4537cb07fedf4b47eddcd0d0fbb2fd55539e5e16abb39771ff9b4d918aed1d774c05e448148d09d2e0a2c6daaf1719b00a048643f5fa1c464d580\", \"key\": 0, \"line\": 1200, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"d4c7ebf71e4e0b45e5266139a3f7914da131ae493d608019070632555346151d5dc450822a496b6975b2a0ab4236cd72bdc635aaf10719a4e4172e45fcdc3522\", \"key\": 1, \"line\": 38307, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 10}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"411dc964a111942c04ebb1cf64fc7cacad8482712d87991c073533618d01b155250b1d66685da36b3766f31ebddd9bbf989b9f0be4d74df21c42ac61b75d271b\", \"key\": 0, \"line\": 1201, \"relation\": \"increases\", \"source\": 1, \"target\": 11}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"267c312c4ac005a85186b6facf3684db3cd1767844a11c8a705eec1a6938ddce42be80c50b0573dfde52ddd3ea59087d427561fe75bb166c096355d7493feb56\", \"key\": 0, \"line\": 38432, \"relation\": \"increases\", \"source\": 1, \"target\": 12}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 0, \"line\": 38289, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"01f8b1194097f7ea34fa18c69828e65d81948bec2dd14628494fc25bfafb77d5ae06cff78cc4c1e4bcc8c4500913a32aedc1f3a8f611d024e13fd9402af9d879\", \"key\": 1, \"line\": 38291, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 13}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"77945159cfc13a663ef04f39e8b1a3d0bdbd7c85e02d121736d49190a6f6f590f871dae8e33e09fd9adf5fc2e1ed39c063ec403af06e79f5b9ce74f9d8eed4db\", \"key\": 0, \"line\": 38287, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"b35b2625396e937414b46bfb43c0bf2046350124f5eb0e57ece2bb25d98561b28b977da014bae29179ae486859ffa698eae00ad46a44bcc11ed0acb64971e19d\", \"key\": 1, \"line\": 38445, \"relation\": \"increases\", \"source\": 1, \"target\": 14}, {\"annotations\": {\"Subgraph\": {\"Amyloidogenic subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\", \"type\": \"PubMed\"}, \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptotic process.\", \"hash\": \"6f38fb5373e42e9777dd248d762af4e115ddd2f7ce3a27de8b3ff8d05b4f2b206e24b8926e6334f137bf8cba1c3831176c2e6487ccdf600af3f2c789d27b100c\", \"key\": 0, \"line\": 20428, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 15}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"0627c31f4899097d836db7dfccf1bf298446ce803a6ad46814ef30dfdafd3735a3b3305631475ef1dff70e4997fc6a561f94136ff1e47eb2787afa94d1a8473f\", \"key\": 0, \"line\": 38441, \"relation\": \"increases\", \"source\": 1, \"target\": 16}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"53a942b2d864220d8ee6ea5cfc18a80e52c515e71d9998c43149702ee2407fd05cbcc45c8841ded72d16db373327261ac00f672ab03388fe364ab4a597703fe8\", \"key\": 0, \"line\": 38297, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 17}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"497ddc22ce832e08bdc029658677233f3a48e77cb9f09a21915bccd25c30601d2fcff70464a56848ab4be3475a944ae973d7a941ad18476eae02f35c6509720d\", \"key\": 0, \"line\": 38295, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 18}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"ffa5f80883d1f67dcf1cc6803d12b339e43e91c77ba62d3d820bf76fc9a3aaff27bf2ebae6c375a359e63e1d71da067e90fc7f457a2a7737b18472d2fe97965b\", \"key\": 0, \"line\": 38299, \"object\": {\"modifier\": \"Activity\"}, \"relation\": \"increases\", \"source\": 1, \"target\": 19}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"882cc46c6072623c131bf8000280d0901a5580ea2840d7a5894a09f298134ea582752aaa2af0c174762443f8e04894a892c95c5f2555ac84965c7335649b8ebe\", \"key\": 0, \"line\": 38443, \"relation\": \"increases\", \"source\": 1, \"target\": 20}, {\"annotations\": {\"MeSHAnatomy\": {\"Astrocytes\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Curr Alzheimer Res. 2011 Feb;8(1):67-80.\", \"reference\": \"21143158\", \"type\": \"PubMed\"}, \"evidence\": \"Astrocyte is the most abundant type of glial cells in the central nervous system (CNS) and appears to be/ involved in the induction of neuroinflammation. Under stress and injury, astrocytes become astrogliotic leading to an / upregulation of the expression of proinflammatory cytokines and chemokines, which are associated with the pathogenesis of AD. \", \"hash\": \"5ab9335e006a69c7cd26de4399acd19038a29046dffdab9dec8c8bb2053707545f76410f1d69eebcfdc6a0a62d9da5115b9f3a9a6d18c72a12ad8c59eec76641\", \"key\": 0, \"line\": 38113, \"relation\": \"increases\", \"source\": 2, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}, \"Subgraph\": {\"Inflammatory response subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"BioMed research international; Vol. 2013; Iss.\", \"reference\": \"24455696\", \"type\": \"PubMed\"}, \"evidence\": \"These proinflammatory factors act as potent stimuli in brain inflammation through upregulation of diverse inflammatory genes, including matrix metalloproteinases (MMPs), cytosolic phospholipase A2 (cPLA2), cyclooxygenase-2 (COX-2), and adhesion molecules.\", \"hash\": \"cee903c343b52932139b2f65fb9cec5b01e2e0b22967448cd6921de35a68ee755df5270bca62c2d2e4265b26d135c2ac709955f805810cec656a47e349b57c02\", \"key\": 0, \"line\": 39268, \"relation\": \"increases\", \"source\": 3, \"target\": 0}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true}}, \"citation\": {\"name\": \"Neurobiol Aging. 2007 Aug;28(8):1215-20. Epub 2006 Jul 5\", \"reference\": \"16822591\", \"type\": \"PubMed\"}, \"evidence\": \"Several proteases were shown to hydrolyze Abeta in vitro or in cell-based assays, and are likely candidates for a role in Abeta clearance in brain. Previous reports suggest that matrix metalloproteinases (MMPs) could be involved in such a mechanism.\", \"hash\": \"67af17501fb6e4eac79df406b81a0d8f1807c062f7b471447635974c097751b7fcfbefde073d11e61816ba96b21db7b18b5fb6c3176b6074b72044091d906b3a\", \"key\": 0, \"line\": 18797, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"MeSHAnatomy\": {\"Brain\": true, \"Microglia\": true}, \"Subgraph\": {\"Amyloidogenic subgraph\": true, \"Matrix metalloproteinase subgraph\": true}}, \"citation\": {\"name\": \"Journal of Alzheimer,s disease\", \"reference\": \"23792694\", \"type\": \"PubMed\"}, \"evidence\": \" However, MMPs can degrade both soluble and fibrillar forms of amyloid- (A). It has also been shown that A enhances the expression of MMPs in neuroglial cultures and induces the release of TIMP-1 by brain cell\", \"hash\": \"02bb8f321fe4119b3dcc46e738ca4169f57fa90e7590e6169e6f33d0b1dc2bb059033876bdf533e715a91c0e28775e0d891b1e509082b62d4b3c706556d7616f\", \"key\": 1, \"line\": 57263, \"object\": {\"modifier\": \"Degradation\"}, \"relation\": \"increases\", \"source\": 3, \"target\": 1}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Inhibition of glycogen synthase kinase 3beta (GSK3beta) decreases inflammatory responses in brain endothelial cells.\", \"hash\": \"4f9e288d8afc39bac15c1877d222af27d881002cfda0254f1132cee79c856b07d99982c036f0187a9b0cbed3053861dc895333024bb3f760add087cad1a809f3\", \"key\": 0, \"line\": 6049, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"GSK3 subgraph\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\", \"type\": \"PubMed\"}, \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"hash\": \"0f2d27530aa9b97add410579eedef954a5c914c053cc5ed47f6955e98fb6a73f0063c35445890eef2e00c784c9b1b14697ef10141bf498202cb68a8fc95f2407\", \"key\": 1, \"line\": 6061, \"relation\": \"increases\", \"source\": 4, \"subject\": {\"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}, \"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"GSK3 subgraph\": true, \"Non-amyloidogenic subgraph\": true}}, \"citation\": {\"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\", \"type\": \"PubMed\"}, \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"hash\": \"8eab374e644c4acbfbf20b7bd346e3a1b26fc478ca972e27e14b168fa174a5e0824a2e19dbe4fcc80287873fbd6af3d61c473fd6fa9312955cc75345722d7af7\", \"key\": 0, \"line\": 12126, \"relation\": \"decreases\", \"source\": 4, \"subject\": {\"modifier\": \"Degradation\"}, \"target\": 1}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1dd71a48b4400dff1aaa98d0472e3659b3a5cc1894e24b9a5b784482b200368a0559b77d63933f3bf84ab24399751cf00133202831601f62ff3fdce81c62c6ab\", \"key\": 0, \"line\": 38302, \"relation\": \"increases\", \"source\": 5, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"1ce74070e5f4bccdae49cef4fc70cc6afd7a5dfd2b7bd81e0bcd67d805cdf08c61ae85659f0a577c44dd6f36e3bf45afef5baacb2827b6a23ee166874dcdf741\", \"key\": 0, \"line\": 38310, \"relation\": \"increases\", \"source\": 6, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"14a3887eb04d47ef16861d71b8dbb615442260a5e053b1edaba1571d5cc80d450178c8a40e5666396882524bf805000269d2a50cc90e7255e9cf62d8352d8deb\", \"key\": 0, \"line\": 38435, \"relation\": \"increases\", \"source\": 7, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"672730d00a691c300de69b2f6ee7d07e9f4cd1295a303c2d7c5bbff6a7a5fd6f1de526cde889356d0d0fdacc9acd0226d761c77816f13199c90aa7dd1e1b99d2\", \"key\": 0, \"line\": 38306, \"relation\": \"increases\", \"source\": 8, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"cfda80e4a954dcc649cf7bd19c2983940b2b5b589338201119c480ecf9d4ef19811f3e952b7e304d7e1d35f933ac8bea331a3aa54ed9e41a07ec82716ffae32f\", \"key\": 0, \"line\": 1186, \"relation\": \"increases\", \"source\": 9, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"12db7f7cc9f85bce7bd1cec9a0048416adcf9313ef5be6b7f6b7b6bd7f4d986722521ed166304c984aef9d91c92f1a44beb1fb4fb638b2d053b2fd720ccd757c\", \"key\": 0, \"line\": 1187, \"relation\": \"increases\", \"source\": 10, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"bfc317b01c60356f50855835e09a1faff561e435495113ba791b0cc496d29ba44d16211b010366d31eebf83c29eecb598156d525c25dd8cdc72d67adf3e9c750\", \"key\": 1, \"line\": 38308, \"relation\": \"increases\", \"source\": 10, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Condition\": {\"Normal Healthy State\": true}, \"Subgraph\": {\"Binding and Uptake of Ligands by Scavenger Receptors\": true, \"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\", \"type\": \"PubMed\"}, \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"hash\": \"9057e11eac794418cf771f4124f1daafb97e16803e381a5c1ee656806df0d2f76917b292846af769cdefca3cf9fbc296c7670127f389c85712f0940af4447744\", \"key\": 0, \"line\": 1188, \"relation\": \"increases\", \"source\": 11, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"5666c2adfe76a10b0f62d3a953239dbc83c1337a91045d261fc183c923d0b6a9f6349f2297914baf96c02c70c6d8583877c89febfc691bcfd87675fa8e727fb6\", \"key\": 0, \"line\": 38433, \"relation\": \"increases\", \"source\": 12, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 1, \"line\": 38290, \"relation\": \"increases\", \"source\": 13, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"352ded28b883a2a933081a4b19c9cae618177ee3910fa64c01674d50c4b6ee50f4f8b9bf0f15210021f547eca22994188e17129ffc4a42e21b8316b5a7112c03\", \"key\": 2, \"line\": 38292, \"relation\": \"increases\", \"source\": 13, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"279a54ffd5588c267b8c927e67a7b190773a1b88e2ff365804a59c382b01f4dffc671f1016884ea19247f81f5a7f52c2706ac78d21193e970e746886c99d7bb7\", \"key\": 1, \"line\": 38288, \"relation\": \"increases\", \"source\": 14, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"e9664597556a52c6000cd806e454d6367e7c55923452d92eca059e8b4e2726d190362174effd4b92ff46fdad745c36dbee34bbefc4f0d87213da18e71e859d95\", \"key\": 2, \"line\": 38446, \"relation\": \"increases\", \"source\": 14, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"MAPK-JNK subgraph\": true}}, \"citation\": {\"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\", \"type\": \"PubMed\"}, \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptotic process, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"hash\": \"f12bc7714924e1c095ed8f322676394f0cab0a88f866cd49594d96d7efbf5b0766f5228964a17ba383403a6b4f5584074bcbc7c7797a8264809cd790a361f8d0\", \"key\": 0, \"line\": 38211, \"relation\": \"increases\", \"source\": 15, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"d9f9a59e824d19d7a6ccba08688635b3bfdff94faa2ccfe3e947c1d15fcfa50819994a165b6aec066c3530dbda98fb459a4e16c963dfa0d03003ba1b9a0ddcaf\", \"key\": 0, \"line\": 38442, \"relation\": \"increases\", \"source\": 16, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"43b7bf9ae85178ba5d2c2f3b092959556fe00834a26708ff435ea32065db4531d6f6809add4b102986bb51ddaadc644b471c5e80e9ed433ffa257039af1b40f6\", \"key\": 0, \"line\": 38298, \"relation\": \"increases\", \"source\": 17, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"c00ae2cbdd0df7d62d9603e0b7047077b765f83ac8748d40aadc62c31c1f3704d131300d4249bc906bb525533365a96aeccbad44cc1207fce970118f015af54c\", \"key\": 0, \"line\": 38296, \"relation\": \"increases\", \"source\": 18, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true}}, \"citation\": {\"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\", \"type\": \"PubMed\"}, \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"hash\": \"dd4777dd826497bfbc85a9340a80fef26042c5d41f87d7288cbf3487b2b5f396666803db1ffb4ef2e199c2c0286b438f10b998f3a0f8eb3d9c4acc8e240b3296\", \"key\": 0, \"line\": 38300, \"relation\": \"increases\", \"source\": 19, \"subject\": {\"modifier\": \"Activity\"}, \"target\": 0}, {\"annotations\": {\"Subgraph\": {\"Inflammatory response subgraph\": true, \"Toll like receptor subgraph\": true}}, \"citation\": {\"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\", \"type\": \"PubMed\"}, \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"hash\": \"147e3e04a79ecfdb6792dee9a9b49557ae03c0dec50505999851a841462bf741754ae924a31147c926e69af2b722abc415a125e89231b6c83bdfd0bb3d85655a\", \"key\": 0, \"line\": 38444, \"relation\": \"increases\", \"source\": 20, \"target\": 0}], \"multigraph\": true, \"nodes\": [{\"cname\": \"inflammatory response\", \"function\": \"BiologicalProcess\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"], \"name\": \"inflammatory response\", \"namespace\": \"GOBP\"}, {\"cname\": \"p(HGNC:APP, frag(672_713))\", \"function\": \"Protein\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]], \"name\": \"APP\", \"namespace\": \"HGNC\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}]}, {\"cname\": \"Gliosis\", \"function\": \"Pathology\", \"id\": [\"Pathology\", \"MESHD\", \"Gliosis\"], \"name\": \"Gliosis\", \"namespace\": \"MESHD\"}, {\"cname\": \"MMP Family\", \"function\": \"Protein\", \"id\": [\"Protein\", \"SFAM\", \"MMP Family\"], \"name\": \"MMP Family\", \"namespace\": \"SFAM\"}, {\"cname\": \"GSK3B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"GSK3B\"], \"name\": \"GSK3B\", \"namespace\": \"HGNC\", \"weight\": 0.32860467}, {\"cname\": \"AGER\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"AGER\"], \"name\": \"AGER\", \"namespace\": \"HGNC\", \"weight\": -0.52367838}, {\"cname\": \"PTX3\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"PTX3\"], \"name\": \"PTX3\", \"namespace\": \"HGNC\", \"weight\": -1.00547039}, {\"cname\": \"IL23A\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL23A\"], \"name\": \"IL23A\", \"namespace\": \"HGNC\", \"weight\": -0.28486748}, {\"cname\": \"MSR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MSR1\"], \"name\": \"MSR1\", \"namespace\": \"HGNC\", \"weight\": 0.60114332}, {\"cname\": \"SCARF1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARF1\"], \"name\": \"SCARF1\", \"namespace\": \"HGNC\", \"weight\": 0.18225965}, {\"cname\": \"SCARB1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARB1\"], \"name\": \"SCARB1\", \"namespace\": \"HGNC\", \"weight\": -0.36691523}, {\"cname\": \"SCARA5\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"SCARA5\"], \"name\": \"SCARA5\", \"namespace\": \"HGNC\", \"weight\": 0.49422828}, {\"cname\": \"IL12B\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"IL12B\"], \"name\": \"IL12B\", \"namespace\": \"HGNC\", \"weight\": -1.61550531}, {\"cname\": \"TLR4\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR4\"], \"name\": \"TLR4\", \"namespace\": \"HGNC\", \"weight\": 0.11823175}, {\"cname\": \"TLR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"TLR2\"], \"name\": \"TLR2\", \"namespace\": \"HGNC\", \"weight\": 0.13520554}, {\"cname\": \"EIF2AK2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"EIF2AK2\"], \"name\": \"EIF2AK2\", \"namespace\": \"HGNC\", \"weight\": 0.47307381}, {\"cname\": \"CD14\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"CD14\"], \"name\": \"CD14\", \"namespace\": \"HGNC\", \"weight\": -0.73001192}, {\"cname\": \"FPR1\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR1\"], \"name\": \"FPR1\", \"namespace\": \"HGNC\", \"weight\": -0.5889405}, {\"cname\": \"NOD2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"NOD2\"], \"name\": \"NOD2\", \"namespace\": \"HGNC\", \"weight\": -0.62084437}, {\"cname\": \"FPR2\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"FPR2\"], \"name\": \"FPR2\", \"namespace\": \"HGNC\", \"weight\": 5.3e-05}, {\"cname\": \"MYD88\", \"function\": \"Gene\", \"id\": [\"Gene\", \"HGNC\", \"MYD88\"], \"name\": \"MYD88\", \"namespace\": \"HGNC\", \"weight\": 0.981683}]}, \"#ebthqvyacfqdjspz\", 1000, 400, {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'});\n",
       "});"
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "runner.run()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "77.20570941000088"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.get_final_score()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Robustness Analysis\n",
    "\n",
    "The algorithm is random, so it can be run multiple times to assess the robustness of the final scores using [pybel_tools.analysis.heat.multirun](http://pybel-tools.readthedocs.io/en/latest/heatdiffusion.html#pybel_tools.analysis.heat.multirun)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|| 500/500 [00:22<00:00, 22.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 22 s, sys: 423 ms, total: 22.4 s\n",
      "Wall time: 22.7 s\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "runners = list(multirun(subgraph, inflamation_node, runs=500, use_tqdm=True))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"277pt\" version=\"1.1\" viewBox=\"0 0 409 277\" width=\"409pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 277.314375 \n",
       "L 409.635625 277.314375 \n",
       "L 409.635625 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 50.14375 239.758125 \n",
       "L 384.94375 239.758125 \n",
       "L 384.94375 22.318125 \n",
       "L 50.14375 22.318125 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 116.087535 239.758125 \n",
       "L 136.378778 239.758125 \n",
       "L 136.378778 38.269322 \n",
       "L 116.087535 38.269322 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 136.378778 239.758125 \n",
       "L 156.670021 239.758125 \n",
       "L 156.670021 122.22299 \n",
       "L 136.378778 122.22299 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 156.670021 239.758125 \n",
       "L 176.961264 239.758125 \n",
       "L 176.961264 32.672411 \n",
       "L 156.670021 32.672411 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 176.961264 239.758125 \n",
       "L 197.252507 239.758125 \n",
       "L 197.252507 131.551175 \n",
       "L 176.961264 131.551175 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 197.252507 239.758125 \n",
       "L 217.54375 239.758125 \n",
       "L 217.54375 208.042295 \n",
       "L 197.252507 208.042295 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 217.54375 239.758125 \n",
       "L 237.834993 239.758125 \n",
       "L 237.834993 174.460828 \n",
       "L 217.54375 174.460828 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 237.834993 239.758125 \n",
       "L 258.126236 239.758125 \n",
       "L 258.126236 206.176658 \n",
       "L 237.834993 206.176658 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 258.126236 239.758125 \n",
       "L 278.417479 239.758125 \n",
       "L 278.417479 148.341909 \n",
       "L 258.126236 148.341909 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 278.417479 239.758125 \n",
       "L 298.708722 239.758125 \n",
       "L 298.708722 180.057739 \n",
       "L 278.417479 180.057739 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 298.708722 239.758125 \n",
       "L 318.999965 239.758125 \n",
       "L 318.999965 222.967391 \n",
       "L 298.708722 222.967391 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m95b58c93af\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"54.353509\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- 40 -->\n",
       "      <defs>\n",
       "       <path d=\"M 37.796875 64.3125 \n",
       "L 12.890625 25.390625 \n",
       "L 37.796875 25.390625 \n",
       "z\n",
       "M 35.203125 72.90625 \n",
       "L 47.609375 72.90625 \n",
       "L 47.609375 25.390625 \n",
       "L 58.015625 25.390625 \n",
       "L 58.015625 17.1875 \n",
       "L 47.609375 17.1875 \n",
       "L 47.609375 0 \n",
       "L 37.796875 0 \n",
       "L 37.796875 17.1875 \n",
       "L 4.890625 17.1875 \n",
       "L 4.890625 26.703125 \n",
       "z\n",
       "\" id=\"DejaVuSans-34\"/>\n",
       "       <path d=\"M 31.78125 66.40625 \n",
       "Q 24.171875 66.40625 20.328125 58.90625 \n",
       "Q 16.5 51.421875 16.5 36.375 \n",
       "Q 16.5 21.390625 20.328125 13.890625 \n",
       "Q 24.171875 6.390625 31.78125 6.390625 \n",
       "Q 39.453125 6.390625 43.28125 13.890625 \n",
       "Q 47.125 21.390625 47.125 36.375 \n",
       "Q 47.125 51.421875 43.28125 58.90625 \n",
       "Q 39.453125 66.40625 31.78125 66.40625 \n",
       "z\n",
       "M 31.78125 74.21875 \n",
       "Q 44.046875 74.21875 50.515625 64.515625 \n",
       "Q 56.984375 54.828125 56.984375 36.375 \n",
       "Q 56.984375 17.96875 50.515625 8.265625 \n",
       "Q 44.046875 -1.421875 31.78125 -1.421875 \n",
       "Q 19.53125 -1.421875 13.0625 8.265625 \n",
       "Q 6.59375 17.96875 6.59375 36.375 \n",
       "Q 6.59375 54.828125 13.0625 64.515625 \n",
       "Q 19.53125 74.21875 31.78125 74.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-30\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(47.991009 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"99.509674\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- 50 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.796875 72.90625 \n",
       "L 49.515625 72.90625 \n",
       "L 49.515625 64.59375 \n",
       "L 19.828125 64.59375 \n",
       "L 19.828125 46.734375 \n",
       "Q 21.96875 47.46875 24.109375 47.828125 \n",
       "Q 26.265625 48.1875 28.421875 48.1875 \n",
       "Q 40.625 48.1875 47.75 41.5 \n",
       "Q 54.890625 34.8125 54.890625 23.390625 \n",
       "Q 54.890625 11.625 47.5625 5.09375 \n",
       "Q 40.234375 -1.421875 26.90625 -1.421875 \n",
       "Q 22.3125 -1.421875 17.546875 -0.640625 \n",
       "Q 12.796875 0.140625 7.71875 1.703125 \n",
       "L 7.71875 11.625 \n",
       "Q 12.109375 9.234375 16.796875 8.0625 \n",
       "Q 21.484375 6.890625 26.703125 6.890625 \n",
       "Q 35.15625 6.890625 40.078125 11.328125 \n",
       "Q 45.015625 15.765625 45.015625 23.390625 \n",
       "Q 45.015625 31 40.078125 35.4375 \n",
       "Q 35.15625 39.890625 26.703125 39.890625 \n",
       "Q 22.75 39.890625 18.8125 39.015625 \n",
       "Q 14.890625 38.140625 10.796875 36.28125 \n",
       "z\n",
       "\" id=\"DejaVuSans-35\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(93.147174 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-35\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"144.665839\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- 60 -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.015625 40.375 \n",
       "Q 26.375 40.375 22.484375 35.828125 \n",
       "Q 18.609375 31.296875 18.609375 23.390625 \n",
       "Q 18.609375 15.53125 22.484375 10.953125 \n",
       "Q 26.375 6.390625 33.015625 6.390625 \n",
       "Q 39.65625 6.390625 43.53125 10.953125 \n",
       "Q 47.40625 15.53125 47.40625 23.390625 \n",
       "Q 47.40625 31.296875 43.53125 35.828125 \n",
       "Q 39.65625 40.375 33.015625 40.375 \n",
       "z\n",
       "M 52.59375 71.296875 \n",
       "L 52.59375 62.3125 \n",
       "Q 48.875 64.0625 45.09375 64.984375 \n",
       "Q 41.3125 65.921875 37.59375 65.921875 \n",
       "Q 27.828125 65.921875 22.671875 59.328125 \n",
       "Q 17.53125 52.734375 16.796875 39.40625 \n",
       "Q 19.671875 43.65625 24.015625 45.921875 \n",
       "Q 28.375 48.1875 33.59375 48.1875 \n",
       "Q 44.578125 48.1875 50.953125 41.515625 \n",
       "Q 57.328125 34.859375 57.328125 23.390625 \n",
       "Q 57.328125 12.15625 50.6875 5.359375 \n",
       "Q 44.046875 -1.421875 33.015625 -1.421875 \n",
       "Q 20.359375 -1.421875 13.671875 8.265625 \n",
       "Q 6.984375 17.96875 6.984375 36.375 \n",
       "Q 6.984375 53.65625 15.1875 63.9375 \n",
       "Q 23.390625 74.21875 37.203125 74.21875 \n",
       "Q 40.921875 74.21875 44.703125 73.484375 \n",
       "Q 48.484375 72.75 52.59375 71.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-36\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(138.303339 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"189.822004\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- 70 -->\n",
       "      <defs>\n",
       "       <path d=\"M 8.203125 72.90625 \n",
       "L 55.078125 72.90625 \n",
       "L 55.078125 68.703125 \n",
       "L 28.609375 0 \n",
       "L 18.3125 0 \n",
       "L 43.21875 64.59375 \n",
       "L 8.203125 64.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-37\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(183.459504 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-37\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"234.978169\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- 80 -->\n",
       "      <defs>\n",
       "       <path d=\"M 31.78125 34.625 \n",
       "Q 24.75 34.625 20.71875 30.859375 \n",
       "Q 16.703125 27.09375 16.703125 20.515625 \n",
       "Q 16.703125 13.921875 20.71875 10.15625 \n",
       "Q 24.75 6.390625 31.78125 6.390625 \n",
       "Q 38.8125 6.390625 42.859375 10.171875 \n",
       "Q 46.921875 13.96875 46.921875 20.515625 \n",
       "Q 46.921875 27.09375 42.890625 30.859375 \n",
       "Q 38.875 34.625 31.78125 34.625 \n",
       "z\n",
       "M 21.921875 38.8125 \n",
       "Q 15.578125 40.375 12.03125 44.71875 \n",
       "Q 8.5 49.078125 8.5 55.328125 \n",
       "Q 8.5 64.0625 14.71875 69.140625 \n",
       "Q 20.953125 74.21875 31.78125 74.21875 \n",
       "Q 42.671875 74.21875 48.875 69.140625 \n",
       "Q 55.078125 64.0625 55.078125 55.328125 \n",
       "Q 55.078125 49.078125 51.53125 44.71875 \n",
       "Q 48 40.375 41.703125 38.8125 \n",
       "Q 48.828125 37.15625 52.796875 32.3125 \n",
       "Q 56.78125 27.484375 56.78125 20.515625 \n",
       "Q 56.78125 9.90625 50.3125 4.234375 \n",
       "Q 43.84375 -1.421875 31.78125 -1.421875 \n",
       "Q 19.734375 -1.421875 13.25 4.234375 \n",
       "Q 6.78125 9.90625 6.78125 20.515625 \n",
       "Q 6.78125 27.484375 10.78125 32.3125 \n",
       "Q 14.796875 37.15625 21.921875 38.8125 \n",
       "z\n",
       "M 18.3125 54.390625 \n",
       "Q 18.3125 48.734375 21.84375 45.5625 \n",
       "Q 25.390625 42.390625 31.78125 42.390625 \n",
       "Q 38.140625 42.390625 41.71875 45.5625 \n",
       "Q 45.3125 48.734375 45.3125 54.390625 \n",
       "Q 45.3125 60.0625 41.71875 63.234375 \n",
       "Q 38.140625 66.40625 31.78125 66.40625 \n",
       "Q 25.390625 66.40625 21.84375 63.234375 \n",
       "Q 18.3125 60.0625 18.3125 54.390625 \n",
       "z\n",
       "\" id=\"DejaVuSans-38\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(228.615669 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-38\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"280.134334\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- 90 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.984375 1.515625 \n",
       "L 10.984375 10.5 \n",
       "Q 14.703125 8.734375 18.5 7.8125 \n",
       "Q 22.3125 6.890625 25.984375 6.890625 \n",
       "Q 35.75 6.890625 40.890625 13.453125 \n",
       "Q 46.046875 20.015625 46.78125 33.40625 \n",
       "Q 43.953125 29.203125 39.59375 26.953125 \n",
       "Q 35.25 24.703125 29.984375 24.703125 \n",
       "Q 19.046875 24.703125 12.671875 31.3125 \n",
       "Q 6.296875 37.9375 6.296875 49.421875 \n",
       "Q 6.296875 60.640625 12.9375 67.421875 \n",
       "Q 19.578125 74.21875 30.609375 74.21875 \n",
       "Q 43.265625 74.21875 49.921875 64.515625 \n",
       "Q 56.59375 54.828125 56.59375 36.375 \n",
       "Q 56.59375 19.140625 48.40625 8.859375 \n",
       "Q 40.234375 -1.421875 26.421875 -1.421875 \n",
       "Q 22.703125 -1.421875 18.890625 -0.6875 \n",
       "Q 15.09375 0.046875 10.984375 1.515625 \n",
       "z\n",
       "M 30.609375 32.421875 \n",
       "Q 37.25 32.421875 41.125 36.953125 \n",
       "Q 45.015625 41.5 45.015625 49.421875 \n",
       "Q 45.015625 57.28125 41.125 61.84375 \n",
       "Q 37.25 66.40625 30.609375 66.40625 \n",
       "Q 23.96875 66.40625 20.09375 61.84375 \n",
       "Q 16.21875 57.28125 16.21875 49.421875 \n",
       "Q 16.21875 41.5 20.09375 36.953125 \n",
       "Q 23.96875 32.421875 30.609375 32.421875 \n",
       "z\n",
       "\" id=\"DejaVuSans-39\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(273.771834 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-39\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"325.290499\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- 100 -->\n",
       "      <defs>\n",
       "       <path d=\"M 12.40625 8.296875 \n",
       "L 28.515625 8.296875 \n",
       "L 28.515625 63.921875 \n",
       "L 10.984375 60.40625 \n",
       "L 10.984375 69.390625 \n",
       "L 28.421875 72.90625 \n",
       "L 38.28125 72.90625 \n",
       "L 38.28125 8.296875 \n",
       "L 54.390625 8.296875 \n",
       "L 54.390625 0 \n",
       "L 12.40625 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-31\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(315.746749 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"370.446664\" xlink:href=\"#m95b58c93af\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- 110 -->\n",
       "      <g transform=\"translate(360.902914 254.356562)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_9\">\n",
       "     <!-- Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 53.515625 70.515625 \n",
       "L 53.515625 60.890625 \n",
       "Q 47.90625 63.578125 42.921875 64.890625 \n",
       "Q 37.9375 66.21875 33.296875 66.21875 \n",
       "Q 25.25 66.21875 20.875 63.09375 \n",
       "Q 16.5 59.96875 16.5 54.203125 \n",
       "Q 16.5 49.359375 19.40625 46.890625 \n",
       "Q 22.3125 44.4375 30.421875 42.921875 \n",
       "L 36.375 41.703125 \n",
       "Q 47.40625 39.59375 52.65625 34.296875 \n",
       "Q 57.90625 29 57.90625 20.125 \n",
       "Q 57.90625 9.515625 50.796875 4.046875 \n",
       "Q 43.703125 -1.421875 29.984375 -1.421875 \n",
       "Q 24.8125 -1.421875 18.96875 -0.25 \n",
       "Q 13.140625 0.921875 6.890625 3.21875 \n",
       "L 6.890625 13.375 \n",
       "Q 12.890625 10.015625 18.65625 8.296875 \n",
       "Q 24.421875 6.59375 29.984375 6.59375 \n",
       "Q 38.421875 6.59375 43.015625 9.90625 \n",
       "Q 47.609375 13.234375 47.609375 19.390625 \n",
       "Q 47.609375 24.75 44.3125 27.78125 \n",
       "Q 41.015625 30.8125 33.5 32.328125 \n",
       "L 27.484375 33.5 \n",
       "Q 16.453125 35.6875 11.515625 40.375 \n",
       "Q 6.59375 45.0625 6.59375 53.421875 \n",
       "Q 6.59375 63.09375 13.40625 68.65625 \n",
       "Q 20.21875 74.21875 32.171875 74.21875 \n",
       "Q 37.3125 74.21875 42.625 73.28125 \n",
       "Q 47.953125 72.359375 53.515625 70.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-53\"/>\n",
       "      <path d=\"M 48.78125 52.59375 \n",
       "L 48.78125 44.1875 \n",
       "Q 44.96875 46.296875 41.140625 47.34375 \n",
       "Q 37.3125 48.390625 33.40625 48.390625 \n",
       "Q 24.65625 48.390625 19.8125 42.84375 \n",
       "Q 14.984375 37.3125 14.984375 27.296875 \n",
       "Q 14.984375 17.28125 19.8125 11.734375 \n",
       "Q 24.65625 6.203125 33.40625 6.203125 \n",
       "Q 37.3125 6.203125 41.140625 7.25 \n",
       "Q 44.96875 8.296875 48.78125 10.40625 \n",
       "L 48.78125 2.09375 \n",
       "Q 45.015625 0.34375 40.984375 -0.53125 \n",
       "Q 36.96875 -1.421875 32.421875 -1.421875 \n",
       "Q 20.0625 -1.421875 12.78125 6.34375 \n",
       "Q 5.515625 14.109375 5.515625 27.296875 \n",
       "Q 5.515625 40.671875 12.859375 48.328125 \n",
       "Q 20.21875 56 33.015625 56 \n",
       "Q 37.15625 56 41.109375 55.140625 \n",
       "Q 45.0625 54.296875 48.78125 52.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-63\"/>\n",
       "      <path d=\"M 30.609375 48.390625 \n",
       "Q 23.390625 48.390625 19.1875 42.75 \n",
       "Q 14.984375 37.109375 14.984375 27.296875 \n",
       "Q 14.984375 17.484375 19.15625 11.84375 \n",
       "Q 23.34375 6.203125 30.609375 6.203125 \n",
       "Q 37.796875 6.203125 41.984375 11.859375 \n",
       "Q 46.1875 17.53125 46.1875 27.296875 \n",
       "Q 46.1875 37.015625 41.984375 42.703125 \n",
       "Q 37.796875 48.390625 30.609375 48.390625 \n",
       "z\n",
       "M 30.609375 56 \n",
       "Q 42.328125 56 49.015625 48.375 \n",
       "Q 55.71875 40.765625 55.71875 27.296875 \n",
       "Q 55.71875 13.875 49.015625 6.21875 \n",
       "Q 42.328125 -1.421875 30.609375 -1.421875 \n",
       "Q 18.84375 -1.421875 12.171875 6.21875 \n",
       "Q 5.515625 13.875 5.515625 27.296875 \n",
       "Q 5.515625 40.765625 12.171875 48.375 \n",
       "Q 18.84375 56 30.609375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-6f\"/>\n",
       "      <path d=\"M 41.109375 46.296875 \n",
       "Q 39.59375 47.171875 37.8125 47.578125 \n",
       "Q 36.03125 48 33.890625 48 \n",
       "Q 26.265625 48 22.1875 43.046875 \n",
       "Q 18.109375 38.09375 18.109375 28.8125 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 20.953125 51.171875 25.484375 53.578125 \n",
       "Q 30.03125 56 36.53125 56 \n",
       "Q 37.453125 56 38.578125 55.875 \n",
       "Q 39.703125 55.765625 41.0625 55.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-72\"/>\n",
       "      <path d=\"M 56.203125 29.59375 \n",
       "L 56.203125 25.203125 \n",
       "L 14.890625 25.203125 \n",
       "Q 15.484375 15.921875 20.484375 11.0625 \n",
       "Q 25.484375 6.203125 34.421875 6.203125 \n",
       "Q 39.59375 6.203125 44.453125 7.46875 \n",
       "Q 49.3125 8.734375 54.109375 11.28125 \n",
       "L 54.109375 2.78125 \n",
       "Q 49.265625 0.734375 44.1875 -0.34375 \n",
       "Q 39.109375 -1.421875 33.890625 -1.421875 \n",
       "Q 20.796875 -1.421875 13.15625 6.1875 \n",
       "Q 5.515625 13.8125 5.515625 26.8125 \n",
       "Q 5.515625 40.234375 12.765625 48.109375 \n",
       "Q 20.015625 56 32.328125 56 \n",
       "Q 43.359375 56 49.78125 48.890625 \n",
       "Q 56.203125 41.796875 56.203125 29.59375 \n",
       "z\n",
       "M 47.21875 32.234375 \n",
       "Q 47.125 39.59375 43.09375 43.984375 \n",
       "Q 39.0625 48.390625 32.421875 48.390625 \n",
       "Q 24.90625 48.390625 20.390625 44.140625 \n",
       "Q 15.875 39.890625 15.1875 32.171875 \n",
       "z\n",
       "\" id=\"DejaVuSans-65\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(203.430469 268.034687)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"63.476562\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "      <use x=\"118.457031\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"179.638672\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"220.720703\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m257afd515f\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"239.758125\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- 0.00 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.6875 12.40625 \n",
       "L 21 12.40625 \n",
       "L 21 0 \n",
       "L 10.6875 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-2e\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.878125 243.557344)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"197.841263\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- 0.01 -->\n",
       "      <g transform=\"translate(20.878125 201.640481)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"155.9244\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- 0.02 -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.1875 8.296875 \n",
       "L 53.609375 8.296875 \n",
       "L 53.609375 0 \n",
       "L 7.328125 0 \n",
       "L 7.328125 8.296875 \n",
       "Q 12.9375 14.109375 22.625 23.890625 \n",
       "Q 32.328125 33.6875 34.8125 36.53125 \n",
       "Q 39.546875 41.84375 41.421875 45.53125 \n",
       "Q 43.3125 49.21875 43.3125 52.78125 \n",
       "Q 43.3125 58.59375 39.234375 62.25 \n",
       "Q 35.15625 65.921875 28.609375 65.921875 \n",
       "Q 23.96875 65.921875 18.8125 64.3125 \n",
       "Q 13.671875 62.703125 7.8125 59.421875 \n",
       "L 7.8125 69.390625 \n",
       "Q 13.765625 71.78125 18.9375 73 \n",
       "Q 24.125 74.21875 28.421875 74.21875 \n",
       "Q 39.75 74.21875 46.484375 68.546875 \n",
       "Q 53.21875 62.890625 53.21875 53.421875 \n",
       "Q 53.21875 48.921875 51.53125 44.890625 \n",
       "Q 49.859375 40.875 45.40625 35.40625 \n",
       "Q 44.1875 33.984375 37.640625 27.21875 \n",
       "Q 31.109375 20.453125 19.1875 8.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-32\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.878125 159.723619)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"114.007538\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- 0.03 -->\n",
       "      <defs>\n",
       "       <path d=\"M 40.578125 39.3125 \n",
       "Q 47.65625 37.796875 51.625 33 \n",
       "Q 55.609375 28.21875 55.609375 21.1875 \n",
       "Q 55.609375 10.40625 48.1875 4.484375 \n",
       "Q 40.765625 -1.421875 27.09375 -1.421875 \n",
       "Q 22.515625 -1.421875 17.65625 -0.515625 \n",
       "Q 12.796875 0.390625 7.625 2.203125 \n",
       "L 7.625 11.71875 \n",
       "Q 11.71875 9.328125 16.59375 8.109375 \n",
       "Q 21.484375 6.890625 26.8125 6.890625 \n",
       "Q 36.078125 6.890625 40.9375 10.546875 \n",
       "Q 45.796875 14.203125 45.796875 21.1875 \n",
       "Q 45.796875 27.640625 41.28125 31.265625 \n",
       "Q 36.765625 34.90625 28.71875 34.90625 \n",
       "L 20.21875 34.90625 \n",
       "L 20.21875 43.015625 \n",
       "L 29.109375 43.015625 \n",
       "Q 36.375 43.015625 40.234375 45.921875 \n",
       "Q 44.09375 48.828125 44.09375 54.296875 \n",
       "Q 44.09375 59.90625 40.109375 62.90625 \n",
       "Q 36.140625 65.921875 28.71875 65.921875 \n",
       "Q 24.65625 65.921875 20.015625 65.03125 \n",
       "Q 15.375 64.15625 9.8125 62.3125 \n",
       "L 9.8125 71.09375 \n",
       "Q 15.4375 72.65625 20.34375 73.4375 \n",
       "Q 25.25 74.21875 29.59375 74.21875 \n",
       "Q 40.828125 74.21875 47.359375 69.109375 \n",
       "Q 53.90625 64.015625 53.90625 55.328125 \n",
       "Q 53.90625 49.265625 50.4375 45.09375 \n",
       "Q 46.96875 40.921875 40.578125 39.3125 \n",
       "z\n",
       "\" id=\"DejaVuSans-33\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.878125 117.806757)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-33\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"72.090675\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- 0.04 -->\n",
       "      <g transform=\"translate(20.878125 75.889894)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-34\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"50.14375\" xlink:href=\"#m257afd515f\" y=\"30.173813\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- 0.05 -->\n",
       "      <g transform=\"translate(20.878125 33.973032)scale(0.1 -0.1)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-2e\"/>\n",
       "       <use x=\"95.410156\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"159.033203\" xlink:href=\"#DejaVuSans-35\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_16\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 9.8125 72.90625 \n",
       "L 51.703125 72.90625 \n",
       "L 51.703125 64.59375 \n",
       "L 19.671875 64.59375 \n",
       "L 19.671875 43.109375 \n",
       "L 48.578125 43.109375 \n",
       "L 48.578125 34.8125 \n",
       "L 19.671875 34.8125 \n",
       "L 19.671875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-46\"/>\n",
       "      <path d=\"M 14.796875 27.296875 \n",
       "Q 14.796875 17.390625 18.875 11.75 \n",
       "Q 22.953125 6.109375 30.078125 6.109375 \n",
       "Q 37.203125 6.109375 41.296875 11.75 \n",
       "Q 45.40625 17.390625 45.40625 27.296875 \n",
       "Q 45.40625 37.203125 41.296875 42.84375 \n",
       "Q 37.203125 48.484375 30.078125 48.484375 \n",
       "Q 22.953125 48.484375 18.875 42.84375 \n",
       "Q 14.796875 37.203125 14.796875 27.296875 \n",
       "z\n",
       "M 45.40625 8.203125 \n",
       "Q 42.578125 3.328125 38.25 0.953125 \n",
       "Q 33.9375 -1.421875 27.875 -1.421875 \n",
       "Q 17.96875 -1.421875 11.734375 6.484375 \n",
       "Q 5.515625 14.40625 5.515625 27.296875 \n",
       "Q 5.515625 40.1875 11.734375 48.09375 \n",
       "Q 17.96875 56 27.875 56 \n",
       "Q 33.9375 56 38.25 53.625 \n",
       "Q 42.578125 51.265625 45.40625 46.390625 \n",
       "L 45.40625 54.6875 \n",
       "L 54.390625 54.6875 \n",
       "L 54.390625 -20.796875 \n",
       "L 45.40625 -20.796875 \n",
       "z\n",
       "\" id=\"DejaVuSans-71\"/>\n",
       "      <path d=\"M 8.5 21.578125 \n",
       "L 8.5 54.6875 \n",
       "L 17.484375 54.6875 \n",
       "L 17.484375 21.921875 \n",
       "Q 17.484375 14.15625 20.5 10.265625 \n",
       "Q 23.53125 6.390625 29.59375 6.390625 \n",
       "Q 36.859375 6.390625 41.078125 11.03125 \n",
       "Q 45.3125 15.671875 45.3125 23.6875 \n",
       "L 45.3125 54.6875 \n",
       "L 54.296875 54.6875 \n",
       "L 54.296875 0 \n",
       "L 45.3125 0 \n",
       "L 45.3125 8.40625 \n",
       "Q 42.046875 3.421875 37.71875 1 \n",
       "Q 33.40625 -1.421875 27.6875 -1.421875 \n",
       "Q 18.265625 -1.421875 13.375 4.4375 \n",
       "Q 8.5 10.296875 8.5 21.578125 \n",
       "z\n",
       "M 31.109375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-75\"/>\n",
       "      <path d=\"M 54.890625 33.015625 \n",
       "L 54.890625 0 \n",
       "L 45.90625 0 \n",
       "L 45.90625 32.71875 \n",
       "Q 45.90625 40.484375 42.875 44.328125 \n",
       "Q 39.84375 48.1875 33.796875 48.1875 \n",
       "Q 26.515625 48.1875 22.3125 43.546875 \n",
       "Q 18.109375 38.921875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.34375 51.125 25.703125 53.5625 \n",
       "Q 30.078125 56 35.796875 56 \n",
       "Q 45.21875 56 50.046875 50.171875 \n",
       "Q 54.890625 44.34375 54.890625 33.015625 \n",
       "z\n",
       "\" id=\"DejaVuSans-6e\"/>\n",
       "      <path d=\"M 32.171875 -5.078125 \n",
       "Q 28.375 -14.84375 24.75 -17.8125 \n",
       "Q 21.140625 -20.796875 15.09375 -20.796875 \n",
       "L 7.90625 -20.796875 \n",
       "L 7.90625 -13.28125 \n",
       "L 13.1875 -13.28125 \n",
       "Q 16.890625 -13.28125 18.9375 -11.515625 \n",
       "Q 21 -9.765625 23.484375 -3.21875 \n",
       "L 25.09375 0.875 \n",
       "L 2.984375 54.6875 \n",
       "L 12.5 54.6875 \n",
       "L 29.59375 11.921875 \n",
       "L 46.6875 54.6875 \n",
       "L 56.203125 54.6875 \n",
       "z\n",
       "\" id=\"DejaVuSans-79\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.798438 157.335781)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-46\"/>\n",
       "      <use x=\"57.410156\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"98.492188\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"160.015625\" xlink:href=\"#DejaVuSans-71\"/>\n",
       "      <use x=\"223.492188\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"286.871094\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"348.394531\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "      <use x=\"411.773438\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "      <use x=\"466.753906\" xlink:href=\"#DejaVuSans-79\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"line2d_15\">\n",
       "    <path clip-path=\"url(#pad10a48887)\" d=\"M 65.361932 239.412251 \n",
       "L 70.15506 238.933533 \n",
       "L 72.551624 238.508907 \n",
       "L 74.948188 237.898189 \n",
       "L 77.344752 237.03916 \n",
       "L 79.741316 235.856805 \n",
       "L 82.13788 234.264148 \n",
       "L 84.534444 232.16466 \n",
       "L 86.931008 229.456567 \n",
       "L 89.327572 226.039212 \n",
       "L 91.724137 221.821371 \n",
       "L 94.120701 216.731101 \n",
       "L 96.517265 210.726344 \n",
       "L 98.913829 203.805177 \n",
       "L 101.310393 196.014367 \n",
       "L 103.706957 187.454822 \n",
       "L 108.500085 168.70499 \n",
       "L 113.293213 149.352235 \n",
       "L 115.689777 140.132695 \n",
       "L 118.086341 131.577993 \n",
       "L 120.482905 123.918342 \n",
       "L 122.879469 117.328241 \n",
       "L 125.276033 111.911582 \n",
       "L 127.672598 107.693048 \n",
       "L 130.069162 104.616841 \n",
       "L 132.465726 102.553021 \n",
       "L 134.86229 101.310861 \n",
       "L 137.258854 100.657884 \n",
       "L 144.448546 99.771582 \n",
       "L 146.84511 99.131932 \n",
       "L 149.241674 98.098983 \n",
       "L 151.638238 96.645733 \n",
       "L 154.034802 94.821713 \n",
       "L 161.224494 88.607739 \n",
       "L 163.621059 87.002949 \n",
       "L 166.017623 86.025571 \n",
       "L 168.414187 85.887985 \n",
       "L 170.810751 86.759011 \n",
       "L 173.207315 88.750026 \n",
       "L 175.603879 91.906948 \n",
       "L 178.000443 96.208368 \n",
       "L 180.397007 101.569428 \n",
       "L 182.793571 107.850403 \n",
       "L 185.190135 114.86864 \n",
       "L 189.983263 130.254688 \n",
       "L 194.776391 145.933471 \n",
       "L 197.172955 153.355685 \n",
       "L 199.56952 160.266379 \n",
       "L 201.966084 166.532497 \n",
       "L 204.362648 172.059572 \n",
       "L 206.759212 176.793453 \n",
       "L 209.155776 180.719922 \n",
       "L 211.55234 183.86219 \n",
       "L 213.948904 186.276279 \n",
       "L 216.345468 188.044471 \n",
       "L 218.742032 189.267138 \n",
       "L 221.138596 190.053495 \n",
       "L 223.53516 190.512004 \n",
       "L 225.931724 190.741251 \n",
       "L 230.724852 190.812374 \n",
       "L 235.51798 190.619598 \n",
       "L 240.311109 190.118449 \n",
       "L 242.707673 189.658833 \n",
       "L 245.104237 188.997134 \n",
       "L 247.500801 188.09362 \n",
       "L 249.897365 186.924312 \n",
       "L 252.293929 185.487372 \n",
       "L 254.690493 183.807032 \n",
       "L 259.483621 179.946632 \n",
       "L 264.276749 176.02482 \n",
       "L 266.673313 174.316822 \n",
       "L 269.069877 172.927787 \n",
       "L 271.466441 171.957743 \n",
       "L 273.863006 171.488031 \n",
       "L 276.25957 171.576449 \n",
       "L 278.656134 172.254674 \n",
       "L 281.052698 173.527987 \n",
       "L 283.449262 175.376976 \n",
       "L 285.845826 177.760756 \n",
       "L 288.24239 180.62113 \n",
       "L 290.638954 183.887112 \n",
       "L 293.035518 187.479372 \n",
       "L 297.828646 195.307338 \n",
       "L 305.018338 207.435751 \n",
       "L 309.811467 214.955178 \n",
       "L 312.208031 218.383209 \n",
       "L 314.604595 221.54206 \n",
       "L 317.001159 224.409355 \n",
       "L 319.397723 226.973417 \n",
       "L 321.794287 229.232539 \n",
       "L 324.190851 231.193847 \n",
       "L 326.587415 232.87184 \n",
       "L 328.983979 234.286732 \n",
       "L 331.380543 235.46272 \n",
       "L 333.777107 236.426306 \n",
       "L 336.173671 237.204788 \n",
       "L 338.570235 237.824994 \n",
       "L 343.363363 238.689887 \n",
       "L 348.156492 239.195655 \n",
       "L 352.94962 239.473551 \n",
       "L 360.139312 239.643028 \n",
       "L 367.329004 239.605731 \n",
       "L 369.725568 239.533398 \n",
       "L 369.725568 239.533398 \n",
       "\" style=\"fill:none;stroke:#1f77b4;stroke-linecap:square;stroke-width:1.5;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 50.14375 239.758125 \n",
       "L 50.14375 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 384.94375 239.758125 \n",
       "L 384.94375 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 50.14375 239.758125 \n",
       "L 384.94375 239.758125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 50.14375 22.318125 \n",
       "L 384.94375 22.318125 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_17\">\n",
       "    <!-- Histogram of heat diffusion scores for inflammatory response -->\n",
       "    <defs>\n",
       "     <path d=\"M 9.8125 72.90625 \n",
       "L 19.671875 72.90625 \n",
       "L 19.671875 43.015625 \n",
       "L 55.515625 43.015625 \n",
       "L 55.515625 72.90625 \n",
       "L 65.375 72.90625 \n",
       "L 65.375 0 \n",
       "L 55.515625 0 \n",
       "L 55.515625 34.71875 \n",
       "L 19.671875 34.71875 \n",
       "L 19.671875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-48\"/>\n",
       "     <path d=\"M 9.421875 54.6875 \n",
       "L 18.40625 54.6875 \n",
       "L 18.40625 0 \n",
       "L 9.421875 0 \n",
       "z\n",
       "M 9.421875 75.984375 \n",
       "L 18.40625 75.984375 \n",
       "L 18.40625 64.59375 \n",
       "L 9.421875 64.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-69\"/>\n",
       "     <path d=\"M 44.28125 53.078125 \n",
       "L 44.28125 44.578125 \n",
       "Q 40.484375 46.53125 36.375 47.5 \n",
       "Q 32.28125 48.484375 27.875 48.484375 \n",
       "Q 21.1875 48.484375 17.84375 46.4375 \n",
       "Q 14.5 44.390625 14.5 40.28125 \n",
       "Q 14.5 37.15625 16.890625 35.375 \n",
       "Q 19.28125 33.59375 26.515625 31.984375 \n",
       "L 29.59375 31.296875 \n",
       "Q 39.15625 29.25 43.1875 25.515625 \n",
       "Q 47.21875 21.78125 47.21875 15.09375 \n",
       "Q 47.21875 7.46875 41.1875 3.015625 \n",
       "Q 35.15625 -1.421875 24.609375 -1.421875 \n",
       "Q 20.21875 -1.421875 15.453125 -0.5625 \n",
       "Q 10.6875 0.296875 5.421875 2 \n",
       "L 5.421875 11.28125 \n",
       "Q 10.40625 8.6875 15.234375 7.390625 \n",
       "Q 20.0625 6.109375 24.8125 6.109375 \n",
       "Q 31.15625 6.109375 34.5625 8.28125 \n",
       "Q 37.984375 10.453125 37.984375 14.40625 \n",
       "Q 37.984375 18.0625 35.515625 20.015625 \n",
       "Q 33.0625 21.96875 24.703125 23.78125 \n",
       "L 21.578125 24.515625 \n",
       "Q 13.234375 26.265625 9.515625 29.90625 \n",
       "Q 5.8125 33.546875 5.8125 39.890625 \n",
       "Q 5.8125 47.609375 11.28125 51.796875 \n",
       "Q 16.75 56 26.8125 56 \n",
       "Q 31.78125 56 36.171875 55.265625 \n",
       "Q 40.578125 54.546875 44.28125 53.078125 \n",
       "z\n",
       "\" id=\"DejaVuSans-73\"/>\n",
       "     <path d=\"M 18.3125 70.21875 \n",
       "L 18.3125 54.6875 \n",
       "L 36.8125 54.6875 \n",
       "L 36.8125 47.703125 \n",
       "L 18.3125 47.703125 \n",
       "L 18.3125 18.015625 \n",
       "Q 18.3125 11.328125 20.140625 9.421875 \n",
       "Q 21.96875 7.515625 27.59375 7.515625 \n",
       "L 36.8125 7.515625 \n",
       "L 36.8125 0 \n",
       "L 27.59375 0 \n",
       "Q 17.1875 0 13.234375 3.875 \n",
       "Q 9.28125 7.765625 9.28125 18.015625 \n",
       "L 9.28125 47.703125 \n",
       "L 2.6875 47.703125 \n",
       "L 2.6875 54.6875 \n",
       "L 9.28125 54.6875 \n",
       "L 9.28125 70.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-74\"/>\n",
       "     <path d=\"M 45.40625 27.984375 \n",
       "Q 45.40625 37.75 41.375 43.109375 \n",
       "Q 37.359375 48.484375 30.078125 48.484375 \n",
       "Q 22.859375 48.484375 18.828125 43.109375 \n",
       "Q 14.796875 37.75 14.796875 27.984375 \n",
       "Q 14.796875 18.265625 18.828125 12.890625 \n",
       "Q 22.859375 7.515625 30.078125 7.515625 \n",
       "Q 37.359375 7.515625 41.375 12.890625 \n",
       "Q 45.40625 18.265625 45.40625 27.984375 \n",
       "z\n",
       "M 54.390625 6.78125 \n",
       "Q 54.390625 -7.171875 48.1875 -13.984375 \n",
       "Q 42 -20.796875 29.203125 -20.796875 \n",
       "Q 24.46875 -20.796875 20.265625 -20.09375 \n",
       "Q 16.0625 -19.390625 12.109375 -17.921875 \n",
       "L 12.109375 -9.1875 \n",
       "Q 16.0625 -11.328125 19.921875 -12.34375 \n",
       "Q 23.78125 -13.375 27.78125 -13.375 \n",
       "Q 36.625 -13.375 41.015625 -8.765625 \n",
       "Q 45.40625 -4.15625 45.40625 5.171875 \n",
       "L 45.40625 9.625 \n",
       "Q 42.625 4.78125 38.28125 2.390625 \n",
       "Q 33.9375 0 27.875 0 \n",
       "Q 17.828125 0 11.671875 7.65625 \n",
       "Q 5.515625 15.328125 5.515625 27.984375 \n",
       "Q 5.515625 40.671875 11.671875 48.328125 \n",
       "Q 17.828125 56 27.875 56 \n",
       "Q 33.9375 56 38.28125 53.609375 \n",
       "Q 42.625 51.21875 45.40625 46.390625 \n",
       "L 45.40625 54.6875 \n",
       "L 54.390625 54.6875 \n",
       "z\n",
       "\" id=\"DejaVuSans-67\"/>\n",
       "     <path d=\"M 34.28125 27.484375 \n",
       "Q 23.390625 27.484375 19.1875 25 \n",
       "Q 14.984375 22.515625 14.984375 16.5 \n",
       "Q 14.984375 11.71875 18.140625 8.90625 \n",
       "Q 21.296875 6.109375 26.703125 6.109375 \n",
       "Q 34.1875 6.109375 38.703125 11.40625 \n",
       "Q 43.21875 16.703125 43.21875 25.484375 \n",
       "L 43.21875 27.484375 \n",
       "z\n",
       "M 52.203125 31.203125 \n",
       "L 52.203125 0 \n",
       "L 43.21875 0 \n",
       "L 43.21875 8.296875 \n",
       "Q 40.140625 3.328125 35.546875 0.953125 \n",
       "Q 30.953125 -1.421875 24.3125 -1.421875 \n",
       "Q 15.921875 -1.421875 10.953125 3.296875 \n",
       "Q 6 8.015625 6 15.921875 \n",
       "Q 6 25.140625 12.171875 29.828125 \n",
       "Q 18.359375 34.515625 30.609375 34.515625 \n",
       "L 43.21875 34.515625 \n",
       "L 43.21875 35.40625 \n",
       "Q 43.21875 41.609375 39.140625 45 \n",
       "Q 35.0625 48.390625 27.6875 48.390625 \n",
       "Q 23 48.390625 18.546875 47.265625 \n",
       "Q 14.109375 46.140625 10.015625 43.890625 \n",
       "L 10.015625 52.203125 \n",
       "Q 14.9375 54.109375 19.578125 55.046875 \n",
       "Q 24.21875 56 28.609375 56 \n",
       "Q 40.484375 56 46.34375 49.84375 \n",
       "Q 52.203125 43.703125 52.203125 31.203125 \n",
       "z\n",
       "\" id=\"DejaVuSans-61\"/>\n",
       "     <path d=\"M 52 44.1875 \n",
       "Q 55.375 50.25 60.0625 53.125 \n",
       "Q 64.75 56 71.09375 56 \n",
       "Q 79.640625 56 84.28125 50.015625 \n",
       "Q 88.921875 44.046875 88.921875 33.015625 \n",
       "L 88.921875 0 \n",
       "L 79.890625 0 \n",
       "L 79.890625 32.71875 \n",
       "Q 79.890625 40.578125 77.09375 44.375 \n",
       "Q 74.3125 48.1875 68.609375 48.1875 \n",
       "Q 61.625 48.1875 57.5625 43.546875 \n",
       "Q 53.515625 38.921875 53.515625 30.90625 \n",
       "L 53.515625 0 \n",
       "L 44.484375 0 \n",
       "L 44.484375 32.71875 \n",
       "Q 44.484375 40.625 41.703125 44.40625 \n",
       "Q 38.921875 48.1875 33.109375 48.1875 \n",
       "Q 26.21875 48.1875 22.15625 43.53125 \n",
       "Q 18.109375 38.875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.1875 51.21875 25.484375 53.609375 \n",
       "Q 29.78125 56 35.6875 56 \n",
       "Q 41.65625 56 45.828125 52.96875 \n",
       "Q 50 49.953125 52 44.1875 \n",
       "z\n",
       "\" id=\"DejaVuSans-6d\"/>\n",
       "     <path id=\"DejaVuSans-20\"/>\n",
       "     <path d=\"M 37.109375 75.984375 \n",
       "L 37.109375 68.5 \n",
       "L 28.515625 68.5 \n",
       "Q 23.6875 68.5 21.796875 66.546875 \n",
       "Q 19.921875 64.59375 19.921875 59.515625 \n",
       "L 19.921875 54.6875 \n",
       "L 34.71875 54.6875 \n",
       "L 34.71875 47.703125 \n",
       "L 19.921875 47.703125 \n",
       "L 19.921875 0 \n",
       "L 10.890625 0 \n",
       "L 10.890625 47.703125 \n",
       "L 2.296875 47.703125 \n",
       "L 2.296875 54.6875 \n",
       "L 10.890625 54.6875 \n",
       "L 10.890625 58.5 \n",
       "Q 10.890625 67.625 15.140625 71.796875 \n",
       "Q 19.390625 75.984375 28.609375 75.984375 \n",
       "z\n",
       "\" id=\"DejaVuSans-66\"/>\n",
       "     <path d=\"M 54.890625 33.015625 \n",
       "L 54.890625 0 \n",
       "L 45.90625 0 \n",
       "L 45.90625 32.71875 \n",
       "Q 45.90625 40.484375 42.875 44.328125 \n",
       "Q 39.84375 48.1875 33.796875 48.1875 \n",
       "Q 26.515625 48.1875 22.3125 43.546875 \n",
       "Q 18.109375 38.921875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 75.984375 \n",
       "L 18.109375 75.984375 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.34375 51.125 25.703125 53.5625 \n",
       "Q 30.078125 56 35.796875 56 \n",
       "Q 45.21875 56 50.046875 50.171875 \n",
       "Q 54.890625 44.34375 54.890625 33.015625 \n",
       "z\n",
       "\" id=\"DejaVuSans-68\"/>\n",
       "     <path d=\"M 45.40625 46.390625 \n",
       "L 45.40625 75.984375 \n",
       "L 54.390625 75.984375 \n",
       "L 54.390625 0 \n",
       "L 45.40625 0 \n",
       "L 45.40625 8.203125 \n",
       "Q 42.578125 3.328125 38.25 0.953125 \n",
       "Q 33.9375 -1.421875 27.875 -1.421875 \n",
       "Q 17.96875 -1.421875 11.734375 6.484375 \n",
       "Q 5.515625 14.40625 5.515625 27.296875 \n",
       "Q 5.515625 40.1875 11.734375 48.09375 \n",
       "Q 17.96875 56 27.875 56 \n",
       "Q 33.9375 56 38.25 53.625 \n",
       "Q 42.578125 51.265625 45.40625 46.390625 \n",
       "z\n",
       "M 14.796875 27.296875 \n",
       "Q 14.796875 17.390625 18.875 11.75 \n",
       "Q 22.953125 6.109375 30.078125 6.109375 \n",
       "Q 37.203125 6.109375 41.296875 11.75 \n",
       "Q 45.40625 17.390625 45.40625 27.296875 \n",
       "Q 45.40625 37.203125 41.296875 42.84375 \n",
       "Q 37.203125 48.484375 30.078125 48.484375 \n",
       "Q 22.953125 48.484375 18.875 42.84375 \n",
       "Q 14.796875 37.203125 14.796875 27.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-64\"/>\n",
       "     <path d=\"M 9.421875 75.984375 \n",
       "L 18.40625 75.984375 \n",
       "L 18.40625 0 \n",
       "L 9.421875 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-6c\"/>\n",
       "     <path d=\"M 18.109375 8.203125 \n",
       "L 18.109375 -20.796875 \n",
       "L 9.078125 -20.796875 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.390625 \n",
       "Q 20.953125 51.265625 25.265625 53.625 \n",
       "Q 29.59375 56 35.59375 56 \n",
       "Q 45.5625 56 51.78125 48.09375 \n",
       "Q 58.015625 40.1875 58.015625 27.296875 \n",
       "Q 58.015625 14.40625 51.78125 6.484375 \n",
       "Q 45.5625 -1.421875 35.59375 -1.421875 \n",
       "Q 29.59375 -1.421875 25.265625 0.953125 \n",
       "Q 20.953125 3.328125 18.109375 8.203125 \n",
       "z\n",
       "M 48.6875 27.296875 \n",
       "Q 48.6875 37.203125 44.609375 42.84375 \n",
       "Q 40.53125 48.484375 33.40625 48.484375 \n",
       "Q 26.265625 48.484375 22.1875 42.84375 \n",
       "Q 18.109375 37.203125 18.109375 27.296875 \n",
       "Q 18.109375 17.390625 22.1875 11.75 \n",
       "Q 26.265625 6.109375 33.40625 6.109375 \n",
       "Q 40.53125 6.109375 44.609375 11.75 \n",
       "Q 48.6875 17.390625 48.6875 27.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-70\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(32.651875 16.318125)scale(0.12 -0.12)\">\n",
       "     <use xlink:href=\"#DejaVuSans-48\"/>\n",
       "     <use x=\"75.195312\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"102.978516\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"155.078125\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "     <use x=\"194.287109\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"255.46875\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "     <use x=\"318.945312\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"360.058594\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"421.337891\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "     <use x=\"518.75\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"550.537109\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"611.71875\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"646.923828\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"678.710938\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "     <use x=\"742.089844\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"803.613281\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"864.892578\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "     <use x=\"904.101562\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"935.888672\" xlink:href=\"#DejaVuSans-64\"/>\n",
       "     <use x=\"999.365234\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"1027.148438\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"1062.353516\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"1097.558594\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "     <use x=\"1160.9375\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"1213.037109\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"1240.820312\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"1302.001953\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"1365.380859\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1397.167969\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"1449.267578\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "     <use x=\"1504.248047\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"1565.429688\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"1606.511719\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"1668.035156\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"1720.134766\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1751.921875\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"1787.126953\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"1848.308594\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"1889.421875\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"1921.208984\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "     <use x=\"1948.992188\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"2012.371094\" xlink:href=\"#DejaVuSans-66\"/>\n",
       "     <use x=\"2047.576172\" xlink:href=\"#DejaVuSans-6c\"/>\n",
       "     <use x=\"2075.359375\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"2136.638672\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "     <use x=\"2234.050781\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "     <use x=\"2331.462891\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "     <use x=\"2392.742188\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "     <use x=\"2431.951172\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"2493.132812\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"2534.246094\" xlink:href=\"#DejaVuSans-79\"/>\n",
       "     <use x=\"2593.425781\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "     <use x=\"2625.212891\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "     <use x=\"2666.294922\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     <use x=\"2727.818359\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"2779.917969\" xlink:href=\"#DejaVuSans-70\"/>\n",
       "     <use x=\"2843.394531\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "     <use x=\"2904.576172\" xlink:href=\"#DejaVuSans-6e\"/>\n",
       "     <use x=\"2967.955078\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     <use x=\"3020.054688\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pad10a48887\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"50.14375\" y=\"22.318125\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x10ad01cc0>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "sns.distplot([runner.get_final_score() for runner in runners])\n",
    "plt.title(\n",
    "    f\"Histogram of heat diffusion scores for {calculate_canonical_name(subgraph, inflamation_node)}\"\n",
    ")\n",
    "plt.ylabel(\"Frequency\")\n",
    "plt.xlabel(\"Score\")\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Full Workflow\n",
    "\n",
    "This workflow can all be run with [pybel_tools.analysis.heat.workflow_all_average](http://pybel-tools.readthedocs.io/en/latest/heat.html#pybel_tools.analysis.heat.workflow_all_average) but is outlined here for a deeper understanding.\n",
    "\n",
    "### Generate Candidate Mechanisms\n",
    "\n",
    "Candidate mechanism are generated with the aforementioned function [pybel_tools.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism) to assess the upstream causal subgraphs of all biological processes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 426 bioprocesses\n"
     ]
    }
   ],
   "source": [
    "all_bioprocesses = list(pybel_tools.filters.get_nodes_by_function(graph, BIOPROCESS))\n",
    "\n",
    "print(f\"There are {len(all_bioprocesses)} bioprocesses\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 610 ms, sys: 15.8 ms, total: 626 ms\n",
      "Wall time: 628 ms\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "candidate_mechanisms = {\n",
    "    node: pybel_tools.generation.generate_mechanism(graph, node)\n",
    "    for node in all_bioprocesses\n",
    "}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The algorithm is run over all candidate subgraphs that are nontrivial (have more than one node in them) and a pandas DataFrame is prepared."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|| 426/426 [01:40<00:00,  4.25it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 1min 39s, sys: 1.01 s, total: 1min 40s\n",
      "Wall time: 1min 40s\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "scores = {}\n",
    "\n",
    "for node, subgraph in tqdm(\n",
    "    candidate_mechanisms.items(), total=len(candidate_mechanisms)\n",
    "):\n",
    "\n",
    "    fneighbors = subgraph.in_degree(node)\n",
    "    fneighbors = 0 if isinstance(fneighbors, dict) else fneighbors\n",
    "    size = subgraph.number_of_nodes()\n",
    "\n",
    "    if size <= 1:  # Can't calculate a score for empty subgraphs\n",
    "        average_score = None\n",
    "    else:\n",
    "        average_score = workflow_aggregate(subgraph, node, aggregator=np.mean)\n",
    "\n",
    "    scores[node] = (average_score, fneighbors, size)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(426, 215)"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(candidate_mechanisms), sum(1 for score, _, _ in scores.values() if score)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Optionally, calcualte \"cute names\" for the nodes\n",
    "# scores = {calculate_canonical_name(graph, n): s for n, s in scores.items()}\n",
    "\n",
    "scores_df = pd.DataFrame.from_items(\n",
    "    scores.items(),\n",
    "    orient=\"index\",\n",
    "    columns=[\"Average Score\", \"Number First Neighbors\", \"Subgraph Size\"],\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A scatter matrix is shown below. The distribution of average scores is shown to be mostly normal. The following scatter plots show small, non-linear relationships between average score and the subgraph size and the number of first neighbors of the seed node. The trivial positive correlation between number of first neighbors is also shown."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/lib/python3.6/site-packages/ipykernel_launcher.py:1: FutureWarning: 'pandas.tools.plotting.scatter_matrix' is deprecated, import 'pandas.plotting.scatter_matrix' instead.\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    },
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"435pt\" version=\"1.1\" viewBox=\"0 0 611 435\" width=\"611pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 435.981875 \n",
       "L 611.848125 435.981875 \n",
       "L 611.848125 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 43.148125 137.54 \n",
       "L 229.148125 137.54 \n",
       "L 229.148125 10.7 \n",
       "L 43.148125 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 47.576696 137.54 \n",
       "L 65.290982 137.54 \n",
       "L 65.290982 137.090929 \n",
       "L 47.576696 137.090929 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 65.290982 137.54 \n",
       "L 83.005268 137.54 \n",
       "L 83.005268 137.54 \n",
       "L 65.290982 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 83.005268 137.54 \n",
       "L 100.719554 137.54 \n",
       "L 100.719554 137.54 \n",
       "L 83.005268 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 100.719554 137.54 \n",
       "L 118.433839 137.54 \n",
       "L 118.433839 137.54 \n",
       "L 100.719554 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 118.433839 137.54 \n",
       "L 136.148125 137.54 \n",
       "L 136.148125 137.090929 \n",
       "L 118.433839 137.090929 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 136.148125 137.54 \n",
       "L 153.862411 137.54 \n",
       "L 153.862411 136.641859 \n",
       "L 136.148125 136.641859 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 153.862411 137.54 \n",
       "L 171.576696 137.54 \n",
       "L 171.576696 131.253011 \n",
       "L 153.862411 131.253011 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 171.576696 137.54 \n",
       "L 189.290982 137.54 \n",
       "L 189.290982 16.74 \n",
       "L 171.576696 16.74 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 189.290982 137.54 \n",
       "L 207.005268 137.54 \n",
       "L 207.005268 128.558587 \n",
       "L 189.290982 128.558587 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#pb4bed2fb86)\" d=\"M 207.005268 137.54 \n",
       "L 224.719554 137.54 \n",
       "L 224.719554 136.192788 \n",
       "L 207.005268 136.192788 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"md377befec4\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"121.316796\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- -30 -->\n",
       "      <defs>\n",
       "       <path d=\"M 4.890625 31.390625 \n",
       "L 31.203125 31.390625 \n",
       "L 31.203125 23.390625 \n",
       "L 4.890625 23.390625 \n",
       "z\n",
       "\" id=\"DejaVuSans-2d\"/>\n",
       "       <path d=\"M 40.578125 39.3125 \n",
       "Q 47.65625 37.796875 51.625 33 \n",
       "Q 55.609375 28.21875 55.609375 21.1875 \n",
       "Q 55.609375 10.40625 48.1875 4.484375 \n",
       "Q 40.765625 -1.421875 27.09375 -1.421875 \n",
       "Q 22.515625 -1.421875 17.65625 -0.515625 \n",
       "Q 12.796875 0.390625 7.625 2.203125 \n",
       "L 7.625 11.71875 \n",
       "Q 11.71875 9.328125 16.59375 8.109375 \n",
       "Q 21.484375 6.890625 26.8125 6.890625 \n",
       "Q 36.078125 6.890625 40.9375 10.546875 \n",
       "Q 45.796875 14.203125 45.796875 21.1875 \n",
       "Q 45.796875 27.640625 41.28125 31.265625 \n",
       "Q 36.765625 34.90625 28.71875 34.90625 \n",
       "L 20.21875 34.90625 \n",
       "L 20.21875 43.015625 \n",
       "L 29.109375 43.015625 \n",
       "Q 36.375 43.015625 40.234375 45.921875 \n",
       "Q 44.09375 48.828125 44.09375 54.296875 \n",
       "Q 44.09375 59.90625 40.109375 62.90625 \n",
       "Q 36.140625 65.921875 28.71875 65.921875 \n",
       "Q 24.65625 65.921875 20.015625 65.03125 \n",
       "Q 15.375 64.15625 9.8125 62.3125 \n",
       "L 9.8125 71.09375 \n",
       "Q 15.4375 72.65625 20.34375 73.4375 \n",
       "Q 25.25 74.21875 29.59375 74.21875 \n",
       "Q 40.828125 74.21875 47.359375 69.109375 \n",
       "Q 53.90625 64.015625 53.90625 55.328125 \n",
       "Q 53.90625 49.265625 50.4375 45.09375 \n",
       "Q 46.96875 40.921875 40.578125 39.3125 \n",
       "z\n",
       "\" id=\"DejaVuSans-33\"/>\n",
       "       <path d=\"M 31.78125 66.40625 \n",
       "Q 24.171875 66.40625 20.328125 58.90625 \n",
       "Q 16.5 51.421875 16.5 36.375 \n",
       "Q 16.5 21.390625 20.328125 13.890625 \n",
       "Q 24.171875 6.390625 31.78125 6.390625 \n",
       "Q 39.453125 6.390625 43.28125 13.890625 \n",
       "Q 47.125 21.390625 47.125 36.375 \n",
       "Q 47.125 51.421875 43.28125 58.90625 \n",
       "Q 39.453125 66.40625 31.78125 66.40625 \n",
       "z\n",
       "M 31.78125 74.21875 \n",
       "Q 44.046875 74.21875 50.515625 64.515625 \n",
       "Q 56.984375 54.828125 56.984375 36.375 \n",
       "Q 56.984375 17.96875 50.515625 8.265625 \n",
       "Q 44.046875 -1.421875 31.78125 -1.421875 \n",
       "Q 19.53125 -1.421875 13.0625 8.265625 \n",
       "Q 6.59375 17.96875 6.59375 36.375 \n",
       "Q 6.59375 54.828125 13.0625 64.515625 \n",
       "Q 19.53125 74.21875 31.78125 74.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-30\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(23.081875 124.356171)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2d\"/>\n",
       "       <use x=\"36.083984\" xlink:href=\"#DejaVuSans-33\"/>\n",
       "       <use x=\"99.707031\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"95.105273\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- -20 -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.1875 8.296875 \n",
       "L 53.609375 8.296875 \n",
       "L 53.609375 0 \n",
       "L 7.328125 0 \n",
       "L 7.328125 8.296875 \n",
       "Q 12.9375 14.109375 22.625 23.890625 \n",
       "Q 32.328125 33.6875 34.8125 36.53125 \n",
       "Q 39.546875 41.84375 41.421875 45.53125 \n",
       "Q 43.3125 49.21875 43.3125 52.78125 \n",
       "Q 43.3125 58.59375 39.234375 62.25 \n",
       "Q 35.15625 65.921875 28.609375 65.921875 \n",
       "Q 23.96875 65.921875 18.8125 64.3125 \n",
       "Q 13.671875 62.703125 7.8125 59.421875 \n",
       "L 7.8125 69.390625 \n",
       "Q 13.765625 71.78125 18.9375 73 \n",
       "Q 24.125 74.21875 28.421875 74.21875 \n",
       "Q 39.75 74.21875 46.484375 68.546875 \n",
       "Q 53.21875 62.890625 53.21875 53.421875 \n",
       "Q 53.21875 48.921875 51.53125 44.890625 \n",
       "Q 49.859375 40.875 45.40625 35.40625 \n",
       "Q 44.1875 33.984375 37.640625 27.21875 \n",
       "Q 31.109375 20.453125 19.1875 8.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-32\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(23.081875 98.144648)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2d\"/>\n",
       "       <use x=\"36.083984\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"99.707031\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"68.893749\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- -10 -->\n",
       "      <defs>\n",
       "       <path d=\"M 12.40625 8.296875 \n",
       "L 28.515625 8.296875 \n",
       "L 28.515625 63.921875 \n",
       "L 10.984375 60.40625 \n",
       "L 10.984375 69.390625 \n",
       "L 28.421875 72.90625 \n",
       "L 38.28125 72.90625 \n",
       "L 38.28125 8.296875 \n",
       "L 54.390625 8.296875 \n",
       "L 54.390625 0 \n",
       "L 12.40625 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-31\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(23.081875 71.933124)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2d\"/>\n",
       "       <use x=\"36.083984\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"99.707031\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"42.682226\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(31.058125 45.721601)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"16.470703\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- 10 -->\n",
       "      <g transform=\"translate(25.968125 19.510078)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_6\">\n",
       "     <!-- Average Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 34.1875 63.1875 \n",
       "L 20.796875 26.90625 \n",
       "L 47.609375 26.90625 \n",
       "z\n",
       "M 28.609375 72.90625 \n",
       "L 39.796875 72.90625 \n",
       "L 67.578125 0 \n",
       "L 57.328125 0 \n",
       "L 50.6875 18.703125 \n",
       "L 17.828125 18.703125 \n",
       "L 11.1875 0 \n",
       "L 0.78125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-41\"/>\n",
       "      <path d=\"M 2.984375 54.6875 \n",
       "L 12.5 54.6875 \n",
       "L 29.59375 8.796875 \n",
       "L 46.6875 54.6875 \n",
       "L 56.203125 54.6875 \n",
       "L 35.6875 0 \n",
       "L 23.484375 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-76\"/>\n",
       "      <path d=\"M 56.203125 29.59375 \n",
       "L 56.203125 25.203125 \n",
       "L 14.890625 25.203125 \n",
       "Q 15.484375 15.921875 20.484375 11.0625 \n",
       "Q 25.484375 6.203125 34.421875 6.203125 \n",
       "Q 39.59375 6.203125 44.453125 7.46875 \n",
       "Q 49.3125 8.734375 54.109375 11.28125 \n",
       "L 54.109375 2.78125 \n",
       "Q 49.265625 0.734375 44.1875 -0.34375 \n",
       "Q 39.109375 -1.421875 33.890625 -1.421875 \n",
       "Q 20.796875 -1.421875 13.15625 6.1875 \n",
       "Q 5.515625 13.8125 5.515625 26.8125 \n",
       "Q 5.515625 40.234375 12.765625 48.109375 \n",
       "Q 20.015625 56 32.328125 56 \n",
       "Q 43.359375 56 49.78125 48.890625 \n",
       "Q 56.203125 41.796875 56.203125 29.59375 \n",
       "z\n",
       "M 47.21875 32.234375 \n",
       "Q 47.125 39.59375 43.09375 43.984375 \n",
       "Q 39.0625 48.390625 32.421875 48.390625 \n",
       "Q 24.90625 48.390625 20.390625 44.140625 \n",
       "Q 15.875 39.890625 15.1875 32.171875 \n",
       "z\n",
       "\" id=\"DejaVuSans-65\"/>\n",
       "      <path d=\"M 41.109375 46.296875 \n",
       "Q 39.59375 47.171875 37.8125 47.578125 \n",
       "Q 36.03125 48 33.890625 48 \n",
       "Q 26.265625 48 22.1875 43.046875 \n",
       "Q 18.109375 38.09375 18.109375 28.8125 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 20.953125 51.171875 25.484375 53.578125 \n",
       "Q 30.03125 56 36.53125 56 \n",
       "Q 37.453125 56 38.578125 55.875 \n",
       "Q 39.703125 55.765625 41.0625 55.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-72\"/>\n",
       "      <path d=\"M 34.28125 27.484375 \n",
       "Q 23.390625 27.484375 19.1875 25 \n",
       "Q 14.984375 22.515625 14.984375 16.5 \n",
       "Q 14.984375 11.71875 18.140625 8.90625 \n",
       "Q 21.296875 6.109375 26.703125 6.109375 \n",
       "Q 34.1875 6.109375 38.703125 11.40625 \n",
       "Q 43.21875 16.703125 43.21875 25.484375 \n",
       "L 43.21875 27.484375 \n",
       "z\n",
       "M 52.203125 31.203125 \n",
       "L 52.203125 0 \n",
       "L 43.21875 0 \n",
       "L 43.21875 8.296875 \n",
       "Q 40.140625 3.328125 35.546875 0.953125 \n",
       "Q 30.953125 -1.421875 24.3125 -1.421875 \n",
       "Q 15.921875 -1.421875 10.953125 3.296875 \n",
       "Q 6 8.015625 6 15.921875 \n",
       "Q 6 25.140625 12.171875 29.828125 \n",
       "Q 18.359375 34.515625 30.609375 34.515625 \n",
       "L 43.21875 34.515625 \n",
       "L 43.21875 35.40625 \n",
       "Q 43.21875 41.609375 39.140625 45 \n",
       "Q 35.0625 48.390625 27.6875 48.390625 \n",
       "Q 23 48.390625 18.546875 47.265625 \n",
       "Q 14.109375 46.140625 10.015625 43.890625 \n",
       "L 10.015625 52.203125 \n",
       "Q 14.9375 54.109375 19.578125 55.046875 \n",
       "Q 24.21875 56 28.609375 56 \n",
       "Q 40.484375 56 46.34375 49.84375 \n",
       "Q 52.203125 43.703125 52.203125 31.203125 \n",
       "z\n",
       "\" id=\"DejaVuSans-61\"/>\n",
       "      <path d=\"M 45.40625 27.984375 \n",
       "Q 45.40625 37.75 41.375 43.109375 \n",
       "Q 37.359375 48.484375 30.078125 48.484375 \n",
       "Q 22.859375 48.484375 18.828125 43.109375 \n",
       "Q 14.796875 37.75 14.796875 27.984375 \n",
       "Q 14.796875 18.265625 18.828125 12.890625 \n",
       "Q 22.859375 7.515625 30.078125 7.515625 \n",
       "Q 37.359375 7.515625 41.375 12.890625 \n",
       "Q 45.40625 18.265625 45.40625 27.984375 \n",
       "z\n",
       "M 54.390625 6.78125 \n",
       "Q 54.390625 -7.171875 48.1875 -13.984375 \n",
       "Q 42 -20.796875 29.203125 -20.796875 \n",
       "Q 24.46875 -20.796875 20.265625 -20.09375 \n",
       "Q 16.0625 -19.390625 12.109375 -17.921875 \n",
       "L 12.109375 -9.1875 \n",
       "Q 16.0625 -11.328125 19.921875 -12.34375 \n",
       "Q 23.78125 -13.375 27.78125 -13.375 \n",
       "Q 36.625 -13.375 41.015625 -8.765625 \n",
       "Q 45.40625 -4.15625 45.40625 5.171875 \n",
       "L 45.40625 9.625 \n",
       "Q 42.625 4.78125 38.28125 2.390625 \n",
       "Q 33.9375 0 27.875 0 \n",
       "Q 17.828125 0 11.671875 7.65625 \n",
       "Q 5.515625 15.328125 5.515625 27.984375 \n",
       "Q 5.515625 40.671875 11.671875 48.328125 \n",
       "Q 17.828125 56 27.875 56 \n",
       "Q 33.9375 56 38.28125 53.609375 \n",
       "Q 42.625 51.21875 45.40625 46.390625 \n",
       "L 45.40625 54.6875 \n",
       "L 54.390625 54.6875 \n",
       "z\n",
       "\" id=\"DejaVuSans-67\"/>\n",
       "      <path id=\"DejaVuSans-20\"/>\n",
       "      <path d=\"M 53.515625 70.515625 \n",
       "L 53.515625 60.890625 \n",
       "Q 47.90625 63.578125 42.921875 64.890625 \n",
       "Q 37.9375 66.21875 33.296875 66.21875 \n",
       "Q 25.25 66.21875 20.875 63.09375 \n",
       "Q 16.5 59.96875 16.5 54.203125 \n",
       "Q 16.5 49.359375 19.40625 46.890625 \n",
       "Q 22.3125 44.4375 30.421875 42.921875 \n",
       "L 36.375 41.703125 \n",
       "Q 47.40625 39.59375 52.65625 34.296875 \n",
       "Q 57.90625 29 57.90625 20.125 \n",
       "Q 57.90625 9.515625 50.796875 4.046875 \n",
       "Q 43.703125 -1.421875 29.984375 -1.421875 \n",
       "Q 24.8125 -1.421875 18.96875 -0.25 \n",
       "Q 13.140625 0.921875 6.890625 3.21875 \n",
       "L 6.890625 13.375 \n",
       "Q 12.890625 10.015625 18.65625 8.296875 \n",
       "Q 24.421875 6.59375 29.984375 6.59375 \n",
       "Q 38.421875 6.59375 43.015625 9.90625 \n",
       "Q 47.609375 13.234375 47.609375 19.390625 \n",
       "Q 47.609375 24.75 44.3125 27.78125 \n",
       "Q 41.015625 30.8125 33.5 32.328125 \n",
       "L 27.484375 33.5 \n",
       "Q 16.453125 35.6875 11.515625 40.375 \n",
       "Q 6.59375 45.0625 6.59375 53.421875 \n",
       "Q 6.59375 63.09375 13.40625 68.65625 \n",
       "Q 20.21875 74.21875 32.171875 74.21875 \n",
       "Q 37.3125 74.21875 42.625 73.28125 \n",
       "Q 47.953125 72.359375 53.515625 70.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-53\"/>\n",
       "      <path d=\"M 48.78125 52.59375 \n",
       "L 48.78125 44.1875 \n",
       "Q 44.96875 46.296875 41.140625 47.34375 \n",
       "Q 37.3125 48.390625 33.40625 48.390625 \n",
       "Q 24.65625 48.390625 19.8125 42.84375 \n",
       "Q 14.984375 37.3125 14.984375 27.296875 \n",
       "Q 14.984375 17.28125 19.8125 11.734375 \n",
       "Q 24.65625 6.203125 33.40625 6.203125 \n",
       "Q 37.3125 6.203125 41.140625 7.25 \n",
       "Q 44.96875 8.296875 48.78125 10.40625 \n",
       "L 48.78125 2.09375 \n",
       "Q 45.015625 0.34375 40.984375 -0.53125 \n",
       "Q 36.96875 -1.421875 32.421875 -1.421875 \n",
       "Q 20.0625 -1.421875 12.78125 6.34375 \n",
       "Q 5.515625 14.109375 5.515625 27.296875 \n",
       "Q 5.515625 40.671875 12.859375 48.328125 \n",
       "Q 20.21875 56 33.015625 56 \n",
       "Q 37.15625 56 41.109375 55.140625 \n",
       "Q 45.0625 54.296875 48.78125 52.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-63\"/>\n",
       "      <path d=\"M 30.609375 48.390625 \n",
       "Q 23.390625 48.390625 19.1875 42.75 \n",
       "Q 14.984375 37.109375 14.984375 27.296875 \n",
       "Q 14.984375 17.484375 19.15625 11.84375 \n",
       "Q 23.34375 6.203125 30.609375 6.203125 \n",
       "Q 37.796875 6.203125 41.984375 11.859375 \n",
       "Q 46.1875 17.53125 46.1875 27.296875 \n",
       "Q 46.1875 37.015625 41.984375 42.703125 \n",
       "Q 37.796875 48.390625 30.609375 48.390625 \n",
       "z\n",
       "M 30.609375 56 \n",
       "Q 42.328125 56 49.015625 48.375 \n",
       "Q 55.71875 40.765625 55.71875 27.296875 \n",
       "Q 55.71875 13.875 49.015625 6.21875 \n",
       "Q 42.328125 -1.421875 30.609375 -1.421875 \n",
       "Q 18.84375 -1.421875 12.171875 6.21875 \n",
       "Q 5.515625 13.875 5.515625 27.296875 \n",
       "Q 5.515625 40.765625 12.171875 48.375 \n",
       "Q 18.84375 56 30.609375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-6f\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(17.002187 110.645)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-41\"/>\n",
       "      <use x=\"68.330078\" xlink:href=\"#DejaVuSans-76\"/>\n",
       "      <use x=\"127.509766\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"189.033203\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"230.146484\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "      <use x=\"291.425781\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"354.902344\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"416.425781\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"448.212891\" xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"511.689453\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "      <use x=\"566.669922\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"627.851562\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"668.933594\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 43.148125 137.54 \n",
       "L 43.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 229.148125 137.54 \n",
       "L 229.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 43.148125 137.54 \n",
       "L 229.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 43.148125 10.7 \n",
       "L 229.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 229.148125 137.54 \n",
       "L 415.148125 137.54 \n",
       "L 415.148125 10.7 \n",
       "L 229.148125 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_1\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C0_0_dc54733189\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"260.829444\" xlink:href=\"#C0_0_dc54733189\" y=\"47.423738\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"39.820589\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"50.161872\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"315.334938\" xlink:href=\"#C0_0_dc54733189\" y=\"34.294562\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"44.074486\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"322.148125\" xlink:href=\"#C0_0_dc54733189\" y=\"69.85323\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"410.719554\" xlink:href=\"#C0_0_dc54733189\" y=\"64.537286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"263.100506\" xlink:href=\"#C0_0_dc54733189\" y=\"31.495474\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.987393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"301.708565\" xlink:href=\"#C0_0_dc54733189\" y=\"13.72\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"46.928447\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"51.853207\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.745702\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"39.991151\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"36.179562\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.941314\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"324.419187\" xlink:href=\"#C0_0_dc54733189\" y=\"40.446251\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"285.811129\" xlink:href=\"#C0_0_dc54733189\" y=\"43.080438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"48.882107\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"45.391119\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"44.306227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"41.963954\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.32309\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C0_0_dc54733189\" y=\"45.780228\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"46.090764\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"39.669619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"47.203328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.09818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"39.241215\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"38.767652\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.689206\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.675246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.675246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.222388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"42.105843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.046428\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"267.64263\" xlink:href=\"#C0_0_dc54733189\" y=\"38.103277\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.32309\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"36.251823\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"51.086434\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"267.64263\" xlink:href=\"#C0_0_dc54733189\" y=\"41.276981\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"37.69476\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"44.756036\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"45.344767\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"37.626165\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"43.460225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"44.228315\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"37.090737\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.586181\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"39.368007\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"41.658027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"254.016257\" xlink:href=\"#C0_0_dc54733189\" y=\"44.097697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"254.016257\" xlink:href=\"#C0_0_dc54733189\" y=\"40.932679\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"52.807917\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.608417\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.638783\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"44.946408\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"38.222757\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"57.866642\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C0_0_dc54733189\" y=\"83.089013\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"41.629686\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"45.244673\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"51.405723\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"43.366073\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"40.278257\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"44.132487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"36.868491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"43.003489\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"48.542798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.783485\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"40.584437\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"40.455093\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"36.46058\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"33.662315\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"37.69476\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"41.938941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.09818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"45.378642\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.381193\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"272.184755\" xlink:href=\"#C0_0_dc54733189\" y=\"39.395145\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"43.377179\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.973526\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"53.201342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.326434\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"33.993363\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"40.063412\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"27.556224\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.371568\" xlink:href=\"#C0_0_dc54733189\" y=\"35.053799\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.158552\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.006435\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.629976\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.993667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.084086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"40.872038\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"260.829444\" xlink:href=\"#C0_0_dc54733189\" y=\"47.482392\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.687199\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"59.517467\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.02415\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.919896\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.677181\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.236246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"45.598545\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"41.220796\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"49.564249\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"37.03611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C0_0_dc54733189\" y=\"34.850514\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"54.761706\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C0_0_dc54733189\" y=\"31.788605\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.084086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.25164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.155789\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"44.975299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C0_0_dc54733189\" y=\"46.099754\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"38.127591\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"36.563138\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"263.100506\" xlink:href=\"#C0_0_dc54733189\" y=\"54.197415\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"44.975299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.978713\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.231045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"44.530006\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"49.227376\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"47.21766\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.710717\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.762549\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.993667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.844452\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.805893\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.869272\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C0_0_dc54733189\" y=\"35.433602\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.25164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"43.048788\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"47.21766\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.844886\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.207653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.061537\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.784299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"43.158782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.104446\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.690164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"37.318405\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.022242\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.022197\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"25.376487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.474132\" xlink:href=\"#C0_0_dc54733189\" y=\"24.882789\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.604666\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.061537\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"39.564456\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.864327\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"56.613\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.536249\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.474132\" xlink:href=\"#C0_0_dc54733189\" y=\"134.52\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"51.380783\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"38.038635\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"47.552736\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"44.499625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.706425\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.175355\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"50.285366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"258.558381\" xlink:href=\"#C0_0_dc54733189\" y=\"53.025194\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.784299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.580153\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.996556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"38.580522\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"43.048788\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.976281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.976281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.222388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.142065\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"38.580522\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C0_0_dc54733189\" y=\"45.762549\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"47.091676\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.273487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C0_0_dc54733189\" y=\"42.40163\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"40.782033\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.847523\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"41.1922\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9db39a5f1a)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C0_0_dc54733189\" y=\"43.35054\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path d=\"M 229.148125 137.54 \n",
       "L 229.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 415.148125 137.54 \n",
       "L 415.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path d=\"M 229.148125 137.54 \n",
       "L 415.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path d=\"M 229.148125 10.7 \n",
       "L 415.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_3\">\n",
       "   <g id=\"patch_22\">\n",
       "    <path d=\"M 415.148125 137.54 \n",
       "L 601.148125 137.54 \n",
       "L 601.148125 10.7 \n",
       "L 415.148125 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_2\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C1_0_f64e14bb0f\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"522.224269\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.423738\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"515.727587\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.820589\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.837612\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"50.161872\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"538.682529\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"34.294562\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.074486\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"596.719554\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"69.85323\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"531.752736\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"64.537286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"521.358045\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"31.495474\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.207154\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.987393\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.938205\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"13.72\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.434887\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"46.928447\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"51.853207\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.774042\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.745702\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.991151\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"36.179562\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.941314\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"554.707678\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.446251\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"517.460036\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.080438\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"439.066742\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"48.882107\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.837612\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.391119\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.306227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.963954\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.32309\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.939602\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.780228\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.434887\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"536.950081\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"46.090764\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"538.249417\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.669619\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.289967\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.203328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.09818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.241215\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.767652\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.689206\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.675246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.675246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.222388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.03356\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.105843\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.046428\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"437.334293\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.103277\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.207154\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.32309\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.364681\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"36.251823\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"520.924933\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"51.086434\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"472.849487\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.276981\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.69476\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.774042\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.756036\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"436.034957\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.344767\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.626165\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.460225\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.228315\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.792554\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.090737\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.586181\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.03356\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.368007\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.658027\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"434.302509\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.097697\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.105163\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.932679\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"512.695802\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"52.807917\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.608417\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.638783\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.946408\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.222757\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.09713\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"57.866642\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.525002\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"83.089013\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.629686\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.34093\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.244673\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"51.405723\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.366073\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.278257\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.136948\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.132487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"36.868491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.474706\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.003489\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"48.542798\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.783485\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.404499\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.584437\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.931569\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.455093\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"36.46058\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.899784\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"33.662315\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.69476\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.938941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.09818\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.105163\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.378642\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.381193\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"515.727587\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.395145\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.377179\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.973526\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"53.201342\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.326434\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.238939\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"33.993363\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.734223\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.063412\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.359442\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"27.556224\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"526.988502\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"35.053799\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.271328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.158552\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.006435\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.629976\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.993667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.084086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.872038\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.869396\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.482392\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.687199\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"481.51173\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"59.517467\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.02415\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.919896\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.677181\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.236246\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.598545\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.220796\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"49.564249\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.332896\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.03611\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"459.856124\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"34.850514\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.098527\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"54.761706\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.797793\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"31.788605\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.084086\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.25164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.155789\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.975299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"537.816305\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"46.099754\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.498457\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.127591\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"503.600448\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"36.563138\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.09713\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"54.197415\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.975299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.978713\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.231045\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.530006\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"49.227376\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.21766\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.710717\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.653735\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.762549\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.993667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.844452\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.805893\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.869272\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.34093\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"35.433602\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.25164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.048788\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.21766\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.519566\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.844886\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.207653\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.061537\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.784299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.158782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.104446\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.690164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"37.318405\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.022242\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.022197\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"49.466466\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"25.376487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"517.026923\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"24.882789\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.604666\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.061537\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.665415\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.564456\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.864327\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.232302\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"56.613\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.665415\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.325817\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.536249\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"448.162096\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"134.52\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"51.380783\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.038635\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.098527\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.552736\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.499625\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.706425\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.175355\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"434.302509\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"50.285366\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.830975\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"53.025194\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.784299\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.933558\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.580153\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.996556\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.580522\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.048788\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.976281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.976281\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.222388\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.142065\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"38.580522\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"45.762549\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.073378\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"47.091676\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"39.430894\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.273487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.40163\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"40.782033\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"44.06846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.847523\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"41.1922\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"42.682226\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pcd438bef5e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C1_0_f64e14bb0f\" y=\"43.35054\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path d=\"M 415.148125 137.54 \n",
       "L 415.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path d=\"M 601.148125 137.54 \n",
       "L 601.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path d=\"M 415.148125 137.54 \n",
       "L 601.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path d=\"M 415.148125 10.7 \n",
       "L 601.148125 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_4\">\n",
       "   <g id=\"patch_27\">\n",
       "    <path d=\"M 43.148125 264.38 \n",
       "L 229.148125 264.38 \n",
       "L 229.148125 137.54 \n",
       "L 43.148125 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_3\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C2_0_7b6dac6d08\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.295908\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"242.775385\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.445275\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"171.280668\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"194.54872\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"205.606154\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.207299\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"142.404977\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"200.96\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"150.200354\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"140.56\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"198.653343\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"241.226667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.801428\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"224.719554\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"214.898462\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"176.02221\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"168.800471\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.223019\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.19516\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.784528\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.268241\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.527795\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"199.411282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.664986\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"225.739487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"173.157334\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.276569\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.867471\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.302214\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.775571\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.705974\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.2506\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.666658\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.619119\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.773311\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.294878\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.989317\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.238693\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.259164\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.259164\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.990414\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.094145\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.782029\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.963566\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"238.129231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.775571\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.678564\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.924877\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.3096\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"238.129231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.562621\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.207864\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.344541\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.663209\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.10806\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.981721\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.448369\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.923356\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.108947\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.750829\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.173262\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"247.421538\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.81449\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"247.421538\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"167.40047\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.289993\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.312635\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.928699\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.788359\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"159.982283\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"122.995835\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.792389\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.49132\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.456668\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.246125\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.774143\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.122245\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.774273\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.777825\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"173.654902\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.100442\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.325156\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.514828\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.37244\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"195.475857\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.562621\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.338892\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.773311\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.294866\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.623197\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.069152\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"235.031795\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.22984\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.355348\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"166.823545\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.837839\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"194.990403\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.089196\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"204.429915\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"193.435363\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"239.677949\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.550437\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.239919\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.325548\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.325813\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.12605\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.903414\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.209897\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"242.775385\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.241636\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"157.56149\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.747527\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.36682\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.256326\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.503677\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.972397\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.39199\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.157031\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.528475\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"193.733464\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"164.535406\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"198.223492\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.12605\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.481104\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.55449\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.886334\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.237417\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.927911\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.222047\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"165.36289\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"241.226667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.886334\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.746984\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.910548\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.539318\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.651026\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.598104\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.27432\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.731899\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.325813\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.011039\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.601167\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.441056\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"192.878415\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.481104\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.711397\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.598104\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.476816\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.944849\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.159116\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.032076\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.550101\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.629779\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.703703\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.114513\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.750325\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.750391\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"207.626314\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"208.350279\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"250.518974\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.89625\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.159116\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.820872\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.515479\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"161.820642\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.530163\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"47.576696\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"250.518974\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.49324\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.058358\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.106742\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.583868\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.747029\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.592968\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"171.099575\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"167.081851\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"244.324103\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.032076\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.465781\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.78799\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.263726\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.711397\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.817723\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.817723\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.990414\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.507444\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.263726\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.731899\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.782848\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.84831\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.660399\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.035399\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.006535\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.433925\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pb51cd27403)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.268903\" xlink:href=\"#C2_0_7b6dac6d08\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"262.908718\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(31.058125 265.948093)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"231.934359\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- 20 -->\n",
       "      <g transform=\"translate(25.968125 234.973734)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"200.96\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- 40 -->\n",
       "      <defs>\n",
       "       <path d=\"M 37.796875 64.3125 \n",
       "L 12.890625 25.390625 \n",
       "L 37.796875 25.390625 \n",
       "z\n",
       "M 35.203125 72.90625 \n",
       "L 47.609375 72.90625 \n",
       "L 47.609375 25.390625 \n",
       "L 58.015625 25.390625 \n",
       "L 58.015625 17.1875 \n",
       "L 47.609375 17.1875 \n",
       "L 47.609375 0 \n",
       "L 37.796875 0 \n",
       "L 37.796875 17.1875 \n",
       "L 4.890625 17.1875 \n",
       "L 4.890625 26.703125 \n",
       "z\n",
       "\" id=\"DejaVuSans-34\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.968125 203.999375)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"169.985641\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- 60 -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.015625 40.375 \n",
       "Q 26.375 40.375 22.484375 35.828125 \n",
       "Q 18.609375 31.296875 18.609375 23.390625 \n",
       "Q 18.609375 15.53125 22.484375 10.953125 \n",
       "Q 26.375 6.390625 33.015625 6.390625 \n",
       "Q 39.65625 6.390625 43.53125 10.953125 \n",
       "Q 47.40625 15.53125 47.40625 23.390625 \n",
       "Q 47.40625 31.296875 43.53125 35.828125 \n",
       "Q 39.65625 40.375 33.015625 40.375 \n",
       "z\n",
       "M 52.59375 71.296875 \n",
       "L 52.59375 62.3125 \n",
       "Q 48.875 64.0625 45.09375 64.984375 \n",
       "Q 41.3125 65.921875 37.59375 65.921875 \n",
       "Q 27.828125 65.921875 22.671875 59.328125 \n",
       "Q 17.53125 52.734375 16.796875 39.40625 \n",
       "Q 19.671875 43.65625 24.015625 45.921875 \n",
       "Q 28.375 48.1875 33.59375 48.1875 \n",
       "Q 44.578125 48.1875 50.953125 41.515625 \n",
       "Q 57.328125 34.859375 57.328125 23.390625 \n",
       "Q 57.328125 12.15625 50.6875 5.359375 \n",
       "Q 44.046875 -1.421875 33.015625 -1.421875 \n",
       "Q 20.359375 -1.421875 13.671875 8.265625 \n",
       "Q 6.984375 17.96875 6.984375 36.375 \n",
       "Q 6.984375 53.65625 15.1875 63.9375 \n",
       "Q 23.390625 74.21875 37.203125 74.21875 \n",
       "Q 40.921875 74.21875 44.703125 73.484375 \n",
       "Q 48.484375 72.75 52.59375 71.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-36\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.968125 173.025016)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"139.011282\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- 80 -->\n",
       "      <defs>\n",
       "       <path d=\"M 31.78125 34.625 \n",
       "Q 24.75 34.625 20.71875 30.859375 \n",
       "Q 16.703125 27.09375 16.703125 20.515625 \n",
       "Q 16.703125 13.921875 20.71875 10.15625 \n",
       "Q 24.75 6.390625 31.78125 6.390625 \n",
       "Q 38.8125 6.390625 42.859375 10.171875 \n",
       "Q 46.921875 13.96875 46.921875 20.515625 \n",
       "Q 46.921875 27.09375 42.890625 30.859375 \n",
       "Q 38.875 34.625 31.78125 34.625 \n",
       "z\n",
       "M 21.921875 38.8125 \n",
       "Q 15.578125 40.375 12.03125 44.71875 \n",
       "Q 8.5 49.078125 8.5 55.328125 \n",
       "Q 8.5 64.0625 14.71875 69.140625 \n",
       "Q 20.953125 74.21875 31.78125 74.21875 \n",
       "Q 42.671875 74.21875 48.875 69.140625 \n",
       "Q 55.078125 64.0625 55.078125 55.328125 \n",
       "Q 55.078125 49.078125 51.53125 44.71875 \n",
       "Q 48 40.375 41.703125 38.8125 \n",
       "Q 48.828125 37.15625 52.796875 32.3125 \n",
       "Q 56.78125 27.484375 56.78125 20.515625 \n",
       "Q 56.78125 9.90625 50.3125 4.234375 \n",
       "Q 43.84375 -1.421875 31.78125 -1.421875 \n",
       "Q 19.734375 -1.421875 13.25 4.234375 \n",
       "Q 6.78125 9.90625 6.78125 20.515625 \n",
       "Q 6.78125 27.484375 10.78125 32.3125 \n",
       "Q 14.796875 37.15625 21.921875 38.8125 \n",
       "z\n",
       "M 18.3125 54.390625 \n",
       "Q 18.3125 48.734375 21.84375 45.5625 \n",
       "Q 25.390625 42.390625 31.78125 42.390625 \n",
       "Q 38.140625 42.390625 41.71875 45.5625 \n",
       "Q 45.3125 48.734375 45.3125 54.390625 \n",
       "Q 45.3125 60.0625 41.71875 63.234375 \n",
       "Q 38.140625 66.40625 31.78125 66.40625 \n",
       "Q 25.390625 66.40625 21.84375 63.234375 \n",
       "Q 18.3125 60.0625 18.3125 54.390625 \n",
       "z\n",
       "\" id=\"DejaVuSans-38\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.968125 142.050657)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-38\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_12\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <defs>\n",
       "      <path d=\"M 9.8125 72.90625 \n",
       "L 23.09375 72.90625 \n",
       "L 55.421875 11.921875 \n",
       "L 55.421875 72.90625 \n",
       "L 64.984375 72.90625 \n",
       "L 64.984375 0 \n",
       "L 51.703125 0 \n",
       "L 19.390625 60.984375 \n",
       "L 19.390625 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-4e\"/>\n",
       "      <path d=\"M 8.5 21.578125 \n",
       "L 8.5 54.6875 \n",
       "L 17.484375 54.6875 \n",
       "L 17.484375 21.921875 \n",
       "Q 17.484375 14.15625 20.5 10.265625 \n",
       "Q 23.53125 6.390625 29.59375 6.390625 \n",
       "Q 36.859375 6.390625 41.078125 11.03125 \n",
       "Q 45.3125 15.671875 45.3125 23.6875 \n",
       "L 45.3125 54.6875 \n",
       "L 54.296875 54.6875 \n",
       "L 54.296875 0 \n",
       "L 45.3125 0 \n",
       "L 45.3125 8.40625 \n",
       "Q 42.046875 3.421875 37.71875 1 \n",
       "Q 33.40625 -1.421875 27.6875 -1.421875 \n",
       "Q 18.265625 -1.421875 13.375 4.4375 \n",
       "Q 8.5 10.296875 8.5 21.578125 \n",
       "z\n",
       "M 31.109375 56 \n",
       "z\n",
       "\" id=\"DejaVuSans-75\"/>\n",
       "      <path d=\"M 52 44.1875 \n",
       "Q 55.375 50.25 60.0625 53.125 \n",
       "Q 64.75 56 71.09375 56 \n",
       "Q 79.640625 56 84.28125 50.015625 \n",
       "Q 88.921875 44.046875 88.921875 33.015625 \n",
       "L 88.921875 0 \n",
       "L 79.890625 0 \n",
       "L 79.890625 32.71875 \n",
       "Q 79.890625 40.578125 77.09375 44.375 \n",
       "Q 74.3125 48.1875 68.609375 48.1875 \n",
       "Q 61.625 48.1875 57.5625 43.546875 \n",
       "Q 53.515625 38.921875 53.515625 30.90625 \n",
       "L 53.515625 0 \n",
       "L 44.484375 0 \n",
       "L 44.484375 32.71875 \n",
       "Q 44.484375 40.625 41.703125 44.40625 \n",
       "Q 38.921875 48.1875 33.109375 48.1875 \n",
       "Q 26.21875 48.1875 22.15625 43.53125 \n",
       "Q 18.109375 38.875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.1875 51.21875 25.484375 53.609375 \n",
       "Q 29.78125 56 35.6875 56 \n",
       "Q 41.65625 56 45.828125 52.96875 \n",
       "Q 50 49.953125 52 44.1875 \n",
       "z\n",
       "\" id=\"DejaVuSans-6d\"/>\n",
       "      <path d=\"M 48.6875 27.296875 \n",
       "Q 48.6875 37.203125 44.609375 42.84375 \n",
       "Q 40.53125 48.484375 33.40625 48.484375 \n",
       "Q 26.265625 48.484375 22.1875 42.84375 \n",
       "Q 18.109375 37.203125 18.109375 27.296875 \n",
       "Q 18.109375 17.390625 22.1875 11.75 \n",
       "Q 26.265625 6.109375 33.40625 6.109375 \n",
       "Q 40.53125 6.109375 44.609375 11.75 \n",
       "Q 48.6875 17.390625 48.6875 27.296875 \n",
       "z\n",
       "M 18.109375 46.390625 \n",
       "Q 20.953125 51.265625 25.265625 53.625 \n",
       "Q 29.59375 56 35.59375 56 \n",
       "Q 45.5625 56 51.78125 48.09375 \n",
       "Q 58.015625 40.1875 58.015625 27.296875 \n",
       "Q 58.015625 14.40625 51.78125 6.484375 \n",
       "Q 45.5625 -1.421875 35.59375 -1.421875 \n",
       "Q 29.59375 -1.421875 25.265625 0.953125 \n",
       "Q 20.953125 3.328125 18.109375 8.203125 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 75.984375 \n",
       "L 18.109375 75.984375 \n",
       "z\n",
       "\" id=\"DejaVuSans-62\"/>\n",
       "      <path d=\"M 9.8125 72.90625 \n",
       "L 51.703125 72.90625 \n",
       "L 51.703125 64.59375 \n",
       "L 19.671875 64.59375 \n",
       "L 19.671875 43.109375 \n",
       "L 48.578125 43.109375 \n",
       "L 48.578125 34.8125 \n",
       "L 19.671875 34.8125 \n",
       "L 19.671875 0 \n",
       "L 9.8125 0 \n",
       "z\n",
       "\" id=\"DejaVuSans-46\"/>\n",
       "      <path d=\"M 9.421875 54.6875 \n",
       "L 18.40625 54.6875 \n",
       "L 18.40625 0 \n",
       "L 9.421875 0 \n",
       "z\n",
       "M 9.421875 75.984375 \n",
       "L 18.40625 75.984375 \n",
       "L 18.40625 64.59375 \n",
       "L 9.421875 64.59375 \n",
       "z\n",
       "\" id=\"DejaVuSans-69\"/>\n",
       "      <path d=\"M 44.28125 53.078125 \n",
       "L 44.28125 44.578125 \n",
       "Q 40.484375 46.53125 36.375 47.5 \n",
       "Q 32.28125 48.484375 27.875 48.484375 \n",
       "Q 21.1875 48.484375 17.84375 46.4375 \n",
       "Q 14.5 44.390625 14.5 40.28125 \n",
       "Q 14.5 37.15625 16.890625 35.375 \n",
       "Q 19.28125 33.59375 26.515625 31.984375 \n",
       "L 29.59375 31.296875 \n",
       "Q 39.15625 29.25 43.1875 25.515625 \n",
       "Q 47.21875 21.78125 47.21875 15.09375 \n",
       "Q 47.21875 7.46875 41.1875 3.015625 \n",
       "Q 35.15625 -1.421875 24.609375 -1.421875 \n",
       "Q 20.21875 -1.421875 15.453125 -0.5625 \n",
       "Q 10.6875 0.296875 5.421875 2 \n",
       "L 5.421875 11.28125 \n",
       "Q 10.40625 8.6875 15.234375 7.390625 \n",
       "Q 20.0625 6.109375 24.8125 6.109375 \n",
       "Q 31.15625 6.109375 34.5625 8.28125 \n",
       "Q 37.984375 10.453125 37.984375 14.40625 \n",
       "Q 37.984375 18.0625 35.515625 20.015625 \n",
       "Q 33.0625 21.96875 24.703125 23.78125 \n",
       "L 21.578125 24.515625 \n",
       "Q 13.234375 26.265625 9.515625 29.90625 \n",
       "Q 5.8125 33.546875 5.8125 39.890625 \n",
       "Q 5.8125 47.609375 11.28125 51.796875 \n",
       "Q 16.75 56 26.8125 56 \n",
       "Q 31.78125 56 36.171875 55.265625 \n",
       "Q 40.578125 54.546875 44.28125 53.078125 \n",
       "z\n",
       "\" id=\"DejaVuSans-73\"/>\n",
       "      <path d=\"M 18.3125 70.21875 \n",
       "L 18.3125 54.6875 \n",
       "L 36.8125 54.6875 \n",
       "L 36.8125 47.703125 \n",
       "L 18.3125 47.703125 \n",
       "L 18.3125 18.015625 \n",
       "Q 18.3125 11.328125 20.140625 9.421875 \n",
       "Q 21.96875 7.515625 27.59375 7.515625 \n",
       "L 36.8125 7.515625 \n",
       "L 36.8125 0 \n",
       "L 27.59375 0 \n",
       "Q 17.1875 0 13.234375 3.875 \n",
       "Q 9.28125 7.765625 9.28125 18.015625 \n",
       "L 9.28125 47.703125 \n",
       "L 2.6875 47.703125 \n",
       "L 2.6875 54.6875 \n",
       "L 9.28125 54.6875 \n",
       "L 9.28125 70.21875 \n",
       "z\n",
       "\" id=\"DejaVuSans-74\"/>\n",
       "      <path d=\"M 54.890625 33.015625 \n",
       "L 54.890625 0 \n",
       "L 45.90625 0 \n",
       "L 45.90625 32.71875 \n",
       "Q 45.90625 40.484375 42.875 44.328125 \n",
       "Q 39.84375 48.1875 33.796875 48.1875 \n",
       "Q 26.515625 48.1875 22.3125 43.546875 \n",
       "Q 18.109375 38.921875 18.109375 30.90625 \n",
       "L 18.109375 0 \n",
       "L 9.078125 0 \n",
       "L 9.078125 75.984375 \n",
       "L 18.109375 75.984375 \n",
       "L 18.109375 46.1875 \n",
       "Q 21.34375 51.125 25.703125 53.5625 \n",
       "Q 30.078125 56 35.796875 56 \n",
       "Q 45.21875 56 50.046875 50.171875 \n",
       "Q 54.890625 44.34375 54.890625 33.015625 \n",
       "z\n",
       "\" id=\"DejaVuSans-68\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(19.888437 260.546719)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-4e\"/>\n",
       "      <use x=\"74.804688\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"138.183594\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "      <use x=\"235.595703\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"299.072266\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"360.595703\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"401.708984\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"433.496094\" xlink:href=\"#DejaVuSans-46\"/>\n",
       "      <use x=\"490.90625\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"518.689453\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"559.802734\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "      <use x=\"611.902344\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "      <use x=\"651.111328\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"682.898438\" xlink:href=\"#DejaVuSans-4e\"/>\n",
       "      <use x=\"757.703125\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"819.226562\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"847.009766\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"910.486328\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "      <use x=\"973.865234\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"1037.341797\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"1098.523438\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"1139.636719\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path d=\"M 43.148125 264.38 \n",
       "L 43.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path d=\"M 229.148125 264.38 \n",
       "L 229.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path d=\"M 43.148125 264.38 \n",
       "L 229.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path d=\"M 43.148125 137.54 \n",
       "L 229.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_5\">\n",
       "   <g id=\"patch_32\">\n",
       "    <path d=\"M 229.148125 264.38 \n",
       "L 415.148125 264.38 \n",
       "L 415.148125 137.54 \n",
       "L 229.148125 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 233.576696 264.38 \n",
       "L 251.290982 264.38 \n",
       "L 251.290982 143.58 \n",
       "L 233.576696 143.58 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 251.290982 264.38 \n",
       "L 269.005268 264.38 \n",
       "L 269.005268 258.964828 \n",
       "L 251.290982 258.964828 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 269.005268 264.38 \n",
       "L 286.719554 264.38 \n",
       "L 286.719554 263.546897 \n",
       "L 269.005268 263.546897 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 286.719554 264.38 \n",
       "L 304.433839 264.38 \n",
       "L 304.433839 263.963448 \n",
       "L 286.719554 263.963448 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 304.433839 264.38 \n",
       "L 322.148125 264.38 \n",
       "L 322.148125 263.963448 \n",
       "L 304.433839 263.963448 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 322.148125 264.38 \n",
       "L 339.862411 264.38 \n",
       "L 339.862411 263.546897 \n",
       "L 322.148125 263.546897 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_39\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 339.862411 264.38 \n",
       "L 357.576696 264.38 \n",
       "L 357.576696 264.38 \n",
       "L 339.862411 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_40\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 357.576696 264.38 \n",
       "L 375.290982 264.38 \n",
       "L 375.290982 264.38 \n",
       "L 357.576696 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_41\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 375.290982 264.38 \n",
       "L 393.005268 264.38 \n",
       "L 393.005268 264.38 \n",
       "L 375.290982 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_42\">\n",
       "    <path clip-path=\"url(#p2878240e32)\" d=\"M 393.005268 264.38 \n",
       "L 410.719554 264.38 \n",
       "L 410.719554 263.963448 \n",
       "L 393.005268 263.963448 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_43\">\n",
       "    <path d=\"M 229.148125 264.38 \n",
       "L 229.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_44\">\n",
       "    <path d=\"M 415.148125 264.38 \n",
       "L 415.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_45\">\n",
       "    <path d=\"M 229.148125 264.38 \n",
       "L 415.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_46\">\n",
       "    <path d=\"M 229.148125 137.54 \n",
       "L 415.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_6\">\n",
       "   <g id=\"patch_47\">\n",
       "    <path d=\"M 415.148125 264.38 \n",
       "L 601.148125 264.38 \n",
       "L 601.148125 137.54 \n",
       "L 415.148125 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_4\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C3_0_2f6aa201e3\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"522.224269\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"242.775385\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"515.727587\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.837612\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"538.682529\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"205.606154\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"596.719554\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"200.96\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"531.752736\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"140.56\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"521.358045\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"241.226667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.207154\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"494.938205\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"214.898462\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.434887\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.774042\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"554.707678\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"199.411282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"517.460036\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"225.739487\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"439.066742\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.837612\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.939602\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.434887\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"536.950081\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"538.249417\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"516.160699\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.971387\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.03356\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"437.334293\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"238.129231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.207154\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.364681\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"520.924933\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"472.849487\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"238.129231\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.774042\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"436.034957\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.792554\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.03356\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"434.302509\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"247.421538\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.105163\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"247.421538\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"512.695802\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.09713\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"455.525002\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.34093\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.136948\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.474706\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"425.640266\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"430.404499\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.931569\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"504.899784\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"432.57006\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.105163\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"515.727587\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"235.031795\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.003172\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.238939\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.734223\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"453.359442\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"526.988502\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"239.677949\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"433.869396\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"242.775385\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"481.51173\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"505.332896\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"459.856124\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"245.872821\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.098527\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"508.797793\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"252.067692\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"537.816305\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"253.61641\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"507.498457\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"503.600448\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"510.09713\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"241.226667\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"456.824339\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"424.34093\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"255.165128\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"428.672051\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.608481\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"517.026923\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"250.518974\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"429.538275\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.665415\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"422.175369\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.232302\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"441.665415\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"448.162096\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"250.518974\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.50649\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"440.366078\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"442.098527\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.041593\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"434.302509\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"443.830975\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"244.324103\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"502.301111\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"256.713846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"423.907818\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"259.811282\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"426.073378\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"501.001775\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"258.262564\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.876033\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.309145\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"421.742257\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.009809\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"419.576696\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#pc1f1d1e2b7)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"420.442921\" xlink:href=\"#C3_0_2f6aa201e3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_48\">\n",
       "    <path d=\"M 415.148125 264.38 \n",
       "L 415.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_49\">\n",
       "    <path d=\"M 601.148125 264.38 \n",
       "L 601.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_50\">\n",
       "    <path d=\"M 415.148125 264.38 \n",
       "L 601.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_51\">\n",
       "    <path d=\"M 415.148125 137.54 \n",
       "L 601.148125 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_7\">\n",
       "   <g id=\"patch_52\">\n",
       "    <path d=\"M 43.148125 391.22 \n",
       "L 229.148125 391.22 \n",
       "L 229.148125 264.38 \n",
       "L 43.148125 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_5\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C4_0_a65845ac24\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.295908\" xlink:href=\"#C4_0_a65845ac24\" y=\"318.200978\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C4_0_a65845ac24\" y=\"322.631296\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C4_0_a65845ac24\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.445275\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"171.280668\" xlink:href=\"#C4_0_a65845ac24\" y=\"380.520782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"194.54872\" xlink:href=\"#C4_0_a65845ac24\" y=\"306.977506\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.207299\" xlink:href=\"#C4_0_a65845ac24\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"142.404977\" xlink:href=\"#C4_0_a65845ac24\" y=\"267.4\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"150.200354\" xlink:href=\"#C4_0_a65845ac24\" y=\"311.703178\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"198.653343\" xlink:href=\"#C4_0_a65845ac24\" y=\"318.791687\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.801428\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.360391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"224.719554\" xlink:href=\"#C4_0_a65845ac24\" y=\"336.808313\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"176.02221\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.377995\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"168.800471\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.223019\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.655746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.19516\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.784528\" xlink:href=\"#C4_0_a65845ac24\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.268241\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.527795\" xlink:href=\"#C4_0_a65845ac24\" y=\"296.049389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.664986\" xlink:href=\"#C4_0_a65845ac24\" y=\"321.449878\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"173.157334\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.909046\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.276569\" xlink:href=\"#C4_0_a65845ac24\" y=\"380.520782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.867471\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.302214\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.775571\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.705974\" xlink:href=\"#C4_0_a65845ac24\" y=\"383.178973\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.377995\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.2506\" xlink:href=\"#C4_0_a65845ac24\" y=\"308.158924\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.666658\" xlink:href=\"#C4_0_a65845ac24\" y=\"307.272861\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.290558\" xlink:href=\"#C4_0_a65845ac24\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.619119\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.773311\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.294878\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.989317\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.238693\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.259164\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.259164\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.990414\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.094145\" xlink:href=\"#C4_0_a65845ac24\" y=\"330.605868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.782029\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.963566\" xlink:href=\"#C4_0_a65845ac24\" y=\"376.090465\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.775571\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.360391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C4_0_a65845ac24\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.678564\" xlink:href=\"#C4_0_a65845ac24\" y=\"327.652323\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.924877\" xlink:href=\"#C4_0_a65845ac24\" y=\"319.087042\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C4_0_a65845ac24\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.3096\" xlink:href=\"#C4_0_a65845ac24\" y=\"351.871394\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.562621\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.207864\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.655746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.344541\" xlink:href=\"#C4_0_a65845ac24\" y=\"376.976528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.663209\" xlink:href=\"#C4_0_a65845ac24\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.10806\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.981721\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.448369\" xlink:href=\"#C4_0_a65845ac24\" y=\"364.866993\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.923356\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.108947\" xlink:href=\"#C4_0_a65845ac24\" y=\"330.605868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.750829\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.173262\" xlink:href=\"#C4_0_a65845ac24\" y=\"378.157946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.81449\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.7022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"167.40047\" xlink:href=\"#C4_0_a65845ac24\" y=\"324.698778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.289993\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.312635\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.928699\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.788359\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"159.982283\" xlink:href=\"#C4_0_a65845ac24\" y=\"326.470905\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"122.995835\" xlink:href=\"#C4_0_a65845ac24\" y=\"363.685575\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.792389\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.49132\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.9511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.456668\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.246125\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.774143\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.122245\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.634719\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.774273\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.777825\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.541809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"173.654902\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.100442\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.325156\" xlink:href=\"#C4_0_a65845ac24\" y=\"380.816137\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.514828\" xlink:href=\"#C4_0_a65845ac24\" y=\"327.947677\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.37244\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"195.475857\" xlink:href=\"#C4_0_a65845ac24\" y=\"330.015159\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.562621\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.338892\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.773311\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.294866\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.7022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.623197\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.069152\" xlink:href=\"#C4_0_a65845ac24\" y=\"322.631296\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C4_0_a65845ac24\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.22984\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.355348\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"166.823545\" xlink:href=\"#C4_0_a65845ac24\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.837839\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"194.990403\" xlink:href=\"#C4_0_a65845ac24\" y=\"382.29291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.089196\" xlink:href=\"#C4_0_a65845ac24\" y=\"331.491932\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"204.429915\" xlink:href=\"#C4_0_a65845ac24\" y=\"365.162347\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"193.435363\" xlink:href=\"#C4_0_a65845ac24\" y=\"314.952078\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.385061\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.550437\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.239919\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.325548\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.325813\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.12605\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.903414\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.209897\" xlink:href=\"#C4_0_a65845ac24\" y=\"378.453301\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.241636\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"157.56149\" xlink:href=\"#C4_0_a65845ac24\" y=\"345.964303\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.747527\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.36682\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.256326\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.503677\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.972397\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.39199\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.157031\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.528475\" xlink:href=\"#C4_0_a65845ac24\" y=\"329.719804\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"193.733464\" xlink:href=\"#C4_0_a65845ac24\" y=\"360.732029\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"164.535406\" xlink:href=\"#C4_0_a65845ac24\" y=\"372.841565\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"198.223492\" xlink:href=\"#C4_0_a65845ac24\" y=\"327.356968\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.12605\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.481104\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.55449\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.886334\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.237417\" xlink:href=\"#C4_0_a65845ac24\" y=\"307.568215\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.927911\" xlink:href=\"#C4_0_a65845ac24\" y=\"328.243032\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"191.222047\" xlink:href=\"#C4_0_a65845ac24\" y=\"330.901222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"165.36289\" xlink:href=\"#C4_0_a65845ac24\" y=\"326.470905\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.886334\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.746984\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C4_0_a65845ac24\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.910548\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.539318\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.651026\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.598104\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.27432\" xlink:href=\"#C4_0_a65845ac24\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.223537\" xlink:href=\"#C4_0_a65845ac24\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.731899\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C4_0_a65845ac24\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.325813\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.011039\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.601167\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.441056\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"192.878415\" xlink:href=\"#C4_0_a65845ac24\" y=\"384.9511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.481104\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.711397\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.598104\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.021041\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.476816\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.944849\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.159116\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.032076\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.550101\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.629779\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.703703\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"190.114513\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.750325\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.750391\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"172.300422\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"207.626314\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"208.350279\" xlink:href=\"#C4_0_a65845ac24\" y=\"321.745232\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.89625\" xlink:href=\"#C4_0_a65845ac24\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"183.159116\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"373.136919\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"186.820872\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.515479\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"161.820642\" xlink:href=\"#C4_0_a65845ac24\" y=\"373.432274\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.4395\" xlink:href=\"#C4_0_a65845ac24\" y=\"373.136919\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.530163\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"47.576696\" xlink:href=\"#C4_0_a65845ac24\" y=\"368.706601\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"169.49324\" xlink:href=\"#C4_0_a65845ac24\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"189.058358\" xlink:href=\"#C4_0_a65845ac24\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.106742\" xlink:href=\"#C4_0_a65845ac24\" y=\"372.841565\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.583868\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.747029\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"178.592968\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"171.099575\" xlink:href=\"#C4_0_a65845ac24\" y=\"378.157946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"167.081851\" xlink:href=\"#C4_0_a65845ac24\" y=\"371.660147\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.032076\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.481129\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.465781\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.78799\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.263726\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.711397\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.817723\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.817723\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"179.990414\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.507444\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"188.263726\" xlink:href=\"#C4_0_a65845ac24\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"177.731899\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"175.782848\" xlink:href=\"#C4_0_a65845ac24\" y=\"383.769682\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"187.016728\" xlink:href=\"#C4_0_a65845ac24\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.84831\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.660399\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"185.035399\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"180.216136\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.006535\" xlink:href=\"#C4_0_a65845ac24\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"184.433925\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"182.248929\" xlink:href=\"#C4_0_a65845ac24\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p89e041b7a4)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"181.268903\" xlink:href=\"#C4_0_a65845ac24\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m49b4dede80\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"66.938065\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- 30 -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 35.5 \n",
       "L 73.1875 35.5 \n",
       "L 73.1875 27.203125 \n",
       "L 10.59375 27.203125 \n",
       "z\n",
       "\" id=\"DejaVuSans-2212\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(69.145565 415.10375)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
       "       <use x=\"83.789062\" xlink:href=\"#DejaVuSans-33\"/>\n",
       "       <use x=\"147.412109\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"105.375019\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- 20 -->\n",
       "      <g transform=\"translate(107.582519 415.10375)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
       "       <use x=\"83.789062\" xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"147.412109\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"143.811974\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- 10 -->\n",
       "      <g transform=\"translate(146.019474 415.10375)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-2212\"/>\n",
       "       <use x=\"83.789062\" xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"147.412109\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"182.248929\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(184.456429 403.31)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"220.685883\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- 10 -->\n",
       "      <g transform=\"translate(222.893383 408.4)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_18\">\n",
       "     <!-- Average Score -->\n",
       "     <g transform=\"translate(99.623125 426.702187)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-41\"/>\n",
       "      <use x=\"68.330078\" xlink:href=\"#DejaVuSans-76\"/>\n",
       "      <use x=\"127.509766\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"189.033203\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"230.146484\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "      <use x=\"291.425781\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"354.902344\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"416.425781\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"448.212891\" xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"511.689453\" xlink:href=\"#DejaVuSans-63\"/>\n",
       "      <use x=\"566.669922\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"627.851562\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"668.933594\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_11\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"388.790709\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(31.058125 391.830084)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_12\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"359.255257\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- 100 -->\n",
       "      <g transform=\"translate(20.878125 362.294632)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_13\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"329.719804\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- 200 -->\n",
       "      <g transform=\"translate(20.878125 332.759179)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_14\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"300.184352\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_22\">\n",
       "      <!-- 300 -->\n",
       "      <g transform=\"translate(20.878125 303.223727)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-33\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_15\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"43.148125\" xlink:href=\"#md377befec4\" y=\"270.6489\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_23\">\n",
       "      <!-- 400 -->\n",
       "      <g transform=\"translate(20.878125 273.688275)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_24\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <defs>\n",
       "      <path d=\"M 18.109375 8.203125 \n",
       "L 18.109375 -20.796875 \n",
       "L 9.078125 -20.796875 \n",
       "L 9.078125 54.6875 \n",
       "L 18.109375 54.6875 \n",
       "L 18.109375 46.390625 \n",
       "Q 20.953125 51.265625 25.265625 53.625 \n",
       "Q 29.59375 56 35.59375 56 \n",
       "Q 45.5625 56 51.78125 48.09375 \n",
       "Q 58.015625 40.1875 58.015625 27.296875 \n",
       "Q 58.015625 14.40625 51.78125 6.484375 \n",
       "Q 45.5625 -1.421875 35.59375 -1.421875 \n",
       "Q 29.59375 -1.421875 25.265625 0.953125 \n",
       "Q 20.953125 3.328125 18.109375 8.203125 \n",
       "z\n",
       "M 48.6875 27.296875 \n",
       "Q 48.6875 37.203125 44.609375 42.84375 \n",
       "Q 40.53125 48.484375 33.40625 48.484375 \n",
       "Q 26.265625 48.484375 22.1875 42.84375 \n",
       "Q 18.109375 37.203125 18.109375 27.296875 \n",
       "Q 18.109375 17.390625 22.1875 11.75 \n",
       "Q 26.265625 6.109375 33.40625 6.109375 \n",
       "Q 40.53125 6.109375 44.609375 11.75 \n",
       "Q 48.6875 17.390625 48.6875 27.296875 \n",
       "z\n",
       "\" id=\"DejaVuSans-70\"/>\n",
       "      <path d=\"M 5.515625 54.6875 \n",
       "L 48.1875 54.6875 \n",
       "L 48.1875 46.484375 \n",
       "L 14.40625 7.171875 \n",
       "L 48.1875 7.171875 \n",
       "L 48.1875 0 \n",
       "L 4.296875 0 \n",
       "L 4.296875 8.203125 \n",
       "L 38.09375 47.515625 \n",
       "L 5.515625 47.515625 \n",
       "z\n",
       "\" id=\"DejaVuSans-7a\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.798437 363.807031)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"63.476562\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"126.855469\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"190.332031\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"253.808594\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"294.921875\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "      <use x=\"356.201172\" xlink:href=\"#DejaVuSans-70\"/>\n",
       "      <use x=\"419.677734\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "      <use x=\"483.056641\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"514.84375\" xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"578.320312\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"606.103516\" xlink:href=\"#DejaVuSans-7a\"/>\n",
       "      <use x=\"658.59375\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_53\">\n",
       "    <path d=\"M 43.148125 391.22 \n",
       "L 43.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_54\">\n",
       "    <path d=\"M 229.148125 391.22 \n",
       "L 229.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_55\">\n",
       "    <path d=\"M 43.148125 391.22 \n",
       "L 229.148125 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_56\">\n",
       "    <path d=\"M 43.148125 264.38 \n",
       "L 229.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_8\">\n",
       "   <g id=\"patch_57\">\n",
       "    <path d=\"M 229.148125 391.22 \n",
       "L 415.148125 391.22 \n",
       "L 415.148125 264.38 \n",
       "L 229.148125 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_6\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"C5_0_cf64af4a25\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"260.829444\" xlink:href=\"#C5_0_cf64af4a25\" y=\"318.200978\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"322.631296\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"380.520782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"315.334938\" xlink:href=\"#C5_0_cf64af4a25\" y=\"306.977506\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"322.148125\" xlink:href=\"#C5_0_cf64af4a25\" y=\"267.4\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"410.719554\" xlink:href=\"#C5_0_cf64af4a25\" y=\"311.703178\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"263.100506\" xlink:href=\"#C5_0_cf64af4a25\" y=\"318.791687\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.360391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"301.708565\" xlink:href=\"#C5_0_cf64af4a25\" y=\"336.808313\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.377995\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.655746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"324.419187\" xlink:href=\"#C5_0_cf64af4a25\" y=\"296.049389\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"285.811129\" xlink:href=\"#C5_0_cf64af4a25\" y=\"321.449878\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.909046\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"380.520782\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C5_0_cf64af4a25\" y=\"383.178973\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.377995\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"308.158924\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"307.272861\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"322.335941\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.111491\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"330.605868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"267.64263\" xlink:href=\"#C5_0_cf64af4a25\" y=\"376.090465\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.360391\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"327.652323\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"319.087042\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"267.64263\" xlink:href=\"#C5_0_cf64af4a25\" y=\"351.871394\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.655746\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"376.976528\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"364.866993\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"330.605868\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"254.016257\" xlink:href=\"#C5_0_cf64af4a25\" y=\"378.157946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"254.016257\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.7022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"324.698778\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"326.470905\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C5_0_cf64af4a25\" y=\"363.685575\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.9511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.634719\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.541809\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.065037\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"380.816137\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"327.947677\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"330.015159\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.339364\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.7022\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"272.184755\" xlink:href=\"#C5_0_cf64af4a25\" y=\"322.631296\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"379.04401\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"382.29291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"331.491932\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"365.162347\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"265.371568\" xlink:href=\"#C5_0_cf64af4a25\" y=\"314.952078\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"260.829444\" xlink:href=\"#C5_0_cf64af4a25\" y=\"378.453301\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"345.964303\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"329.719804\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"256.287319\" xlink:href=\"#C5_0_cf64af4a25\" y=\"360.732029\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"372.841565\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"247.20307\" xlink:href=\"#C5_0_cf64af4a25\" y=\"327.356968\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"244.932008\" xlink:href=\"#C5_0_cf64af4a25\" y=\"307.568215\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"328.243032\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"330.901222\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"263.100506\" xlink:href=\"#C5_0_cf64af4a25\" y=\"326.470905\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"362.799511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"242.660946\" xlink:href=\"#C5_0_cf64af4a25\" y=\"384.9511\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.997555\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.132518\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.474132\" xlink:href=\"#C5_0_cf64af4a25\" y=\"321.745232\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"381.406846\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"373.136919\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.427873\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"373.432274\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"373.136919\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"249.474132\" xlink:href=\"#C5_0_cf64af4a25\" y=\"368.706601\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"383.474328\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"374.022983\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"372.841565\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.837164\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"378.157946\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"258.558381\" xlink:href=\"#C5_0_cf64af4a25\" y=\"371.660147\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"240.389883\" xlink:href=\"#C5_0_cf64af4a25\" y=\"331.787286\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"385.246455\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"235.847759\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"383.769682\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"332.67335\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"238.118821\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.313936\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.018582\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"386.723227\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.904645\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"388.2\"/>\n",
       "    </g>\n",
       "    <g clip-path=\"url(#p9979a80287)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;\" x=\"233.576696\" xlink:href=\"#C5_0_cf64af4a25\" y=\"387.609291\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_5\">\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"231.305634\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(233.513134 403.31)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"276.72688\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_26\">\n",
       "      <!-- 20 -->\n",
       "      <g transform=\"translate(278.93438 408.4)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"322.148125\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_27\">\n",
       "      <!-- 40 -->\n",
       "      <g transform=\"translate(324.355625 408.4)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"367.56937\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_28\">\n",
       "      <!-- 60 -->\n",
       "      <g transform=\"translate(369.77687 408.4)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-36\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_25\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"412.990616\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_29\">\n",
       "      <!-- 80 -->\n",
       "      <g transform=\"translate(415.198116 408.4)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-38\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_30\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <g transform=\"translate(262.561406 419.998437)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-4e\"/>\n",
       "      <use x=\"74.804688\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"138.183594\" xlink:href=\"#DejaVuSans-6d\"/>\n",
       "      <use x=\"235.595703\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"299.072266\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"360.595703\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"401.708984\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"433.496094\" xlink:href=\"#DejaVuSans-46\"/>\n",
       "      <use x=\"490.90625\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"518.689453\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"559.802734\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "      <use x=\"611.902344\" xlink:href=\"#DejaVuSans-74\"/>\n",
       "      <use x=\"651.111328\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"682.898438\" xlink:href=\"#DejaVuSans-4e\"/>\n",
       "      <use x=\"757.703125\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "      <use x=\"819.226562\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"847.009766\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"910.486328\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "      <use x=\"973.865234\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"1037.341797\" xlink:href=\"#DejaVuSans-6f\"/>\n",
       "      <use x=\"1098.523438\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"1139.636719\" xlink:href=\"#DejaVuSans-73\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_58\">\n",
       "    <path d=\"M 229.148125 391.22 \n",
       "L 229.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_59\">\n",
       "    <path d=\"M 415.148125 391.22 \n",
       "L 415.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_60\">\n",
       "    <path d=\"M 229.148125 391.22 \n",
       "L 415.148125 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_61\">\n",
       "    <path d=\"M 229.148125 264.38 \n",
       "L 415.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_9\">\n",
       "   <g id=\"patch_62\">\n",
       "    <path d=\"M 415.148125 391.22 \n",
       "L 601.148125 391.22 \n",
       "L 601.148125 264.38 \n",
       "L 415.148125 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_63\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 419.576696 391.22 \n",
       "L 437.290982 391.22 \n",
       "L 437.290982 270.42 \n",
       "L 419.576696 270.42 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_64\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 437.290982 391.22 \n",
       "L 455.005268 391.22 \n",
       "L 455.005268 383.132971 \n",
       "L 437.290982 383.132971 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_65\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 455.005268 391.22 \n",
       "L 472.719554 391.22 \n",
       "L 472.719554 387.681925 \n",
       "L 455.005268 387.681925 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_66\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 472.719554 391.22 \n",
       "L 490.433839 391.22 \n",
       "L 490.433839 390.209121 \n",
       "L 472.719554 390.209121 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_67\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 490.433839 391.22 \n",
       "L 508.148125 391.22 \n",
       "L 508.148125 379.594895 \n",
       "L 490.433839 379.594895 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_68\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 508.148125 391.22 \n",
       "L 525.862411 391.22 \n",
       "L 525.862411 383.63841 \n",
       "L 508.148125 383.63841 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_69\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 525.862411 391.22 \n",
       "L 543.576696 391.22 \n",
       "L 543.576696 388.187364 \n",
       "L 525.862411 388.187364 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_70\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 543.576696 391.22 \n",
       "L 561.290982 391.22 \n",
       "L 561.290982 390.714561 \n",
       "L 543.576696 390.714561 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_71\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 561.290982 391.22 \n",
       "L 579.005268 391.22 \n",
       "L 579.005268 391.22 \n",
       "L 561.290982 391.22 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_72\">\n",
       "    <path clip-path=\"url(#p66b7a092f6)\" d=\"M 579.005268 391.22 \n",
       "L 596.719554 391.22 \n",
       "L 596.719554 390.714561 \n",
       "L 579.005268 390.714561 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_6\">\n",
       "    <g id=\"xtick_11\">\n",
       "     <g id=\"line2d_26\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"418.710472\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_31\">\n",
       "      <!-- 0 -->\n",
       "      <g transform=\"translate(420.917972 403.31)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_12\">\n",
       "     <g id=\"line2d_27\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"462.021684\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_32\">\n",
       "      <!-- 100 -->\n",
       "      <g transform=\"translate(464.229184 413.49)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-31\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_13\">\n",
       "     <g id=\"line2d_28\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"505.332896\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_33\">\n",
       "      <!-- 200 -->\n",
       "      <g transform=\"translate(507.540396 413.49)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-32\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_14\">\n",
       "     <g id=\"line2d_29\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"548.644108\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_34\">\n",
       "      <!-- 300 -->\n",
       "      <g transform=\"translate(550.851608 413.49)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-33\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_15\">\n",
       "     <g id=\"line2d_30\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"591.95532\" xlink:href=\"#m49b4dede80\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_35\">\n",
       "      <!-- 400 -->\n",
       "      <g transform=\"translate(594.16282 413.49)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use xlink:href=\"#DejaVuSans-34\"/>\n",
       "       <use x=\"63.623047\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "       <use x=\"127.246094\" xlink:href=\"#DejaVuSans-30\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_36\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <g transform=\"translate(472.141094 425.088437)scale(0.1 -0.1)\">\n",
       "      <use xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"63.476562\" xlink:href=\"#DejaVuSans-75\"/>\n",
       "      <use x=\"126.855469\" xlink:href=\"#DejaVuSans-62\"/>\n",
       "      <use x=\"190.332031\" xlink:href=\"#DejaVuSans-67\"/>\n",
       "      <use x=\"253.808594\" xlink:href=\"#DejaVuSans-72\"/>\n",
       "      <use x=\"294.921875\" xlink:href=\"#DejaVuSans-61\"/>\n",
       "      <use x=\"356.201172\" xlink:href=\"#DejaVuSans-70\"/>\n",
       "      <use x=\"419.677734\" xlink:href=\"#DejaVuSans-68\"/>\n",
       "      <use x=\"483.056641\" xlink:href=\"#DejaVuSans-20\"/>\n",
       "      <use x=\"514.84375\" xlink:href=\"#DejaVuSans-53\"/>\n",
       "      <use x=\"578.320312\" xlink:href=\"#DejaVuSans-69\"/>\n",
       "      <use x=\"606.103516\" xlink:href=\"#DejaVuSans-7a\"/>\n",
       "      <use x=\"658.59375\" xlink:href=\"#DejaVuSans-65\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_73\">\n",
       "    <path d=\"M 415.148125 391.22 \n",
       "L 415.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_74\">\n",
       "    <path d=\"M 601.148125 391.22 \n",
       "L 601.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_75\">\n",
       "    <path d=\"M 415.148125 391.22 \n",
       "L 601.148125 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_76\">\n",
       "    <path d=\"M 415.148125 264.38 \n",
       "L 601.148125 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pb4bed2fb86\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"43.148125\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p9db39a5f1a\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"229.148125\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pcd438bef5e\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"415.148125\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pb51cd27403\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"43.148125\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p2878240e32\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"229.148125\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pc1f1d1e2b7\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"415.148125\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p89e041b7a4\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"43.148125\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p9979a80287\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"229.148125\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p66b7a092f6\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"415.148125\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x10ade33c8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "scatter_matrix(scores_df[scores_df[\"Average Score\"].notnull()], figsize=(10, 7))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "scores_df.to_csv(os.path.expanduser(\"~/Desktop/heat_diffusion_demo.csv\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Average Score</th>\n",
       "      <th>Number First Neighbors</th>\n",
       "      <th>Subgraph Size</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glial cell differentiation)</th>\n",
       "      <td>-35.037175</td>\n",
       "      <td>8</td>\n",
       "      <td>68</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, production of molecular mediator involved in inflammatory response)</th>\n",
       "      <td>-15.415658</td>\n",
       "      <td>11</td>\n",
       "      <td>85</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron apoptotic process)</th>\n",
       "      <td>-10.366053</td>\n",
       "      <td>40</td>\n",
       "      <td>411</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, apoptotic process)</th>\n",
       "      <td>-8.337959</td>\n",
       "      <td>79</td>\n",
       "      <td>261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell migration)</th>\n",
       "      <td>-6.422840</td>\n",
       "      <td>5</td>\n",
       "      <td>145</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, chronic inflammatory response)</th>\n",
       "      <td>-5.793031</td>\n",
       "      <td>5</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell-matrix adhesion)</th>\n",
       "      <td>-5.314752</td>\n",
       "      <td>3</td>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of microtubule depolymerization)</th>\n",
       "      <td>-4.608462</td>\n",
       "      <td>1</td>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, beta-amyloid formation)</th>\n",
       "      <td>-4.393178</td>\n",
       "      <td>14</td>\n",
       "      <td>211</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, phagocytosis)</th>\n",
       "      <td>-4.013165</td>\n",
       "      <td>7</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, CBP, neuroinflammation)</th>\n",
       "      <td>-3.945962</td>\n",
       "      <td>12</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of synaptic activity)</th>\n",
       "      <td>-3.863068</td>\n",
       "      <td>4</td>\n",
       "      <td>217</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, response to endoplasmic reticulum stress)</th>\n",
       "      <td>-3.498835</td>\n",
       "      <td>5</td>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, immune response)</th>\n",
       "      <td>-3.328115</td>\n",
       "      <td>3</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, gliogenesis)</th>\n",
       "      <td>-3.318600</td>\n",
       "      <td>1</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of neuron projection regeneration)</th>\n",
       "      <td>-3.206303</td>\n",
       "      <td>4</td>\n",
       "      <td>236</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, pyroptosis)</th>\n",
       "      <td>-2.900686</td>\n",
       "      <td>4</td>\n",
       "      <td>36</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microglial cell activation involved in immune response)</th>\n",
       "      <td>-2.853572</td>\n",
       "      <td>7</td>\n",
       "      <td>28</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cellular senescence)</th>\n",
       "      <td>-2.625572</td>\n",
       "      <td>2</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, stem cell proliferation)</th>\n",
       "      <td>-2.588266</td>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuronal signal transduction)</th>\n",
       "      <td>-2.588266</td>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, central nervous system development)</th>\n",
       "      <td>-2.588266</td>\n",
       "      <td>3</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microtubule depolymerization)</th>\n",
       "      <td>-2.497051</td>\n",
       "      <td>1</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of glial cell apoptotic process)</th>\n",
       "      <td>-2.365327</td>\n",
       "      <td>3</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of synaptic transmission, glutamatergic)</th>\n",
       "      <td>-2.235876</td>\n",
       "      <td>5</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, DNA biosynthetic process)</th>\n",
       "      <td>-1.858156</td>\n",
       "      <td>3</td>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, PMIBP, neuroprotection)</th>\n",
       "      <td>-1.831319</td>\n",
       "      <td>13</td>\n",
       "      <td>35</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, response to oxidative stress)</th>\n",
       "      <td>-1.808942</td>\n",
       "      <td>13</td>\n",
       "      <td>239</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell adhesion)</th>\n",
       "      <td>-1.771584</td>\n",
       "      <td>4</td>\n",
       "      <td>53</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell-cell signaling)</th>\n",
       "      <td>-1.771584</td>\n",
       "      <td>1</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell proliferation)</th>\n",
       "      <td>1.493455</td>\n",
       "      <td>5</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cytokine secretion)</th>\n",
       "      <td>1.564848</td>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cytokine production)</th>\n",
       "      <td>1.564848</td>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, chemokine biosynthetic process)</th>\n",
       "      <td>1.701339</td>\n",
       "      <td>3</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, vasodilation)</th>\n",
       "      <td>1.737646</td>\n",
       "      <td>4</td>\n",
       "      <td>205</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, calcium ion import)</th>\n",
       "      <td>1.746922</td>\n",
       "      <td>16</td>\n",
       "      <td>43</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of cellular response to oxidative stress)</th>\n",
       "      <td>1.771584</td>\n",
       "      <td>1</td>\n",
       "      <td>50</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, maintenance of permeability of blood-brain barrier)</th>\n",
       "      <td>1.902776</td>\n",
       "      <td>1</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, generation of neurons)</th>\n",
       "      <td>1.902776</td>\n",
       "      <td>1</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glucose metabolic process)</th>\n",
       "      <td>1.928946</td>\n",
       "      <td>7</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of voltage-gated potassium channel activity)</th>\n",
       "      <td>2.046360</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of synaptic plasticity)</th>\n",
       "      <td>2.133218</td>\n",
       "      <td>6</td>\n",
       "      <td>81</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neurogenesis)</th>\n",
       "      <td>2.154059</td>\n",
       "      <td>4</td>\n",
       "      <td>200</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, long-term synaptic potentiation)</th>\n",
       "      <td>2.218007</td>\n",
       "      <td>7</td>\n",
       "      <td>33</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, vasoconstriction)</th>\n",
       "      <td>2.334503</td>\n",
       "      <td>4</td>\n",
       "      <td>196</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of dendritic cell dendrite assembly)</th>\n",
       "      <td>2.373630</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of calcium-mediated signaling)</th>\n",
       "      <td>2.453273</td>\n",
       "      <td>6</td>\n",
       "      <td>207</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Lipid Peroxidation)</th>\n",
       "      <td>2.480842</td>\n",
       "      <td>4</td>\n",
       "      <td>193</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, amyloid fibril formation)</th>\n",
       "      <td>2.765434</td>\n",
       "      <td>5</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Oxidative Stress)</th>\n",
       "      <td>2.910333</td>\n",
       "      <td>15</td>\n",
       "      <td>250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, learning)</th>\n",
       "      <td>2.987889</td>\n",
       "      <td>11</td>\n",
       "      <td>95</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, inflammatory response)</th>\n",
       "      <td>3.199991</td>\n",
       "      <td>37</td>\n",
       "      <td>277</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Insulin Resistance)</th>\n",
       "      <td>3.314902</td>\n",
       "      <td>6</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, brain development)</th>\n",
       "      <td>3.441201</td>\n",
       "      <td>3</td>\n",
       "      <td>199</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron development)</th>\n",
       "      <td>4.156043</td>\n",
       "      <td>7</td>\n",
       "      <td>208</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell death)</th>\n",
       "      <td>4.267876</td>\n",
       "      <td>14</td>\n",
       "      <td>237</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of catalytic activity)</th>\n",
       "      <td>5.770745</td>\n",
       "      <td>3</td>\n",
       "      <td>80</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, energy reserve metabolic process)</th>\n",
       "      <td>6.602340</td>\n",
       "      <td>3</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of neuron apoptotic process)</th>\n",
       "      <td>6.790692</td>\n",
       "      <td>8</td>\n",
       "      <td>227</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, memory)</th>\n",
       "      <td>11.049425</td>\n",
       "      <td>31</td>\n",
       "      <td>176</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>310 rows  3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                    Average Score  \\\n",
       "(BiologicalProcess, GOBP, glial cell differenti...     -35.037175   \n",
       "(BiologicalProcess, GOBP, production of molecul...     -15.415658   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...     -10.366053   \n",
       "(BiologicalProcess, GOBP, apoptotic process)            -8.337959   \n",
       "(BiologicalProcess, GOBP, cell migration)               -6.422840   \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...      -5.793031   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)         -5.314752   \n",
       "(BiologicalProcess, GOBP, negative regulation o...      -4.608462   \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)       -4.393178   \n",
       "(BiologicalProcess, GOBP, phagocytosis)                 -4.013165   \n",
       "(BiologicalProcess, CBP, neuroinflammation)             -3.945962   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...      -3.863068   \n",
       "(BiologicalProcess, GOBP, response to endoplasm...      -3.498835   \n",
       "(BiologicalProcess, GOBP, immune response)              -3.328115   \n",
       "(BiologicalProcess, GOBP, gliogenesis)                  -3.318600   \n",
       "(BiologicalProcess, GOBP, negative regulation o...      -3.206303   \n",
       "(BiologicalProcess, GOBP, pyroptosis)                   -2.900686   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...      -2.853572   \n",
       "(BiologicalProcess, GOBP, cellular senescence)          -2.625572   \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)      -2.588266   \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...      -2.588266   \n",
       "(BiologicalProcess, GOBP, central nervous syste...      -2.588266   \n",
       "(BiologicalProcess, GOBP, microtubule depolymer...      -2.497051   \n",
       "(BiologicalProcess, GOBP, positive regulation o...      -2.365327   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...      -2.235876   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...      -1.858156   \n",
       "(BiologicalProcess, PMIBP, neuroprotection)             -1.831319   \n",
       "(BiologicalProcess, GOBP, response to oxidative...      -1.808942   \n",
       "(BiologicalProcess, GOBP, cell adhesion)                -1.771584   \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)          -1.771584   \n",
       "...                                                           ...   \n",
       "(BiologicalProcess, GOBP, cell proliferation)            1.493455   \n",
       "(BiologicalProcess, GOBP, cytokine secretion)            1.564848   \n",
       "(BiologicalProcess, GOBP, cytokine production)           1.564848   \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...       1.701339   \n",
       "(BiologicalProcess, GOBP, vasodilation)                  1.737646   \n",
       "(BiologicalProcess, GOBP, calcium ion import)            1.746922   \n",
       "(BiologicalProcess, GOBP, regulation of cellula...       1.771584   \n",
       "(BiologicalProcess, GOBP, maintenance of permea...       1.902776   \n",
       "(BiologicalProcess, GOBP, generation of neurons)         1.902776   \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...       1.928946   \n",
       "(BiologicalProcess, GOBP, positive regulation o...       2.046360   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...       2.133218   \n",
       "(BiologicalProcess, GOBP, neurogenesis)                  2.154059   \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...       2.218007   \n",
       "(BiologicalProcess, GOBP, vasoconstriction)              2.334503   \n",
       "(BiologicalProcess, GOBP, regulation of dendrit...       2.373630   \n",
       "(BiologicalProcess, GOBP, negative regulation o...       2.453273   \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)          2.480842   \n",
       "(BiologicalProcess, GOBP, amyloid fibril format...       2.765434   \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)            2.910333   \n",
       "(BiologicalProcess, GOBP, learning)                      2.987889   \n",
       "(BiologicalProcess, GOBP, inflammatory response)         3.199991   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)          3.314902   \n",
       "(BiologicalProcess, GOBP, brain development)             3.441201   \n",
       "(BiologicalProcess, GOBP, neuron development)            4.156043   \n",
       "(BiologicalProcess, GOBP, cell death)                    4.267876   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...       5.770745   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...       6.602340   \n",
       "(BiologicalProcess, GOBP, negative regulation o...       6.790692   \n",
       "(BiologicalProcess, GOBP, memory)                       11.049425   \n",
       "\n",
       "                                                    Number First Neighbors  \\\n",
       "(BiologicalProcess, GOBP, glial cell differenti...                       8   \n",
       "(BiologicalProcess, GOBP, production of molecul...                      11   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...                      40   \n",
       "(BiologicalProcess, GOBP, apoptotic process)                            79   \n",
       "(BiologicalProcess, GOBP, cell migration)                                5   \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...                       5   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                          3   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)                       14   \n",
       "(BiologicalProcess, GOBP, phagocytosis)                                  7   \n",
       "(BiologicalProcess, CBP, neuroinflammation)                             12   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...                       4   \n",
       "(BiologicalProcess, GOBP, response to endoplasm...                       5   \n",
       "(BiologicalProcess, GOBP, immune response)                               3   \n",
       "(BiologicalProcess, GOBP, gliogenesis)                                   1   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       4   \n",
       "(BiologicalProcess, GOBP, pyroptosis)                                    4   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...                       7   \n",
       "(BiologicalProcess, GOBP, cellular senescence)                           2   \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)                       1   \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...                       1   \n",
       "(BiologicalProcess, GOBP, central nervous syste...                       3   \n",
       "(BiologicalProcess, GOBP, microtubule depolymer...                       1   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       3   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...                       5   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...                       3   \n",
       "(BiologicalProcess, PMIBP, neuroprotection)                             13   \n",
       "(BiologicalProcess, GOBP, response to oxidative...                      13   \n",
       "(BiologicalProcess, GOBP, cell adhesion)                                 4   \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)                           1   \n",
       "...                                                                    ...   \n",
       "(BiologicalProcess, GOBP, cell proliferation)                            5   \n",
       "(BiologicalProcess, GOBP, cytokine secretion)                            2   \n",
       "(BiologicalProcess, GOBP, cytokine production)                           2   \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...                       3   \n",
       "(BiologicalProcess, GOBP, vasodilation)                                  4   \n",
       "(BiologicalProcess, GOBP, calcium ion import)                           16   \n",
       "(BiologicalProcess, GOBP, regulation of cellula...                       1   \n",
       "(BiologicalProcess, GOBP, maintenance of permea...                       1   \n",
       "(BiologicalProcess, GOBP, generation of neurons)                         1   \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...                       7   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, regulation of synapti...                       6   \n",
       "(BiologicalProcess, GOBP, neurogenesis)                                  4   \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...                       7   \n",
       "(BiologicalProcess, GOBP, vasoconstriction)                              4   \n",
       "(BiologicalProcess, GOBP, regulation of dendrit...                       1   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       6   \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)                          4   \n",
       "(BiologicalProcess, GOBP, amyloid fibril format...                       5   \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)                           15   \n",
       "(BiologicalProcess, GOBP, learning)                                     11   \n",
       "(BiologicalProcess, GOBP, inflammatory response)                        37   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                          6   \n",
       "(BiologicalProcess, GOBP, brain development)                             3   \n",
       "(BiologicalProcess, GOBP, neuron development)                            7   \n",
       "(BiologicalProcess, GOBP, cell death)                                   14   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...                       3   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...                       3   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       8   \n",
       "(BiologicalProcess, GOBP, memory)                                       31   \n",
       "\n",
       "                                                    Subgraph Size  \n",
       "(BiologicalProcess, GOBP, glial cell differenti...             68  \n",
       "(BiologicalProcess, GOBP, production of molecul...             85  \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...            411  \n",
       "(BiologicalProcess, GOBP, apoptotic process)                  261  \n",
       "(BiologicalProcess, GOBP, cell migration)                     145  \n",
       "(BiologicalProcess, GOBP, chronic inflammatory ...            211  \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                52  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             54  \n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)             211  \n",
       "(BiologicalProcess, GOBP, phagocytosis)                        33  \n",
       "(BiologicalProcess, CBP, neuroinflammation)                    58  \n",
       "(BiologicalProcess, GOBP, regulation of synapti...            217  \n",
       "(BiologicalProcess, GOBP, response to endoplasm...             26  \n",
       "(BiologicalProcess, GOBP, immune response)                     25  \n",
       "(BiologicalProcess, GOBP, gliogenesis)                         18  \n",
       "(BiologicalProcess, GOBP, negative regulation o...            236  \n",
       "(BiologicalProcess, GOBP, pyroptosis)                          36  \n",
       "(BiologicalProcess, GOBP, microglial cell activ...             28  \n",
       "(BiologicalProcess, GOBP, cellular senescence)                  7  \n",
       "(BiologicalProcess, GOBP, stem cell proliferation)             12  \n",
       "(BiologicalProcess, GOBP, neuronal signal trans...             12  \n",
       "(BiologicalProcess, GOBP, central nervous syste...             18  \n",
       "(BiologicalProcess, GOBP, microtubule depolymer...              8  \n",
       "(BiologicalProcess, GOBP, positive regulation o...             47  \n",
       "(BiologicalProcess, GOBP, regulation of synapti...             16  \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...             54  \n",
       "(BiologicalProcess, PMIBP, neuroprotection)                    35  \n",
       "(BiologicalProcess, GOBP, response to oxidative...            239  \n",
       "(BiologicalProcess, GOBP, cell adhesion)                       53  \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)                 50  \n",
       "...                                                           ...  \n",
       "(BiologicalProcess, GOBP, cell proliferation)                  12  \n",
       "(BiologicalProcess, GOBP, cytokine secretion)                  12  \n",
       "(BiologicalProcess, GOBP, cytokine production)                 12  \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...             16  \n",
       "(BiologicalProcess, GOBP, vasodilation)                       205  \n",
       "(BiologicalProcess, GOBP, calcium ion import)                  43  \n",
       "(BiologicalProcess, GOBP, regulation of cellula...             50  \n",
       "(BiologicalProcess, GOBP, maintenance of permea...             32  \n",
       "(BiologicalProcess, GOBP, generation of neurons)               32  \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...             18  \n",
       "(BiologicalProcess, GOBP, positive regulation o...              3  \n",
       "(BiologicalProcess, GOBP, regulation of synapti...             81  \n",
       "(BiologicalProcess, GOBP, neurogenesis)                       200  \n",
       "(BiologicalProcess, GOBP, long-term synaptic po...             33  \n",
       "(BiologicalProcess, GOBP, vasoconstriction)                   196  \n",
       "(BiologicalProcess, GOBP, regulation of dendrit...              9  \n",
       "(BiologicalProcess, GOBP, negative regulation o...            207  \n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)               193  \n",
       "(BiologicalProcess, GOBP, amyloid fibril format...             13  \n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)                 250  \n",
       "(BiologicalProcess, GOBP, learning)                            95  \n",
       "(BiologicalProcess, GOBP, inflammatory response)              277  \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                22  \n",
       "(BiologicalProcess, GOBP, brain development)                  199  \n",
       "(BiologicalProcess, GOBP, neuron development)                 208  \n",
       "(BiologicalProcess, GOBP, cell death)                         237  \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...             80  \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...             10  \n",
       "(BiologicalProcess, GOBP, negative regulation o...            227  \n",
       "(BiologicalProcess, GOBP, memory)                             176  \n",
       "\n",
       "[310 rows x 3 columns]"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scores_df[scores_df[\"Average Score\"].notnull()].sort_values(\"Average Score\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Conclusions\n",
    "\n",
    "## Problems\n",
    "\n",
    "This algorithm overcomes deterministic problems with cyclic graphs by randomly removing edges. It can be run multiple times to assess the stability of the score with `pybel_tools.analysis.heat.average_run`. The effect of the default score can also be checked with a grid search.\n",
    "\n",
    "## Looking Forward\n",
    "\n",
    "While this new algorithm is generally applicable and overcomes the original algorithm, it must throw away information to do so. Further algorithmic development, such as heat diffusion approaches could allow for a more thorough investigation of the propogation of effects of the differential gene expression.\n",
    "\n",
    "This algorithm can be run over the mechanisms generated for each biological process. Further investigation into the upstream mechanism generation can yield bias-free candidate mechanisms. They can also be matched to canonical mechanisms in NeuroMMSigDB to identify concordance between the biological dogma and all possible mechanisms, then to identify previously unknown cross-talk between mechanisms."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
