{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Abstract\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt)\n",
    "\n",
    "**Estimated Run Time:** 4 minutes\n",
    "\n",
    "This notebooks demonstrates the usage of a variant on the [Network Perturbation Amplitude](http://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-54) algorithm to analyze an BEL knowledge assembly in the context of a differential gene expression experiment. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Imports"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import logging\n",
    "import os\n",
    "import time\n",
    "\n",
    "import pandas as pd\n",
    "import matplotlib.pyplot as plt\n",
    "from pandas.tools.plotting import scatter_matrix\n",
    "from matplotlib_venn import venn2\n",
    "\n",
    "import pybel\n",
    "import pybel_tools as pbt\n",
    "import pybel_tools.analysis.npa as npa\n",
    "from pybel.constants import *\n",
    "from pybel.canonicalize import calculate_canonical_name\n",
    "from pybel_tools.visualization import to_jupyter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "%config InlineBackend.figure_format = 'svg'\n",
    "%matplotlib inline"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Notebook Provenance\n",
    "\n",
    "The time of execution, random number generator seed, and the versions of the software packages used are displayed explicitly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Tue Apr 25 11:17:04 2017'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "time.asctime()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# seed the random number generator\n",
    "import random\n",
    "random.seed(127)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.5.4-dev'"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.1.8-dev'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.__version__"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Local Path Definitions\n",
    "\n",
    "To make this notebook interoperable across many machines, locations to the repositories that contain the data used in this notebook are referenced from the environment, set in `~/.bashrc` to point to the place where the repositories have been cloned. Assuming the repositories have been `git clone`'d into the `~/dev` folder, the entries in `~/.bashrc` should look like:\n",
    "\n",
    "```bash\n",
    "...\n",
    "export BMS_BASE=~/dev/bms\n",
    "...\n",
    "```\n",
    "\n",
    "#### BMS \n",
    "\n",
    "The biological model store (BMS) is the internal Fraunhofer SCAI repository for keeping BEL models under version control. It can be downloaded from https://tor-2.scai.fraunhofer.de/gf/project/bms/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "bms_base = os.environ['BMS_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### OwnCloud\n",
    "\n",
    "The differential gene expression data used in this notebook is currently not published, and is obfuscated with reference through our team's internal data storage system with [OwnCloud](https://owncloud.org/)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "owncloud_base = os.environ['OWNCLOUD_BASE']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Data\n",
    "\n",
    "## Alzheimer's Disease Knowledge Assembly\n",
    "\n",
    "The Alzheimer's Disease Knowledge Assembly has been precompiled with the following command line script, and will be loaded from this format for improved performance. In general, derived data, such as the gpickle representation of a BEL script, are not saved under version control to ensure that the most up-to-date data is always used.\n",
    "\n",
    "```sh\n",
    "pybel convert --path \"$BMS_BASE/aetionomy/alzheimers.bel\" --pickle \"$BMS_BASE/aetionomy/alzheimers.gpickle\"\n",
    "```\n",
    "\n",
    "The BEL script can also be compiled from inside this notebook with the following python code:\n",
    "\n",
    "```python\n",
    ">>> import os\n",
    ">>> import pybel\n",
    ">>> # Input from BEL script\n",
    ">>> bel_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.bel')\n",
    ">>> graph = pybel.from_path(bel_path)\n",
    ">>> # Output to gpickle for fast loading later\n",
    ">>> pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers.gpickle')\n",
    ">>> pybel.to_pickle(graph, pickle_path)\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "pickle_path = os.path.join(bms_base, 'aetionomy', 'alzheimers', 'alzheimers.gpickle')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "graph = pybel.from_pickle(pickle_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'3.0.2'"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "graph.version"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "All orthologies are discared before analysis."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.filters.remove_nodes_by_namespace(graph, 'MGI')\n",
    "pbt.filters.remove_nodes_by_namespace(graph, 'RGD')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "To merge the differential gene expression annotations, the entire graph is collapsed to genes using [pbt.mutation.collapse_by_central_dogma_to_genes](http://pybel-tools.readthedocs.io/en/latest/mutation.html#pybel_tools.mutation.collapse_by_central_dogma_to_genes)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.mutation.opening_on_central_dogma(graph)\n",
    "\n",
    "pbt.mutation.collapse_variants_to_genes(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 4071\n",
      "Number of edges: 19726\n",
      "Network density: 0.0011905386997501956\n",
      "Number weakly connected components: 75\n",
      "Average in-degree: 4.8454925079832964\n",
      "Average out-degree: 4.8454925079832964\n"
     ]
    }
   ],
   "source": [
    "pbt.summary.print_summary(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"280pt\" version=\"1.1\" viewBox=\"0 0 712 280\" width=\"712pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 280.8 \n",
       "L 712.838891 280.8 \n",
       "L 712.838891 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 331.372261 257.744323 \n",
       "L 331.372261 21.50218 \n",
       "L 89.335068 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 251.83827 \n",
       "L 100.336758 251.83827 \n",
       "L 100.336758 240.026163 \n",
       "L -34236.324752 240.026163 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 228.214055 \n",
       "L 155.903281 228.214055 \n",
       "L 155.903281 216.401948 \n",
       "L -34236.324752 216.401948 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 204.589841 \n",
       "L 170.721259 204.589841 \n",
       "L 170.721259 192.777734 \n",
       "L -34236.324752 192.777734 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 180.965627 \n",
       "L 202.414774 180.965627 \n",
       "L 202.414774 169.153519 \n",
       "L -34236.324752 169.153519 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 157.341412 \n",
       "L 266.465664 157.341412 \n",
       "L 266.465664 145.529305 \n",
       "L -34236.324752 145.529305 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 133.717198 \n",
       "L 269.923214 133.717198 \n",
       "L 269.923214 121.905091 \n",
       "L -34236.324752 121.905091 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 110.092984 \n",
       "L 276.693543 110.092984 \n",
       "L 276.693543 98.280876 \n",
       "L -34236.324752 98.280876 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 86.468769 \n",
       "L 286.748963 86.468769 \n",
       "L 286.748963 74.656662 \n",
       "L -34236.324752 74.656662 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 62.844555 \n",
       "L 295.927279 62.844555 \n",
       "L 295.927279 51.032448 \n",
       "L -34236.324752 51.032448 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p7289b9365d)\" d=\"M -34236.324752 39.220341 \n",
       "L 320.370571 39.220341 \n",
       "L 320.370571 27.408233 \n",
       "L -34236.324752 27.408233 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m348a6af32d\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"197.695183\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(190.728229 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"311.715117\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(304.748163 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 2 \n",
       "\" id=\"mf7f977acbd\" style=\"stroke:#000000;stroke-width:0.6;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"117.99867\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"138.076583\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"152.32209\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"163.371763\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"172.400003\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"180.033272\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"186.64551\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"192.477917\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_11\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"232.018603\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_12\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"252.096517\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_13\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"266.342023\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_14\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"277.391697\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_15\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"286.419937\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_16\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"294.053205\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_17\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"300.665443\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_18\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"306.497851\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mc40aa56ede\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"245.932216\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- Reaction -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.203125 33.53125 \n",
       "C 48.703125 35.4375 56.703125 41.8125 56.703125 49.78125 \n",
       "C 56.703125 59.140625 46.09375 67.59375 32.09375 67.59375 \n",
       "L 5.203125 67.59375 \n",
       "L 5.203125 65 \n",
       "C 12.09375 65 13.203125 65 13.203125 60.515625 \n",
       "L 13.203125 6.484375 \n",
       "C 13.203125 2 12.09375 2 5.203125 2 \n",
       "L 5.203125 -0.40625 \n",
       "C 8.09375 -0.40625 13.296875 -0.40625 16.40625 -0.40625 \n",
       "C 19.5 -0.40625 24.703125 -0.40625 27.59375 -0.40625 \n",
       "L 27.59375 2 \n",
       "C 20.703125 2 19.59375 2 19.59375 6.515625 \n",
       "L 19.59375 33 \n",
       "L 31.296875 33 \n",
       "C 37 33 40.203125 30.6875 41.59375 29.390625 \n",
       "C 45.59375 25.5 45.59375 22.796875 45.59375 16 \n",
       "C 45.59375 9.296875 45.59375 5.890625 48.59375 2.5 \n",
       "C 52.40625 -1.59375 57.5 -2 59.703125 -2 \n",
       "C 67.09375 -2 68.203125 6.1875 68.203125 7.859375 \n",
       "C 68.203125 8.4375 68.203125 9.140625 67.296875 9.140625 \n",
       "C 66.5 9.140625 66.40625 8.546875 66.40625 8.15625 \n",
       "C 65.90625 1.75 63 -0.40625 59.90625 -0.40625 \n",
       "C 54.40625 -0.40625 53.796875 5.859375 53.203125 11.9375 \n",
       "C 52.90625 14.03125 52.703125 15.515625 52.5 17.703125 \n",
       "C 51.90625 22.6875 51.09375 29.953125 39.203125 33.53125 \n",
       "M 31 34.59375 \n",
       "L 19.59375 34.59375 \n",
       "L 19.59375 61.140625 \n",
       "C 19.59375 64.515625 19.796875 65 23.796875 65 \n",
       "L 31.09375 65 \n",
       "C 39.40625 65 49.09375 62.140625 49.09375 49.84375 \n",
       "C 49.09375 37.0625 38.5 34.59375 31 34.59375 \n",
       "\" id=\"CMR17-82\"/>\n",
       "       <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "       <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "       <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "       <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "       <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "       <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(44.00651 249.391465)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(67.791102 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(108.276762 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(153.96724 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(194.4529 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(229.733668 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(254.604726 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(300.295203 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"222.308002\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- miRNA -->\n",
       "      <defs>\n",
       "       <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "       <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "       <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(48.98783 225.767251)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(76.91959 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(101.790648 0)scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(169.58175 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "       <use transform=\"translate(238.65418 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_21\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"198.683787\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- Composite -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "       <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "       <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(36.064035 202.143037)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(242.617756 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(288.308233 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(324.109571 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(348.980629 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(384.261397 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_22\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"175.059573\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- Pathology -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "       <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "       <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "       <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "       <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(38.471647 178.518822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(59.983847 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(105.674324 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(140.955092 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(191.850461 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(237.540938 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(262.411997 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(308.102474 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(353.792951 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_23\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"151.435359\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- Abundance -->\n",
       "      <defs>\n",
       "       <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "       <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "       <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(33.351976 154.894608)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "       <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(170.863169 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(221.758539 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(272.653909 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(318.344386 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(369.239756 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(409.725416 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_24\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"127.811144\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- BiologicalProcess -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 39.59375 0 \n",
       "C 52.40625 0 60.703125 9.109375 60.703125 18.1875 \n",
       "C 60.703125 27.203125 52.5 34.890625 41.796875 34.890625 \n",
       "C 51.40625 36.8125 58.09375 43.296875 58.09375 51 \n",
       "C 58.09375 59.40625 49.90625 68 37 68 \n",
       "z\n",
       "M 19.5 36 \n",
       "L 19.5 61.515625 \n",
       "C 19.5 64.921875 19.703125 65.40625 23.703125 65.40625 \n",
       "L 36.5 65.40625 \n",
       "C 46.40625 65.40625 50.796875 57.40625 50.796875 51.203125 \n",
       "C 50.796875 43.609375 44.703125 36 33.90625 36 \n",
       "z\n",
       "M 23.703125 2.59375 \n",
       "C 19.703125 2.59375 19.5 3.078125 19.5 6.453125 \n",
       "L 19.5 34.40625 \n",
       "L 38 34.40625 \n",
       "C 48.296875 34.40625 53.296875 25.390625 53.296875 18.15625 \n",
       "C 53.296875 10.921875 47.59375 2.59375 36.703125 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-66\"/>\n",
       "       <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(7.2 131.270394)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(65.188754 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(90.059813 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(135.75029 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(160.621349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(206.311826 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(252.002303 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(276.873361 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(317.359022 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(363.049499 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(387.920557 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(450.506858 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(485.787626 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(534.080541 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(574.566202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(615.051862 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(650.8532 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_25\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"104.18693\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- RNA -->\n",
       "      <g transform=\"translate(60.057464 107.646179)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(67.791102 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "       <use transform=\"translate(136.863532 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_26\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"80.562716\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- Gene -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(60.126662 84.021965)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "       <use transform=\"translate(72.355346 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(112.841006 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(163.736376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_27\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"56.938501\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- Complex -->\n",
       "      <defs>\n",
       "       <path d=\"M 26 23.078125 \n",
       "C 29 26.96875 32.40625 31.34375 35.203125 34.625 \n",
       "C 38.5 38.40625 42.296875 40 46.703125 40 \n",
       "L 46.703125 42.59375 \n",
       "C 45 42.59375 41.5 42.59375 39.703125 42.59375 \n",
       "C 37 42.59375 33.90625 42.59375 31.59375 42.6875 \n",
       "L 31.59375 40.09375 \n",
       "C 33 39.890625 33.796875 38.890625 33.796875 37.5 \n",
       "C 33.796875 35.703125 32.796875 34.609375 32.296875 33.921875 \n",
       "L 24.90625 24.578125 \n",
       "L 16.09375 36.03125 \n",
       "C 15.203125 37.125 15.203125 37.3125 15.203125 37.8125 \n",
       "C 15.203125 39.015625 16.203125 40 17.90625 40 \n",
       "L 17.90625 42.6875 \n",
       "C 15.5 42.59375 11.203125 42.59375 8.703125 42.59375 \n",
       "C 6 42.59375 2.796875 42.59375 0.59375 42.6875 \n",
       "L 0.59375 40 \n",
       "C 6 40 7.203125 39.59375 9.59375 36.609375 \n",
       "L 21.09375 21.796875 \n",
       "C 21.296875 21.59375 21.796875 21 21.796875 20.703125 \n",
       "C 21.796875 20.296875 13.09375 9.671875 12 8.28125 \n",
       "C 7.5 2.890625 2.796875 2.5 0.09375 2.5 \n",
       "L 0.09375 0 \n",
       "C 1.796875 0 5.296875 0 7.09375 0 \n",
       "C 9.09375 0 13.5 0 15.296875 -0.09375 \n",
       "L 15.296875 2.5 \n",
       "C 14.703125 2.59375 13.09375 2.796875 13.09375 5.078125 \n",
       "C 13.09375 6.875 13.90625 7.96875 14.796875 9.15625 \n",
       "L 23.09375 19.3125 \n",
       "L 31.796875 8 \n",
       "C 32.59375 6.90625 33.796875 5.40625 33.796875 4.703125 \n",
       "C 33.796875 3.296875 32.59375 2.5 31 2.5 \n",
       "L 31 -0.09375 \n",
       "C 33.40625 0 37.703125 0 40.203125 0 \n",
       "C 42.90625 0 46.09375 0 48.296875 -0.09375 \n",
       "L 48.296875 2.5 \n",
       "C 43.296875 2.5 41.796875 2.6875 39.40625 5.78125 \n",
       "z\n",
       "\" id=\"CMR17-120\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(43.868113 60.397751)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(66.509881 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(112.200358 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(189.119948 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(240.015317 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(264.886376 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(305.372036 0)scale(0.996264)\" xlink:href=\"#CMR17-120\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_28\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"89.335068\" xlink:href=\"#mc40aa56ede\" y=\"33.314287\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- Protein -->\n",
       "      <g transform=\"translate(50.067111 36.773536)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(97.867068 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(143.557545 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(178.838313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(219.323974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(244.195032 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 89.335068 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 331.372261 257.744323 \n",
       "L 331.372261 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 89.335068 257.744323 \n",
       "L 331.372261 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 89.335068 21.50218 \n",
       "L 331.372261 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_13\">\n",
       "    <!-- Number of nodes: 4071 -->\n",
       "    <defs>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 16.59375 38.921875 \n",
       "C 16.59375 41.40625 14.59375 43 12.5 43 \n",
       "C 10.5 43 8.40625 41.40625 8.40625 38.921875 \n",
       "C 8.40625 36.4375 10.40625 34.84375 12.5 34.84375 \n",
       "C 14.5 34.84375 16.59375 36.4375 16.59375 38.921875 \n",
       "M 16.59375 4.078125 \n",
       "C 16.59375 6.5625 14.59375 8.15625 12.5 8.15625 \n",
       "C 10.5 8.15625 8.40625 6.5625 8.40625 4.078125 \n",
       "C 8.40625 1.59375 10.40625 0 12.5 0 \n",
       "C 14.5 0 16.59375 1.59375 16.59375 4.078125 \n",
       "\" id=\"CMR17-58\"/>\n",
       "     <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "     <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(150.655239 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(507.760464 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(556.05338 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(606.94875 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(647.43441 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(683.235748 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(748.592467 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "     <use transform=\"translate(794.282944 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(839.973421 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(885.663898 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_1\">\n",
       "    <g id=\"patch_17\">\n",
       "     <path d=\"M 287.390926 252.744323 \n",
       "L 324.372261 252.744323 \n",
       "Q 326.372261 252.744323 326.372261 250.744323 \n",
       "L 326.372261 237.807145 \n",
       "Q 326.372261 235.807145 324.372261 235.807145 \n",
       "L 287.390926 235.807145 \n",
       "Q 285.390926 235.807145 285.390926 237.807145 \n",
       "L 285.390926 250.744323 \n",
       "Q 285.390926 252.744323 287.390926 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_18\">\n",
       "     <path d=\"M 289.390926 246.807145 \n",
       "L 309.390926 246.807145 \n",
       "L 309.390926 239.807145 \n",
       "L 289.390926 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_14\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(317.390926 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 457.446275 257.744323 \n",
       "L 699.483468 257.744323 \n",
       "L 699.483468 21.50218 \n",
       "L 457.446275 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 254.635874 \n",
       "L 468.447965 254.635874 \n",
       "L 468.447965 248.418976 \n",
       "L -16798.34992 248.418976 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 242.202077 \n",
       "L 468.447965 242.202077 \n",
       "L 468.447965 235.985178 \n",
       "L -16798.34992 235.985178 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_22\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 229.76828 \n",
       "L 540.696421 229.76828 \n",
       "L 540.696421 223.551381 \n",
       "L -16798.34992 223.551381 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 217.334483 \n",
       "L 545.712436 217.334483 \n",
       "L 545.712436 211.117584 \n",
       "L -16798.34992 211.117584 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 204.900686 \n",
       "L 552.617734 204.900686 \n",
       "L 552.617734 198.683787 \n",
       "L -16798.34992 198.683787 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 192.466889 \n",
       "L 575.693363 192.466889 \n",
       "L 575.693363 186.24999 \n",
       "L -16798.34992 186.24999 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 180.033092 \n",
       "L 576.368978 180.033092 \n",
       "L 576.368978 173.816193 \n",
       "L -16798.34992 173.816193 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_27\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 167.599295 \n",
       "L 576.700059 167.599295 \n",
       "L 576.700059 161.382396 \n",
       "L -16798.34992 161.382396 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 155.165498 \n",
       "L 585.71407 155.165498 \n",
       "L 585.71407 148.948599 \n",
       "L -16798.34992 148.948599 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 142.731701 \n",
       "L 588.734529 142.731701 \n",
       "L 588.734529 136.514802 \n",
       "L -16798.34992 136.514802 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 130.297904 \n",
       "L 595.925502 130.297904 \n",
       "L 595.925502 124.081005 \n",
       "L -16798.34992 124.081005 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 117.864107 \n",
       "L 605.650826 117.864107 \n",
       "L 605.650826 111.647208 \n",
       "L -16798.34992 111.647208 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_32\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 105.43031 \n",
       "L 606.860558 105.43031 \n",
       "L 606.860558 99.213411 \n",
       "L -16798.34992 99.213411 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 92.996513 \n",
       "L 638.758531 92.996513 \n",
       "L 638.758531 86.779614 \n",
       "L -16798.34992 86.779614 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 80.562716 \n",
       "L 641.708768 80.562716 \n",
       "L 641.708768 74.345817 \n",
       "L -16798.34992 74.345817 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 68.128919 \n",
       "L 652.957833 68.128919 \n",
       "L 652.957833 61.91202 \n",
       "L -16798.34992 61.91202 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 55.695122 \n",
       "L 661.485944 55.695122 \n",
       "L 661.485944 49.478223 \n",
       "L -16798.34992 49.478223 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 43.261325 \n",
       "L 686.267161 43.261325 \n",
       "L 686.267161 37.044426 \n",
       "L -16798.34992 37.044426 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path clip-path=\"url(#p93f0f215ed)\" d=\"M -16798.34992 30.827528 \n",
       "L 688.481777 30.827528 \n",
       "L 688.481777 24.610629 \n",
       "L -16798.34992 24.610629 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_19\">\n",
       "     <g id=\"line2d_29\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"468.447965\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $10^{0}$ -->\n",
       "      <g transform=\"translate(461.481011 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_20\">\n",
       "     <g id=\"line2d_30\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"526.003958\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $10^{1}$ -->\n",
       "      <g transform=\"translate(519.037004 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-49\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_21\">\n",
       "     <g id=\"line2d_31\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"583.559951\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $10^{2}$ -->\n",
       "      <g transform=\"translate(576.592997 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_22\">\n",
       "     <g id=\"line2d_32\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"641.115944\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_18\">\n",
       "      <!-- $10^{3}$ -->\n",
       "      <g transform=\"translate(634.14899 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-51\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_23\">\n",
       "     <g id=\"line2d_33\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"698.671937\" xlink:href=\"#m348a6af32d\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- $10^{4}$ -->\n",
       "      <g transform=\"translate(691.704983 271.662822)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 36.153639)scale(0.697382)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_24\">\n",
       "     <g id=\"line2d_34\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"459.532429\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_25\">\n",
       "     <g id=\"line2d_35\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"462.870213\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_26\">\n",
       "     <g id=\"line2d_36\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"465.814347\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_27\">\n",
       "     <g id=\"line2d_37\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"485.774046\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_28\">\n",
       "     <g id=\"line2d_38\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"495.909153\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_29\">\n",
       "     <g id=\"line2d_39\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"503.100126\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_30\">\n",
       "     <g id=\"line2d_40\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"508.677878\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_31\">\n",
       "     <g id=\"line2d_41\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"513.235233\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_32\">\n",
       "     <g id=\"line2d_42\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"517.088422\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_33\">\n",
       "     <g id=\"line2d_43\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"520.426206\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_34\">\n",
       "     <g id=\"line2d_44\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"523.37034\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_35\">\n",
       "     <g id=\"line2d_45\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"543.330038\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_36\">\n",
       "     <g id=\"line2d_46\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"553.465146\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_37\">\n",
       "     <g id=\"line2d_47\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"560.656119\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_38\">\n",
       "     <g id=\"line2d_48\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"566.233871\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_39\">\n",
       "     <g id=\"line2d_49\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"570.791226\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_40\">\n",
       "     <g id=\"line2d_50\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"574.644415\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_41\">\n",
       "     <g id=\"line2d_51\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"577.982199\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_42\">\n",
       "     <g id=\"line2d_52\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"580.926333\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_43\">\n",
       "     <g id=\"line2d_53\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"600.886031\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_44\">\n",
       "     <g id=\"line2d_54\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"611.021139\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_45\">\n",
       "     <g id=\"line2d_55\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"618.212112\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_46\">\n",
       "     <g id=\"line2d_56\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"623.789864\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_47\">\n",
       "     <g id=\"line2d_57\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"628.347219\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_48\">\n",
       "     <g id=\"line2d_58\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"632.200408\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_49\">\n",
       "     <g id=\"line2d_59\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"635.538192\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_50\">\n",
       "     <g id=\"line2d_60\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"638.482326\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_51\">\n",
       "     <g id=\"line2d_61\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"658.442024\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_52\">\n",
       "     <g id=\"line2d_62\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"668.577132\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_53\">\n",
       "     <g id=\"line2d_63\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"675.768105\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_54\">\n",
       "     <g id=\"line2d_64\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"681.345857\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_55\">\n",
       "     <g id=\"line2d_65\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"685.903212\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_56\">\n",
       "     <g id=\"line2d_66\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"689.756401\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_57\">\n",
       "     <g id=\"line2d_67\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"693.094185\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_58\">\n",
       "     <g id=\"line2d_68\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.6;\" x=\"696.038319\" xlink:href=\"#mf7f977acbd\" y=\"257.744323\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_11\">\n",
       "     <g id=\"line2d_69\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"251.527425\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- prognosticBiomarkerFor -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.09375 26.140625 \n",
       "C 26 26.34375 25.5 26.9375 25.5 27.234375 \n",
       "C 25.5 27.4375 32.59375 33.921875 33.5 34.71875 \n",
       "C 39.203125 39.5 42.796875 40 45.40625 40 \n",
       "L 45.40625 42.703125 \n",
       "C 44.796875 42.703125 44.59375 42.703125 44.5 42.5 \n",
       "L 39.59375 42.5 \n",
       "C 36.796875 42.5 32.5 42.5 29.796875 42.703125 \n",
       "L 29.796875 40.09375 \n",
       "C 31.203125 39.984375 32.203125 39.1875 32.203125 37.796875 \n",
       "C 32.203125 35.796875 30.203125 33.90625 30.09375 33.90625 \n",
       "L 15 20.140625 \n",
       "L 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 6.65625 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 14.90625 0 19.296875 0 21.703125 -0.203125 \n",
       "L 21.703125 2.390625 \n",
       "C 15.296875 2.390625 14.90625 2.78125 14.90625 6.6875 \n",
       "L 14.90625 17.65625 \n",
       "L 21.796875 23.9375 \n",
       "L 31.90625 8.96875 \n",
       "C 33.09375 7.171875 34.09375 5.765625 34.09375 4.46875 \n",
       "C 34.09375 2.6875 32.296875 2.390625 31.09375 2.390625 \n",
       "L 31.09375 -0.203125 \n",
       "C 33.5 0 37.796875 0 40.296875 0 \n",
       "C 42.09375 0 45.796875 0 47.5 0 \n",
       "L 47.5 2.390625 \n",
       "C 42.796875 2.390625 41.703125 3.28125 38.703125 7.671875 \n",
       "z\n",
       "\" id=\"CMR17-107\"/>\n",
       "       <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(345.672261 254.986674)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(131.866615 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(177.557092 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(228.452462 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(274.142939 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(309.944276 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(345.225044 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(370.096103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(410.581763 0)scale(0.996264)\" xlink:href=\"#CMR17-66\"/>\n",
       "       <use transform=\"translate(475.770518 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(500.641576 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(546.332053 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(623.251643 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(668.94212 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(704.222888 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(749.91335 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(790.39901 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(825.679778 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(877.856369 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(923.546846 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_12\">\n",
       "     <g id=\"line2d_70\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"239.093628\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- rateLimitingStepOf -->\n",
       "      <defs>\n",
       "       <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "       <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "       <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "\" id=\"CMR17-79\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(365.874305 242.552877)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(80.971245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(116.252013 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(156.737673 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "       <use transform=\"translate(214.119081 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(238.99014 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(315.909729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(340.780788 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(376.061556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(400.932614 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(451.827984 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(497.518461 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "       <use transform=\"translate(548.413831 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(583.694599 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(624.180259 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(675.075629 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "       <use transform=\"translate(746.790417 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_13\">\n",
       "     <g id=\"line2d_71\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"226.659831\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_22\">\n",
       "      <!-- hasProduct -->\n",
       "      <g transform=\"translate(400.688302 230.11908)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "       <use transform=\"translate(194.973485 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(230.254253 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(278.547169 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(329.442538 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(380.337908 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(420.823569 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_14\">\n",
       "     <g id=\"line2d_72\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"214.226034\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_23\">\n",
       "      <!-- hasReactant -->\n",
       "      <g transform=\"translate(396.841643 217.685283)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-82\"/>\n",
       "       <use transform=\"translate(200.178287 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(240.663947 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(286.354424 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(326.840085 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(362.120852 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(407.811329 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(456.104245 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_15\">\n",
       "     <g id=\"line2d_73\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"201.792237\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_24\">\n",
       "      <!-- biomarkerFor -->\n",
       "      <g transform=\"translate(392.109344 205.251486)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(121.456905 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(198.376495 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(244.066972 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(279.34774 0)scale(0.996264)\" xlink:href=\"#CMR17-107\"/>\n",
       "       <use transform=\"translate(325.038202 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(365.523863 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(400.80463 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "       <use transform=\"translate(452.981222 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(498.671699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_16\">\n",
       "     <g id=\"line2d_74\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"189.35844\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- hasMember -->\n",
       "      <defs>\n",
       "       <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(400.273218 192.817689)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "       <use transform=\"translate(217.074125 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(257.559786 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(331.876922 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(385.37473 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(425.86039 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_17\">\n",
       "     <g id=\"line2d_75\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"176.924643\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_26\">\n",
       "      <!-- isA -->\n",
       "      <g transform=\"translate(436.277165 180.383892)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(60.672396 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_18\">\n",
       "     <g id=\"line2d_76\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"164.490846\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_27\">\n",
       "      <!-- directlyDecreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(375.421967 167.950095)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "       <use transform=\"translate(370.856815 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(411.342475 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(451.828136 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(487.108904 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(527.594564 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(573.285041 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(609.086379 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(649.572039 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_19\">\n",
       "     <g id=\"line2d_77\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"152.057049\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_28\">\n",
       "      <!-- causesNoChange -->\n",
       "      <g transform=\"translate(378.106313 155.516298)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(40.48566 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(86.176138 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(172.872845 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(213.358505 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(249.159843 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "       <use transform=\"translate(318.232273 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(363.92275 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(430.432631 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(481.328 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(527.018477 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(577.913847 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(623.604324 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_20\">\n",
       "     <g id=\"line2d_78\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"139.623252\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_29\">\n",
       "      <!-- transcribedTo -->\n",
       "      <defs>\n",
       "       <path d=\"M 62.203125 67.59375 \n",
       "L 4.5 67.59375 \n",
       "L 2.796875 45.59375 \n",
       "L 4.59375 45.59375 \n",
       "C 5.90625 62.125 7.296875 65 22.703125 65 \n",
       "C 24.5 65 27.40625 65 28.203125 64.90625 \n",
       "C 30.09375 64.609375 30.09375 63.40625 30.09375 61.125 \n",
       "L 30.09375 7.171875 \n",
       "C 30.09375 3.59375 29.796875 2.5 21.5 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 23.5 0 28.5 0 33.40625 0 \n",
       "C 38.296875 0 43.296875 0 48.09375 0 \n",
       "L 48.09375 2.5 \n",
       "L 45.296875 2.5 \n",
       "C 37 2.5 36.703125 3.59375 36.703125 7.171875 \n",
       "L 36.703125 61.125 \n",
       "C 36.703125 63.3125 36.703125 64.515625 38.5 64.90625 \n",
       "C 39.296875 65 42.203125 65 44 65 \n",
       "C 59.296875 65 60.796875 62.125 62.09375 45.59375 \n",
       "L 63.90625 45.59375 \n",
       "z\n",
       "\" id=\"CMR17-84\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(390.006147 143.082501)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(70.561536 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(116.252013 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(167.147382 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(202.94872 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(243.434381 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(278.715148 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(303.586207 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "       <use transform=\"translate(357.084015 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(397.569676 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(448.465046 0)scale(0.996264)\" xlink:href=\"#CMR17-84\"/>\n",
       "       <use transform=\"translate(507.167671 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_21\">\n",
       "     <g id=\"line2d_79\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"127.189455\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_30\">\n",
       "      <!-- directlyIncreases -->\n",
       "      <defs>\n",
       "       <path d=\"M 19.703125 61.015625 \n",
       "C 19.703125 65.5 20.90625 65.5 28.09375 65.5 \n",
       "L 28.09375 68 \n",
       "C 24.90625 68 19.796875 68 16.40625 68 \n",
       "C 13 68 7.90625 68 4.703125 68 \n",
       "L 4.703125 65.5 \n",
       "C 11.90625 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 11.90625 2.5 4.703125 2.5 \n",
       "L 4.703125 0 \n",
       "C 7.90625 0 13 0 16.40625 0 \n",
       "C 19.796875 0 24.90625 0 28.09375 0 \n",
       "L 28.09375 2.5 \n",
       "C 20.90625 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-73\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(378.327727 130.648704)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(111.047196 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(227.299285 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(252.170343 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "       <use transform=\"translate(300.463259 0)scale(0.996264)\" xlink:href=\"#CMR17-73\"/>\n",
       "       <use transform=\"translate(333.101668 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(383.997038 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(424.482699 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(459.763466 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(500.249127 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(545.939604 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(581.740942 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(622.226602 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_22\">\n",
       "     <g id=\"line2d_80\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"114.755658\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_31\">\n",
       "      <!-- hasVariant -->\n",
       "      <defs>\n",
       "       <path d=\"M 59 58.515625 \n",
       "C 59.90625 60.71875 61.5 65 68.203125 65 \n",
       "L 68.203125 67.6875 \n",
       "C 65.90625 67.484375 63.296875 67.484375 60.796875 67.484375 \n",
       "C 58.40625 67.484375 54.296875 67.484375 52 67.6875 \n",
       "L 52 65 \n",
       "C 57 65 57.203125 61.5 57.203125 60.90625 \n",
       "C 57.203125 59.796875 56.90625 59.109375 56.59375 58.3125 \n",
       "L 36.90625 6.96875 \n",
       "L 16.296875 60.71875 \n",
       "C 16 61.421875 15.796875 61.921875 15.796875 62.40625 \n",
       "C 15.796875 65 19.796875 65 21.90625 65 \n",
       "L 21.90625 67.484375 \n",
       "C 18.703125 67.484375 14 67.484375 10.703125 67.484375 \n",
       "C 8.09375 67.484375 3.40625 67.484375 1 67.6875 \n",
       "L 1 65 \n",
       "C 6.796875 65 7.90625 64.609375 9.09375 61.421875 \n",
       "L 32.90625 -0.40625 \n",
       "C 33.40625 -1.796875 33.5 -2 34.59375 -2 \n",
       "C 35.703125 -2 35.796875 -1.796875 36.296875 -0.40625 \n",
       "z\n",
       "\" id=\"CMR17-86\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(403.594092 118.214907)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-86\"/>\n",
       "       <use transform=\"translate(193.652359 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(239.342836 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(274.623604 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(299.494662 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(345.185139 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(393.478055 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_23\">\n",
       "     <g id=\"line2d_81\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"102.32186\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_32\">\n",
       "      <!-- translatedTo -->\n",
       "      <g transform=\"translate(395.291882 105.78111)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(35.280768 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(70.561536 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(116.252013 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(167.147382 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(202.94872 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(227.819779 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(273.510256 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(308.791024 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(349.276684 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(400.172054 0)scale(0.996264)\" xlink:href=\"#CMR17-84\"/>\n",
       "       <use transform=\"translate(458.874679 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_24\">\n",
       "     <g id=\"line2d_82\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"89.888063\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_33\">\n",
       "      <!-- negativeCorrelation -->\n",
       "      <defs>\n",
       "       <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(365.15482 93.347313)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "       <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(182.761984 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(218.042752 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(242.913811 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(288.604273 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.089933 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(395.599814 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(441.290291 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(476.571059 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(511.851826 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(552.337487 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(577.208545 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(622.899022 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(658.17979 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(683.050849 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(728.741326 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_25\">\n",
       "     <g id=\"line2d_83\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"77.454266\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_34\">\n",
       "      <!-- positiveCorrelation -->\n",
       "      <g transform=\"translate(367.59013 80.913516)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(53.497809 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(99.188286 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(134.989623 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(159.860682 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(195.14145 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(220.012508 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "       <use transform=\"translate(265.70297 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(306.18863 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(372.698511 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(418.388988 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(453.669756 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(488.950524 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(529.436184 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "       <use transform=\"translate(554.307243 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(599.99772 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(635.278488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(660.149546 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(705.840023 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_26\">\n",
       "     <g id=\"line2d_84\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"65.020469\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_35\">\n",
       "      <!-- hasComponent -->\n",
       "      <g transform=\"translate(385.633612 68.479719)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(96.585847 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(132.387185 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "       <use transform=\"translate(198.897065 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(244.587542 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "       <use transform=\"translate(321.507132 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "       <use transform=\"translate(375.004941 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(420.695418 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(471.590787 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(512.076448 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(560.369364 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_27\">\n",
       "     <g id=\"line2d_85\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"52.586672\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_36\">\n",
       "      <!-- decreases -->\n",
       "      <g transform=\"translate(410.457317 56.045922)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "       <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(131.866691 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(167.147459 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(207.633119 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(253.323596 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(289.124934 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(329.610594 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_28\">\n",
       "     <g id=\"line2d_86\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"40.152875\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_37\">\n",
       "      <!-- increases -->\n",
       "      <g transform=\"translate(412.117747 43.612125)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(24.871058 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "       <use transform=\"translate(75.766428 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(116.252089 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "       <use transform=\"translate(151.532857 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(192.018517 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(237.708994 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(273.510332 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "       <use transform=\"translate(313.995992 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_29\">\n",
       "     <g id=\"line2d_87\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"457.446275\" xlink:href=\"#mc40aa56ede\" y=\"27.719078\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_38\">\n",
       "      <!-- association -->\n",
       "      <g transform=\"translate(403.012928 31.178328)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(81.491815 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "       <use transform=\"translate(117.293152 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(165.586068 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "       <use transform=\"translate(206.071729 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(230.942787 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "       <use transform=\"translate(276.633264 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "       <use transform=\"translate(311.914032 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "       <use transform=\"translate(336.785091 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "       <use transform=\"translate(382.475568 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_39\">\n",
       "    <path d=\"M 457.446275 257.744323 \n",
       "L 457.446275 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_40\">\n",
       "    <path d=\"M 699.483468 257.744323 \n",
       "L 699.483468 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_41\">\n",
       "    <path d=\"M 457.446275 257.744323 \n",
       "L 699.483468 257.744323 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_42\">\n",
       "    <path d=\"M 457.446275 21.50218 \n",
       "L 699.483468 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_39\">\n",
       "    <!-- Number of edges: 19726 -->\n",
       "    <defs>\n",
       "     <path d=\"M 35.203125 29.125 \n",
       "C 35.203125 7.25 26.09375 0.484375 18.59375 0.484375 \n",
       "C 16.296875 0.484375 10.703125 0.484375 8.40625 4.59375 \n",
       "C 11 4.1875 12.90625 5.578125 12.90625 8.09375 \n",
       "C 12.90625 10.8125 10.703125 11.703125 9.296875 11.703125 \n",
       "C 8.40625 11.703125 5.703125 11.296875 5.703125 7.90625 \n",
       "C 5.703125 1.109375 11.59375 -2 18.796875 -2 \n",
       "C 30.296875 -2 42 10.53125 42 32.390625 \n",
       "C 42 59.640625 30.796875 65.296875 23.203125 65.296875 \n",
       "C 13.296875 65.296875 3.796875 56.6875 3.796875 43.40625 \n",
       "C 3.796875 31.109375 11.90625 22 22.09375 22 \n",
       "C 30.5 22 34.203125 29.734375 35.203125 33 \n",
       "z\n",
       "M 22.296875 23.59375 \n",
       "C 19.59375 23.59375 15.796875 24.09375 12.703125 30.03125 \n",
       "C 10.59375 33.890625 10.59375 38.453125 10.59375 43.296875 \n",
       "C 10.59375 49.15625 10.59375 53.203125 13.40625 57.5625 \n",
       "C 14.796875 59.640625 17.40625 63 23.203125 63 \n",
       "C 35 63 35 45.078125 35 41.125 \n",
       "C 35 34.09375 31.796875 23.59375 22.296875 23.59375 \n",
       "\" id=\"CMR17-57\"/>\n",
       "     <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(516.652866 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(399.315616 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(456.865095 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(497.350755 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(548.246125 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(593.936602 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(634.422262 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(670.2236 0)scale(0.996264)\" xlink:href=\"#CMR17-58\"/>\n",
       "     <use transform=\"translate(735.580319 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(781.270796 0)scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "     <use transform=\"translate(826.961273 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(872.65175 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(918.342227 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"legend_2\">\n",
       "    <g id=\"patch_43\">\n",
       "     <path d=\"M 655.502133 252.744323 \n",
       "L 692.483468 252.744323 \n",
       "Q 694.483468 252.744323 694.483468 250.744323 \n",
       "L 694.483468 237.807145 \n",
       "Q 694.483468 235.807145 692.483468 235.807145 \n",
       "L 655.502133 235.807145 \n",
       "Q 653.502133 235.807145 653.502133 237.807145 \n",
       "L 653.502133 250.744323 \n",
       "Q 653.502133 252.744323 655.502133 252.744323 \n",
       "z\n",
       "\" style=\"fill:#ffffff;opacity:0.8;stroke:#cccccc;stroke-linejoin:miter;\"/>\n",
       "    </g>\n",
       "    <g id=\"patch_44\">\n",
       "     <path d=\"M 657.502133 246.807145 \n",
       "L 677.502133 246.807145 \n",
       "L 677.502133 239.807145 \n",
       "L 657.502133 239.807145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "    </g>\n",
       "    <g id=\"text_40\">\n",
       "     <!-- 0 -->\n",
       "     <g transform=\"translate(685.502133 246.807145)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p7289b9365d\">\n",
       "   <rect height=\"236.242143\" width=\"242.037193\" x=\"89.335068\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p93f0f215ed\">\n",
       "   <rect height=\"236.242143\" width=\"242.037193\" x=\"457.446275\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x10ba63a90>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "pbt.summary.plot_summary(graph, plt, figsize=(10, 4))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Differential Gene Expression\n",
    "\n",
    "Differential gene expression data can be obtained from many sources, including ADNI and other large clinical studies. This analysis is concerned with the log-fold-changes on each gene, and not necessarily the p-value. This is better as a data-driven process becuase it does not require a model or multiple hypothesis testing on raw data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "data_path = os.path.join(owncloud_base, 'SevAD.csv')\n",
    "target_columns = ['Gene.symbol', 'logFC']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene.symbol</th>\n",
       "      <th>logFC</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ZNF616</td>\n",
       "      <td>-4.244691</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DEFB125</td>\n",
       "      <td>3.974393</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SNAP23</td>\n",
       "      <td>3.337636</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PHLDB2</td>\n",
       "      <td>3.192559</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>LOC389895</td>\n",
       "      <td>-4.296850</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{llr}\n",
       "\\toprule\n",
       "{} & Gene.symbol &     logFC \\\\\n",
       "\\midrule\n",
       "0 &      ZNF616 & -4.244691 \\\\\n",
       "1 &     DEFB125 &  3.974393 \\\\\n",
       "3 &      SNAP23 &  3.337636 \\\\\n",
       "4 &      PHLDB2 &  3.192559 \\\\\n",
       "5 &   LOC389895 & -4.296850 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "  Gene.symbol     logFC\n",
       "0      ZNF616 -4.244691\n",
       "1     DEFB125  3.974393\n",
       "3      SNAP23  3.337636\n",
       "4      PHLDB2  3.192559\n",
       "5   LOC389895 -4.296850"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df = pd.read_csv(data_path)\n",
    "df = df.loc[df['Gene.symbol'].notnull(), target_columns]\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A histogram of the log-fold-changes shows that the data are normally distributed, as expected for differential gene expression data."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# max_fc = {}\n",
    "# for row, symbol, fc in df.itertuples():\n",
    "#    if symbol not in max_fc or abs(max_fc[symbol]) < abs(fc):\n",
    "#        max_fc[symbol] = fc\n",
    "        \n",
    "data = {k: v for _, k, v in df.itertuples()}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"274pt\" version=\"1.1\" viewBox=\"0 0 397 274\" width=\"397pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 274.853533 \n",
       "L 397.462353 274.853533 \n",
       "L 397.462353 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 386.762353 238.94218 \n",
       "L 386.762353 21.50218 \n",
       "L 51.962353 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 67.180535 238.94218 \n",
       "L 97.616898 238.94218 \n",
       "L 97.616898 238.904231 \n",
       "L 67.180535 238.904231 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 97.616898 238.94218 \n",
       "L 128.053262 238.94218 \n",
       "L 128.053262 238.657565 \n",
       "L 97.616898 238.657565 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 128.053262 238.94218 \n",
       "L 158.489626 238.94218 \n",
       "L 158.489626 234.122703 \n",
       "L 128.053262 234.122703 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 158.489626 238.94218 \n",
       "L 188.925989 238.94218 \n",
       "L 188.925989 212.397108 \n",
       "L 158.489626 212.397108 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 188.925989 238.94218 \n",
       "L 219.362353 238.94218 \n",
       "L 219.362353 101.017857 \n",
       "L 188.925989 101.017857 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 219.362353 238.94218 \n",
       "L 249.798717 238.94218 \n",
       "L 249.798717 31.856466 \n",
       "L 219.362353 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 249.798717 238.94218 \n",
       "L 280.23508 238.94218 \n",
       "L 280.23508 201.373029 \n",
       "L 249.798717 201.373029 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 280.23508 238.94218 \n",
       "L 310.671444 238.94218 \n",
       "L 310.671444 233.060141 \n",
       "L 280.23508 233.060141 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 310.671444 238.94218 \n",
       "L 341.107808 238.94218 \n",
       "L 341.107808 238.221156 \n",
       "L 310.671444 238.221156 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p75af197ac9)\" d=\"M 341.107808 238.94218 \n",
       "L 371.544171 238.94218 \n",
       "L 371.544171 238.828334 \n",
       "L 341.107808 238.828334 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m8de105a4f7\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"77.978435\" xlink:href=\"#m8de105a4f7\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $-4$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(71.613396 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"150.728385\" xlink:href=\"#m8de105a4f7\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $-2$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(144.363345 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"223.478335\" xlink:href=\"#m8de105a4f7\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $0$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(220.987667 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"296.228284\" xlink:href=\"#m8de105a4f7\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $2$ -->\n",
       "      <g transform=\"translate(293.737617 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"368.978234\" xlink:href=\"#m8de105a4f7\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $4$ -->\n",
       "      <g transform=\"translate(366.487567 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_6\">\n",
       "     <!-- Log-Fold-Change -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.203125 25.234375 \n",
       "L 52.40625 25.234375 \n",
       "C 51.59375 15.359375 50.296875 2.59375 33.203125 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 60.90625 \n",
       "C 19.703125 64.515625 19.90625 65.5 27.5 65.5 \n",
       "L 29.703125 65.5 \n",
       "L 29.703125 68 \n",
       "C 25.59375 68 20.796875 68 16.703125 68 \n",
       "C 13.59375 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 7.0625 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 51.796875 0 \n",
       "z\n",
       "\" id=\"CMR17-76\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "      <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "      <path d=\"M 29.203125 67.59375 \n",
       "L 29.203125 65 \n",
       "C 35.09375 65 36 64.40625 36 59.6875 \n",
       "L 36 36.640625 \n",
       "C 35.59375 37.140625 31.5 43.59375 23.40625 43.59375 \n",
       "C 13.203125 43.59375 3.296875 34.5 3.296875 21.3125 \n",
       "C 3.296875 8.203125 12.59375 -1 22.40625 -1 \n",
       "C 30.90625 -1 35.296875 5.265625 35.796875 5.9375 \n",
       "L 35.796875 -0.984375 \n",
       "L 47.90625 -0.984375 \n",
       "L 47.90625 2 \n",
       "C 42 2 41.09375 2.59375 41.09375 7.390625 \n",
       "L 41.09375 68.6875 \n",
       "z\n",
       "M 35.796875 11.796875 \n",
       "C 35.796875 8.796875 34 6.09375 31.703125 4.09375 \n",
       "C 28.296875 1.09375 24.90625 0.59375 23 0.59375 \n",
       "C 20.09375 0.59375 9.703125 2.09375 9.703125 21.1875 \n",
       "C 9.703125 40.8125 21.296875 42 23.90625 42 \n",
       "C 28.5 42 32.203125 39.40625 34.5 35.796875 \n",
       "C 35.796875 33.703125 35.796875 33.40625 35.796875 31.609375 \n",
       "z\n",
       "\" id=\"CMR17-100\"/>\n",
       "      <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "      <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(182.140743 265.716355)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "      <use transform=\"translate(57.381408 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(103.071885 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(148.762362 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(178.838328 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(231.01492 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(276.705397 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "      <use transform=\"translate(301.576455 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "      <use transform=\"translate(352.471825 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "      <use transform=\"translate(382.547791 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "      <use transform=\"translate(449.057672 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(499.953042 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(545.643519 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(596.538889 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(642.229366 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mdc86cbaf3a\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(39.981018 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"200.993542\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $2000$ -->\n",
       "      <g transform=\"translate(25.037012 204.452792)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"163.044905\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $4000$ -->\n",
       "      <g transform=\"translate(25.037012 166.504154)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"125.096268\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $6000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 128.555517)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"87.14763\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $8000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(25.037012 90.606879)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"51.962353\" xlink:href=\"#mdc86cbaf3a\" y=\"49.198993\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $10000$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.055677 52.658242)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_13\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 152.444442)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 51.962353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 386.762353 238.94218 \n",
       "L 386.762353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 51.962353 238.94218 \n",
       "L 386.762353 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 51.962353 21.50218 \n",
       "L 386.762353 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_14\">\n",
       "    <!-- Distribution of Log-Fold-Changes -->\n",
       "    <defs>\n",
       "     <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(133.537855 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(70.393556 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(95.264614 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(131.065952 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(166.34672 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(201.627488 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(226.498546 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "     <use transform=\"translate(277.393916 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "     <use transform=\"translate(328.289286 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(363.570053 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(388.441112 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(434.131589 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(515.102925 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(560.793402 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(618.342881 0)scale(0.996264)\" xlink:href=\"#CMR17-76\"/>\n",
       "     <use transform=\"translate(675.724289 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(721.414766 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(767.105243 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(797.18121 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "     <use transform=\"translate(849.357801 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(895.048278 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(919.919336 0)scale(0.996264)\" xlink:href=\"#CMR17-100\"/>\n",
       "     <use transform=\"translate(970.814706 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(1000.890672 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(1067.400553 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(1118.295923 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1163.9864 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1214.88177 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(1260.572247 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1301.057907 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p75af197ac9\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"51.962353\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x11147c9b0>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.title('Distribution of Log-Fold-Changes')\n",
    "plt.xlabel('Log-Fold-Change')\n",
    "plt.ylabel('Frequency')\n",
    "plt.hist(list(data.values()))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Overlap Summary"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The number of shared elements in the knowledge assembly and differential gene data set are counted."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"249pt\" version=\"1.1\" viewBox=\"0 0 333 249\" width=\"333pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 249.64218 \n",
       "L 333.252183 249.64218 \n",
       "L 333.252183 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path clip-path=\"url(#paaf6e74d29)\" d=\"M 122.868937 146.021899 \n",
       "C 121.962646 140.80414 121.506979 135.518063 121.506979 130.22218 \n",
       "C 121.506979 124.926297 121.962646 119.64022 122.868937 114.422461 \n",
       "C 119.032647 118.790127 116.915404 124.40895 116.915404 130.22218 \n",
       "C 116.915404 136.035409 119.032647 141.654232 122.868937 146.021899 \n",
       "\" style=\"fill:#ff0000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#paaf6e74d29)\" d=\"M 122.868937 114.422461 \n",
       "C 124.465686 112.604541 126.328831 111.039136 128.394789 109.77967 \n",
       "C 130.460746 108.520203 132.705744 107.581167 135.053165 106.994608 \n",
       "C 137.400586 106.408049 139.823419 106.180716 142.23898 106.320369 \n",
       "C 144.654541 106.460022 147.035034 106.965052 149.299219 107.818226 \n",
       "C 151.563403 108.671399 153.685226 109.862898 155.592275 111.352059 \n",
       "C 157.499324 112.84122 159.169655 114.610909 160.546264 116.600729 \n",
       "C 161.922873 118.590549 162.989919 120.777606 163.710988 123.087259 \n",
       "C 164.432056 125.396912 164.798849 127.802585 164.798849 130.22218 \n",
       "C 164.798849 132.641774 164.432056 135.047448 163.710988 137.357101 \n",
       "C 162.989919 139.666754 161.922873 141.85381 160.546264 143.843631 \n",
       "C 159.169655 145.833451 157.499324 147.603139 155.592275 149.092301 \n",
       "C 153.685226 150.581462 151.563403 151.772961 149.299219 152.626134 \n",
       "C 147.035034 153.479307 144.654541 153.984338 142.23898 154.123991 \n",
       "C 139.823419 154.263643 137.400586 154.036311 135.053165 153.449752 \n",
       "C 132.705744 152.863193 130.460746 151.924157 128.394789 150.66469 \n",
       "C 126.328831 149.405223 124.465686 147.839819 122.868937 146.021899 \n",
       "C 124.829494 157.309352 128.87399 168.133906 134.795379 177.94143 \n",
       "C 140.716769 187.748954 148.412255 196.369187 157.48789 203.360842 \n",
       "C 166.563525 210.352496 176.861753 215.594195 187.855388 218.817605 \n",
       "C 198.849023 222.041015 210.347212 223.190177 221.761339 222.206264 \n",
       "C 233.175466 221.22235 244.307379 218.122442 254.587213 213.065229 \n",
       "C 264.867046 208.008016 274.11634 201.081294 281.861604 192.639654 \n",
       "C 289.606867 184.198015 295.713641 174.388008 299.869307 163.711828 \n",
       "C 304.024972 153.035648 306.157387 141.678636 306.157387 130.22218 \n",
       "C 306.157387 118.765724 304.024972 107.408712 299.869307 96.732531 \n",
       "C 295.713641 86.056351 289.606867 76.246344 281.861604 67.804705 \n",
       "C 274.11634 59.363066 264.867046 52.436343 254.587213 47.379131 \n",
       "C 244.307379 42.321918 233.175466 39.222009 221.761339 38.238096 \n",
       "C 210.347212 37.254182 198.849023 38.403344 187.855388 41.626755 \n",
       "C 176.861753 44.850165 166.563525 50.091864 157.48789 57.083518 \n",
       "C 148.412255 64.075172 140.716769 72.695405 134.795379 82.502929 \n",
       "C 128.87399 92.310453 124.829494 103.135007 122.868937 114.422461 \n",
       "\" style=\"fill:#008000;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#paaf6e74d29)\" d=\"M 122.868937 146.021899 \n",
       "C 124.465686 147.839819 126.328831 149.405223 128.394789 150.66469 \n",
       "C 130.460746 151.924157 132.705744 152.863193 135.053165 153.449752 \n",
       "C 137.400586 154.036311 139.823419 154.263643 142.23898 154.123991 \n",
       "C 144.654541 153.984338 147.035034 153.479307 149.299219 152.626134 \n",
       "C 151.563403 151.772961 153.685226 150.581462 155.592275 149.092301 \n",
       "C 157.499324 147.603139 159.169655 145.833451 160.546264 143.843631 \n",
       "C 161.922873 141.85381 162.989919 139.666754 163.710988 137.357101 \n",
       "C 164.432056 135.047448 164.798849 132.641774 164.798849 130.22218 \n",
       "C 164.798849 127.802585 164.432056 125.396912 163.710988 123.087259 \n",
       "C 162.989919 120.777606 161.922873 118.590549 160.546264 116.600729 \n",
       "C 159.169655 114.610909 157.499324 112.84122 155.592275 111.352059 \n",
       "C 153.685226 109.862898 151.563403 108.671399 149.299219 107.818226 \n",
       "C 147.035034 106.965052 144.654541 106.460022 142.23898 106.320369 \n",
       "C 139.823419 106.180716 137.400586 106.408049 135.053165 106.994608 \n",
       "C 132.705744 107.581167 130.460746 108.520203 128.394789 109.77967 \n",
       "C 126.328831 111.039136 124.465686 112.604541 122.868937 114.422461 \n",
       "C 121.962646 119.64022 121.506979 124.926297 121.506979 130.22218 \n",
       "C 121.506979 135.518063 121.962646 140.80414 122.868937 146.021899 \n",
       "\" style=\"fill:#b25900;opacity:0.4;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_1\">\n",
       "    <!-- 83 -->\n",
       "    <defs>\n",
       "     <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "     <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(114.229856 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_2\">\n",
       "    <!-- 20778 -->\n",
       "    <defs>\n",
       "     <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "     <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "     <path d=\"M 45.09375 61.578125 \n",
       "L 45.09375 63.765625 \n",
       "L 21.59375 63.765625 \n",
       "C 9.90625 63.765625 9.703125 65.078125 9.296875 67 \n",
       "L 7.5 67 \n",
       "L 4.59375 48.34375 \n",
       "L 6.40625 48.34375 \n",
       "C 6.703125 50.234375 7.40625 55.3125 8.59375 57.203125 \n",
       "C 9.203125 58 16.59375 58 18.296875 58 \n",
       "L 40.296875 58 \n",
       "L 29.203125 41.578125 \n",
       "C 21.796875 30.53125 16.703125 15.609375 16.703125 2.578125 \n",
       "C 16.703125 1.390625 16.703125 -2 20.296875 -2 \n",
       "C 23.90625 -2 23.90625 1.390625 23.90625 2.671875 \n",
       "L 23.90625 7.265625 \n",
       "C 23.90625 23.5625 26.703125 34.3125 31.296875 41.1875 \n",
       "z\n",
       "\" id=\"CMR17-55\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(223.02478 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-55\"/>\n",
       "     <use transform=\"translate(182.761908 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_3\">\n",
       "    <!-- 1409 -->\n",
       "    <defs>\n",
       "     <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "     <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "     <path d=\"M 35.203125 29.125 \n",
       "C 35.203125 7.25 26.09375 0.484375 18.59375 0.484375 \n",
       "C 16.296875 0.484375 10.703125 0.484375 8.40625 4.59375 \n",
       "C 11 4.1875 12.90625 5.578125 12.90625 8.09375 \n",
       "C 12.90625 10.8125 10.703125 11.703125 9.296875 11.703125 \n",
       "C 8.40625 11.703125 5.703125 11.296875 5.703125 7.90625 \n",
       "C 5.703125 1.109375 11.59375 -2 18.796875 -2 \n",
       "C 30.296875 -2 42 10.53125 42 32.390625 \n",
       "C 42 59.640625 30.796875 65.296875 23.203125 65.296875 \n",
       "C 13.296875 65.296875 3.796875 56.6875 3.796875 43.40625 \n",
       "C 3.796875 31.109375 11.90625 22 22.09375 22 \n",
       "C 30.5 22 34.203125 29.734375 35.203125 33 \n",
       "z\n",
       "M 22.296875 23.59375 \n",
       "C 19.59375 23.59375 15.796875 24.09375 12.703125 30.03125 \n",
       "C 10.59375 33.890625 10.59375 38.453125 10.59375 43.296875 \n",
       "C 10.59375 49.15625 10.59375 53.203125 13.40625 57.5625 \n",
       "C 14.796875 59.640625 17.40625 63 23.203125 63 \n",
       "C 35 63 35 45.078125 35 41.125 \n",
       "C 35 34.09375 31.796875 23.59375 22.296875 23.59375 \n",
       "\" id=\"CMR17-57\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(133.190244 132.71284)scale(0.1 -0.1)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "     <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "     <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "     <use transform=\"translate(137.071431 0)scale(0.996264)\" xlink:href=\"#CMR17-57\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_4\">\n",
       "    <!-- Alzheimer's Disease KAM -->\n",
       "    <defs>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 15.296875 68.6875 \n",
       "L 3.40625 67.59375 \n",
       "L 3.40625 65 \n",
       "C 9.296875 65 10.203125 64.40625 10.203125 59.640625 \n",
       "L 10.203125 6.65625 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.203125 0 12.703125 0 \n",
       "C 15.296875 0 19.703125 0 22.09375 -0.203125 \n",
       "L 22.09375 2.390625 \n",
       "C 15.703125 2.390625 15.296875 2.78125 15.296875 6.65625 \n",
       "z\n",
       "\" id=\"CMR17-108\"/>\n",
       "     <path d=\"M 36.09375 40.703125 \n",
       "C 36.90625 41.71875 36.90625 41.90625 36.90625 42.109375 \n",
       "C 36.90625 43 36.5 43 34.703125 43 \n",
       "L 4.296875 43 \n",
       "L 3.296875 27.5 \n",
       "L 5.09375 27.5 \n",
       "C 5.796875 37.203125 6.90625 41.40625 18.40625 41.40625 \n",
       "L 30.59375 41.40625 \n",
       "L 2 1.5 \n",
       "C 2 0 2.09375 0 4.203125 0 \n",
       "L 35.796875 0 \n",
       "L 37.203125 18 \n",
       "L 35.40625 18 \n",
       "C 34.59375 6.90625 33.296875 1.796875 21.09375 1.796875 \n",
       "L 8.296875 1.796875 \n",
       "z\n",
       "\" id=\"CMR17-122\"/>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "     <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "     <path d=\"M 16 62.890625 \n",
       "C 16.203125 61.796875 16.296875 61.390625 16.296875 59.6875 \n",
       "C 16.296875 53.359375 13.796875 47.53125 9.90625 43.40625 \n",
       "C 9.5 43 9.296875 42.703125 9.296875 42.296875 \n",
       "C 9.296875 41.890625 9.703125 41.5 10.09375 41.5 \n",
       "C 10.796875 41.5 17.90625 48.515625 17.90625 59.546875 \n",
       "C 17.90625 61.75 17.703125 69.15625 12.5 69.15625 \n",
       "C 10.296875 69.15625 8.40625 67.671875 8.40625 65.078125 \n",
       "C 8.40625 62.484375 10.296875 61 12.5 61 \n",
       "C 13.40625 61 15.09375 61.296875 16 62.890625 \n",
       "\" id=\"CMR17-39\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.9375 \n",
       "L 13.203125 7.0625 \n",
       "C 13.203125 2.59375 12.09375 2.59375 5.203125 2.59375 \n",
       "L 5.203125 0 \n",
       "L 37 0 \n",
       "C 53.09375 0 66 14.734375 66 33.453125 \n",
       "C 66 52.28125 53.296875 68 37 68 \n",
       "z\n",
       "M 23.796875 2.59375 \n",
       "C 19.796875 2.59375 19.59375 3.078125 19.59375 6.46875 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 35 65.40625 \n",
       "C 43.296875 65.40625 58.59375 60.53125 58.59375 33.453125 \n",
       "C 58.59375 19.40625 54.5 13.734375 52.59375 11.046875 \n",
       "C 50.203125 7.96875 44.5 2.59375 35 2.59375 \n",
       "z\n",
       "\" id=\"CMR17-68\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 36.90625 41.796875 \n",
       "L 54.203125 59.09375 \n",
       "C 57.796875 62.59375 61.59375 65.40625 67.203125 65.40625 \n",
       "L 67.203125 68 \n",
       "C 65.203125 68 62.703125 68 60.703125 68 \n",
       "C 58 68 53.296875 68 50.703125 68.09375 \n",
       "L 50.703125 65.5 \n",
       "C 53.296875 65.3125 53.703125 63.515625 53.703125 62.796875 \n",
       "C 53.703125 62.109375 53.296875 61.015625 52.90625 60.421875 \n",
       "L 19.703125 27.15625 \n",
       "L 19.703125 60.9375 \n",
       "C 19.703125 65.40625 20.796875 65.40625 27.703125 65.40625 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.40625 \n",
       "C 12 65.40625 13.09375 65.40625 13.09375 60.921875 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.984375 \n",
       "L 19.703125 24.671875 \n",
       "L 32.5 37.34375 \n",
       "L 51.40625 8.390625 \n",
       "C 52 7.390625 52.703125 6.09375 52.703125 5.09375 \n",
       "C 52.703125 2.5 49.90625 2.5 48.59375 2.5 \n",
       "L 48.59375 0 \n",
       "C 51.796875 0 56.40625 0 59.703125 0 \n",
       "C 62.09375 0 66.203125 0 68.5 -0.09375 \n",
       "L 68.5 2.5 \n",
       "C 63.703125 2.5 62.203125 3.09375 59.203125 7.703125 \n",
       "z\n",
       "\" id=\"CMR17-75\"/>\n",
       "     <path d=\"M 20.90625 66 \n",
       "C 20.296875 67.5 20.203125 67.5 18.09375 67.5 \n",
       "L 5.296875 67.5 \n",
       "L 5.296875 65 \n",
       "C 12.203125 65 13.296875 65 13.296875 60.546875 \n",
       "L 13.296875 9.53125 \n",
       "C 13.296875 7.046875 13.296875 2.390625 5.296875 2.390625 \n",
       "L 5.296875 -0.203125 \n",
       "C 7.59375 0 11.796875 0 14.296875 0 \n",
       "C 16.796875 0 21 0 23.296875 -0.203125 \n",
       "L 23.296875 2.390625 \n",
       "C 15.296875 2.390625 15.296875 7.046875 15.296875 9.53125 \n",
       "L 15.296875 64.609375 \n",
       "L 15.40625 64.609375 \n",
       "L 38.90625 1.390625 \n",
       "C 39.296875 0.390625 39.5 -0.203125 40.296875 -0.203125 \n",
       "C 41.09375 -0.203125 41.296875 0.390625 41.703125 1.390625 \n",
       "L 65.40625 65.09375 \n",
       "L 65.5 65.09375 \n",
       "L 65.5 6.84375 \n",
       "C 65.5 2.390625 64.40625 2.390625 57.5 2.390625 \n",
       "L 57.5 -0.203125 \n",
       "C 60.203125 0 65.59375 0 68.5 0 \n",
       "C 71.40625 0 76.90625 0 79.59375 -0.203125 \n",
       "L 79.59375 2.390625 \n",
       "C 72.703125 2.390625 71.59375 2.390625 71.59375 6.84375 \n",
       "L 71.59375 60.546875 \n",
       "C 71.59375 65 72.703125 65 79.59375 65 \n",
       "L 79.59375 67.5 \n",
       "L 66.90625 67.5 \n",
       "C 64.796875 67.5 64.703125 67.40625 64.09375 65.90625 \n",
       "L 42.5 7.84375 \n",
       "z\n",
       "\" id=\"CMR17-77\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(7.2 168.279429)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-108\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "     <use transform=\"translate(134.429149 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "     <use transform=\"translate(185.324519 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(225.810179 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(250.681237 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(327.600827 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(368.086488 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(403.367256 0)scale(0.996264)\" xlink:href=\"#CMR17-39\"/>\n",
       "     <use transform=\"translate(428.238314 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(494.115618 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(564.509174 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(589.380232 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(625.18157 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(665.667231 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(711.357708 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(747.159045 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(817.720672 0)scale(0.996264)\" xlink:href=\"#CMR17-75\"/>\n",
       "     <use transform=\"translate(889.395449 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(958.467879 0)scale(0.996264)\" xlink:href=\"#CMR17-77\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_5\">\n",
       "    <!-- E-GEOD-5281 -->\n",
       "    <defs>\n",
       "     <path d=\"M 60.703125 25.234375 \n",
       "L 58.90625 25.234375 \n",
       "C 56.703125 10.5625 55.40625 2.59375 37.796875 2.59375 \n",
       "L 23.90625 2.59375 \n",
       "C 19.90625 2.59375 19.703125 3.078125 19.703125 6.46875 \n",
       "L 19.703125 34 \n",
       "L 29.09375 34 \n",
       "C 38.5 34 39.40625 30.625 39.40625 22.34375 \n",
       "L 41.203125 22.34375 \n",
       "L 41.203125 48.140625 \n",
       "L 39.40625 48.140625 \n",
       "C 39.40625 39.953125 38.5 36.59375 29.09375 36.59375 \n",
       "L 19.703125 36.59375 \n",
       "L 19.703125 61.140625 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 37.59375 65 \n",
       "C 53.09375 65 54.90625 58.890625 56.296875 45.765625 \n",
       "L 58.09375 45.765625 \n",
       "L 55.703125 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 7.046875 \n",
       "C 13.09375 2.59375 12 2.59375 5.09375 2.59375 \n",
       "L 5.09375 0 \n",
       "L 57.09375 0 \n",
       "z\n",
       "\" id=\"CMR17-69\"/>\n",
       "     <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "     <path d=\"M 62.09375 19.265625 \n",
       "C 62.09375 23.015625 62.40625 23.515625 68.59375 23.515625 \n",
       "L 68.59375 26.09375 \n",
       "C 66.203125 26 60.796875 26 58.09375 26 \n",
       "C 55.296875 26 47.5 26 43.703125 26 \n",
       "L 43.703125 23.515625 \n",
       "L 46.703125 23.515625 \n",
       "C 55.296875 23.515625 55.5 22.40625 55.5 18.765625 \n",
       "L 55.5 12.09375 \n",
       "C 55.5 0.984375 42.5 0.984375 39.40625 0.984375 \n",
       "C 29.09375 0.984375 12.203125 7.984375 12.203125 33.75 \n",
       "C 12.203125 59.5 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.65625 59.90625 42.828125 \n",
       "C 60.09375 41.625 60.09375 41.421875 61 41.421875 \n",
       "C 62.09375 41.421875 62.09375 41.625 62.09375 43.421875 \n",
       "L 62.09375 67 \n",
       "C 62.09375 68.78125 62 68.890625 61.40625 68.890625 \n",
       "C 61 68.890625 60.90625 68.78125 60.203125 67.59375 \n",
       "L 55.796875 59.234375 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 13.203125 19.796875 -2 37.40625 -2 \n",
       "C 45.5 -2 53.40625 0.875 56.5 6.515625 \n",
       "C 58.203125 2.46875 61 -0.3125 61.5 -0.3125 \n",
       "C 62 -0.3125 62.09375 -0.203125 62.09375 1.5625 \n",
       "z\n",
       "\" id=\"CMR17-71\"/>\n",
       "     <path d=\"M 67.296875 33.359375 \n",
       "C 67.296875 53.703125 52.59375 69 36 69 \n",
       "C 19 69 4.59375 53.5 4.59375 33.359375 \n",
       "C 4.59375 13.203125 19.203125 -2 35.90625 -2 \n",
       "C 53 -2 67.296875 13.5 67.296875 33.359375 \n",
       "M 36 -0.203125 \n",
       "C 24.59375 -0.203125 12.203125 11.109375 12.203125 34.640625 \n",
       "C 12.203125 58.0625 25.796875 67.203125 35.90625 67.203125 \n",
       "C 46.5 67.203125 59.703125 57.78125 59.703125 34.640625 \n",
       "C 59.703125 10.71875 47 -0.203125 36 -0.203125 \n",
       "\" id=\"CMR17-79\"/>\n",
       "     <path d=\"M 11.40625 57.6875 \n",
       "C 12.40625 57.28125 16.5 56 20.703125 56 \n",
       "C 30 56 35.09375 61.109375 38 64.0625 \n",
       "C 38 64.953125 38 65.5 37.40625 65.5 \n",
       "C 37.296875 65.5 37.09375 65.5 36.296875 65.046875 \n",
       "C 32.796875 63.40625 28.703125 62.078125 23.703125 62.078125 \n",
       "C 20.703125 62.078125 16.203125 62.484375 11.296875 64.671875 \n",
       "C 10.203125 65.171875 10 65.171875 9.90625 65.171875 \n",
       "C 9.40625 65.171875 9.296875 65.0625 9.296875 63.078125 \n",
       "L 9.296875 34.421875 \n",
       "C 9.296875 32.640625 9.296875 32.140625 10.296875 32.140625 \n",
       "C 10.796875 32.140625 11 32.34375 11.5 33.03125 \n",
       "C 14.703125 37.515625 19.09375 39.40625 24.09375 39.40625 \n",
       "C 27.59375 39.40625 35.09375 37.21875 35.09375 20.140625 \n",
       "C 35.09375 16.953125 35.09375 11.171875 32.09375 6.578125 \n",
       "C 29.59375 2.484375 25.703125 0.390625 21.40625 0.390625 \n",
       "C 14.796875 0.390625 8.09375 5 6.296875 12.71875 \n",
       "C 6.703125 12.609375 7.5 12.421875 7.90625 12.421875 \n",
       "C 9.203125 12.421875 11.703125 13.125 11.703125 16.21875 \n",
       "C 11.703125 18.90625 9.796875 20 7.90625 20 \n",
       "C 5.59375 20 4.09375 18.59375 4.09375 15.796875 \n",
       "C 4.09375 7.09375 11 -2 21.59375 -2 \n",
       "C 31.90625 -2 41.703125 6.875 41.703125 19.75 \n",
       "C 41.703125 31.71875 33.90625 41 24.203125 41 \n",
       "C 19.09375 41 14.796875 39.109375 11.40625 35.53125 \n",
       "z\n",
       "\" id=\"CMR17-53\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(213.832183 236.66291)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(62.586301 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(92.662267 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(165.017613 0)scale(0.996264)\" xlink:href=\"#CMR17-69\"/>\n",
       "     <use transform=\"translate(227.603913 0)scale(0.996264)\" xlink:href=\"#CMR17-79\"/>\n",
       "     <use transform=\"translate(299.318702 0)scale(0.996264)\" xlink:href=\"#CMR17-68\"/>\n",
       "     <use transform=\"translate(369.712258 0)scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "     <use transform=\"translate(399.788224 0)scale(0.996264)\" xlink:href=\"#CMR17-53\"/>\n",
       "     <use transform=\"translate(445.478701 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "     <use transform=\"translate(491.169178 0)scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "     <use transform=\"translate(536.859655 0)scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"text_6\">\n",
       "    <!-- HGNC Gene Names -->\n",
       "    <defs>\n",
       "     <path d=\"M 56.09375 61.015625 \n",
       "C 56.09375 65.5 57.203125 65.5 64.09375 65.5 \n",
       "L 64.09375 68 \n",
       "C 61.203125 68 55.90625 68 52.796875 68 \n",
       "C 49.703125 68 44.40625 68 41.5 68 \n",
       "L 41.5 65.5 \n",
       "C 48.40625 65.5 49.5 65.5 49.5 61 \n",
       "L 49.5 36.578125 \n",
       "L 19.703125 36.578125 \n",
       "L 19.703125 61 \n",
       "C 19.703125 65.5 20.796875 65.5 27.703125 65.5 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.96875 \n",
       "L 19.703125 34 \n",
       "L 49.5 34 \n",
       "L 49.5 6.96875 \n",
       "C 49.5 2.5 48.40625 2.5 41.5 2.5 \n",
       "L 41.5 0 \n",
       "C 44.40625 0 49.703125 0 52.796875 0 \n",
       "C 55.90625 0 61.203125 0 64.09375 0 \n",
       "L 64.09375 2.5 \n",
       "C 57.203125 2.5 56.09375 2.5 56.09375 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-72\"/>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 62.09375 67.015625 \n",
       "C 62.09375 68.796875 62 68.90625 61.40625 68.90625 \n",
       "C 61 68.90625 60.90625 68.796875 60.203125 67.609375 \n",
       "L 55.796875 59.25 \n",
       "C 50.90625 65.421875 44.90625 69 37.203125 69 \n",
       "C 20.09375 69 4.59375 54 4.59375 33.5625 \n",
       "C 4.59375 12.90625 20.09375 -2 37.296875 -2 \n",
       "C 52.59375 -2 62.09375 11.40625 62.09375 22.828125 \n",
       "C 62.09375 23.8125 62.09375 24.21875 61.203125 24.21875 \n",
       "C 60.40625 24.21875 60.40625 23.921875 60.296875 23.03125 \n",
       "C 59.5 9.140625 49.5 0.59375 38.703125 0.59375 \n",
       "C 28.5 0.59375 12.203125 7.734375 12.203125 33.546875 \n",
       "C 12.203125 59.453125 28.796875 66.40625 38.5 66.40625 \n",
       "C 49.796875 66.40625 58 56.6875 59.90625 42.890625 \n",
       "C 60.09375 41.6875 60.09375 41.484375 61 41.484375 \n",
       "C 62.09375 41.484375 62.09375 41.6875 62.09375 43.484375 \n",
       "z\n",
       "\" id=\"CMR17-67\"/>\n",
       "     <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(159.891007 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-72\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(141.427776 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(210.500206 0)scale(0.996264)\" xlink:href=\"#CMR17-67\"/>\n",
       "     <use transform=\"translate(307.086052 0)scale(0.996264)\" xlink:href=\"#CMR17-71\"/>\n",
       "     <use transform=\"translate(379.441398 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(419.927059 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(470.822429 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(541.384055 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(610.456485 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(656.146962 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(733.066552 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(773.552213 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"paaf6e74d29\">\n",
       "   <rect height=\"217.44\" width=\"222.031576\" x=\"100.520607\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x1112bb7b8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "hgnc_names = pbt.summary.get_names(graph, 'HGNC')\n",
    "df_names = set(df['Gene.symbol'])\n",
    "overlapping_hgnc_names = hgnc_names & df_names\n",
    "\n",
    "plt.title('HGNC Gene Names')\n",
    "venn2([hgnc_names, df_names], set_labels=[\"Alzheimer's Disease KAM\", 'E-GEOD-5281'])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Integration\n",
    "\n",
    "Finally, the differential gene expression data are ovelayed on the BEL graph with [pbt.integration.overlay_type_data](http://pybel-tools.readthedocs.io/en/latest/integration.html#pybel_tools.integration.overlay_type_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "label = 'weight'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.integration.overlay_type_data(graph, data, label, GENE, 'HGNC', overwrite=False, impute=0)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Subgraph Generation\n",
    "\n",
    "A subgraph is induced around an example biological process, Inflammatory Response, by selecting all of the source nodes of upstream causal edges, where this process is the target node. These networks are then enriched by expandind around all of those nodes' upstream edges as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "inflamation_node = BIOPROCESS, 'GOBP', 'inflammatory response'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 1/2\n",
    "\n",
    "All of the nodes that have upstream causal relations to Inflammatory Response are added."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 32\n",
      "Number of edges: 53\n",
      "Network density: 0.05342741935483871\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 1.65625\n",
      "Average out-degree: 1.65625\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"bqvxdaujygremoqt\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"cname\": \"SCARF1\", \"id\": [\"Protein\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"cname\": \"SCARB1\", \"id\": [\"Protein\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"cname\": \"IRAK1\", \"id\": [\"RNA\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"cname\": \"IRAK2\", \"id\": [\"RNA\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"cname\": \"TLR2\", \"id\": [\"Protein\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"leukotriene\", \"cname\": \"leukotriene\", \"id\": [\"Abundance\", \"CHEBI\", \"leukotriene\"]}, {\"function\": \"Abundance\", \"namespace\": \"ADO\", \"name\": \"neuroprotectin\", \"cname\": \"neuroprotectin\", \"id\": [\"Abundance\", \"ADO\", \"neuroprotectin\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"melatonin\", \"cname\": \"melatonin\", \"id\": [\"Abundance\", \"CHEBI\", \"melatonin\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"NOD2\", \"cname\": \"NOD2\", \"id\": [\"Protein\", \"HGNC\", \"NOD2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"FPR1\", \"cname\": \"FPR1\", \"id\": [\"Protein\", \"HGNC\", \"FPR1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"FPR2\", \"cname\": \"FPR2\", \"id\": [\"Protein\", \"HGNC\", \"FPR2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PTX3\", \"cname\": \"PTX3\", \"id\": [\"Protein\", \"HGNC\", \"PTX3\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PLA2G7\", \"cname\": \"PLA2G7\", \"id\": [\"Protein\", \"HGNC\", \"PLA2G7\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"cname\": \"IL12B\", \"id\": [\"Protein\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"cname\": \"IL23A\", \"id\": [\"Protein\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"cname\": \"CD14\", \"id\": [\"Protein\", \"HGNC\", \"CD14\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MYD88\", \"cname\": \"MYD88\", \"id\": [\"Protein\", \"HGNC\", \"MYD88\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Abundance\", \"namespace\": \"CHEBI\", \"name\": \"noradrenaline\", \"cname\": \"noradrenaline\", \"id\": [\"Abundance\", \"CHEBI\", \"noradrenaline\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"cname\": \"CX3CR1\", \"id\": [\"Protein\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CHI3L1\", \"cname\": \"CHI3L1\", \"id\": [\"Protein\", \"HGNC\", \"CHI3L1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TREM2\", \"cname\": \"TREM2\", \"id\": [\"Protein\", \"HGNC\", \"TREM2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TREM1\", \"cname\": \"TREM1\", \"id\": [\"Protein\", \"HGNC\", \"TREM1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Hypersensitivity\"}, \"source\": 10, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Diabetes Mellitus, Type 2\"}, \"source\": 10, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Asthma\"}, \"source\": 10, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Obesity\"}, \"source\": 10, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These leukotrienes are known to produce inflammatory responses in asthma and allergic reactions, to induce a reduction of tyrosine hydroxylase in brain, and are involved in the development of cardiac strokes, obesity and type 2 diabetes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"CNS Neurol Disord Drug Targets. 2014 Mar;13(2):197-202\", \"reference\": \"24059322\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Eicosanoids signaling subgraph\", \"MeSHDisease\": \"Stroke\"}, \"source\": 10, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011 Jan 5;6(1):e15816. doi: 10.1371/journal.pone.0015816.\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces A beta peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2011 Jan 5;6(1):e15816. doi: 10.1371/journal.pone.0015816.\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Neuroprotection subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Neuroprotectin D1 (NPD1) is a stereoselective mediator derived from the omega-3 essential fatty acid docosahexaenoic acid (DHA) with potent inflammatory resolving and neuroprotective bioactivity. NPD1 reduces Abeta42 peptide release from aging human brain cells and is severely depleted in Alzheimer's disease (AD) brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One2011\", \"reference\": \"21246057\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"Recent studies showed that melatonin, an indoleamine secreted by the pineal gland, may play an important/ role in aging and AD as an antioxidant and neuroprotector. Melatonin decreases during aging and patients with AD have a/ more profound reduction in this hormone.Melatonin efficiently protects neuronal cells from Abeta-mediated toxicity via/ antioxidant and anti-amyloid properties: it not only inhibits Abeta generation, but also arrests the formation of amyloid/ fibrils by a structure-dependent interaction with Abeta. Our recent studies have demonstrated that melatonin efficiently/ attenuates Alzheimer-like tau hyperphosphorylation. Although the exact mechanism is still not fully understood, a direct/ regulatory influence of melatonin on the activities of protein kinases and protein phosphatases is proposed. Additionally,/ melatonin also plays a role in protecting cholinergic neurons and in anti-inflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Pharmacol Sin. 2006 Jan;27(1):41-9.\", \"reference\": \"16364209\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 13, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 18, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 19, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 20, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Platelets are an important source of amyloid-ss (Ass) in the circulatory system and play an important pro-inflammatory role. Upon activation, they adhere to leukocytes and endothelial cells by means of adhesive proteins like P-selectin, platelet endothelial cell adhesion molecule-1 (PECAM) and intercellular adhesion molecule-1 and -2 (ICAM-1 and -2) and secrete inflammatory mediators (chemokines, interleukins). In addition, platelets contain important enzymes involved in inflammatory intermediary synthesis like phospholipase A(2) (PLA(2)) and cyclooxygenase-2 (COX-2), and recent reports demonstrated significant changes in platelet levels and activities in Alzheimer's disease. Thus, as platelets represent an important link between Ass deposition and inflammatory reactions especially at endothelial level, they can be considered a valuable cellular model to evaluate potential peripheral inflammatory biomarkers in Alzheimer's disease.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biogerontology. 2010 Oct;11(5):627-33. doi: 10.1007/s10522-010-9281-8. Epub 2010 May 9.\", \"reference\": \"20454929\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 21, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 22, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 23, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 24, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 25, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 26, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2003 Aug;86(4):907-16.\", \"reference\": \"12887689\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 27, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Experimental evidence suggests that cortical noradrenaline (NA) depletion due to degeneration of the locus/ ceruleus (LC) - a pathological hallmark of AD - plays a permissive role in the development of inflammation in AD. Our/ results indicate for the first time that PPARgamma expression can be modulated by the cAMP signalling pathway, and/ suggest that the anti-inflammatory effects of NA on brain cells may be partly mediated by increasing PPARgamma levels./ Conversely, decreased NA due to LC cell death in AD may reduce endogenous PPARgamma expression and therefore potentiate/ neuroinflammatory processes.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem. 2003 Aug;86(4):907-16.\", \"reference\": \"12887689\"}, \"annotations\": {\"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 27, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\"}, \"annotations\": {\"MeSHAnatomy\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 28, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"There was a strong correlation between YKL-40 and markers of neurodegeneration (total tau and p-tau). In the whole sample, we found a negative correlation between YKL-40 and CTh in AD vulnerable areas in Abeta42+ subjects but not in Abeta42 participants. Our results suggest that YKL-40 could track the inflammatory processes associated to tau-related neurodegeneration in the presence of the AD pathophysiological process.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2015 Jun;36(6):2018-23. doi: 10.1016/j.neurobiolaging.2015.03.001\", \"reference\": \"25865441\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 29, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Recently, a novel variant in the gene encoding the triggering receptor expressed on myeloid cells 2 (TREM2) has been identified that has refocused the spotlight back onto inflammation as a major contributing factor in AD. Variants in TREM2 triple one's risk of developing late-onset AD. TREM2 is expressed on microglial cells, the resident macrophages in the CNS, and functions to stimulate phagocytosis on one hand and to suppress cytokine production and inflammation on the other hand. \", \"citation\": {\"type\": \"PubMed\", \"name\": \"N Engl J Med. 2013 Jan 10;368(2):117-27. doi: 10.1056/NEJMoa1211851. Epub 2012 Nov 14.\", \"reference\": \"23150934\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 30, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"monocyte\"}, \"source\": 31, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"TREM-1 is an activating receptor on neutrophils and monocytes that plays an important role in the amplification of inflammation\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Recent Pat Antiinfect Drug Discov. 2009 Jan;4(1):51-6.\", \"reference\": \"19149696\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\", \"Cell\": \"neutrophil\"}, \"source\": 31, \"target\": 1, \"key\": 1}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "subgraph = pbt.mutation.get_upstream_causal_subgraph(graph, inflamation_node)\n",
    "\n",
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Iteration 2/2\n",
    "\n",
    "All of the nodes upstream of the nodes within the subgraph are added as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 266\n",
      "Number of edges: 1081\n",
      "Network density: 0.015335508582777699\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 4.06390977443609\n",
      "Average out-degree: 4.06390977443609\n"
     ]
    }
   ],
   "source": [
    "pbt.mutation.expand_upstream_causal_subgraph(graph, subgraph)\n",
    "pbt.summary.print_summary(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Post-Processing\n",
    "\n",
    "Inconsistient edges are thrown away, literature bias is removed, and nodes on the periphery of the subgraph that are not annotated with experimental data are also removed, since they cannot contribute to the propogation of scores."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.mutation.remove_inconsistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.mutation.collapse_consistent_edges(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_leaves(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "pbt.generation.remove_unweighted_sources(subgraph, label)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of nodes: 51\n",
      "Number of edges: 132\n",
      "Network density: 0.05176470588235294\n",
      "Number weakly connected components: 1\n",
      "Average in-degree: 2.588235294117647\n",
      "Average out-degree: 2.588235294117647\n"
     ]
    },
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"mkudpqcofahtyrge\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"cname\": \"SCARF1\", \"id\": [\"Protein\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"cname\": \"SCARB1\", \"id\": [\"Protein\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"IRAK1\", \"cname\": \"IRAK1\", \"id\": [\"RNA\", \"HGNC\", \"IRAK1\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"IRAK2\", \"cname\": \"IRAK2\", \"id\": [\"RNA\", \"HGNC\", \"IRAK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR2\", \"cname\": \"TLR2\", \"id\": [\"Protein\", \"HGNC\", \"TLR2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}, {\"function\": \"Abundance\", \"namespace\": \"MESHCS\", \"name\": \"Neurofibrillary Tangles\", \"cname\": \"Neurofibrillary Tangles\", \"id\": [\"Abundance\", \"MESHCS\", \"Neurofibrillary Tangles\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"IL12B\", \"cname\": \"IL12B\", \"id\": [\"Protein\", \"HGNC\", \"IL12B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"IL23A\", \"cname\": \"IL23A\", \"id\": [\"Protein\", \"HGNC\", \"IL23A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CD14\", \"cname\": \"CD14\", \"id\": [\"Protein\", \"HGNC\", \"CD14\"]}, {\"function\": \"Abundance\", \"namespace\": \"BRCO\", \"name\": \"Astrocyte\", \"cname\": \"Astrocyte\", \"id\": [\"Abundance\", \"BRCO\", \"Astrocyte\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CX3CR1\", \"cname\": \"CX3CR1\", \"id\": [\"Protein\", \"HGNC\", \"CX3CR1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"weight\": 0.32860467, \"cname\": \"g(HGNC:GSK3B, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 9}], \"weight\": 0.32860467, \"cname\": \"g(HGNC:GSK3B, pmod(Ph, Ser, 9))\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 9]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Glyco\"}}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Glyco))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Glyco\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"IRS2\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\"}], \"weight\": 0.20342729, \"cname\": \"g(HGNC:IRS2, pmod(Ph, Thr))\", \"id\": [\"Protein\", \"HGNC\", \"IRS2\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.275341G>C\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.275341G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.275341G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.717G>C\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(c.717G>C))\", \"id\": [\"Gene\", \"HGNC\", \"APP\", [\"hgvs\", \"c.717G>C\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Asn141Ile\"}], \"weight\": -0.02429789, \"cname\": \"g(HGNC:PSEN2, var(p.Asn141Ile))\", \"id\": [\"Protein\", \"HGNC\", \"PSEN2\", [\"hgvs\", \"p.Asn141Ile\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN1\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Glu280Ala\"}], \"weight\": 0.00266298, \"cname\": \"g(HGNC:PSEN1, var(p.Glu280Ala))\", \"id\": [\"Protein\", \"HGNC\", \"PSEN1\", [\"hgvs\", \"p.Glu280Ala\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APOE\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Arg132Ser\"}], \"weight\": 0.04406715, \"cname\": \"g(HGNC:APOE, var(p.Arg132Ser))\", \"id\": [\"Protein\", \"HGNC\", \"APOE\", [\"hgvs\", \"p.Arg132Ser\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Asn10Tyr\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(p.Asn10Tyr))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"hgvs\", \"p.Asn10Tyr\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CTSD\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"c.224C>T\"}], \"weight\": 0.88300839, \"cname\": \"g(HGNC:CTSD, var(c.224C>T))\", \"id\": [\"Gene\", \"HGNC\", \"CTSD\", [\"hgvs\", \"c.224C>T\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ub\"}}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, pmod(Ub))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"pmod\", [\"bel\", \"Ub\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 668}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, pmod(Ph, Thr, 668))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 668]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 262}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Ph, Ser, 262))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 262]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Thr\", \"pos\": 231}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Ph, Thr, 231))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Thr\", 231]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 396}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Ph, Ser, 396))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 396]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 404}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Ph, Ser, 404))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 404]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"MAPT\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"weight\": 0.02358135, \"cname\": \"g(HGNC:MAPT, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"MAPT\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"CALHM1\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Pro86Leu\"}], \"weight\": -0.04955734, \"cname\": \"g(HGNC:CALHM1, var(p.Pro86Leu))\", \"id\": [\"Protein\", \"HGNC\", \"CALHM1\", [\"hgvs\", \"p.Pro86Leu\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"pmod\", [\"bel\", \"Ph\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APOE\", \"weight\": 0.04406715, \"cname\": \"APOE\", \"id\": [\"Gene\", \"HGNC\", \"APOE\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"weight\": -0.13701466, \"cname\": \"APP\", \"id\": [\"Gene\", \"HGNC\", \"APP\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN1\", \"weight\": 0.00266298, \"cname\": \"PSEN1\", \"id\": [\"Gene\", \"HGNC\", \"PSEN1\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PSEN2\", \"weight\": -0.02429789, \"cname\": \"PSEN2\", \"id\": [\"Gene\", \"HGNC\", \"PSEN2\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"PPARG\", \"weight\": -0.06808657, \"cname\": \"PPARG\", \"id\": [\"Gene\", \"HGNC\", \"PPARG\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCB1\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Cys1236Thr\"}], \"weight\": -0.32392887, \"cname\": \"g(HGNC:ABCB1, var(p.Cys1236Thr))\", \"id\": [\"Protein\", \"HGNC\", \"ABCB1\", [\"hgvs\", \"p.Cys1236Thr\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCB1\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Gly2677Thr\"}], \"weight\": -0.32392887, \"cname\": \"g(HGNC:ABCB1, var(p.Gly2677Thr))\", \"id\": [\"Protein\", \"HGNC\", \"ABCB1\", [\"hgvs\", \"p.Gly2677Thr\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"ABCB1\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Cys3435Thr\"}], \"weight\": -0.32392887, \"cname\": \"g(HGNC:ABCB1, var(p.Cys3435Thr))\", \"id\": [\"Protein\", \"HGNC\", \"ABCB1\", [\"hgvs\", \"p.Cys3435Thr\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Lys595Met\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(p.Lys595Met))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"hgvs\", \"p.Lys595Met\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"hgvs\", \"identifier\": \"p.Asn596Leu\"}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, var(p.Asn596Leu))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"hgvs\", \"p.Asn596Leu\"]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"H3F3A\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ac\"}}], \"weight\": 1.04528382, \"cname\": \"g(HGNC:H3F3A, pmod(Ac))\", \"id\": [\"Protein\", \"HGNC\", \"H3F3A\", [\"pmod\", [\"bel\", \"Ac\"]]]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}}], \"weight\": 0.47307381, \"cname\": \"g(HGNC:EIF2AK2, pmod(Ph))\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\", [\"pmod\", [\"bel\", \"Ph\"]]]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 1, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 4, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"miRNA subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Differential regulation of interleukin-1 receptor-associated kinase-1 (IRAK-1) and IRAK-2 by microRNA-146a and NF-kappaB in stressed human astroglial cells and in Alzheimer disease.In monocytes, increased expression of an NF-B-regulated miRNA-146a down-regulates expression of the interleukin-1 receptor-associated kinase-1 (IRAK-1), an essential component of Toll-like/IL-1 receptor signaling. Here we extend those observations to the hippocampus and neocortex of Alzheimer disease (AD) brain and to stressed human astroglial (HAG) cells in primary culture. In 66 control and AD samples we note a significant up-regulation of miRNA-146a coupled to down-regulation of IRAK-1 and a compensatory up-regulation of IRAK-2. Using miRNA-146a-, IRAK-1-, or IRAK-2 promoter-luciferase reporter constructs, we observe decreases in IRAK-1 and increases in miRNA-146a and IRAK-2 expression in interleukin-1 (IL-1) and amyloid--42 (Abeta42) peptide-stressed HAG cells. NF-B-mediated transcriptional control of human IRAK-2 was localized to between -119 and +12 bp of the immediate IRAK-2 promoter. The NF-B inhibitors curcumin, pyrrolidine dithiocarbamate or CAY10512 abrogated both IRAK-2 and miRNA-146a expression, whereas IRAK-1 was up-regulated. Incubation of a protected antisense miRNA-146a was found to inhibit miRNA-146a and restore IRAK-1, whereas IRAK-2 remained unaffected. These data suggest a significantly independent regulation of IRAK-1 and IRAK-2 in AD and in IL-1+Abeta42 peptide-stressed HAG cells and that an inducible, NF-B-sensitive, miRNA-146a-mediated down-regulation of IRAK-1 coupled to an NF-B-induced up-regulation of IRAK-2 expression drives an extensively sustained inflammatory response. The interactive signaling of NF-B and miRNA-146a further illustrate interplay between inducible transcription factors and pro-inflammatory miRNAs that regulate brain IRAK expression. The combinatorial use of NF-B inhibitors with miRNA-146a or antisense miRNA-146a may have potential as a bi-pronged therapeutic strategy directed against IRAK-2-driven pathogenic signaling.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2010 Dec 10;285(50):38951-60. doi: 10.1074/jbc.M110.178848. Epub 2010 Oct 11.\", \"reference\": \"20937840\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 9, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 9, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 9, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 9, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 9, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 9, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 9, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 9, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 9, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 9, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 9, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 9, \"target\": 11, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 9, \"target\": 11, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 9, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 10, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 11, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 12, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 13, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 14, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 15, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"In the present study, Abeta(1-42) synergistically elevated the expression of IL-12 and IL-23 triggered by inflammatory activation of microglia, and the peroxisome proliferator-activated receptor (PPAR)-gamma agonist 15-deoxy-Delta(12,14)-PGJ(2) (15d-PGJ(2)) effectively blocked the elevation of these proinflammatory cytokines. Furthermore, 15d-PGJ(2) suppressed the Abeta-related synergistic induction of CD14, MyD88, and Toll-like receptor 2, molecules that play critical roles in neuroinflammatory conditions. Collectively, these studies suggest that PPAR-gamma agonists may be effective in modulating the development of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PPAR Res. 2008;2008:349185. doi: 10.1155/2008/349185.\", \"reference\": \"18615183\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 16, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Accumulating data indicate that astrocytes play an important role in the neuroinflammation related to the/ pathogenesis of AD. It has been shown that microglia and astrocytes are activated in AD brain and amyloid-beta (Abeta)/ can increase the expression of cyclooxygenase 2 (COX-2), interleukin-1, and interleukin-6.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;20(4):1189-99. doi: 10.3233/JAD-2010-091336.\", \"reference\": \"20413894\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 17, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Chemokine signaling subgraph\"}, \"source\": 17, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These findings suggest that the absence of CCR5 increases expression of CCR2, which leads to the activation of astrocytes causing Abeta deposit, and thereby impairs memory function. These results suggest that CCR5 may be a critical suppressor of the development and progression of AD pathology.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Learn Mem. 2009 Oct;92(3):356-63. Epub 2009 Apr 24.\", \"reference\": \"19394434\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 17, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 17, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"-Amyloid (Abeta) plays a central role in Alzheimer's disease (AD) pathogenesis. Neurons are major sources of Abeta in the brain. However, astrocytes outnumber neurons by at least five-fold. Thus, even a small level of astrocytic Abeta production could make a significant contribution to Abeta burden in AD. Moreover, activated astrocytes may increase Abeta generation. -Site APP cleaving enzyme 1 (BACE1) cleavage of amyloid precursor protein (APP) initiates Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neuroinflammation. 2011 Nov 2;8:150.\", \"reference\": \"22047170\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 17, \"target\": 9, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Microglia produced only IL-12p40 and CXCL10, whereas astroglia produced these cytokines plus CCL2, CCL3, CCL5, CXCL1, G-CSF, IL-1beta, IL-6, IL-12p70, and IL-13.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Virol. 2009 Nov;83(21):11244-53. Epub 2009 Aug 26\", \"reference\": \"19710140\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Interleukin signaling subgraph\"}, \"source\": 17, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\"}, \"annotations\": {\"MeSHAnatomy\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 18, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Activation of the catalytic subunit of the Pl3-kinase re sults in phosphorylation of phosphatidylinositide-diphos phate (PI4,sP) to generate phosphatidylinositide-triphosphate (PI3,4.sP). This leads to activation of several downstream tar gets, such as phosphoinositide-dependent Rrotein kinase (PDK)-1, rotein k nase B (PKB, AKT), p705 kinase, glycogen synthase kmase(GSK)-3and BAD a proapoptotic member ?f thBcl-2 family. Phosphorylation of GSK-3P and BAD mact1vates these proteins and thereby inhibits further sia nal ing leading to apoptosis. Activated AKT phosphorylatethe forkhed transcription factor Foxo, which triggers its nuclear excl usiOn and thereby regulates transcription of aenes in volved in development, growth, stress resistance, :poptosis, metabolism an d ag ing [29-31]. Several Foxo target genes might play an importarJt role for the patho<> enetics of AD (re.viein [32]): Catalase a nd MnSOD (m:nganese super ox rd dr.smutase) re .crucial in prom oting longevity by possi?ly ctmg. as ant10 rdants due to redu ced insulin-like signal m g m varrous spec res such as drosophila [33] and C. elegans\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 19, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Tau is a microtubule-binding protein that ligates tubulin and accounts for the stability of microtubules. If hyperphos phorylated, tau aggregates and interferes with intraneu ronal metabolism and transport lead ing to neurodegeneration [I 02]. Tau phosphorylation state is regulated by site-specific dephosphorylation through certain phosphatases and by kinases phosphorylating tau protein at specific sites. Protein phosphatase 2A (PP2A) is the major phosphatase with 70% tau phosphatase activity in human brains [I 03]. This implies a protective role of PP2A in neurodegeneration which is consistent with the finding that PP2A activity is reduced in AD brains [104, 105]. The phosphorylation of tau is mainly promoted by GSK-3and cyclin-dependent kinase 5 (Cdk5). Besides these kinases, activated c-Jun N-terminal kinases (JNK) and ERK-1 /-2 signaling lead to an increase in tau phosphorylation and th erefore might be of importance in AD pathogenesis. GSK-3is a serine/threonine kinase, modu lated by insulin/IGF-1 signaling. When the IRIIGF-IR path way is activated, GSK-3is phosphorylated by AKT at Scr 9 leading to its inactivation [I 06-109]. However, PP2A dephosphorylates GSK-3(review in [II 0]), which then phosphorylates tau at several sites leading to an equilibrium of phosphorylation and dephosphorylation of tau (Overview : see Fig. 1). Thus, impaired IRIIGF-1 R signaling might lead to hyperphosphory lation of tau protein and an in creased for mation of neurofibrillary tangles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Akt subgraph\"}, \"source\": 20, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Tau is a microtubule-binding protein that ligates tubulin and accounts for the stability of microtubules. If hyperphos phorylated, tau aggregates and interferes with intraneu ronal metabolism and transport lead ing to neurodegeneration [I 02]. Tau phosphorylation state is regulated by site-specific dephosphorylation through certain phosphatases and by kinases phosphorylating tau protein at specific sites. Protein phosphatase 2A (PP2A) is the major phosphatase with 70% tau phosphatase activity in human brains [I 03]. This implies a protective role of PP2A in neurodegeneration which is consistent with the finding that PP2A activity is reduced in AD brains [104, 105]. The phosphorylation of tau is mainly promoted by GSK-3and cyclin-dependent kinase 5 (Cdk5). Besides these kinases, activated c-Jun N-terminal kinases (JNK) and ERK-1 /-2 signaling lead to an increase in tau phosphorylation and th erefore might be of importance in AD pathogenesis. GSK-3is a serine/threonine kinase, modu lated by insulin/IGF-1 signaling. When the IRIIGF-IR path way is activated, GSK-3is phosphorylated by AKT at Scr 9 leading to its inactivation [I 06-109]. However, PP2A dephosphorylates GSK-3(review in [II 0]), which then phosphorylates tau at several sites leading to an equilibrium of phosphorylation and dephosphorylation of tau (Overview : see Fig. 1). Thus, impaired IRIIGF-1 R signaling might lead to hyperphosphory lation of tau protein and an in creased for mation of neurofibrillary tangles.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Alzheimer Res. 2009 Jun;6(3):213-23.\", \"reference\": \"19519303\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 20, \"target\": 4, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Estrogen subgraph\"}, \"source\": 20, \"target\": 4, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Simultaneously, it increased the level of Ser9-phosphorylated (inactive) GSK-3beta\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Journal of Neural Transmission2008\", \"reference\": \"18217188\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 20, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that aberrant glycosylation of tau in AD might be involved in neurofibrillary degeneration by promoting abnormal hyperphosphorylation by cdk5 and GSK-3beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 21, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"These data suggest that aberrant glycosylation of tau in AD might be involved in neurofibrillary degeneration by promoting abnormal hyperphosphorylation by cdk5 and GSK-3beta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS Lett2002\", \"reference\": \"12387894\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 21, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane -subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the -subunit, and causes autophosphorylation of the -subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3 and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3 signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta monomers may bind to IGF-1/insulin receptors, as already shown for ABeta oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Regulation of cytoskeleton subgraph\"}, \"source\": 22, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Insulin like growth factor-I (IGF-I) and insulin signaling: The underlying mechanism of neuronal survival with ABeta is emerging. Neuronal synapses and astrocytes of memory-processing brain regions possess insulin receptors (IRs) [99] which when activated by insulin facilitate synaptic plasticity in normal brain [100]. IR and Insulin-like growth factor I (IGF-I) receptors consist of a-subunits and transmembrane -subunits. Binding of insulin or IGF-I to the a-subunit increases the intrinsic tyrosine kinase activity of the -subunit, and causes autophosphorylation of the -subunit, thus triggering tyrosine phosphorylation of insulin receptor substrate (IRS)-1 and IRS-2, as well as Shc [101] as an important pathway of cell survival. To protect against ABeta toxicity, the tyrosine-phosphorylated sites create binding sites for various signal-transducing molecules containing Src homology-2 domain, such as phosphoinositide 3-kinase (PI3K) and growth factor receptor-bound protein 2 (Grb2), thus activating PI3K/phosphoinostide-dependent kinase 1 (PDK1)/Akt (protein kinase B)/glycogen synthase kinase (GSK)-3a/-3 and Ras/Raf-1/mitogen-activated protein kinase/extracellular-signal regulated kinase (MEK/ERK) signaling pathways [101]. In normal brain, IGF-I and insulin promote glucose utilization, energy metabolism, and neuronal survival [102], largely through PI3K/Akt/GSK-3 signaling [103, 104].Consistent with positive effects of insulin on synaptic plasticity [105], acute insulin treatment improved memory in rats [106] and also in normaladults and AD patients [107] by strongly activating ERK and Akt and blocking c-Jun N-terminal kinase (JNK) activation in a PI3K-dependent manner [108]. The mechanism involves many steps beginning with ABeta activation of IGF-1/insulin receptors by locally produced IGF-1 or, possibly, ABeta monomers may bind to IGF-1/insulin receptors, as already shown for ABeta oligomers \", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis. 2010;22(3):741-63\", \"reference\": \"20847424\"}, \"subject\": {\"modifier\": \"Activity\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Insulin signal transduction\"}, \"source\": 22, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 23, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 24, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 25, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 25, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 26, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 27, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The main factors responsible for Abeta formation are mutation of APP or PS1 and PS2 genes or ApoE gene. All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specifically the more amyloidogenic form, Abeta42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Clin Neurosci. 2004 Jun;11(5):456-67\", \"reference\": \"15177383\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 27, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"NO contributes to cell signaling by inducing posttranslational protein modifications. Under pathological conditions there is a shift from the signal transducing actions to the formation of protein tyrosine nitration by secondary products like peroxynitrite and nitrogen dioxide. We identified amyloid Abeta (Abeta) as an NO target, which is nitrated at tyrosine 10 (3NTyr(10)-Abeta). Nitration of Abeta accelerated its aggregation and was detected in the core of Abeta plaques of APP/PS1 mice and AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron. 2011 Sep 8;71(5):833-44.\", \"reference\": \"21903077\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 28, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"NO contributes to cell signaling by inducing posttranslational protein modifications. Under pathological conditions there is a shift from the signal transducing actions to the formation of protein tyrosine nitration by secondary products like peroxynitrite and nitrogen dioxide. We identified amyloid Abeta (Abeta) as an NO target, which is nitrated at tyrosine 10 (3NTyr(10)-Abeta). Nitration of Abeta accelerated its aggregation and was detected in the core of Abeta plaques of APP/PS1 mice and AD brains.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuron. 2011 Sep 8;71(5):833-44.\", \"reference\": \"21903077\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Nitric oxide subgraph\"}, \"source\": 28, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (Abeta). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurol Scand. 2011 Jun;123(6):419-23. doi: 10.1111/j.1600-0404.2010.01400.x.\", \"reference\": \"20597865\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Endosomal lysosomal subgraph\"}, \"source\": 29, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Cathepsin D, the most abundant lysosomal and endosomal aspartyl protease, shows beta and gamma secretase activity in vitro by cleaving the amyloid precursor protein (APP) into amyloid beta protein (Abeta). Polymorphism at position 224, C224T, on exon 2 of cathepsin D gene (CTSD) has been associated with an increased risk for Alzheimer's disease (AD) \", \"citation\": {\"type\": \"PubMed\", \"name\": \"Acta Neurol Scand. 2011 Jun;123(6):419-23. doi: 10.1111/j.1600-0404.2010.01400.x.\", \"reference\": \"20597865\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 29, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Endoplasmic reticulum (ER)-associated degradation (ERAD) is a protective mechanism against ER stress in which unfolded proteins accumulated in the ER are selectively transported to the cytosol for degradation by the ubiquitin-proteasome system. We cloned the novel ubiquitin ligase HRD1, which is involved in ERAD, and showed that HRD1 promoted amyloid precursor protein (APP) ubiquitination and degradation, resulting in decreased generation of amyloid Abeta (Abeta). In addition, suppression of HRD1 expression caused APP accumulation and promoted Abeta generation associated with ER stress and apoptosis. Interestingly, HRD1 levels were significantly decreased in the cerebral cortex of patients with Alzheimer's disease (AD), and the brains of these patients experienced ER stress. Our recent study revealed that this decrease in HRD1 was due to its insolubilization; however, controversy persists about whether the decrease in HRD1 protein promotes Abeta generation or whether Abeta neurotoxicity causes the decrease in HRD1 protein levels. Here, we review current findings on the mechanism of HRD1 protein loss in the AD brain and the involvement of HRD1 in the pathogenesis of AD. Furthermore, we propose that HRD1 may be a target for novel AD therapeutics.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2012;118(3):325-30. Epub 2012 Mar 2.\", \"reference\": \"22382662\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Endoplasmic reticulum (ER)-associated degradation (ERAD) is a protective mechanism against ER stress in which unfolded proteins accumulated in the ER are selectively transported to the cytosol for degradation by the ubiquitin-proteasome system. We cloned the novel ubiquitin ligase HRD1, which is involved in ERAD, and showed that HRD1 promoted amyloid precursor protein (APP) ubiquitination and degradation, resulting in decreased generation of amyloid Abeta (Abeta). In addition, suppression of HRD1 expression caused APP accumulation and promoted Abeta generation associated with ER stress and apoptosis. Interestingly, HRD1 levels were significantly decreased in the cerebral cortex of patients with Alzheimer's disease (AD), and the brains of these patients experienced ER stress. Our recent study revealed that this decrease in HRD1 was due to its insolubilization; however, controversy persists about whether the decrease in HRD1 protein promotes Abeta generation or whether Abeta neurotoxicity causes the decrease in HRD1 protein levels. Here, we review current findings on the mechanism of HRD1 protein loss in the AD brain and the involvement of HRD1 in the pathogenesis of AD. Furthermore, we propose that HRD1 may be a target for novel AD therapeutics.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2012;118(3):325-30. Epub 2012 Mar 2.\", \"reference\": \"22382662\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Tumor necrosis factor subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Endoplasmic reticulum (ER)-associated degradation (ERAD) is a protective mechanism against ER stress in which unfolded proteins accumulated in the ER are selectively transported to the cytosol for degradation by the ubiquitin-proteasome system. We cloned the novel ubiquitin ligase HRD1, which is involved in ERAD, and showed that HRD1 promoted amyloid precursor protein (APP) ubiquitination and degradation, resulting in decreased generation of amyloid Abeta (Abeta). In addition, suppression of HRD1 expression caused APP accumulation and promoted Abeta generation associated with ER stress and apoptosis. Interestingly, HRD1 levels were significantly decreased in the cerebral cortex of patients with Alzheimer's disease (AD), and the brains of these patients experienced ER stress. Our recent study revealed that this decrease in HRD1 was due to its insolubilization; however, controversy persists about whether the decrease in HRD1 protein promotes Abeta generation or whether Abeta neurotoxicity causes the decrease in HRD1 protein levels. Here, we review current findings on the mechanism of HRD1 protein loss in the AD brain and the involvement of HRD1 in the pathogenesis of AD. Furthermore, we propose that HRD1 may be a target for novel AD therapeutics.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2012;118(3):325-30. Epub 2012 Mar 2.\", \"reference\": \"22382662\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Unfolded protein response subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"A possible model for involvement of HRD1 in Alzheimer's disease (AD). Suppression of AD: Upregulation of HRD1 promotes the ubiquitination and degradation of APP in the ERAD pathway, resulting in decreased Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2012;118(3):325-30. Epub 2012 Mar 2.\", \"reference\": \"22382662\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"A possible model for involvement of HRD1 in Alzheimer's disease (AD). Suppression of AD: Upregulation of HRD1 promotes the ubiquitination and degradation of APP in the ERAD pathway, resulting in decreased Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Pharmacol Sci. 2012;118(3):325-30. Epub 2012 Mar 2.\", \"reference\": \"22382662\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Unfolded protein response subgraph\"}, \"source\": 30, \"target\": 9, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"Background and Objective: Could a normal - but persistent - stress response to impeded axonal transport lead to late-onset Alzheimer's disease (AD)? Our results offer an affirmative answer, suggesting a mechanism for the abnormal production of amyloid-Abeta (Abeta), triggered by the slowed axonal transport at old age. We hypothesize that Abeta precursor protein (APP) is a sensor at the endoplasmic reticulum (ER) that detects, and signals to the nucleus, abnormalities in axonal transport. When persistently activated, due to chronically slowed-down transport, this signaling pathway leads to accumulation of Abeta within the ER. Methods and Results: We tested this hypothesis with the neuronal cell line CAD. We show that, normally, a fraction of APP is transported into neurites by recruiting kinesin-1 via the adaptor protein, Fe65. Under conditions that block kinesin-1-dependent transport, APP, Fe65 and kinesin-1 accumulate in the soma, and form a complex at the ER. This complex recruits active c-Jun N-terminal kinase (JNK), which phosphorylates APP at Thr(668). This phosphorylation increases the cleavage of APP by the amyloidogenic pathway, which generates Abeta within the ER lumen, and releases Fe65 into the cytoplasm. Part of the released Fe65 translocates into the nucleus, likely to initiate a gene transcription response to arrested transport. Prolonged arrest of kinesin-1-dependent transport could thus lead to accumulation and oligomerization of Abeta in the ER. Conclusion: These results support a model where the APP:Fe65 complex is a sensor at the ER for detecting the increased level of kinesin-1 caused by halted transport, which signals to the nucleus, while concomitantly generating an oligomerization-prone pool of Abeta in the ER. Our hypothesis could thus explain a pathogenic mechanism in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurodegener Dis. 2012;10(1-4):60-3. Epub 2011 Dec 7.\", \"reference\": \"22156573\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14557249\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Together, these results suggest that T668 phosphorylation may facilitate the BACE1 cleavage of APP to increase Abeta generation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol2003\", \"reference\": \"14557249\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 31, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 32, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 32, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 32, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 33, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 33, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 33, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 34, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 34, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 34, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Cholesterol metabolism subgraph\"}, \"source\": 35, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Akt subgraph\"}, \"source\": 35, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"AMP-activated protein kinase (AMPK), a master regulator of cellular energy homeostasis and a central player in glucose and lipid metabolism, is potentially implicated in the pathogenesis of Alzheimer's disease (AD). AMPK activity decreases in AD brain, indicating decreased mitochondrial biogenesis and function. Emerging evidence demonstrates that AMPK activation is a potential target for improving perturbed brain energy metabolism that is involved in the pathogenesis of AD. The roles of AMPK in the pathogenesis of AD include Abeta-amyloid protein (Abeta) generation and tau phosphorylation. In particular, AMPK may regulate Abeta generation through modulating neuronal cholesterol and sphingomyelin levels and through regulating APP distribution in the lipid rafts. AMPK is activated by phosphorylation of Thr-172 by LKB1 complex in response to increase in the AMP/ATP ratio and by calmodulin-dependent protein kinase kinase-beta in response to elevated Ca(2+) levels, which contributes to regulating Abeta generation. AMPK is a physiological tau kinase and can increase the phosphorylation of tau at Ser-262. AMPK can also directly phosphorylate tau at Thr-231 and Ser-396/404. Furthermore, AMPK activation decreases mTOR signaling activity to facilitate autophagy and promotes lysosomal degradation of Abeta. However, AMPK activation has non-neuroprotective property and may lead to detrimental outcomes, including Abeta generation and tau phosphorylation. Therefore, it is still unclear whether AMPK could serve a potential therapeutic target for AD, and hence, further studies will be needed to clarify the role of AMPK in AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neuromolecular Med. 2012 Mar;14(1):1-14. Epub 2012 Feb 26.\", \"reference\": \"22367557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Sphingolipid metabolic subgraph\"}, \"source\": 35, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"During the course of AD tau becomes hyperphosphorylated and dissociates from microtubules which then depolymerize. The hyperphosphorylated tau self-aggregates and accumulates in the cell body where it forms paired-helical filaments (neurofibrillary tangles). As a consequence of accumulation of Abeta at synapses, Ca2+ regulation is impaired\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Trends Neurosci. 2008 Sep;31(9):454-63. Epub 2008 Jul 31\", \"reference\": \"18675468\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 36, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"During the course of AD tau becomes hyperphosphorylated and dissociates from microtubules which then depolymerize. The hyperphosphorylated tau self-aggregates and accumulates in the cell body where it forms paired-helical filaments (neurofibrillary tangles). As a consequence of accumulation of Abeta at synapses, Ca2+ regulation is impaired\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Trends Neurosci. 2008 Sep;31(9):454-63. Epub 2008 Jul 31\", \"reference\": \"18675468\"}, \"annotations\": {\"MeSHAnatomy\": \"Neurons\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 36, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates tau at 11 sites, including Thr212; primes tau phosphorylation by GSK-3; promotes tau aggregation into NFTs and the several-fold increase in the number of DYRK1A-positive NFTs. Phosphorylation of ASF by nuclear DYRK1A increases the level of 3R tau, leading to an imbalance in the 3R/4R-tau ratio and triggering neurofibrillary degeneration with a several-fold increase of 3R-taupositive NFTs. Both cytoplasmic and nuclear pathways contribute to neurofibrillary degeneration, loss of neuron function, and neuronal death.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Tau protein subgraph\"}, \"source\": 36, \"target\": 13, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"DYRK1A phosphorylates tau at 11 sites, including Thr212; primes tau phosphorylation by GSK-3; promotes tau aggregation into NFTs and the several-fold increase in the number of DYRK1A-positive NFTs. Phosphorylation of ASF by nuclear DYRK1A increases the level of 3R tau, leading to an imbalance in the 3R/4R-tau ratio and triggering neurofibrillary degeneration with a several-fold increase of 3R-taupositive NFTs. Both cytoplasmic and nuclear pathways contribute to neurofibrillary degeneration, loss of neuron function, and neuronal death.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"FEBS J. 2011 Jan;278(2):236-45. doi: 10.1111/j.1742-4658.2010.07955.x.Epub 2010 Dec 13.\", \"reference\": \"21156028\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 36, \"target\": 13, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is an age-related form of dementia due to a progressive loss of neuronal functioning, also characterized by the deposition of extracellular amyloid throughout the brain (amyloid plaques) and a significant decrease in brain mass. An overactivity of the cyclin-dependent protein kinase CDK5 in the brain has been implicated in neuronal degeneration and the formation of neurofibrillary tangles due to hyperphosphorylation of essential neuronal proteins such as MAP and Tau (2). CDK5 is regulated by the neuron-specific and cyclin-related protein p35 (also known as p35Ncdk5a). During the development of Alzheimer's disease, p35 is processed to a smaller protein, p25, which is a constitutive activator of CDK5. The generation of p25 disrupts CDK5 localization and substrate preferences (2). In addition, the expression of p25 in neurons results in a collapse of microtubules, neurite retraction, and apoptosis (2).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Endocrinology. 2004 Jun;145(6):3023-31. Epub 2004 Feb 19.\", \"reference\": \"14976144\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 36, \"target\": 13, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is an age-related form of dementia due to a progressive loss of neuronal functioning, also characterized by the deposition of extracellular amyloid throughout the brain (amyloid plaques) and a significant decrease in brain mass. An overactivity of the cyclin-dependent protein kinase CDK5 in the brain has been implicated in neuronal degeneration and the formation of neurofibrillary tangles due to hyperphosphorylation of essential neuronal proteins such as MAP and Tau (2). CDK5 is regulated by the neuron-specific and cyclin-related protein p35 (also known as p35Ncdk5a). During the development of Alzheimer's disease, p35 is processed to a smaller protein, p25, which is a constitutive activator of CDK5. The generation of p25 disrupts CDK5 localization and substrate preferences (2). In addition, the expression of p25 in neurons results in a collapse of microtubules, neurite retraction, and apoptosis (2).\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Endocrinology. 2004 Jun;145(6):3023-31. Epub 2004 Feb 19.\", \"reference\": \"14976144\"}, \"annotations\": {\"Subgraph\": \"Cyclin-CDK subgraph\"}, \"source\": 36, \"target\": 13, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer disease, the most common tauopathy, is characterized by neurofibrillary tangles that are mainly composed of abnormally phosphorylated Tau. Tau phosphorylation occurs mainly at proline-directed Ser/Thr sites, which are targeted by protein kinases such as GSK3 and Cdk5. We reported previously that dephosphorylation of Tau at Cdk5-mediated sites was enhanced by Pin1, a peptidyl-prolyl isomerase that stimulates dephosphorylation at proline-directed sites by protein phosphatase 2A. Pin1 deficiency is suggested to cause Tau hyperphosphorylation in Alzheimer disease. Up to the present, Pin1 binding was only shown for two Tau phosphorylation sites (Thr-212 and Thr-231) despite the presence of many more hyperphosphorylated sites. Here, we analyzed the interaction of Pin1 with Tau phosphorylated by Cdk5-p25 using a GST pulldown assay and Biacore approach. We found that Pin1 binds and stimulates dephosphorylation of Tau at all Cdk5-mediated sites (Ser-202, Thr-205, Ser-235, and Ser-404). Furthermore, FTDP-17 mutant Tau (P301L or R406W) showed slightly weaker Pin1 binding than non-mutated Tau, suggesting that FTDP-17 mutations induce hyperphosphorylation by reducing the interaction between Pin1 and Tau. Together, these results indicate that Pin1 is generally involved in the regulation of Tau hyperphosphorylation and hence the etiology of tauopathies.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Biol Chem. 2013 Mar 15;288(11):7968-77. doi: 10.1074/jbc.M112.433326. Epub 2013 Jan 28.\", \"reference\": \"23362255\"}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 36, \"target\": 13, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"it has been demonstrated that micromolar S100B concentrations stimulate c-Jun N-terminal kinase (JNK) phosphorylation through the receptor for advanced glycation ending products, and subsequently activate nuclear AP-1/cJun transcription, in cultured human neural stem cells. In addition, as revealed by Western blot, small interfering RNA and immunofluorescence analysis, S100B-induced JNK activation increased expression of Dickopff-1 that, in turn, promoted glycogen synthase kinase 3beta phosphorylation and beta-catenin degradation, causing canonical Wnt signaling pathway disruption and tau protein hyperphosphorylation. These findings propose a previously unrecognized link between S100B and tau hyperphosphorylation, suggesting S100B can contribute to NFT formation in AD and in all other conditions in which neuroinflammation may have a crucial role.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Mol Med. 2008 Jun;12(3):914-27. doi: 10.1111/j.1582-4934.2008.00159.x.\", \"reference\": \"18494933\"}, \"annotations\": {}, \"source\": 36, \"target\": 13, \"key\": 5}, {\"relation\": \"increases\", \"evidence\": \"Further investigation demonstrated the functional significance of the rs2986017 SNP by showing that the P86L polymorphism promotes Abeta accumulation via a loss of CALHM1 control on calcium permeability and cytosolic calcium levels\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurodegener. 2009 May 6;4:20.\", \"reference\": \"19419557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 37, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Further investigation demonstrated the functional significance of the rs2986017 SNP by showing that the P86L polymorphism promotes Abeta accumulation via a loss of CALHM1 control on calcium permeability and cytosolic calcium levels\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Mol Neurodegener. 2009 May 6;4:20.\", \"reference\": \"19419557\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Calcium-dependent signal transduction\"}, \"source\": 37, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Dyrk1A activity is involved in cell proliferation and differentiation (Dierssen and de Lagran, 2006). For example, while Ras-dependent signaling is required for promoting or maintaining neuronal differentiation, Dyrk1A modulates ERK activation by interacting with Ras, B-Raf, and MEK1 and by facilitating the formation of a Ras/B-Raf/MEK1 multi-protein complex (Kelly and Rahmani, 2005). Recently, many studies implicated a potential role of Dyrk1A during cell death. For example, Dyrk1A caused the formation of abnormal protein aggregates through the phosphorylation of a-synuclein, APP, and tau. Furthermore, these cells show a marked increase of apoptotic cell death (Park et al., 2007), indicating that the overexpression of Dyrk1A induces cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurobiol. 2011 Mar;20(1):35-44. Epub 2011 Mar 31.\", \"reference\": \"22110360\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 38, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Dyrk1A activity is involved in cell proliferation and differentiation (Dierssen and de Lagran, 2006). For example, while Ras-dependent signaling is required for promoting or maintaining neuronal differentiation, Dyrk1A modulates ERK activation by interacting with Ras, B-Raf, and MEK1 and by facilitating the formation of a Ras/B-Raf/MEK1 multi-protein complex (Kelly and Rahmani, 2005). Recently, many studies implicated a potential role of Dyrk1A during cell death. For example, Dyrk1A caused the formation of abnormal protein aggregates through the phosphorylation of a-synuclein, APP, and tau. Furthermore, these cells show a marked increase of apoptotic cell death (Park et al., 2007), indicating that the overexpression of Dyrk1A induces cell death\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurobiol. 2011 Mar;20(1):35-44. Epub 2011 Mar 31.\", \"reference\": \"22110360\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"DYRK1A subgraph\"}, \"source\": 38, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"APOE subgraph\"}, \"source\": 39, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Subgraph\": \"APOE subgraph\"}, \"source\": 39, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease is associated with impaired clearance of Abeta-amyloid from the brain, a process normally facilitated by apolipoprotein E (ApoE)\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Science. 2012 Feb 9. [Epub ahead of print]\", \"reference\": \"22323736\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 39, \"target\": 9, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 40, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 41, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"annotations\": {\"Subgraph\": \"Beta secretase subgraph\"}, \"source\": 41, \"target\": 9, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"The PSEN1 AD mutations giving rise to CWP produce unusually high levels of the amyloid beta peptide (Abeta) ending at position 42 or 43, and the main component of CWP is amino-terminally truncated forms of amyloid beta peptide starting after the alternative beta-secretase cleavage site at position 11.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Neurochem2008\", \"reference\": \"17995932\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 41, \"target\": 9, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Gamma secretase subgraph\"}, \"source\": 42, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Alzheimer's disease (AD) is a chronic disorder that slowly destroys neurons and causes serious cognitive disability. AD is associated with senile plaques and neurofibrillary tangles (NFTs). Amyloid-beta (Abeta), a major component of senile plaques, has various pathological effects on cell and organelle function. The extracellular Abeta oligomers may activate caspases through activation of cell surface death receptors. Alternatively, intracellular Abeta may contribute to pathology by facilitating tau hyper-phosphorylation, disrupting mitochondria function, and triggering calcium dysfunction. To date genetic studies have revealed four genes that may be linked to autosomal dominant or familial early onset AD (FAD). These four genes include: amyloid precursor protein (APP), presenilin 1 (PS1), presenilin 2 (PS2) and apolipoprotein E (ApoE). All mutations associated with APP and PS proteins can lead to an increase in the production of Abeta peptides, specfically the more amyloidogenic form, Abeta42. FAD-linked PS1 mutation downregulates the unfolded protein response and leads to vulnerability to ER stress.\", \"citation\": {\"type\": \"Online Resource\", \"name\": \"Alzheimer's disease - Homo sapiens (human)\", \"reference\": \"hsa05010\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 42, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 43, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Ubiquitin degradation subgraph\"}, \"source\": 43, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 43, \"target\": 9, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Peroxisome proliferator activated receptor subgraph\"}, \"source\": 43, \"target\": 9, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Rosiglitazone is a TZD derivative that was recently observed to improve cognition in both APP transgenic mice and AD patients [Watson et al., 2005; Pedersen et al., 2006]. It activates PPAR leading to increased glucose disposal rates [Jung et al., 2005] and is also neuroprotective against Abeta neurotoxicity [Inestrosa et al., 2005]. Neuroprotection may result through modulation of Wnt signaling since an increase in -catenin and inhibition of GSK-3 is observed upon exposure to rosiglitazone [Inestrosa et al., 2005], although a different study has shown that activation of PPAR by a potent ligand leads to -catenin degradation [Liu and Farmer 2004]. Nonetheless, in support of the role for PPAR activation in neuroprotection, PPAR agonists have potent anti-inflammatory effects [Luna-Medina et al., 2005], inhibit microglial activation [Bernardo et al., 2005; Heneka et al., 2005] and have been shown to improve verbal memory in AD patients with T2DM, possibly in a mechanism dependent on PPAR activation . A recent report demonstrates that overexpression of PPAR in cultured cells leads to a dramatic decrease in the production of Abeta, by increasing the rate of APP degradation via ubiquitination [D'Abramo et al., 2005]. This study also showed that by decreasing Abeta secretion, PPAR protects the cells against H2O2-mediated necrosis [D'Abramo et al., 2005].\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Med Genet B Neuropsychiatr Genet. 2007 Jun 5;144B(4):508-16.\", \"reference\": \"19885299\"}, \"annotations\": {\"UserdefinedCellLine\": \"App transgenic\", \"MeSHDisease\": \"Alzheimer Disease\", \"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 43, \"target\": 9, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 44, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 44, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 45, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 45, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 46, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"As such, genetic variations in ABCB1 might contribute to the progression of amyloid-beta deposition in the brain.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EJNMMI Res. 2012 Oct 16;2(1):57\", \"reference\": \"23067778\"}, \"annotations\": {\"MeSHAnatomy\": \"Brain\", \"Subgraph\": \"ATP binding cassette transport subgraph\"}, \"source\": 46, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \" Introduction of the APP KM595/596NL Swedish mutation causing overproduction of Abeta, however, surprisingly diminished the concentration of Abeta 3(pE)-40/42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 47, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" Introduction of the APP KM595/596NL Swedish mutation causing overproduction of Abeta, however, surprisingly diminished the concentration of Abeta 3(pE)-40/42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Biochemistry2008\", \"reference\": \"18570439\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 48, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We also found that the increasing trend of cellular histone H3 acetylation is consistent with the increased expression of Abeta1-42 and Abeta1-40 peptides in N2a/APPswe and N2a/APPwt cells. This data suggested that the cellular histone hyperacetylation may regulate the genes that produce the Abeta peptides\", \"citation\": {\"type\": \"PubMed\", \"name\": \"PLoS One. 2014 Jul 22;9(7):e103067. doi: 10.1371/journal.pone.0103067. eCollection 2014.\", \"reference\": \"25051175\"}, \"annotations\": {\"Encode_Feature_Types\": \"Promoter\"}, \"source\": 49, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \" We found that bilateral microinjection of amyloid beta (Abeta)1-40 fibrils into the hippocampal CA1 area of resulted in significant upregulation of CX3CR1 messenger RNA (mRNA) and protein expression (via increasing histone H3 acetylation in the Cx3cr1 promoter region), synaptic dysfunction, and cognitive impairment,\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Aging. 2013 Dec;34(12):2843-52. doi: 10.1016/j.neurobiolaging.2013.06.003. Epub 2013 Jul 12.\", \"reference\": \"23855980\"}, \"annotations\": {\"MeSHAnatomy\": \"Hippocampus\", \"Encode_Feature_Types\": \"Promoter\"}, \"source\": 49, \"target\": 18, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Signaling pathways involving PKR-mediated inflammation and immune regulation. (A) PKR is required for / inflammasome activation. PAMPs and DAMPs promote the autophosphorylation and activation of PKR. Active PKR physically / interacts with NLRP1, NLRP3, NLRC4, and AIM2 and promotes the release of inflammasome-dependent cytokines, including IL-1, / IL-18, and HMGB1. LPS, lipopolysaccharide. (B) The activation of PKR is induced by many different stimuli, such as / cytokines, bacterial and viral infection, and DNA damage. Activation of PKR results in its dimerization and / autophosphorylation. The activation of PKR in turn triggers the production of IFN and proinflammatory cytokines / through transcriptional regulatory signals. In addition, the FADDcaspase-8 pathway mediates the proapoptotic activity / of PKR, whereas the eIF2-LC3 pathway mediates the proautophagic activity of PKR. Both apoptosis and autophagy are / important in the regulation of inflammation and immunity.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Response to oxidative stress\"}, \"source\": 50, \"target\": 10, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 30,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pbt.summary.print_summary(subgraph)\n",
    "to_jupyter(subgraph)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This entire procedure can be run automatically with [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Running"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [],
   "source": [
    "runner = npa.NpaRunner(subgraph, inflamation_node, label)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "While the algorithm can be immediately run with [pbt.analysis.npa.NpaRunner.run](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.NpaRunner.run), the process is outlined to show the evolution of the graph throughouts its steps. For animation purposes, the functin [pbt.analysis.npa.NpaRunner.run_with_graph_transformation](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.NpaRunner.run_with_graph_transformation) yields a new `BELGraph` at each step through the process as well."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Abundance', 'MESHCS', 'Neurofibrillary Tangles'),\n",
       " ('Protein', 'HGNC', 'CX3CR1'),\n",
       " ('Protein', 'HGNC', 'IL12B')}"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"dtzjxqobeyqvgrmp\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"cname\": \"SCARF1\", \"id\": [\"Protein\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"cname\": \"SCARB1\", \"id\": [\"Protein\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 1, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"ybjfvhurdgpoczts\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"cname\": \"SCARF1\", \"id\": [\"Protein\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"cname\": \"SCARB1\", \"id\": [\"Protein\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 1, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"kvgqyqdpahbmtsjx\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARF1\", \"cname\": \"SCARF1\", \"id\": [\"Protein\", \"HGNC\", \"SCARF1\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARB1\", \"cname\": \"SCARB1\", \"id\": [\"Protein\", \"HGNC\", \"SCARB1\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 1, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 2, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 3, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 1, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 1, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 6, \"target\": 4, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 6, \"target\": 7, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 6, \"target\": 8, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 6, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 8, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 9, \"target\": 1, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "\n",
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{('Protein', 'HGNC', 'SCARB1')}"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:no leaves.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "set()"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.score_leaves()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.remove_random_edge()\n",
    "leaves = runner.score_leaves()\n",
    "len(leaves)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"gzcpevkqtuqsfyjh\";\n",
       "        var width = 1000;\n",
       "        var height = 400;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.5.4.dev0\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"inflammatory response\", \"cname\": \"inflammatory response\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"inflammatory response\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"SCARA5\", \"cname\": \"SCARA5\", \"id\": [\"Protein\", \"HGNC\", \"SCARA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"GSK3B\", \"cname\": \"GSK3B\", \"id\": [\"Protein\", \"HGNC\", \"GSK3B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"TLR4\", \"cname\": \"TLR4\", \"id\": [\"Protein\", \"HGNC\", \"TLR4\"]}, {\"function\": \"Gene\", \"namespace\": \"HGNC\", \"name\": \"APP\", \"variants\": [{\"kind\": \"frag\", \"start\": 672, \"stop\": 713}], \"weight\": -0.13701466, \"cname\": \"g(HGNC:APP, frag(672_713))\", \"id\": [\"Protein\", \"HGNC\", \"APP\", [\"frag\", [672, 713]]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"EIF2AK2\", \"cname\": \"EIF2AK2\", \"id\": [\"Protein\", \"HGNC\", \"EIF2AK2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"AGER\", \"cname\": \"AGER\", \"id\": [\"Protein\", \"HGNC\", \"AGER\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MSR1\", \"cname\": \"MSR1\", \"id\": [\"Protein\", \"HGNC\", \"MSR1\"]}], \"links\": [{\"relation\": \"increases\", \"evidence\": \"The serum IL-1 and TNF- level were significantly increased, and the expressions of TLR4 and NF-B p65 mRNA and protein in the brain were up-regulated, indicating inflammation response was initiated following administration of Abeta1-42.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Int J Dev Neurosci. 2011 Aug;29(5):537-42\", \"reference\": \"21515354\"}, \"annotations\": {\"Subgraph\": \"Toll like receptor subgraph\", \"MeSHAnatomy\": \"Serum\"}, \"source\": 0, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 1, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 1, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Genes known to contribute to neuroinflammation that were most negatively affected by GSK3beta inactivation included IP-10/CXCL10, MCP-1/CCL2, IL-8/CXCL8, RANTES/CCL5, and Groalpha/CXCL1. Interactions of monocytes with TNFalpha-activated BMVECs led to barrier disruption, and GSK3beta suppression in the endothelium restored barrier integrity. GSK3beta inhibition in vivo substantially decreased leukocyte adhesion to brain endothelium under inflammatory conditions. In summary, inhibition of GSK3beta emerges as an important target for stabilization of the blood-brain barrier in neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol. 2010 Feb;176(2):881-92. Epub 2010 Jan 7\", \"reference\": \"20056834\"}, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 2, \"target\": 0, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 2, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"The inhibition of this kinase can prevent the aggregation of -amyloid (Abeta) and hyperphosphorylation of tau protein.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Rev Neurosci. 2011 Dec 21;23(1):1-11. doi: 10.1515/rns.2011.061.\", \"reference\": \"22718609\"}, \"subject\": {\"modifier\": \"Degradation\"}, \"annotations\": {\"Subgraph\": \"Non-amyloidogenic subgraph\"}, \"source\": 2, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 3, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Blood vessel dilation subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Moreover, in examination of this pathway in another cell type pertinent to AD, we find that Abeta induces a proinflammatory response in microglia as evidenced by increased leukotriene B4 release. We show that both dipyridamole and compounds which increase cGMP levels prevent Abeta-induced microglial inflammation. Our results suggest that therapeutic intervention aimed at reduction of microglial-mediated inflammation via inhibition of cGMP-PDE or elevation of cGMP may be beneficial in the treatment of AD.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Exp Neurol. 1999 May;157(1):211-21\", \"reference\": \"10222124\"}, \"annotations\": {\"MeSHAnatomy\": \"Microglia\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In disease state, LPS (lipoploysacharide) induces TLR4, which increases NFKB1 activities. NFKB1 increases MIR34A which targets TREM2, decreasing normal TREM2 and increases the mutant variant. Recent GWAS studies associated SNP rs75932628 with TREM2 in LOAD patients. Also there are studies suggesting the link of this TREM2 variant with certain clinical and neuroimaging AD features such as frontobasal gray atrophy. Moreover, in disease brain TREM2 forms complex with TYROBP which triggers immune responses through activating macrophages and dendritic cells which leads to chronic neuroinflammation.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Immunol. 2015 Mar 15;194(6):2862-70. doi: 10.4049/jimmunol.1400874. Epub 2015 Feb 13.\", \"reference\": \"25681350\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 4, \"target\": 0, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 4, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 2, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Abeta-Amyloid (Abeta) plaques in Alzheimer (AD) brains are surrounded by severe dendritic and axonal changes, including local spine loss, axonal swellings and distorted neurite trajectories. Whether and how plaques induce these neuropil abnormalities remains unknown. We tested the hypothesis that oligomeric assemblies of Abeta, seen in the periphery of plaques, mediate the neurodegenerative phenotype of AD by triggering activation of the enzyme GSK-3Abeta, which in turn appears to inhibit a transcriptional program mediated by CREB. We detect increased activity of GSK-3Abeta after exposure to oligomeric Abeta in neurons in culture, in the brain of double transgenic APP/tau mice and in AD brains. Activation of GSK-3Abeta, even in the absence of Abeta, is sufficient to produce a phenocopy of Abeta-induced dendritic spine loss in neurons in culture, while pharmacological inhibition of GSK-3Abeta prevents spine loss and increases expression of CREB-target genes like BDNF. Of note, in transgenic mice GSK-3Abeta inhibition ameliorated plaque-related neuritic changes and increased CREB-mediated gene expression. Moreover, GSK-3Abeta inhibition robustly decreased the oligomeric Abeta load in the mouse brain. All these findings support the idea that GSK3Abeta is aberrantly activated by the presence of Abeta, and contributes, at least in part, to the neuronal anatomical derangement associated with Abeta plaques in AD brains and to Abeta pathology itself.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurobiol Dis. 2012 Jan;45(1):425-37. Epub 2011 Sep 13.\", \"reference\": \"21945540\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Disease\": \"Alzheimer's disease\", \"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 2, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Tau protein subgraph\"}, \"source\": 4, \"target\": 2, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 2, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"In the present review, we discuss our initial in vitro results and additional investigations showing that Abeta activates GSK-3beta through impairment of phosphatidylinositol-3 (PI3)/Akt signaling; that Abeta-activated GSK-3beta induces hyperphosphorylation of tau, NFT formation, neuronal death, and synaptic loss (all found in the AD brain); that GSK-3beta can induce memory deficits in vivo; and that inhibition of GSK-3alpha (an isoform of GSK-3beta) reduces Abeta production.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Alzheimers Dis2006\", \"reference\": \"16914869\"}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Subgraph\": \"GSK3 subgraph\"}, \"source\": 4, \"target\": 2, \"key\": 4}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Amyloidogenic subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have shown previously that activation of PKR in Abeta-triggered apoptosis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Apoptosis. 2006 May;11(5):687-700\", \"reference\": \"16532272\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"MAPK-JNK subgraph\"}, \"source\": 4, \"target\": 5, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem. 2006;13(17):1971-8.\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Immunoglobulin subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The receptor for advanced glycation end products (RAGE) is a cell-bound receptor of the immunoglobulin superfamily which may be activated by a variety of proinflammatory ligands including advanced glycoxidation end products, S100/calgranulins, high mobility group box 1, and amyloid beta-peptide.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Curr Med Chem2006\", \"reference\": \"16842191\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Cytokine signaling subgraph\"}, \"source\": 4, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Our results suggest that SRs play a role on inflammatory activation, inducing production of NO and IL1Abeta, and show potentiation by Abeta. Potentiation of the inflammatory response of Abeta could be meaningful for the activation of glia observed in AD.We propose that scavenger receptors (SR) participate in the activation of glia by Abeta.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Neurotox Res. 2012 Jan 12. [Epub ahead of print]\", \"reference\": \"22237943\"}, \"object\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Condition\": \"Normal Healthy State\", \"Subgraph\": \"Binding and Uptake of Ligands by Scavenger Receptors\"}, \"source\": 4, \"target\": 7, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Double-stranded RNA-dependent protein kinase (PKR) is implicated in inflammation and immune dysfunction/ through its regulation of mitogen-activated protein kinases, interferon regulatory factor 3, nuclear factor B,/ apoptosis, and autophagy pathways. A study shows that PKR is also required for the activation of inflammasomes and/ the subsequent release of high-mobility group box 1 (HMGB1) protein, a proinflammatory cytokine. Thus, the cell stress/ kinase PKR has multifaceted roles in the regulation of inflammatory immune responses, and PKR and HMGB1 are attractive/ targets for inflammasome-associated diseases.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Sci Signal. 2012 Oct 23;5(247):pe47. doi: 10.1126/scisignal.2003511.\", \"reference\": \"23092889\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 5, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 6, \"target\": 0, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"increased production of amyloid-beta peptide species can activate the innate immunity system via pattern recognition receptors (PRRs) and evoke Alzheimer's pathology. We will focus on the role of innate immunity system of brain in the initiation and the propagation of inflammatory process in AD. We examine here in detail the significance of amyloid-beta oligomers and fibrils as danger-associated molecular patterns (DAMPs) in the activation of a wide array of PRRs in glial cells and neurons, such as Toll-like, NOD-like, formyl peptide, RAGE and scavenger receptors along with complement and pentraxin systems.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Prog Neurobiol. 2009 Feb;87(3):181-94.\", \"reference\": \"19388207\"}, \"subject\": {\"modifier\": \"Activity\"}, \"annotations\": {\"Subgraph\": \"Inflammatory response subgraph\"}, \"source\": 7, \"target\": 0, \"key\": 0}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(runner.get_remaining_graph(), height=400)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [],
   "source": [
    "runner.run()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "5.6699960399999956"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "runner.get_final_score()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Robustness Analysis\n",
    "\n",
    "The algorithm is random, so it can be run multiple times to assess the robustness of the final scores using [pbt.analysis.npa.multirun](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.multirun)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 7.59 s, sys: 77.7 ms, total: 7.66 s\n",
      "Wall time: 7.69 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "runners = list(npa.multirun(subgraph, inflamation_node, label, default_score=0, runs=500))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"274pt\" version=\"1.1\" viewBox=\"0 0 387 274\" width=\"387pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M 0 274.853533 \n",
       "L 387.499682 274.853533 \n",
       "L 387.499682 0 \n",
       "L 0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "L 41.999682 21.50218 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 57.217864 238.94218 \n",
       "L 87.654228 238.94218 \n",
       "L 87.654228 211.242817 \n",
       "L 57.217864 211.242817 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 87.654228 238.94218 \n",
       "L 118.090592 238.94218 \n",
       "L 118.090592 31.856466 \n",
       "L 87.654228 31.856466 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 118.090592 238.94218 \n",
       "L 148.526955 238.94218 \n",
       "L 148.526955 83.29814 \n",
       "L 118.090592 83.29814 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 148.526955 238.94218 \n",
       "L 178.963319 238.94218 \n",
       "L 178.963319 89.893226 \n",
       "L 148.526955 89.893226 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 178.963319 238.94218 \n",
       "L 209.399682 238.94218 \n",
       "L 209.399682 202.009696 \n",
       "L 178.963319 202.009696 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 209.399682 238.94218 \n",
       "L 239.836046 238.94218 \n",
       "L 239.836046 196.733627 \n",
       "L 209.399682 196.733627 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 239.836046 238.94218 \n",
       "L 270.27241 238.94218 \n",
       "L 270.27241 213.880851 \n",
       "L 239.836046 213.880851 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 270.27241 238.94218 \n",
       "L 300.708773 238.94218 \n",
       "L 300.708773 229.709059 \n",
       "L 270.27241 229.709059 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 300.708773 238.94218 \n",
       "L 331.145137 238.94218 \n",
       "L 331.145137 234.985128 \n",
       "L 300.708773 234.985128 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#p8e5da269d9)\" d=\"M 331.145137 238.94218 \n",
       "L 361.581501 238.94218 \n",
       "L 361.581501 236.304145 \n",
       "L 331.145137 236.304145 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m4707f4657b\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"71.706603\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- $2$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(69.215936 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"119.900281\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- $4$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(117.409613 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"168.093959\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- $6$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(165.603291 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"216.287637\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- $8$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 27.203125 35.375 \n",
       "C 33.5 38.546875 39.90625 43.328125 39.90625 50.984375 \n",
       "C 39.90625 60.015625 31.09375 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 58.734375 5.90625 49.6875 \n",
       "C 5.90625 47.203125 6.5 42.921875 10.40625 39.140625 \n",
       "C 11.40625 38.15625 15.59375 35.171875 18.296875 33.28125 \n",
       "C 13.796875 30.984375 3.296875 25.53125 3.296875 14.59375 \n",
       "C 3.296875 4.359375 13.09375 -2 22.796875 -2 \n",
       "C 33.5 -2 42.5 5.65625 42.5 15.78125 \n",
       "C 42.5 24.84375 36.40625 29.015625 32.40625 31.703125 \n",
       "z\n",
       "M 14.09375 44.09375 \n",
       "C 13.296875 44.59375 9.296875 47.671875 9.296875 52.359375 \n",
       "C 9.296875 58.421875 15.59375 63 22.796875 63 \n",
       "C 30.703125 63 36.5 57.4375 36.5 50.96875 \n",
       "C 36.5 41.703125 26.09375 36.421875 25.59375 36.421875 \n",
       "C 25.5 36.421875 25.40625 36.421875 24.59375 37.03125 \n",
       "z\n",
       "M 32.5 23.734375 \n",
       "C 34 22.640625 38.796875 19.375 38.796875 13.296875 \n",
       "C 38.796875 5.953125 31.40625 0.390625 23 0.390625 \n",
       "C 13.90625 0.390625 7 6.84375 7 14.6875 \n",
       "C 7 22.546875 13.09375 29.09375 20 32.1875 \n",
       "z\n",
       "\" id=\"CMR17-56\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(213.796969 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"264.481314\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_5\">\n",
       "      <!-- $10$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(259.499979 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"312.674992\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- $12$ -->\n",
       "      <g transform=\"translate(307.693657 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"360.86867\" xlink:href=\"#m4707f4657b\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $14$ -->\n",
       "      <g transform=\"translate(355.887334 252.860678)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_8\">\n",
       "     <!-- Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(197.762748 265.716355)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(91.38103 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(137.071507 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(172.352275 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"m77175bf265\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"238.94218\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_9\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(30.018347 242.401429)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"212.561834\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(25.037012 216.021083)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"186.181488\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(25.037012 189.640737)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"159.801143\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $60$ -->\n",
       "      <g transform=\"translate(25.037012 163.260392)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"133.420797\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $80$ -->\n",
       "      <g transform=\"translate(25.037012 136.880046)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-56\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"107.040451\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_14\">\n",
       "      <!-- $100$ -->\n",
       "      <g transform=\"translate(20.055677 110.4997)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"80.660105\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $120$ -->\n",
       "      <g transform=\"translate(20.055677 84.119354)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"54.279759\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $140$ -->\n",
       "      <g transform=\"translate(20.055677 57.739009)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"41.999682\" xlink:href=\"#m77175bf265\" y=\"27.899414\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $160$ -->\n",
       "      <g transform=\"translate(20.055677 31.358663)scale(0.1 -0.1)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(91.380954 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_18\">\n",
       "     <!-- Frequency -->\n",
       "     <defs>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 41.09375 43.78125 \n",
       "L 39.5 43.78125 \n",
       "L 36.5 33.9375 \n",
       "C 34.203125 39.203125 30 43.78125 23.296875 43.78125 \n",
       "C 13.09375 43.78125 3.296875 34.734375 3.296875 21.609375 \n",
       "C 3.296875 8.5625 12.5 -0.59375 22.5 -0.59375 \n",
       "C 30.203125 -0.59375 34.5 5.28125 35.796875 7.46875 \n",
       "L 35.796875 -12.328125 \n",
       "C 35.796875 -16.109375 35.40625 -16.609375 29 -16.609375 \n",
       "L 29 -19.1875 \n",
       "C 31.40625 -19 35.796875 -19 38.40625 -19 \n",
       "C 41 -19 45.5 -19 47.90625 -19.1875 \n",
       "L 47.90625 -16.609375 \n",
       "C 41.5 -16.609375 41.09375 -16.203125 41.09375 -12.328125 \n",
       "z\n",
       "M 36 13.84375 \n",
       "C 36 9.5625 32.40625 5.578125 32.09375 5.28125 \n",
       "C 29 1.796875 25.40625 1 23 1 \n",
       "C 15.5 1 9.703125 10.453125 9.703125 21.484375 \n",
       "C 9.703125 33.140625 16.296875 42 24 42 \n",
       "C 32 42 36 32.9375 36 27.765625 \n",
       "z\n",
       "\" id=\"CMR17-113\"/>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-110\"/>\n",
       "      <path d=\"M 39 34.328125 \n",
       "C 41.5 40.40625 46 40.40625 47.40625 40.40625 \n",
       "L 47.40625 43 \n",
       "C 45.5 43 43 43 41.09375 43 \n",
       "C 39 43 35.703125 43 33.703125 43.09375 \n",
       "L 33.703125 40.5 \n",
       "C 37.5 40.203125 37.59375 37.40625 37.59375 36.609375 \n",
       "C 37.59375 35.625 37.40625 35.125 36.90625 33.921875 \n",
       "L 26 7.015625 \n",
       "L 14.09375 36.03125 \n",
       "C 13.59375 37.21875 13.59375 37.90625 13.59375 38.015625 \n",
       "C 13.59375 40.203125 15.90625 40.40625 18.296875 40.40625 \n",
       "L 18.296875 43.09375 \n",
       "C 15.90625 43 11.59375 43 9.09375 43 \n",
       "C 6.40625 43 3.203125 43 1 43.09375 \n",
       "L 1 40.40625 \n",
       "C 6.40625 40.40625 7 39.90625 8.296875 36.71875 \n",
       "L 23.203125 0.234375 \n",
       "C 18.703125 -11.53125 16.09375 -18.40625 9.796875 -18.40625 \n",
       "C 8.703125 -18.40625 6.203125 -18.109375 4.40625 -16.3125 \n",
       "C 6.703125 -16.109375 7.703125 -14.71875 7.703125 -13.015625 \n",
       "C 7.703125 -11.328125 6.5 -9.828125 4.5 -9.828125 \n",
       "C 2.296875 -9.828125 1.203125 -11.328125 1.203125 -13.125 \n",
       "C 1.203125 -17.109375 5.296875 -20 9.796875 -20 \n",
       "C 15.59375 -20 19.203125 -14.40625 21.203125 -9.4375 \n",
       "z\n",
       "\" id=\"CMR17-121\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 152.444442)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(52.176591 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(87.457359 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(127.943019 0)scale(0.996264)\" xlink:href=\"#CMR17-113\"/>\n",
       "      <use transform=\"translate(176.235935 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(227.131305 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(267.616966 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "      <use transform=\"translate(318.512335 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(358.997996 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 41.999682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 376.799682 238.94218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 41.999682 238.94218 \n",
       "L 376.799682 238.94218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 41.999682 21.50218 \n",
       "L 376.799682 21.50218 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"text_19\">\n",
       "    <!-- Histogram of NPA scores for inflammatory response -->\n",
       "    <defs>\n",
       "     <path d=\"M 56.09375 61.015625 \n",
       "C 56.09375 65.5 57.203125 65.5 64.09375 65.5 \n",
       "L 64.09375 68 \n",
       "C 61.203125 68 55.90625 68 52.796875 68 \n",
       "C 49.703125 68 44.40625 68 41.5 68 \n",
       "L 41.5 65.5 \n",
       "C 48.40625 65.5 49.5 65.5 49.5 61 \n",
       "L 49.5 36.578125 \n",
       "L 19.703125 36.578125 \n",
       "L 19.703125 61 \n",
       "C 19.703125 65.5 20.796875 65.5 27.703125 65.5 \n",
       "L 27.703125 68 \n",
       "C 24.796875 68 19.5 68 16.40625 68 \n",
       "C 13.296875 68 8 68 5.09375 68 \n",
       "L 5.09375 65.5 \n",
       "C 12 65.5 13.09375 65.5 13.09375 61.015625 \n",
       "L 13.09375 6.984375 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.296875 0 16.40625 0 \n",
       "C 19.5 0 24.796875 0 27.703125 0 \n",
       "L 27.703125 2.5 \n",
       "C 20.796875 2.5 19.703125 2.5 19.703125 6.96875 \n",
       "L 19.703125 34 \n",
       "L 49.5 34 \n",
       "L 49.5 6.96875 \n",
       "C 49.5 2.5 48.40625 2.5 41.5 2.5 \n",
       "L 41.5 0 \n",
       "C 44.40625 0 49.703125 0 52.796875 0 \n",
       "C 55.90625 0 61.203125 0 64.09375 0 \n",
       "L 64.09375 2.5 \n",
       "C 57.203125 2.5 56.09375 2.5 56.09375 6.984375 \n",
       "z\n",
       "\" id=\"CMR17-72\"/>\n",
       "     <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "     <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "     <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "     <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "     <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "     <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "     <path d=\"M 15.296875 40 \n",
       "L 26.703125 40 \n",
       "L 26.703125 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 54.59375 \n",
       "C 15.09375 62.84375 19.296875 68 23.59375 68 \n",
       "C 24.90625 68 26.40625 67.609375 27.40625 66.90625 \n",
       "C 26.59375 66.703125 24.5 66.015625 24.5 63.546875 \n",
       "C 24.5 60.953125 26.5 60.15625 27.90625 60.15625 \n",
       "C 29.296875 60.15625 31.296875 60.953125 31.296875 63.546875 \n",
       "C 31.296875 67.3125 27.703125 69.59375 23.703125 69.59375 \n",
       "C 18 69.59375 10.203125 64.9375 10.203125 54.390625 \n",
       "L 10.203125 42.578125 \n",
       "L 2.203125 42.578125 \n",
       "L 2.203125 40 \n",
       "L 10.203125 40 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.40625 0 13 0 \n",
       "C 15.90625 0 21 0 23.703125 -0.203125 \n",
       "L 23.703125 2.390625 \n",
       "C 16.5 2.390625 15.296875 2.390625 15.296875 6.859375 \n",
       "z\n",
       "\" id=\"CMR17-102\"/>\n",
       "     <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "     <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "     <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "     <path d=\"M 39.90625 68.890625 \n",
       "C 38.796875 68.6875 37.90625 68.5 36.59375 68.203125 \n",
       "C 33.703125 69.6875 29.90625 69.6875 28.90625 69.6875 \n",
       "C 20.40625 69.6875 9.90625 64.90625 9.90625 54.25 \n",
       "L 9.90625 42.578125 \n",
       "L 1.90625 42.578125 \n",
       "L 1.90625 40 \n",
       "L 9.90625 40 \n",
       "L 9.90625 6.671875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 9.90625 0 12.40625 0 \n",
       "C 15 0 19.40625 0 21.796875 -0.203125 \n",
       "L 21.796875 2.390625 \n",
       "C 15.40625 2.390625 15 2.78125 15 6.671875 \n",
       "L 15 40 \n",
       "L 34.796875 40 \n",
       "L 34.796875 6.671875 \n",
       "C 34.796875 2.875 34.40625 2.390625 28 2.390625 \n",
       "L 28 -0.203125 \n",
       "C 30.40625 0 34.796875 0 37.296875 0 \n",
       "C 39.90625 0 44.296875 0 46.703125 -0.203125 \n",
       "L 46.703125 2.390625 \n",
       "C 40.296875 2.390625 39.90625 2.78125 39.90625 6.671875 \n",
       "z\n",
       "M 14.796875 42.578125 \n",
       "L 14.796875 54.203125 \n",
       "C 14.796875 62.9375 21.40625 68 28.703125 68 \n",
       "C 29.203125 68 34.703125 68 34.703125 66.90625 \n",
       "C 34.703125 66.796875 34.59375 66.703125 34.09375 66.203125 \n",
       "C 33.203125 65.609375 32.796875 64.609375 32.796875 63.609375 \n",
       "C 32.796875 62.203125 33.5 61.015625 34.796875 60.515625 \n",
       "L 34.796875 42.578125 \n",
       "z\n",
       "\" id=\"CMR17-13\"/>\n",
       "     <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "    </defs>\n",
       "    <g transform=\"translate(76.170122 15.50218)scale(0.12 -0.12)\">\n",
       "     <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-72\"/>\n",
       "     <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(93.943488 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(129.744826 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(165.025594 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(210.716071 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "     <use transform=\"translate(256.406548 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(291.687316 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(337.377793 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(444.373349 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(490.063826 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(547.613305 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "     <use transform=\"translate(616.685735 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "     <use transform=\"translate(671.46478 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "     <use transform=\"translate(770.613176 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(806.414514 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "     <use transform=\"translate(846.900174 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(892.590651 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(927.871419 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(968.35708 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1034.234384 0)scale(0.996264)\" xlink:href=\"#CMR17-102\"/>\n",
       "     <use transform=\"translate(1061.707896 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1107.398373 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1172.755107 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "     <use transform=\"translate(1197.626166 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1248.521535 0)scale(0.996264)\" xlink:href=\"#CMR17-13\"/>\n",
       "     <use transform=\"translate(1298.263759 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1343.954236 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1420.873826 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "     <use transform=\"translate(1497.793416 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "     <use transform=\"translate(1543.483893 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "     <use transform=\"translate(1578.764661 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1624.455138 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1659.735906 0)scale(0.996264)\" xlink:href=\"#CMR17-121\"/>\n",
       "     <use transform=\"translate(1738.104788 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "     <use transform=\"translate(1773.385556 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     <use transform=\"translate(1813.871216 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(1849.672554 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "     <use transform=\"translate(1903.170362 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "     <use transform=\"translate(1948.860839 0)scale(0.996264)\" xlink:href=\"#CMR17-110\"/>\n",
       "     <use transform=\"translate(1999.756209 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     <use transform=\"translate(2035.557547 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"p8e5da269d9\">\n",
       "   <rect height=\"217.44\" width=\"334.8\" x=\"41.999682\" y=\"21.50218\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x10c756240>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "plt.title('Histogram of NPA scores for {}'.format(calculate_canonical_name(subgraph, inflamation_node)))\n",
    "plt.ylabel('Frequency')\n",
    "plt.xlabel('Score')\n",
    "plt.hist([runner.get_final_score() for runner in runners])\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Full Workflow\n",
    "\n",
    "This workflow can all be run with [pbt.analysis.npa.workflow_all_average](http://pybel-tools.readthedocs.io/en/latest/npa.html#pybel_tools.analysis.npa.workflow_all_average) but is outlined here for a deeper understanding.\n",
    "\n",
    "### Generate Candidate Mechanisms\n",
    "\n",
    "Candidate mechanism are generated with the aforementioned function [pbt.generation.generate_mechanism](http://pybel-tools.readthedocs.io/en/latest/generation.html#pybel_tools.generation.generate_mechanism) to assess the upstream causal subgraphs of all biological processes."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "all_bioprocesses = pbt.filters.get_nodes_by_function(graph, BIOPROCESS)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 1.88 s, sys: 20.1 ms, total: 1.9 s\n",
      "Wall time: 1.91 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "candidate_mechanisms = {node: pbt.generation.generate_mechanism(graph, node, label) for node in all_bioprocesses}"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The algorithm is run over all candidate subgraphs that are nontrivial (have more than one node in them) and a pandas DataFrame is prepared."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'endoplasmic reticulum calcium ion homeostasis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'neuron migration')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'endocytosis')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'synaptic vesicle budding from endosome')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'positive regulation of mitochondrial membrane potential')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'cellular senescence')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'synaptic transmission, cholinergic')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'xenobiotic metabolic process')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'rhythmic excitation')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'phagocytosis, engulfment')\n",
      "WARNING:pybel_tools.analysis.npa:Unable to run NPA on ('BiologicalProcess', 'GOBP', 'regulation of dendritic cell antigen processing and presentation')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 10.8 s, sys: 81.8 ms, total: 10.9 s\n",
      "Wall time: 10.9 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "\n",
    "scores = {}\n",
    "\n",
    "for node, subgraph in candidate_mechanisms.items():\n",
    "    \n",
    "    fneighbors = subgraph.in_degree(node)\n",
    "    fneighbors = 0 if isinstance(fneighbors, dict) else fneighbors\n",
    "    size = subgraph.number_of_nodes()\n",
    "    \n",
    "    if size <= 1:  # Can't calculate a score for empty subgraphs\n",
    "        average_score = None\n",
    "    else:\n",
    "        average_score = npa.workflow_average(subgraph, node, label, runs=40)\n",
    "    \n",
    "    scores[node] = (average_score, fneighbors, size)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(430, 139)"
      ]
     },
     "execution_count": 49,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(candidate_mechanisms), sum(1 for score, _, _ in scores.values() if score)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Optionally, calcualte \"cute names\" for the nodes\n",
    "# scores = {calculate_canonical_name(graph, n): s for n, s in scores.items()}\n",
    "\n",
    "scores_df = pd.DataFrame.from_items(scores.items(), orient='index', columns=['Average NPA Score','Number First Neighbors','Subgraph Size'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "A scatter matrix is shown below. The distribution of average NPA scores is shown to be mostly normal. The following scatter plots show small, non-linear relationships between average NPA score and the subgraph size and the number of first neighbors of the seed node. The trivial positive correlation between number of first neighbors is also shown."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/svg+xml": [
       "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"no\"?>\n",
       "<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n",
       "  \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n",
       "<!-- Created with matplotlib (http://matplotlib.org/) -->\n",
       "<svg height=\"429pt\" version=\"1.1\" viewBox=\"0 0 604 429\" width=\"604pt\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n",
       " <defs>\n",
       "  <style type=\"text/css\">\n",
       "*{stroke-linecap:butt;stroke-linejoin:round;}\n",
       "  </style>\n",
       " </defs>\n",
       " <g id=\"figure_1\">\n",
       "  <g id=\"patch_1\">\n",
       "   <path d=\"M -0 429.096366 \n",
       "L 604.224042 429.096366 \n",
       "L 604.224042 0 \n",
       "L -0 0 \n",
       "z\n",
       "\" style=\"fill:none;\"/>\n",
       "  </g>\n",
       "  <g id=\"axes_1\">\n",
       "   <g id=\"patch_2\">\n",
       "    <path d=\"M 35.524042 137.54 \n",
       "L 221.524042 137.54 \n",
       "L 221.524042 10.7 \n",
       "L 35.524042 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_3\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 39.952614 137.54 \n",
       "L 57.6669 137.54 \n",
       "L 57.6669 128.911429 \n",
       "L 39.952614 128.911429 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_4\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 57.6669 137.54 \n",
       "L 75.381185 137.54 \n",
       "L 75.381185 122.44 \n",
       "L 57.6669 122.44 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_5\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 75.381185 137.54 \n",
       "L 93.095471 137.54 \n",
       "L 93.095471 46.94 \n",
       "L 75.381185 46.94 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_6\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 93.095471 137.54 \n",
       "L 110.809757 137.54 \n",
       "L 110.809757 16.74 \n",
       "L 93.095471 16.74 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_7\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 110.809757 137.54 \n",
       "L 128.524042 137.54 \n",
       "L 128.524042 92.24 \n",
       "L 110.809757 92.24 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_8\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 128.524042 137.54 \n",
       "L 146.238328 137.54 \n",
       "L 146.238328 126.754286 \n",
       "L 128.524042 126.754286 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_9\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 146.238328 137.54 \n",
       "L 163.952614 137.54 \n",
       "L 163.952614 133.225714 \n",
       "L 146.238328 133.225714 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_10\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 163.952614 137.54 \n",
       "L 181.6669 137.54 \n",
       "L 181.6669 137.54 \n",
       "L 163.952614 137.54 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_11\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 181.6669 137.54 \n",
       "L 199.381185 137.54 \n",
       "L 199.381185 131.068571 \n",
       "L 181.6669 131.068571 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_12\">\n",
       "    <path clip-path=\"url(#pbd76cf1ee0)\" d=\"M 199.381185 137.54 \n",
       "L 217.095471 137.54 \n",
       "L 217.095471 135.382857 \n",
       "L 199.381185 135.382857 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_1\">\n",
       "    <g id=\"ytick_1\">\n",
       "     <g id=\"line2d_1\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L -3.5 0 \n",
       "\" id=\"mf24b69ec4d\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"135.362817\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_1\">\n",
       "      <!-- -2 -->\n",
       "      <defs>\n",
       "       <path d=\"M 25.5 19 \n",
       "L 25.5 23.375 \n",
       "L 0.40625 23.375 \n",
       "L 0.40625 19 \n",
       "z\n",
       "\" id=\"CMR17-45\"/>\n",
       "       <path d=\"M 41.703125 15.453125 \n",
       "L 39.90625 15.453125 \n",
       "C 38.90625 8.375 38.09375 7.171875 37.703125 6.5625 \n",
       "C 37.203125 5.765625 30 5.765625 28.59375 5.765625 \n",
       "L 9.40625 5.765625 \n",
       "C 13 9.671875 20 16.75 28.5 24.953125 \n",
       "C 34.59375 30.734375 41.703125 37.53125 41.703125 47.421875 \n",
       "C 41.703125 59.21875 32.296875 66 21.796875 66 \n",
       "C 10.796875 66 4.09375 56.3125 4.09375 47.34375 \n",
       "C 4.09375 43.4375 7 42.9375 8.203125 42.9375 \n",
       "C 9.203125 42.9375 12.203125 43.546875 12.203125 47.03125 \n",
       "C 12.203125 50.109375 9.59375 51 8.203125 51 \n",
       "C 7.59375 51 7 50.90625 6.59375 50.703125 \n",
       "C 8.5 59.21875 14.296875 63.40625 20.40625 63.40625 \n",
       "C 29.09375 63.40625 34.796875 56.515625 34.796875 47.421875 \n",
       "C 34.796875 38.734375 29.703125 31.25 24 24.75 \n",
       "L 4.09375 2.28125 \n",
       "L 4.09375 0 \n",
       "L 39.296875 0 \n",
       "z\n",
       "\" id=\"CMR17-50\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(21.467071 138.130218)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-45\"/>\n",
       "       <use transform=\"translate(30.075966 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_2\">\n",
       "     <g id=\"line2d_2\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"95.716497\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_2\">\n",
       "      <!-- 0 -->\n",
       "      <defs>\n",
       "       <path d=\"M 42 31.640625 \n",
       "C 42 37.75 41.90625 48.125 37.703125 56.109375 \n",
       "C 34 63.109375 28.09375 65.59375 22.90625 65.59375 \n",
       "C 18.09375 65.59375 12 63.40625 8.203125 56.203125 \n",
       "C 4.203125 48.71875 3.796875 39.4375 3.796875 31.640625 \n",
       "C 3.796875 25.953125 3.90625 17.28125 7 9.671875 \n",
       "C 11.296875 -0.609375 19 -2 22.90625 -2 \n",
       "C 27.5 -2 34.5 -0.109375 38.59375 9.375 \n",
       "C 41.59375 16.28125 42 24.359375 42 31.640625 \n",
       "M 22.90625 -0.40625 \n",
       "C 16.5 -0.40625 12.703125 5.078125 11.296875 12.6875 \n",
       "C 10.203125 18.5625 10.203125 27.15625 10.203125 32.75 \n",
       "C 10.203125 40.4375 10.203125 46.828125 11.5 52.921875 \n",
       "C 13.40625 61.390625 19 64 22.90625 64 \n",
       "C 27 64 32.296875 61.296875 34.203125 53.125 \n",
       "C 35.5 47.4375 35.59375 40.734375 35.59375 32.75 \n",
       "C 35.59375 26.25 35.59375 18.265625 34.40625 12.375 \n",
       "C 32.296875 1.484375 26.40625 -0.40625 22.90625 -0.40625 \n",
       "\" id=\"CMR17-48\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 98.483898)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_3\">\n",
       "     <g id=\"line2d_3\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"56.070176\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_3\">\n",
       "      <!-- 2 -->\n",
       "      <g transform=\"translate(24.28986 58.837577)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_4\">\n",
       "     <g id=\"line2d_4\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"16.423855\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_4\">\n",
       "      <!-- 4 -->\n",
       "      <defs>\n",
       "       <path d=\"M 33.59375 64.40625 \n",
       "C 33.59375 66.5 33.5 66.5 31.703125 66.5 \n",
       "L 2 19.59375 \n",
       "L 2 17 \n",
       "L 27.796875 17 \n",
       "L 27.796875 7.140625 \n",
       "C 27.796875 3.5 27.59375 2.5 20.59375 2.5 \n",
       "L 18.703125 2.5 \n",
       "L 18.703125 0 \n",
       "C 21.90625 0 27.296875 0 30.703125 0 \n",
       "C 34.09375 0 39.5 0 42.703125 0 \n",
       "L 42.703125 2.5 \n",
       "L 40.796875 2.5 \n",
       "C 33.796875 2.5 33.59375 3.5 33.59375 7.140625 \n",
       "L 33.59375 17 \n",
       "L 43.796875 17 \n",
       "L 43.796875 19.59375 \n",
       "L 33.59375 19.59375 \n",
       "z\n",
       "M 28.09375 57.859375 \n",
       "L 28.09375 19.59375 \n",
       "L 4 19.59375 \n",
       "z\n",
       "\" id=\"CMR17-52\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(24.28986 19.191256)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_5\">\n",
       "     <!-- Average NPA Score -->\n",
       "     <defs>\n",
       "      <path d=\"M 36.296875 68.40625 \n",
       "C 35.796875 69.703125 35.59375 70 34.59375 70 \n",
       "C 33.59375 70 33.40625 69.703125 32.90625 68.40625 \n",
       "L 11.703125 8.84375 \n",
       "C 9.796875 3.484375 5.796875 2.390625 2.296875 2.390625 \n",
       "L 2.296875 -0.203125 \n",
       "C 4.09375 0 8.09375 0 10 0 \n",
       "C 12.5 0 16.5 0 18.90625 -0.203125 \n",
       "L 18.90625 2.390625 \n",
       "C 14.203125 2.390625 13.5 5.640625 13.5 6.859375 \n",
       "C 13.5 7.765625 13.703125 8.28125 13.90625 8.890625 \n",
       "L 19.203125 24 \n",
       "L 45.40625 24 \n",
       "L 51.40625 6.90625 \n",
       "C 51.90625 5.703125 51.90625 5.5 51.90625 5.109375 \n",
       "C 51.90625 2.390625 47.59375 2.390625 45.703125 2.390625 \n",
       "L 45.703125 0 \n",
       "C 48.59375 0 53.796875 0 56.90625 0 \n",
       "C 59.5 0 64.5 0 66.90625 -0.203125 \n",
       "L 66.90625 2.390625 \n",
       "C 61.59375 2.390625 59.796875 2.390625 58.5 6.046875 \n",
       "z\n",
       "M 32.296875 60.828125 \n",
       "L 44.5 26.59375 \n",
       "L 20.09375 26.59375 \n",
       "z\n",
       "\" id=\"CMR17-65\"/>\n",
       "      <path d=\"M 39.09375 33.796875 \n",
       "C 40 36 41.59375 40 47.40625 40 \n",
       "L 47.40625 42.59375 \n",
       "C 45.5 42.59375 43 42.59375 41.09375 42.59375 \n",
       "C 39 42.59375 35.703125 42.59375 33.703125 42.6875 \n",
       "L 33.703125 40.09375 \n",
       "C 36.203125 39.890625 37.59375 38.703125 37.59375 36.109375 \n",
       "C 37.59375 35.203125 37.59375 35 37 33.609375 \n",
       "L 26 5.890625 \n",
       "L 14.09375 35.71875 \n",
       "C 13.5 37.015625 13.5 37.203125 13.5 37.609375 \n",
       "C 13.5 40 16.796875 40 18.296875 40 \n",
       "L 18.296875 42.6875 \n",
       "C 15.90625 42.59375 11.5 42.59375 9 42.59375 \n",
       "C 6.296875 42.59375 3.203125 42.59375 1 42.6875 \n",
       "L 1 40 \n",
       "C 4.796875 40 6.90625 40 8.09375 36.90625 \n",
       "L 22.59375 0.6875 \n",
       "C 23.203125 -0.796875 23.296875 -1 24.203125 -1 \n",
       "C 25.09375 -1 25.203125 -0.796875 25.796875 0.6875 \n",
       "z\n",
       "\" id=\"CMR17-118\"/>\n",
       "      <path d=\"M 38.09375 23 \n",
       "C 38.5 23.40625 38.5 23.609375 38.5 24.625 \n",
       "C 38.5 34.96875 33.09375 44 21.703125 44 \n",
       "C 11.09375 44 2.703125 33.890625 2.703125 21.609375 \n",
       "C 2.703125 8.609375 12.203125 -1 22.796875 -1 \n",
       "C 34 -1 38.40625 9.484375 38.40625 11.5625 \n",
       "C 38.40625 12.25 37.796875 12.25 37.59375 12.25 \n",
       "C 36.90625 12.25 36.796875 12.046875 36.40625 10.875 \n",
       "C 34.203125 4.140625 28.703125 0.796875 23.5 0.796875 \n",
       "C 19.203125 0.796875 14.90625 3.171875 12.203125 7.5 \n",
       "C 9.09375 12.546875 9.09375 18.375 9.09375 23 \n",
       "z\n",
       "M 9.203125 24.5 \n",
       "C 9.90625 39.140625 17.59375 42.40625 21.59375 42.40625 \n",
       "C 28.40625 42.40625 33 35.890625 33.09375 24.5 \n",
       "z\n",
       "\" id=\"CMR17-101\"/>\n",
       "      <path d=\"M 15 23.390625 \n",
       "C 15 33 19.09375 42 26.59375 42 \n",
       "C 27.296875 42 28 41.90625 28.703125 41.59375 \n",
       "C 28.703125 41.59375 26.5 40.890625 26.5 38.3125 \n",
       "C 26.5 35.90625 28.40625 34.90625 29.90625 34.90625 \n",
       "C 31.09375 34.90625 33.296875 35.609375 33.296875 38.40625 \n",
       "C 33.296875 41.609375 30.09375 43.59375 26.703125 43.59375 \n",
       "C 19.09375 43.59375 15.796875 36.203125 14.796875 32.703125 \n",
       "L 14.703125 32.703125 \n",
       "L 14.703125 43.59375 \n",
       "L 3.09375 42.5 \n",
       "L 3.09375 39.90625 \n",
       "C 9 39.90625 9.90625 39.296875 9.90625 34.5 \n",
       "L 9.90625 6.6875 \n",
       "C 9.90625 2.875 9.5 2.390625 3.09375 2.390625 \n",
       "L 3.09375 -0.203125 \n",
       "C 5.5 0 10.09375 0 12.703125 0 \n",
       "C 15.59375 0 20.703125 0 23.40625 -0.203125 \n",
       "L 23.40625 2.390625 \n",
       "C 16.203125 2.390625 15 2.390625 15 6.875 \n",
       "z\n",
       "\" id=\"CMR17-114\"/>\n",
       "      <path d=\"M 36 25.828125 \n",
       "C 36 32.375 36 35.84375 31.796875 39.734375 \n",
       "C 28.09375 43 23.796875 44 20.40625 44 \n",
       "C 12.5 44 6.796875 37.609375 6.796875 30.796875 \n",
       "C 6.796875 27 9.796875 27 10.40625 27 \n",
       "C 11.703125 27 14 27.78125 14 30.53125 \n",
       "C 14 33 12.09375 34.078125 10.40625 34.078125 \n",
       "C 10 34.078125 9.5 33.96875 9.203125 33.875 \n",
       "C 11.296875 40.5 16.703125 42.40625 20.203125 42.40625 \n",
       "C 25.203125 42.40625 30.703125 37.953125 30.703125 29.453125 \n",
       "L 30.703125 25 \n",
       "C 24.796875 25 17.703125 24.203125 12.09375 21.203125 \n",
       "C 5.796875 17.703125 4 12.703125 4 8.90625 \n",
       "C 4 1.203125 13 -1 18.296875 -1 \n",
       "C 23.796875 -1 28.90625 2.171875 31.09375 7.984375 \n",
       "C 31.296875 3.640625 34.09375 -0.28125 38.5 -0.28125 \n",
       "C 40.59375 -0.28125 45.90625 1.09375 45.90625 8.859375 \n",
       "L 45.90625 14.375 \n",
       "L 44.09375 14.375 \n",
       "L 44.09375 8.78125 \n",
       "C 44.09375 2.78125 41.40625 2 40.09375 2 \n",
       "C 36 2 36 7.15625 36 11.53125 \n",
       "z\n",
       "M 30.703125 13.59375 \n",
       "C 30.703125 4.953125 24.5 0.59375 19 0.59375 \n",
       "C 14 0.59375 10.09375 4.296875 10.09375 8.90625 \n",
       "C 10.09375 11.890625 11.40625 17.1875 17.203125 20.390625 \n",
       "C 22 23.09375 27.5 23.5 30.703125 23.5 \n",
       "z\n",
       "\" id=\"CMR17-97\"/>\n",
       "      <path d=\"M 11.09375 18.1875 \n",
       "C 13 16.6875 16.296875 15 20.296875 15 \n",
       "C 28.203125 15 35 21.203125 35 29.5 \n",
       "C 35 32.09375 34.203125 36.09375 31.203125 39.203125 \n",
       "C 34 42.0625 38.09375 43 40.40625 43 \n",
       "C 40.796875 43 41.40625 43 41.90625 42.71875 \n",
       "C 41.5 42.625 40.59375 42.234375 40.59375 40.703125 \n",
       "C 40.59375 39.484375 41.5 38.609375 42.796875 38.609375 \n",
       "C 44.296875 38.609375 45.09375 39.578125 45.09375 40.84375 \n",
       "C 45.09375 42.671875 43.59375 44.59375 40.40625 44.59375 \n",
       "C 36.40625 44.59375 32.90625 42.78125 30.09375 40.3125 \n",
       "C 27.09375 43.015625 23.5 44 20.296875 44 \n",
       "C 12.40625 44 5.59375 37.796875 5.59375 29.5 \n",
       "C 5.59375 23.78125 8.90625 20.1875 9.90625 19.1875 \n",
       "C 6.90625 15.6875 6.90625 11.609375 6.90625 11.109375 \n",
       "C 6.90625 8.515625 7.90625 4.71875 11.296875 2.625 \n",
       "C 6.09375 1.328125 2 -2.640625 2 -7.59375 \n",
       "C 2 -14.734375 11.5 -20 22.90625 -20 \n",
       "C 33.90625 -20 43.796875 -14.9375 43.796875 -7.484375 \n",
       "C 43.796875 6 29 6 21.296875 6 \n",
       "C 19 6 14.90625 6 14.40625 6.109375 \n",
       "C 11.296875 6.609375 9.203125 9.40625 9.203125 12.90625 \n",
       "C 9.203125 13.796875 9.203125 16 11.09375 18.1875 \n",
       "M 20.296875 16.6875 \n",
       "C 11.5 16.6875 11.5 27.5 11.5 29.515625 \n",
       "C 11.5 31.5 11.5 42.3125 20.296875 42.3125 \n",
       "C 29.09375 42.3125 29.09375 31.5 29.09375 29.515625 \n",
       "C 29.09375 27.5 29.09375 16.6875 20.296875 16.6875 \n",
       "M 22.90625 -18.296875 \n",
       "C 12.90625 -18.296875 6 -13.046875 6 -7.59375 \n",
       "C 6 -4.515625 7.703125 -1.640625 9.90625 -0.0625 \n",
       "C 12.40625 1.625 13.40625 1.625 20.203125 1.625 \n",
       "C 28.40625 1.625 39.796875 1.625 39.796875 -7.59375 \n",
       "C 39.796875 -13.046875 32.90625 -18.296875 22.90625 -18.296875 \n",
       "\" id=\"CMR17-103\"/>\n",
       "      <path d=\"M 20.5 66.40625 \n",
       "C 19.796875 67.5 19.703125 67.5 17.796875 67.5 \n",
       "L 5.09375 67.5 \n",
       "L 5.09375 65 \n",
       "L 7 65 \n",
       "C 11 65 12.796875 64.515625 13.09375 64.40625 \n",
       "L 13.09375 9.53125 \n",
       "C 13.09375 7.046875 13.09375 2.390625 5.09375 2.390625 \n",
       "L 5.09375 -0.203125 \n",
       "C 7.40625 0 11.59375 0 14.09375 0 \n",
       "C 16.59375 0 20.796875 0 23.09375 -0.203125 \n",
       "L 23.09375 2.390625 \n",
       "C 15.09375 2.390625 15.09375 7.046875 15.09375 9.53125 \n",
       "L 15.09375 63.015625 \n",
       "C 15.796875 62.421875 15.796875 62.21875 16.5 61.234375 \n",
       "L 53.59375 1.1875 \n",
       "C 54.296875 0 54.59375 0 55.09375 0 \n",
       "C 56.09375 0 56.09375 0.296875 56.09375 2.171875 \n",
       "L 56.09375 57.890625 \n",
       "C 56.09375 60.359375 56.09375 65 64.09375 65 \n",
       "L 64.09375 67.59375 \n",
       "C 61.796875 67.5 57.59375 67.5 55.09375 67.5 \n",
       "C 52.59375 67.5 48.40625 67.5 46.09375 67.59375 \n",
       "L 46.09375 65 \n",
       "C 54.09375 65 54.09375 60.34375 54.09375 57.859375 \n",
       "L 54.09375 11.921875 \n",
       "z\n",
       "\" id=\"CMR17-78\"/>\n",
       "      <path d=\"M 19.796875 31.90625 \n",
       "L 35.796875 31.90625 \n",
       "C 48.703125 31.90625 58.09375 40.28125 58.09375 49.75 \n",
       "C 58.09375 59.234375 48.90625 68 35.796875 68 \n",
       "L 5.203125 68 \n",
       "L 5.203125 65.40625 \n",
       "C 12.09375 65.40625 13.203125 65.40625 13.203125 60.921875 \n",
       "L 13.203125 6.96875 \n",
       "C 13.203125 2.5 12.09375 2.5 5.203125 2.5 \n",
       "L 5.203125 0 \n",
       "C 8.09375 0 13.40625 0 16.5 0 \n",
       "C 19.59375 0 24.90625 0 27.796875 0 \n",
       "L 27.796875 2.5 \n",
       "C 20.90625 2.5 19.796875 2.5 19.796875 6.96875 \n",
       "z\n",
       "M 19.59375 34 \n",
       "L 19.59375 61.53125 \n",
       "C 19.59375 64.921875 19.796875 65.40625 23.796875 65.40625 \n",
       "L 33.90625 65.40625 \n",
       "C 46.796875 65.40625 50.5 58.03125 50.5 49.75 \n",
       "C 50.5 40.6875 46.09375 34 33.90625 34 \n",
       "z\n",
       "\" id=\"CMR17-80\"/>\n",
       "      <path d=\"M 43.796875 67.03125 \n",
       "C 43.796875 68.8125 43.703125 68.921875 43.09375 68.921875 \n",
       "C 42.5 68.921875 42.296875 68.515625 41.90625 67.625 \n",
       "L 38.90625 61.046875 \n",
       "C 35.09375 66.515625 29.296875 69 23 69 \n",
       "C 12.796875 69 4.59375 60.953125 4.59375 50.828125 \n",
       "C 4.59375 47.75 5.296875 43.375 8.703125 39.21875 \n",
       "C 12.40625 34.65625 15.90625 33.859375 23.90625 31.765625 \n",
       "C 26.90625 30.96875 31.90625 29.671875 32.90625 29.28125 \n",
       "C 38.59375 26.90625 41.59375 20.953125 41.59375 15.390625 \n",
       "C 41.59375 8.234375 36.5 0.59375 27.796875 0.59375 \n",
       "C 17.09375 0.59375 7.09375 6.546875 6.40625 20.484375 \n",
       "C 6.296875 22.078125 6.203125 22.171875 5.5 22.171875 \n",
       "C 4.703125 22.171875 4.59375 22.078125 4.59375 20.078125 \n",
       "L 4.59375 -0.015625 \n",
       "C 4.59375 -1.796875 4.703125 -1.90625 5.296875 -1.90625 \n",
       "C 6 -1.90625 6.203125 -1.40625 7.59375 1.6875 \n",
       "C 7.703125 1.984375 7.703125 2.1875 9.59375 5.953125 \n",
       "C 14.59375 -0.609375 23 -2 27.796875 -2 \n",
       "C 38.703125 -2 46.40625 7.046875 46.40625 17.5625 \n",
       "C 46.40625 22.71875 44.5 26.6875 43.40625 28.484375 \n",
       "C 39.203125 34.84375 34.703125 36.03125 29.90625 37.21875 \n",
       "C 29.203125 37.53125 29 37.53125 23.40625 38.90625 \n",
       "C 18 40.296875 15.703125 41 13.203125 43.6875 \n",
       "C 10.40625 46.75 9.40625 49.9375 9.40625 53.109375 \n",
       "C 9.40625 59.5625 14.59375 66.609375 23.09375 66.609375 \n",
       "C 33.703125 66.609375 40.296875 59.15625 41.796875 46.765625 \n",
       "C 42.09375 44.96875 42.203125 44.875 42.90625 44.875 \n",
       "C 43.796875 44.875 43.796875 45.171875 43.796875 46.859375 \n",
       "z\n",
       "\" id=\"CMR17-83\"/>\n",
       "      <path d=\"M 34.90625 35 \n",
       "C 33 35 30.203125 35 30.203125 31.515625 \n",
       "C 30.203125 28.71875 32.5 27.921875 33.796875 27.921875 \n",
       "C 34.5 27.921875 37.40625 28.234375 37.40625 31.625 \n",
       "C 37.40625 38.546875 30.59375 43.796875 22.90625 43.796875 \n",
       "C 12.296875 43.796875 3.296875 34.03125 3.296875 21.296875 \n",
       "C 3.296875 8.0625 12.703125 -1 22.90625 -1 \n",
       "C 35.296875 -1 38.40625 10.578125 38.40625 11.6875 \n",
       "C 38.40625 12.09375 38.296875 12.390625 37.59375 12.390625 \n",
       "C 36.90625 12.390625 36.796875 12.296875 36.40625 10.984375 \n",
       "C 33.796875 2.8125 28.09375 0.796875 23.796875 0.796875 \n",
       "C 17.40625 0.796875 9.703125 6.671875 9.703125 21.390625 \n",
       "C 9.703125 36.53125 17.09375 42 23 42 \n",
       "C 26.90625 42 32.703125 40.203125 34.90625 35 \n",
       "\" id=\"CMR17-99\"/>\n",
       "      <path d=\"M 43.09375 21.1875 \n",
       "C 43.09375 34 33.796875 44 22.90625 44 \n",
       "C 12 44 2.703125 34 2.703125 21.1875 \n",
       "C 2.703125 8.609375 12 -1 22.90625 -1 \n",
       "C 33.796875 -1 43.09375 8.609375 43.09375 21.1875 \n",
       "M 22.90625 0.796875 \n",
       "C 18.203125 0.796875 14.203125 3.59375 11.90625 7.5 \n",
       "C 9.40625 11.984375 9.09375 17.59375 9.09375 22 \n",
       "C 9.09375 26.203125 9.296875 31.40625 11.90625 35.90625 \n",
       "C 13.90625 39.203125 17.796875 42.40625 22.90625 42.40625 \n",
       "C 27.40625 42.40625 31.203125 39.90625 33.59375 36.40625 \n",
       "C 36.703125 31.703125 36.703125 25.109375 36.703125 22 \n",
       "C 36.703125 18.09375 36.5 12.109375 33.796875 7.296875 \n",
       "C 31 2.703125 26.703125 0.796875 22.90625 0.796875 \n",
       "\" id=\"CMR17-111\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(15.529893 117.111621)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "      <use transform=\"translate(114.762892 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(155.248552 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(190.52932 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(236.219797 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(281.910274 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(352.471901 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(421.544331 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "      <use transform=\"translate(476.323376 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(575.471772 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(626.367142 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(666.852802 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(712.543279 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(747.824047 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_13\">\n",
       "    <path d=\"M 35.524042 137.54 \n",
       "L 35.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_14\">\n",
       "    <path d=\"M 221.524042 137.54 \n",
       "L 221.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_15\">\n",
       "    <path d=\"M 35.524042 137.54 \n",
       "L 221.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_16\">\n",
       "    <path d=\"M 35.524042 10.7 \n",
       "L 221.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_2\">\n",
       "   <g id=\"patch_17\">\n",
       "    <path d=\"M 221.524042 137.54 \n",
       "L 407.524042 137.54 \n",
       "L 407.524042 10.7 \n",
       "L 221.524042 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_1\">\n",
       "    <defs>\n",
       "     <path d=\"M 0 1.5 \n",
       "C 0.397805 1.5 0.77937 1.341951 1.06066 1.06066 \n",
       "C 1.341951 0.77937 1.5 0.397805 1.5 0 \n",
       "C 1.5 -0.397805 1.341951 -0.77937 1.06066 -1.06066 \n",
       "C 0.77937 -1.341951 0.397805 -1.5 0 -1.5 \n",
       "C -0.397805 -1.5 -0.77937 -1.341951 -1.06066 -1.06066 \n",
       "C -1.341951 -0.77937 -1.5 -0.397805 -1.5 0 \n",
       "C -1.5 0.397805 -1.341951 0.77937 -1.06066 1.06066 \n",
       "C -0.77937 1.341951 -0.397805 1.5 0 1.5 \n",
       "z\n",
       "\" id=\"m92d7d176f3\" style=\"stroke:#1f77b4;stroke-opacity:0.5;\"/>\n",
       "    </defs>\n",
       "    <g clip-path=\"url(#pa38e8881a1)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"117.338049\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"115.681036\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"104.883211\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"106.960498\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"75.873952\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"377.045051\" xlink:href=\"#m92d7d176f3\" y=\"57.909651\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"278.053454\" xlink:href=\"#m92d7d176f3\" y=\"70.919228\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"403.095471\" xlink:href=\"#m92d7d176f3\" y=\"108.209064\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"309.313958\" xlink:href=\"#m92d7d176f3\" y=\"115.500384\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"293.683706\" xlink:href=\"#m92d7d176f3\" y=\"65.593499\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"125.752789\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"96.145369\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"314.524042\" xlink:href=\"#m92d7d176f3\" y=\"134.52\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"288.473622\" xlink:href=\"#m92d7d176f3\" y=\"53.919551\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"89.054742\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"83.905848\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"66.111389\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"118.781562\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"93.846\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.271117\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.271117\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"288.473622\" xlink:href=\"#m92d7d176f3\" y=\"75.740003\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"94.021188\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"262.423202\" xlink:href=\"#m92d7d176f3\" y=\"97.990619\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"130.869955\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"96.802932\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"87.97072\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"101.148624\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"90.824339\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"262.423202\" xlink:href=\"#m92d7d176f3\" y=\"110.088975\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"74.686151\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"96.89711\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.923118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"103.952831\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"100.74621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"76.284625\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"96.192016\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"74.931908\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"93.429305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"110.103438\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"105.482297\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"109.999856\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"72.773006\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"95.716497\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"113.767215\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"257.213118\" xlink:href=\"#m92d7d176f3\" y=\"93.707909\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"92.987323\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"92.987323\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"94.630061\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"94.630061\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"95.769285\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"37.420881\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"113.245085\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"77.1621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"105.409102\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"86.023892\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.95095\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"36.598106\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"78.689982\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"88.881329\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"100.74621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"74.995668\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"257.213118\" xlink:href=\"#m92d7d176f3\" y=\"100.638972\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"88.209772\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"95.716497\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.590926\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"87.760505\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"125.780873\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"105.23147\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"83.930505\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"96.38352\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"95.049473\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"95.049473\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.977514\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"106.960498\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"91.02236\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"95.610919\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"75.806954\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.701891\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"37.420881\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"75.347815\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"75.932735\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"74.024504\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"76.068029\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"13.72\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"113.049744\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"78.950879\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.462273\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.462273\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"90.417471\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"98.084188\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_18\">\n",
       "    <path d=\"M 221.524042 137.54 \n",
       "L 221.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_19\">\n",
       "    <path d=\"M 407.524042 137.54 \n",
       "L 407.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_20\">\n",
       "    <path d=\"M 221.524042 137.54 \n",
       "L 407.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_21\">\n",
       "    <path d=\"M 221.524042 10.7 \n",
       "L 407.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_3\">\n",
       "   <g id=\"patch_22\">\n",
       "    <path d=\"M 407.524042 137.54 \n",
       "L 593.524042 137.54 \n",
       "L 593.524042 10.7 \n",
       "L 407.524042 10.7 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_2\">\n",
       "    <g clip-path=\"url(#p0198bef5da)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"117.338049\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"115.681036\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"104.883211\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"106.960498\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"75.873952\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"589.095471\" xlink:href=\"#m92d7d176f3\" y=\"57.909651\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"509.777774\" xlink:href=\"#m92d7d176f3\" y=\"70.919228\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"541.504853\" xlink:href=\"#m92d7d176f3\" y=\"108.209064\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"115.500384\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"530.92916\" xlink:href=\"#m92d7d176f3\" y=\"65.593499\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"125.752789\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"96.145369\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"536.217006\" xlink:href=\"#m92d7d176f3\" y=\"134.52\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"478.050695\" xlink:href=\"#m92d7d176f3\" y=\"53.919551\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"89.054742\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"83.905848\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"554.724469\" xlink:href=\"#m92d7d176f3\" y=\"66.111389\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"118.781562\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"93.846\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"98.271117\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"98.271117\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"451.611462\" xlink:href=\"#m92d7d176f3\" y=\"75.740003\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"491.270311\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"491.270311\" xlink:href=\"#m92d7d176f3\" y=\"94.021188\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"475.406772\" xlink:href=\"#m92d7d176f3\" y=\"97.990619\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"130.869955\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"96.802932\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"87.97072\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"101.148624\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"90.824339\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"446.323616\" xlink:href=\"#m92d7d176f3\" y=\"110.088975\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"433.104\" xlink:href=\"#m92d7d176f3\" y=\"74.686151\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"96.89711\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"98.923118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"103.952831\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"100.74621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"76.284625\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"96.192016\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"74.931908\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"93.429305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"110.103438\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"105.482297\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"109.999856\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"438.391846\" xlink:href=\"#m92d7d176f3\" y=\"72.773006\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"95.716497\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"443.679693\" xlink:href=\"#m92d7d176f3\" y=\"113.767215\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"441.035769\" xlink:href=\"#m92d7d176f3\" y=\"93.707909\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"92.987323\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"92.987323\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"94.630061\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"94.630061\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"95.769285\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"37.420881\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"113.245085\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"77.1621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"105.409102\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"86.023892\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"99.95095\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"441.035769\" xlink:href=\"#m92d7d176f3\" y=\"36.598106\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"78.689982\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"433.104\" xlink:href=\"#m92d7d176f3\" y=\"88.881329\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"91.597393\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"446.323616\" xlink:href=\"#m92d7d176f3\" y=\"100.74621\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"74.995668\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"100.638972\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"88.209772\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"95.716497\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"88.631563\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"99.258964\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"98.590926\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"92.17403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"84.667305\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"87.760505\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"125.780873\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"105.23147\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"83.930505\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"97.586994\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"93.000433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"98.43256\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"96.38352\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"95.049473\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"95.049473\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"427.816153\" xlink:href=\"#m92d7d176f3\" y=\"103.977514\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"106.960498\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"91.02236\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"95.610919\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"75.806954\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"84.701891\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"37.420881\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"98.003688\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"75.347815\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"90.28437\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"75.932735\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"435.747923\" xlink:href=\"#m92d7d176f3\" y=\"74.024504\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"76.068029\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"443.679693\" xlink:href=\"#m92d7d176f3\" y=\"13.72\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"113.049744\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"435.747923\" xlink:href=\"#m92d7d176f3\" y=\"78.950879\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"103.462273\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"103.462273\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"90.417471\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"98.084188\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"103.672488\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_23\">\n",
       "    <path d=\"M 407.524042 137.54 \n",
       "L 407.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_24\">\n",
       "    <path d=\"M 593.524042 137.54 \n",
       "L 593.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_25\">\n",
       "    <path d=\"M 407.524042 137.54 \n",
       "L 593.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_26\">\n",
       "    <path d=\"M 407.524042 10.7 \n",
       "L 593.524042 10.7 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_4\">\n",
       "   <g id=\"patch_27\">\n",
       "    <path d=\"M 35.524042 264.38 \n",
       "L 221.524042 264.38 \n",
       "L 221.524042 137.54 \n",
       "L 35.524042 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_3\">\n",
       "    <g clip-path=\"url(#pbf28c37d7e)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"65.148474\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"67.578342\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"83.412427\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"80.366264\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.952022\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"152.295132\" xlink:href=\"#m92d7d176f3\" y=\"158.324706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"133.2177\" xlink:href=\"#m92d7d176f3\" y=\"225.830588\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"78.535348\" xlink:href=\"#m92d7d176f3\" y=\"140.56\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"67.843253\" xlink:href=\"#m92d7d176f3\" y=\"204.512941\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"141.027426\" xlink:href=\"#m92d7d176f3\" y=\"215.171765\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"52.808978\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.225725\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"39.952614\" xlink:href=\"#m92d7d176f3\" y=\"200.96\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"158.146272\" xlink:href=\"#m92d7d176f3\" y=\"218.724706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"106.623522\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"114.173934\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"140.267985\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"63.031685\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.597553\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.108496\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.108496\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.148447\" xlink:href=\"#m92d7d176f3\" y=\"218.724706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.340655\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.519823\" xlink:href=\"#m92d7d176f3\" y=\"236.489412\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"45.305092\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.261465\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"108.213147\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"88.88888\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.02856\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.77862\" xlink:href=\"#m92d7d176f3\" y=\"236.489412\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.69383\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.12336\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.152393\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"84.77675\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.478986\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.349805\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.15732\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.333449\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.208599\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.757412\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.533919\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.909306\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"130.499294\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.854629\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"70.384797\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.800051\" xlink:href=\"#m92d7d176f3\" y=\"240.042353\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.856728\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.856728\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"98.447793\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"98.447793\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.777219\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"182.340158\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"71.150455\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"124.063063\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.641254\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"111.068004\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"90.645165\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"183.546688\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"121.822555\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"106.877817\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.478986\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.23995\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.63624\" xlink:href=\"#m92d7d176f3\" y=\"240.042353\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.862598\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.854629\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.639524\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"108.521409\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"52.767796\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.901735\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"114.137777\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.876496\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.832762\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.832762\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"84.740555\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"80.366264\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"103.738178\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.009449\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.050268\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.006605\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"182.340158\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.723557\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.865822\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"128.664079\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.667424\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"217.095471\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"71.436906\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"121.439972\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.496111\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.496111\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.625196\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.382612\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_2\">\n",
       "    <g id=\"ytick_5\">\n",
       "     <g id=\"line2d_5\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"229.383529\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_6\">\n",
       "      <!-- $10$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 26.59375 63.40625 \n",
       "C 26.59375 65.5 26.5 65.5 25.09375 65.5 \n",
       "C 21.203125 61.1875 15.296875 59.796875 9.703125 59.796875 \n",
       "C 9.40625 59.796875 8.90625 59.796875 8.796875 59.5 \n",
       "C 8.703125 59.296875 8.703125 59.09375 8.703125 57 \n",
       "C 11.796875 57 17 57.59375 21 59.984375 \n",
       "L 21 7.203125 \n",
       "C 21 3.6875 20.796875 2.5 12.203125 2.5 \n",
       "L 9.203125 2.5 \n",
       "L 9.203125 0 \n",
       "C 14 0 19 0 23.796875 0 \n",
       "C 28.59375 0 33.59375 0 38.40625 0 \n",
       "L 38.40625 2.5 \n",
       "L 35.40625 2.5 \n",
       "C 26.796875 2.5 26.59375 3.59375 26.59375 7.15625 \n",
       "z\n",
       "\" id=\"CMR17-49\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.055677 232.15093)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_6\">\n",
       "     <g id=\"line2d_6\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"193.854118\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_7\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(20.055677 196.621519)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_7\">\n",
       "     <g id=\"line2d_7\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"158.324706\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_8\">\n",
       "      <!-- $30$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 22.09375 33.703125 \n",
       "C 31 33.703125 34.90625 25.96875 34.90625 17.03125 \n",
       "C 34.90625 5 28.5 0.390625 22.703125 0.390625 \n",
       "C 17.40625 0.390625 8.796875 3.03125 6.09375 10.828125 \n",
       "C 6.59375 10.625 7.09375 10.625 7.59375 10.625 \n",
       "C 10 10.625 11.796875 12.21875 11.796875 14.8125 \n",
       "C 11.796875 17.703125 9.59375 19 7.59375 19 \n",
       "C 5.90625 19 3.296875 18.203125 3.296875 14.46875 \n",
       "C 3.296875 5.234375 12.296875 -2 22.90625 -2 \n",
       "C 34 -2 42.5 6.71875 42.5 16.953125 \n",
       "C 42.5 26.671875 34.5 33.703125 25 34.796875 \n",
       "C 32.59375 36.375 39.90625 43.0625 39.90625 52.015625 \n",
       "C 39.90625 59.6875 32 65.296875 23 65.296875 \n",
       "C 13.90625 65.296875 5.90625 59.828125 5.90625 52.015625 \n",
       "C 5.90625 48.59375 8.5 48 9.796875 48 \n",
       "C 11.90625 48 13.703125 49.296875 13.703125 51.890625 \n",
       "C 13.703125 54.46875 11.90625 55.765625 9.796875 55.765625 \n",
       "C 9.40625 55.765625 8.90625 55.765625 8.5 55.578125 \n",
       "C 11.40625 61.859375 19.296875 63 22.796875 63 \n",
       "C 26.296875 63 32.90625 61.328125 32.90625 51.875 \n",
       "C 32.90625 49.109375 32.5 44.1875 29.09375 39.84375 \n",
       "C 26.09375 36 22.703125 36 19.40625 35.6875 \n",
       "C 18.90625 35.6875 16.59375 35.453125 16.203125 35.453125 \n",
       "C 15.5 35.359375 15.09375 35.265625 15.09375 34.5 \n",
       "C 15.09375 33.796875 15.203125 33.703125 17.203125 33.703125 \n",
       "z\n",
       "\" id=\"CMR17-51\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.055677 161.092107)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_9\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <defs>\n",
       "      <path d=\"M 29.40625 42.078125 \n",
       "L 29.40625 39.5 \n",
       "C 35.296875 39.5 36.203125 38.890625 36.203125 34.15625 \n",
       "L 36.203125 16.125 \n",
       "C 36.203125 7.8125 31.703125 0.59375 24.296875 0.59375 \n",
       "C 16.09375 0.59375 15.40625 5.4375 15.40625 10.59375 \n",
       "L 15.40625 43.171875 \n",
       "L 3.296875 42.078125 \n",
       "L 3.296875 39.5 \n",
       "C 7.296875 39.5 10 39.5 10.09375 35.53125 \n",
       "L 10.09375 16.53125 \n",
       "C 10.09375 9.890625 10.09375 5.640625 12.703125 2.859375 \n",
       "C 14 1.578125 16.5 -1 23.703125 -1 \n",
       "C 32.203125 -1 35.5 5.890625 36.296875 7.90625 \n",
       "L 36.40625 7.90625 \n",
       "L 36.40625 -0.984375 \n",
       "L 48.296875 -0.984375 \n",
       "L 48.296875 2 \n",
       "C 42.40625 2 41.5 2.59375 41.5 7.390625 \n",
       "L 41.5 43.171875 \n",
       "z\n",
       "\" id=\"CMR17-117\"/>\n",
       "      <path d=\"M 67.59375 30.203125 \n",
       "C 67.59375 35.1875 66.703125 43.59375 54.796875 43.59375 \n",
       "C 48 43.59375 43.296875 39 41.5 33.609375 \n",
       "L 41.40625 33.609375 \n",
       "C 40.203125 41.8125 34.296875 43.59375 28.703125 43.59375 \n",
       "C 20.796875 43.59375 16.703125 37.5 15.203125 33.5 \n",
       "L 15.09375 33.5 \n",
       "L 15.09375 43.59375 \n",
       "L 3.296875 42.5 \n",
       "L 3.296875 39.90625 \n",
       "C 9.203125 39.90625 10.09375 39.296875 10.09375 34.5 \n",
       "L 10.09375 6.6875 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "L 41.5 25.90625 \n",
       "C 41.5 35.109375 47.09375 42 54.09375 42 \n",
       "C 61.5 42 62.296875 35.40625 62.296875 30.59375 \n",
       "L 62.296875 6.6875 \n",
       "C 62.296875 2.875 61.90625 2.390625 55.5 2.390625 \n",
       "L 55.5 -0.203125 \n",
       "C 57.90625 0 62.296875 0 64.90625 0 \n",
       "C 67.5 0 72 0 74.40625 -0.203125 \n",
       "L 74.40625 2.390625 \n",
       "C 68 2.390625 67.59375 2.78125 67.59375 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-109\"/>\n",
       "      <path d=\"M 15 68.6875 \n",
       "L 3.09375 67.59375 \n",
       "L 3.09375 65 \n",
       "C 9 65 9.90625 64.40625 9.90625 59.640625 \n",
       "L 9.90625 -0.203125 \n",
       "L 11.703125 -0.203125 \n",
       "L 14.59375 6.890625 \n",
       "C 17.5 2.203125 21.796875 -1 27.59375 -1 \n",
       "C 37.59375 -1 47.703125 7.8125 47.703125 21.40625 \n",
       "C 47.703125 34.1875 38.703125 43.59375 28.59375 43.59375 \n",
       "C 22.40625 43.59375 18 40.296875 15 36.40625 \n",
       "z\n",
       "M 15.203125 31.3125 \n",
       "C 15.203125 33.09375 15.203125 33.40625 16.59375 35.5 \n",
       "C 20.5 41.515625 26.09375 42 28 42 \n",
       "C 31 42 41.296875 40.390625 41.296875 21.5 \n",
       "C 41.296875 1.6875 29.5 0.59375 27.09375 0.59375 \n",
       "C 24 0.59375 19.5 1.78125 16.296875 7.59375 \n",
       "C 15.203125 9.5 15.203125 9.703125 15.203125 11.484375 \n",
       "z\n",
       "\" id=\"CMR17-98\"/>\n",
       "      <path d=\"M 54.40625 67.578125 \n",
       "L 5.09375 67.578125 \n",
       "L 5.09375 65 \n",
       "C 12 65 13.09375 65 13.09375 60.53125 \n",
       "L 13.09375 6.96875 \n",
       "C 13.09375 2.5 12 2.5 5.09375 2.5 \n",
       "L 5.09375 0 \n",
       "C 8 0 13.59375 0 16.703125 0 \n",
       "C 20.796875 0 25.59375 0 29.703125 0 \n",
       "L 29.703125 2.5 \n",
       "L 27.5 2.5 \n",
       "C 19.90625 2.5 19.703125 3.484375 19.703125 7.0625 \n",
       "L 19.703125 32.421875 \n",
       "L 28.703125 32.421875 \n",
       "C 38 32.421875 38.90625 29.140625 38.90625 20.8125 \n",
       "L 40.703125 20.8125 \n",
       "L 40.703125 46.671875 \n",
       "L 38.90625 46.671875 \n",
       "C 38.90625 38.28125 38 35 28.703125 35 \n",
       "L 19.703125 35 \n",
       "L 19.703125 61.109375 \n",
       "C 19.703125 64.5 19.90625 65 23.90625 65 \n",
       "L 36.796875 65 \n",
       "C 51.90625 65 53.59375 58.828125 55 45.5625 \n",
       "L 56.796875 45.5625 \n",
       "z\n",
       "\" id=\"CMR17-70\"/>\n",
       "      <path d=\"M 15.5 61.078125 \n",
       "C 15.5 63.265625 13.703125 65.15625 11.40625 65.15625 \n",
       "C 9.203125 65.15625 7.296875 63.375 7.296875 61.078125 \n",
       "C 7.296875 58.890625 9.09375 57 11.40625 57 \n",
       "C 13.59375 57 15.5 58.796875 15.5 61.078125 \n",
       "M 3.796875 42.28125 \n",
       "L 3.796875 39.703125 \n",
       "C 9.40625 39.703125 10.203125 39.09375 10.203125 34.328125 \n",
       "L 10.203125 6.671875 \n",
       "C 10.203125 2.875 9.796875 2.390625 3.40625 2.390625 \n",
       "L 3.40625 -0.203125 \n",
       "C 5.796875 0 10.09375 0 12.59375 0 \n",
       "C 15 0 19.09375 0 21.40625 -0.203125 \n",
       "L 21.40625 2.390625 \n",
       "C 15.5 2.390625 15.296875 2.984375 15.296875 6.5625 \n",
       "L 15.296875 43.375 \n",
       "z\n",
       "\" id=\"CMR17-105\"/>\n",
       "      <path d=\"M 30.90625 42.140625 \n",
       "C 30.90625 43.953125 30.796875 44.046875 30.203125 44.046875 \n",
       "C 29.796875 44.046875 29.703125 43.953125 28.5 42.453125 \n",
       "C 28.203125 42.046875 27.296875 41.03125 27 40.640625 \n",
       "C 23.796875 44.046875 19.296875 44.046875 17.59375 44.046875 \n",
       "C 6.5 44.046875 2.5 38.234375 2.5 32.4375 \n",
       "C 2.5 23.40625 12.703125 21.328125 15.59375 20.703125 \n",
       "C 21.90625 19.421875 24.09375 19.015625 26.203125 17.203125 \n",
       "C 27.5 16.015625 29.703125 13.8125 29.703125 10.21875 \n",
       "C 29.703125 6 27.296875 0.59375 18.09375 0.59375 \n",
       "C 9.40625 0.59375 6.296875 7.1875 4.5 15.953125 \n",
       "C 4.203125 17.34375 4.203125 17.453125 3.40625 17.453125 \n",
       "C 2.59375 17.453125 2.5 17.34375 2.5 15.359375 \n",
       "L 2.5 0.984375 \n",
       "C 2.5 -0.796875 2.59375 -0.90625 3.203125 -0.90625 \n",
       "C 3.703125 -0.90625 3.796875 -0.796875 4.296875 0 \n",
       "C 4.90625 0.890625 6.40625 3.296875 7 4.3125 \n",
       "C 9 1.609375 12.5 -1 18.09375 -1 \n",
       "C 28 -1 33.296875 4.421875 33.296875 12.25 \n",
       "C 33.296875 17.375 30.59375 20.078125 29.296875 21.28125 \n",
       "C 26.296875 24.390625 22.796875 25.09375 18.59375 25.890625 \n",
       "C 13.09375 27.109375 6.09375 28.515625 6.09375 34.640625 \n",
       "C 6.09375 37.234375 7.5 42.765625 17.59375 42.765625 \n",
       "C 28.296875 42.765625 28.90625 32.71875 29.09375 29.5 \n",
       "C 29.203125 29 29.703125 28.890625 30 28.890625 \n",
       "C 30.90625 28.890625 30.90625 29.203125 30.90625 30.90625 \n",
       "z\n",
       "\" id=\"CMR17-115\"/>\n",
       "      <path d=\"M 15.09375 40 \n",
       "L 29.09375 40 \n",
       "L 29.09375 42.578125 \n",
       "L 15.09375 42.578125 \n",
       "L 15.09375 61 \n",
       "L 13.296875 61 \n",
       "C 13.09375 50.703125 9.59375 41.6875 1.09375 41.6875 \n",
       "L 1.09375 40 \n",
       "L 9.796875 40 \n",
       "L 9.796875 12.046875 \n",
       "C 9.796875 10.15625 9.796875 -1 21.40625 -1 \n",
       "C 27.296875 -1 30.703125 4.78125 30.703125 12.140625 \n",
       "L 30.703125 17.8125 \n",
       "L 28.90625 17.8125 \n",
       "L 28.90625 12.234375 \n",
       "C 28.90625 5.375 26.203125 0.796875 22 0.796875 \n",
       "C 19.09375 0.796875 15.09375 2.78125 15.09375 11.84375 \n",
       "z\n",
       "\" id=\"CMR17-116\"/>\n",
       "      <path d=\"M 41.5 30.203125 \n",
       "C 41.5 35.296875 40.5 43.59375 28.703125 43.59375 \n",
       "C 20.5 43.59375 16.5 37 15.296875 33.703125 \n",
       "L 15.203125 33.703125 \n",
       "L 15.203125 68.6875 \n",
       "L 3.296875 67.59375 \n",
       "L 3.296875 65 \n",
       "C 9.203125 65 10.09375 64.40625 10.09375 59.640625 \n",
       "L 10.09375 6.65625 \n",
       "C 10.09375 2.875 9.703125 2.390625 3.296875 2.390625 \n",
       "L 3.296875 -0.203125 \n",
       "C 5.703125 0 10.09375 0 12.703125 0 \n",
       "C 15.296875 0 19.796875 0 22.203125 -0.203125 \n",
       "L 22.203125 2.390625 \n",
       "C 15.796875 2.390625 15.40625 2.78125 15.40625 6.6875 \n",
       "L 15.40625 25.90625 \n",
       "C 15.40625 35.109375 21 42 28 42 \n",
       "C 35.40625 42 36.203125 35.40625 36.203125 30.59375 \n",
       "L 36.203125 6.6875 \n",
       "C 36.203125 2.875 35.796875 2.390625 29.40625 2.390625 \n",
       "L 29.40625 -0.203125 \n",
       "C 31.796875 0 36.203125 0 38.796875 0 \n",
       "C 41.40625 0 45.90625 0 48.296875 -0.203125 \n",
       "L 48.296875 2.390625 \n",
       "C 41.90625 2.390625 41.5 2.78125 41.5 6.6875 \n",
       "z\n",
       "\" id=\"CMR17-104\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 254.121857)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "      <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(413.608985 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(438.480044 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(473.760812 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      <use transform=\"translate(509.562149 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      <use transform=\"translate(574.918883 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(643.991313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(684.476974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(709.348032 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(755.038509 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(803.331425 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(856.829234 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(902.519711 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(937.800478 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_28\">\n",
       "    <path d=\"M 35.524042 264.38 \n",
       "L 35.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_29\">\n",
       "    <path d=\"M 221.524042 264.38 \n",
       "L 221.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_30\">\n",
       "    <path d=\"M 35.524042 264.38 \n",
       "L 221.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_31\">\n",
       "    <path d=\"M 35.524042 137.54 \n",
       "L 221.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_5\">\n",
       "   <g id=\"patch_32\">\n",
       "    <path d=\"M 221.524042 264.38 \n",
       "L 407.524042 264.38 \n",
       "L 407.524042 137.54 \n",
       "L 221.524042 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_33\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 225.952614 264.38 \n",
       "L 243.6669 264.38 \n",
       "L 243.6669 143.58 \n",
       "L 225.952614 143.58 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_34\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 243.6669 264.38 \n",
       "L 261.381185 264.38 \n",
       "L 261.381185 253.306667 \n",
       "L 243.6669 253.306667 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_35\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 261.381185 264.38 \n",
       "L 279.095471 264.38 \n",
       "L 279.095471 261.36 \n",
       "L 261.381185 261.36 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_36\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 279.095471 264.38 \n",
       "L 296.809757 264.38 \n",
       "L 296.809757 261.36 \n",
       "L 279.095471 261.36 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_37\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 296.809757 264.38 \n",
       "L 314.524042 264.38 \n",
       "L 314.524042 263.373333 \n",
       "L 296.809757 263.373333 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_38\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 314.524042 264.38 \n",
       "L 332.238328 264.38 \n",
       "L 332.238328 263.373333 \n",
       "L 314.524042 263.373333 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_39\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 332.238328 264.38 \n",
       "L 349.952614 264.38 \n",
       "L 349.952614 264.38 \n",
       "L 332.238328 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_40\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 349.952614 264.38 \n",
       "L 367.6669 264.38 \n",
       "L 367.6669 264.38 \n",
       "L 349.952614 264.38 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_41\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 367.6669 264.38 \n",
       "L 385.381185 264.38 \n",
       "L 385.381185 263.373333 \n",
       "L 367.6669 263.373333 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_42\">\n",
       "    <path clip-path=\"url(#pfdbcd8ecd9)\" d=\"M 385.381185 264.38 \n",
       "L 403.095471 264.38 \n",
       "L 403.095471 263.373333 \n",
       "L 385.381185 263.373333 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_43\">\n",
       "    <path d=\"M 221.524042 264.38 \n",
       "L 221.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_44\">\n",
       "    <path d=\"M 407.524042 264.38 \n",
       "L 407.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_45\">\n",
       "    <path d=\"M 221.524042 264.38 \n",
       "L 407.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_46\">\n",
       "    <path d=\"M 221.524042 137.54 \n",
       "L 407.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_6\">\n",
       "   <g id=\"patch_47\">\n",
       "    <path d=\"M 407.524042 264.38 \n",
       "L 593.524042 264.38 \n",
       "L 593.524042 137.54 \n",
       "L 407.524042 137.54 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_4\">\n",
       "    <g clip-path=\"url(#pf631313b9b)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"589.095471\" xlink:href=\"#m92d7d176f3\" y=\"158.324706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"509.777774\" xlink:href=\"#m92d7d176f3\" y=\"225.830588\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"541.504853\" xlink:href=\"#m92d7d176f3\" y=\"140.56\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"204.512941\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"530.92916\" xlink:href=\"#m92d7d176f3\" y=\"215.171765\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"536.217006\" xlink:href=\"#m92d7d176f3\" y=\"200.96\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"478.050695\" xlink:href=\"#m92d7d176f3\" y=\"218.724706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"554.724469\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"496.558158\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"451.611462\" xlink:href=\"#m92d7d176f3\" y=\"218.724706\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"491.270311\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"491.270311\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"475.406772\" xlink:href=\"#m92d7d176f3\" y=\"236.489412\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"446.323616\" xlink:href=\"#m92d7d176f3\" y=\"236.489412\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"433.104\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"438.391846\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"443.679693\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"441.035769\" xlink:href=\"#m92d7d176f3\" y=\"240.042353\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"441.035769\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"433.104\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"446.323616\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"240.042353\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"552.080546\" xlink:href=\"#m92d7d176f3\" y=\"250.701176\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"485.982465\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"483.338541\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"422.528307\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"493.914234\" xlink:href=\"#m92d7d176f3\" y=\"247.148235\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"488.626388\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"427.816153\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"425.17223\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"419.884384\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"435.747923\" xlink:href=\"#m92d7d176f3\" y=\"254.254118\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"443.679693\" xlink:href=\"#m92d7d176f3\" y=\"243.595294\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"430.460077\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"435.747923\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"417.24046\" xlink:href=\"#m92d7d176f3\" y=\"257.807059\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"411.952614\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"414.596537\" xlink:href=\"#m92d7d176f3\" y=\"261.36\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_48\">\n",
       "    <path d=\"M 407.524042 264.38 \n",
       "L 407.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_49\">\n",
       "    <path d=\"M 593.524042 264.38 \n",
       "L 593.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_50\">\n",
       "    <path d=\"M 407.524042 264.38 \n",
       "L 593.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_51\">\n",
       "    <path d=\"M 407.524042 137.54 \n",
       "L 593.524042 137.54 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_7\">\n",
       "   <g id=\"patch_52\">\n",
       "    <path d=\"M 35.524042 391.22 \n",
       "L 221.524042 391.22 \n",
       "L 221.524042 264.38 \n",
       "L 35.524042 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_5\">\n",
       "    <g clip-path=\"url(#p7eaf6073cb)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"65.148474\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"67.578342\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"83.412427\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"80.366264\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.952022\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"152.295132\" xlink:href=\"#m92d7d176f3\" y=\"267.4\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"133.2177\" xlink:href=\"#m92d7d176f3\" y=\"321.489552\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"78.535348\" xlink:href=\"#m92d7d176f3\" y=\"299.853731\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"67.843253\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"141.027426\" xlink:href=\"#m92d7d176f3\" y=\"307.065672\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"52.808978\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.225725\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"39.952614\" xlink:href=\"#m92d7d176f3\" y=\"303.459701\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"158.146272\" xlink:href=\"#m92d7d176f3\" y=\"343.125373\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"106.623522\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"114.173934\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"140.267985\" xlink:href=\"#m92d7d176f3\" y=\"290.838806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"63.031685\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.597553\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.108496\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.108496\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.148447\" xlink:href=\"#m92d7d176f3\" y=\"361.155224\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"334.110448\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.340655\" xlink:href=\"#m92d7d176f3\" y=\"334.110448\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.519823\" xlink:href=\"#m92d7d176f3\" y=\"344.928358\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"45.305092\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.261465\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"108.213147\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"88.88888\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.02856\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.77862\" xlink:href=\"#m92d7d176f3\" y=\"364.761194\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.69383\" xlink:href=\"#m92d7d176f3\" y=\"373.776119\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.12336\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.152393\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"84.77675\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.478986\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.349805\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.15732\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.333449\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.208599\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.757412\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.533919\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"75.909306\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"130.499294\" xlink:href=\"#m92d7d176f3\" y=\"370.170149\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.854629\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"70.384797\" xlink:href=\"#m92d7d176f3\" y=\"366.564179\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"99.800051\" xlink:href=\"#m92d7d176f3\" y=\"368.367164\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.856728\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.856728\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"98.447793\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"98.447793\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.777219\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"182.340158\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"71.150455\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"124.063063\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.641254\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"111.068004\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"90.645165\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"183.546688\" xlink:href=\"#m92d7d176f3\" y=\"368.367164\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"121.822555\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"106.877817\" xlink:href=\"#m92d7d176f3\" y=\"373.776119\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.894942\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.478986\" xlink:href=\"#m92d7d176f3\" y=\"364.761194\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"127.23995\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"89.63624\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.862598\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"96.854629\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"107.244078\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"91.659905\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.639524\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"102.049354\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.057323\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"108.521409\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"52.767796\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"82.901735\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"114.137777\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"94.111705\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"100.837504\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"92.871754\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"95.876496\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.832762\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.832762\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"84.740555\" xlink:href=\"#m92d7d176f3\" y=\"377.38209\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"80.366264\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"103.738178\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"97.009449\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.050268\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"113.006605\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"182.340158\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.500659\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"126.723557\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.820378\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.865822\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"128.664079\" xlink:href=\"#m92d7d176f3\" y=\"371.973134\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"125.667424\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"217.095471\" xlink:href=\"#m92d7d176f3\" y=\"366.564179\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"71.436906\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"121.439972\" xlink:href=\"#m92d7d176f3\" y=\"371.973134\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.496111\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.496111\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"104.625196\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"93.382612\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"85.187849\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_3\">\n",
       "    <g id=\"xtick_1\">\n",
       "     <g id=\"line2d_8\">\n",
       "      <defs>\n",
       "       <path d=\"M 0 0 \n",
       "L 0 3.5 \n",
       "\" id=\"m82ad270b99\" style=\"stroke:#000000;stroke-width:0.8;\"/>\n",
       "      </defs>\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"38.716694\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_10\">\n",
       "      <!-- $-2$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 65.90625 23 \n",
       "C 67.59375 23 69.40625 23 69.40625 25 \n",
       "C 69.40625 27 67.59375 27 65.90625 27 \n",
       "L 11.796875 27 \n",
       "C 10.09375 27 8.296875 27 8.296875 25 \n",
       "C 8.296875 23 10.09375 23 11.796875 23 \n",
       "z\n",
       "\" id=\"CMSY10-0\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(40.709222 409.04069)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMSY10-0\"/>\n",
       "       <use transform=\"translate(77.487468 0)scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_2\">\n",
       "     <g id=\"line2d_9\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"96.854629\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_11\">\n",
       "      <!-- $0$ -->\n",
       "      <g transform=\"translate(98.847157 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_3\">\n",
       "     <g id=\"line2d_10\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"154.992564\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_12\">\n",
       "      <!-- $2$ -->\n",
       "      <g transform=\"translate(156.985092 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_4\">\n",
       "     <g id=\"line2d_11\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"213.130499\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_13\">\n",
       "      <!-- $4$ -->\n",
       "      <g transform=\"translate(215.123027 402.454183)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_14\">\n",
       "     <!-- Average NPA Score -->\n",
       "     <g transform=\"translate(85.532421 419.959188)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-118\"/>\n",
       "      <use transform=\"translate(114.762892 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(155.248552 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(190.52932 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(236.219797 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(281.910274 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(352.471901 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(421.544331 0)scale(0.996264)\" xlink:href=\"#CMR17-80\"/>\n",
       "      <use transform=\"translate(476.323376 0)scale(0.996264)\" xlink:href=\"#CMR17-65\"/>\n",
       "      <use transform=\"translate(575.471772 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(626.367142 0)scale(0.996264)\" xlink:href=\"#CMR17-99\"/>\n",
       "      <use transform=\"translate(666.852802 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(712.543279 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(747.824047 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_4\">\n",
       "    <g id=\"ytick_8\">\n",
       "     <g id=\"line2d_12\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"355.746269\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_15\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(20.055677 358.51367)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_9\">\n",
       "     <g id=\"line2d_13\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"319.686567\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_16\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(20.055677 322.453968)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"ytick_10\">\n",
       "     <g id=\"line2d_14\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"35.524042\" xlink:href=\"#mf24b69ec4d\" y=\"283.626866\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_17\">\n",
       "      <!-- $60$ -->\n",
       "      <defs>\n",
       "       <path d=\"M 10.59375 34 \n",
       "C 10.59375 57.65625 21.796875 63 28.296875 63 \n",
       "C 30.40625 63 35.5 62.640625 37.5 59 \n",
       "C 35.90625 59 32.90625 59 32.90625 55.515625 \n",
       "C 32.90625 52.828125 35.09375 51.921875 36.5 51.921875 \n",
       "C 37.40625 51.921875 40.09375 52.3125 40.09375 55.625 \n",
       "C 40.09375 61.78125 35.09375 65.296875 28.203125 65.296875 \n",
       "C 16.296875 65.296875 3.796875 52.96875 3.796875 31 \n",
       "C 3.796875 3.953125 15.09375 -2 23.09375 -2 \n",
       "C 32.796875 -2 42 6.671875 42 20.046875 \n",
       "C 42 32.515625 33.90625 41.59375 23.703125 41.59375 \n",
       "C 17.59375 41.59375 13.09375 37.609375 10.59375 30.625 \n",
       "z\n",
       "M 23.09375 0.390625 \n",
       "C 10.796875 0.390625 10.796875 18.75 10.796875 22.4375 \n",
       "C 10.796875 29.625 14.203125 40 23.5 40 \n",
       "C 25.203125 40 30.09375 40 33.40625 33.125 \n",
       "C 35.203125 29.21875 35.203125 25.140625 35.203125 20.140625 \n",
       "C 35.203125 14.75 35.203125 10.78125 33.09375 6.78125 \n",
       "C 30.90625 2.671875 27.703125 0.390625 23.09375 0.390625 \n",
       "\" id=\"CMR17-54\"/>\n",
       "      </defs>\n",
       "      <g transform=\"translate(20.055677 286.394267)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_18\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <defs>\n",
       "      <path d=\"M 22 -16.609375 \n",
       "C 15.59375 -16.609375 15.203125 -16.203125 15.203125 -12.328125 \n",
       "L 15.203125 6.28125 \n",
       "C 18.09375 2.09375 22.296875 -0.59375 27.59375 -0.59375 \n",
       "C 37.59375 -0.59375 47.703125 8.171875 47.703125 21.6875 \n",
       "C 47.703125 34.234375 39 43.78125 28.703125 43.78125 \n",
       "C 22.703125 43.78125 17.796875 40.5 15 36.53125 \n",
       "L 15 43.78125 \n",
       "L 3.09375 42.6875 \n",
       "L 3.09375 40.109375 \n",
       "C 9 40.109375 9.90625 39.5 9.90625 34.734375 \n",
       "L 9.90625 -12.328125 \n",
       "C 9.90625 -16.109375 9.5 -16.609375 3.09375 -16.609375 \n",
       "L 3.09375 -19.1875 \n",
       "C 5.5 -19 9.90625 -19 12.5 -19 \n",
       "C 15.09375 -19 19.59375 -19 22 -19.1875 \n",
       "z\n",
       "M 15.203125 31.4375 \n",
       "C 15.203125 32.84375 15.203125 34.921875 19.09375 38.609375 \n",
       "C 19.59375 39.015625 23 42 28 42 \n",
       "C 35.296875 42 41.296875 32.9375 41.296875 21.59375 \n",
       "C 41.296875 10.25 34.90625 1 27 1 \n",
       "C 23.40625 1 19.5 2.6875 16.5 7.375 \n",
       "C 15.203125 9.5625 15.203125 10.15625 15.203125 11.75 \n",
       "z\n",
       "\" id=\"CMR17-112\"/>\n",
       "      <path d=\"M 36.09375 40.703125 \n",
       "C 36.90625 41.71875 36.90625 41.90625 36.90625 42.109375 \n",
       "C 36.90625 43 36.5 43 34.703125 43 \n",
       "L 4.296875 43 \n",
       "L 3.296875 27.5 \n",
       "L 5.09375 27.5 \n",
       "C 5.796875 37.203125 6.90625 41.40625 18.40625 41.40625 \n",
       "L 30.59375 41.40625 \n",
       "L 2 1.5 \n",
       "C 2 0 2.09375 0 4.203125 0 \n",
       "L 35.796875 0 \n",
       "L 37.203125 18 \n",
       "L 35.40625 18 \n",
       "C 34.59375 6.90625 33.296875 1.796875 21.09375 1.796875 \n",
       "L 8.296875 1.796875 \n",
       "z\n",
       "\" id=\"CMR17-122\"/>\n",
       "     </defs>\n",
       "     <g transform=\"translate(14.118498 358.808786)rotate(-90)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(101.79074 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(152.686109 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(198.376586 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(233.657354 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(279.347831 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "      <use transform=\"translate(330.243201 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(411.214537 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(462.109907 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(486.980966 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "      <use transform=\"translate(527.466626 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_53\">\n",
       "    <path d=\"M 35.524042 391.22 \n",
       "L 35.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_54\">\n",
       "    <path d=\"M 221.524042 391.22 \n",
       "L 221.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_55\">\n",
       "    <path d=\"M 35.524042 391.22 \n",
       "L 221.524042 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_56\">\n",
       "    <path d=\"M 35.524042 264.38 \n",
       "L 221.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_8\">\n",
       "   <g id=\"patch_57\">\n",
       "    <path d=\"M 221.524042 391.22 \n",
       "L 407.524042 391.22 \n",
       "L 407.524042 264.38 \n",
       "L 221.524042 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"PathCollection_6\">\n",
       "    <g clip-path=\"url(#p08bd2a22a5)\">\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"377.045051\" xlink:href=\"#m92d7d176f3\" y=\"267.4\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"278.053454\" xlink:href=\"#m92d7d176f3\" y=\"321.489552\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"403.095471\" xlink:href=\"#m92d7d176f3\" y=\"299.853731\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"309.313958\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"293.683706\" xlink:href=\"#m92d7d176f3\" y=\"307.065672\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"314.524042\" xlink:href=\"#m92d7d176f3\" y=\"303.459701\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"288.473622\" xlink:href=\"#m92d7d176f3\" y=\"343.125373\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"290.838806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"330.504478\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"288.473622\" xlink:href=\"#m92d7d176f3\" y=\"361.155224\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"334.110448\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"334.110448\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"262.423202\" xlink:href=\"#m92d7d176f3\" y=\"344.928358\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"262.423202\" xlink:href=\"#m92d7d176f3\" y=\"364.761194\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"373.776119\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"370.170149\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"366.564179\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"257.213118\" xlink:href=\"#m92d7d176f3\" y=\"368.367164\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"368.367164\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"373.776119\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"364.761194\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"257.213118\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"241.582866\" xlink:href=\"#m92d7d176f3\" y=\"292.641791\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"337.716418\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"339.519403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"380.98806\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"246.79295\" xlink:href=\"#m92d7d176f3\" y=\"332.307463\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"335.913433\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"377.38209\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"379.185075\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"382.791045\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"236.372782\" xlink:href=\"#m92d7d176f3\" y=\"371.973134\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"252.003034\" xlink:href=\"#m92d7d176f3\" y=\"366.564179\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"375.579104\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"371.973134\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"231.162698\" xlink:href=\"#m92d7d176f3\" y=\"384.59403\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"388.2\"/>\n",
       "     <use style=\"fill:#1f77b4;fill-opacity:0.5;stroke:#1f77b4;stroke-opacity:0.5;\" x=\"225.952614\" xlink:href=\"#m92d7d176f3\" y=\"386.397015\"/>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_5\">\n",
       "    <g id=\"xtick_5\">\n",
       "     <g id=\"line2d_15\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"272.84337\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_19\">\n",
       "      <!-- $10$ -->\n",
       "      <g transform=\"translate(274.835898 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-49\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_6\">\n",
       "     <g id=\"line2d_16\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"324.94421\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_20\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(326.936738 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_7\">\n",
       "     <g id=\"line2d_17\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"377.045051\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_21\">\n",
       "      <!-- $30$ -->\n",
       "      <g transform=\"translate(379.037579 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-51\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_22\">\n",
       "     <!-- Number First Neighbors -->\n",
       "     <g transform=\"translate(261.362186 417.606864)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(69.07243 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(119.9678 0)scale(0.996264)\" xlink:href=\"#CMR17-109\"/>\n",
       "      <use transform=\"translate(194.284936 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(247.782744 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(288.268404 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(353.625139 0)scale(0.996264)\" xlink:href=\"#CMR17-70\"/>\n",
       "      <use transform=\"translate(413.608985 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(438.480044 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(473.760812 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "      <use transform=\"translate(509.562149 0)scale(0.996264)\" xlink:href=\"#CMR17-116\"/>\n",
       "      <use transform=\"translate(574.918883 0)scale(0.996264)\" xlink:href=\"#CMR17-78\"/>\n",
       "      <use transform=\"translate(643.991313 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "      <use transform=\"translate(684.476974 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(709.348032 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(755.038509 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(803.331425 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(856.829234 0)scale(0.996264)\" xlink:href=\"#CMR17-111\"/>\n",
       "      <use transform=\"translate(902.519711 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(937.800478 0)scale(0.996264)\" xlink:href=\"#CMR17-115\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_58\">\n",
       "    <path d=\"M 221.524042 391.22 \n",
       "L 221.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_59\">\n",
       "    <path d=\"M 407.524042 391.22 \n",
       "L 407.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_60\">\n",
       "    <path d=\"M 221.524042 391.22 \n",
       "L 407.524042 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_61\">\n",
       "    <path d=\"M 221.524042 264.38 \n",
       "L 407.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       "  <g id=\"axes_9\">\n",
       "   <g id=\"patch_62\">\n",
       "    <path d=\"M 407.524042 391.22 \n",
       "L 593.524042 391.22 \n",
       "L 593.524042 264.38 \n",
       "L 407.524042 264.38 \n",
       "z\n",
       "\" style=\"fill:#ffffff;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_63\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 411.952614 391.22 \n",
       "L 429.6669 391.22 \n",
       "L 429.6669 270.42 \n",
       "L 411.952614 270.42 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_64\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 429.6669 391.22 \n",
       "L 447.381185 391.22 \n",
       "L 447.381185 360.283415 \n",
       "L 429.6669 360.283415 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_65\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 447.381185 391.22 \n",
       "L 465.095471 391.22 \n",
       "L 465.095471 389.746829 \n",
       "L 447.381185 389.746829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_66\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 465.095471 391.22 \n",
       "L 482.809757 391.22 \n",
       "L 482.809757 388.273659 \n",
       "L 465.095471 388.273659 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_67\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 482.809757 391.22 \n",
       "L 500.524042 391.22 \n",
       "L 500.524042 352.917561 \n",
       "L 482.809757 352.917561 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_68\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 500.524042 391.22 \n",
       "L 518.238328 391.22 \n",
       "L 518.238328 389.746829 \n",
       "L 500.524042 389.746829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_69\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 518.238328 391.22 \n",
       "L 535.952614 391.22 \n",
       "L 535.952614 389.746829 \n",
       "L 518.238328 389.746829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_70\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 535.952614 391.22 \n",
       "L 553.6669 391.22 \n",
       "L 553.6669 383.854146 \n",
       "L 535.952614 383.854146 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_71\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 553.6669 391.22 \n",
       "L 571.381185 391.22 \n",
       "L 571.381185 389.746829 \n",
       "L 553.6669 389.746829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_72\">\n",
       "    <path clip-path=\"url(#p19c035e4e1)\" d=\"M 571.381185 391.22 \n",
       "L 589.095471 391.22 \n",
       "L 589.095471 389.746829 \n",
       "L 571.381185 389.746829 \n",
       "z\n",
       "\" style=\"fill:#1f77b4;\"/>\n",
       "   </g>\n",
       "   <g id=\"matplotlib.axis_6\">\n",
       "    <g id=\"xtick_8\">\n",
       "     <g id=\"line2d_18\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"459.543232\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_23\">\n",
       "      <!-- $20$ -->\n",
       "      <g transform=\"translate(461.53576 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-50\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_9\">\n",
       "     <g id=\"line2d_19\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"512.421697\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_24\">\n",
       "      <!-- $40$ -->\n",
       "      <g transform=\"translate(514.414225 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-52\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"xtick_10\">\n",
       "     <g id=\"line2d_20\">\n",
       "      <g>\n",
       "       <use style=\"stroke:#000000;stroke-width:0.8;\" x=\"565.300162\" xlink:href=\"#m82ad270b99\" y=\"391.22\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "     <g id=\"text_25\">\n",
       "      <!-- $60$ -->\n",
       "      <g transform=\"translate(567.29269 406.688366)rotate(-90)scale(0.08 -0.08)\">\n",
       "       <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-54\"/>\n",
       "       <use transform=\"translate(45.690477 0)scale(0.996264)\" xlink:href=\"#CMR17-48\"/>\n",
       "      </g>\n",
       "     </g>\n",
       "    </g>\n",
       "    <g id=\"text_26\">\n",
       "     <!-- Subgraph Size -->\n",
       "     <g transform=\"translate(469.515257 417.606864)scale(0.1 -0.1)\">\n",
       "      <use transform=\"scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(50.89537 0)scale(0.996264)\" xlink:href=\"#CMR17-117\"/>\n",
       "      <use transform=\"translate(101.79074 0)scale(0.996264)\" xlink:href=\"#CMR17-98\"/>\n",
       "      <use transform=\"translate(152.686109 0)scale(0.996264)\" xlink:href=\"#CMR17-103\"/>\n",
       "      <use transform=\"translate(198.376586 0)scale(0.996264)\" xlink:href=\"#CMR17-114\"/>\n",
       "      <use transform=\"translate(233.657354 0)scale(0.996264)\" xlink:href=\"#CMR17-97\"/>\n",
       "      <use transform=\"translate(279.347831 0)scale(0.996264)\" xlink:href=\"#CMR17-112\"/>\n",
       "      <use transform=\"translate(330.243201 0)scale(0.996264)\" xlink:href=\"#CMR17-104\"/>\n",
       "      <use transform=\"translate(411.214537 0)scale(0.996264)\" xlink:href=\"#CMR17-83\"/>\n",
       "      <use transform=\"translate(462.109907 0)scale(0.996264)\" xlink:href=\"#CMR17-105\"/>\n",
       "      <use transform=\"translate(486.980966 0)scale(0.996264)\" xlink:href=\"#CMR17-122\"/>\n",
       "      <use transform=\"translate(527.466626 0)scale(0.996264)\" xlink:href=\"#CMR17-101\"/>\n",
       "     </g>\n",
       "    </g>\n",
       "   </g>\n",
       "   <g id=\"patch_73\">\n",
       "    <path d=\"M 407.524042 391.22 \n",
       "L 407.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_74\">\n",
       "    <path d=\"M 593.524042 391.22 \n",
       "L 593.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_75\">\n",
       "    <path d=\"M 407.524042 391.22 \n",
       "L 593.524042 391.22 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "   <g id=\"patch_76\">\n",
       "    <path d=\"M 407.524042 264.38 \n",
       "L 593.524042 264.38 \n",
       "\" style=\"fill:none;stroke:#000000;stroke-linecap:square;stroke-linejoin:miter;stroke-width:0.8;\"/>\n",
       "   </g>\n",
       "  </g>\n",
       " </g>\n",
       " <defs>\n",
       "  <clipPath id=\"pbd76cf1ee0\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"35.524042\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pa38e8881a1\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"221.524042\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p0198bef5da\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"407.524042\" y=\"10.7\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pbf28c37d7e\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"35.524042\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pfdbcd8ecd9\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"221.524042\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"pf631313b9b\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"407.524042\" y=\"137.54\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p7eaf6073cb\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"35.524042\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p08bd2a22a5\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"221.524042\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       "  <clipPath id=\"p19c035e4e1\">\n",
       "   <rect height=\"126.84\" width=\"186\" x=\"407.524042\" y=\"264.38\"/>\n",
       "  </clipPath>\n",
       " </defs>\n",
       "</svg>\n"
      ],
      "text/plain": [
       "<matplotlib.figure.Figure at 0x11166c6d8>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "scatter_matrix(scores_df[scores_df['Average NPA Score'].notnull()], figsize=(10, 7))\n",
    "plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "scores_df.to_csv(os.path.expanduser('~/Desktop/npa_demo.csv'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Average NPA Score</th>\n",
       "      <th>Number First Neighbors</th>\n",
       "      <th>Subgraph Size</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cognition)</th>\n",
       "      <td>-1.957483</td>\n",
       "      <td>18</td>\n",
       "      <td>49</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of glucose metabolic process)</th>\n",
       "      <td>-1.773353</td>\n",
       "      <td>2</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, central nervous system development)</th>\n",
       "      <td>-1.516629</td>\n",
       "      <td>2</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Unfolded Protein Response)</th>\n",
       "      <td>-1.515212</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, protein import into mitochondrial matrix)</th>\n",
       "      <td>-1.163541</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondrial calcium ion homeostasis)</th>\n",
       "      <td>-1.090722</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, electron transport chain)</th>\n",
       "      <td>-1.007132</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, memory)</th>\n",
       "      <td>-0.998019</td>\n",
       "      <td>17</td>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of catalytic activity)</th>\n",
       "      <td>-0.910587</td>\n",
       "      <td>3</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, signaling)</th>\n",
       "      <td>-0.884248</td>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of cellular response to oxidative stress)</th>\n",
       "      <td>-0.874394</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of endocytosis)</th>\n",
       "      <td>-0.725764</td>\n",
       "      <td>3</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, production of molecular mediator involved in inflammatory response)</th>\n",
       "      <td>-0.725035</td>\n",
       "      <td>8</td>\n",
       "      <td>15</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of lysosomal membrane permeability)</th>\n",
       "      <td>-0.720539</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, apoptotic process)</th>\n",
       "      <td>-0.630201</td>\n",
       "      <td>35</td>\n",
       "      <td>51</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of synaptic plasticity)</th>\n",
       "      <td>-0.567215</td>\n",
       "      <td>1</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, PTS, inflammatory response pathway)</th>\n",
       "      <td>-0.567215</td>\n",
       "      <td>1</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, plasma membrane fusion)</th>\n",
       "      <td>-0.492646</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of cAMP-dependent protein kinase activity)</th>\n",
       "      <td>-0.488954</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, apoptotic signaling pathway)</th>\n",
       "      <td>-0.479993</td>\n",
       "      <td>1</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microglial cell activation involved in immune response)</th>\n",
       "      <td>-0.462424</td>\n",
       "      <td>4</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, PMIBP, ST6GAL1 secretion)</th>\n",
       "      <td>-0.416736</td>\n",
       "      <td>1</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of neuronal synaptic plasticity)</th>\n",
       "      <td>-0.415490</td>\n",
       "      <td>2</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cytokine production)</th>\n",
       "      <td>-0.401348</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of phagocytosis)</th>\n",
       "      <td>-0.401348</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, chemokine biosynthetic process)</th>\n",
       "      <td>-0.401348</td>\n",
       "      <td>2</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cytokine secretion)</th>\n",
       "      <td>-0.401348</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of ERK1 and ERK2 cascade)</th>\n",
       "      <td>-0.390744</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, positive regulation of ERK5 cascade)</th>\n",
       "      <td>-0.390744</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, microglial cell activation)</th>\n",
       "      <td>-0.274029</td>\n",
       "      <td>3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, zinc ion homeostasis)</th>\n",
       "      <td>0.557388</td>\n",
       "      <td>1</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, leukocyte mediated immunity)</th>\n",
       "      <td>0.557388</td>\n",
       "      <td>1</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of synaptic transmission, glutamatergic)</th>\n",
       "      <td>0.557388</td>\n",
       "      <td>1</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cAMP-mediated signaling)</th>\n",
       "      <td>0.557388</td>\n",
       "      <td>1</td>\n",
       "      <td>31</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, growth)</th>\n",
       "      <td>0.594557</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondria-nucleus signaling pathway)</th>\n",
       "      <td>0.595800</td>\n",
       "      <td>4</td>\n",
       "      <td>55</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, DNA biosynthetic process)</th>\n",
       "      <td>0.845759</td>\n",
       "      <td>2</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, RNA metabolic process)</th>\n",
       "      <td>0.858920</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell migration)</th>\n",
       "      <td>0.935996</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, brain development)</th>\n",
       "      <td>0.980261</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, synaptic transmission, glutamatergic)</th>\n",
       "      <td>0.991187</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, regulation of growth)</th>\n",
       "      <td>0.998013</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, reactive oxygen species metabolic process)</th>\n",
       "      <td>1.000978</td>\n",
       "      <td>2</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell-cell signaling)</th>\n",
       "      <td>1.004358</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, calcium ion import)</th>\n",
       "      <td>1.007735</td>\n",
       "      <td>13</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell adhesion)</th>\n",
       "      <td>1.027519</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cholesterol biosynthetic process via desmosterol)</th>\n",
       "      <td>1.045284</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, synapse assembly)</th>\n",
       "      <td>1.048500</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, stress-activated protein kinase signaling cascade)</th>\n",
       "      <td>1.060898</td>\n",
       "      <td>2</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell-matrix adhesion)</th>\n",
       "      <td>1.094275</td>\n",
       "      <td>3</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, MESHPP, Insulin Resistance)</th>\n",
       "      <td>1.157408</td>\n",
       "      <td>6</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, cell death)</th>\n",
       "      <td>1.250924</td>\n",
       "      <td>11</td>\n",
       "      <td>39</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, mitochondrion organization)</th>\n",
       "      <td>1.493461</td>\n",
       "      <td>5</td>\n",
       "      <td>56</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, inflammatory response)</th>\n",
       "      <td>1.519586</td>\n",
       "      <td>14</td>\n",
       "      <td>47</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron apoptotic process)</th>\n",
       "      <td>1.907206</td>\n",
       "      <td>30</td>\n",
       "      <td>69</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, neuron death)</th>\n",
       "      <td>2.108491</td>\n",
       "      <td>13</td>\n",
       "      <td>27</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, energy reserve metabolic process)</th>\n",
       "      <td>2.940783</td>\n",
       "      <td>3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glucose metabolic process)</th>\n",
       "      <td>2.940783</td>\n",
       "      <td>3</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, negative regulation of neuron apoptotic process)</th>\n",
       "      <td>2.982289</td>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>(BiologicalProcess, GOBP, glial cell differentiation)</th>\n",
       "      <td>4.136399</td>\n",
       "      <td>6</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>141 rows  3 columns</p>\n",
       "</div>"
      ],
      "text/latex": [
       "\\begin{center}{\\begin{tabular}{lrrr}\n",
       "\\toprule\n",
       "{} &  Average NPA Score &  Number First Neighbors &  Subgraph Size \\\\\n",
       "\\midrule\n",
       "(BiologicalProcess, GOBP, cognition)               &          -1.957483 &                      18 &             49 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of glucose... &          -1.773353 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, central nervous syste... &          -1.516629 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, MESHPP, Unfolded Protein Re... &          -1.515212 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, protein import into m... &          -1.163541 &                       2 &             34 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial calcium... &          -1.090722 &                       2 &             34 \\\\\n",
       "(BiologicalProcess, GOBP, electron transport ch... &          -1.007132 &                       2 &             34 \\\\\n",
       "(BiologicalProcess, GOBP, memory)                  &          -0.998019 &                      17 &             55 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of catalyt... &          -0.910587 &                       3 &             14 \\\\\n",
       "(BiologicalProcess, GOBP, signaling)               &          -0.884248 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cellula... &          -0.874394 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.725764 &                       3 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, production of molecul... &          -0.725035 &                       8 &             15 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.720539 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, apoptotic process)       &          -0.630201 &                      35 &             51 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &          -0.567215 &                       1 &              7 \\\\\n",
       "(BiologicalProcess, PTS, inflammatory response ... &          -0.567215 &                       1 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, plasma membrane fusion)  &          -0.492646 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cAMP-de... &          -0.488954 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, apoptotic signaling p... &          -0.479993 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, microglial cell activ... &          -0.462424 &                       4 &              6 \\\\\n",
       "(BiologicalProcess, PMIBP, ST6GAL1 secretion)      &          -0.416736 &                       1 &              8 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of neurona... &          -0.415490 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, cytokine production)     &          -0.401348 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &          -0.401348 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti... &          -0.401348 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cytokine secretion)      &          -0.401348 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.390744 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.390744 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, microglial cell activ... &          -0.274029 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, neuron development)      &          -0.253729 &                       5 &             15 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of dendrit... &          -0.253729 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, MESHPP, Apoptosis)             &          -0.248319 &                       7 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, activation of GTPase ... &          -0.213611 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &          -0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, sodium ion homeostasis)  &          -0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, nitric oxide-cGMP-med... &          -0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial transport) &          -0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, reelin-mediated signa... &          -0.178703 &                       2 &             30 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.161761 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.145004 &                       1 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of MAPK ca... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, beta-amyloid clearance)  &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of N-methy... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, TOR signaling)           &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, immune response)         &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, transcription, DNA-te... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, response to endoplasm... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, lipid oxidation)         &          -0.137015 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle process)      &          -0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, astrocyte activation)    &          -0.137015 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, calcium-dependent cel... &          -0.128871 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of cell-ce... &          -0.128871 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, PMIBP, beta amyloid induced... &          -0.119441 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.115380 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &          -0.115380 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of short-t... &          -0.115380 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.114720 &                       8 &             26 \\\\\n",
       "(BiologicalProcess, GOBP, cell proliferation)      &          -0.094359 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &          -0.094359 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.094359 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, transmission of nerve... &          -0.059557 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, long term synaptic de... &          -0.054806 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, ADO, loss\\_of\\_synapses)         &          -0.033649 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &          -0.023988 &                       3 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of ryanodi... &          -0.021635 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Endoplasmic Reticul... &          -0.002663 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, vasodilation)            &           0.000000 &                       2 &             30 \\\\\n",
       "(BiologicalProcess, MESHPP, Long-Term Potentiat... &           0.000000 &                       2 &             30 \\\\\n",
       "(BiologicalProcess, PTS, Wnt/beta\\_catenin pathway) &           0.005326 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, PMIBP, spatial memory)         &           0.033649 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, PMIBP, spatial learning)       &           0.033649 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Neuronal Plasticity)   &           0.054806 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, acetylcholine secretion) &           0.054806 &                       1 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.085522 &                       3 &             32 \\\\\n",
       "(BiologicalProcess, GOBP, cell cycle)              &           0.094359 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, MESHPP, Oxidative Stress)      &           0.101325 &                       7 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, positive regulation o... &           0.115380 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, long-term synaptic po... &           0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, neuron projection dev... &           0.137015 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, iron ion homeostasis)    &           0.137015 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.137676 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.137676 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, beta-amyloid formation)  &           0.178703 &                       5 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, leukotriene productio... &           0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, response to oxidative... &           0.178703 &                       5 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, copper ion export)       &           0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, response to reactive ... &           0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, PMIBP, Calcium dysfunction)    &           0.178703 &                       1 &             29 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &           0.207792 &                       2 &              6 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.207792 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, calcium ion homeostasis) &           0.207792 &                       3 &              7 \\\\\n",
       "(BiologicalProcess, GOBP, generation of neurons)   &           0.207792 &                       1 &              5 \\\\\n",
       "(BiologicalProcess, PMIBP, growth cone collapse... &           0.236801 &                       2 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, chronic inflammatory ... &           0.246790 &                       6 &             33 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of protein... &           0.267315 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of long-te... &           0.274029 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of neuron ... &           0.274029 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of synapti... &           0.274029 &                       2 &              4 \\\\\n",
       "(BiologicalProcess, GOBP, autophagy)               &           0.336059 &                       2 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, learning)                &           0.344807 &                       6 &             10 \\\\\n",
       "(BiologicalProcess, GOBP, vasoconstriction)        &           0.357407 &                       3 &             31 \\\\\n",
       "(BiologicalProcess, MESHPP, Lipid Peroxidation)    &           0.357407 &                       2 &             30 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.357407 &                       4 &             32 \\\\\n",
       "(BiologicalProcess, GOBP, blood circulation)       &           0.378685 &                       4 &             55 \\\\\n",
       "(BiologicalProcess, GOBP, pre-B cell receptor e... &           0.390744 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of inflamm... &           0.401348 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, dopamine receptor sig... &           0.488954 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrial genome ... &           0.555643 &                       1 &              2 \\\\\n",
       "(BiologicalProcess, GOBP, zinc ion homeostasis)    &           0.557388 &                       1 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, leukocyte mediated im... &           0.557388 &                       1 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           0.557388 &                       1 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, cAMP-mediated signaling) &           0.557388 &                       1 &             31 \\\\\n",
       "(BiologicalProcess, GOBP, growth)                  &           0.594557 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ... &           0.595800 &                       4 &             55 \\\\\n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc... &           0.845759 &                       2 &             11 \\\\\n",
       "(BiologicalProcess, GOBP, RNA metabolic process)   &           0.858920 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, cell migration)          &           0.935996 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, brain development)       &           0.980261 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, synaptic transmission... &           0.991187 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, regulation of growth)    &           0.998013 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, reactive oxygen speci... &           1.000978 &                       2 &             34 \\\\\n",
       "(BiologicalProcess, GOBP, cell-cell signaling)     &           1.004358 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, calcium ion import)      &           1.007735 &                      13 &             17 \\\\\n",
       "(BiologicalProcess, GOBP, cell adhesion)           &           1.027519 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, cholesterol biosynthe... &           1.045284 &                       1 &              3 \\\\\n",
       "(BiologicalProcess, GOBP, synapse assembly)        &           1.048500 &                       1 &              9 \\\\\n",
       "(BiologicalProcess, GOBP, stress-activated prot... &           1.060898 &                       2 &             10 \\\\\n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)    &           1.094275 &                       3 &             11 \\\\\n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)    &           1.157408 &                       6 &             12 \\\\\n",
       "(BiologicalProcess, GOBP, cell death)              &           1.250924 &                      11 &             39 \\\\\n",
       "(BiologicalProcess, GOBP, mitochondrion organiz... &           1.493461 &                       5 &             56 \\\\\n",
       "(BiologicalProcess, GOBP, inflammatory response)   &           1.519586 &                      14 &             47 \\\\\n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc... &           1.907206 &                      30 &             69 \\\\\n",
       "(BiologicalProcess, GOBP, neuron death)            &           2.108491 &                      13 &             27 \\\\\n",
       "(BiologicalProcess, GOBP, energy reserve metabo... &           2.940783 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, glucose metabolic pro... &           2.940783 &                       3 &              5 \\\\\n",
       "(BiologicalProcess, GOBP, negative regulation o... &           2.982289 &                       6 &             13 \\\\\n",
       "(BiologicalProcess, GOBP, glial cell differenti... &           4.136399 &                       6 &             14 \\\\\n",
       "\\bottomrule\n",
       "\\end{tabular}\n",
       "}\\end{center}"
      ],
      "text/plain": [
       "                                                    Average NPA Score  \\\n",
       "(BiologicalProcess, GOBP, cognition)                        -1.957483   \n",
       "(BiologicalProcess, GOBP, regulation of glucose...          -1.773353   \n",
       "(BiologicalProcess, GOBP, central nervous syste...          -1.516629   \n",
       "(BiologicalProcess, MESHPP, Unfolded Protein Re...          -1.515212   \n",
       "(BiologicalProcess, GOBP, protein import into m...          -1.163541   \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...          -1.090722   \n",
       "(BiologicalProcess, GOBP, electron transport ch...          -1.007132   \n",
       "(BiologicalProcess, GOBP, memory)                           -0.998019   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...          -0.910587   \n",
       "(BiologicalProcess, GOBP, signaling)                        -0.884248   \n",
       "(BiologicalProcess, GOBP, regulation of cellula...          -0.874394   \n",
       "(BiologicalProcess, GOBP, positive regulation o...          -0.725764   \n",
       "(BiologicalProcess, GOBP, production of molecul...          -0.725035   \n",
       "(BiologicalProcess, GOBP, positive regulation o...          -0.720539   \n",
       "(BiologicalProcess, GOBP, apoptotic process)                -0.630201   \n",
       "(BiologicalProcess, GOBP, negative regulation o...          -0.567215   \n",
       "(BiologicalProcess, PTS, inflammatory response ...          -0.567215   \n",
       "(BiologicalProcess, GOBP, plasma membrane fusion)           -0.492646   \n",
       "(BiologicalProcess, GOBP, regulation of cAMP-de...          -0.488954   \n",
       "(BiologicalProcess, GOBP, apoptotic signaling p...          -0.479993   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...          -0.462424   \n",
       "(BiologicalProcess, PMIBP, ST6GAL1 secretion)               -0.416736   \n",
       "(BiologicalProcess, GOBP, regulation of neurona...          -0.415490   \n",
       "(BiologicalProcess, GOBP, cytokine production)              -0.401348   \n",
       "(BiologicalProcess, GOBP, negative regulation o...          -0.401348   \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...          -0.401348   \n",
       "(BiologicalProcess, GOBP, cytokine secretion)               -0.401348   \n",
       "(BiologicalProcess, GOBP, positive regulation o...          -0.390744   \n",
       "(BiologicalProcess, GOBP, positive regulation o...          -0.390744   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...          -0.274029   \n",
       "...                                                               ...   \n",
       "(BiologicalProcess, GOBP, zinc ion homeostasis)              0.557388   \n",
       "(BiologicalProcess, GOBP, leukocyte mediated im...           0.557388   \n",
       "(BiologicalProcess, GOBP, negative regulation o...           0.557388   \n",
       "(BiologicalProcess, GOBP, cAMP-mediated signaling)           0.557388   \n",
       "(BiologicalProcess, GOBP, growth)                            0.594557   \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...           0.595800   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...           0.845759   \n",
       "(BiologicalProcess, GOBP, RNA metabolic process)             0.858920   \n",
       "(BiologicalProcess, GOBP, cell migration)                    0.935996   \n",
       "(BiologicalProcess, GOBP, brain development)                 0.980261   \n",
       "(BiologicalProcess, GOBP, synaptic transmission...           0.991187   \n",
       "(BiologicalProcess, GOBP, regulation of growth)              0.998013   \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...           1.000978   \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)               1.004358   \n",
       "(BiologicalProcess, GOBP, calcium ion import)                1.007735   \n",
       "(BiologicalProcess, GOBP, cell adhesion)                     1.027519   \n",
       "(BiologicalProcess, GOBP, cholesterol biosynthe...           1.045284   \n",
       "(BiologicalProcess, GOBP, synapse assembly)                  1.048500   \n",
       "(BiologicalProcess, GOBP, stress-activated prot...           1.060898   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)              1.094275   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)              1.157408   \n",
       "(BiologicalProcess, GOBP, cell death)                        1.250924   \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...           1.493461   \n",
       "(BiologicalProcess, GOBP, inflammatory response)             1.519586   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...           1.907206   \n",
       "(BiologicalProcess, GOBP, neuron death)                      2.108491   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...           2.940783   \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...           2.940783   \n",
       "(BiologicalProcess, GOBP, negative regulation o...           2.982289   \n",
       "(BiologicalProcess, GOBP, glial cell differenti...           4.136399   \n",
       "\n",
       "                                                    Number First Neighbors  \\\n",
       "(BiologicalProcess, GOBP, cognition)                                    18   \n",
       "(BiologicalProcess, GOBP, regulation of glucose...                       2   \n",
       "(BiologicalProcess, GOBP, central nervous syste...                       2   \n",
       "(BiologicalProcess, MESHPP, Unfolded Protein Re...                       1   \n",
       "(BiologicalProcess, GOBP, protein import into m...                       2   \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...                       2   \n",
       "(BiologicalProcess, GOBP, electron transport ch...                       2   \n",
       "(BiologicalProcess, GOBP, memory)                                       17   \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...                       3   \n",
       "(BiologicalProcess, GOBP, signaling)                                     2   \n",
       "(BiologicalProcess, GOBP, regulation of cellula...                       1   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       3   \n",
       "(BiologicalProcess, GOBP, production of molecul...                       8   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, apoptotic process)                            35   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       1   \n",
       "(BiologicalProcess, PTS, inflammatory response ...                       1   \n",
       "(BiologicalProcess, GOBP, plasma membrane fusion)                        1   \n",
       "(BiologicalProcess, GOBP, regulation of cAMP-de...                       1   \n",
       "(BiologicalProcess, GOBP, apoptotic signaling p...                       1   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...                       4   \n",
       "(BiologicalProcess, PMIBP, ST6GAL1 secretion)                            1   \n",
       "(BiologicalProcess, GOBP, regulation of neurona...                       2   \n",
       "(BiologicalProcess, GOBP, cytokine production)                           1   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...                       2   \n",
       "(BiologicalProcess, GOBP, cytokine secretion)                            1   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, positive regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, microglial cell activ...                       3   \n",
       "...                                                                    ...   \n",
       "(BiologicalProcess, GOBP, zinc ion homeostasis)                          1   \n",
       "(BiologicalProcess, GOBP, leukocyte mediated im...                       1   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       1   \n",
       "(BiologicalProcess, GOBP, cAMP-mediated signaling)                       1   \n",
       "(BiologicalProcess, GOBP, growth)                                        1   \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...                       4   \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...                       2   \n",
       "(BiologicalProcess, GOBP, RNA metabolic process)                         1   \n",
       "(BiologicalProcess, GOBP, cell migration)                                1   \n",
       "(BiologicalProcess, GOBP, brain development)                             1   \n",
       "(BiologicalProcess, GOBP, synaptic transmission...                       1   \n",
       "(BiologicalProcess, GOBP, regulation of growth)                          1   \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...                       2   \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)                           1   \n",
       "(BiologicalProcess, GOBP, calcium ion import)                           13   \n",
       "(BiologicalProcess, GOBP, cell adhesion)                                 1   \n",
       "(BiologicalProcess, GOBP, cholesterol biosynthe...                       1   \n",
       "(BiologicalProcess, GOBP, synapse assembly)                              1   \n",
       "(BiologicalProcess, GOBP, stress-activated prot...                       2   \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                          3   \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                          6   \n",
       "(BiologicalProcess, GOBP, cell death)                                   11   \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...                       5   \n",
       "(BiologicalProcess, GOBP, inflammatory response)                        14   \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...                      30   \n",
       "(BiologicalProcess, GOBP, neuron death)                                 13   \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...                       3   \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...                       3   \n",
       "(BiologicalProcess, GOBP, negative regulation o...                       6   \n",
       "(BiologicalProcess, GOBP, glial cell differenti...                       6   \n",
       "\n",
       "                                                    Subgraph Size  \n",
       "(BiologicalProcess, GOBP, cognition)                           49  \n",
       "(BiologicalProcess, GOBP, regulation of glucose...              5  \n",
       "(BiologicalProcess, GOBP, central nervous syste...              5  \n",
       "(BiologicalProcess, MESHPP, Unfolded Protein Re...              3  \n",
       "(BiologicalProcess, GOBP, protein import into m...             34  \n",
       "(BiologicalProcess, GOBP, mitochondrial calcium...             34  \n",
       "(BiologicalProcess, GOBP, electron transport ch...             34  \n",
       "(BiologicalProcess, GOBP, memory)                              55  \n",
       "(BiologicalProcess, GOBP, regulation of catalyt...             14  \n",
       "(BiologicalProcess, GOBP, signaling)                            3  \n",
       "(BiologicalProcess, GOBP, regulation of cellula...              9  \n",
       "(BiologicalProcess, GOBP, positive regulation o...              4  \n",
       "(BiologicalProcess, GOBP, production of molecul...             15  \n",
       "(BiologicalProcess, GOBP, positive regulation o...              3  \n",
       "(BiologicalProcess, GOBP, apoptotic process)                   51  \n",
       "(BiologicalProcess, GOBP, negative regulation o...              7  \n",
       "(BiologicalProcess, PTS, inflammatory response ...              7  \n",
       "(BiologicalProcess, GOBP, plasma membrane fusion)               2  \n",
       "(BiologicalProcess, GOBP, regulation of cAMP-de...              2  \n",
       "(BiologicalProcess, GOBP, apoptotic signaling p...              4  \n",
       "(BiologicalProcess, GOBP, microglial cell activ...              6  \n",
       "(BiologicalProcess, PMIBP, ST6GAL1 secretion)                   8  \n",
       "(BiologicalProcess, GOBP, regulation of neurona...              5  \n",
       "(BiologicalProcess, GOBP, cytokine production)                  3  \n",
       "(BiologicalProcess, GOBP, negative regulation o...              2  \n",
       "(BiologicalProcess, GOBP, chemokine biosyntheti...              4  \n",
       "(BiologicalProcess, GOBP, cytokine secretion)                   3  \n",
       "(BiologicalProcess, GOBP, positive regulation o...              3  \n",
       "(BiologicalProcess, GOBP, positive regulation o...              3  \n",
       "(BiologicalProcess, GOBP, microglial cell activ...              5  \n",
       "...                                                           ...  \n",
       "(BiologicalProcess, GOBP, zinc ion homeostasis)                31  \n",
       "(BiologicalProcess, GOBP, leukocyte mediated im...             31  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             31  \n",
       "(BiologicalProcess, GOBP, cAMP-mediated signaling)             31  \n",
       "(BiologicalProcess, GOBP, growth)                               9  \n",
       "(BiologicalProcess, GOBP, mitochondria-nucleus ...             55  \n",
       "(BiologicalProcess, GOBP, DNA biosynthetic proc...             11  \n",
       "(BiologicalProcess, GOBP, RNA metabolic process)                9  \n",
       "(BiologicalProcess, GOBP, cell migration)                       9  \n",
       "(BiologicalProcess, GOBP, brain development)                    3  \n",
       "(BiologicalProcess, GOBP, synaptic transmission...              9  \n",
       "(BiologicalProcess, GOBP, regulation of growth)                 9  \n",
       "(BiologicalProcess, GOBP, reactive oxygen speci...             34  \n",
       "(BiologicalProcess, GOBP, cell-cell signaling)                  9  \n",
       "(BiologicalProcess, GOBP, calcium ion import)                  17  \n",
       "(BiologicalProcess, GOBP, cell adhesion)                        9  \n",
       "(BiologicalProcess, GOBP, cholesterol biosynthe...              3  \n",
       "(BiologicalProcess, GOBP, synapse assembly)                     9  \n",
       "(BiologicalProcess, GOBP, stress-activated prot...             10  \n",
       "(BiologicalProcess, GOBP, cell-matrix adhesion)                11  \n",
       "(BiologicalProcess, MESHPP, Insulin Resistance)                12  \n",
       "(BiologicalProcess, GOBP, cell death)                          39  \n",
       "(BiologicalProcess, GOBP, mitochondrion organiz...             56  \n",
       "(BiologicalProcess, GOBP, inflammatory response)               47  \n",
       "(BiologicalProcess, GOBP, neuron apoptotic proc...             69  \n",
       "(BiologicalProcess, GOBP, neuron death)                        27  \n",
       "(BiologicalProcess, GOBP, energy reserve metabo...              5  \n",
       "(BiologicalProcess, GOBP, glucose metabolic pro...              5  \n",
       "(BiologicalProcess, GOBP, negative regulation o...             13  \n",
       "(BiologicalProcess, GOBP, glial cell differenti...             14  \n",
       "\n",
       "[141 rows x 3 columns]"
      ]
     },
     "execution_count": 53,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scores_df[scores_df['Average NPA Score'].notnull()].sort_values('Average NPA Score')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Conclusions\n",
    "\n",
    "## Problems\n",
    "\n",
    "This algorithm overcomes deterministic problems with cyclic graphs by randomly removing edges. It can be run multiple times to assess the stability of the score with `pbt.analysis.npa.average_npa_run`. The effect of the default score can also be checked with a grid search.\n",
    "\n",
    "## Looking Forward\n",
    "\n",
    "While this new algorithm is generally applicable and overcomes the original algorithm, it must throw away information to do so. Further algorithmic development, such as heat diffusion approaches could allow for a more thorough investigation of the propogation of effects of the differential gene expression.\n",
    "\n",
    "This algorithm can be run over the mechanisms generated for each biological process. Further investigation into the upstream mechanism generation can yield bias-free candidate mechanisms. They can also be matched to canonical mechanisms in NeuroMMSigDB to identify concordance between the biological dogma and all possible mechanisms, then to identify previously unknown cross-talk between mechanisms."
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
