{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Parsing the Causal Biological Network Database\n",
    "\n",
    "**Author:** [Charles Tapley Hoyt](https://github.com/cthoyt/)\n",
    "\n",
    "**Estimated Run Time:** 1 minute\n",
    "\n",
    "This notebook outlines the process of parsing the JSON Graph File format used in the Causal Biological Network (CBN) Database. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "import requests\n",
    "import os\n",
    "import time\n",
    "\n",
    "import networkx as nx\n",
    "\n",
    "import pybel\n",
    "from pybel.constants import *\n",
    "import pybel_tools\n",
    "from pybel_tools.visualization import to_jupyter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.7.2-dev'"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'0.1.17-dev'"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pybel_tools.__version__"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Thu Aug 10 14:07:57 2017'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "time.asctime()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Data Acquisition\n",
    "\n",
    "Data can be downloaded directly from the `GetJSONGraphFile` endpoint, and are returned in the response as JSON."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "res = requests.get(\n",
    "    \"http://causalbionet.com/Networks/GetJSONGraphFile?networkId=hox_2.0_hs\"\n",
    ").json()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Parsing\n",
    "\n",
    "The structure is traversed, and the [BELParser](http://pybel.readthedocs.io/en/latest/parser.html#pybel.parser.parse_bel.BelParser) is manipulated directly. Normally, during BEL compilation, the usage of this class is hidden from the user."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "graph = pybel.BELGraph()\n",
    "parser = pybel.parser.BelParser(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_citation(evidence):\n",
    "    return {\n",
    "        CITATION_NAME: evidence[\"citation\"][\"name\"],\n",
    "        CITATION_TYPE: evidence[\"citation\"][\"type\"],\n",
    "        CITATION_REFERENCE: evidence[\"citation\"][\"id\"],\n",
    "    }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "annotation_map = {\n",
    "    \"tissue\": \"Tissue\",\n",
    "    \"disease\": \"Disease\",\n",
    "    \"species_common_name\": \"Species\",\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "species_map = {\"human\": \"9606\", \"rat\": \"10116\", \"mouse\": \"10090\"}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "annotation_value_map = {\"Species\": species_map}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "for edge in res[\"graph\"][\"edges\"]:\n",
    "    for evidence in edge[\"metadata\"][\"evidences\"]:\n",
    "        if \"citation\" not in evidence or not evidence[\"citation\"]:\n",
    "            continue\n",
    "\n",
    "        parser.control_parser.clear()\n",
    "        parser.control_parser.citation = get_citation(evidence)\n",
    "        parser.control_parser.evidence = evidence[\"summary_text\"]\n",
    "\n",
    "        d = {}\n",
    "\n",
    "        if \"biological_context\" in evidence:\n",
    "            annotations = evidence[\"biological_context\"]\n",
    "\n",
    "            if annotations[\"tissue\"]:\n",
    "                d[\"Tissue\"] = annotations[\"tissue\"]\n",
    "\n",
    "            if annotations[\"disease\"]:\n",
    "                d[\"Disease\"] = annotations[\"disease\"]\n",
    "\n",
    "            if annotations[\"species_common_name\"]:\n",
    "                d[\"Species\"] = species_map[annotations[\"species_common_name\"].lower()]\n",
    "\n",
    "        parser.control_parser.annotations.update(d)\n",
    "        bel = \"{source} {relation} {target}\".format_map(edge)\n",
    "        try:\n",
    "            parser.parseString(bel)\n",
    "        except Exception as e:\n",
    "            print(e, bel)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Visualization\n",
    "\n",
    "Finally, the graph is vizualized in the notebook diretly with `pybel_tools.visualization.to_jupyter`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"rdexfctbjzqohavu\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {}, \"pybel_version\": \"0.7.2-dev\", \"namespace_url\": {}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MEOX2\", \"cname\": \"MEOX2\", \"id\": [\"Protein\", \"HGNC\", \"MEOX2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1A\", \"cname\": \"CDKN1A\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA5\", \"cname\": \"HOXA5\", \"id\": [\"Protein\", \"HGNC\", \"HOXA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"KDR\", \"cname\": \"KDR\", \"id\": [\"Protein\", \"HGNC\", \"KDR\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXD3\", \"cname\": \"HOXD3\", \"id\": [\"Protein\", \"HGNC\", \"HOXD3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGA5\", \"cname\": \"ITGA5\", \"id\": [\"Protein\", \"HGNC\", \"ITGA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGB3\", \"cname\": \"ITGB3\", \"id\": [\"Protein\", \"HGNC\", \"ITGB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXB4\", \"cname\": \"HOXB4\", \"id\": [\"Protein\", \"HGNC\", \"HOXB4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1B\", \"cname\": \"CDKN1B\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1B\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cell proliferation\", \"cname\": \"cell proliferation\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cell proliferation\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA3\", \"cname\": \"HOXA3\", \"id\": [\"Protein\", \"HGNC\", \"HOXA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MMP14\", \"cname\": \"MMP14\", \"id\": [\"Protein\", \"HGNC\", \"MMP14\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PLAU\", \"cname\": \"PLAU\", \"id\": [\"Protein\", \"HGNC\", \"PLAU\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PDGFRB\", \"cname\": \"PDGFRB\", \"id\": [\"Protein\", \"HGNC\", \"PDGFRB\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN2D\", \"cname\": \"CDKN2D\", \"id\": [\"Protein\", \"HGNC\", \"CDKN2D\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1C\", \"cname\": \"CDKN1C\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1C\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CDKN1A\", \"cname\": \"CDKN1A\", \"id\": [\"RNA\", \"HGNC\", \"CDKN1A\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"endothelial cell activation\", \"cname\": \"endothelial cell activation\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"endothelial cell activation\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CDKN2A\", \"cname\": \"CDKN2A\", \"id\": [\"RNA\", \"HGNC\", \"CDKN2A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGAV\", \"cname\": \"ITGAV\", \"id\": [\"Protein\", \"HGNC\", \"ITGAV\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"AKT Family\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 473}], \"cname\": \"p(SFAM:\\\"AKT Family\\\", pmod(Ph, Ser, 473))\", \"id\": [\"Protein\", \"SFAM\", \"AKT Family\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 473]]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"AKT Family\", \"cname\": \"AKT Family\", \"id\": [\"Protein\", \"SFAM\", \"AKT Family\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PTGS2\", \"cname\": \"PTGS2\", \"id\": [\"Protein\", \"HGNC\", \"PTGS2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PLAUR\", \"cname\": \"PLAUR\", \"id\": [\"Protein\", \"HGNC\", \"PLAUR\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA9\", \"cname\": \"HOXA9\", \"id\": [\"Protein\", \"HGNC\", \"HOXA9\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"JUN\", \"cname\": \"JUN\", \"id\": [\"Protein\", \"HGNC\", \"JUN\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:ITGAV), p(HGNC:ITGB3))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"ITGAV\"], [\"Protein\", \"HGNC\", \"ITGB3\"]]}], \"links\": [{\"relation\": \"directlyIncreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"\\\"The homeodomain transcription factor MEOX2 is an important regulator of vascular cell proliferation and is a direct transcriptional activator of both p21(CIP1/WAF1) and p16(INK4a)\\\"\", \"citation\": {\"name\": \"PLoS One. 2011;6(12):e29099. doi: 10.1371/journal.pone.0029099. Epub 2011 Dec 20.\", \"type\": \"Other\", \"reference\": \"22206000\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"12842453\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"HUVEC\", \"Disease\": \"Normal\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 1, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Gax strongly inhibited HUVEC proliferation and mitogen-stimulated 3H-thymidine uptake. p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold. Tube formation on Matrigel was strongly inhibited by Gax expression\", \"citation\": {\"name\": \"J Surg Res 2003 May 1 111(1) 91-9\", \"type\": \"Other\", \"reference\": \"12842453\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"Endothelium\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"constitutive Gax expression inhibits cell proliferation in a p21(WAF/CIP)-dependent manner\", \"citation\": {\"name\": \"EMBO J 1998 Jul 1 17(13) 3576-86\", \"type\": \"Other\", \"reference\": \"9649428\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"10090\"}, \"source\": 0, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"12842453\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"HUVEC\", \"Disease\": \"Normal\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 16, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9224717\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"mouse embryonic fibroblasts\", \"Disease\": \"normal\", \"Species\": \"10090\"}, \"source\": 0, \"target\": 17, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9224717\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"vascular smooth muscle cells\", \"Disease\": \"normal\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 17, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Inhibition of endothelial cell activation by the homeobox gene Gax.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"12842453\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"HUVEC\", \"Disease\": \"Normal\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 17, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9224717\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"vascular smooth muscle cells\", \"Disease\": \"normal\", \"Species\": \"10116\"}, \"source\": 0, \"target\": 17, \"key\": 3}, {\"relation\": \"directlyIncreases\", \"evidence\": \"MEOX1 and MEOX2 activate p16(INK4a) in a DNA binding dependent manner.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"22206000\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"HUVECs\", \"Disease\": \"normal\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 18, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 2, \"target\": 3, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"we treated HoxA5 expressing EOMA cells with LY294002, an inhibitor of PI3Kinase and Akt activity. As expected, addition of LY294002 reduced levels of phospho-Akt in EOMA cells expressing HoxA5 cells but did not influence total Akt protein â¦ HoxA5 mediated stabilization of junctions requires Akt activity. (A) Western blot showing levels of Ser473 phoshorylated AkT (upper panel) in EOMA cells expressing HoxA5\\nNote: EOMA were used as in-vitro system for virus-mediated expression of proteins studied\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"19262140\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"cell line eoma\", \"Species\": \"9606\"}, \"source\": 2, \"target\": 20, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 2, \"target\": 20, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"Sustained expression of Hox A5 leads to down regulation of many pro-angiogenic genes including VEGFR2, ephrin A1, Hif1alpha and COX-2\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"16379594\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 2, \"target\": 22, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 2, \"target\": 22, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Compared with wild-type samples, decreased proliferation was first observed in P0 and P5 Hoxa5â/â lungs (Figure 2, B, C, and H) . This was followed by increased proliferation at P15 and P31 (Figure 2, DâH) . This increase could not be associated to a specific cell type\", \"citation\": {\"name\": \"am j pathol 2006 oct 169(4) 1312-27\", \"type\": \"Other\", \"reference\": \"17003488\"}, \"line\": 0, \"annotations\": {\"Species\": \"10090\"}, \"source\": 2, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Compared with wild-type samples, decreased proliferation was first observed in P0 and P5 Hoxa5â/â lungs (Figure 2, B, C, and H) . This was followed by increased proliferation at P15 and P31 (Figure 2, DâH) . This increase could not be associated to a specific cell type. No significant difference was noted at P60 (Figure 2H) . The changes in the temporal profile of proliferation in Hoxa5â/â lungs could explain the lower lung weight/body weight ratios observed at early stages and the compensatory effect seen later on.\", \"citation\": {\"name\": \"Am J Pathol 2006 Oct 169(4) 1312-27\", \"type\": \"Other\", \"reference\": \"17003488\"}, \"line\": 0, \"annotations\": {\"Species\": \"10090\"}, \"source\": 2, \"target\": 9, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Hoxa5 transcription factor proteins uniquely impact lung morphogenesis at the developmental time point when extremely preterm infants are born. The effect of O2 exposure (0.4 FiO2) used in preterm infant care on these Hox proteins is unknown. We used ex vivo fetal mouse lung organ cultures to explore the effects of 0.4 FiO2 on lung airway and vascular formation in the context of Hoxb5 and Hoxa5 expression and regulation.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"25073509\"}, \"line\": 0, \"annotations\": {\"Species\": \"10090\"}, \"source\": 2, \"target\": 9, \"key\": 2}, {\"relation\": \"directlyIncreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 5, \"key\": 0}, {\"relation\": \"directlyIncreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"exposure of EC to the angiogenic cytokine bFGF leads to increased levels of Hox D3, resulting in the functional expression of the angiogenic-promoting genes integrin Î²3 and uPA in these cells.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9314544\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"endothelial cells\", \"Species\": \"9606\"}, \"source\": 4, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"type\": \"Other\", \"reference\": \"10648567\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 6, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"type\": \"Other\", \"reference\": \"10648567\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 19, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"type\": \"Other\", \"reference\": \"10648567\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 12, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"exposure of EC to the angiogenic cytokine bFGF leads to increased levels of Hox D3, resulting in the functional expression of the angiogenic-promoting genes integrin Î²3 and uPA in these cells\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9314544\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"endothelial cells\", \"Species\": \"9606\"}, \"source\": 4, \"target\": 12, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 4, \"target\": 12, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"Stimulation of EC with basic fibroblast growth factor (bFGF) resulted in increased expression of Hox D3, integrin alphavbeta3, and the urokinase plasminogen activator (uPA). Hox D3 antisense blocked the ability of bFGF to induce uPA and integrin alphavbeta3 expression, yet had no effect on EC cell proliferation or bFGF-mediated cyclin D1 expression. Expression of Hox D3, in the absence of bFGF, resulted in enhanced expression of integrin alphavbeta3 and uPA.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"9314544\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"endothelial cells\", \"Species\": \"9606\"}, \"source\": 4, \"target\": 12, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"uPAR activation by uPA induces its association with platelet-derived growth factor receptor (PDGFR)-beta. The interaction results in PDGF-independent PDGFR-beta activation by phosphorylation of cytoplasmic tyrosine kinase domains and receptor dimerization. Association of the receptors as well as the tyrosine kinase activity of PDGFR-beta are decisive in mediating uPA-induced downstream signaling that regulates\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"15889147\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"vascular smooth muscle cells\", \"Species\": \"9606\"}, \"source\": 4, \"target\": 12, \"key\": 4}, {\"relation\": \"decreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Q-PCR analyses further revealed a downregulation of Cdknlb (p27Kipl) and Mxdl (MadI) transcript levels in Hoxb4hiPbxl(l0) primitive cells\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"17577929\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"hematopoietic stem cells\", \"Species\": \"10090\"}, \"source\": 7, \"target\": 1, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 7, \"target\": 1, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 10, \"target\": 11, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"annotations\": {\"Species\": \"9606\"}, \"source\": 10, \"target\": 23, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Here we show that HOXA3 increases endothelial cell migration, induces angiogenesis in vivo, and leads to increased expression of the matrix metalloproteinase-14 (MMP-14) and urokinase-type plasminogen activator receptor (uPAR) genes in endothelial cells in culture and in vivo in response to injury.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"15914537\"}, \"line\": 0, \"annotations\": {\"Tissue\": \"endothelial cells\", \"Species\": \"10090\"}, \"source\": 10, \"target\": 23, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"MMP-2 binds to ?v?3 at the cell surface, and blocking this binding inhibits MMP-2 protease activity in cultured cells, suppressing angiogenesis and tumour growth in chorioallantoic membrane assays43.Conversely, the membrane MMP (MT-1) cleaves ?v?3, enhancing the ligand binding activity of this integrin44. So,?v?3 seems to be important in regulating angiogenesis in adult animals.\", \"citation\": {\"name\": \"Nat Rev Cancer 2002 Feb 2(2) 83-90\", \"type\": \"Other\", \"reference\": \"12635171\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"cat\", \"namespace\": \"bel\"}}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"cat\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"Endothelium, Vascular\", \"Species\": \"9606\"}, \"source\": 11, \"target\": 26, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Kiyan, J., Kiyan, R., Haller, H., and Dumler, I. 2005. Urokinase-induced signaling in human vascular smooth muscle cells is mediated by PDGFR-beta. EMBO J. 24: 1787â1797. doi:10. 1038/sj.emboj.7600669. PMID:15889147\", \"citation\": {\"name\": \"J Lipid Res 2009 Apr 50 Suppl S189-94\", \"type\": \"Other\", \"reference\": \"19064999\"}, \"line\": 0, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"kin\", \"namespace\": \"bel\"}}, \"annotations\": {\"Tissue\": \"vascular tissue\", \"Species\": \"9606\"}, \"source\": 12, \"target\": 13, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 21, \"target\": 20, \"key\": -4}, {\"relation\": \"increases\", \"evidence\": \"Retroviral transduction with a Hoxa-9 expression vector dramatically enhanced the cytokine-driven proliferation and in vivo engraftment of Hoxa-9-/- marrow cells.\", \"citation\": {\"name\": \"Blood 2005 Dec 1 106(12) 3988-94\", \"type\": \"Other\", \"reference\": \"16091451\"}, \"line\": 0, \"annotations\": {\"Species\": \"10090\"}, \"source\": 24, \"target\": 9, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"This Network edge has no supporting evidence.  Please add real evidence to this edge prior to deleting.\", \"citation\": {\"name\": \"\", \"type\": \"Other\", \"reference\": \"0\"}, \"line\": 0, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"name\": \"tscript\", \"namespace\": \"bel\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 24, \"target\": 25, \"key\": 0}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 26, \"target\": 19, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 26, \"target\": 6, \"key\": -3}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pybel.to_database(graph)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pybel.get_ver"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Using PyBEL Functions\n",
    "\n",
    "This pipeline is implemented directly in PyBEL at [pybel.from_cbn_jgif]()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\n",
    "    os.path.join(os.environ[\"BMS_BASE\"], \"cbn\", \"Human-2.0\", \"Hox-2.0-Hs.jgf\")\n",
    ") as f:\n",
    "    graph_jgif_dict = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CPU times: user 4.73 s, sys: 69.2 ms, total: 4.79 s\n",
      "Wall time: 4.8 s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "graph = pybel.from_cbn_jgif(graph_jgif_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING:pybel.parser:Line 0000006 - VersionFormatWarning: Version string \"2.0\" neither is a date like YYYYMMDD nor adheres to semantic versioning\n",
      "ERROR:pybel.io.line_utils:Missing required document metadata: ContactInfo\n",
      "WARNING:pybel.parser:Line 0000021 - IllegalAnnotationValueWarning: \"normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000023 - IllegalAnnotationValueWarning: \"HUVECs\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000025 - MissingAnnotationKeyWarning: \"Disease\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000030 - IllegalAnnotationValueWarning: \"Cell line\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000031 - IllegalAnnotationValueWarning: \"Normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000033 - IllegalAnnotationValueWarning: \"HUVEC\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000035 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000048 - IllegalAnnotationValueWarning: \"endothelial cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000050 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000057 - IllegalAnnotationValueWarning: \"hematopoietic stem cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000059 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000064 - IllegalAnnotationValueWarning: \"Cell line\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000065 - IllegalAnnotationValueWarning: \"Normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000067 - IllegalAnnotationValueWarning: \"HUVEC\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000069 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000090 - IllegalAnnotationValueWarning: \"endothelial cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000092 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000104 - UndefinedAnnotationWarning: \"SupportingText\" is not defined\n",
      "WARNING:pybel.parser:Line 0000106 - IllegalAnnotationValueWarning: \"cell line eoma\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000107 - MissingSupportWarning: Missing evidence; can't add: p(HGNC:HOXA5) increases p(SFAM:\"AKT Family\", pmod(Ph, Ser, 473))\n",
      "WARNING:pybel.parser:Line 0000108 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000109 - MissingAnnotationKeyWarning: \"SupportingText\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000113 - IllegalAnnotationValueWarning: \"Myocytes, Smooth Muscle\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000115 - IllegalAnnotationValueWarning: \"vascular tissue\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000117 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000123 - IllegalAnnotationValueWarning: \"endothelial cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000125 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000131 - IllegalAnnotationValueWarning: \"normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000133 - IllegalAnnotationValueWarning: \"mouse embryonic fibroblasts\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000135 - MissingAnnotationKeyWarning: \"Disease\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000162 - IllegalAnnotationValueWarning: \"vascular smooth muscle cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000164 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000171 - IllegalAnnotationValueWarning: \"endothelial cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000173 - MissingAnnotationKeyWarning: \"Tissue\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000179 - IllegalAnnotationValueWarning: \"normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000181 - IllegalAnnotationValueWarning: \"vascular smooth muscle cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000183 - MissingAnnotationKeyWarning: \"Disease\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000185 - IllegalAnnotationValueWarning: \"normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000187 - IllegalAnnotationValueWarning: \"vascular smooth muscle cells\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000189 - MissingAnnotationKeyWarning: \"Disease\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000194 - IllegalAnnotationValueWarning: \"Stem Cells\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000197 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000202 - IllegalAnnotationValueWarning: \"Endothelial Cells\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000205 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000217 - IllegalAnnotationValueWarning: \"Cell line\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000218 - IllegalAnnotationValueWarning: \"Normal\" is not defined in the Disease annotation\n",
      "WARNING:pybel.parser:Line 0000220 - IllegalAnnotationValueWarning: \"HUVEC\" is not defined in the Tissue annotation\n",
      "WARNING:pybel.parser:Line 0000222 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000227 - IllegalAnnotationValueWarning: \"Endothelial Cells\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000230 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Line 0000231 - IllegalAnnotationValueWarning: \"Endothelial Cells\" is not defined in the Cell annotation\n",
      "WARNING:pybel.parser:Line 0000234 - MissingAnnotationKeyWarning: \"Cell\" is not set, so it can't be unset\n",
      "WARNING:pybel.parser:Added singleton [line 243]: p(HGNC:KDR). Putative error - needs checking.\n",
      "WARNING:pybel.parser:Added singleton [line 244]: p(HGNC:CDKN1C). Putative error - needs checking.\n",
      "WARNING:pybel.parser:Added singleton [line 245]: p(HGNC:JUN). Putative error - needs checking.\n",
      "WARNING:pybel.parser:Added singleton [line 246]: p(HGNC:CDKN1B). Putative error - needs checking.\n",
      "WARNING:pybel.parser:Added singleton [line 247]: p(HGNC:ITGA5). Putative error - needs checking.\n",
      "WARNING:pybel.parser:Added singleton [line 248]: p(HGNC:CDKN2D). Putative error - needs checking.\n"
     ]
    }
   ],
   "source": [
    "bel_lines = pybel.to_bel_lines(graph)\n",
    "\n",
    "graph_reloaded = pybel.from_lines(bel_lines)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/javascript": [
       "function init_d3_force(d3, graph, chart, width, height, function_colors) {\n",
       "    var focus_node = null;\n",
       "    var highlight_node = null;\n",
       "\n",
       "    // Highlight color variables\n",
       "\n",
       "    // Highlight color of the node boundering\n",
       "    const highlight_node_boundering = \"#4EB2D4\";\n",
       "\n",
       "    // Highlight color of the edge\n",
       "    const highlighted_link_color = \"#4EB2D4\";\n",
       "\n",
       "    // Text highlight color\n",
       "    const highlight_text = \"#4EB2D4\";\n",
       "\n",
       "    // Size when zooming scale\n",
       "    var size = d3.scalePow().exponent(1)\n",
       "        .domain([1, 100])\n",
       "        .range([8, 24]);\n",
       "\n",
       "    // Simulation parameters\n",
       "    const linkDistance = 100;\n",
       "    const fCharge = -1000;\n",
       "    const linkStrength = 0.7;\n",
       "    const collideStrength = 1;\n",
       "\n",
       "    // Simulation defined with variables\n",
       "    var simulation = d3.forceSimulation()\n",
       "        .force(\"link\", d3.forceLink()\n",
       "            .distance(linkDistance)\n",
       "            .strength(linkStrength)\n",
       "        )\n",
       "        .force(\"collide\", d3.forceCollide()\n",
       "            .radius(function (d) {\n",
       "                return d.r + 10\n",
       "            })\n",
       "            .strength(collideStrength)\n",
       "        )\n",
       "        .force(\"charge\", d3.forceManyBody()\n",
       "            .strength(fCharge)\n",
       "        )\n",
       "        .force(\"center\", d3.forceCenter(width / 2, height / 2))\n",
       "        .force(\"y\", d3.forceY(0))\n",
       "        .force(\"x\", d3.forceX(0));\n",
       "\n",
       "    // Pin down functionality\n",
       "    var node_drag = d3.drag()\n",
       "        .on(\"start\", dragstarted)\n",
       "        .on(\"drag\", dragged)\n",
       "        .on(\"end\", dragended);\n",
       "\n",
       "    function dragstarted(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0.3).restart();\n",
       "        d.fx = d.x;\n",
       "        d.fy = d.y;\n",
       "    }\n",
       "\n",
       "    function dragged(d) {\n",
       "        d.fx = d3.event.x;\n",
       "        d.fy = d3.event.y;\n",
       "    }\n",
       "\n",
       "    function dragended(d) {\n",
       "        if (!d3.event.active) simulation.alphaTarget(0);\n",
       "    }\n",
       "\n",
       "    function releasenode(d) {\n",
       "        d.fx = null;\n",
       "        d.fy = null;\n",
       "    }\n",
       "\n",
       "    //END Pin down functionality\n",
       "\n",
       "    const color_circunferencia = \"black\";\n",
       "    const default_link_color = \"#AAAAAA\";\n",
       "    const nominal_base_node_size = 8;\n",
       "\n",
       "    // Normal and highlighted stroke of the links (double the width of the link when highlighted)\n",
       "    const nominal_stroke = 1.5;\n",
       "\n",
       "    // Zoom variables\n",
       "    const min_zoom = 0.1;\n",
       "    const max_zoom = 10;\n",
       "    const border = 1;\n",
       "    const bordercolor = 'black';\n",
       "\n",
       "    var svg = d3.select(chart).append(\"svg\")\n",
       "        .attr(\"width\", width)\n",
       "        .attr(\"height\", height);\n",
       "\n",
       "    // // Create definition for arrowhead.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"arrowhead\")\n",
       "        .attr(\"viewBox\", \"0 -5 10 10\")\n",
       "        .attr(\"refX\", 20)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M0,-5L10,0L0,5\");\n",
       "\n",
       "    // // Create definition for stub.\n",
       "    svg.append(\"defs\").append(\"marker\")\n",
       "        .attr(\"id\", \"stub\")\n",
       "        .attr(\"viewBox\", \"-1 -5 2 10\")\n",
       "        .attr(\"refX\", 15)\n",
       "        .attr(\"refY\", 0)\n",
       "        .attr(\"markerUnits\", \"strokeWidth\")\n",
       "        .attr(\"markerWidth\", 6)\n",
       "        .attr(\"markerHeight\", 6)\n",
       "        .attr(\"orient\", \"auto\")\n",
       "        .append(\"path\")\n",
       "        .attr(\"d\", \"M 0,0 m -1,-5 L 1,-5 L 1,5 L -1,5 Z\");\n",
       "\n",
       "    // Background\n",
       "    svg.append(\"rect\")\n",
       "        .attr(\"width\", \"100%\")\n",
       "        .attr(\"height\", \"100%\")\n",
       "        .attr(\"fill\", \"#ffffff\")\n",
       "        .style(\"pointer-events\", \"all\")\n",
       "        // Zoom + panning functionality\n",
       "        .call(d3.zoom()\n",
       "            .scaleExtent([min_zoom, max_zoom])\n",
       "            .on(\"zoom\", zoomed))\n",
       "        .on(\"dblclick.zoom\", null);\n",
       "\n",
       "\n",
       "    function zoomed() {\n",
       "        g.attr(\"transform\", d3.event.transform);\n",
       "    }\n",
       "\n",
       "    // g = svg object where the graph will be appended\n",
       "    var g = svg.append(\"g\");\n",
       "\n",
       "    var linkedByIndex = {};\n",
       "    graph.links.forEach(function (d) {\n",
       "        linkedByIndex[d.source + \",\" + d.target] = true;\n",
       "    });\n",
       "\n",
       "    function isConnected(a, b) {\n",
       "        return linkedByIndex[a.index + \",\" + b.index] || linkedByIndex[b.index + \",\" + a.index] || a.index == b.index;\n",
       "    }\n",
       "\n",
       "    function ticked() {\n",
       "        link.attr(\"x1\", function (d) {\n",
       "            return d.source.x;\n",
       "        })\n",
       "            .attr(\"y1\", function (d) {\n",
       "                return d.source.y;\n",
       "            })\n",
       "            .attr(\"x2\", function (d) {\n",
       "                return d.target.x;\n",
       "            })\n",
       "            .attr(\"y2\", function (d) {\n",
       "                return d.target.y;\n",
       "            });\n",
       "\n",
       "        node\n",
       "            .attr(\"transform\", function (d) {\n",
       "                return \"translate(\" + d.x + \", \" + d.y + \")\";\n",
       "            });\n",
       "    }\n",
       "\n",
       "\n",
       "    simulation\n",
       "        .nodes(graph.nodes)\n",
       "        .on(\"tick\", ticked);\n",
       "\n",
       "    simulation.force(\"link\")\n",
       "        .links(graph.links);\n",
       "\n",
       "    // Definition of links nodes text...\n",
       "\n",
       "    var link = g.selectAll(\".link\")\n",
       "        .data(graph.links)\n",
       "        .enter().append(\"line\")\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", default_link_color)\n",
       "        .style(\"stroke-dasharray\", function (d) {\n",
       "            if (['decreases', 'directlyDecreases', 'increases', 'directlyIncreases', 'negativeCorrelation',\n",
       "                    'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"none\"\n",
       "            } else {\n",
       "                return \"4, 4\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-start\", function (d) {\n",
       "            if ('positiveCorrelation' == d.relation) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            }\n",
       "            else if ('negativeCorrelation' == d.relation) {\n",
       "                return \"url(#stub)\"\n",
       "            }\n",
       "            else {\n",
       "                return \"\"\n",
       "            }\n",
       "        })\n",
       "        .attr(\"marker-end\", function (d) {\n",
       "            if (['increases', 'directlyIncreases', 'positiveCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#arrowhead)\"\n",
       "            } else if (['decreases', 'directlyDecreases', 'negativeCorrelation'].indexOf(d.relation) >= 0) {\n",
       "                return \"url(#stub)\"\n",
       "            } else {\n",
       "                return \"\"\n",
       "            }\n",
       "        });\n",
       "\n",
       "    var node = g.selectAll(\".nodes\")\n",
       "        .data(graph.nodes)\n",
       "        .enter().append(\"g\")\n",
       "        .attr(\"class\", \"node\")\n",
       "        // Next two lines -> Pin down functionality\n",
       "        .on('dblclick', releasenode)\n",
       "        .call(node_drag);\n",
       "\n",
       "    var circle = node.append(\"path\")\n",
       "        .attr(\"d\", d3.symbol()\n",
       "            .size(function (d) {\n",
       "                return Math.PI * Math.pow(size(d.size) || nominal_base_node_size, 2);\n",
       "            })\n",
       "        )\n",
       "        .attr(\"class\", function (d) {\n",
       "            return d.function\n",
       "        })\n",
       "        .style('fill', function (d) {\n",
       "            return function_colors[d.function]\n",
       "        })\n",
       "        .style(\"stroke-width\", nominal_stroke)\n",
       "        .style(\"stroke\", color_circunferencia);\n",
       "\n",
       "    var text = node.append(\"text\")\n",
       "        .attr(\"class\", \"node-name\")\n",
       "        // .attr(\"id\", nodehashes[d])\n",
       "        .attr(\"fill\", \"black\")\n",
       "        .attr(\"dx\", 12)\n",
       "        .attr(\"dy\", \".35em\")\n",
       "        .text(function (d) {\n",
       "            return d.cname\n",
       "        });\n",
       "\n",
       "    // Highlight on mouseenter and back to normal on mouseout\n",
       "    node.on(\"mouseenter\", function (d) {\n",
       "        set_highlight(d);\n",
       "    })\n",
       "        .on(\"mousedown\", function () {\n",
       "            d3.event.stopPropagation();\n",
       "        }).on(\"mouseout\", function () {\n",
       "        exit_highlight();\n",
       "    });\n",
       "\n",
       "    function exit_highlight() {\n",
       "        highlight_node = null;\n",
       "        if (focus_node === null) {\n",
       "            if (highlight_node_boundering != color_circunferencia) {\n",
       "                circle.style(\"stroke\", color_circunferencia);\n",
       "                text.style(\"fill\", \"black\");\n",
       "                link.style(\"stroke\", default_link_color);\n",
       "            }\n",
       "        }\n",
       "    }\n",
       "\n",
       "    function set_highlight(d) {\n",
       "        if (focus_node !== null) d = focus_node;\n",
       "        highlight_node = d;\n",
       "\n",
       "        if (highlight_node_boundering != color_circunferencia) {\n",
       "            circle.style(\"stroke\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_node_boundering : color_circunferencia;\n",
       "            });\n",
       "            text.style(\"fill\", function (o) {\n",
       "                return isConnected(d, o) ? highlight_text : \"black\";\n",
       "            });\n",
       "            link.style(\"stroke\", function (o) {\n",
       "                return o.source.index == d.index || o.target.index == d.index ? highlighted_link_color : default_link_color;\n",
       "            });\n",
       "        }\n",
       "    }\n",
       "\n",
       "\n",
       "    // Freeze the graph when space is pressed\n",
       "    function freezeGraph() {\n",
       "        // Space button Triggers STOP\n",
       "        if (d3.event.keyCode == 32) {\n",
       "            simulation.stop();\n",
       "        }\n",
       "    }\n",
       "\n",
       "    // Call freezeGraph when a key is pressed, freezeGraph checks whether this key is \"Space\" that triggers the freeze\n",
       "    d3.select(window).on(\"keydown\", freezeGraph);\n",
       "}\n",
       "        var chart = \"ojafcuxgepqdbqhs\";\n",
       "        var width = 1000;\n",
       "        var height = 650;\n",
       "        var graph = {\"directed\": true, \"multigraph\": true, \"graph\": {\"document_metadata\": {\"authors\": \"Causal Biological Networks Database\", \"description\": \"The Hox network describes the causal mechanisms in the Hox signaling pathway that leads to cell proliferation as well as downstream targets of the HOX family members such as CDKN1A, CDKN1B, CDKN1C, ITGAV, and ITGB3.\", \"licenses\": \"               Please cite:                            - www.causalbionet.com          - https://bionet.sbvimprover.com                        as well as any relevant publications.                           The sbv IMPROVER project, the website and the Symposia are part of a collaborative project              designed to enable scientists to learn about and contribute to the development of a new crowd           sourcing method for verification of scientific data and results. The current challenges, website                and biological network models were developed and are maintained as part of a collaboration among                Selventa, OrangeBus and ADS. The project is led and funded by Philip Morris International. For more             information on the focus of Philip Morris International’s research, please visit www.pmi.com.           \", \"name\": \"Hox-2.0-Hs\", \"version\": \"2.0\"}, \"pybel_version\": \"0.7.2-dev\", \"namespace_url\": {\"GOBP\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-biological-process/go-biological-process-20150601.belns\", \"GOCC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/go-cellular-component/go-cellular-component-20170511.belns\", \"HGNC\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/hgnc-human-genes/hgnc-human-genes-20150601.belns\", \"SFAM\": \"https://arty.scai.fraunhofer.de/artifactory/bel/namespace/selventa-protein-families/selventa-protein-families-20150601.belns\"}, \"namespace_owl\": {}, \"namespace_pattern\": {}, \"annotation_url\": {\"Cell\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/cell-line/cell-line-20150601.belanno\", \"Disease\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/disease/disease-20150601.belanno\", \"Species\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/species-taxonomy-id/species-taxonomy-id-20170511.belanno\", \"Tissue\": \"https://arty.scai.fraunhofer.de/artifactory/bel/annotation/mesh-anatomy/mesh-anatomy-20150601.belanno\"}, \"annotation_owl\": {}, \"annotation_pattern\": {}, \"annotation_list\": {}}, \"nodes\": [{\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MEOX2\", \"cname\": \"MEOX2\", \"id\": [\"Protein\", \"HGNC\", \"MEOX2\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CDKN2A\", \"cname\": \"CDKN2A\", \"id\": [\"RNA\", \"HGNC\", \"CDKN2A\"]}, {\"function\": \"RNA\", \"namespace\": \"HGNC\", \"name\": \"CDKN1A\", \"cname\": \"CDKN1A\", \"id\": [\"RNA\", \"HGNC\", \"CDKN1A\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA5\", \"cname\": \"HOXA5\", \"id\": [\"Protein\", \"HGNC\", \"HOXA5\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"cell proliferation\", \"cname\": \"cell proliferation\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"cell proliferation\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXD3\", \"cname\": \"HOXD3\", \"id\": [\"Protein\", \"HGNC\", \"HOXD3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGB3\", \"cname\": \"ITGB3\", \"id\": [\"Protein\", \"HGNC\", \"ITGB3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXB4\", \"cname\": \"HOXB4\", \"id\": [\"Protein\", \"HGNC\", \"HOXB4\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1A\", \"cname\": \"CDKN1A\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1A\"]}, {\"function\": \"BiologicalProcess\", \"namespace\": \"GOBP\", \"name\": \"endothelial cell activation\", \"cname\": \"endothelial cell activation\", \"id\": [\"BiologicalProcess\", \"GOBP\", \"endothelial cell activation\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PLAU\", \"cname\": \"PLAU\", \"id\": [\"Protein\", \"HGNC\", \"PLAU\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"MMP14\", \"cname\": \"MMP14\", \"id\": [\"Protein\", \"HGNC\", \"MMP14\"]}, {\"function\": \"Complex\", \"cname\": \"complex(p(HGNC:ITGAV), p(HGNC:ITGB3))\", \"id\": [\"Complex\", [\"Protein\", \"HGNC\", \"ITGAV\"], [\"Protein\", \"HGNC\", \"ITGB3\"]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGAV\", \"cname\": \"ITGAV\", \"id\": [\"Protein\", \"HGNC\", \"ITGAV\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PDGFRB\", \"cname\": \"PDGFRB\", \"id\": [\"Protein\", \"HGNC\", \"PDGFRB\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PTGS2\", \"cname\": \"PTGS2\", \"id\": [\"Protein\", \"HGNC\", \"PTGS2\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA3\", \"cname\": \"HOXA3\", \"id\": [\"Protein\", \"HGNC\", \"HOXA3\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"PLAUR\", \"cname\": \"PLAUR\", \"id\": [\"Protein\", \"HGNC\", \"PLAUR\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"HOXA9\", \"cname\": \"HOXA9\", \"id\": [\"Protein\", \"HGNC\", \"HOXA9\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"AKT Family\", \"cname\": \"AKT Family\", \"id\": [\"Protein\", \"SFAM\", \"AKT Family\"]}, {\"function\": \"Protein\", \"namespace\": \"SFAM\", \"name\": \"AKT Family\", \"variants\": [{\"kind\": \"pmod\", \"identifier\": {\"namespace\": \"bel\", \"name\": \"Ph\"}, \"code\": \"Ser\", \"pos\": 473}], \"cname\": \"p(SFAM:\\\"AKT Family\\\", pmod(Ph, Ser, 473))\", \"id\": [\"Protein\", \"SFAM\", \"AKT Family\", [\"pmod\", [\"bel\", \"Ph\"], \"Ser\", 473]]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"KDR\", \"cname\": \"KDR\", \"id\": [\"Protein\", \"HGNC\", \"KDR\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1C\", \"cname\": \"CDKN1C\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1C\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"JUN\", \"cname\": \"JUN\", \"id\": [\"Protein\", \"HGNC\", \"JUN\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN1B\", \"cname\": \"CDKN1B\", \"id\": [\"Protein\", \"HGNC\", \"CDKN1B\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"ITGA5\", \"cname\": \"ITGA5\", \"id\": [\"Protein\", \"HGNC\", \"ITGA5\"]}, {\"function\": \"Protein\", \"namespace\": \"HGNC\", \"name\": \"CDKN2D\", \"cname\": \"CDKN2D\", \"id\": [\"Protein\", \"HGNC\", \"CDKN2D\"]}], \"links\": [{\"relation\": \"directlyIncreases\", \"evidence\": \"MEOX1 and MEOX2 activate p16(INK4a) in a DNA binding dependent manner.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"22206000\"}, \"line\": 24, \"annotations\": {\"Cell\": \"cell line\", \"Species\": \"9606\"}, \"source\": 0, \"target\": 1, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"12842453\"}, \"line\": 34, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 2, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Inhibition of endothelial cell activation by the homeobox gene Gax.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"12842453\"}, \"line\": 68, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 9, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9224717\"}, \"line\": 134, \"annotations\": {\"Cell\": \"cell line\", \"Species\": \"10090\"}, \"source\": 0, \"target\": 9, \"key\": 1}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9224717\"}, \"line\": 182, \"annotations\": {\"Cell\": \"cell line\", \"Species\": \"10116\"}, \"source\": 0, \"target\": 9, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"p21CIP1-mediated inhibition of cell proliferation by overexpression of the gax homeodomain gene\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9224717\"}, \"line\": 188, \"annotations\": {\"Species\": \"9606\", \"Cell\": \"cell line\"}, \"source\": 0, \"target\": 9, \"key\": 3}, {\"relation\": \"decreases\", \"evidence\": \"Gax strongly inhibited HUVEC proliferation and mitogen-stimulated 3H-thymidine uptake. p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold. Tube formation on Matrigel was strongly inhibited by Gax expression\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Surg Res 2003 May 1 111(1) 91-9\", \"reference\": \"12842453\"}, \"line\": 76, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Species\": \"9606\", \"Tissue\": \"Endothelium\"}, \"source\": 0, \"target\": 4, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"constitutive Gax expression inhibits cell proliferation in a p21(WAF/CIP)-dependent manner\", \"citation\": {\"type\": \"PubMed\", \"name\": \"EMBO J 1998 Jul 1 17(13) 3576-86\", \"reference\": \"9649428\"}, \"line\": 155, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Species\": \"10090\"}, \"source\": 0, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"The homeodomain transcription factor MEOX2 is an important regulator of vascular cell proliferation and is a direct transcriptional activator of both p21(CIP1/WAF1) and p16(INK4a)\", \"citation\": {\"type\": \"Other\", \"name\": \"PLoS One. 2011;6(12):e29099. doi: 10.1371/journal.pone.0029099. Epub 2011 Dec 20.\", \"reference\": \"22206000\"}, \"line\": 83, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"p21 expression in HUVECs transduced with Gax was increased up to 5-fold as measured by Northern blot, and p21 promoter activity was activated by 4- to 5-fold.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"12842453\"}, \"line\": 221, \"annotations\": {\"Species\": \"9606\"}, \"source\": 0, \"target\": 8, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Compared with wild-type samples, decreased proliferation was first observed in P0 and P5 Hoxa5–/– lungs (Figure 2, B, C, and H) . This was followed by increased proliferation at P15 and P31 (Figure 2, D–H) . This increase could not be associated to a specific cell type\", \"citation\": {\"type\": \"Other\", \"name\": \"am j pathol 2006 oct 169(4) 1312-27\", \"reference\": \"17003488\"}, \"line\": 41, \"annotations\": {\"Species\": \"10090\"}, \"source\": 3, \"target\": 4, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Hoxa5 transcription factor proteins uniquely impact lung morphogenesis at the developmental time point when extremely preterm infants are born. The effect of O2 exposure (0.4 FiO2) used in preterm infant care on these Hox proteins is unknown. We used ex vivo fetal mouse lung organ cultures to explore the effects of 0.4 FiO2 on lung airway and vascular formation in the context of Hoxb5 and Hoxa5 expression and regulation.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"25073509\"}, \"line\": 148, \"annotations\": {\"Species\": \"10090\"}, \"source\": 3, \"target\": 4, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Compared with wild-type samples, decreased proliferation was first observed in P0 and P5 Hoxa5–/– lungs (Figure 2, B, C, and H) . This was followed by increased proliferation at P15 and P31 (Figure 2, D–H) . This increase could not be associated to a specific cell type. No significant difference was noted at P60 (Figure 2H) . The changes in the temporal profile of proliferation in Hoxa5–/– lungs could explain the lower lung weight/body weight ratios observed at early stages and the compensatory effect seen later on.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Am J Pathol 2006 Oct 169(4) 1312-27\", \"reference\": \"17003488\"}, \"line\": 211, \"annotations\": {\"Species\": \"10090\"}, \"source\": 3, \"target\": 4, \"key\": 2}, {\"relation\": \"decreases\", \"evidence\": \"Sustained expression of Hox A5 leads to down regulation of many pro-angiogenic genes including VEGFR2, ephrin A1, Hif1alpha and COX-2\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"16379594\"}, \"line\": 141, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 3, \"target\": 15, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"exposure of EC to the angiogenic cytokine bFGF leads to increased levels of Hox D3, resulting in the functional expression of the angiogenic-promoting genes integrin β3 and uPA in these cells.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9314544\"}, \"line\": 49, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 6, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"reference\": \"10648567\"}, \"line\": 204, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 6, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"Stimulation of EC with basic fibroblast growth factor (bFGF) resulted in increased expression of Hox D3, integrin alphavbeta3, and the urokinase plasminogen activator (uPA). Hox D3 antisense blocked the ability of bFGF to induce uPA and integrin alphavbeta3 expression, yet had no effect on EC cell proliferation or bFGF-mediated cyclin D1 expression. Expression of Hox D3, in the absence of bFGF, resulted in enhanced expression of integrin alphavbeta3 and uPA.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9314544\"}, \"line\": 91, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 10, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"exposure of EC to the angiogenic cytokine bFGF leads to increased levels of Hox D3, resulting in the functional expression of the angiogenic-promoting genes integrin β3 and uPA in these cells\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"9314544\"}, \"line\": 124, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 10, \"key\": 1}, {\"relation\": \"increases\", \"evidence\": \"uPAR activation by uPA induces its association with platelet-derived growth factor receptor (PDGFR)-beta. The interaction results in PDGF-independent PDGFR-beta activation by phosphorylation of cytoplasmic tyrosine kinase domains and receptor dimerization. Association of the receptors as well as the tyrosine kinase activity of PDGFR-beta are decisive in mediating uPA-induced downstream signaling that regulates\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"15889147\"}, \"line\": 163, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 10, \"key\": 2}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"reference\": \"10648567\"}, \"line\": 233, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 10, \"key\": 3}, {\"relation\": \"increases\", \"evidence\": \"We have previously established that Hox D3 is required for expression of integrin alphavbeta3 and urokinase plasminogen activator (uPA), which contribute to EC adhesion, invasion, and migration during angiogenesis.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Cell Biol 2000 Jan 24 148(2) 343-51\", \"reference\": \"10648567\"}, \"line\": 229, \"annotations\": {\"Species\": \"9606\"}, \"source\": 5, \"target\": 13, \"key\": 0}, {\"relation\": \"decreases\", \"evidence\": \"Q-PCR analyses further revealed a downregulation of Cdknlb (p27Kipl) and Mxdl (MadI) transcript levels in Hoxb4hiPbxl(l0) primitive cells\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"17577929\"}, \"line\": 58, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"tscript\"}}, \"annotations\": {\"Cell\": \"cell line\", \"Species\": \"10090\"}, \"source\": 7, \"target\": 8, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Kiyan, J., Kiyan, R., Haller, H., and Dumler, I. 2005. Urokinase-induced signaling in human vascular smooth muscle cells is mediated by PDGFR-beta. EMBO J. 24: 1787–1797. doi:10. 1038/sj.emboj.7600669. PMID:15889147\", \"citation\": {\"type\": \"PubMed\", \"name\": \"J Lipid Res 2009 Apr 50 Suppl S189-94\", \"reference\": \"19064999\"}, \"line\": 116, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"kin\"}}, \"annotations\": {\"Species\": \"9606\"}, \"source\": 10, \"target\": 14, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"MMP-2 binds to ?v?3 at the cell surface, and blocking this binding inhibits MMP-2 protease activity in cultured cells, suppressing angiogenesis and tumour growth in chorioallantoic membrane assays43.Conversely, the membrane MMP (MT-1) cleaves ?v?3, enhancing the ligand binding activity of this integrin44. So,?v?3 seems to be important in regulating angiogenesis in adult animals.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Nat Rev Cancer 2002 Feb 2(2) 83-90\", \"reference\": \"12635171\"}, \"line\": 99, \"subject\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"cat\"}}, \"object\": {\"modifier\": \"Activity\", \"effect\": {\"namespace\": \"bel\", \"name\": \"cat\"}}, \"annotations\": {\"Species\": \"9606\", \"Tissue\": \"Endothelium, Vascular\"}, \"source\": 11, \"target\": 12, \"key\": 0}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 12, \"target\": 13, \"key\": -3}, {\"relation\": \"hasComponent\", \"annotations\": {}, \"source\": 12, \"target\": 6, \"key\": -3}, {\"relation\": \"increases\", \"evidence\": \"Here we show that HOXA3 increases endothelial cell migration, induces angiogenesis in vivo, and leads to increased expression of the matrix metalloproteinase-14 (MMP-14) and urokinase-type plasminogen activator receptor (uPAR) genes in endothelial cells in culture and in vivo in response to injury.\", \"citation\": {\"type\": \"Other\", \"name\": \"\", \"reference\": \"15914537\"}, \"line\": 172, \"annotations\": {\"Cell\": \"cell line\", \"Species\": \"10090\"}, \"source\": 16, \"target\": 17, \"key\": 0}, {\"relation\": \"increases\", \"evidence\": \"Retroviral transduction with a Hoxa-9 expression vector dramatically enhanced the cytokine-driven proliferation and in vivo engraftment of Hoxa-9-/- marrow cells.\", \"citation\": {\"type\": \"PubMed\", \"name\": \"Blood 2005 Dec 1 106(12) 3988-94\", \"reference\": \"16091451\"}, \"line\": 196, \"annotations\": {\"Species\": \"10090\"}, \"source\": 18, \"target\": 4, \"key\": 0}, {\"relation\": \"hasVariant\", \"annotations\": {}, \"source\": 19, \"target\": 20, \"key\": -4}]};\n",
       "        const color_map = {'Protein': '#1F77B4', 'Pathology': '#FF7F0E', 'BiologicalProcess': '#2CA02C', 'miRNA': '#D62728', 'Complex': '#9467bd', 'Composite': '#9467bd', 'Reaction': '#8c564b', 'Gene': '#e377c2', 'Abundance': '#bcbd22', 'RNA': '#17becf'};\n",
       "    \n",
       "        require.config({\n",
       "          paths: {\n",
       "              d3: '//cdnjs.cloudflare.com/ajax/libs/d3/4.5.0/d3.min'\n",
       "          }\n",
       "        });\n",
       "\n",
       "        var elementInnerHTML = \"<div id='\" + chart + \"'></div>\";\n",
       "\n",
       "        element.append(elementInnerHTML);\n",
       "\n",
       "        var chartQualified = \"#\" + chart;\n",
       "\n",
       "        require(['d3'], function(d3) {\n",
       "            return init_d3_force(d3, graph, chartQualified, width, height, color_map);\n",
       "        });\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Javascript object>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "to_jupyter(graph_reloaded)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
